
<!DOCTYPE html>

<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

    <title>Cc-by-4.0 (112) &#8212; NFDI4BioImage Training Materials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "light";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=796348d33e8b1d947c94" rel="stylesheet">
<link href="../_static/styles/bootstrap.css?digest=796348d33e8b1d947c94" rel="stylesheet">
<link href="../_static/styles/pydata-sphinx-theme.css?digest=796348d33e8b1d947c94" rel="stylesheet">

  
  <link href="../_static/vendor/fontawesome/6.1.2/css/all.min.css?digest=796348d33e8b1d947c94" rel="stylesheet">
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.woff2">
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.woff2">
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.woff2">

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" href="../_static/styles/sphinx-book-theme.css?digest=4ec06e9971c5264fbd345897d5258098f11cc577" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=796348d33e8b1d947c94">
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=796348d33e8b1d947c94">

    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/_sphinx_javascript_frameworks_compat.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/clipboard.min.js"></script>
    <script src="../_static/copybutton.js"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?digest=8bf782fb4ee92b3d3646425e50f299c4e1fd152d"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../_static/sphinx-thebe.js"></script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'licenses/cc-by-4.0';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Cc0-1.0 (5)" href="cc0-1.0.html" />
    <link rel="prev" title="Bsd-3-clause (17)" href="bsd-3-clause.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="docsearch:language" content="en">
  </head>
  
  
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="180" data-default-mode="">

  
  
  <a class="skip-link" href="#main-content">Skip to main content</a>

  
  <input type="checkbox" class="sidebar-toggle" name="__primary" id="__primary">
  <label class="overlay overlay-primary" for="__primary"></label>

  
  <input type="checkbox" class="sidebar-toggle" name="__secondary" id="__secondary">
  <label class="overlay overlay-secondary" for="__secondary"></label>

  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
      
<form class="bd-search d-flex align-items-center" action="../search.html" method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false">
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form>
    </div>
  </div>

  
  <nav class="bd-header navbar navbar-expand-lg bd-navbar" id="navbar-main"><div class="bd-header__inner bd-page-width">
  <label class="sidebar-toggle primary-toggle" for="__primary">
      <span class="fa-solid fa-bars"></span>
  </label>
  <div id="navbar-start">
    
    
  


<a class="navbar-brand logo" href="../readme.html">

  
  
  
  
  
  
  

  
    <img src="../_static/logo.png" class="logo__image only-light" alt="Logo image">
    <img src="../_static/logo.png" class="logo__image only-dark" alt="Logo image">
  
  
</a>
    
  </div>

  
  <div class="col-lg-9 navbar-header-items">
    <div id="navbar-center" class="mr-auto">
      
      <div class="navbar-center-item">
        <nav class="navbar-nav">
    <p class="sidebar-header-items__title" role="heading" aria-level="1" aria-label="Site Navigation">
        Site Navigation
    </p>
    <ul id="navbar-main-elements" class="navbar-nav">
        
                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../contributing/index.html">
                        How to contribute
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../contributing/submit_app.html">
                        Using the Training Materials Submission App
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../contributing/format.html">
                        YML format
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/artificial_intelligence.html">
                        Artificial intelligence (26)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/bio-image_analysis.html">
                        Bio-image analysis (5)
                      </a>
                    </li>
                
            <div class="nav-item dropdown">
                <button class="btn dropdown-toggle nav-item" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    More
                </button>
                <div class="dropdown-menu">
                    
                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/bioimage_analysis.html">
                        Bioimage analysis (140)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/deep_learning.html">
                        Deep learning (9)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/fair-principles.html">
                        Fair-principles (16)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/imagej.html">
                        Imagej (16)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/licensing.html">
                        Licensing (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/meta_data.html">
                        Meta data (6)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/microscopy_image_analysis.html">
                        Microscopy image analysis (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/napari.html">
                        Napari (12)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/neubias.html">
                        Neubias (26)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/nfdi4bioimage.html">
                        Nfdi4bioimage (22)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/omero.html">
                        Omero (19)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/python.html">
                        Python (65)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/research_data_management.html">
                        Research data management (77)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/sharing.html">
                        Sharing (7)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/workflow_engine.html">
                        Workflow engine (12)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/blog.html">
                        Blog (17)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/book.html">
                        Book (16)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/code.html">
                        Code (10)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/collection.html">
                        Collection (58)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/data.html">
                        Data (8)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/document.html">
                        Document (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/documentation.html">
                        Documentation (14)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/event.html">
                        Event (7)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/github%20repository.html">
                        Github repository (11)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/notebook.html">
                        Notebook (47)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/poster.html">
                        Poster (10)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/preprint.html">
                        Preprint (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/publication.html">
                        Publication (42)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/slide.html">
                        Slide (39)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/slides.html">
                        Slides (33)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/tutorial.html">
                        Tutorial (16)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/video.html">
                        Video (20)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/workshop.html">
                        Workshop (6)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="all_rights_reserved.html">
                        All rights reserved (14)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="bsd-2-clause.html">
                        Bsd-2-clause (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="bsd-3-clause.html">
                        Bsd-3-clause (17)
                      </a>
                    </li>
                

                    <li class="nav-item current active">
                      <a class="nav-link nav-internal" href="#">
                        Cc-by-4.0 (112)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="cc0-1.0.html">
                        Cc0-1.0 (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="mit.html">
                        Mit (19)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="unknown.html">
                        Unknown (59)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../statistics/readme.html">
                        Statistics
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../export/readme.html">
                        Open data
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../imprint.html">
                        Imprint
                      </a>
                    </li>
                
                </div>
            </div>
            
    </ul>
</nav>
      </div>
      
    </div>

    <div id="navbar-end">
      
        <div class="navbar-end-item navbar-persistent--container">
          
<button class="btn btn-sm navbar-btn search-button search-button__button" title="Search" aria-label="Search" data-toggle="tooltip">
  <i class="fa-solid fa-magnifying-glass"></i>
</button>
        </div>
      
      
      <div class="navbar-end-item">
        <button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="light/dark" aria-label="light/dark" data-toggle="tooltip">
    <span class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></span>
    <span class="theme-switch" data-mode="dark"><i class="fa-solid fa-moon"></i></span>
    <span class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></span>
</button>
      </div>
      
      <div class="navbar-end-item">
        <ul id="navbar-icon-links" class="navbar-nav" aria-label="Icon Links">
      </ul>
      </div>
      
    </div>
  </div>


  
  
    <div class="navbar-persistent--mobile">
<button class="btn btn-sm navbar-btn search-button search-button__button" title="Search" aria-label="Search" data-toggle="tooltip">
  <i class="fa-solid fa-magnifying-glass"></i>
</button>
    </div>
  

  
  <label class="sidebar-toggle secondary-toggle" for="__secondary">
      <span class="fa-solid fa-outdent"></span>
  </label>
  

</div>
  </nav>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      <div class="bd-sidebar-primary bd-sidebar">
        
  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
      <div class="sidebar-header-items__center">
      
      <div class="navbar-center-item">
        <nav class="navbar-nav">
    <p class="sidebar-header-items__title" role="heading" aria-level="1" aria-label="Site Navigation">
        Site Navigation
    </p>
    <ul id="navbar-main-elements" class="navbar-nav">
        
                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../contributing/index.html">
                        How to contribute
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../contributing/submit_app.html">
                        Using the Training Materials Submission App
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../contributing/format.html">
                        YML format
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/artificial_intelligence.html">
                        Artificial intelligence (26)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/bio-image_analysis.html">
                        Bio-image analysis (5)
                      </a>
                    </li>
                
            <div class="nav-item dropdown">
                <button class="btn dropdown-toggle nav-item" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    More
                </button>
                <div class="dropdown-menu">
                    
                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/bioimage_analysis.html">
                        Bioimage analysis (140)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/deep_learning.html">
                        Deep learning (9)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/fair-principles.html">
                        Fair-principles (16)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/imagej.html">
                        Imagej (16)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/licensing.html">
                        Licensing (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/meta_data.html">
                        Meta data (6)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/microscopy_image_analysis.html">
                        Microscopy image analysis (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/napari.html">
                        Napari (12)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/neubias.html">
                        Neubias (26)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/nfdi4bioimage.html">
                        Nfdi4bioimage (22)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/omero.html">
                        Omero (19)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/python.html">
                        Python (65)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/research_data_management.html">
                        Research data management (77)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/sharing.html">
                        Sharing (7)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../tags/workflow_engine.html">
                        Workflow engine (12)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/blog.html">
                        Blog (17)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/book.html">
                        Book (16)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/code.html">
                        Code (10)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/collection.html">
                        Collection (58)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/data.html">
                        Data (8)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/document.html">
                        Document (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/documentation.html">
                        Documentation (14)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/event.html">
                        Event (7)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/github%20repository.html">
                        Github repository (11)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/notebook.html">
                        Notebook (47)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/poster.html">
                        Poster (10)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/preprint.html">
                        Preprint (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/publication.html">
                        Publication (42)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/slide.html">
                        Slide (39)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/slides.html">
                        Slides (33)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/tutorial.html">
                        Tutorial (16)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/video.html">
                        Video (20)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../content_types/workshop.html">
                        Workshop (6)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="all_rights_reserved.html">
                        All rights reserved (14)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="bsd-2-clause.html">
                        Bsd-2-clause (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="bsd-3-clause.html">
                        Bsd-3-clause (17)
                      </a>
                    </li>
                

                    <li class="nav-item current active">
                      <a class="nav-link nav-internal" href="#">
                        Cc-by-4.0 (112)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="cc0-1.0.html">
                        Cc0-1.0 (5)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="mit.html">
                        Mit (19)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="unknown.html">
                        Unknown (59)
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../statistics/readme.html">
                        Statistics
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../export/readme.html">
                        Open data
                      </a>
                    </li>
                

                    <li class="nav-item">
                      <a class="nav-link nav-internal" href="../imprint.html">
                        Imprint
                      </a>
                    </li>
                
                </div>
            </div>
            
    </ul>
</nav>
      </div>
      
      </div>
    

    
    
    <div class="sidebar-header-items__end">
      
      <div class="navbar-end-item">
        <button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="light/dark" aria-label="light/dark" data-toggle="tooltip">
    <span class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></span>
    <span class="theme-switch" data-mode="dark"><i class="fa-solid fa-moon"></i></span>
    <span class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></span>
</button>
      </div>
      
      <div class="navbar-end-item">
        <ul id="navbar-icon-links" class="navbar-nav" aria-label="Icon Links">
      </ul>
      </div>
      
    </div>
    
  </div>

  
  <div class="sidebar-start-items sidebar-primary__section">
    <div class="sidebar-start-items__item">
  


<a class="navbar-brand logo" href="../readme.html">

  
  
  
  
  
  
  

  
    <img src="../_static/logo.png" class="logo__image only-light" alt="Logo image">
    <img src="../_static/logo.png" class="logo__image only-dark" alt="Logo image">
  
  
</a>
    </div>
    <div class="sidebar-start-items__item">
<form class="bd-search d-flex align-items-center" action="../search.html" method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false">
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form>
    </div>
    <div class="sidebar-start-items__item"><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../readme.html">
                    NFDI4BioImage Training Materials
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">Contributing</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../contributing/index.html">How to contribute</a></li>

<li class="toctree-l1"><a class="reference internal" href="../contributing/submit_app.html">Using the Training Materials Submission App</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contributing/format.html">YML format</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By tag</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../tags/artificial_intelligence.html">Artificial intelligence (26)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bio-image_analysis.html">Bio-image analysis (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bioimage_analysis.html">Bioimage analysis (140)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/deep_learning.html">Deep learning (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/fair-principles.html">Fair-principles (16)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/imagej.html">Imagej (16)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/licensing.html">Licensing (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/meta_data.html">Meta data (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/microscopy_image_analysis.html">Microscopy image analysis (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/napari.html">Napari (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/neubias.html">Neubias (26)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/nfdi4bioimage.html">Nfdi4bioimage (22)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/omero.html">Omero (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/python.html">Python (65)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/research_data_management.html">Research data management (77)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/sharing.html">Sharing (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/workflow_engine.html">Workflow engine (12)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By content type</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../content_types/blog.html">Blog (17)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/book.html">Book (16)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/code.html">Code (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/collection.html">Collection (58)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/data.html">Data (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/document.html">Document (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/documentation.html">Documentation (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/event.html">Event (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/github%20repository.html">Github repository (11)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/notebook.html">Notebook (47)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/poster.html">Poster (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/preprint.html">Preprint (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/publication.html">Publication (42)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/slide.html">Slide (39)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/slides.html">Slides (33)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/tutorial.html">Tutorial (16)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/video.html">Video (20)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/workshop.html">Workshop (6)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By license</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="all_rights_reserved.html">All rights reserved (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="bsd-2-clause.html">Bsd-2-clause (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="bsd-3-clause.html">Bsd-3-clause (17)</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Cc-by-4.0 (112)</a></li>
<li class="toctree-l1"><a class="reference internal" href="cc0-1.0.html">Cc0-1.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="mit.html">Mit (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="unknown.html">Unknown (59)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Appendix</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../statistics/readme.html">Statistics</a></li>
<li class="toctree-l1"><a class="reference internal" href="../export/readme.html">Open data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../imprint.html">Imprint</a></li>

</ul>

    </div>
</nav>
    </div>
  </div>
  

  
  <div class="sidebar-end-items sidebar-primary__section">
    <div class="sidebar-end-items__item">
    </div>
  </div>

  
  <div id="rtd-footer-container"></div>

      </div>
      <main id="main-content" class="bd-main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

        <div class="bd-content">
          <div class="bd-article-container">
            
            <div class="bd-header-article">
                



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        <label class="sidebar-toggle primary-toggle btn btn-sm" for="__primary" data-toggle="tooltip" data-placement="right" title="Toggle primary sidebar">
            <span class="fa-solid fa-bars"></span>
        </label>
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="btn btn-sm"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<div class="dropdown dropdown-repository-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      <li><a href="https://github.com/NFDI4BIOIMAGE/training" target="_blank"
   class="btn btn-sm dropdown-item"
   data-toggle="tooltip"
data-placement="left"
title="Source repository"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">repository</span>
</a>
</a>
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training/issues/new?title=Issue%20on%20page%20%2Flicenses/cc-by-4.0.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm dropdown-item"
   data-toggle="tooltip"
data-placement="left"
title="Open an issue"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">open issue</span>
</a>
</a>
      
  </ul>
</div>



<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      <li><a href="../_sources/licenses/cc-by-4.0.md" target="_blank"
   class="btn btn-sm dropdown-item"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</a>
      
      <li>
<button onclick="printPdf(this)"
  class="btn btn-sm dropdown-item"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</a>
      
  </ul>
</div>
<label class="sidebar-toggle secondary-toggle btn btn-sm" for="__secondary" data-toggle="tooltip" data-placement="left" title="Toggle secondary sidebar">
            <span class="fa-solid fa-list"></span>
        </label>
    </div>
</div>
            </div>
            
            

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Cc-by-4.0 (112)</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium">
   “ZENODO und Co.” Was bringt und wer braucht ein Repositorium?
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#steps-towards-reproducible-research">
   6 Steps Towards Reproducible Research
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#a-cloud-optimized-storage-for-interactive-access-of-large-arrays">
   A Cloud-Optimized Storage for Interactive Access of Large Arrays
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">
   A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe">
   A Hitchhiker’s guide through the bio-image analysis software universe
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten">
   Alles meins – oder!? Urheberrechte klären für Forschungsdaten
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bids-lecture-2024">
   BIDS-lecture-2024
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bigdataprocessor2-a-free-and-open-source-fiji-plugin-for-inspection-and-processing-of-tb-sized-image-data">
   BigDataProcessor2: A free and open-source Fiji plugin for inspection and processing of TB sized image data
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">
   Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-analysis-with-the-help-of-large-language-models">
   Bio-image Analysis with the Help of Large Language Models
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-data-science">
   Bio-image Data Science
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-data-science-lectures-uni-leipzig-scads-ai">
   Bio-image Data Science Lectures @ Uni Leipzig / ScaDS.AI
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">
   Bio-image analysis, biostatistics, programming and machine learning for computational biology
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-tools-database">
   Bio.tools database
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bioimage-analysis-notebooks">
   BioImage Analysis Notebooks
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#browsing-the-open-microscopy-image-data-resource-with-python">
   Browsing the Open Microscopy Image Data Resource with Python
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">
   Challenges and opportunities for bio-image analysis core-facilities
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#challenges-and-opportunities-for-bioimage-analysis-core-facilities">
   Challenges and opportunities for bioimage analysis core-facilities
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#chatgpt-for-image-analysis">
   ChatGPT for Image Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#collaborative-bio-image-analysis-script-editing-with-git">
   Collaborative bio-image analysis script editing with git
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#combining-the-bids-and-arc-directory-structures-for-multimodal-research-data-organization">
   Combining the BIDS and ARC Directory Structures for Multimodal Research Data Organization
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#crashkurs-forschungsdatenmanagement">
   Crashkurs Forschungsdatenmanagement
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#creating-open-computational-curricula">
   Creating open computational curricula
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#cultivating-open-training">
   Cultivating Open Training
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#cultivating-open-training-to-advance-bio-image-analysis">
   Cultivating Open Training to advance Bio-image Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#dl4miceverywhere">
   DL4MicEverywhere
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#data-carpentry-for-biologists">
   Data Carpentry for Biologists
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#datenmanagement">
   Datenmanagement
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz">
   Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-1">
   Datenmanagementpläne erstellen - Teil 1
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-2">
   Datenmanagementpläne erstellen - Teil 2
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#developing-semi-automatic-analysis-pipelines-and-technological-solutions-for-metadata-annotation-and-management-in-high-content-screening-hcs-bioimaging">
   Developing (semi)automatic analysis pipelines and technological solutions for metadata annotation and management in high-content screening (HCS) bioimaging
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#developing-open-source-software-for-bioimage-analysis-opportunities-and-challenges">
   Developing open-source software for bioimage analysis: opportunities and challenges
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats">
   EDAM-bioimaging - The ontology of bioimage informatics operations, topics, data, and formats
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats-update-2020">
   EDAM-bioimaging: The ontology of bioimage informatics operations, topics, data, and formats (update 2020)
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#efficiently-starting-institutional-research-data-management">
   Efficiently starting institutional research data management
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten">
   Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">
   Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#euro-bioimaging-s-template-for-research-data-management-plans">
   Euro-BioImaging’s Template for Research Data Management Plans
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#fair-bioimage-data">
   FAIR BioImage Data
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#fairy-deep-learning-for-bioimage-analysis">
   FAIRy deep-learning for bioImage analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#forschungsdaten-org">
   Forschungsdaten.org
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers">
   From Paper to Pixels: Navigation through your Research Data - presentations of speakers
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#generative-artificial-intelligence-for-bio-image-analysis">
   Generative artificial intelligence for bio-image analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#getting-started-with-mambaforge-and-python">
   Getting started with Mambaforge and Python
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#hackaton-results-conversion-of-knime-image-analysis-workflows-to-galaxy">
   Hackaton Results - Conversion of KNIME image analysis workflows to Galaxy
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero">
   High throughput &amp; automated data analysis and data management workflow with Cellprofiler and OMERO
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists">
   Highlights from the 2016-2020 NEUBIAS training schools for Bioimage Analysts: a success story and key asset for analysts and life scientists
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">
   I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work">
   If you license it, it’ll be harder to steal it. Why we should license our work
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#image-analysis-training-resources">
   Image Analysis Training Resources
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#image-processing-with-python">
   Image Processing with Python
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#insights-and-impact-from-five-cycles-of-essential-open-source-software-for-science">
   Insights and Impact From Five Cycles of Essential Open Source Software for Science
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#introduction-to-bioimage-analysis">
   Introduction to Bioimage Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#kollaboratives-arbeiten-und-versionskontrolle-mit-git">
   Kollaboratives Arbeiten und Versionskontrolle mit Git
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#large-language-models-an-introduction-for-life-scientists">
   Large Language Models: An Introduction for Life Scientists
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#making-your-package-available-on-conda-forge">
   Making your package available on conda-forge
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#managing-scientific-python-environments-using-conda-mamba-and-friends">
   Managing Scientific Python environments using Conda, Mamba and friends
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration">
   Meeting in the Middle: Towards Successful Multidisciplinary Bioimage Analysis Collaboration
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#methods-in-bioimage-analysis">
   Methods in bioimage analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#microsam-talks">
   MicroSam-Talks
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive">
   Microscopy data analysis: machine learning and the BioImage Archive
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud">
   My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage">
   NFDI4BIOIMAGE
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">
   NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">
   NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis">
   NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and Bioimage Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-perspective-for-a-national-bioimaging-standard">
   NFDI4BIOIMAGE: Perspective for a national bioimaging standard
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon">
   NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon)
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">
   NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nextflow-scalable-and-reproducible-scientific-workflows">
   Nextflow: Scalable and reproducible scientific workflows
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#ome-documentation">
   OME Documentation
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#ome-ngff-a-next-generation-file-format-for-expanding-bioimaging-data-access-strategies">
   OME-NGFF: a next-generation file format for expanding bioimaging data-access strategies
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#omero-tools">
   Omero-tools
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#open-image-data-handbook">
   Open Image Data Handbook
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#open-science-sharing-licensing">
   Open Science, Sharing &amp; Licensing
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing">
   Parallelization and heterogeneous computing: from pure CPU to GPU-accelerated image processing
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#photonic-data-analysis-in-2050">
   Photonic data analysis in 2050
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis">
   PoL Bio-Image Analysis Training School on GPU-Accelerated Image Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#practical-guide-to-the-international-alignment-of-research-data-management-extended-edition">
   Practical Guide to the International Alignment of Research Data Management - Extended Edition
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#qi-2024-analysis-lab-manual">
   QI 2024 Analysis Lab Manual
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#qm-course-lectures-on-bio-image-analysis-with-napari-2024">
   QM Course Lectures on Bio-Image Analysis with napari 2024
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#quarep-limi-a-community-driven-initiative-to-establish-guidelines-for-quality-assessment-and-reproducibility-for-instruments-and-images-in-light-microscopy">
   QUAREP-LiMi: A community-driven initiative to establish guidelines for quality assessment and reproducibility for instruments and images in light microscopy
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#qupath-open-source-software-for-analysing-awkward-images">
   QuPath: Open source software for analysing (awkward) images
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again">
   RDF as a bridge to domain-platforms like OMERO, or There and back again.
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#rdm4mic-presentations">
   RDM4Mic Presentations
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#rdmkit-training-resources">
   RDMKit Training Resources
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#research-data-management-seminar-slides">
   Research Data Management Seminar - Slides
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey">
   Research data management for bioimaging: the 2021 NFDI4BIOIMAGE community survey
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#running-deep-learning-scripts-in-the-bia-pol-omero-server">
   Running Deep-Learning Scripts in the BiA-PoL Omero Server
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sharing-and-licensing-material">
   Sharing and licensing material
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sharing-research-data-with-zenodo">
   Sharing research data with Zenodo
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten">
   So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#spatialdata-an-open-and-universal-data-framework-for-spatial-omics">
   SpatialData: an open and universal data framework for spatial omics
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data">
   Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sustainable-data-stewardship">
   Sustainable Data Stewardship
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#the-turing-way-guide-for-reproducible-research">
   The Turing Way: Guide for reproducible research
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#thinking-data-management-on-different-scales">
   Thinking data management on different scales
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#towards-preservation-of-life-science-data-with-nfdi4bioimage">
   Towards Preservation of Life Science Data with NFDI4BIOIMAGE
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#train-the-trainer-concept-on-research-data-management">
   Train-the-Trainer Concept on Research Data Management
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#using-glittr-org-to-find-compare-and-re-use-online-training-materials">
   Using Glittr.org to find, compare and re-use online training materials
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#welcome-to-bioimage-town">
   Welcome to BioImage Town
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#who-you-gonna-call-data-stewards-to-the-rescue">
   Who you gonna call? - Data Stewards to the rescue
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#cordi-2023-zarr-a-cloud-optimized-storage-for-interactive-access-of-large-arrays">
   [CORDI 2023] Zarr: A Cloud-Optimized Storage for Interactive Access of Large Arrays
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">
   [ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#gbi-eoe-vii-five-or-ten-must-have-items-for-making-it-infrastructure-for-managing-bioimage-data">
   [GBI EOE VII] Five (or ten) must-have items for making IT infrastructure for managing bioimage data
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#n4bi-ahm-welcome-to-bioimage-town">
   [N4BI AHM] Welcome to BioImage Town
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard">
   [SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">
   [Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero">
   [Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#ilastik-interactive-machine-learning-for-bio-image-analysis">
   ilastik: interactive machine learning for (bio)image analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#scikit-learn-mooc">
   scikit-learn MOOC
  </a>
 </li>
</ul>

            </nav>
        </div>
    </div>
</div>

            <article class="bd-article" role="main">
              
  <section class="tex2jax_ignore mathjax_ignore" id="cc-by-4-0-112">
<h1>Cc-by-4.0 (112)<a class="headerlink" href="#cc-by-4-0-112" title="Permalink to this heading">#</a></h1>
<section id="zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium">
<h2>“ZENODO und Co.” Was bringt und wer braucht ein Repositorium?<a class="headerlink" href="#zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium" title="Permalink to this heading">#</a></h2>
<p>[‘Elfi Hesse’, ‘Jan-Christoph Deinert’, ‘Christian Löschen’]</p>
<p>Published 2021-01-25</p>
<p>Licensed CC-BY-4.0</p>
<p>Die Online-Veranstaltung fand am 21.01.2021 im Rahmen der SaxFDM-Veranstaltungsreihe “Digital Kitchen - Küchengespräche mit SaxFDM” statt. SaxFDM-Sprecherin Elfi Hesse (HTW Dresden) erläuterte zunächst Grundsätzliches zum Thema Repositorien. Anschließend teilten Nutzer (Jan Deinert – HZDR) und Anbieter (Christian Löschen – TU Dresden/ZIH) lokaler Repositorien ihre Erfahrungen mit uns.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4461261">https://zenodo.org/records/4461261</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4461261">https://doi.org/10.5281/zenodo.4461261</a></p>
</section>
<hr class="docutils" />
<section id="steps-towards-reproducible-research">
<h2>6 Steps Towards Reproducible Research<a class="headerlink" href="#steps-towards-reproducible-research" title="Permalink to this heading">#</a></h2>
<p>[‘Heidi Seibold’]</p>
<p>Licensed CC-BY-4.0</p>
<p>A short book with 6 steps that get you closer to making your work reproducible.</p>
<p>Tags: Reproducibility, Research Data Management</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://zenodo.org/records/12744715">https://zenodo.org/records/12744715</a></p>
</section>
<hr class="docutils" />
<section id="a-cloud-optimized-storage-for-interactive-access-of-large-arrays">
<h2>A Cloud-Optimized Storage for Interactive Access of Large Arrays<a class="headerlink" href="#a-cloud-optimized-storage-for-interactive-access-of-large-arrays" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’, ‘Susanne Kunis’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Publication, Conference Abstract</p>
<p><a class="reference external" href="https://doi.org/10.52825/cordi.v1i.285">https://doi.org/10.52825/cordi.v1i.285</a></p>
</section>
<hr class="docutils" />
<section id="a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">
<h2>A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter<a class="headerlink" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter" title="Permalink to this heading">#</a></h2>
<p>[‘Anca Margineanu’, ‘Chiara Stringari’, ‘Marcelo Zoccoler’, ‘Cornelia Wetzker’]</p>
<p>Licensed CC-BY-4.0</p>
<p>The presentations introduce open-source software to read in, visualize and analyse fluorescence lifetime imaging microscopy (FLIM) raw data developed for life scientists. The slides were presented at German Bioimaging (GerBI) FLIM Workshop held February 26 to 29 2024 at the Biomedical Center of LMU München by Anca Margineanu, Chiara Stringari and Conni Wetzker.</p>
<p>Tags: Bioimage Analysis, Flim</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10886749">https://zenodo.org/doi/10.5281/zenodo.10886749</a></p>
</section>
<hr class="docutils" />
<section id="a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe">
<h2>A Hitchhiker’s guide through the bio-image analysis software universe<a class="headerlink" href="#a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’, ‘Elnaz Fazeli’, ‘David Legland’, ‘Michael Doube’, ‘Siân Culley’, ‘Ilya Belevich’, ‘Eija Jokitalo’, ‘Martin Schorb’, ‘Anna Klemm’, ‘Christian Tischer’]</p>
<p>Licensed CC-BY-4.0</p>
<p>This article gives an overview about commonly used bioimage analysis software and which aspects to consider when choosing a software for a specific project.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://febs.onlinelibrary.wiley.com/doi/full/10.1002/1873-3468.14451">https://febs.onlinelibrary.wiley.com/doi/full/10.1002/1873-3468.14451</a></p>
</section>
<hr class="docutils" />
<section id="alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten">
<h2>Alles meins – oder!? Urheberrechte klären für Forschungsdaten<a class="headerlink" href="#alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten" title="Permalink to this heading">#</a></h2>
<p>[‘Stephan Wünsche’]</p>
<p>Published 2024-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Wem gehören Forschungsdaten? Diese Frage stellt sich bei Daten, an deren Entstehung mehrere Personen beteiligt waren, und besonders bei Textdaten, Bildern und Videos. Hier lernen Sie, für Ihr eigenes Forschungsvorhaben zu erkennen, wessen Urheber- und Leistungsschutzrechte zu berücksichtigen sind. Sie erfahren, wie Sie mit Hilfe von Vereinbarungen frühzeitig Rechtssicherheit herstellen, etwa um Daten weitergeben oder publizieren zu können.
 
 </p>
<p>Tags: Research Data Management, Licensing</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11472148">https://zenodo.org/records/11472148</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11472148">https://doi.org/10.5281/zenodo.11472148</a></p>
</section>
<hr class="docutils" />
<section id="bids-lecture-2024">
<h2>BIDS-lecture-2024<a class="headerlink" href="#bids-lecture-2024" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python. The material developed here between April and July 2024.</p>
<p>Tags: Bioimage Analysis, Deep Learning, Microscopy Image Analysis, Python</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ScaDS/BIDS-lecture-2024/">ScaDS/BIDS-lecture-2024</a></p>
</section>
<hr class="docutils" />
<section id="bigdataprocessor2-a-free-and-open-source-fiji-plugin-for-inspection-and-processing-of-tb-sized-image-data">
<h2>BigDataProcessor2: A free and open-source Fiji plugin for inspection and processing of TB sized image data<a class="headerlink" href="#bigdataprocessor2-a-free-and-open-source-fiji-plugin-for-inspection-and-processing-of-tb-sized-image-data" title="Permalink to this heading">#</a></h2>
<p>[‘Christian Tischer’, ‘Ashis Ravindran’, ‘Sabine Reither’, ‘Nicolas Chiaruttini’, ‘Rainer Pepperkok’, ‘Nils Norlin’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1093/bioinformatics/btab106">https://doi.org/10.1093/bioinformatics/btab106</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">
<h2>Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities<a class="headerlink" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities" title="Permalink to this heading">#</a></h2>
<p>[‘Cornelia Wetzker’]</p>
<p>Published 2023-11-08</p>
<p>Licensed CC-BY-4.0</p>
<p>This presentation gives a short outline of the complexity of data and metadata in the bioimaging universe. It introduces NFDI4BIOIMAGE as a newly formed consortium as part of the German ‘Nationale Forschungsdateninfrastruktur’ (NFDI) and its goals and tools for data management including its current members on TU Dresden campus.  </p>
<p>Tags: Research Data Management, Tu Dresden, Bioimage Data, Nfdi4Bioimage</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://zenodo.org/records/10083555">https://zenodo.org/records/10083555</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10083555">https://doi.org/10.5281/zenodo.10083555</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-with-the-help-of-large-language-models">
<h2>Bio-image Analysis with the Help of Large Language Models<a class="headerlink" href="#bio-image-analysis-with-the-help-of-large-language-models" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Published 2024-03-13</p>
<p>Licensed CC-BY-4.0</p>
<p>Large Language Models (LLMs) change the way how we use computers. This also has impact on the bio-image analysis community. We can generate code that analyzes biomedical image data if we ask the right prompts. This talk outlines introduces basic principles, explains prompt engineering and how to apply it to bio-image analysis. We also compare how different LLM vendors perform on code generation tasks and which challenges are ahead for the bio-image analysis community.</p>
<p>Tags: Large Language Models, Python</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://zenodo.org/records/10815329">https://zenodo.org/records/10815329</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10815329">https://doi.org/10.5281/zenodo.10815329</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-science">
<h2>Bio-image Data Science<a class="headerlink" href="#bio-image-data-science" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository contains training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python.</p>
<p>Tags: Image Data Management, Deep Learning, Microscopy Image Analysis, Python</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/ScaDS/BIDS-lecture-2024">ScaDS/BIDS-lecture-2024</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-science-lectures-uni-leipzig-scads-ai">
<h2>Bio-image Data Science Lectures &#64; Uni Leipzig / <a class="reference external" href="http://ScaDS.AI">ScaDS.AI</a><a class="headerlink" href="#bio-image-data-science-lectures-uni-leipzig-scads-ai" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>These are the PPTx training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python. The material developed here between April and July 2024.</p>
<p>Tags: Bioimage Analysis, Deep Learning, Microscopy Image Analysis, Python</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/12623730">https://zenodo.org/records/12623730</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">
<h2>Bio-image analysis, biostatistics, programming and machine learning for computational biology<a class="headerlink" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology" title="Permalink to this heading">#</a></h2>
<p>[‘Anna Poetsch’, ‘Biotec Dresden’, ‘Marcelo Leomil Zoccoler’, ‘Johannes Richard Müller’, ‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Bioimage Analysis, Napari</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/BiAPoL/Bio-image_Analysis_with_Python">BiAPoL/Bio-image_Analysis_with_Python</a></p>
</section>
<hr class="docutils" />
<section id="bio-tools-database">
<h2>Bio.tools database<a class="headerlink" href="#bio-tools-database" title="Permalink to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioinformatics</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://bio.tools/">https://bio.tools/</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-analysis-notebooks">
<h2>BioImage Analysis Notebooks<a class="headerlink" href="#bioimage-analysis-notebooks" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase et al.’]</p>
<p>Licensed [‘CC-BY-4.0’, ‘BSD-3-CLAUSE’]</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Book, Notebook</p>
<p><a class="reference external" href="https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/intro.html">https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="browsing-the-open-microscopy-image-data-resource-with-python">
<h2>Browsing the Open Microscopy Image Data Resource with Python<a class="headerlink" href="#browsing-the-open-microscopy-image-data-resource-with-python" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Omero, Python</p>
<p>Content type: Blog</p>
<p><a class="reference external" href="https://biapol.github.io/blog/robert_haase/browsing_idr/readme.html">https://biapol.github.io/blog/robert_haase/browsing_idr/readme.html</a></p>
</section>
<hr class="docutils" />
<section id="challenges-and-opportunities-for-bio-image-analysis-core-facilities">
<h2>Challenges and opportunities for bio-image analysis core-facilities<a class="headerlink" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Bio-Image Analysis, Nfdi4Bioimage</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://f1000research.com/slides/12-1054">https://f1000research.com/slides/12-1054</a></p>
</section>
<hr class="docutils" />
<section id="challenges-and-opportunities-for-bioimage-analysis-core-facilities">
<h2>Challenges and opportunities for bioimage analysis core-facilities<a class="headerlink" href="#challenges-and-opportunities-for-bioimage-analysis-core-facilities" title="Permalink to this heading">#</a></h2>
<p>[‘Johannes Richard Soltwedel’, ‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>This article outlines common reasons for founding bioimage analysis core-facilities, services they can provide to fulfill certain need and conflicts of interest that arise from these services.</p>
<p>Tags: Bioimage Analysis, Research Data Management</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://onlinelibrary.wiley.com/doi/full/10.1111/jmi.13192">https://onlinelibrary.wiley.com/doi/full/10.1111/jmi.13192</a></p>
</section>
<hr class="docutils" />
<section id="chatgpt-for-image-analysis">
<h2>ChatGPT for Image Analysis<a class="headerlink" href="#chatgpt-for-image-analysis" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Published 2024-08-25</p>
<p>Licensed CC-BY-4.0</p>
<p>Large Language Models (LLMs) such as ChatGPT are changing the way we interact with computers, including how we analye microscopy imaging data. In this talk I introduce basic concepts of asking LLMs to write code and how to modify the questions to get the best out of it. For trying out these prompt engineering basics there are additional online resources available: <a class="reference external" href="https://scads.github.io/prompt-engineering-basics-2024/intro.html">https://scads.github.io/prompt-engineering-basics-2024/intro.html</a></p>
<p><a class="reference external" href="https://zenodo.org/records/13371196">https://zenodo.org/records/13371196</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13371196">https://doi.org/10.5281/zenodo.13371196</a></p>
</section>
<hr class="docutils" />
<section id="collaborative-bio-image-analysis-script-editing-with-git">
<h2>Collaborative bio-image analysis script editing with git<a class="headerlink" href="#collaborative-bio-image-analysis-script-editing-with-git" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to version control using git for collaborative, reproducible script editing.</p>
<p>Tags: Sharing, Research Data Management</p>
<p>Content type: Blog</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2021/09/04/collaborative-bio-image-analysis-script-editing-with-git/">https://focalplane.biologists.com/2021/09/04/collaborative-bio-image-analysis-script-editing-with-git/</a></p>
</section>
<hr class="docutils" />
<section id="combining-the-bids-and-arc-directory-structures-for-multimodal-research-data-organization">
<h2>Combining the BIDS and ARC Directory Structures for Multimodal Research Data Organization<a class="headerlink" href="#combining-the-bids-and-arc-directory-structures-for-multimodal-research-data-organization" title="Permalink to this heading">#</a></h2>
<p>[‘Torsten Stöter’, ‘Tobias Gottschall’, ‘Andrea Schrader’, ‘Peter Zentis’, ‘Monica Valencia-Schneider’, ‘Niraj Kandpal’, ‘Werner Zuschratter’, ‘Astrid Schauss’, ‘Timo Dickscheid’, ‘Timo Mühlhaus’, ‘Dirk von Suchodoletz’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Interdisciplinary collaboration and integrating large, diverse datasets are crucial for answering complex research questions, requiring multimodal data analysis and adherence to FAIR principles. To address challenges in capturing the full research cycle and contextualizing data, DataPLANT developed the Annotated Research Context (ARC), while the neuroimaging community extended the Brain Imaging Data Structure (BIDS) for microscopic image data, both providing standardized, file system-based storage structures for organizing and sharing data with metadata.</p>
<p>Tags: Research Data Management, Fair-Principles</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8349562">https://zenodo.org/doi/10.5281/zenodo.8349562</a></p>
</section>
<hr class="docutils" />
<section id="crashkurs-forschungsdatenmanagement">
<h2>Crashkurs Forschungsdatenmanagement<a class="headerlink" href="#crashkurs-forschungsdatenmanagement" title="Permalink to this heading">#</a></h2>
<p>[‘Barbara Weiner’, ‘Stephan Wünsche’, ‘Stefan Kühne’, ‘Pia Voigt’, ‘Sebastian Frericks’, ‘Clemens Hoffmann’, ‘Romy Elze’, ‘Ronny Gey’]</p>
<p>Published 2020-04-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Diese Präsentation bietet einen Einstieg in alle relevanten Bereiche des Forschungsdatenmanagements an der Universität Leipzig. Behandelt werden Grundlagen des Forschungsdatenmanagements, technische, ethische und rechtliche Aspekte sowie die Archivierung und Publikation von Forschungsdaten. Die Präsentation enthält zahlreiche weiterführende Links (rot) und Literaturhinweise.</p>
<p>Ergänzend hierzu wird eine Präsentation mit Übungsaufgaben angeboten, die helfen soll, das Gelernte zu festigen und in der eigenen Forschungspraxis umzusetzen. Den Aufgaben folgen jeweils eine Antwortfolie sowie deren Auflösung.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/3778431">https://zenodo.org/records/3778431</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.3778431">https://doi.org/10.5281/zenodo.3778431</a></p>
</section>
<hr class="docutils" />
<section id="creating-open-computational-curricula">
<h2>Creating open computational curricula<a class="headerlink" href="#creating-open-computational-curricula" title="Permalink to this heading">#</a></h2>
<p>[‘Kari Jordan’, ‘Zhian Kamvar’, ‘Toby Hodges’]</p>
<p>Published 2020-12-11</p>
<p>Licensed CC-BY-4.0</p>
<p>In this interactive session, Carpentries team members will guide attendees through three stages of the backward design process to create a lesson development plan for the open source tool of their choosing. Attendees will leave having identified what practical skills they aim to teach (learning objectives), an approach for designing challenge questions (formative assessment), and mechanisms to give and receive feedback.</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://zenodo.org/records/4317149">https://zenodo.org/records/4317149</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4317149">https://doi.org/10.5281/zenodo.4317149</a></p>
</section>
<hr class="docutils" />
<section id="cultivating-open-training">
<h2>Cultivating Open Training<a class="headerlink" href="#cultivating-open-training" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Published 2024-03-14</p>
<p>Licensed CC-BY-4.0</p>
<p>In this SaxFDM Digital Kitchen, I introduced current challenges and potential solutions for openly sharing training materials, softly focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and uploading materials to publicly acessible repositories are proposed to fill this gap.</p>
<p>Tags: Open Science, Research Data Management, Fair-Principles, Bio-Image Analysis, Licensing</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10816895">https://zenodo.org/records/10816895</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10816895">https://doi.org/10.5281/zenodo.10816895</a></p>
</section>
<hr class="docutils" />
<section id="cultivating-open-training-to-advance-bio-image-analysis">
<h2>Cultivating Open Training to advance Bio-image Analysis<a class="headerlink" href="#cultivating-open-training-to-advance-bio-image-analysis" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Published 2024-04-25</p>
<p>Licensed CC-BY-4.0</p>
<p>These slides introduce current challenges and potential solutions for openly sharing training materials, focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and publicly acessible repositories are proposed to fill this gap.</p>
<p>Tags: Research Data Management, Licensing, Fair-Principles</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11066250">https://zenodo.org/records/11066250</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11066250">https://doi.org/10.5281/zenodo.11066250</a></p>
</section>
<hr class="docutils" />
<section id="dl4miceverywhere">
<h2>DL4MicEverywhere<a class="headerlink" href="#dl4miceverywhere" title="Permalink to this heading">#</a></h2>
<p>[‘Iván Hidalgo’, ‘et al.’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Content type: Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/HenriquesLab/DL4MicEverywhere">HenriquesLab/DL4MicEverywhere</a></p>
</section>
<hr class="docutils" />
<section id="data-carpentry-for-biologists">
<h2>Data Carpentry for Biologists<a class="headerlink" href="#data-carpentry-for-biologists" title="Permalink to this heading">#</a></h2>
<p>Licensed [‘CC-BY-4.0’, ‘MIT’]</p>
<p>Content type: Tutorial, Code</p>
<p><a class="reference external" href="https://datacarpentry.org/semester-biology/">https://datacarpentry.org/semester-biology/</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagement">
<h2>Datenmanagement<a class="headerlink" href="#datenmanagement" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Published 2024-04-14</p>
<p>Licensed CC-BY-4.0</p>
<p>In dieser Data Management Session wird der Lebenszyklus von Daten näher beleuchtet. Wie entstehen Daten, was passiert mit ihnen, wenn sie verarbeitet werden? Wem gehören die Daten und wer ist dafür verantwortlich, sie zu veröffentlichen, zu archivieren und gegebenenfalls wiederzuverwenden? Wir werden einen Datenmanagementplan in Gruppenarbeit entwerfen, ggf. mit Hilfe von ChatGPT.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10970869">https://zenodo.org/records/10970869</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10970869">https://doi.org/10.5281/zenodo.10970869</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz">
<h2>Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz<a class="headerlink" href="#datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz" title="Permalink to this heading">#</a></h2>
<p>[‘Pia Voigt’, ‘Carolin Hundt’]</p>
<p>Published 2024-04-19</p>
<p>Licensed CC-BY-4.0</p>
<p>Workshop zum Thema „Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz“ auf der Data Week Leipzig
Der Umgang mit Daten ist im Alltag nicht immer leicht: Wie und wo speichert man Daten idealerweise? Welche Strategien helfen, den Überblick zu behalten und wie geht man mit personenbezogenen Daten um? Diese Fragen möchten wir gemeinsam mit Ihnen anhand individueller Datenprobleme besprechen und Ihnen Lösungen aufzeigen, wie Sie ihr Datenmanagement effizient gestalten können.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11107798">https://zenodo.org/records/11107798</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11107798">https://doi.org/10.5281/zenodo.11107798</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagementplane-erstellen-teil-1">
<h2>Datenmanagementpläne erstellen - Teil 1<a class="headerlink" href="#datenmanagementplane-erstellen-teil-1" title="Permalink to this heading">#</a></h2>
<p>[‘Pia Voigt’, ‘Barbara Weiner’]</p>
<p>Published 2021-03-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Was ist ein Datenmanagementplan? Welche Vorgaben sollte ich beachten? Wie erstelle ich einen solchen für mein Forschungsprojekt und welche nützlichen Tools kann ich hierfür verwenden?</p>
<p>Die Anforderungen der Forschungsförderer zum Datenmanagement steigen stetig. Damit verbunden ist häufig auch das Erstellen eines Datenmanagementplans. Dabei erwarten DFG, BMBF oder die EU jeweils unterschiedliche Angaben zur Erhebung, Speicherung und Veröffentlichung von projektbezogenen Forschungsdaten. Zudem bietet das Erstellen eines Datenmanagementplans viele Vorteile und hilft Ihnen nicht zuletzt, die Anforderungen der guten wissenschaftlichen Praxis strukturiert umzusetzen.</p>
<p>Was im ersten Moment unübersichtlich und überfordernd wirkt, soll in diesem Kurs anhand einer grundlegenden theoretischen Einführung im ersten und praxisorientierter Beispiele im zweiten Teil der Veranstaltung handhabbar gemacht werden. Sie lernen, was hinter den Anforderungen der Forschungsförderer steckt, welche Elemente ein Datenmanagementplan enthalten sollte und wie sie einen solchen mithilfe interaktiver Tools selbst erstellen können.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4630788">https://zenodo.org/records/4630788</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4630788">https://doi.org/10.5281/zenodo.4630788</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagementplane-erstellen-teil-2">
<h2>Datenmanagementpläne erstellen - Teil 2<a class="headerlink" href="#datenmanagementplane-erstellen-teil-2" title="Permalink to this heading">#</a></h2>
<p>[‘Pia Voigt’, ‘Barbara Weiner’]</p>
<p>Published 2021-03-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Was ist ein Datenmanagementplan? Welche Vorgaben sollte ich beachten? Wie erstelle ich einen solchen für mein Forschungsprojekt und welche nützlichen Tools kann ich hierfür verwenden?</p>
<p>Die Anforderungen der Forschungsförderer zum Datenmanagement steigen stetig. Damit verbunden ist häufig auch das Erstellen eines Datenmanagementplans. Dabei erwarten DFG, BMBF oder die EU jeweils unterschiedliche Angaben zur Erhebung, Speicherung und Veröffentlichung von projektbezogenen Forschungsdaten. Zudem bietet das Erstellen eines Datenmanagementplans viele Vorteile und hilft Ihnen nicht zuletzt, die Anforderungen der guten wissenschaftlichen Praxis strukturiert umzusetzen.</p>
<p>Was im ersten Moment unübersichtlich und überfordernd wirkt, soll in diesem Kurs anhand einer grundlegenden theoretischen Einführung im ersten und praxisorientierter Beispiele im zweiten Teil der Veranstaltung handhabbar gemacht werden. Sie lernen, was hinter den Anforderungen der Forschungsförderer steckt, welche Elemente ein Datenmanagementplan enthalten sollte und wie sie einen solchen mithilfe interaktiver Tools selbst erstellen können.</p>
<p>Version 2 enthält aktuelle Links und weiterführende Hinweise zu einzelnen Aspekten eines Datenmanagementplans.</p>
<p>Version 3 ist die überarbeitete und aktualisierte Version der ersten beiden und enthält u.a. Hinweise zur Lizenzierung und zu Nutzungsrechten an Forschungsdaten.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4748534">https://zenodo.org/records/4748534</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4748534">https://doi.org/10.5281/zenodo.4748534</a></p>
</section>
<hr class="docutils" />
<section id="developing-semi-automatic-analysis-pipelines-and-technological-solutions-for-metadata-annotation-and-management-in-high-content-screening-hcs-bioimaging">
<h2>Developing (semi)automatic analysis pipelines and technological solutions for metadata annotation and management in high-content screening (HCS) bioimaging<a class="headerlink" href="#developing-semi-automatic-analysis-pipelines-and-technological-solutions-for-metadata-annotation-and-management-in-high-content-screening-hcs-bioimaging" title="Permalink to this heading">#</a></h2>
<p>[‘Scholz Massei’, ‘Schnike Busch’, ‘Bumberger Bohring’]</p>
<p>Licensed CC-BY-4.0</p>
<p>High-content screening (HCS) bioimaging automates the imaging and analysis of numerous biological samples, generating extensive metadata that is crucial for effective image management and interpretation. Efficiently handling this complex data is essential to implementing FAIR principles and realizing HCS’s full potential for scientific discoveries.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8434325">https://doi.org/10.5281/zenodo.8434325</a></p>
</section>
<hr class="docutils" />
<section id="developing-open-source-software-for-bioimage-analysis-opportunities-and-challenges">
<h2>Developing open-source software for bioimage analysis: opportunities and challenges<a class="headerlink" href="#developing-open-source-software-for-bioimage-analysis-opportunities-and-challenges" title="Permalink to this heading">#</a></h2>
<p>[‘Florian Levet’, ‘Anne E. Carpenter’, ‘Kevin W. Eliceiri’, ‘Anna Kreshuk’, ‘Peter Bankhead’, ‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>This article outlines common challenges and practices when developing open-source software for bio-image analysis.</p>
<p>Tags: Neubias</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://f1000research.com/articles/10-302">https://f1000research.com/articles/10-302</a></p>
</section>
<hr class="docutils" />
<section id="edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats">
<h2>EDAM-bioimaging - The ontology of bioimage informatics operations, topics, data, and formats<a class="headerlink" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats" title="Permalink to this heading">#</a></h2>
<p>[‘Matúš Kalaš et al.’]</p>
<p>Licensed CC-BY-4.0</p>
<p>EDAM-bioimaging is an extension of the EDAM ontology, dedicated to bioimage analysis, bioimage informatics, and bioimaging.</p>
<p>Tags: Ontology, Bioimage Analysis</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://hal.science/hal-02267597/document">https://hal.science/hal-02267597/document</a></p>
</section>
<hr class="docutils" />
<section id="edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats-update-2020">
<h2>EDAM-bioimaging: The ontology of bioimage informatics operations, topics, data, and formats (update 2020)<a class="headerlink" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats-update-2020" title="Permalink to this heading">#</a></h2>
<p>[‘Matúš Kalaš’, ‘Laure Plantard’, ‘Joakim Lindblad’, ‘Martin Jones’, ‘Nataša Sladoje’, ‘Moritz A Kirschmann’, ‘Anatole Chessel’, ‘Leandro Scholz’, ‘Fabianne Rössler’, ‘Laura Nicolás Sáenz’, ‘Estibaliz Gómez de Mariscal’, ‘John Bogovic’, ‘Alexandre Dufour’, ‘Xavier Heiligenstein’, ‘Dominic Waithe’, ‘Marie-Charlotte Domart’, ‘Matthia Karreman’, ‘Raf Van de Plas’, ‘Robert Haase’, ‘David Hörl’, ‘Lassi Paavolainen’, ‘Ivana Vrhovac Madunić’, ‘Dean Karaica’, ‘Arrate Muñoz-Barrutia’, ‘Paula Sampaio’, ‘Daniel Sage’, ‘Sebastian Munck’, ‘Ofra Golani’, ‘Josh Moore’, ‘Florian Levet’, ‘Jon Ison’, ‘Alban Gaignard’, ‘Hervé Ménager’, ‘Chong Zhang’, ‘Kota Miura’, ‘Julien Colombelli’, ‘Perrine Paul-Gilloteaux’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Meta Data</p>
<p>Content type: Publication, Poster</p>
<p><a class="reference external" href="https://f1000research.com/posters/9-162">https://f1000research.com/posters/9-162</a></p>
</section>
<hr class="docutils" />
<section id="efficiently-starting-institutional-research-data-management">
<h2>Efficiently starting institutional research data management<a class="headerlink" href="#efficiently-starting-institutional-research-data-management" title="Permalink to this heading">#</a></h2>
<p>[‘Katarzyna Biernacka’, ‘Katrin Cortez’, ‘Kerstin Helbig’]</p>
<p>Published 2019-10-15</p>
<p>Licensed CC-BY-4.0</p>
<p>Researchers are increasingly often confronted with research data management (RDM) topics during their work. Higher education institutions therefore begin to offer services for RDM at some point to give support and advice. However, many groundbreaking decisions have to be made at the very beginning of RDM services. Priorities must be set and policies formulated. Likewise, the staff must first be qualified in order to provide advice and adequately deal with the manifold problems awaiting.
The FDMentor project has therefore bundled the expertise of five German universities with different experiences and levels of RDM knowledge to jointly develop strategies, roadmaps, guidelines, and open access training material. Humboldt-Universität zu Berlin, Freie Universität Berlin, Technische Universität Berlin, University of Potsdam, and European University Viadrina Frankfurt (Oder) have worked together on common solutions that are easy to adapt. With funding of the German Federal Ministry of Education and Research, the collaborative project addressed four problem areas: strategy development, legal issues, policy development, and competence enhancement. The aim of the project outcomes is to provide other higher education institutions with the best possible support for the efficient introduction of research data management. Therefore, all project results are freely accessible under the CC-BY 4.0 international license. The early involvement of the community in the form of workshops and the collection of feedback has proven its worth: the FDMentor strategies, roadmaps, guidelines, and training materials are applied and adapted beyond the partner universities.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Document</p>
<p><a class="reference external" href="https://zenodo.org/record/3490058">https://zenodo.org/record/3490058</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.3490058">https://doi.org/10.5281/zenodo.3490058</a></p>
</section>
<hr class="docutils" />
<section id="einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten">
<h2>Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten<a class="headerlink" href="#einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten" title="Permalink to this heading">#</a></h2>
<p>[‘Stephan Wünsche’, ‘Pia Voigt’]</p>
<p>Published 2021-05-11</p>
<p>Licensed CC-BY-4.0</p>
<p>Diese Präsentation wurde im Zuge der digitalen Veranstaltungsreihe “Einblicke ins Forschungsdatenmanagement” erstellt. Diese findet seit dem SS 2020 an der Universität Leipzig für alle Interessierten zu verschiedenen Themen des Forschungsdatenmanagements statt.</p>
<p>Dieser Teil der Reihe dreht sich um Rechtsfragen im Umgang mit Forschungsdaten und deren Bedeutung für die wissenschaftliche Praxis. Sie finden in der vorliegenden Präsentation einen Überblick über relevante Rechtsbereiche sowie Erläuterungen zum Datenschutz, Urheberrecht und den Grundsätzen der guten wissenschaftlichen Praxis mit Fokus auf deren Bedeutung im Forschungsdatenmanagement.</p>
<p>Tags: Research Data Management, Data Protection</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4748510">https://zenodo.org/records/4748510</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4748510">https://doi.org/10.5281/zenodo.4748510</a></p>
</section>
<hr class="docutils" />
<section id="euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">
<h2>Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks<a class="headerlink" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks" title="Permalink to this heading">#</a></h2>
<p>[‘Isabel Kemmer’, ‘Euro-BioImaging ERIC’]</p>
<p>Published 2024-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Hands-on exercises on FAIR Bioimage Data from the interactive online workshop “Euro-BioImaging’s Guide to FAIR BioImage Data 2024” (<a class="reference external" href="https://www.eurobioimaging.eu/news/a-guide-to-fair-bioimage-data-2024/">https://www.eurobioimaging.eu/news/a-guide-to-fair-bioimage-data-2024/</a>).  Types of tasks included: FAIR characteristics of a real world dataset Data Management Plan (DMP) Journal Policies on FAIR data sharing Ontology search Metadata according to REMBI scheme (Image from: Sarkans, U., Chiu, W., Collinson, L. et al. REMBI: Recommended Metadata for Biological Images—enabling reuse of microscopy data in biology. Nat Methods 18, 1418–1422 (2021). <a class="reference external" href="https://doi.org/10.1038/s41592-021-01166-8">https://doi.org/10.1038/s41592-021-01166-8</a>) Matching datasets to bioimage repositories Browsing bioimage repositories</p>
<p>Tags: Bioimage Analysis, Fair-Principles, Research Data Management</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="reference external" href="https://zenodo.org/records/11474407">https://zenodo.org/records/11474407</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11474407">https://doi.org/10.5281/zenodo.11474407</a></p>
</section>
<hr class="docutils" />
<section id="euro-bioimaging-s-template-for-research-data-management-plans">
<h2>Euro-BioImaging’s Template for Research Data Management Plans<a class="headerlink" href="#euro-bioimaging-s-template-for-research-data-management-plans" title="Permalink to this heading">#</a></h2>
<p>[‘Isabel Kemmer’, ‘Euro-BioImaging ERIC’]</p>
<p>Published 2024-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Euro-BioImaging has developed a Data Management Plan (DMP) template with questions tailored to bioimaging research projects. Outlining data management practices in this way ensures traceability of project data, allowing for a continuous and unambiguous flow of information throughout the research project. This template can be used to satisfy the requirement to submit a DMP to certain funders. Regardless of the funder, Euro-BioImaging users are encouraged to provide a DMP and can use this template accordingly. 
This DMP template is available as a fillable PDF with further instructions and sample responses available by hovering over the fillable fields. </p>
<p>Tags: Bioimage Analysis, Fair-Principles, Research Data Management</p>
<p>Content type: Collection, Tutorial</p>
<p><a class="reference external" href="https://zenodo.org/records/11473803">https://zenodo.org/records/11473803</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11473803">https://doi.org/10.5281/zenodo.11473803</a></p>
</section>
<hr class="docutils" />
<section id="fair-bioimage-data">
<h2>FAIR BioImage Data<a class="headerlink" href="#fair-bioimage-data" title="Permalink to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Fair, Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=8zd4KTy-oYI&amp;list=PLW-oxncaXRqU4XqduJzwFHvWLF06PvdVm">https://www.youtube.com/watch?v=8zd4KTy-oYI&amp;list=PLW-oxncaXRqU4XqduJzwFHvWLF06PvdVm</a></p>
</section>
<hr class="docutils" />
<section id="fairy-deep-learning-for-bioimage-analysis">
<h2>FAIRy deep-learning for bioImage analysis<a class="headerlink" href="#fairy-deep-learning-for-bioimage-analysis" title="Permalink to this heading">#</a></h2>
<p>[‘Estibaliz Gómez de Mariscal’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to FAIR deep learning. Furthermore, tools to deploy trained DL models (deepImageJ), easily train and evaluate them (ZeroCostDL4Mic and DeepBacs) ensure reproducibility (DL4MicEverywhere), and share this technology in an open-source and reproducible manner (BioImage Model Zoo) are introduced.</p>
<p>Tags: Deep Learning, Fair-Principles, Microscopy Image Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/13-147">https://f1000research.com/slides/13-147</a></p>
</section>
<hr class="docutils" />
<section id="forschungsdaten-org">
<h2><a class="reference external" href="http://Forschungsdaten.org">Forschungsdaten.org</a><a class="headerlink" href="#forschungsdaten-org" title="Permalink to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Research Data Management Wiki in German</p>
<p>Tags: Research Data Management</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://www.forschungsdaten.org/">https://www.forschungsdaten.org/</a></p>
</section>
<hr class="docutils" />
<section id="from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers">
<h2>From Paper to Pixels: Navigation through your Research Data - presentations of speakers<a class="headerlink" href="#from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers" title="Permalink to this heading">#</a></h2>
<p>[‘Marcelo Zoccoler’, ‘Simon Bekemeier’, ‘Tom Boissonnet’, ‘Simon Parker’, ‘Luca Bertinetti’, ‘Marc Gentzel’, ‘Riccardo Massei’, ‘Cornelia Wetzker’]</p>
<p>Published 2024-06-10</p>
<p>Licensed CC-BY-4.0</p>
<p>The workshop introduced key topics of research data management (RDM) and the implementation thereof on a life science campus. Internal and external experts of RDM including scientists that apply chosen software tools presented the basic concepts and their implementation to a broad audience. 
Talks covered general aspects of data handling and sorting, naming conventions, data storage repositories and archives, licensing of material, data and code management using git, data protection particularly regarding patient data and in genome sequencing and more. Two data management concepts and exemplary tools were highlighted in particular, being electronic lab notebooks with eLabFTW and the bio-image management software OMERO. Those were chosen because of three aspects: the large benefit of these management tools for a life science campus, their free availability as open source tools with the option of contribution of required functionalities and first existing use cases on campus already supported by CMCB/PoL IT.
Two talks by Robert Haase (<a class="reference external" href="http://ScaDS.AI/">ScaDS.AI/</a> Uni Leipzig) and Robert Müller (Kontaktstelle Forschungsdaten, TU Dresden with contributions from Denise Dörfel) that opened the symposium were shared independently:
<a class="reference external" href="https://zenodo.org/records/11382341">https://zenodo.org/records/11382341</a>
<a class="reference external" href="https://zenodo.org/records/11261115">https://zenodo.org/records/11261115</a>
The workshop organization was funded by the CMCB/PoL Networking Grant and supported by the consortium NFDI4BIOIMAGE (funded by DFG grant number NFDI 46/1, project number 501864659).</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11548617">https://zenodo.org/records/11548617</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11548617">https://doi.org/10.5281/zenodo.11548617</a></p>
</section>
<hr class="docutils" />
<section id="generative-artificial-intelligence-for-bio-image-analysis">
<h2>Generative artificial intelligence for bio-image analysis<a class="headerlink" href="#generative-artificial-intelligence-for-bio-image-analysis" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://f1000research.com/slides/12-971">https://f1000research.com/slides/12-971</a></p>
</section>
<hr class="docutils" />
<section id="getting-started-with-mambaforge-and-python">
<h2>Getting started with Mambaforge and Python<a class="headerlink" href="#getting-started-with-mambaforge-and-python" title="Permalink to this heading">#</a></h2>
<p>[‘Mara Lampert’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Conda, Mamba</p>
<p>Content type: Blog</p>
<p><a class="reference external" href="https://biapol.github.io/blog/mara_lampert/getting_started_with_mambaforge_and_python/readme.html">https://biapol.github.io/blog/mara_lampert/getting_started_with_mambaforge_and_python/readme.html</a></p>
</section>
<hr class="docutils" />
<section id="hackaton-results-conversion-of-knime-image-analysis-workflows-to-galaxy">
<h2>Hackaton Results - Conversion of KNIME image analysis workflows to Galaxy<a class="headerlink" href="#hackaton-results-conversion-of-knime-image-analysis-workflows-to-galaxy" title="Permalink to this heading">#</a></h2>
<p>[‘Riccardo Massei’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Results of the project ‘Conversion of KNIME image analysis workflows to Galaxy’ during the Hackathon ‘Image Analysis in Galaxy’ (Freiburg 26 Feb - 01 Mar 2024)</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10793699">https://zenodo.org/doi/10.5281/zenodo.10793699</a></p>
</section>
<hr class="docutils" />
<section id="high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero">
<h2>High throughput &amp; automated data analysis and data management workflow with Cellprofiler and OMERO<a class="headerlink" href="#high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero" title="Permalink to this heading">#</a></h2>
<p>[‘Sarah Weischer’, ‘Jens Wendt’, ‘Thomas Zobel’]</p>
<p>Licensed CC-BY-4.0</p>
<p>In this workshop a fully integrated data analysis solutions employing OMERO and commonly applied image analysis tools (e.g., CellProfiler, Fiji) using existing python interfaces (OMERO Python language bindings, ezOmero, Cellprofiler Python API) is presented.</p>
<p>Tags: Omero, Data Analysis, Bioimage Analysis</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8139353">https://zenodo.org/doi/10.5281/zenodo.8139353</a></p>
</section>
<hr class="docutils" />
<section id="highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists">
<h2>Highlights from the 2016-2020 NEUBIAS training schools for Bioimage Analysts: a success story and key asset for analysts and life scientists<a class="headerlink" href="#highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists" title="Permalink to this heading">#</a></h2>
<p>[‘Gabriel G. Martins’, ‘Fabrice P. Cordelières’, ‘Julien Colombelli’, ‘Rocco D’Antuono’, ‘Ofra Golani’, ‘Romain Guiet’, ‘Robert Haase’, ‘Anna H. Klemm’, ‘Marion Louveaux’, ‘Perrine Paul-Gilloteaux’, ‘Jean-Yves Tinevez’, ‘Kota Miura’]</p>
<p>Published 2021</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis, Neubias</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://f1000research.com/articles/10-334/v1">https://f1000research.com/articles/10-334/v1</a></p>
</section>
<hr class="docutils" />
<section id="i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">
<h2>I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training<a class="headerlink" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training" title="Permalink to this heading">#</a></h2>
<p>[‘Christian Schmidt’, ‘Michele Bortolomeazzi’, ‘Tom Boissonnet’, ‘Carsten Fortmann-Grote’, ‘Julia Dohle’, ‘Peter Zentis’, ‘Niraj Kandpal’, ‘Susanne Kunis’, ‘Thomas Zobel’, ‘Stefanie Weidtkamp-Peters’, ‘Elisa Ferrando-May’]</p>
<p>Published 2023-11-13</p>
<p>Licensed CC-BY-4.0</p>
<p>The open-source software OME Remote Objects (OMERO) is a data management software that allows storing, organizing, and annotating bioimaging/microscopy data. OMERO has become one of the best-known systems for bioimage data management in the bioimaging community. The Information Infrastructure for BioImage Data (I3D:bio) project facilitates the uptake of OMERO into research data management (RDM) practices at universities and research institutions in Germany. Since the adoption of OMERO into researchers’ daily routines requires intensive training, a broad portfolio of training resources for OMERO is an asset. On top of using the OMERO guides curated by the Open Microscopy Environment Consortium (OME) team, imaging core facility staff at institutions where OMERO is used often prepare additional material tailored to be applicable for their own OMERO instances. Based on experience gathered in the Research Data Management for Microscopy group (RDM4mic) in Germany, and in the use cases in the I3D:bio project, we created a set of reusable, adjustable, openly available slide decks to serve as the basis for tailored training lectures, video tutorials, and self-guided instruction manuals directed at beginners in using OMERO. The material is published as an open educational resource complementing the existing resources for OMERO contributed by the community.</p>
<p>Tags: Omero, Research Data Management, Nfdi4Bioimage, I3Dbio</p>
<p>Content type: Slide, Video</p>
<p><a class="reference external" href="https://zenodo.org/records/8323588">https://zenodo.org/records/8323588</a></p>
<p><a class="reference external" href="https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU">https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8323588">https://doi.org/10.5281/zenodo.8323588</a></p>
</section>
<hr class="docutils" />
<section id="if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work">
<h2>If you license it, it’ll be harder to steal it. Why we should license our work<a class="headerlink" href="#if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Blog post about why we should license our work and what is important when choosing a license.</p>
<p>Tags: Licensing, Research Data Management</p>
<p>Content type: Blog</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/05/06/if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work/">https://focalplane.biologists.com/2023/05/06/if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work/</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-training-resources">
<h2>Image Analysis Training Resources<a class="headerlink" href="#image-analysis-training-resources" title="Permalink to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://neubias.github.io/training-resources/">https://neubias.github.io/training-resources/</a></p>
</section>
<hr class="docutils" />
<section id="image-processing-with-python">
<h2>Image Processing with Python<a class="headerlink" href="#image-processing-with-python" title="Permalink to this heading">#</a></h2>
<p>[‘Jacob Deppen’, ‘Kimberly Meechan’, ‘David Palmquist’, ‘Ulf Schiller’, ‘Robert Turner’, ‘Marianne Corvellec’, ‘Toby Hodges’, ‘et al.’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Content type: Python, Bioimage Analysis</p>
<p><a class="reference external" href="https://datacarpentry.org/image-processing/">https://datacarpentry.org/image-processing/</a></p>
<p><a class="github reference external" href="https://github.com/datacarpentry/image-processing">datacarpentry/image-processing</a></p>
</section>
<hr class="docutils" />
<section id="insights-and-impact-from-five-cycles-of-essential-open-source-software-for-science">
<h2>Insights and Impact From Five Cycles of Essential Open Source Software for Science<a class="headerlink" href="#insights-and-impact-from-five-cycles-of-essential-open-source-software-for-science" title="Permalink to this heading">#</a></h2>
<p>[‘Kate Hertweck’, ‘Carly Strasser’, ‘Dario Taraborelli’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Open source software (OSS) is essential for advancing scientific discovery, particularly in biomedical research, yet funding to support these vital tools has been limited. The Chan Zuckerberg Initiative’s Essential Open Source Software for Science (EOSS) program has significantly contributed to this field by providing $51.8 million in funding over five years to support the maintenance, growth, and community engagement of critical OSS tools. The program has impacted scientific OSS projects by improving their technical outputs, community building, and sustainability practices, and fostering collaborations within the OSS community. Additionally, EOSS funding has enhanced diversity, equity, and inclusion within the OSS community, although changes in principal investigator demographics were not observed. The funded projects have had a substantial impact on biomedical research by improving the usability and accessibility of scientific software, which has led to increased adoption and advancements in various biomedical fields.</p>
<p>Tags: Open Source Software, Funding, Sustainability</p>
<p>Content type: Report</p>
<p><a class="reference external" href="https://zenodo.org/records/11201216">https://zenodo.org/records/11201216</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-bioimage-analysis">
<h2>Introduction to Bioimage Analysis<a class="headerlink" href="#introduction-to-bioimage-analysis" title="Permalink to this heading">#</a></h2>
<p>[‘Pete Bankhead’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Imagej, Bioimage Analysis</p>
<p>Content type: Book, Notebook</p>
<p><a class="reference external" href="https://bioimagebook.github.io/index.html">https://bioimagebook.github.io/index.html</a></p>
</section>
<hr class="docutils" />
<section id="kollaboratives-arbeiten-und-versionskontrolle-mit-git">
<h2>Kollaboratives Arbeiten und Versionskontrolle mit Git<a class="headerlink" href="#kollaboratives-arbeiten-und-versionskontrolle-mit-git" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Published 2024-04-15</p>
<p>Licensed CC-BY-4.0</p>
<p>Gemeinsames Arbeiten im Internet stellt uns vor neue Herausforderungen: Wer hat eine Datei wann hochgeladen? Wer hat zum Inhalt beigetragen? Wie kann man Inhalte zusammenfuehren, wenn mehrere Mitarbeiter gleichzeitig Aenderungen gemacht haben? Das Versionskontrollwerkzeug git stellt eine umfassende Loesung fuer solche Fragen bereit. Die Onlineplatform <a class="reference external" href="http://github.com">github.com</a> stellt nicht nur Softwareentwicklern weltweit eine git-getriebene Platform zur Verfuegung und erlaubt ihnen effektiv zusammen zu arbeiten. In diesem Workshop lernen wir:</p>
<p>Infuerung in FAIR-Prinzipien im Softwarecontext
Arbeiten mit git: Pull-requests
Aufloesen von Merge-Konflikten
Automatisiertes Archivieren von Inhalten nach <a class="reference external" href="http://Zenodo.org">Zenodo.org</a>
Eigene Webseiten auf <a class="reference external" href="http://github.io">github.io</a> publizieren</p>
<p>Tags: Research Data Management, Fair-Principles, Git, Zenodo</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10972692">https://zenodo.org/records/10972692</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10972692">https://doi.org/10.5281/zenodo.10972692</a></p>
</section>
<hr class="docutils" />
<section id="large-language-models-an-introduction-for-life-scientists">
<h2>Large Language Models: An Introduction for Life Scientists<a class="headerlink" href="#large-language-models-an-introduction-for-life-scientists" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Published 2024-08-27</p>
<p>Licensed CC-BY-4.0</p>
<p>Large Language Models (LLMs) are changing the way how humans interact with computers. This has impact on all scientific fields by enabling new ways to achieve for example data analysis goals. In this talk we will go through an introduction to LLMs with respect to applications in the life sciences, focusing on bio-image analysis. We will see how to generate text and images using LLMs and how LLMs can extract information from reproducibly images through code-generation. We will go through selected prompt engineering techniques enabling scientists to tune the output of LLMs towards their scientific goal and how to do quality assurance in this context.</p>
<p><a class="reference external" href="https://zenodo.org/records/13379394">https://zenodo.org/records/13379394</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13379394">https://doi.org/10.5281/zenodo.13379394</a></p>
</section>
<hr class="docutils" />
<section id="making-your-package-available-on-conda-forge">
<h2>Making your package available on conda-forge<a class="headerlink" href="#making-your-package-available-on-conda-forge" title="Permalink to this heading">#</a></h2>
<p>[‘Kevin Yamauchi’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Deployment, Python</p>
<p>Content type: Documentation</p>
<p><a class="reference external" href="https://kevinyamauchi.github.io/open-image-data/how_tos/conda_forge_packaging.html">https://kevinyamauchi.github.io/open-image-data/how_tos/conda_forge_packaging.html</a></p>
</section>
<hr class="docutils" />
<section id="managing-scientific-python-environments-using-conda-mamba-and-friends">
<h2>Managing Scientific Python environments using Conda, Mamba and friends<a class="headerlink" href="#managing-scientific-python-environments-using-conda-mamba-and-friends" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Conda, Mamba</p>
<p>Content type: Blog</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/">https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/</a></p>
</section>
<hr class="docutils" />
<section id="meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration">
<h2>Meeting in the Middle: Towards Successful Multidisciplinary Bioimage Analysis Collaboration<a class="headerlink" href="#meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration" title="Permalink to this heading">#</a></h2>
<p>[‘Anjalie Schlaeppi’, ‘Wilson Adams’, ‘Robert Haase’, ‘Jan Huisken’, ‘Ryan B. MacDonald’, ‘Kevin W. Eliceiri’, ‘Elisabeth C. Kugler’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.frontiersin.org/articles/10.3389/fbinf.2022.889755/full">https://www.frontiersin.org/articles/10.3389/fbinf.2022.889755/full</a></p>
</section>
<hr class="docutils" />
<section id="methods-in-bioimage-analysis">
<h2>Methods in bioimage analysis<a class="headerlink" href="#methods-in-bioimage-analysis" title="Permalink to this heading">#</a></h2>
<p>[‘Christian Tischer’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Online Tutorial, Video, Slide</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/events/methods-bioimage-analysis/">https://www.ebi.ac.uk/training/events/methods-bioimage-analysis/</a></p>
<p><a class="reference external" href="https://doi.org/10.6019/TOL.BioImageAnalysis22-w.2022.00001.1">https://doi.org/10.6019/TOL.BioImageAnalysis22-w.2022.00001.1</a></p>
<p><a class="reference external" href="https://drive.google.com/file/d/1MhuqfKhZcYu3bchWMqogIybKjamU5Msg/view">https://drive.google.com/file/d/1MhuqfKhZcYu3bchWMqogIybKjamU5Msg/view</a></p>
</section>
<hr class="docutils" />
<section id="microsam-talks">
<h2>MicroSam-Talks<a class="headerlink" href="#microsam-talks" title="Permalink to this heading">#</a></h2>
<p>[‘Constantin Pape’]</p>
<p>Published 2024-05-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Talks about Segment Anything for Microscopy: <a class="github reference external" href="https://github.com/computational-cell-analytics/micro-sam">computational-cell-analytics/micro-sam</a>.
Currently contains slides for two talks:</p>
<p>Overview of Segment Anythign for Microscopy given at the SWISSBIAS online meeting in April 2024
Talk about vision foundation models and Segment Anything for Microscopy given at Human Technopole as part of the EMBO Deep Learning Course in May 2024</p>
<p>Tags: Image Segmentation, Bio-Image Analysis, Deep Learning</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11265038">https://zenodo.org/records/11265038</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11265038">https://doi.org/10.5281/zenodo.11265038</a></p>
</section>
<hr class="docutils" />
<section id="microscopy-data-analysis-machine-learning-and-the-bioimage-archive">
<h2>Microscopy data analysis: machine learning and the BioImage Archive<a class="headerlink" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive" title="Permalink to this heading">#</a></h2>
<p>[‘Andrii Iudin’, ‘Anna Foix-Romero’, ‘Anna Kreshuk’, ‘Awais Athar’, ‘Beth Cimini’, ‘Dominik Kutra’, ‘Estibalis Gomez de Mariscal’, ‘Frances Wong’, ‘Guillaume Jacquemet’, ‘Kedar Narayan’, ‘Martin Weigert’, ‘Nodar Gogoberidze’, ‘Osman Salih’, ‘Petr Walczysko’, ‘Ryan Conrad’, ‘Simone Weyend’, ‘Sriram Sundar Somasundharam’, ‘Suganya Sivagurunathan’, ‘Ugis Sarkans’]</p>
<p>Licensed CC-BY-4.0</p>
<p>The Microscopy data analysis: machine learning and the BioImage Archive course, which focused on introducing programmatic approaches used in the analysis of bioimage data via the BioImage Archive, ran in May 2023.</p>
<p>Tags: Microscopy Image Analysis, Python, Deep Learning</p>
<p>Content type: Videos, Practicals, Slides</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/materials/microscopy-data-analysis-machine-learning-and-the-bioimage-archive-materials/">https://www.ebi.ac.uk/training/materials/microscopy-data-analysis-machine-learning-and-the-bioimage-archive-materials/</a></p>
</section>
<hr class="docutils" />
<section id="my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud">
<h2>My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud<a class="headerlink" href="#my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud" title="Permalink to this heading">#</a></h2>
<p>[‘Elnaz Fazeli’]</p>
<p>Published 2024-02-19</p>
<p>Licensed CC-BY-4.0</p>
<p>In these slides I introducemy journey through teaching bioimage analysis courses in different formats, from in person courses to online material. I have an overview of different training formats and comparing these for different audiences. </p>
<p>Tags: Teaching</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10679054">https://zenodo.org/records/10679054</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10679054">https://doi.org/10.5281/zenodo.10679054</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage">
<h2>NFDI4BIOIMAGE<a class="headerlink" href="#nfdi4bioimage" title="Permalink to this heading">#</a></h2>
<p>[‘Carsten Fortmann.Grote’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation was given at the 2nd MPG-NFDI Workshop on April 18th about e NFDI4BIOIMAGE Consortium, FAIRification of Image (meta)data, Zarr, RFC, Training (TA5), contributing.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Fair-Principles, Zarr, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.11031746">https://zenodo.org/doi/10.5281/zenodo.11031746</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023" title="Permalink to this heading">#</a></h2>
<p>[‘Stefanie Weidtkamp-Peters’]</p>
<p>Licensed CC-BY-4.0</p>
<p>NFDI4BIOIMAGE core mission, bioimage data challenge, task areas, FAIR bioimage workflows.</p>
<p>Tags: Research Data Management, Fair-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8070038">https://doi.org/10.5281/zenodo.8070038</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg" title="Permalink to this heading">#</a></h2>
<p>[‘Riccardo Massei’]</p>
<p>Licensed CC-BY-4.0</p>
<p>NFDI4BIOIMAGE is a consortium within the framework of the National Research Data Infrastructure (NFDI) in Germany. In this talk, the consortium and the contribution to the work programme by the Helmholtz Centre for Environmental Research (UFZ) in Leipzig are outlined.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8414318">https://zenodo.org/doi/10.5281/zenodo.8414318</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and Bioimage Analysis<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis" title="Permalink to this heading">#</a></h2>
<p>[‘Marcelo Zoccoler’]</p>
<p>Published 2024-08-07</p>
<p>Licensed CC-BY-4.0</p>
<p>Bioimaging refers to a collection of methods to visualize the internal structures and mechanisms of living organisms. The fundamental tool, the microscope, has enabled seminal discoveries like that of the cell as the smallest unit of life, and continues to expand our understanding of biological processes. Today, we can follow the interaction of single molecules within nanoseconds in a living cell, and the development of complete small organisms like fish and flies over several days starting from the fertilized egg. Each image pixel encodes multiple spatiotemporal and spectral dimensions, compounding the massive volume and complexity of bioimage data. Proper handling of this data is indispensable for analysis and its lack has become a growing hindrance for the many disciplines of the life and biomedical sciences relying on bioimaging. No single domain has the expertise to tackle this bottleneck alone.
As a method-specific consortium, NFDI4BIOMAGE seeks to address these issues, enabling bioimaging data to be shared and re-used like they are acquired, i.e., independently of disciplinary boundaries. We will provide solutions for exploiting the full information content of bioimage data and enable new discoveries through sharing and re-analysis. Our RDM strategy is based on a robust needs analysis that derives not only from a community survey but also from over a decade of experience in German BioImaging, the German Society for Microscopy and Image Analysis. It considers the entire lifecycle of bioimaging data, from acquisition to archiving, including analysis and enabling re-use. A foundational element of this strategy is the definition of a common, cloud-compatible, and interoperable digital object that bundles binary images with their descriptive and provenance metadata. With members from plant biology to neuroscience, NFDI4BIOIMAGE will champion the standardization of bioimage data to create a framework that answers discipline-specific needs while ensuring communication and interoperability with data types and RDM systems across domains. Integration of bioimage data with, e.g., omics data as the basis for spatial omics, holds great promise for fields such as cancer medicine. Unlocking the full potential of bioimage data will rely on the development and broad availability of exceptional analysis tools and training sets. NFDI4BIOIMAGE will make these accessible and usable including cutting-edge AI-based methods in scalable cloud environments. NFDI4BIOIMAGE intersects with multiple NFDI consortia, most prominently with GHGA for linking image and genomics data and with DataPLANT on the definition of FAIR data objects. Last but not least, NFDI4BIOIMAGE is internationally well connected and represents the opportunity for German scientists to keep path with and have a voice in several international initiatives focusing on the FAIRification of bioimage data as one of the main challenges for the advancement of knowledge in the life and biomedical sciences.</p>
<p><a class="reference external" href="https://zenodo.org/records/13168693">https://zenodo.org/records/13168693</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13168693">https://doi.org/10.5281/zenodo.13168693</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-perspective-for-a-national-bioimaging-standard">
<h2>NFDI4BIOIMAGE: Perspective for a national bioimaging standard<a class="headerlink" href="#nfdi4bioimage-perspective-for-a-national-bioimaging-standard" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’, ‘Susanne Kunis’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://ceur-ws.org/Vol-3415/paper-27.pdf">https://ceur-ws.org/Vol-3415/paper-27.pdf</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon">
<h2>NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon)<a class="headerlink" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon" title="Permalink to this heading">#</a></h2>
<p>[‘Mohamed M. Abdrabbou’, ‘Mehrnaz Babaki’, ‘Tom Boissonnet’, ‘Michele Bortolomeazzi’, ‘Eik Dahms’, ‘Vanessa A. F. Fuchs’, ‘Moritz Hoevels’, ‘Niraj Kandpal’, ‘Christoph Möhl’, ‘Joshua A. Moore’, ‘Astrid Schauss’, ‘Andrea Schrader’, ‘Torsten Stöter’, ‘Julia Thönnißen’, ‘Monica Valencia-S.’, ‘H. Lukas Weil’, ‘Jens Wendt and Peter Zentis’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Arc, Dataplant, Hackathon, Nfdi4Bioimage, Omero, Python, Research Data Management</p>
<p>Content type: Event, Publication, Documentation</p>
<p><a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/Cologne-Hackathon-2023">NFDI4BIOIMAGE/Cologne-Hackathon-2023</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10609770">https://doi.org/10.5281/zenodo.10609770</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">
<h2>NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)<a class="headerlink" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository" title="Permalink to this heading">#</a></h2>
<p>[‘Mohamed Abdrabbou’, ‘Mehrnaz Babaki’, ‘Tom Boissonnet’, ‘Michele Bortolomeazzi’, ‘Eik Dahms’, ‘Vanessa Fuchs’, ‘A. F. Moritz Hoevels’, ‘Niraj Kandpal’, ‘Christoph Möhl’, ‘Joshua A. Moore’, ‘Astrid Schauss’, ‘Andrea Schrader’, ‘Torsten Stöter’, ‘Julia Thönnißen’, ‘Monica Valencia-S.’, ‘H. Lukas Weil’, ‘Jens Wendt’, ‘Peter Zentis’]</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository documents the first NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon), where topics like ‘Interoperability’, ‘REMBI / Mapping’, and ‘Neuroglancer (OMERO / zarr)’ were explored through collaborative discussions and workflow sessions, culminating in reports that bridge NFDI4Bioimage to DataPLANT. Funded by various DFG initiatives, this event emphasized documentation and use cases, contributing preparatory work for future interoperability projects at the 2nd de.NBI BioHackathon in Bielefeld.</p>
<p>Tags: Research Data Management, Fair-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Github Repository</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10609770">https://zenodo.org/doi/10.5281/zenodo.10609770</a></p>
</section>
<hr class="docutils" />
<section id="nextflow-scalable-and-reproducible-scientific-workflows">
<h2>Nextflow: Scalable and reproducible scientific workflows<a class="headerlink" href="#nextflow-scalable-and-reproducible-scientific-workflows" title="Permalink to this heading">#</a></h2>
<p>[‘Floden Evan’, ‘Di Tommaso Paolo’]</p>
<p>Published 2020-12-17</p>
<p>Licensed CC-BY-4.0</p>
<p>Nextflow is an open-source workflow management system that prioritizes portability and reproducibility. It enables users to develop and seamlessly scale genomics workflows locally, on HPC clusters, or in major cloud providers’ infrastructures. Developed since 2014 and backed by a fast-growing community, the Nextflow ecosystem is made up of users and developers across academia, government and industry. It counts over 1M downloads and over 10K users worldwide.</p>
<p>Tags: Workflow Engine</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://zenodo.org/records/4334697">https://zenodo.org/records/4334697</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4334697">https://doi.org/10.5281/zenodo.4334697</a></p>
</section>
<hr class="docutils" />
<section id="ome-documentation">
<h2>OME Documentation<a class="headerlink" href="#ome-documentation" title="Permalink to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Omero</p>
<p>Content type: Documentation</p>
<p><a class="reference external" href="https://www.openmicroscopy.org/docs/">https://www.openmicroscopy.org/docs/</a></p>
</section>
<hr class="docutils" />
<section id="ome-ngff-a-next-generation-file-format-for-expanding-bioimaging-data-access-strategies">
<h2>OME-NGFF: a next-generation file format for expanding bioimaging data-access strategies<a class="headerlink" href="#ome-ngff-a-next-generation-file-format-for-expanding-bioimaging-data-access-strategies" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’, ‘Chris Allan’, ‘Sébastien Besson’, ‘Jean-Marie Burel’, ‘Erin Diel’, ‘David Gault’, ‘Kevin Kozlowski’, ‘Dominik Lindner’, ‘Melissa Linkert’, ‘Trevor Manz’, ‘Will Moore’, ‘Constantin Pape’, ‘Christian Tischer’, ‘Jason R. Swedlow’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.nature.com/articles/s41592-021-01326-w">https://www.nature.com/articles/s41592-021-01326-w</a></p>
</section>
<hr class="docutils" />
<section id="omero-tools">
<h2>Omero-tools<a class="headerlink" href="#omero-tools" title="Permalink to this heading">#</a></h2>
<p>[‘Johannes Soltwedel’]</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository contains a collection of tools for working with OMERO. Such tools can be working with the OMERO command line interface to transfer datasets between repositories, etc.</p>
<p>Tags: Omero, Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="reference external" href="https://biapol.github.io/omero-tools/">https://biapol.github.io/omero-tools/</a></p>
</section>
<hr class="docutils" />
<section id="open-image-data-handbook">
<h2>Open Image Data Handbook<a class="headerlink" href="#open-image-data-handbook" title="Permalink to this heading">#</a></h2>
<p>[‘Kevin Yamauchi’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Neubias, Research Data Management, Napari, Python, Bioimage Analysis</p>
<p>Content type: Book, Notebook</p>
<p><a class="reference external" href="https://kevinyamauchi.github.io/open-image-data/intro.html">https://kevinyamauchi.github.io/open-image-data/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="open-science-sharing-licensing">
<h2>Open Science, Sharing &amp; Licensing<a class="headerlink" href="#open-science-sharing-licensing" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Published 2024-04-18</p>
<p>Licensed CC-BY-4.0</p>
<p>Wir tauchen ein in die Welt der Open Science und definieren Begriffe wie Open Source, Open Access und die FAIR-Prinzipien (Findable, Accessible, Interoperable and Reuasable). Wir diskutieren, wie diese Methoden der [wissenschaftlichen] Kommunikation und des Datenmanagements die Welt verändern und wie wir sie praktisch in unsere Arbeit integrieren können. Dabei spielen Aspekte wie Copyright und Lizenzierung eine wichtige Rolle.</p>
<p>Tags: Research Data Management, Open Access, Fair-Principles, Licensing</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10990107">https://zenodo.org/records/10990107</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10990107">https://doi.org/10.5281/zenodo.10990107</a></p>
</section>
<hr class="docutils" />
<section id="parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing">
<h2>Parallelization and heterogeneous computing: from pure CPU to GPU-accelerated image processing<a class="headerlink" href="#parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://f1000research.com/slides/11-1171">https://f1000research.com/slides/11-1171</a></p>
<p><a class="reference external" href="https://doi.org/10.7490/f1000research.1119154.1">https://doi.org/10.7490/f1000research.1119154.1</a></p>
</section>
<hr class="docutils" />
<section id="photonic-data-analysis-in-2050">
<h2>Photonic data analysis in 2050<a class="headerlink" href="#photonic-data-analysis-in-2050" title="Permalink to this heading">#</a></h2>
<p>[‘Oleg Ryabchykov’, ‘Shuxia Guo’, ‘Thomas Bocklitz’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Photonic data analysis, combining imaging, spectroscopy, machine learning, and computer science, requires flexible methods and interdisciplinary collaborations to advance. Essential developments include standardizing data infrastructure for comparability, optimizing data-driven models for complex investigations, and creating techniques to handle limited or unbalanced data and device variations.</p>
<p>Tags: Fair-Principles, Machine Learning, Research Data Management</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1016/j.vibspec.2024.103685">https://doi.org/10.1016/j.vibspec.2024.103685</a></p>
</section>
<hr class="docutils" />
<section id="pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis">
<h2>PoL Bio-Image Analysis Training School on GPU-Accelerated Image Analysis<a class="headerlink" href="#pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis" title="Permalink to this heading">#</a></h2>
<p>[‘Stephane Rigaud’, ‘Brian Northan’, ‘Till Korten’, ‘Neringa Jurenaite’, ‘Apurv Deepak Kulkarni’, ‘Peter Steinbach’, ‘Sebastian Starke’, ‘Johannes Soltwedel’, ‘Marvin Albert’, ‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository hosts notebooks, information and data for the GPU-Accelerated Image Analysis Track of the PoL Bio-Image Analysis Symposium.</p>
<p>Tags: Gpu, Clesperanto, Dask, Python</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/BiAPoL/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/">BiAPoL/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis</a></p>
</section>
<hr class="docutils" />
<section id="practical-guide-to-the-international-alignment-of-research-data-management-extended-edition">
<h2>Practical Guide to the International Alignment of Research Data Management - Extended Edition<a class="headerlink" href="#practical-guide-to-the-international-alignment-of-research-data-management-extended-edition" title="Permalink to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://www.scienceeurope.org/our-resources/practical-guide-to-the-international-alignment-of-research-data-management/">https://www.scienceeurope.org/our-resources/practical-guide-to-the-international-alignment-of-research-data-management/</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4915861">https://doi.org/10.5281/zenodo.4915861</a></p>
</section>
<hr class="docutils" />
<section id="qi-2024-analysis-lab-manual">
<h2>QI 2024 Analysis Lab Manual<a class="headerlink" href="#qi-2024-analysis-lab-manual" title="Permalink to this heading">#</a></h2>
<p>[‘Beth Cimini’, ‘Florian Jug’, ‘QI 2024’]</p>
<p>Licensed CC-BY-4.0</p>
<p>This book contains the quantitative analysis labs for the QI CSHL course, 2024</p>
<p>Tags: Segmentation, Python</p>
<p>Content type: Notebook</p>
<p><a class="reference external" href="https://bethac07.github.io/qi_2024_analysis_lab_manual/intro.html">https://bethac07.github.io/qi_2024_analysis_lab_manual/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="qm-course-lectures-on-bio-image-analysis-with-napari-2024">
<h2>QM Course Lectures on Bio-Image Analysis with napari 2024<a class="headerlink" href="#qm-course-lectures-on-bio-image-analysis-with-napari-2024" title="Permalink to this heading">#</a></h2>
<p>[‘Marcelo Leomil Zoccoler’]</p>
<p>Licensed CC-BY-4.0</p>
<p>In these lectures, we will explore ways to analyze microscopy images with Python and visualize them with napari, an nD viewer open-source software. The analysis will be done in Python mostly using the scikit-image, pyclesperanto and apoc libraries, via Jupyter notebooks. We will also explore some napari plugins as an interactive and convenient alternative way of performing these analysis, especially the napari-assistant, napari-apoc and napari-flim-phasor-plotter plugins.</p>
<p>Tags: Napari, Python</p>
<p>Content type: Notebook</p>
<p><a class="reference external" href="https://zoccoler.github.io/QM_Course_Bio_Image_Analysis_with_napari_2024">https://zoccoler.github.io/QM_Course_Bio_Image_Analysis_with_napari_2024</a></p>
</section>
<hr class="docutils" />
<section id="quarep-limi-a-community-driven-initiative-to-establish-guidelines-for-quality-assessment-and-reproducibility-for-instruments-and-images-in-light-microscopy">
<h2>QUAREP-LiMi: A community-driven initiative to establish guidelines for quality assessment and reproducibility for instruments and images in light microscopy<a class="headerlink" href="#quarep-limi-a-community-driven-initiative-to-establish-guidelines-for-quality-assessment-and-reproducibility-for-instruments-and-images-in-light-microscopy" title="Permalink to this heading">#</a></h2>
<p>[‘Glyn Nelson’, ‘Ulrike Boehme’, ‘et al.’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Quareo-Limi</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://onlinelibrary.wiley.com/doi/10.1111/jmi.13041">https://onlinelibrary.wiley.com/doi/10.1111/jmi.13041</a></p>
</section>
<hr class="docutils" />
<section id="qupath-open-source-software-for-analysing-awkward-images">
<h2>QuPath: Open source software for analysing (awkward) images<a class="headerlink" href="#qupath-open-source-software-for-analysing-awkward-images" title="Permalink to this heading">#</a></h2>
<p>[‘Peter Bankhead’]</p>
<p>Published 2020-12-16</p>
<p>Licensed CC-BY-4.0</p>
<p>Slides from the CZI/EOSS online meeting in December 2020.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://zenodo.org/records/4328911">https://zenodo.org/records/4328911</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4328911">https://doi.org/10.5281/zenodo.4328911</a></p>
</section>
<hr class="docutils" />
<section id="rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again">
<h2>RDF as a bridge to domain-platforms like OMERO, or There and back again.<a class="headerlink" href="#rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’, ‘Andra Waagmeester’, ‘Kristina Hettne’, ‘Katherine Wolstencroft’, ‘Susanne Kunis’]</p>
<p>Licensed CC-BY-4.0</p>
<p>In 2005, the first version of OMERO stored RDF natively. However, just a year after the 1.0 release of RDF, performance considerations led to the development of a more traditional SQL approach for OMERO. A binary protocol makes it possible to query and retrieve metadata but the resulting information cannot immediately be combined with other sources. This is the adventure of rediscovering the benefit of RDF triples as a – if not the – common exchange mechanism.</p>
<p>Tags: Research Data Management, Fair-Principles, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10687658">https://zenodo.org/doi/10.5281/zenodo.10687658</a></p>
</section>
<hr class="docutils" />
<section id="rdm4mic-presentations">
<h2>RDM4Mic Presentations<a class="headerlink" href="#rdm4mic-presentations" title="Permalink to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management</p>
<p>Content type: Collection</p>
<p><a class="github reference external" href="https://github.com/German-BioImaging/RDM4mic/tree/master/presentations">German-BioImaging/RDM4mic</a></p>
</section>
<hr class="docutils" />
<section id="rdmkit-training-resources">
<h2>RDMKit Training Resources<a class="headerlink" href="#rdmkit-training-resources" title="Permalink to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://rdmkit.elixir-europe.org/all_training_resources">https://rdmkit.elixir-europe.org/all_training_resources</a></p>
</section>
<hr class="docutils" />
<section id="research-data-management-seminar-slides">
<h2>Research Data Management Seminar - Slides<a class="headerlink" href="#research-data-management-seminar-slides" title="Permalink to this heading">#</a></h2>
<p>[‘Della Chiesa’, ‘Stefano’]</p>
<p>Published 2022-05-18</p>
<p>Licensed CC-BY-4.0</p>
<p>This Research Data Management (RDM) Slides introduce to the multidisciplinary knowledge and competencies required to address policy compliance and research data management best practices throughout a project lifecycle, and beyond it.</p>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>Module 1 - Introduces the RDM giving its context in the Research Data Governance
Module 2 - Illustrates the most important RDM policies and principles
Module 3 - Provides the most relevant RDM knowledge bricks
Module 4 - Discuss the Data Management Plans (DMPs), examples, templates and guidance
</pre></div>
</div>
<p> </p>
<p>Tags: Research Data Management</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://zenodo.org/record/6602101">https://zenodo.org/record/6602101</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.6602101">https://doi.org/10.5281/zenodo.6602101</a></p>
</section>
<hr class="docutils" />
<section id="research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey">
<h2>Research data management for bioimaging: the 2021 NFDI4BIOIMAGE community survey<a class="headerlink" href="#research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey" title="Permalink to this heading">#</a></h2>
<p>[‘Christian Schmidt’, ‘Janina Hanne’, ‘Josh Moore’, ‘Christian Meesters’, ‘Elisa Ferrando-May’, ‘Stefanie Weidtkamp-Peters’, ‘members of the NFDI4BIOIMAGE initiative’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://f1000research.com/articles/11-638">https://f1000research.com/articles/11-638</a></p>
</section>
<hr class="docutils" />
<section id="running-deep-learning-scripts-in-the-bia-pol-omero-server">
<h2>Running Deep-Learning Scripts in the BiA-PoL Omero Server<a class="headerlink" href="#running-deep-learning-scripts-in-the-bia-pol-omero-server" title="Permalink to this heading">#</a></h2>
<p>[‘Marcelo Zoccoler’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Blog</p>
<p><a class="reference external" href="https://biapol.github.io/blog/marcelo_zoccoler/omero_scripts/readme.html">https://biapol.github.io/blog/marcelo_zoccoler/omero_scripts/readme.html</a></p>
</section>
<hr class="docutils" />
<section id="sharing-and-licensing-material">
<h2>Sharing and licensing material<a class="headerlink" href="#sharing-and-licensing-material" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to sharing resources online and licensing</p>
<p>Tags: Sharing, Research Data Management</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://f1000research.com/slides/10-519">https://f1000research.com/slides/10-519</a></p>
</section>
<hr class="docutils" />
<section id="sharing-research-data-with-zenodo">
<h2>Sharing research data with Zenodo<a class="headerlink" href="#sharing-research-data-with-zenodo" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Blog post about how to share data using <a class="reference external" href="http://zenodo.org">zenodo.org</a></p>
<p>Tags: Sharing, Research Data Management</p>
<p>Content type: Blog</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/02/15/sharing-research-data-with-zenodo/">https://focalplane.biologists.com/2023/02/15/sharing-research-data-with-zenodo/</a></p>
</section>
<hr class="docutils" />
<section id="so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten">
<h2>So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten<a class="headerlink" href="#so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten" title="Permalink to this heading">#</a></h2>
<p>[‘Pia Voigt’]</p>
<p>Published 2024-05-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Der Umgang mit personenbezogenen Daten stellt Forschende oft vor rechtliche Herausforderungen: Unter welchen Bedingungen dürfen personenbezogene Daten verarbeitet werden? Welche Voraussetzungen müssen erfüllt sein und welche Strategien können angewendet werden, um Daten sicher speichern, verarbeiten, teilen und aufbewahren zu können? Mit Hilfe dieses Foliensatzes erhalten Sie Einblicke in datenschutzrechtliche Aspekte beim Umgang mit Ihren Forschungsdaten. </p>
<p>Tags: Research Data Management, Data Protection, Fair-Principles</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11396199">https://zenodo.org/records/11396199</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11396199">https://doi.org/10.5281/zenodo.11396199</a></p>
</section>
<hr class="docutils" />
<section id="spatialdata-an-open-and-universal-data-framework-for-spatial-omics">
<h2>SpatialData: an open and universal data framework for spatial omics<a class="headerlink" href="#spatialdata-an-open-and-universal-data-framework-for-spatial-omics" title="Permalink to this heading">#</a></h2>
<p>[‘Luca Marconato’, ‘Giovanni Palla’, ‘Kevin A Yamauchi’, ‘Isaac Virshup’, ‘Elyas Heidari’, ‘Tim Treis’, ‘Marcella Toth’, ‘Rahul Shrestha’, ‘Harald Vöhringer’, ‘Wolfgang Huber’, ‘Moritz Gerstung’, ‘Josh Moore’, ‘Fabian J Theis’, ‘Oliver Stegle’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python</p>
<p>Content type: Publication, Preprint</p>
<p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2023.05.05.539647v1.abstract">https://www.biorxiv.org/content/10.1101/2023.05.05.539647v1.abstract</a></p>
</section>
<hr class="docutils" />
<section id="structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data">
<h2>Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data<a class="headerlink" href="#structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data" title="Permalink to this heading">#</a></h2>
<p>[‘Susanne Kunis’]</p>
<p>Published 2022-08-24</p>
<p>Licensed CC-BY-4.0</p>
<p>guided walkthrough of poster at <a class="reference external" href="https://doi.org/10.5281/zenodo.6821815">https://doi.org/10.5281/zenodo.6821815</a></p>
<p>which provides an overview of contexts, frameworks, and models from the world of bioimage data as well as metadata and the techniques for structuring this data as Linked Data.</p>
<p>You can also watch the video in the browser on the I3D:bio website.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Video</p>
<p><a class="reference external" href="https://zenodo.org/record/7018929">https://zenodo.org/record/7018929</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.7018929">https://doi.org/10.5281/zenodo.7018929</a></p>
</section>
<hr class="docutils" />
<section id="sustainable-data-stewardship">
<h2>Sustainable Data Stewardship<a class="headerlink" href="#sustainable-data-stewardship" title="Permalink to this heading">#</a></h2>
<p>[‘Stefano Della Chiesa’]</p>
<p>Published 2024-03-25</p>
<p>Licensed CC-BY-4.0</p>
<p>These slides were presented at the 2. SaxFDM-Beratungsstammtisch and delve into the strategic integration of Research Data Management (RDM) within research organizations. The Leibniz IOER presented an insightful overview of RDM activities and approaches, emphasizing the criticality of embedding RDM strategically within research institutions. The presentation showcases some best practices in RDM implementation through practical examples, offering valuable insights for optimizing data stewardship processes.</p>
<p>Tags: Research Data Management, Data Stewardship</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10942559">https://zenodo.org/records/10942559</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10942559">https://doi.org/10.5281/zenodo.10942559</a></p>
</section>
<hr class="docutils" />
<section id="the-turing-way-guide-for-reproducible-research">
<h2>The Turing Way: Guide for reproducible research<a class="headerlink" href="#the-turing-way-guide-for-reproducible-research" title="Permalink to this heading">#</a></h2>
<p>Licensed [‘CC-BY-4.0’, ‘MIT’]</p>
<p>A guide which covers topics related to skills, tools and best practices for research reproducibility.</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://the-turing-way.netlify.app/reproducible-research/reproducible-research">https://the-turing-way.netlify.app/reproducible-research/reproducible-research</a></p>
</section>
<hr class="docutils" />
<section id="thinking-data-management-on-different-scales">
<h2>Thinking data management on different scales<a class="headerlink" href="#thinking-data-management-on-different-scales" title="Permalink to this heading">#</a></h2>
<p>[‘Susanne Kunis’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation given at PoL BioImage Analysis Symposium Dresden 2023</p>
<p>Tags: Research Data Management, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8329305">https://zenodo.org/doi/10.5281/zenodo.8329305</a></p>
</section>
<hr class="docutils" />
<section id="towards-preservation-of-life-science-data-with-nfdi4bioimage">
<h2>Towards Preservation of Life Science Data with NFDI4BIOIMAGE<a class="headerlink" href="#towards-preservation-of-life-science-data-with-nfdi4bioimage" title="Permalink to this heading">#</a></h2>
<p>[‘Robert Haase’]</p>
<p>Published 2024-08-29</p>
<p>Licensed CC-BY-4.0</p>
<p>This talk will present the initiatives of the NFDI4BioImage consortium aimed at the long-term preservation of life science data. We will discuss our efforts to establish metadata standards, which are crucial for ensuring data reusability and integrity. The development of sustainable infrastructure is another key focus, enabling seamless data integration and analysis in the cloud. We will take a look at how we manage training materials and communicate with our community. Through these actions, NFDI4BioImage seeks to enable FAIR bioimage data management for German researchers, across disciplines and embedded in the international framework.</p>
<p><a class="reference external" href="https://zenodo.org/records/13506641">https://zenodo.org/records/13506641</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13506641">https://doi.org/10.5281/zenodo.13506641</a></p>
</section>
<hr class="docutils" />
<section id="train-the-trainer-concept-on-research-data-management">
<h2>Train-the-Trainer Concept on Research Data Management<a class="headerlink" href="#train-the-trainer-concept-on-research-data-management" title="Permalink to this heading">#</a></h2>
<p>[‘Katarzyna Biernacka’, ‘Maik Bierwirth’, ‘Petra Buchholz’, ‘Dominika Dolzycka’, ‘Kerstin Helbig’, ‘Janna Neumann’, ‘Carolin Odebrecht’, ‘Cord Wiljes’, ‘Ulrike Wuttke’]</p>
<p>Published 2020-11-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Within the project FDMentor, a German Train-the-Trainer Programme on Research Data Management (RDM) was developed and piloted in a series of workshops. The topics cover many aspects of research data management, such as data management plans and the publication of research data, as well as didactic units on learning concepts, workshop design and a range of didactic methods.</p>
<p>After the end of the project, the concept was supplemented and updated by members of the Sub-Working Group Training/Further Education (UAG Schulungen/Fortbildungen) of the DINI/nestor Working Group Research Data (DINI/nestor-AG Forschungsdaten). The newly published English version of the Train-the-Trainer Concept contains the translated concept, the materials and all methods of the Train-the-Trainer Programme. Furthermore, additional English references and materials complement this version.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://zenodo.org/record/4071471">https://zenodo.org/record/4071471</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4071471">https://doi.org/10.5281/zenodo.4071471</a></p>
</section>
<hr class="docutils" />
<section id="using-glittr-org-to-find-compare-and-re-use-online-training-materials">
<h2>Using <a class="reference external" href="http://Glittr.org">Glittr.org</a> to find, compare and re-use online training materials<a class="headerlink" href="#using-glittr-org-to-find-compare-and-re-use-online-training-materials" title="Permalink to this heading">#</a></h2>
<p>[‘Geert van Geest’, ‘Yann Haefliger’, ‘Monique Zahn-Zabal’, ‘Patricia M. Palagi’]</p>
<p>Licensed CC-BY-4.0</p>
<p><a class="reference external" href="http://Glittr.org">Glittr.org</a> is a platform that aggregates and indexes training materials on computational life sciences from public git repositories, making it easier for users to find, compare, and analyze these resources based on various metrics. By providing insights into the availability of materials, collaboration patterns, and licensing practices, <a class="reference external" href="http://Glittr.org">Glittr.org</a> supports adherence to the FAIR principles, benefiting the broader life sciences community.</p>
<p>Tags: Training, Bioimage Analysis, Research Data Management</p>
<p>Content type: Publication, Preprint</p>
<p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2024.08.20.608021v1">https://www.biorxiv.org/content/10.1101/2024.08.20.608021v1</a></p>
</section>
<hr class="docutils" />
<section id="welcome-to-bioimage-town">
<h2>Welcome to BioImage Town<a class="headerlink" href="#welcome-to-bioimage-town" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Welcome at NFDI4BIOIMAGE All-Hands Meeting in Düsseldorf, Germany, October 16, 2023</p>
<p>Tags: Omero, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10008464">https://zenodo.org/doi/10.5281/zenodo.10008464</a></p>
</section>
<hr class="docutils" />
<section id="who-you-gonna-call-data-stewards-to-the-rescue">
<h2>Who you gonna call? - Data Stewards to the rescue<a class="headerlink" href="#who-you-gonna-call-data-stewards-to-the-rescue" title="Permalink to this heading">#</a></h2>
<p>[‘Vanessa Fuchs’, ‘Aphaia Fiona’, ‘Jens Wendt’, ‘Maximilian Müller’, ‘Mohsen Ahmadi’, ‘Riccardo Massei’, ‘Cornelia Wetzker’]</p>
<p>Licensed CC-BY-4.0</p>
<p>The Data Steward Team of the NFDI4BIOIMAGE consortium presents themselves and the services (including the Helpdesk) that we offer.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Data Stewardship, Nfdi4Bioimage</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10730423">https://zenodo.org/doi/10.5281/zenodo.10730423</a></p>
</section>
<hr class="docutils" />
<section id="cordi-2023-zarr-a-cloud-optimized-storage-for-interactive-access-of-large-arrays">
<h2>[CORDI 2023] Zarr: A Cloud-Optimized Storage for Interactive Access of Large Arrays<a class="headerlink" href="#cordi-2023-zarr-a-cloud-optimized-storage-for-interactive-access-of-large-arrays" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’]</p>
<p>Licensed CC-BY-4.0</p>
<p>For decades, the sharing of large N-dimensional datasets has posed issues across multiple domains. Interactively accessing terabyte-scale data has previously required significant server resources to properly prepare cropped or down-sampled representations on the fly. Now, a cloud-native chunked format easing this burden has been adopted in the bioimaging domain for standardization. The format — Zarr — is potentially of interest for other consortia and sections of NFDI.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Data Science</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8340247">https://zenodo.org/doi/10.5281/zenodo.8340247</a></p>
</section>
<hr class="docutils" />
<section id="elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">
<h2>[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math<a class="headerlink" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’, ‘Susanne Kunis’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presented at the European Light Microscopy Initiative meeting in Liverpool (<a class="reference external" href="https://www.elmi2024.org/">https://www.elmi2024.org/</a>)</p>
<p>Tags: Research Data Management, Fair-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.11235512">https://zenodo.org/doi/10.5281/zenodo.11235512</a></p>
</section>
<hr class="docutils" />
<section id="gbi-eoe-vii-five-or-ten-must-have-items-for-making-it-infrastructure-for-managing-bioimage-data">
<h2>[GBI EOE VII] Five (or ten) must-have items for making IT infrastructure for managing bioimage data<a class="headerlink" href="#gbi-eoe-vii-five-or-ten-must-have-items-for-making-it-infrastructure-for-managing-bioimage-data" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’]</p>
<p>Published 2024-05-26</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation made to the GBI Image Data Management Working Group during the 7th Exchange of Experience in Uruguay.</p>
<p><a class="reference external" href="https://zenodo.org/records/11318151">https://zenodo.org/records/11318151</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11318151">https://doi.org/10.5281/zenodo.11318151</a></p>
</section>
<hr class="docutils" />
<section id="n4bi-ahm-welcome-to-bioimage-town">
<h2>[N4BI AHM] Welcome to BioImage Town<a class="headerlink" href="#n4bi-ahm-welcome-to-bioimage-town" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’]</p>
<p>Published 2023-10-16</p>
<p>Licensed CC-BY-4.0</p>
<p>Keynote at the NFDI4BIOIMAGE All-Hands Meeting in Düsseldorf, Germany, October 16, 2023.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10008465">https://zenodo.org/records/10008465</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10008465">https://doi.org/10.5281/zenodo.10008465</a></p>
</section>
<hr class="docutils" />
<section id="swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard">
<h2>[SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard<a class="headerlink" href="#swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard" title="Permalink to this heading">#</a></h2>
<p>[‘Josh Moore’, ‘Susanne Kunis’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presented at Semantic Web Applications and Tools for Health Care and Life Sciences (SWAT4HCLS 2023), Feb 13–16, 2023, Basel, Switzerland. NFDI4BIOIMAGE is a newly established German consortium dedicated to the FAIR representation of biological imaging data. A key deliverable is the definition of a semantically-compatible FAIR image object integrating RDF metadata with web-compatible storage of large n-dimensional binary data in OME-Zarr. We invite feedback from and collaboration with other endeavors during the soon-to-begin 5 year funding period.</p>
<p>Tags: Research Data Management, Fair-Principles, Nfdi4Bioimage</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.7928332">https://zenodo.org/doi/10.5281/zenodo.7928332</a></p>
</section>
<hr class="docutils" />
<section id="short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">
<h2>[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure<a class="headerlink" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure" title="Permalink to this heading">#</a></h2>
<p>[‘Christian Schmidt’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Short Talk about the NFDI4BIOIMAGE consortium presented at the RDM in (Bio-)Medicine Information Event on April 10th, 2024, organized C³RDM &amp; ZB MED.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10939519">https://zenodo.org/doi/10.5281/zenodo.10939519</a></p>
</section>
<hr class="docutils" />
<section id="workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero">
<h2>[Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO<a class="headerlink" href="#workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero" title="Permalink to this heading">#</a></h2>
<p>[‘Fuchs’, ‘Vanessa Aphaia Fiona’, ‘Schmidt’, ‘Christian’, ‘Boissonnet’, ‘Tom’]</p>
<p>Published 2024-05-06</p>
<p>Licensed CC-BY-4.0</p>
<p>Description
Microscopy experiments generate information-rich, multi-dimensional data, allowing us to investigate biological processes at high spatial and temporal resolution. Image processing and analysis is a standard procedure to retrieve quantitative information from biological imaging. Due to the complex nature of bioimaging files that often come in proprietary formats, it can be challenging to organize, structure, and annotate bioimaging data throughout a project. Data often needs to be moved between collaboration partners, transformed into open formats, processed with a variety of software tools, and exported to smaller-sized images for presentation. The path from image acquisition to final publication figures with quantitative results must be documented and reproducible.
In this workshop, participants learn how to use structured metadata annotations in the image data management platform OMERO (OME Remote Objects) to optimize their data handling. This strategy helps both with organizing data for easier processing and analysis and for the preparation of data publication in journal manuscripts and in public repositories such as the BioImage Archive. Participants learn the principles of leveraging object-oriented data organization in OMERO to enhance findability and usability of their data, also in collaborative settings. The integration of OMERO with image analysis tools, in particular ImageJ/Fiji, will be trained. Moreover, users learn about community-accepted metadata checklists (REMBI) to enrich the value of their data toward reproducibility and reusability. In this workshop, we will provide hands-on training and recommendations on:</p>
<p>Structured metadata annotation features in OMERO and how to use them
Types of metadata in bioimaging: Technical metadata, sample metadata, analysis metadata
The use of ontologies and terminologies for metadata annotation
REMBI, the recommended metadata for biological images
Metadata-assisted image analysis streamlining
Tools for metadata annotation in OMERO</p>
<p>The target group for this workshop
This workshop is directed at researchers at all career levels who have started using OMERO for their microscopy research data management. We encourage the workshop participants to bring example data from their research to discuss suitable metadata annotation for their everyday practice.
Who are the trainers (see trainer description below for more details)</p>
<p>Dr. Vanessa Fuchs (NFDI4BIOIMAGE Data Steward, Center for Advanced Imaging, Heinrich-Heine University of Düsseldorf)
Dr. Tom Boissonnet (OMERO admin and image metadata specialist, Center for Advanced Imaging, Heinrich-Heine University of Düsseldorf)
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center, Heidelberg)</p>
<p>Material Description
Published here are the presentation slides that were used for input from the trainers during the different sessions of the programme. Additionally, a Fiji Macro is published that depends on the OMERO Extensions Plugin by Pouchin et al, 2022, F100Research, <a class="reference external" href="https://doi.org/10.12688/f1000research.110385.2">https://doi.org/10.12688/f1000research.110385.2</a> 
Programme Overview
Day 1 - April 29th, 2024 09.00 a.m. to 10.00 a.m.: Session 1 - Welcome and Introduction
10.00 a.m. to 10.30 a.m.:  Session 2 - Introduction to the FAIR principles &amp; data annotation
10:30 a.m. to 10:45 a.m.: Coffee break
10.45 a.m. to 12.00 a.m.: Session 3 - Data structure (datasets in OMERO) and organization with Tags 
12.00 a.m. to 1.00 p.m.:  Lunch Break
1.00 p.m. to 2.00 p.m.:  Session 4 - REMBI, Key-Value pair annotations in bioimaging
2:00 p.m. to 2.30 p.m.:  Session 5 - Ontologies for Key-Value Pairs in OMERO
2:30 p.m. to 2:45 p.m. Coffee break
2.45 p.m. to 3.45 p.m.:  Wrap-up, discussion, outlook on day 2
Day 2 - April 30th, 2024
09.00 a.m. to 09.30 a.m.:  Arrival and Start into day 2
09.30 a.m. to 11.30 a.m.:  Session 6 - Hands-on : REMBI-based Key-Value Pair annotation in OMERO
11.30 a.m. to 12.30 a.m.:  Lunch Break
12.30 a.m. to 1.15 p.m.: Session 7 - OMERO and OMERO.plugins
1.15 p.m. to 2.00 p.m.: Session 8 - Loading OMERO-hosted data into Fiji
2.00 p.m. to 2.15 p.m.: Coffee break 
2.15 p.m. to 3.00 p.m.: Discussion, Outlook</p>
<p><a class="reference external" href="https://zenodo.org/records/11109616">https://zenodo.org/records/11109616</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11109616">https://doi.org/10.5281/zenodo.11109616</a></p>
</section>
<hr class="docutils" />
<section id="ilastik-interactive-machine-learning-for-bio-image-analysis">
<h2>ilastik: interactive machine learning for (bio)image analysis<a class="headerlink" href="#ilastik-interactive-machine-learning-for-bio-image-analysis" title="Permalink to this heading">#</a></h2>
<p>[‘Anna Kreshuk’, ‘Dominik Kutra’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slide</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.4330625">https://zenodo.org/doi/10.5281/zenodo.4330625</a></p>
</section>
<hr class="docutils" />
<section id="scikit-learn-mooc">
<h2>scikit-learn MOOC<a class="headerlink" href="#scikit-learn-mooc" title="Permalink to this heading">#</a></h2>
<p>[‘Loïc Estève et al.’]</p>
<p>Licensed CC-BY-4.0</p>
<p>Machine learning in Python with scikit-learn MOOC</p>
<p>Tags: Bioimage Analysis, Machine Learning</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/INRIA/scikit-learn-mooc">INRIA/scikit-learn-mooc</a></p>
<hr class="docutils" />
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./licenses"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

            </article>
            

            
            
            <footer class="bd-footer-article">
                <!-- Previous / next buttons -->
<div class='prev-next-area'>
  <a class='left-prev' id="prev-link" href="bsd-3-clause.html" title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
          <p class="prev-next-subtitle">previous</p>
          <p class="prev-next-title">Bsd-3-clause (17)</p>
      </div>
  </a>
  <a class='right-next' id="next-link" href="cc0-1.0.html" title="next page">
  <div class="prev-next-info">
      <p class="prev-next-subtitle">next</p>
      <p class="prev-next-title">Cc0-1.0 (5)</p>
  </div>
  <i class="fa-solid fa-angle-right"></i>
  </a>
</div>
            </footer>
            
          </div>
          
          
          
            <div class="bd-sidebar-secondary bd-toc">
              
<div class="toc-item">
  
<div class="tocsection onthispage">
    <i class="fa-solid fa-list"></i> On this page
</div>
<nav id="bd-toc-nav" class="page-toc">
    <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium">
   “ZENODO und Co.” Was bringt und wer braucht ein Repositorium?
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#steps-towards-reproducible-research">
   6 Steps Towards Reproducible Research
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#a-cloud-optimized-storage-for-interactive-access-of-large-arrays">
   A Cloud-Optimized Storage for Interactive Access of Large Arrays
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">
   A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe">
   A Hitchhiker’s guide through the bio-image analysis software universe
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten">
   Alles meins – oder!? Urheberrechte klären für Forschungsdaten
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bids-lecture-2024">
   BIDS-lecture-2024
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bigdataprocessor2-a-free-and-open-source-fiji-plugin-for-inspection-and-processing-of-tb-sized-image-data">
   BigDataProcessor2: A free and open-source Fiji plugin for inspection and processing of TB sized image data
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">
   Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-analysis-with-the-help-of-large-language-models">
   Bio-image Analysis with the Help of Large Language Models
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-data-science">
   Bio-image Data Science
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-data-science-lectures-uni-leipzig-scads-ai">
   Bio-image Data Science Lectures @ Uni Leipzig / ScaDS.AI
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">
   Bio-image analysis, biostatistics, programming and machine learning for computational biology
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bio-tools-database">
   Bio.tools database
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#bioimage-analysis-notebooks">
   BioImage Analysis Notebooks
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#browsing-the-open-microscopy-image-data-resource-with-python">
   Browsing the Open Microscopy Image Data Resource with Python
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">
   Challenges and opportunities for bio-image analysis core-facilities
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#challenges-and-opportunities-for-bioimage-analysis-core-facilities">
   Challenges and opportunities for bioimage analysis core-facilities
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#chatgpt-for-image-analysis">
   ChatGPT for Image Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#collaborative-bio-image-analysis-script-editing-with-git">
   Collaborative bio-image analysis script editing with git
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#combining-the-bids-and-arc-directory-structures-for-multimodal-research-data-organization">
   Combining the BIDS and ARC Directory Structures for Multimodal Research Data Organization
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#crashkurs-forschungsdatenmanagement">
   Crashkurs Forschungsdatenmanagement
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#creating-open-computational-curricula">
   Creating open computational curricula
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#cultivating-open-training">
   Cultivating Open Training
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#cultivating-open-training-to-advance-bio-image-analysis">
   Cultivating Open Training to advance Bio-image Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#dl4miceverywhere">
   DL4MicEverywhere
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#data-carpentry-for-biologists">
   Data Carpentry for Biologists
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#datenmanagement">
   Datenmanagement
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz">
   Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-1">
   Datenmanagementpläne erstellen - Teil 1
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-2">
   Datenmanagementpläne erstellen - Teil 2
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#developing-semi-automatic-analysis-pipelines-and-technological-solutions-for-metadata-annotation-and-management-in-high-content-screening-hcs-bioimaging">
   Developing (semi)automatic analysis pipelines and technological solutions for metadata annotation and management in high-content screening (HCS) bioimaging
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#developing-open-source-software-for-bioimage-analysis-opportunities-and-challenges">
   Developing open-source software for bioimage analysis: opportunities and challenges
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats">
   EDAM-bioimaging - The ontology of bioimage informatics operations, topics, data, and formats
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats-update-2020">
   EDAM-bioimaging: The ontology of bioimage informatics operations, topics, data, and formats (update 2020)
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#efficiently-starting-institutional-research-data-management">
   Efficiently starting institutional research data management
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten">
   Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">
   Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#euro-bioimaging-s-template-for-research-data-management-plans">
   Euro-BioImaging’s Template for Research Data Management Plans
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#fair-bioimage-data">
   FAIR BioImage Data
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#fairy-deep-learning-for-bioimage-analysis">
   FAIRy deep-learning for bioImage analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#forschungsdaten-org">
   Forschungsdaten.org
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers">
   From Paper to Pixels: Navigation through your Research Data - presentations of speakers
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#generative-artificial-intelligence-for-bio-image-analysis">
   Generative artificial intelligence for bio-image analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#getting-started-with-mambaforge-and-python">
   Getting started with Mambaforge and Python
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#hackaton-results-conversion-of-knime-image-analysis-workflows-to-galaxy">
   Hackaton Results - Conversion of KNIME image analysis workflows to Galaxy
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero">
   High throughput &amp; automated data analysis and data management workflow with Cellprofiler and OMERO
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists">
   Highlights from the 2016-2020 NEUBIAS training schools for Bioimage Analysts: a success story and key asset for analysts and life scientists
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">
   I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work">
   If you license it, it’ll be harder to steal it. Why we should license our work
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#image-analysis-training-resources">
   Image Analysis Training Resources
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#image-processing-with-python">
   Image Processing with Python
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#insights-and-impact-from-five-cycles-of-essential-open-source-software-for-science">
   Insights and Impact From Five Cycles of Essential Open Source Software for Science
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#introduction-to-bioimage-analysis">
   Introduction to Bioimage Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#kollaboratives-arbeiten-und-versionskontrolle-mit-git">
   Kollaboratives Arbeiten und Versionskontrolle mit Git
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#large-language-models-an-introduction-for-life-scientists">
   Large Language Models: An Introduction for Life Scientists
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#making-your-package-available-on-conda-forge">
   Making your package available on conda-forge
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#managing-scientific-python-environments-using-conda-mamba-and-friends">
   Managing Scientific Python environments using Conda, Mamba and friends
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration">
   Meeting in the Middle: Towards Successful Multidisciplinary Bioimage Analysis Collaboration
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#methods-in-bioimage-analysis">
   Methods in bioimage analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#microsam-talks">
   MicroSam-Talks
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive">
   Microscopy data analysis: machine learning and the BioImage Archive
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud">
   My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage">
   NFDI4BIOIMAGE
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">
   NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">
   NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis">
   NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and Bioimage Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-perspective-for-a-national-bioimaging-standard">
   NFDI4BIOIMAGE: Perspective for a national bioimaging standard
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon">
   NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon)
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">
   NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#nextflow-scalable-and-reproducible-scientific-workflows">
   Nextflow: Scalable and reproducible scientific workflows
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#ome-documentation">
   OME Documentation
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#ome-ngff-a-next-generation-file-format-for-expanding-bioimaging-data-access-strategies">
   OME-NGFF: a next-generation file format for expanding bioimaging data-access strategies
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#omero-tools">
   Omero-tools
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#open-image-data-handbook">
   Open Image Data Handbook
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#open-science-sharing-licensing">
   Open Science, Sharing &amp; Licensing
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing">
   Parallelization and heterogeneous computing: from pure CPU to GPU-accelerated image processing
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#photonic-data-analysis-in-2050">
   Photonic data analysis in 2050
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis">
   PoL Bio-Image Analysis Training School on GPU-Accelerated Image Analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#practical-guide-to-the-international-alignment-of-research-data-management-extended-edition">
   Practical Guide to the International Alignment of Research Data Management - Extended Edition
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#qi-2024-analysis-lab-manual">
   QI 2024 Analysis Lab Manual
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#qm-course-lectures-on-bio-image-analysis-with-napari-2024">
   QM Course Lectures on Bio-Image Analysis with napari 2024
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#quarep-limi-a-community-driven-initiative-to-establish-guidelines-for-quality-assessment-and-reproducibility-for-instruments-and-images-in-light-microscopy">
   QUAREP-LiMi: A community-driven initiative to establish guidelines for quality assessment and reproducibility for instruments and images in light microscopy
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#qupath-open-source-software-for-analysing-awkward-images">
   QuPath: Open source software for analysing (awkward) images
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again">
   RDF as a bridge to domain-platforms like OMERO, or There and back again.
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#rdm4mic-presentations">
   RDM4Mic Presentations
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#rdmkit-training-resources">
   RDMKit Training Resources
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#research-data-management-seminar-slides">
   Research Data Management Seminar - Slides
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey">
   Research data management for bioimaging: the 2021 NFDI4BIOIMAGE community survey
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#running-deep-learning-scripts-in-the-bia-pol-omero-server">
   Running Deep-Learning Scripts in the BiA-PoL Omero Server
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sharing-and-licensing-material">
   Sharing and licensing material
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sharing-research-data-with-zenodo">
   Sharing research data with Zenodo
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten">
   So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#spatialdata-an-open-and-universal-data-framework-for-spatial-omics">
   SpatialData: an open and universal data framework for spatial omics
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data">
   Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#sustainable-data-stewardship">
   Sustainable Data Stewardship
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#the-turing-way-guide-for-reproducible-research">
   The Turing Way: Guide for reproducible research
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#thinking-data-management-on-different-scales">
   Thinking data management on different scales
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#towards-preservation-of-life-science-data-with-nfdi4bioimage">
   Towards Preservation of Life Science Data with NFDI4BIOIMAGE
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#train-the-trainer-concept-on-research-data-management">
   Train-the-Trainer Concept on Research Data Management
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#using-glittr-org-to-find-compare-and-re-use-online-training-materials">
   Using Glittr.org to find, compare and re-use online training materials
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#welcome-to-bioimage-town">
   Welcome to BioImage Town
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#who-you-gonna-call-data-stewards-to-the-rescue">
   Who you gonna call? - Data Stewards to the rescue
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#cordi-2023-zarr-a-cloud-optimized-storage-for-interactive-access-of-large-arrays">
   [CORDI 2023] Zarr: A Cloud-Optimized Storage for Interactive Access of Large Arrays
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">
   [ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#gbi-eoe-vii-five-or-ten-must-have-items-for-making-it-infrastructure-for-managing-bioimage-data">
   [GBI EOE VII] Five (or ten) must-have items for making IT infrastructure for managing bioimage data
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#n4bi-ahm-welcome-to-bioimage-town">
   [N4BI AHM] Welcome to BioImage Town
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard">
   [SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">
   [Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero">
   [Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#ilastik-interactive-machine-learning-for-bio-image-analysis">
   ilastik: interactive machine learning for (bio)image analysis
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#scikit-learn-mooc">
   scikit-learn MOOC
  </a>
 </li>
</ul>

</nav>
</div>

            </div>
          
          
        </div>
        <footer class="bd-footer-content">
          <div class="bd-footer-content__inner">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Robert Haase, Clément Caporal,... and the NFDI4BioImage Initiative
</p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    <p class="last-updated">
Last updated on 2024-10-08.<br>
</p>
  </div>
  
  <div class="footer-item">
    
<div class="extra_footer">
  <p>
Copyright: Licensed <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY 4.0</a> unless mentioned otherwise. 
Contributions and feedback are welcome.
</p>

</div>
  </div>
  
</div>
          </div>
        </footer>
        

      </main>
    </div>
  </div>

  
    
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=796348d33e8b1d947c94"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=796348d33e8b1d947c94"></script>

  </body>
</html>