# Zenodo.org (96)
## "ZENODO und Co." Was bringt und wer braucht ein Repositorium?

Elfi Hesse, Jan-Christoph Deinert, Christian Löschen

Published 2021-01-25

Licensed CC-BY-4.0



Die Online-Veranstaltung fand am 21.01.2021 im Rahmen der SaxFDM-Veranstaltungsreihe &quot;Digital Kitchen - K&uuml;chengespr&auml;che mit SaxFDM&quot; statt. SaxFDM-Sprecherin Elfi Hesse (HTW Dresden) erl&auml;uterte zun&auml;chst Grunds&auml;tzliches zum Thema Repositorien. Anschlie&szlig;end teilten Nutzer (Jan Deinert &ndash; HZDR) und Anbieter (Christian L&ouml;schen &ndash; TU Dresden/ZIH) lokaler Repositorien ihre Erfahrungen mit uns.

Tags: Research Data Management

Content type: Slides

[https://zenodo.org/records/4461261](https://zenodo.org/records/4461261)

[https://doi.org/10.5281/zenodo.4461261](https://doi.org/10.5281/zenodo.4461261)


---

## 6 Steps Towards Reproducible Research

Heidi Seibold

Licensed CC-BY-4.0



A short book with 6 steps that get you closer to making your work reproducible.

Tags: Reproducibility, Research Data Management

Content type: Book

[https://zenodo.org/records/12744715](https://zenodo.org/records/12744715)


---

## A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter

Anca Margineanu, Chiara Stringari, Marcelo Zoccoler, Cornelia Wetzker

Published 2024-03-27

Licensed CC-BY-4.0



The presentations introduce open-source software to read in, visualize and analyse fluorescence lifetime imaging microscopy (FLIM) raw data developed for life scientists. The slides were presented at German Bioimaging (GerBI) FLIM Workshop held February 26 to 29 2024 at the Biomedical Center of LMU M&uuml;nchen by Anca Margineanu, Chiara Stringari and Conni Wetzker.&nbsp;

[https://zenodo.org/records/10886750](https://zenodo.org/records/10886750)

[https://doi.org/10.5281/zenodo.10886750](https://doi.org/10.5281/zenodo.10886750)


---

## A journey to FAIR microscopy data

Stefanie Weidtkamp-Peters, Janina Hanne, Christian Schmidt

Published 2023-05-03

Licensed CC-BY-4.0



Oral presentation, 32nd MoMAN &quot;From Molecules to Man&quot; Seminar, Ulm, online. Monday February 6th, 2023

Abstract:

Research data management is essential in nowadays research, and one of the big opportunities to accelerate collaborative and innovative scientific projects. To achieve this goal, all our data needs to be FAIR (findable, accessible, interoperable, reproducible). For data acquired on microscopes, however, a common ground for FAIR data sharing is still to be established. Plenty of work on file formats, data bases, and training needs to be performed to highlight the value of data sharing and exploit its potential for bioimaging data.

In this presentation, Stefanie Weidtkamp-Peters will introduce the challenges for bioimaging data management, and the necessary steps to achieve data FAIRification. German BioImaging - GMB e.V., together with other institutions, contributes to this endeavor. Janina Hanne will present how the network of imaging core facilities, research groups and industry partners is key to the German bioimaging community&rsquo;s aligned collaboration toward FAIR bioimaging data. These activities have paved the way for two data management initiatives in Germany: I3D:bio (Information Infrastructure for BioImage Data) and NFDI4BIOIMAGE, a consortium of the National Research Data Infrastructure. Christian Schmidt will introduce the goals and measures of these initiatives to the benefit of imaging scientist&rsquo;s work and everyday practice. &nbsp;

[https://zenodo.org/records/7890311](https://zenodo.org/records/7890311)

[https://doi.org/10.5281/zenodo.7890311](https://doi.org/10.5281/zenodo.7890311)


---

## Alles meins – oder!? Urheberrechte klären für Forschungsdaten

Stephan Wünsche

Published 2024-06-04

Licensed CC-BY-4.0



Wem geh&ouml;ren Forschungsdaten? Diese Frage stellt sich bei Daten, an deren Entstehung mehrere Personen beteiligt waren, und besonders bei Textdaten, Bildern und Videos. Hier lernen Sie, f&uuml;r Ihr eigenes Forschungsvorhaben zu erkennen, wessen Urheber- und Leistungsschutzrechte zu ber&uuml;cksichtigen sind. Sie erfahren, wie Sie mit Hilfe von Vereinbarungen fr&uuml;hzeitig Rechtssicherheit herstellen, etwa um Daten weitergeben oder publizieren zu k&ouml;nnen.
&nbsp;
&nbsp;

Tags: Research Data Management, Licensing

Content type: Slides

[https://zenodo.org/records/11472148](https://zenodo.org/records/11472148)

[https://doi.org/10.5281/zenodo.11472148](https://doi.org/10.5281/zenodo.11472148)


---

## Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities

Cornelia Wetzker

Published 2023-11-08

Licensed CC-BY-4.0



This presentation gives a short outline of the complexity of data and metadata in the bioimaging universe. It introduces NFDI4BIOIMAGE as a newly formed consortium as part of the German 'Nationale Forschungsdateninfrastruktur' (NFDI) and its goals and tools for data management including its current members on TU Dresden campus. &nbsp;

Tags: Research Data Management, Tu Dresden, Bioimage Data, Nfdi4Bioimage

Content type: Slide

[https://zenodo.org/records/10083555](https://zenodo.org/records/10083555)

[https://doi.org/10.5281/zenodo.10083555](https://doi.org/10.5281/zenodo.10083555)


---

## Bio-image Analysis Code Generation

Robert Haase

Published 2024-10-28

Licensed CC-BY-4.0



Large Language Models are changing the way we interact with computers, especially how we write code. In this tutorial, we will generate bio-image analysis code using two LLM-based code generators, bia-bob and git-bob.
https://github.com/haesleinhuepf/bia-bob
https://github.com/haesleinhuepf/git-bob
&nbsp;

[https://zenodo.org/records/14001044](https://zenodo.org/records/14001044)

[https://doi.org/10.5281/zenodo.14001044](https://doi.org/10.5281/zenodo.14001044)


---

## Bio-image Analysis Code Generation using bia-bob

Robert Haase

Published 2024-10-09

Licensed CC-BY-4.0



In this presentation I introduce bia-bob, an AI-based code generator that integrates into Jupyter Lab and allows for easy generation of Bio-Image Analysis Python code. It highlights how to get started with using large language models and prompt engineering to get high-quality bio-image analysis code.

[https://zenodo.org/records/13908108](https://zenodo.org/records/13908108)

[https://doi.org/10.5281/zenodo.13908108](https://doi.org/10.5281/zenodo.13908108)


---

## Bio-image Analysis with the Help of Large Language Models

Robert Haase

Published 2024-03-13

Licensed CC-BY-4.0



Large Language Models (LLMs) change the way how we use computers. This also has impact on the bio-image analysis community. We can generate code that analyzes biomedical image data if we ask the right prompts. This talk outlines introduces basic principles, explains prompt engineering and how to apply it to bio-image analysis. We also compare how different LLM vendors perform on code generation tasks and which challenges are ahead for the bio-image analysis community.

Tags: Large Language Models, Python

Content type: Slide

[https://zenodo.org/records/10815329](https://zenodo.org/records/10815329)

[https://doi.org/10.5281/zenodo.10815329](https://doi.org/10.5281/zenodo.10815329)


---

## Bio-image Data Science Lectures @ Uni Leipzig / ScaDS.AI

Robert Haase

Licensed CC-BY-4.0



These are the PPTx training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python. The material developed here between April and July 2024.

Tags: Bioimage Analysis, Deep Learning, Microscopy Image Analysis, Python

Content type: Slides

[https://zenodo.org/records/12623730](https://zenodo.org/records/12623730)


---

## Building FAIR image analysis pipelines for high-content-screening (HCS) data using Galaxy

Riccardo Massei, Matthias Berndt, Beatriz Serrano-Solano, Wibke Busch, Stefan Scholz, Hannes Bohring, Jo Nyffeler, Luise Reger, Jan Bumberger, Lucille Lopez-Delisle

Published 2024-11-06

Licensed CC-BY-4.0



Imaging is crucial across various scientific disciplines, particularly in life sciences, where it plays a key role in studies ranging from single molecules to whole organisms. However, the complexity and sheer volume of image data, especially from high-content screening (HCS) experiments involving cell lines or other organisms, present significant challenges. Managing and analysing this data efficiently requires well-defined image processing tools and analysis pipelines that align with the FAIR principles&mdash;ensuring they are findable, accessible, interoperable, and reusable across different domains.
In the frame of NFDI4BioImaging (the National Research Data Infrastructure focusing on bioimaging in Germany), we want to find viable solutions for storing, processing, analysing, and sharing HCS data. In particular, we want to develop solutions to make findable and machine-readable metadata using (semi)automatic analysis pipelines. In scientific research, such pipelines are crucial for maintaining data integrity, supporting reproducibility, and enabling interdisciplinary collaboration. These tools can be used by different users to retrieve images based on specific attributes as well as support quality control by identifying appropriate metadata.
Galaxy, an open-source, web-based platform for data-intensive research, offers a solution by enabling the construction of reproducible pipelines for image analysis. By integrating popular analysis software like CellProfiler and connecting with cloud services such as OMERO and IDR, Galaxy facilitates the seamless access and management of image data. This capability is particularly valuable in bioimaging, where automated pipelines can streamline the handling of complex metadata, ensuring data integrity and fostering interdisciplinary collaboration. This approach not only increases the efficiency of HCS bioimaging but also contributes to the broader scientific community's efforts to embrace FAIR principles, ultimately advancing scientific discovery and innovation.
In the present study, we proposed an automated analysis pipeline for storing, processing, analysing, and sharing HCS bioimaging data. The (semi)automatic workflow was developed by taking as a case study a dataset of zebrafish larvae and cell lines images previously obtained from an automated imaging system generating data in an HCS fashion. In our workflows, images are automatically enriched with metadata (i.e. key-value pairs, tags, raw data, regions of interest) and uploaded to the UFZ-OME Remote Objects (OMERO) server using a novel OMERO tool suite developed with GALAXY. Workflows give the possibility to the user to intuitively fetch images from the local server and perform image analysis (i.e. annotation) or even more complex toxicological analyses (dose response modelling). Furthermore, we want to improve the FAIRness of the protocol by adding a direct upload link to the Image Data Resource (IDR) repository to automatically prepare the data for publication and sharing.

[https://zenodo.org/records/14044640](https://zenodo.org/records/14044640)

[https://doi.org/10.5281/zenodo.14044640](https://doi.org/10.5281/zenodo.14044640)

[https://galaxyproject.org/news/2024-11-08-galaxy-imaging-fair-pipelines/](https://galaxyproject.org/news/2024-11-08-galaxy-imaging-fair-pipelines/)


---

## CZI file examples

Nicolas Chiaruttini

Published 2023-08-18

Licensed CC-BY-4.0



A set of public CZI files. These can be used for testing CZI readers.

- Demo LISH 4x8 15pct 647.czi: A cleared mouse brain acquired with a Zeiss LightSheet Z1 with 32 tiles. Courtesy of the Carl Petersen lab LSENS (https://www.epfl.ch/labs/lsens). Sampled prepared by Yanqi Liu an imaged by Olivier Burri.
- test_gray.czi: a synthetically generated CZI file without metadata, made by Sebastian Rhode

- Image_1_2023_08_18__14_32_31_964.czi: an example multi-part CZI file, containing only camera noise

- a xt scan, xz scan, xzt scan

- a set of multi angle, multi illumination, mutli tile acquisition, taken on the LightSheet Z1 microscope of the PTBIOP by Lorenzo Tal&agrave;

[https://zenodo.org/records/8305531](https://zenodo.org/records/8305531)

[https://doi.org/10.5281/zenodo.8305531](https://doi.org/10.5281/zenodo.8305531)


---

## CZI: Open Science Program Collection



Content type: Collection

[https://zenodo.org/communities/eoss](https://zenodo.org/communities/eoss)


---

## ChatGPT for Image Analysis

Robert Haase

Published 2024-08-25

Licensed CC-BY-4.0



Large Language Models (LLMs) such as ChatGPT are changing the way we interact with computers, including how we analye microscopy imaging data. In this talk I introduce basic concepts of asking LLMs to write code and how to modify the questions to get the best out of it. For trying out these prompt engineering basics there are additional online resources available: https://scads.github.io/prompt-engineering-basics-2024/intro.html

[https://zenodo.org/records/13371196](https://zenodo.org/records/13371196)

[https://doi.org/10.5281/zenodo.13371196](https://doi.org/10.5281/zenodo.13371196)


---

## Combining the BIDS and ARC Directory Structures for Multimodal Research Data Organization

Torsten Stöter, Tobias Gottschall, Andrea Schrader, Peter Zentis, Monica Valencia-Schneider, Niraj Kandpal, Werner Zuschratter, Astrid Schauss, Timo Dickscheid, Timo Mühlhaus, Dirk von Suchodoletz

Licensed CC-BY-4.0



Interdisciplinary collaboration and integrating large, diverse datasets are crucial for answering complex research questions, requiring multimodal data analysis and adherence to FAIR principles. To address challenges in capturing the full research cycle and contextualizing data, DataPLANT developed the Annotated Research Context (ARC), while the neuroimaging community extended the Brain Imaging Data Structure (BIDS) for microscopic image data, both providing standardized, file system-based storage structures for organizing and sharing data with metadata.

Tags: Research Data Management, FAIR-Principles

Content type: Poster

[https://zenodo.org/doi/10.5281/zenodo.8349562](https://zenodo.org/doi/10.5281/zenodo.8349562)


---

## Crashkurs Forschungsdatenmanagement

Barbara Weiner, Stephan Wünsche, Stefan Kühne, Pia Voigt, Sebastian Frericks, Clemens Hoffmann, Romy Elze, Ronny Gey

Published 2020-04-30

Licensed CC-BY-4.0



Diese Pr&auml;sentation bietet einen Einstieg in alle relevanten Bereiche des Forschungsdatenmanagements an der Universit&auml;t Leipzig. Behandelt werden Grundlagen des Forschungsdatenmanagements, technische, ethische und rechtliche Aspekte sowie die Archivierung und Publikation von Forschungsdaten. Die Pr&auml;sentation enth&auml;lt zahlreiche weiterf&uuml;hrende Links (rot) und Literaturhinweise.

Erg&auml;nzend hierzu wird eine Pr&auml;sentation mit &Uuml;bungsaufgaben angeboten, die helfen soll, das Gelernte zu festigen und in der eigenen Forschungspraxis umzusetzen. Den Aufgaben folgen jeweils eine Antwortfolie sowie deren Aufl&ouml;sung.

Tags: Research Data Management

Content type: Slides

[https://zenodo.org/records/3778431](https://zenodo.org/records/3778431)

[https://doi.org/10.5281/zenodo.3778431](https://doi.org/10.5281/zenodo.3778431)


---

## Creating open computational curricula

Kari Jordan, Zhian Kamvar, Toby Hodges

Published 2020-12-11

Licensed CC-BY-4.0



In this interactive session, Carpentries team members will guide attendees through three stages of the backward design process to create a lesson development plan for the open source tool of their choosing. Attendees will leave having identified what practical skills they aim to teach (learning objectives), an approach for designing challenge questions (formative assessment), and mechanisms to give and receive feedback.

Content type: Slide

[https://zenodo.org/records/4317149](https://zenodo.org/records/4317149)

[https://doi.org/10.5281/zenodo.4317149](https://doi.org/10.5281/zenodo.4317149)


---

## Cultivating Open Training

Robert Haase

Published 2024-03-14

Licensed CC-BY-4.0



In this SaxFDM Digital Kitchen, I introduced current challenges and potential solutions for openly sharing training materials, softly focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and uploading materials to publicly acessible repositories are proposed to fill this gap.

Tags: Open Science, Research Data Management, FAIR-Principles, Bioimage Analysis, Licensing

Content type: Slides

[https://zenodo.org/records/10816895](https://zenodo.org/records/10816895)

[https://doi.org/10.5281/zenodo.10816895](https://doi.org/10.5281/zenodo.10816895)


---

## Cultivating Open Training to advance Bio-image Analysis

Robert Haase

Published 2024-04-25

Licensed CC-BY-4.0




These slides introduce current challenges and potential solutions for openly sharing training materials, focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and publicly acessible repositories are proposed to fill this gap.


Tags: Research Data Management, Licensing, FAIR-Principles

Content type: Slides

[https://zenodo.org/records/11066250](https://zenodo.org/records/11066250)

[https://doi.org/10.5281/zenodo.11066250](https://doi.org/10.5281/zenodo.11066250)


---

## DALIA Interchange Format

Jonathan Geiger, Petra Steiner, Abdelmoneim Amer Desouki, Frank Lange

Published 2024-06-07

Licensed CC-BY-SA-4.0



The DALIA (Data Literacy Alliance) project aims to develop a knowledge graph for FAIR teaching and learning materials on data literacy, data competencies and research data management (RDM) skills within the National Research Data Infrastructure (NFDI) and the RDM landscape. Such a platform thrives on the participation of users who want to search, create, manage or use teaching and learning materials.
A schematization of the metadata is necessary for the interoperability of teaching and learning materials. This is done by the DALIA Interchange Format (DIF), which provides a framework for making the metadata of teaching and learning materials transparent, comparable and smooth to integrate into the DALIA platform. It includes the description and explanation of the data fields for the online publication of educational resources.
The DIF was developed in close consultation with the scientific community. This development process included several feedback rounds in which valuable feedback was provided and subsequently incorporated into the DIF. This not only contributed to the clear, transparent and user-oriented definitions of the data fields, and to a clear structure, but also to the integration of many existing data standards and to the (special) requirements of the scientific community. The selection of elements is based on the&nbsp;Dublin Core Application Profile.
The DIF is provided as a PDF document and in table form (ODS) to convey the attributes of the teaching and learning materials and their definitions in an easily understandable form and to facilitate communication. It also includes a legend and an example in tabular form. In addition, a template (CSV) with the attributes as column headers is provided, which can be used for recording the metadata of the teaching and learning materials. The tables can also be transferred to technical application profiles.
We would like to thank all the commentators of the previous versions, especially Susanne Arndt, Sophie Bo&szlig;e, Sonja Felder, Marc Fuhrmans, Jan-Michael Haugwitz, Marina Lemaire, Karoline Lemke, Birte Lindst&auml;dt, Juliane R&ouml;der, and Jakob Vo&szlig;. Without their feedback and advice, the DIF would be less transparent.

[https://zenodo.org/records/11521029](https://zenodo.org/records/11521029)

[https://doi.org/10.5281/zenodo.11521029](https://doi.org/10.5281/zenodo.11521029)


---

## Data stewardship and research data management tools for multimodal linking of imaging data in plasma medicine

Ahmadi, Mohsen, Wagner, Robert, Mattern, Philipp, Plathe, Nick, Bekeschus, Sander, Becker, Markus M., Stöter, Torsten, Weidtkamp-Peters, Stefanie

Published 2023-11-03

Licensed CC-BY-4.0



A more detailed understanding of the effect of plasmas on biological systems can be fostered by combining data from different imaging modalities, such as optical imaging, fluorescence imaging, and mass spectrometry imaging. This, however, requires the implementation and use of sophisticated research data management (RDM) solutions to incorporate the influence of plasma parameters and treatment procedures as well as the effects of plasma on the treated targets. In order to address this, RDM activities on different levels and from different perspectives are started and brought together within the framework of the NFDI consortium NFDI4BIOIMAGE.

[https://zenodo.org/records/10069368](https://zenodo.org/records/10069368)

[https://doi.org/10.5281/zenodo.10069368](https://doi.org/10.5281/zenodo.10069368)


---

## Datenmanagement

Robert Haase

Published 2024-04-14

Licensed CC-BY-4.0



In dieser Data Management Session wird der Lebenszyklus von Daten n&auml;her beleuchtet. Wie entstehen Daten, was passiert mit ihnen, wenn sie verarbeitet werden? Wem geh&ouml;ren die Daten und wer ist daf&uuml;r verantwortlich, sie zu ver&ouml;ffentlichen, zu archivieren und gegebenenfalls wiederzuverwenden? Wir werden einen Datenmanagementplan in Gruppenarbeit entwerfen, ggf. mit Hilfe von ChatGPT.

Tags: Research Data Management

Content type: Slides

[https://zenodo.org/records/10970869](https://zenodo.org/records/10970869)

[https://doi.org/10.5281/zenodo.10970869](https://doi.org/10.5281/zenodo.10970869)


---

## Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz

Pia Voigt, Carolin Hundt

Published 2024-04-19

Licensed CC-BY-4.0



 Workshop zum Thema &bdquo;Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz&ldquo; auf der Data Week Leipzig
Der Umgang mit Daten ist im Alltag nicht immer leicht: Wie und wo speichert man Daten idealerweise? Welche Strategien helfen, den &Uuml;berblick zu behalten und wie geht man mit personenbezogenen Daten um? Diese Fragen m&ouml;chten wir gemeinsam mit Ihnen anhand individueller Datenprobleme besprechen und Ihnen L&ouml;sungen aufzeigen, wie Sie ihr Datenmanagement effizient gestalten k&ouml;nnen.

Tags: Research Data Management

Content type: Slides

[https://zenodo.org/records/11107798](https://zenodo.org/records/11107798)

[https://doi.org/10.5281/zenodo.11107798](https://doi.org/10.5281/zenodo.11107798)


---

## Datenmanagementpläne erstellen - Teil 1

Pia Voigt, Barbara Weiner

Published 2021-03-23

Licensed CC-BY-4.0



Was ist ein Datenmanagementplan? Welche Vorgaben sollte ich beachten? Wie erstelle ich einen solchen f&uuml;r mein Forschungsprojekt und welche n&uuml;tzlichen Tools kann ich hierf&uuml;r verwenden?

Die Anforderungen der Forschungsf&ouml;rderer zum Datenmanagement steigen stetig. Damit verbunden ist h&auml;ufig auch das Erstellen eines Datenmanagementplans. Dabei erwarten DFG, BMBF oder die EU jeweils unterschiedliche Angaben zur Erhebung, Speicherung und Ver&ouml;ffentlichung von projektbezogenen Forschungsdaten. Zudem bietet das Erstellen eines Datenmanagementplans viele Vorteile und hilft Ihnen nicht zuletzt, die Anforderungen der guten wissenschaftlichen Praxis strukturiert umzusetzen.

Was im ersten Moment un&uuml;bersichtlich und &uuml;berfordernd wirkt, soll in diesem Kurs anhand einer grundlegenden theoretischen Einf&uuml;hrung im ersten und praxisorientierter Beispiele im zweiten Teil der Veranstaltung handhabbar gemacht werden. Sie lernen, was hinter den Anforderungen der Forschungsf&ouml;rderer steckt, welche Elemente ein Datenmanagementplan enthalten sollte und wie sie einen solchen mithilfe interaktiver Tools selbst erstellen k&ouml;nnen.

Tags: Research Data Management

Content type: Slides

[https://zenodo.org/records/4630788](https://zenodo.org/records/4630788)

[https://doi.org/10.5281/zenodo.4630788](https://doi.org/10.5281/zenodo.4630788)


---

## Datenmanagementpläne erstellen - Teil 2

Pia Voigt, Barbara Weiner

Published 2021-03-30

Licensed CC-BY-4.0



Was ist ein Datenmanagementplan? Welche Vorgaben sollte ich beachten? Wie erstelle ich einen solchen f&uuml;r mein Forschungsprojekt und welche n&uuml;tzlichen Tools kann ich hierf&uuml;r verwenden?

Die Anforderungen der Forschungsf&ouml;rderer zum Datenmanagement steigen stetig. Damit verbunden ist h&auml;ufig auch das Erstellen eines Datenmanagementplans. Dabei erwarten DFG, BMBF oder die EU jeweils unterschiedliche Angaben zur Erhebung, Speicherung und Ver&ouml;ffentlichung von projektbezogenen Forschungsdaten. Zudem bietet das Erstellen eines Datenmanagementplans viele Vorteile und hilft Ihnen nicht zuletzt, die Anforderungen der guten wissenschaftlichen Praxis strukturiert umzusetzen.

Was im ersten Moment un&uuml;bersichtlich und &uuml;berfordernd wirkt, soll in diesem Kurs anhand einer grundlegenden theoretischen Einf&uuml;hrung im ersten und praxisorientierter Beispiele im zweiten Teil der Veranstaltung handhabbar gemacht werden. Sie lernen, was hinter den Anforderungen der Forschungsf&ouml;rderer steckt, welche Elemente ein Datenmanagementplan enthalten sollte und wie sie einen solchen mithilfe interaktiver Tools selbst erstellen k&ouml;nnen.

Version 2 enth&auml;lt aktuelle Links und weiterf&uuml;hrende Hinweise zu einzelnen Aspekten eines Datenmanagementplans.

Version 3 ist die &uuml;berarbeitete und aktualisierte Version der ersten beiden und enth&auml;lt u.a. Hinweise zur Lizenzierung und zu Nutzungsrechten an Forschungsdaten.

Tags: Research Data Management

Content type: Slides

[https://zenodo.org/records/4748534](https://zenodo.org/records/4748534)

[https://doi.org/10.5281/zenodo.4748534](https://doi.org/10.5281/zenodo.4748534)


---

## Efficiently starting institutional research data management

Katarzyna Biernacka, Katrin Cortez, Kerstin Helbig

Published 2019-10-15

Licensed CC-BY-4.0



Researchers are increasingly often confronted with research data management (RDM) topics during their work. Higher education institutions therefore begin to offer services for RDM at some point to give support and advice. However, many groundbreaking decisions have to be made at the very beginning of RDM services. Priorities must be set and policies formulated. Likewise, the staff must first be qualified in order to provide advice and adequately deal with the manifold problems awaiting.
The FDMentor project has therefore bundled the expertise of five German universities with different experiences and levels of RDM knowledge to jointly develop strategies, roadmaps, guidelines, and open access training material. Humboldt-Universit&auml;t zu Berlin, Freie Universit&auml;t Berlin, Technische Universit&auml;t Berlin, University of Potsdam, and European University Viadrina Frankfurt (Oder) have worked together on common solutions that are easy to adapt. With funding of the German Federal Ministry of Education and Research, the collaborative project addressed four problem areas: strategy development, legal issues, policy development, and competence enhancement. The aim of the project outcomes is to provide other higher education institutions with the best possible support for the efficient introduction of research data management. Therefore, all project results are freely accessible under the CC-BY 4.0 international license. The early involvement of the community in the form of workshops and the collection of feedback has proven its worth: the FDMentor strategies, roadmaps, guidelines, and training materials are applied and adapted beyond the partner universities.

Tags: Research Data Management

Content type: Document

[https://zenodo.org/record/3490058](https://zenodo.org/record/3490058)

[https://doi.org/10.5281/zenodo.3490058](https://doi.org/10.5281/zenodo.3490058)


---

## Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten

Stephan Wünsche, Pia Voigt

Published 2021-05-11

Licensed CC-BY-4.0



Diese Pr&auml;sentation wurde im Zuge der digitalen Veranstaltungsreihe &quot;Einblicke ins Forschungsdatenmanagement&quot; erstellt. Diese findet seit dem SS 2020 an der Universit&auml;t Leipzig f&uuml;r alle Interessierten zu verschiedenen Themen des Forschungsdatenmanagements statt.

Dieser Teil der Reihe dreht sich um Rechtsfragen im Umgang mit Forschungsdaten und deren Bedeutung f&uuml;r die wissenschaftliche Praxis. Sie finden in der vorliegenden Pr&auml;sentation einen &Uuml;berblick &uuml;ber relevante Rechtsbereiche sowie Erl&auml;uterungen zum Datenschutz, Urheberrecht und den Grunds&auml;tzen der guten wissenschaftlichen Praxis mit Fokus auf deren Bedeutung im Forschungsdatenmanagement.

Tags: Research Data Management, Data Protection

Content type: Slides

[https://zenodo.org/records/4748510](https://zenodo.org/records/4748510)

[https://doi.org/10.5281/zenodo.4748510](https://doi.org/10.5281/zenodo.4748510)


---

## Euro-BioImaging's Guide to FAIR BioImage Data - Practical Tasks

Isabel Kemmer, Euro-BioImaging ERIC

Published 2024-06-04

Licensed CC-BY-4.0



Hands-on exercises on FAIR Bioimage Data from the interactive online workshop "Euro-BioImaging's Guide to FAIR BioImage Data 2024" (https://www.eurobioimaging.eu/news/a-guide-to-fair-bioimage-data-2024/).&nbsp; Types of tasks included: FAIR characteristics of a real world dataset Data Management Plan (DMP) Journal Policies on FAIR data sharing Ontology search Metadata according to REMBI scheme (Image from: Sarkans, U., Chiu, W., Collinson, L. et al. REMBI: Recommended Metadata for Biological Images&mdash;enabling reuse of microscopy data in biology. Nat Methods 18, 1418&ndash;1422 (2021). https://doi.org/10.1038/s41592-021-01166-8) Matching datasets to bioimage repositories Browsing bioimage repositories

Tags: Bioimage Analysis, FAIR-Principles, Research Data Management

Content type: Slides, Tutorial

[https://zenodo.org/records/11474407](https://zenodo.org/records/11474407)

[https://doi.org/10.5281/zenodo.11474407](https://doi.org/10.5281/zenodo.11474407)


---

## Euro-BioImaging's Template for Research Data Management Plans

Isabel Kemmer, Euro-BioImaging ERIC

Published 2024-06-04

Licensed CC-BY-4.0



Euro-BioImaging has developed a Data Management Plan (DMP) template with questions tailored to bioimaging research projects. Outlining data management practices in this way ensures traceability of project data, allowing for a continuous and unambiguous flow of information throughout the research project. This template can be used to satisfy the requirement to submit a DMP to certain funders. Regardless of the funder, Euro-BioImaging users are encouraged to provide a DMP and can use this template accordingly.&nbsp;
This DMP template is available as a fillable PDF with further instructions and sample responses available by hovering over the fillable fields.&nbsp;

Tags: Bioimage Analysis, FAIR-Principles, Research Data Management

Content type: Collection, Tutorial

[https://zenodo.org/records/11473803](https://zenodo.org/records/11473803)

[https://doi.org/10.5281/zenodo.11473803](https://doi.org/10.5281/zenodo.11473803)


---

## Evident OIR sample files tiles + stitched image - FV 4000

Nicolas Chiaruttini

Published 2024-09-04

Licensed CC-BY-4.0



The files contained in this repository are confocal images taken with the Evident FV 4000 of a sample containing DAPI and mCherry stains, excited with a 405 nm laser and a 561 nm laser

individual tiles are named `tiling-sample-brain-section_A01_G001_{i}.oir`
The stiched image is named `Stitch_A01_G001` and contains an extra file `Stitch_A01_G001_00001`
Some metadata like the tiles positions are stored in the extra files (omp2info)

&nbsp;

[https://zenodo.org/records/13680725](https://zenodo.org/records/13680725)

[https://doi.org/10.5281/zenodo.13680725](https://doi.org/10.5281/zenodo.13680725)


---

## Example Microscopy Metadata JSON files produced using Micro-Meta App to document example microscopy experiments performed at individual core facilities

Alessandro Rigano, Ulrike Boehm, Claire M. Brown, Joel Ryan, James J. Chambers, Robert A. Coleman, Orestis Faklaris, Thomas Guilbert, Michelle S. Itano, Judith Lacoste, Alex Laude, Marco Marcello, Paula Montero-Llopis, Glyn Nelson, Roland Nitschke, Jaime A. Pimentel, Stefanie Weidtkamp-Peters, Caterina Strambio-De-Castillia

Published 2022-01-15

Licensed CC-BY-4.0



Example Microscopy Metadata (Microscope.JSON and Settings.JSON) files produced using Micro-Meta App to document the Hardware Specifications of example Microscopes and the Image Acquisition Settings utilized to acquire example images as listed in the table below.


For each facility, the dataset contains two JSON files:


	Microscope.JSON file (e.g., 01_marcello_uliverpool_cci_zeiss_axioobserz1_lsm710.json)
	Settings.JSON file (indicated with the name of the image and with the _AS suffix)



Micro-Meta App was developed as part of a global community initiative including the 4D Nucleome (4DN) Imaging Working Group, BioImaging North America (BINA) Quality Control and Data Management Working Group, and QUAlity and REProducibility for Instrument and Images in Light Microscopy (QUAREP-LiMi), to extend the Open Microscopy Environment (OME) data model.


The works of this global community effort resulted in multiple publications featured on a recent Nature Methods FOCUS ISSUE dedicated to Reporting and reproducibility in microscopy.



Learn More! For a thorough description of Micro-Meta App consult our recent Nature Methods and BioRxiv.org publications!


&nbsp;


	
		
			Nr.
			Manufacturer
			Model
			Tier
			&Epsilon;xperiment Type
			Facility Name
			Department and Institution
			URL
			References
		
		
			1
			Carl Zeiss Microscopy
			Axio Observer Z1 (with LSM 710 scan head)
			1
			3D visualization of superhydrophobic polymer-nanoparticles
			Centre for Cell Imaging (CCI)
			University of Liverpool
			https://cci.liv.ac.uk/equipment_710.html
			Upton et al., 2020
		
		
			2
			Carl Zeiss Microscopy
			Axio Observer (Axiovert 200M)
			2
			&Mu;easurement of illumination stability on Chinese Hamster Ovary cells expressing Paxillin-EGFP
			Advanced BioImaging Facility (ABIF).
			McGill University
			https://www.mcgill.ca/abif/equipment/axiovert-1
			Kiepas et al., 2020
		
		
			3
			Carl Zeiss Microscopy
			Axio Observer Z1 (with Spinning Disk)
			2
			Immunofluorescence imaging of cryosection of Mouse kidney
			Imagerie Cellulaire; Quality Control managed by Miacellavie (https://miacellavie.com/)
			Centre de recherche du Centre Hospitalier Universit&eacute; de Montr&eacute;al (CR CHUM), University of Montreal
			https://www.chumontreal.qc.ca/crchum/plateformes-et-services&nbsp; (the web site is for all core facilities, not specifically for the core facility hosting this microscope)
			Pilliod et al., 2020
		
		
			4
			Carl Zeiss Microscopy
			Axio Imager Z2 (with Apotome)
			2
			Immunofluorescence imaging of mitotic division in Hela cells using&nbsp;&nbsp;
			Bioimaging Unit
			Newcastle University
			https://www.ncl.ac.uk/bioimaging/
			Watson et al., 2020
		
		
			5
			Carl Zeiss Microscopy
			Axio Observer Z1
			2
			Fluorescence microscopy of human skin fibroblasts from Glycogen Storage Disease patients.
			Life Imaging Center (LIC)
			Centre for Integrative Signalling Analysis (CISA), University of Freiburg
			https://miap.eu/equipments/sd-i-abl/
			Hannibal et al., 2020
		
		
			6
			Leica Microsystems
			DMI6000B
			2
			3D immunofluorescence imaging&nbsp; rhinovirus infected macrophages&nbsp;
			IMAG&#39;IC Confocal Microscopy Facility
			Institut Cochin, CNRS, INSERM, Universit&eacute; de Paris
			https://www.institutcochin.fr/core_facilities/confocal-microscopy/cochin-imaging-photonic-microscopy/organigram_team/10054/view
			Jubrail et al., 2020
		
		
			7
			Leica Microsystems
			DM5500B
			2
			Immunofluorescence analysis of the colocalization of PML bodies with DNA double-strand breaks
			Bioimaging Unit
			Edwardson Building on the Campus for Ageing and Vitality, Newcastle University
			https://www.ncl.ac.uk/bioimaging/equipment/leica-dm5500/#overview
			da Silva et al., 2019; Nelson et al., 2012
			&nbsp;&nbsp;
		
		
			8
			Leica Microsystems
			DMI8-CS (with TCS SP8 STED 3X)
			2
			Live-cell imaging of N. benthamiana leaves cells-derived protoplasts
			Center for Advanced Imaging (CAi)
			School of Mathematics/Natural Sciences, Heinrich-Heine-Universit&auml;t D&uuml;sseldorf
			https://www.cai.hhu.de/en/equipment/super-resolution-microscopy/leica-tcs-sp8-sted-3x
			Singer et al., 2017; H&auml;nsch et al., 2020
		
		
			9
			Nikon Instruments
			Eclipse Ti
			2
			Immunofluorescence analysis of the cytoskeleton structure in COS cells
			Advanced Imaging Center (AIC)
			Janelia Research Campus, Howard Hughes Medical Institute
			https://www.janelia.org/support-team/light-microscopy/equipment
			Abdelfattah et al., 2019; Qian et al., 2019; Grimm et al., 2020
		
		
			10
			Nikon Instruments
			Eclipse Ti-E (HCA)
			2
			&Tau;ime-lapse analysis of the bursting behavior of amine-functionalized vesicular assemblies
			Light Microscopy Facility (IALS-LIF)
			Institute for Applied Life Sciences, University of Massachusetts at Amherst
			https://www.umass.edu/ials/light-microscopy
			Fernandez et al., 2020
		
		
			11
			Nikon Instruments/Coleman laboratory (customized)
			TIRF HILO Epifluorescence light Microscope (THEM)/ Eclipse Ti
			2
			Single-particle tracking of Halo-tagged PCNA in Lox cells
			Coleman laboratory
			Anatomy and Structural Biology Department, The Albert Einstein College of Medicine
			https://einsteinmed.org/faculty/12252/robert-coleman/
			Drosopoulos et al., 2020
		
		
			12
			Nikon Instruments
			Eclipse Ti (with Andor Dragon Fly Spinning Disk)
			2
			Investigation of the 3D structure of cerebral organoids
			Montpellier Resources Imagerie
			Centre de Recherche de Biologie cellulaire de Montpellier (MRI-CRBM), CNRS, Univerity of Montpellier
			https://www.mri.cnrs.fr/en/optical-imaging/our-facilities/mri-crbm.html
			Ayala-Nunez et al., 2019
		
		
			13
			Nikon Instruments
			Eclipse Ti2
			2
			&Iota;mmunofluorescence imaging of cryosections of mouse hearth myocardium&nbsp;
			Neuroscience Center Microscopy Core
			Neuroscience Center, University of North Carolina
			https://www.med.unc.edu/neuroscience/core-facilities/neuro-microscopy/
			Aghajanian et al., 2021
		
		
			14
			Nikon Instruments
			Eclipse Ti2
			2
			Live-cell imaging of bacterial cells expressing GFP-PopZ
			Microscopy Resources on the North Quad (MicRoN)
			Harvard Medical School&nbsp;
			https://micron.hms.harvard.edu/
			Lim and Bernhardt 2019; Lim et al., 2019
		
		
			15
			Olympus/Biomedical Imaging Group (customized)
			TIRF Epifluorescence Structured light Microscope (TESM)/IX71
			3
			3D distribution of HIV-1 in the nucleus of human cells
			Biomedical Imaging Group
			Program in Molecular Medicine, University of Massachusetts Medical School
			https://trello.com/b/BQ8zCcQC/tirf-epi-fluorescence-structured-light-microscope
			Navaroli et al., 2012
		
		
			16
			Olympus/Computer Vision Laboratory (customized)
			3D BrightField Scanner/IX71
			3
			Transmitted light brightfield visualization of swimming spermatocytes
			Laboratorio Nacional de Microscopia Avanzada (LNMA) and Computer Vision Laboratory of the Institute of Biotechnology
			Universidad Nacional Autonoma de Mexico (UNAM)
			https://lnma.unam.mx/wp/
			Pimentel et al., 2012; Silva-Villalobos et al., 2014
		
	


Getting started

Use these videos to get started with using Micro-Meta App after installation into OMERO and downloading the example data files:


	Video 1
	Video 2


More information


For full information on how to use Micro-Meta App please utilize the following resources:


	Micro-Meta App website
	Full documentation
	Installation instructions
	Step-by-Step Instructions
	Tutorial Videos



Background

If you want to learn more about the importance of metadata and quality control to ensure full reproducibility, quality and scientific value in light microscopy, please take a look at our recent publications describing the development of community-driven light 4DN-BINA-OME Microscopy Metadata specifications Nature Methods and BioRxiv.org and our overview manuscript entitled A perspective on Microscopy Metadata: data provenance and quality control.

&nbsp;

&nbsp;

[https://zenodo.org/records/5847477](https://zenodo.org/records/5847477)

[https://doi.org/10.5281/zenodo.5847477](https://doi.org/10.5281/zenodo.5847477)


---

## Example Operetta Dataset

Nicolas Chiaruttini

Published 2023-07-17

Licensed CC-BY-4.0



This is a microscopy image dataset generated by the Perkin Elmer Operetta HCS microscope by of the user of the PTBIOP EPFL facility.

As of the 17th of July 2023, opening this file in ImageJ/Fiji using the BioFormats 6.14 library, this dataset generates a Null Pointer Exception.

A post on forum.image.sc is linked to this issue:

https://forum.image.sc/t/null-pointer-exception-in-perkin-elmer-operetta-dataset-with-bio-formats-6-14/83784

&nbsp;

&nbsp;

[https://zenodo.org/records/8153907](https://zenodo.org/records/8153907)

[https://doi.org/10.5281/zenodo.8153907](https://doi.org/10.5281/zenodo.8153907)


---

## Excel template for adding Key-Value Pairs to images

Thomas Zobel, Jens Wendt

Published 2024-10-30

Licensed CC-BY-4.0



This Excel Workbook contains some simple Macros to help with the generation of a .csv in the necessary format for Key-Value pair annotations of images in OMERO.
The format is tailored for the OMERO.web script&nbsp;"KeyVal_from_csv.py"&nbsp; (from the version &lt;=5.8.3 of the core omero-scripts).
Attached is also a video of Thomas Zobel, the head of the imaging core facility Uni M&uuml;nster, showcasing the use of the Excel workbook.The video uses a slightly older version of the workbook and OMERO, but the core functionality remains unchanged.
Please keep in mind, that the OMERO.web script(s) to handle Key-Value Pairs from/to .csv files will undergo a major change very soon.This might break the compatibility with the format used now for the generated .csv from the workbook.

[https://zenodo.org/records/14014252](https://zenodo.org/records/14014252)

[https://doi.org/10.5281/zenodo.14014252](https://doi.org/10.5281/zenodo.14014252)


---

## Forschungsdatenmanagement zukunftsfest gestalten – Impulse für die   Strukturevaluation der Nationalen Forschungsdateninfrastruktur (NFDI)

Steuerungsgremium Allianz-Schwerpunkt, Alexander von Humboldt Foundation, Deutsche Forschungsgemeinschaft, Fraunhofer Society, German Rectors' Conference, Leibniz Association, German National Academy of Sciences Leopoldina, German Academic Exchange Service, Helmholtz Association of German Research Centres, Max Planck Society

Published 2024-11-04

Licensed CC-BY-4.0



Arbeitspapier des Steuerungsgremiums des Allianz-Schwerpunkts "Digitalit&auml;t in der Wissenschaft"

[https://zenodo.org/records/14032908](https://zenodo.org/records/14032908)

[https://doi.org/10.5281/zenodo.14032908](https://doi.org/10.5281/zenodo.14032908)


---

## From Cells to Pixels: Bridging Biologists and  Image Analysts Through a Common Language

Elnaz Fazeli, Haase Robert, Doube Michael, Miura Kota, Legland David

Published 2024-08-16

Licensed CC-BY-4.0



Bioimaging has transformed our understanding of biological processes, yet extracting&nbsp;meaningful information from complex datasets remains a challenge, particularly for&nbsp;early career scientists. This paper proposes a simplified, systematic approach to&nbsp;bioimage analysis, focusing on categorizing commonly observed structures and&nbsp;shapes, and providing relevant analysis methods. Our approach includes illustrative&nbsp;examples and a visual flowchart, enabling researchers to define analysis objectives&nbsp;clearly. By understanding the diversity of bioimage structures and aligning them with&nbsp;appropriate analysis approaches, the framework empowers researchers to navigate&nbsp;bioimage datasets more efficiently. It also aims to foster a common language between&nbsp;researchers and analysts, thereby enhancing mutual understanding and facilitating&nbsp;effective communication.

[https://zenodo.org/records/13331351](https://zenodo.org/records/13331351)

[https://doi.org/10.5281/zenodo.13331351](https://doi.org/10.5281/zenodo.13331351)


---

## From Paper to Pixels: Navigation through your Research Data - presentations of speakers

Marcelo Zoccoler, Simon Bekemeier, Tom Boissonnet, Simon Parker, Luca Bertinetti, Marc Gentzel, Riccardo Massei, Cornelia Wetzker

Published 2024-06-10

Licensed CC-BY-4.0



The workshop introduced key topics of research data management (RDM) and the implementation thereof on a life science campus. Internal and external experts of RDM including scientists that apply chosen software tools presented the basic concepts and their implementation to a broad audience.&nbsp;
Talks covered general aspects of data handling and sorting, naming conventions, data storage repositories and archives, licensing of material, data and code management using git, data protection particularly regarding patient data and in genome sequencing and more. Two data management concepts and exemplary tools were highlighted in particular, being electronic lab notebooks with eLabFTW and the bio-image management software OMERO. Those were chosen because of three aspects: the large benefit of these management tools for a life science campus, their free availability as open source tools with the option of contribution of required functionalities and first existing use cases on campus already supported by CMCB/PoL IT.
Two talks by Robert Haase (ScaDS.AI/ Uni Leipzig) and Robert M&uuml;ller (Kontaktstelle Forschungsdaten, TU Dresden with contributions from Denise D&ouml;rfel) that opened the symposium were shared independently:
https://zenodo.org/records/11382341
https://zenodo.org/records/11261115
The workshop organization was funded by the CMCB/PoL Networking Grant and supported by the consortium NFDI4BIOIMAGE (funded by DFG grant number NFDI 46/1, project number 501864659).

Tags: Research Data Management

Content type: Slides

[https://zenodo.org/records/11548617](https://zenodo.org/records/11548617)

[https://doi.org/10.5281/zenodo.11548617](https://doi.org/10.5281/zenodo.11548617)


---

## Getting started with Python: intro and set-up a conda environment

Riccardo Massei

Published 2024-10-09

Licensed CC-BY-4.0



YMIA python event 2024
Presentation :&nbsp; "Getting started with Python: intro and set-up a conda environment with Dr. Riccardo Massei"

[https://zenodo.org/records/13908480](https://zenodo.org/records/13908480)

[https://doi.org/10.5281/zenodo.13908480](https://doi.org/10.5281/zenodo.13908480)


---

## Hackaton Results - Conversion of KNIME image analysis workflows to Galaxy

Riccardo Massei

Published 2024-03-07

Licensed CC-BY-4.0



Results of the project "Conversion of KNIME image analysis workflows to Galaxy" during the Hackathon "Image Analysis in Galaxy" (Freiburg 26 Feb - 01 Mar 2024)
&nbsp;

[https://zenodo.org/records/10793700](https://zenodo.org/records/10793700)

[https://doi.org/10.5281/zenodo.10793700](https://doi.org/10.5281/zenodo.10793700)


---

## High throughput & automated data analysis and data management workflow with Cellprofiler and OMERO

Sarah Weischer, Jens Wendt, Thomas Zobel

Licensed CC-BY-4.0



In this workshop a fully integrated data analysis solutions employing OMERO and commonly applied image analysis tools (e.g., CellProfiler, Fiji) using existing python interfaces (OMERO Python language bindings, ezOmero, Cellprofiler Python API) is presented.

Tags: OMERO, Data Analysis, Bioimage Analysis

Content type: Collection

[https://zenodo.org/doi/10.5281/zenodo.8139353](https://zenodo.org/doi/10.5281/zenodo.8139353)


---

## I3D:bio's OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training

Christian Schmidt, Michele Bortolomeazzi, Tom Boissonnet, Carsten Fortmann-Grote, Julia Dohle, Peter Zentis, Niraj Kandpal, Susanne Kunis, Thomas Zobel, Stefanie Weidtkamp-Peters, Elisa Ferrando-May

Published 2023-11-13

Licensed CC-BY-4.0



The open-source software OME Remote Objects (OMERO) is a data management software that allows storing, organizing, and annotating bioimaging/microscopy data. OMERO has become one of the best-known systems for bioimage data management in the bioimaging community. The Information Infrastructure for BioImage Data (I3D:bio) project facilitates the uptake of OMERO into research data management (RDM) practices at universities and research institutions in Germany. Since the adoption of OMERO into researchers' daily routines requires intensive training, a broad portfolio of training resources for OMERO is an asset. On top of using the OMERO guides curated by the Open Microscopy Environment Consortium (OME) team, imaging core facility staff at institutions where OMERO is used often prepare additional material tailored to be applicable for their own OMERO instances. Based on experience gathered in the Research Data Management for Microscopy group (RDM4mic) in Germany, and in the use cases in the I3D:bio project, we created a set of reusable, adjustable, openly available slide decks to serve as the basis for tailored training lectures, video tutorials, and self-guided instruction manuals directed at beginners in using OMERO. The material is published as an open educational resource complementing the existing resources for OMERO contributed by the community.

Tags: OMERO, Research Data Management, Nfdi4Bioimage, I3Dbio

Content type: Slide, Video

[https://zenodo.org/records/8323588](https://zenodo.org/records/8323588)

[https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU](https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU)

[https://doi.org/10.5281/zenodo.8323588](https://doi.org/10.5281/zenodo.8323588)


---

## Image Repository Decision Tree - Where do I deposit my imaging data

Kemmer, Isabel, Romdhane, Feriel, Euro-BioImaging ERIC

Published 2024-10-22

Licensed CC-BY-4.0



Depositing data in quality data repositories is one crucial step towards FAIR (Findable, Accessible, Interoperable, and Reusable) data. Accordingly, Euro-BioImaging strongly encourages sharing scientific imaging data in established, thematic repositories.&nbsp;
To guide you in the selection of appropriate repositories, we have created an overview of available repositories for different types of image data, including their scope and requirements. This decision tree guides you through questions about your data and directs you to the correct repository, and/or provides instructions for further processing to meet the critera of the repositories.&nbsp;
Three seperate trees are provided for different classes of imaging data: open bioimage data, preclinical data, and human imaging data.&nbsp;

[https://zenodo.org/records/13945179](https://zenodo.org/records/13945179)

[https://doi.org/10.5281/zenodo.13945179](https://doi.org/10.5281/zenodo.13945179)


---

## Insights and Impact From Five Cycles of Essential Open Source Software for Science

Kate Hertweck, Carly Strasser, Dario Taraborelli

Licensed CC-BY-4.0



Open source software (OSS) is essential for advancing scientific discovery, particularly in biomedical research, yet funding to support these vital tools has been limited. The Chan Zuckerberg Initiative's Essential Open Source Software for Science (EOSS) program has significantly contributed to this field by providing $51.8 million in funding over five years to support the maintenance, growth, and community engagement of critical OSS tools. The program has impacted scientific OSS projects by improving their technical outputs, community building, and sustainability practices, and fostering collaborations within the OSS community. Additionally, EOSS funding has enhanced diversity, equity, and inclusion within the OSS community, although changes in principal investigator demographics were not observed. The funded projects have had a substantial impact on biomedical research by improving the usability and accessibility of scientific software, which has led to increased adoption and advancements in various biomedical fields.

Tags: Open Source Software, Funding, Sustainability

Content type: Publication

[https://zenodo.org/records/11201216](https://zenodo.org/records/11201216)


---

## Insights from Acquiring Open Medical Imaging  Datasets for Foundation Model Development

Stefan Dvoretskii

Published 2024-04-10

Licensed CC-BY-4.0



[https://zenodo.org/records/11503289](https://zenodo.org/records/11503289)

[https://doi.org/10.5281/zenodo.11503289](https://doi.org/10.5281/zenodo.11503289)


---

## Insights from Acquiring Open Medical Imaging Datasets for Foundation Model Development

Stefan Dvoretskii

Published 2024-04-10

Licensed CC-BY-4.0



[https://zenodo.org/records/13380289](https://zenodo.org/records/13380289)

[https://doi.org/10.5281/zenodo.13380289](https://doi.org/10.5281/zenodo.13380289)


---

## Institutionalization and Collaboration as a Way of Addressing the Challenges Open Science Presents to Libraries: The University of Konstanz as a National Pioneer

Sophie Habinger, Maximilian Heber, Sonja Kralj, Emilia Mikautsch

Published 2024-07-09

Licensed CC-BY-4.0



The rise of Open Science (OS) and the academic community&rsquo;s needs that come with it bring about a range of challenges for academic libraries. To face these challenges, the University of Konstanz has created a competence unit called&nbsp;Team Open Science&nbsp;in the Communication, Information, Media Center (KIM) - a joint unit of library and IT infrastructure. The Team creates synergies within itself and across the library. In December 2023, it involved 12 staff members specialising in open access (OA), research data management (RDM), open educational resources (OER) and virtual research environments (VRE). It collaborates closely with other KIM departments. This submission shall serve as a best practice example for the impact of OS on research libraries and, beyond that, the impact of research libraries on universities.
To enhance and foster OS, the Team provides individual consultations, services and office hours for researchers. Here, it collaborates closely with other librarians like subject specialists and the Team University Publications. Along similar lines, the KIM offers institutional repositories for publications (KOPS) and research data (KonDATA). Beyond that, the Team provides solutions to host OA journals and analyses researchers&rsquo; VRE needs to decide on implementation options. In sum, the Team is the central OS contact point for the entire university, underlining the major role the library holds in making institutional impact.
Furthermore, the Team had the leading role in creating the University of Konstanz&rsquo; OS Policy, one of the first ones passed by a German university. This policy stands out because it encompasses various OS domains. It demands, among other things, that text publications be made OA and that research data be managed according to relevant subject-specific standards. If permissible and reasonable, it demands that research data should be made publicly available at the earliest possible time. Along these lines, the policy has a large impact on how the library handles closed access books and subscription-based journals. As a consequence, OA is pursued wherever possible, leading to the highest OA quota of all German universities. In that sense, the Team is a crucial driving force of OS in the University of Konstanz, which ties in with the library&rsquo;s major role of open research transformation.
Beyond the University of Konstanz, the Team is involved in a range of national and international projects collaborating with other libraries. On a national level, they lead the project open.access-network which provides an information platform for researchers and librarians and connects the German-speaking OA community through events like bar camps. The project KOALA-AV supports libraries in establishing consortial solutions for financing Diamond OA publications. Moreover, the Team is involved in the federal state initiative for RDM in Baden-W&uuml;rttemberg (bwFDM). Here, the Team is in charge of forschungsdaten.info, the German-speaking countries&rsquo; leading RDM information platform, which will be offered in English within the next years. Internationally, the Team cooperates with librarians and other OS professionals from the European Reform University Alliance (ERUA) and the European University for Well-Being (EUniWell), establishing formats for best practice exchange, such as monthly OS Meet-Ups.

[https://zenodo.org/records/12699637](https://zenodo.org/records/12699637)

[https://doi.org/10.5281/zenodo.12699637](https://doi.org/10.5281/zenodo.12699637)


---

## Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)

Anett Jannasch, Silke Tulok, Fuchs, Vanessa Aphaia Fiona, Tom Boissonnet, Christian Schmidt, Michele Bortolomeazzi, Gunar Fabig, Chukwuebuka Okafornta

Published 2024-06-26

Licensed CC-BY-4.0



This is a Key-Value pair template used for the annotation of datasets in OMERO. It is tailored for a research study (PERIKLES project) on the biocompatibility of newly designed biomaterials out of pericardial tissue for cardiovascular substitutes (https://doi.org/10.1063/5.0182672) conducted in the research department of Cardiac Surgery at the Faculty of Medicine Carl Gustav Carus at the Technische Universit&auml;t Dresden . A corresponding public example dataset is used in the publication "Setting up an institutional OMERO environment for bioimage data: perspectives from both facility staff and users" and is available here
(https://omero.med.tu-dresden.de/webclient/?show=dataset-1557).
The template is based on the REMBI recommendations (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015) and it was developed during the PoL-Bio-Image Analysis Symposium in Dresden Aug 28th- Sept 1th 2023.&nbsp;
With this template it is possible to create a csv-file, that can be used to annotate a dataset in OMERO using the annotation script (https://github.com/ome/omero-scripts/blob/develop/omero/annotation_scripts/).
How to use:
select and copy the data range containing Keys and Values
open a new excel sheet and paste transpose in column B1
type in A1 'dataset'
insert in A2 the exact name of the dataset, which should be annotated in OMERO
save the new excel sheet in csv- (comma seperated values) file format

Example can be seen in sheet 1 'csv import'. Important note; the code has to be 8-Bit UCS transformation format (UTF-8) otherwise several characters (for example &micro;, %,&deg;) might not be able to decode by the annotation script. We encountered this issue with old Microsoft Office versions (e.g. MS Office 2016).&nbsp;
Note: By filling the values in the excel sheet, avoid the usage of decimal delimiter.
&nbsp;
See cross reference:
10.5281/zenodo.12547566&nbsp;Key-Value pair template for annotation of datasets in OMERO (light- and electron microscopy data within the research group of Prof. Mueller-Reichert)
10.5281/zenodo.12578084 Key-Value pair template for annotation in OMERO for light microscopy data acquired with AxioScan7 - Core Facility Cellular Imaging (CFCI)

[https://zenodo.org/records/12546808](https://zenodo.org/records/12546808)

[https://doi.org/10.5281/zenodo.12546808](https://doi.org/10.5281/zenodo.12546808)


---

## Key-Value pair template for annotation of datasets in OMERO for light- and electron microscopy data within the research group of Prof. Müller-Reichert

Fabig, Gunar, Jannasch, Anett, Okafornta, Chukwuebuka, Boissonnet, Tom, Schmidt, Christian, Bortolomeazzi, Michele, Fuchs, Vanessa Aphaia Fiona, Koeckert, Maria, Poddar, Aayush, Vogel, Martin, Schwarzbach, Hanna-Margareta, Vogelsang, Andy, Gerlach, Michael, Nobst, Anja, Müller-Reichert, Thomas, Tulok, Silke

Published 2024-06-26

Licensed CC-BY-4.0



This are a two Key-Value pair templates used for the annotation of datasets in OMERO. They are tailored for light- and electron microcopy data for all research projects of the research group of Prof. T. Mueller-Reichert.&nbsp; All members of the Core Facility Cellular Imaging agreed for using these templates to annotate data in OMERO. Furthermore, there are a corresponding public example datasets used in the publication "Setting up an institutional OMERO environment for bioimage data: perspectives from both facility staff and users" and are available here:
https://omero.med.tu-dresden.de/webclient/?show=dataset-1552 --&gt; for lattice-light sheet microscopy
https://omero.med.tu-dresden.de/webclient/?show=dataset-1555--&gt; for electron microscopy data
That templates are based on the REMBI recommendations (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015) and were developed during the PoL-Bio-Image Analysis Symposium in Dresden Aug 28th- Sept 1st in 2023 and further adapeted during the usage of OMERO.&nbsp;
With every template it is possible to create a csv-file, that can be used to annotate a dataset in OMERO using the annotation script (https://github.com/ome/omero-scripts/blob/develop/omero/annotation_scripts/).
How to use:

fill the template with metadata
select and copy the data range containing the Keys and Values
open a new excel sheet and paste transpose in cell A1
Important: cell A1 contains always the name 'dataset' and cell A2 contains the exact name of the dataset, which should be annotated in OMERO
save the new excel sheet in csv-file (comma separated values) format

Examples can be seen in sheet 3 'csv_TOMO' and sheet 5 csv_TEM'.
Important note: The code has to be 8-Bit UCS transformation format (UTF-8) otherwise several characters (for example &micro;, %,&deg;) might be not able to decode by the annotation script. We encountered this issue with old Microsoft-Office versions (MS Office 2016).&nbsp;
Note: By filling the values in the excel sheet, avoid the usage of comma as decimal delimiter.
See cross reference:
10.5281/zenodo.12546808&nbsp;Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)
10.5281/zenodo.12578084 Key-Value pair template for annotation in OMERO for light microscopy data acquired with AxioScan7 - Core Facility Cellular Imaging (CFCI)
&nbsp;

[https://zenodo.org/records/12547566](https://zenodo.org/records/12547566)

[https://doi.org/10.5281/zenodo.12547566](https://doi.org/10.5281/zenodo.12547566)


---

## Kollaboratives Arbeiten und Versionskontrolle mit Git

Robert Haase

Published 2024-04-15

Licensed CC-BY-4.0



Gemeinsames Arbeiten im Internet stellt uns vor neue Herausforderungen: Wer hat eine Datei wann hochgeladen? Wer hat zum Inhalt beigetragen? Wie kann man Inhalte zusammenfuehren, wenn mehrere Mitarbeiter gleichzeitig Aenderungen gemacht haben? Das Versionskontrollwerkzeug git stellt eine umfassende Loesung fuer solche Fragen bereit. Die Onlineplatform github.com stellt nicht nur Softwareentwicklern weltweit eine git-getriebene Platform zur Verfuegung und erlaubt ihnen effektiv zusammen zu arbeiten. In diesem Workshop lernen wir:

Infuerung in FAIR-Prinzipien im Softwarecontext
Arbeiten mit git: Pull-requests
Aufloesen von Merge-Konflikten
Automatisiertes Archivieren von Inhalten nach Zenodo.org
Eigene Webseiten auf github.io publizieren


Tags: Research Data Management, FAIR-Principles, Git, Zenodo

Content type: Slides

[https://zenodo.org/records/10972692](https://zenodo.org/records/10972692)

[https://doi.org/10.5281/zenodo.10972692](https://doi.org/10.5281/zenodo.10972692)


---

## LEO: Linking ELN with OMERO

Escobar Diaz Guerrero, Rodrigo

Published 2024-05-08

Licensed CC-BY-4.0



First updates of LEO (Linking ELN with OMERO)

[https://zenodo.org/records/11146807](https://zenodo.org/records/11146807)

[https://doi.org/10.5281/zenodo.11146807](https://doi.org/10.5281/zenodo.11146807)


---

## Large Language Models: An Introduction for Life Scientists

Robert Haase

Published 2024-08-27

Licensed CC-BY-4.0



Large Language Models (LLMs) are changing the way how humans interact with computers. This has impact on all scientific fields by enabling new ways to achieve for example data analysis goals. In this talk we will go through an introduction to LLMs with respect to applications in the life sciences, focusing on bio-image analysis. We will see how to generate text and images using LLMs and how LLMs can extract information from reproducibly images through code-generation. We will go through selected prompt engineering techniques enabling scientists to tune the output of LLMs towards their scientific goal and how to do quality assurance in this context.

[https://zenodo.org/records/13379394](https://zenodo.org/records/13379394)

[https://doi.org/10.5281/zenodo.13379394](https://doi.org/10.5281/zenodo.13379394)


---

## Leitfaden zur digitalen Datensparsamkeit (mit Praxisbeispielen)

Heber, Maximilian, Jakob, Moritz, Landwehr, Matthias, Leendertse, Jan, Müller, Maximilian, Schneider, Gabriel, von Suchodoletz, Dirk, Ulrich, Robert

Published 2024-06-03

Licensed CC-BY-4.0



Im Zuge der stetig wachsenden Brisanz des Forschungsdatenmanagements fallen immer gr&ouml;&szlig;ere Mengen an Forschungsdaten an. Diese an sich begr&uuml;&szlig;enswerte Entwicklung f&uuml;hrt zu technischen und organisatorischen Herausforderungen nicht nur im Bereich der Speicherung von Forschungsdaten, sondern in allen Phasen des Forschungsdatenlebenszyklus. Der vorliegende Beitrag erl&auml;utert vor diesem Hintergrund m&ouml;gliche Motivationen hinter digitaler Datensparsamkeit mit Blick auf organisatorische, technische und ethische Kriterien, Datenschutz und Nachhaltigkeit. Anschlie&szlig;end werden vor dem Hintergrund zentraler Herausforderungen Umsetzungsvorschl&auml;ge f&uuml;r das Vorfeld sowie den Verlauf eines Forschungsvorhabens gemacht. Zudem werden grundlegende Empfehlungen zur digitalen Datensparsamkeit ausgesprochen.
Eine k&uuml;rzere Ausgabe des Leitfadens ist im Mai 2024 in der Zeitschrift o | bib erschienen: https://doi.org/10.5282/o-bib/6036 
Diese Ausgabe enth&auml;lt ein zus&auml;tzliches Kapitel (4.2) mit konkreten Praxisbeispielen.
Dieser Artikel wurde ins Englische &uuml;bersetzt:
Heber, M., Jakob, M., Landwehr, M., Leendertse, J., M&uuml;ller, M., Schneider, G., von Suchodoletz, D., &amp; Ulrich, R. (2024). A Users' Guide to Economical Digital Data Usage. Zenodo. https://doi.org/10.5281/zenodo.13752220

[https://zenodo.org/records/11445843](https://zenodo.org/records/11445843)

[https://doi.org/10.5281/zenodo.11445843](https://doi.org/10.5281/zenodo.11445843)


---

## LimeSeg Test Datasets

Sarah Machado, Vincent Mercier, Nicolas Chiaruttini

Published 2018-10-27

Licensed CC-BY-4.0



Image datasets from the&nbsp;publication :&nbsp;LimeSeg: A coarse-grained lipid membrane simulation for 3D image segmentation


	Vesicles.tif: spinning-disc confocal images of giant unilamellar vesicles
	HelaCell-FIBSEM.tif:&nbsp;a 3D Electron&nbsp;Microscopy (EM)&nbsp;dataset of nearly isotropic sections of a Hela cell, acquired with a focused ion beam scanning electron microscope (FIB-SEM). Sections are aligned with TrackEm2 (doi: ), without additional preprocessing.
	DrosophilaEggChamber.tif: point scanning confocal images of a Drosophila egg chamber. Channel&nbsp;1: cell nuclei &nbsp;stained with DAPI. Channel 2:&nbsp;cell membranes visualized with fused membrane proteins Nrg::GFP and Bsg::GFP.&nbsp;


Image metadata contains extra information including voxel sizes.

&nbsp;

[https://zenodo.org/records/1472859](https://zenodo.org/records/1472859)

[https://doi.org/10.5281/zenodo.1472859](https://doi.org/10.5281/zenodo.1472859)


---

## Linked (Open) Data for Microbial Population Biology

Carsten Fortmann-Grote

Published 2024-03-12

Licensed CC-BY-4.0



[https://zenodo.org/records/10808486](https://zenodo.org/records/10808486)

[https://doi.org/10.5281/zenodo.10808486](https://doi.org/10.5281/zenodo.10808486)


---

## Metadata Annotation Workflow for OMERO with Tabbles

Wendt Jens

Published 2023-09-04

Licensed CC-BY-4.0



Short presentation given at at PoL BioImage Analysis Symposium Dresden 2023

[https://zenodo.org/records/8314968](https://zenodo.org/records/8314968)

[https://doi.org/10.5281/zenodo.8314968](https://doi.org/10.5281/zenodo.8314968)


---

## MicroSam-Talks

Constantin Pape

Published 2024-05-23

Licensed CC-BY-4.0



Talks about Segment Anything for Microscopy: https://github.com/computational-cell-analytics/micro-sam.
Currently contains slides for two talks:

Overview of Segment Anythign for Microscopy given at the SWISSBIAS online meeting in April 2024
Talk about vision foundation models and Segment Anything for Microscopy given at Human Technopole as part of the EMBO Deep Learning Course in May 2024


Tags: Image Segmentation, Bioimage Analysis, Deep Learning

Content type: Slides

[https://zenodo.org/records/11265038](https://zenodo.org/records/11265038)

[https://doi.org/10.5281/zenodo.11265038](https://doi.org/10.5281/zenodo.11265038)


---

## My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud

Elnaz Fazeli

Published 2024-02-19

Licensed CC-BY-4.0



In these slides I introducemy journey through teaching bioimage analysis courses in different formats, from in person courses to online material. I have an overview of different training formats and comparing these for different audiences.&nbsp;

Tags: Teaching

Content type: Slides

[https://zenodo.org/records/10679054](https://zenodo.org/records/10679054)

[https://doi.org/10.5281/zenodo.10679054](https://doi.org/10.5281/zenodo.10679054)


---

## NFDI4BIOIMAGE

Carsten Fortmann-Grote

Published 2024-04-22

Licensed CC-BY-4.0



This presentation was given at the 2nd MPG-NFDI Workshop on April 18th.

[https://zenodo.org/records/11031747](https://zenodo.org/records/11031747)

[https://doi.org/10.5281/zenodo.11031747](https://doi.org/10.5281/zenodo.11031747)


---

## NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]

Riccardo Massei

Licensed CC-BY-4.0



NFDI4BIOIMAGE is a consortium within the framework of the National Research Data Infrastructure (NFDI) in Germany. In this talk, the consortium and the contribution to the work programme by the Helmholtz Centre for Environmental Research (UFZ) in Leipzig are outlined.

Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage

Content type: Slides

[https://zenodo.org/doi/10.5281/zenodo.8414318](https://zenodo.org/doi/10.5281/zenodo.8414318)


---

## NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and Bioimage Analysis

Marcelo Zoccoler

Published 2024-08-07

Licensed CC-BY-4.0



Bioimaging refers to a collection of methods to visualize the internal structures and mechanisms of living organisms. The fundamental tool, the microscope, has enabled seminal discoveries like that of the cell as the smallest unit of life, and continues to expand our understanding of biological processes. Today, we can follow the interaction of single molecules within nanoseconds in a living cell, and the development of complete small organisms like fish and flies over several days starting from the fertilized egg. Each image pixel encodes multiple spatiotemporal and spectral dimensions, compounding the massive volume and complexity of bioimage data. Proper handling of this data is indispensable for analysis and its lack has become a growing hindrance for the many disciplines of the life and biomedical sciences relying on bioimaging. No single domain has the expertise to tackle this bottleneck alone.
As a method-specific consortium, NFDI4BIOMAGE seeks to address these issues, enabling bioimaging data to be shared and re-used like they are acquired, i.e., independently of disciplinary boundaries. We will provide solutions for exploiting the full information content of bioimage data and enable new discoveries through sharing and re-analysis. Our RDM strategy is based on a robust needs analysis that derives not only from a community survey but also from over a decade of experience in German BioImaging, the German Society for Microscopy and Image Analysis. It considers the entire lifecycle of bioimaging data, from acquisition to archiving, including analysis and enabling re-use. A foundational element of this strategy is the definition of a common, cloud-compatible, and interoperable digital object that bundles binary images with their descriptive and provenance metadata. With members from plant biology to neuroscience, NFDI4BIOIMAGE will champion the standardization of bioimage data to create a framework that answers discipline-specific needs while ensuring communication and interoperability with data types and RDM systems across domains. Integration of bioimage data with, e.g., omics data as the basis for spatial omics, holds great promise for fields such as cancer medicine. Unlocking the full potential of bioimage data will rely on the development and broad availability of exceptional analysis tools and training sets. NFDI4BIOIMAGE will make these accessible and usable including cutting-edge AI-based methods in scalable cloud environments. NFDI4BIOIMAGE intersects with multiple NFDI consortia, most prominently with GHGA for linking image and genomics data and with DataPLANT on the definition of FAIR data objects. Last but not least, NFDI4BIOIMAGE is internationally well connected and represents the opportunity for German scientists to keep path with and have a voice in several international initiatives focusing on the FAIRification of bioimage data as one of the main challenges for the advancement of knowledge in the life and biomedical sciences.

[https://zenodo.org/records/13168693](https://zenodo.org/records/13168693)

[https://doi.org/10.5281/zenodo.13168693](https://doi.org/10.5281/zenodo.13168693)


---

## NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)

Mohamed Abdrabbou, Mehrnaz Babaki, Tom Boissonnet, Michele Bortolomeazzi, Eik Dahms, Vanessa Fuchs, A. F. Moritz Hoevels, Niraj Kandpal, Christoph Möhl, Joshua A. Moore, Astrid Schauss, Andrea Schrader, Torsten Stöter, Julia Thönnißen, Monica Valencia-S., H. Lukas Weil, Jens Wendt, Peter Zentis

Licensed CC-BY-4.0



This repository documents the first NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon), where topics like 'Interoperability', 'REMBI / Mapping', and 'Neuroglancer (OMERO / zarr)' were explored through collaborative discussions and workflow sessions, culminating in reports that bridge NFDI4Bioimage to DataPLANT. Funded by various DFG initiatives, this event emphasized documentation and use cases, contributing preparatory work for future interoperability projects at the 2nd de.NBI BioHackathon in Bielefeld.

Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage

Content type: Github Repository

[https://zenodo.org/doi/10.5281/zenodo.10609770](https://zenodo.org/doi/10.5281/zenodo.10609770)


---

## NFDI4Bioimage Calendar 2024 October; original image

Christian Jüngst, Peter Zentis

Published 2024-09-25

Licensed CC-BY-4.0



Raw microscopy image from the NFDI4Bioimage calendar October 2024

[https://zenodo.org/records/13837146](https://zenodo.org/records/13837146)

[https://doi.org/10.5281/zenodo.13837146](https://doi.org/10.5281/zenodo.13837146)


---

## New Kid on the (NFDI) Block: NFDI4BIOIMAGE  - A National Initiative for FAIR Data Management in Bioimaging and Bioimage Analysis

Cornelia Wetzker

Published 2024-10-29

Licensed CC-BY-4.0



The poster introduces the consortium NFDI4BIOIMAGE with its central objectives, provides an overview of challenges in bioimage data handling, sharing and analysis and lists support options by the consortium through its data stewardship team.
It is part of the work of the German consortium NFDI4BIOIMAGE funded by the Deutsche Forschungsgemeinschaft (DFG grant number NFDI 46/1, project number 501864659) and has been presented at the conference FDM@Campus held in G&ouml;ttingen September 23-25, 2024.

[https://zenodo.org/records/14006558](https://zenodo.org/records/14006558)

[https://doi.org/10.5281/zenodo.14006558](https://doi.org/10.5281/zenodo.14006558)


---

## Nextflow: Scalable and reproducible scientific workflows

Floden Evan, Di Tommaso Paolo

Published 2020-12-17

Licensed CC-BY-4.0



Nextflow is an open-source workflow management system that prioritizes portability and reproducibility. It enables users to develop and seamlessly scale genomics workflows locally, on HPC clusters, or in major cloud providers&rsquo; infrastructures. Developed since 2014 and backed by a fast-growing community, the Nextflow ecosystem is made up of users and developers across academia, government and industry. It counts over 1M downloads and over 10K users worldwide.

Tags: Workflow Engine

Content type: Slide

[https://zenodo.org/records/4334697](https://zenodo.org/records/4334697)

[https://doi.org/10.5281/zenodo.4334697](https://doi.org/10.5281/zenodo.4334697)


---

## Open Science, Sharing & Licensing

Robert Haase

Published 2024-04-18

Licensed CC-BY-4.0



Wir tauchen ein in die Welt der Open Science und definieren Begriffe wie Open Source, Open Access und die FAIR-Prinzipien (Findable, Accessible, Interoperable and Reuasable). Wir diskutieren, wie diese Methoden der [wissenschaftlichen] Kommunikation und des Datenmanagements die Welt ver&auml;ndern und wie wir sie praktisch in unsere Arbeit integrieren k&ouml;nnen. Dabei spielen Aspekte wie Copyright und Lizenzierung eine wichtige Rolle.

Tags: Research Data Management, Open Access, FAIR-Principles, Licensing

Content type: Slides

[https://zenodo.org/records/10990107](https://zenodo.org/records/10990107)

[https://doi.org/10.5281/zenodo.10990107](https://doi.org/10.5281/zenodo.10990107)


---

## QuPath: Open source software for analysing (awkward) images

Peter Bankhead

Published 2020-12-16

Licensed CC-BY-4.0



Slides from the CZI/EOSS online meeting in December 2020.

Tags: Bioimage Analysis

Content type: Slide

[https://zenodo.org/records/4328911](https://zenodo.org/records/4328911)

[https://doi.org/10.5281/zenodo.4328911](https://doi.org/10.5281/zenodo.4328911)


---

## RDF as a bridge to domain-platforms like OMERO, or There and back again.

Josh Moore, Andra Waagmeester, Kristina Hettne, Katherine Wolstencroft, Susanne Kunis

Licensed CC-BY-4.0



In 2005, the first version of OMERO stored RDF natively. However, just a year after the 1.0 release of RDF, performance considerations led to the development of a more traditional SQL approach for OMERO. A binary protocol makes it possible to query and retrieve metadata but the resulting information cannot immediately be combined with other sources. This is the adventure of rediscovering the benefit of RDF triples as a -- if not the -- common exchange mechanism.

Tags: Research Data Management, FAIR-Principles, Bioimage Analysis

Content type: Slides

[https://zenodo.org/doi/10.5281/zenodo.10687658](https://zenodo.org/doi/10.5281/zenodo.10687658)


---

## RESEARCH DATA MANAGEMENT on Campus and in NFDI4BIOIMAGE

Cornelia Wetzker, Michael Schlierf

Published 2024-08-29

Licensed CC-BY-4.0



The poster is part of the work of the German consortium NFDI4BIOIMAGE funded by the Deutsche Forschungsgemeinschaft (DFG grant number NFDI 46/1, project number 501864659).

[https://zenodo.org/records/13684187](https://zenodo.org/records/13684187)

[https://doi.org/10.5281/zenodo.13684187](https://doi.org/10.5281/zenodo.13684187)


---

## Report on a pilot study:  Implementation of OMERO for  microscopy data management

Silke Tulok, Gunar Fabig, Andy Vogelsang, Thomas Kugel, Thomas Müller-Reichert

Published 2023-11-10

Licensed CC-BY-4.0



The Core Facility Cellular Imaging (CFCI) at the Faculty of Medicine Carl Gustav Carus (TU Dresden) is currently running a pilot project for testing the use and handling of the OMERO software. This is done together with interested users of the imaging facility and a research group. Currently, we are pushing forward this pilot study on a small scale without any data steward. Our experiences argue so far for giving data management issues into the hands of dedicated personnel not fully involved in research projects. As funding agencies will ask for higher and higher standards for implementing FAIRdata principles in the future, this will be a releva

[https://zenodo.org/records/10103316](https://zenodo.org/records/10103316)

[https://doi.org/10.5281/zenodo.10103316](https://doi.org/10.5281/zenodo.10103316)


---

## Research Data Management Seminar - Slides

Della Chiesa, Stefano

Published 2022-05-18

Licensed CC-BY-4.0



This Research Data Management (RDM) Slides introduce to the multidisciplinary knowledge and competencies required to address policy compliance and research data management best practices throughout a project lifecycle, and beyond it.


	Module 1 - Introduces the RDM giving its context in the Research Data Governance
	Module 2 - Illustrates the most important RDM policies and principles
	Module 3 - Provides the most relevant RDM knowledge bricks
	Module 4 - Discuss the Data Management Plans (DMPs), examples, templates and guidance


&nbsp;

Tags: Research Data Management

Content type: Slide

[https://zenodo.org/record/6602101](https://zenodo.org/record/6602101)

[https://doi.org/10.5281/zenodo.6602101](https://doi.org/10.5281/zenodo.6602101)


---

## So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten

Pia Voigt

Published 2024-05-30

Licensed CC-BY-4.0



Der Umgang mit personenbezogenen Daten stellt Forschende oft vor rechtliche Herausforderungen: Unter welchen Bedingungen d&uuml;rfen personenbezogene Daten verarbeitet werden? Welche Voraussetzungen m&uuml;ssen erf&uuml;llt sein und welche Strategien k&ouml;nnen angewendet werden, um Daten sicher speichern, verarbeiten, teilen und aufbewahren zu k&ouml;nnen? Mit Hilfe dieses Foliensatzes erhalten Sie Einblicke in datenschutzrechtliche Aspekte beim Umgang mit Ihren Forschungsdaten.&nbsp;

Tags: Research Data Management, Data Protection, FAIR-Principles

Content type: Slides

[https://zenodo.org/records/11396199](https://zenodo.org/records/11396199)

[https://doi.org/10.5281/zenodo.11396199](https://doi.org/10.5281/zenodo.11396199)


---

## Stackview sliceplot example data

Robert Haase

Published 2024-11-03

Licensed CC-BY-4.0



This is a dataset of PNG images of [Bio-Image Data Science teaching slides](https://zenodo.org/records/12623730). The png_umap.yml file contains a list of all images and a dimensionality reduced embedding (Uniform Manifold Approximation Projection, UMAP) made using OpenAI's text-embedding-ada-002 model.
A notebook for visualizing this data is published here: https://github.com/haesleinhuepf/stackview/blob/main/docs/sliceplot.ipynb

[https://zenodo.org/records/14030307](https://zenodo.org/records/14030307)

[https://doi.org/10.5281/zenodo.14030307](https://doi.org/10.5281/zenodo.14030307)


---

## Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data

Susanne Kunis

Published 2022-08-24

Licensed CC-BY-4.0



guided walkthrough of poster at https://doi.org/10.5281/zenodo.6821815

which provides an overview of contexts, frameworks, and models from the world of bioimage data as well as metadata and the techniques for structuring this data as Linked Data.

You can also watch the video in the browser on the I3D:bio website.

Tags: Nfdi4Bioimage, Research Data Management

Content type: Video

[https://zenodo.org/record/7018929](https://zenodo.org/record/7018929)

[https://doi.org/10.5281/zenodo.7018929](https://doi.org/10.5281/zenodo.7018929)


---

## Sustainable Data Stewardship

Stefano Della Chiesa

Published 2024-03-25

Licensed CC-BY-4.0



 These slides were presented at the 2. SaxFDM-Beratungsstammtisch and delve into the strategic integration of Research Data Management (RDM) within research organizations. The Leibniz IOER presented an insightful overview of RDM activities and approaches, emphasizing the criticality of embedding RDM strategically within research institutions. The presentation showcases some best practices in RDM implementation through practical examples, offering valuable insights for optimizing data stewardship processes.

Tags: Research Data Management, Data Stewardship

Content type: Slides

[https://zenodo.org/records/10942559](https://zenodo.org/records/10942559)

[https://doi.org/10.5281/zenodo.10942559](https://doi.org/10.5281/zenodo.10942559)


---

## Test Dataset for Whole Slide Image Registration

Romain Guiet, Nicolas Chiaruttini

Published 2021-04-12

Licensed CC-BY-4.0



Mouse duodenum fixed in 4% PFA overnight at 4&deg;C, processed for paraffin infiltration using a standard histology procedure and cut at 4 microns were dewaxed, rehydrated, permeabilized with 0.5% Triton X-100 in PBS 1x and stained with Azide - Alexa Fluor 555 (Thermo Fisher) to detect EdU and DAPI for nuclei. The images were taken using a Leica DM5500 microscope with a 40X N.A.1 objective (black&amp;white camera: DFC350FXR2, pixel dimension: 0.161 microns). Next, the slide was unmounted and stained using the fully automated Ventana Discovery xT autostainer (Roche Diagnostics, Rotkreuz, Switzerland). All steps were performed on automate with Ventana solutions. Sections were pretreated with heat using the CC1 solution under mild conditions. The primary rat anti BrDU (clone: BU1/75 (ICR1), Serotec, diluted 1:300) was incubated 1 hour at 37&deg;C. After incubation with a donkey anti rat biotin diluted 1:200 (Jackson ImmunoResearch Laboratories), chromogenic revelation was performed with DabMap kit. The section was counterstained with Harris hematoxylin (J.T. Baker) before a second round of imaging on DM5500 PL Fluotar 40X N.A.1.0 oil (color camera: DFC 320 R2, pixel dimension: 0.1725 microns). Before acquisition, a white-balance as well as a shading correction is performed according to Leica LAS software wizard. The fluorescence and DAB images were converted in ome.tiff multiresolution file with the kheops Fiji Plugin.

Sampled prepared in the EPFL histology core facility by Nathalie M&uuml;ller and Gian-Filippo Mancini.

Associated documents:


	https://c4science.ch/w/bioimaging_and_optics_platform_biop/teaching/dab-intensity/
	https://imagej.net/plugins/bdv/warpy/warpy


This document contains a full QuPath project with an example of registered image.

&nbsp;

[https://zenodo.org/records/5675686](https://zenodo.org/records/5675686)

[https://doi.org/10.5281/zenodo.5675686](https://doi.org/10.5281/zenodo.5675686)


---

## The role of Helmholtz Centers in NFDI4BIOIMAGE - A national consortium enhancing FAIR data management for microscopy and bioimage analysis

Riccardo Massei, Christian Schmidt, Michele Bortolomeazzi, Julia Thoennissen, Jan Bumberger, Timo Dickscheid, Jan-Philipp Mallm, Elisa Ferrando-May

Published 2024-06-06

Licensed CC-BY-4.0



Germany&rsquo;s National Research Data Infrastructure (NFDI) aims to establish a sustained, cross-disciplinary research data management (RDM) infrastructure that enables researchers to handle FAIR (findable, accessible, interoperable, reusable) data. While&nbsp;FAIR principles have been&nbsp;adopted by funders, policymakers, and publishers, their practical implementation remains an ongoing effort. In the field of bio-imaging, harmonization of&nbsp;data formats, metadata ontologies, and open data repositories is necessary&nbsp;to achieve FAIR data.&nbsp;The NFDI4BIOIMAGE was established&nbsp;to address these issues and&nbsp;develop tools and best practices to facilitate FAIR microscopy and image analysis data in alignment with international community activities. The&nbsp;consortium operates through its Data Stewards team to provide expertise and direct support to help overcome RDM challenges. The three Helmholtz Centers in NFDI4BIOIMAGE aim to collaborate closely with other centers and initiatives, such as HMC, Helmholtz AI, and HIP. Here we present NFDI4BIOIMAGE&rsquo;s work and its significance for research in Helmholtz and beyond

[https://zenodo.org/records/11501662](https://zenodo.org/records/11501662)

[https://doi.org/10.5281/zenodo.11501662](https://doi.org/10.5281/zenodo.11501662)


---

## Thinking data management on different scales

Susanne Kunis

Licensed CC-BY-4.0



Presentation given at PoL BioImage Analysis Symposium Dresden 2023

Tags: Research Data Management, Nfdi4Bioimage

Content type: Slides

[https://zenodo.org/doi/10.5281/zenodo.8329305](https://zenodo.org/doi/10.5281/zenodo.8329305)


---

## Towards Preservation of Life Science Data with NFDI4BIOIMAGE

Robert Haase

Published 2024-09-03

Licensed CC-BY-4.0



This talk will present the initiatives of the NFDI4BioImage consortium aimed at the long-term preservation of life science data. We will discuss our efforts to establish metadata standards, which are crucial for ensuring data reusability and integrity. The development of sustainable infrastructure is another key focus, enabling seamless data integration and analysis in the cloud. We will take a look at how we manage training materials and communicate with our community. Through these actions, NFDI4BioImage seeks to enable FAIR bioimage data management for German researchers, across disciplines and embedded in the international framework.

[https://zenodo.org/records/13640979](https://zenodo.org/records/13640979)

[https://doi.org/10.5281/zenodo.13640979](https://doi.org/10.5281/zenodo.13640979)


---

## Towards Transparency and Knowledge Exchange in AI-assisted Data Analysis Code Generation

Robert Haase

Published 2024-10-14

Licensed CC-BY-4.0



The integration of Large Language Models (LLMs) in scientific research presents both opportunities and challenges for life scientists. Key challenges include ensuring transparency in AI-generated content and facilitating efficient knowledge exchange among researchers. These issues arise from the in-transparent nature of AI-driven code generation and the informal sharing of AI insights, which may hinder reproducibility and collaboration. This paper introduces git-bob, an innovative AI-assistant designed to address these challenges by fostering an interactive and transparent collaboration platform within GitHub. By enabling seamless dialogue between humans and AI, git-bob ensures that AI contributions are transparent and reproducible. Moreover, it supports collaborative knowledge exchange, enhancing the interdisciplinary dialogue necessary for cutting-edge life sciences research. The open-source nature of git-bob further promotes accessibility and customization, positioning it as a vital tool in employing LLMs responsibly and effectively within scientific communities.

[https://zenodo.org/records/13928832](https://zenodo.org/records/13928832)

[https://doi.org/10.5281/zenodo.13928832](https://doi.org/10.5281/zenodo.13928832)


---

## Train-the-Trainer Concept on Research Data Management

Katarzyna Biernacka, Maik Bierwirth, Petra Buchholz, Dominika Dolzycka, Kerstin Helbig, Janna Neumann, Carolin Odebrecht, Cord Wiljes, Ulrike Wuttke

Published 2020-11-04

Licensed CC-BY-4.0



Within the project FDMentor, a German Train-the-Trainer Programme on Research Data Management (RDM) was developed and piloted in a series of workshops. The topics cover many aspects of research data management, such as data management plans and the publication of research data, as well as didactic units on learning concepts, workshop design and a range of didactic methods.

After the end of the project, the concept was supplemented and updated by members of the Sub-Working Group Training/Further Education (UAG Schulungen/Fortbildungen) of the DINI/nestor Working Group Research Data (DINI/nestor-AG Forschungsdaten). The newly published English version of the Train-the-Trainer Concept contains the translated concept, the materials and all methods of the Train-the-Trainer Programme. Furthermore, additional English references and materials complement this version.

Tags: Research Data Management

Content type: Book

[https://zenodo.org/record/4071471](https://zenodo.org/record/4071471)

[https://doi.org/10.5281/zenodo.4071471](https://doi.org/10.5281/zenodo.4071471)


---

## Welcome to BioImage Town

Josh Moore

Licensed CC-BY-4.0



Welcome at NFDI4BIOIMAGE All-Hands Meeting in Düsseldorf, Germany, October 16, 2023

Tags: OMERO, Bioimage Analysis, Nfdi4Bioimage

Content type: Slides

[https://zenodo.org/doi/10.5281/zenodo.10008464](https://zenodo.org/doi/10.5281/zenodo.10008464)


---

## Who you gonna call? - Data Stewards to the rescue

Fuchs, Vanessa Aphaia Fiona, Jens Wendt, Maximilian Müller, Mohsen Ahmadi, Riccardo Massei, Cornelia Wetzker

Published 2024-03-01

Licensed CC-BY-4.0



The Data Steward Team of the NFDI4BIOIMAGE consortium presents themselves and the services (including the Helpdesk) that we offer.

[https://zenodo.org/records/10730424](https://zenodo.org/records/10730424)

[https://doi.org/10.5281/zenodo.10730424](https://doi.org/10.5281/zenodo.10730424)


---

## [BINA CC] Scalable strategies for a next-generation of FAIR bioimaging

Josh Moore

Published 2024-09-24

Licensed CC-BY-4.0



Presented at https://www.bioimagingnorthamerica.org/events/bina-2024-community-congress/

[https://zenodo.org/records/13831274](https://zenodo.org/records/13831274)

[https://doi.org/10.5281/zenodo.13831274](https://doi.org/10.5281/zenodo.13831274)


---

## [CORDI 2023] Zarr: A Cloud-Optimized Storage for Interactive Access of Large Arrays

Josh Moore

Licensed CC-BY-4.0



For decades, the sharing of large N-dimensional datasets has posed issues across multiple domains. Interactively accessing terabyte-scale data has previously required significant server resources to properly prepare cropped or down-sampled representations on the fly. Now, a cloud-native chunked format easing this burden has been adopted in the bioimaging domain for standardization. The format — Zarr — is potentially of interest for other consortia and sections of NFDI.

Tags: Research Data Management, Bioimage Analysis, Data Science

Content type: Poster

[https://zenodo.org/doi/10.5281/zenodo.8340247](https://zenodo.org/doi/10.5281/zenodo.8340247)


---

## [ELMI 2024]  AI's Dirty Little Secret: Without
FAIR Data, It's Just Fancy Math

Josh Moore, Susanne Kunis

Published 2024-05-21

Licensed CC-BY-4.0



Poster presented at the European Light Microscopy Initiative meeting in Liverpool (https://www.elmi2024.org/)

[https://zenodo.org/records/11235513](https://zenodo.org/records/11235513)

[https://doi.org/10.5281/zenodo.11235513](https://doi.org/10.5281/zenodo.11235513)


---

## [ELMI 2024] AI's Dirty Little Secret: Without FAIR Data, It's Just Fancy Math

Josh Moore, Susanne Kunis

Licensed CC-BY-4.0



Poster presented at the European Light Microscopy Initiative meeting in Liverpool (https://www.elmi2024.org/)

Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage

Content type: Poster

[https://zenodo.org/doi/10.5281/zenodo.11235512](https://zenodo.org/doi/10.5281/zenodo.11235512)


---

## [GBI EOE VII] Five (or ten) must-have items for making IT infrastructure for managing bioimage data

Josh Moore

Published 2024-05-26

Licensed CC-BY-4.0



Presentation made to the GBI Image Data Management Working Group during the 7th Exchange of Experience in Uruguay.

[https://zenodo.org/records/11318151](https://zenodo.org/records/11318151)

[https://doi.org/10.5281/zenodo.11318151](https://doi.org/10.5281/zenodo.11318151)


---

## [GBI EoE IX] NFDI4BIOIMAGE
National Research Data Infrastructure 
for Microscopy and BioImage Analysis

Josh Moore

Published 2024-10-29

Licensed CC-BY-4.0



Presented at https://globalbioimaging.org/exchange-of-experience/exchange-of-experience-ix in Okazaki, Japan.

[https://zenodo.org/records/14001388](https://zenodo.org/records/14001388)

[https://doi.org/10.5281/zenodo.14001388](https://doi.org/10.5281/zenodo.14001388)


---

## [I2K] Scalable strategies for a next-generation of FAIR bioimaging

Josh Moore

Published 2024-10-25

Licensed CC-BY-4.0



or, "OME-Zarr: 'even a talk on formats [can be] interesting'"
Presented at https://events.humantechnopole.it/event/1/

[https://zenodo.org/records/13991322](https://zenodo.org/records/13991322)

[https://doi.org/10.5281/zenodo.13991322](https://doi.org/10.5281/zenodo.13991322)


---

## [N4BI AHM] Welcome to BioImage Town

Josh Moore

Published 2023-10-16

Licensed CC-BY-4.0



Keynote at the NFDI4BIOIMAGE All-Hands Meeting in Düsseldorf, Germany, October 16, 2023.

Tags: Research Data Management

Content type: Slides

[https://zenodo.org/records/10008465](https://zenodo.org/records/10008465)

[https://doi.org/10.5281/zenodo.10008465](https://doi.org/10.5281/zenodo.10008465)


---

## [SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard

Josh Moore, Susanne Kunis

Licensed CC-BY-4.0



Poster presented at Semantic Web Applications and Tools for Health Care and Life Sciences (SWAT4HCLS 2023), Feb 13--16, 2023, Basel, Switzerland. NFDI4BIOIMAGE is a newly established German consortium dedicated to the FAIR representation of biological imaging data. A key deliverable is the definition of a semantically-compatible FAIR image object integrating RDF metadata with web-compatible storage of large n-dimensional binary data in OME-Zarr. We invite feedback from and collaboration with other endeavors during the soon-to-begin 5 year funding period.

Tags: Research Data Management, FAIR-Principles, Nfdi4Bioimage

Content type: Poster

[https://zenodo.org/doi/10.5281/zenodo.7928332](https://zenodo.org/doi/10.5281/zenodo.7928332)


---

## [Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure

Christian Schmidt

Published 2024-04-10

Licensed CC-BY-4.0



Short Talk about the NFDI4BIOIMAGE consortium presented at the RDM in (Bio-)Medicine Information Event on April 10th, 2024, organized C&sup3;RDM &amp; ZB MED.

[https://zenodo.org/records/10939520](https://zenodo.org/records/10939520)

[https://doi.org/10.5281/zenodo.10939520](https://doi.org/10.5281/zenodo.10939520)


---

## [Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg

Tom Boissonnet, Bettina Hagen, Susanne Kunis, Christian Schmidt, Stefanie Weidtkamp-Peters

Published 2024-11-18

Licensed CC-BY-4.0



Fit for OMERO: How imaging facilities and IT departments work together to enable RDM for bioimaging
Description:
Research data management (RDM) in bioimaging is challenging because of large file sizes, heterogeneous file formats and the variability of imaging methods. The image data management system OMERO (OME Remote Objects) allows for centralized and secure storage, organization, annotation, and interrogation of microscopy data by researchers. It is an internationally well-supported open-source software tool that has become one of the best-known image data management tools among bioimaging scientists. Nevertheless, the&nbsp;de novo&nbsp;setup of OMERO at an institute is a multi-stakeholder process that demands time, funds, organization and iterative implementation. In this workshop, participants learn how to begin setting up OMERO-based image data management at their institution. The topics include:

Stakeholder identification at the university / research institute
Process management, time line expectations, and resources planning
Learning about each other&lsquo;s perspectives on chances and challenges for RDM
Funding opportunities and strategies for IT and imaging core facilities
Hands-on: Setting up an OMERO server in a virtual machine environment

Target audience:
This workshop was directed at universities and research institutions who consider or plan to implement OMERO, or are in an early phase of implementation. This workshop was intended for teams from IT departments and imaging facilities to participate together with one person from the IT department, and one person from the imaging core facility at the same institution.
The trainers:

Prof. Dr. Stefanie Weidtkamp-Peters (Imaging Core Facility Head, Center for Advanced Imaging, Heinrich Heine University of D&uuml;sseldorf)
Dr. Susanne Kunis (Software architect, OMERO administrator, metadata specialist, University of Osnabr&uuml;ck)
Dr. Tom Boissonnet (OMERO admin and image metadata specialist, Center for Advanced Imaging, Heinrich Heine University of D&uuml;sseldorf)
Dr. Bettina Hagen&nbsp;(IT Administration and service specialist, Max Planck Institute for the Biology of Ageing, Cologne)&nbsp;
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center (DKFZ), Heidelberg)

Time and place
The format was a two-day, in-person workshop (October 16-17, 2024).&nbsp;Location: Heidelberg, Germany
Workshop learning goals

Learn the steps to establish a local RDM environment fit for bioimaging data
Create a network of IT experts and bioimaging specialists for bioimage RDM across institutions
Establish a stakeholder process management for installing OMERO-based RDM
Learn from each other, leverage different expertise
Learn how to train users, establish sustainability strategies, and foster FAIR RDM for bioimaging at your institution


[https://zenodo.org/records/14178789](https://zenodo.org/records/14178789)

[https://doi.org/10.5281/zenodo.14178789](https://doi.org/10.5281/zenodo.14178789)


---

## [Workshop] Bioimage data management and analysis with OMERO

Riccardo Massei, Michele Bortolomeazzi, Christian Schmidt

Published 2024-05-13

Licensed CC-BY-4.0



Here we share the material used in a workshop held on May 13th, 2024, at the German Cancer Research Center in Heidelberg (on-premise)
Description:Microscopy experiments generate information-rich, multi-dimensional data, allowing us to investigate biological processes at high spatial and temporal resolution. Image processing and analysis is a standard procedure to retrieve quantitative information from biological imaging. Due to the complex nature of bioimaging files that often come in proprietary formats, it can be challenging to organize, structure, and annotate bioimaging data throughout a project. Data often needs to be moved between collaboration partners, transformed into open formats, processed with a variety of software tools, and exported to smaller-sized images for presentation. The path from image acquisition to final publication figures with quantitative results must be documented and reproducible.
In this workshop, participants learn how to use OMERO to organize their data and enrich the bioimage data with structured metadata annotations.We also focus on image analysis workflows in combination with OMERO based on the Fiji/ImageJ software and using Jupyter Notebooks. In the last part, we explore how OMERO can be used to create publication figures and prepare bioimage data for publication in a suitable repository such as the Bioimage Archive.
Module 1&nbsp;(9 am - 10.15 am):&nbsp;Basics of OMERO, data structuring and annotation
Module 2&nbsp;(10.45 am - 12.45 pm):&nbsp;OMERO and Fiji
Module 3&nbsp;(1.45 pm - 3.45 pm):&nbsp;OMERO and Jupyter Notebooks
Module 4&nbsp;(4.15 pm - 6. pm):&nbsp;Publication-ready figures and data with OMERO
The target group for this workshopThis workshop is directed at researchers at all career levels who plan to or have started to use OMERO for their microscopy research data management.&nbsp;We encourage the workshop participants to bring example data from their research to discuss suitable metadata annotation for their everyday practice.
Prerequisites:Users should bring their laptops and have access to the internet through one of the following options:- eduroam- institutional WiFi- VPN connection to their institutional networks to access OMERO
Who are the trainers?
Dr. Riccardo Massei (Helmholtz-Center for Environmental Research, UFZ, Leipzig) - Data Steward for Bioimaging Data in NFDI4BIOIMAGE
Dr. Michele Bortolomeazzi (DKFZ, Single cell Open Lab, bioimage data specialist, bioinformatician, staff scientist in the NFDI4BIOIMAGE project)
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center, Heidelberg, Project Coordinator of the NFDI4BIOIMAGE project)

[https://zenodo.org/records/11350689](https://zenodo.org/records/11350689)

[https://doi.org/10.5281/zenodo.11350689](https://doi.org/10.5281/zenodo.11350689)


---

## [Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO

Vanessa Fuchs, Fiona Aphaia, Christian Schmidt, Tom Boissonnet

Published 2024-05-06

Licensed CC-BY-4.0



Description
Microscopy experiments generate information-rich, multi-dimensional data, allowing us to investigate biological processes at high spatial and temporal resolution. Image processing and analysis is a standard procedure to retrieve quantitative information from biological imaging. Due to the complex nature of bioimaging files that often come in proprietary formats, it can be challenging to organize, structure, and annotate bioimaging data throughout a project. Data often needs to be moved between collaboration partners, transformed into open formats, processed with a variety of software tools, and exported to smaller-sized images for presentation. The path from image acquisition to final publication figures with quantitative results must be documented and reproducible.
In this workshop, participants learn how to use structured metadata annotations in the image data management platform OMERO (OME Remote Objects) to optimize their data handling. This strategy helps both with organizing data for easier processing and analysis and for the preparation of data publication in journal manuscripts and in public repositories such as the BioImage Archive. Participants learn the principles of leveraging object-oriented data organization in OMERO to enhance findability and usability of their data, also in collaborative settings. The integration of OMERO with image analysis tools, in particular ImageJ/Fiji, will be trained. Moreover, users learn about community-accepted metadata checklists (REMBI) to enrich the value of their data toward reproducibility and reusability. In this workshop, we will provide hands-on training and recommendations on:

Structured metadata annotation features in OMERO and how to use them
Types of metadata in bioimaging: Technical metadata, sample metadata, analysis metadata
The use of ontologies and terminologies for metadata annotation
REMBI, the recommended metadata for biological images
Metadata-assisted image analysis streamlining
Tools for metadata annotation in OMERO

The target group for this workshop
This workshop is directed at researchers at all career levels who have started using OMERO for their microscopy research data management. We encourage the workshop participants to bring example data from their research to discuss suitable metadata annotation for their everyday practice.
Who are the trainers (see trainer description below for more details)

Dr. Vanessa Fuchs (NFDI4BIOIMAGE Data Steward, Center for Advanced Imaging, Heinrich-Heine University of D&uuml;sseldorf)
Dr. Tom Boissonnet (OMERO admin and image metadata specialist, Center for Advanced Imaging, Heinrich-Heine University of D&uuml;sseldorf)
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center, Heidelberg)

Material Description
Published here are the presentation slides that were used for input from the trainers during the different sessions of the programme. Additionally, a Fiji Macro is published that depends on the OMERO Extensions Plugin by Pouchin et al, 2022, F100Research, https://doi.org/10.12688/f1000research.110385.2&nbsp;
Programme Overview
Day 1 - April 29th, 2024&nbsp;09.00 a.m. to 10.00 a.m.: Session 1 - Welcome and Introduction
10.00 a.m. to 10.30 a.m.:&nbsp; Session 2 - Introduction to the FAIR principles &amp; data annotation
10:30 a.m. to 10:45 a.m.: Coffee break
10.45 a.m. to 12.00 a.m.: Session 3 - Data structure (datasets in OMERO) and organization with Tags&nbsp;
12.00 a.m. to 1.00 p.m.:&nbsp; Lunch Break
1.00 p.m. to 2.00 p.m.:&nbsp; Session 4 - REMBI, Key-Value pair annotations in bioimaging
2:00 p.m. to 2.30 p.m.:&nbsp; Session 5 - Ontologies for Key-Value Pairs in OMERO
2:30 p.m. to 2:45 p.m. Coffee break
2.45 p.m. to 3.45 p.m.:&nbsp; Wrap-up, discussion, outlook on day 2
Day 2 - April 30th, 2024
09.00 a.m. to 09.30 a.m.:&nbsp; Arrival and Start into day 2
09.30 a.m. to 11.30 a.m.:&nbsp; Session 6 - Hands-on : REMBI-based Key-Value Pair annotation in OMERO
11.30 a.m. to 12.30 a.m.:&nbsp; Lunch Break
12.30 a.m. to 1.15 p.m.: Session 7 - OMERO and OMERO.plugins
1.15 p.m. to 2.00 p.m.: Session 8 - Loading OMERO-hosted data into Fiji
2.00 p.m. to 2.15 p.m.:&nbsp;Coffee break&nbsp;
2.15 p.m. to 3.00 p.m.: Discussion, Outlook

[https://zenodo.org/records/11109616](https://zenodo.org/records/11109616)

[https://doi.org/10.5281/zenodo.11109616](https://doi.org/10.5281/zenodo.11109616)


---

## ilastik: interactive machine learning for (bio)image analysis

Anna Kreshuk, Dominik Kutra

Licensed CC-BY-4.0



Tags: Artificial Intelligence, Bioimage Analysis

Content type: Slide

[https://zenodo.org/doi/10.5281/zenodo.4330625](https://zenodo.org/doi/10.5281/zenodo.4330625)


---

## quantixed/TheDigitalCell: First complete code set

Stephen Royle

Published 2019-04-17

Licensed GPL-3.0



First complete code set for The Digital Cell book.

Tags: Bioimage Analysis

Content type: Code

[https://github.com/quantixed/TheDigitalCell](https://github.com/quantixed/TheDigitalCell)

[https://zenodo.org/records/2643411](https://zenodo.org/records/2643411)

[https://doi.org/10.5281/zenodo.2643411](https://doi.org/10.5281/zenodo.2643411)


---

