
<!DOCTYPE html>


<html lang="en" data-content_root="../" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Github.com (148) &#8212; NFDI4BioImage Training Materials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="../_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="../_static/styles/sphinx-book-theme.css?v=eba8b062" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.8ecb98da25f57f5357bf6f572d296f466b2cfe2517ffebfabe82451661e28f02.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="../_static/css/searchbox.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-design.min.css?v=95c83b7e" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="../_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="../_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="../_static/doctools.js?v=9a2dae69"></script>
    <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="../_static/copybutton.js?v=f281be69"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/lunr.js/0.6.0/lunr.min.js"></script>
    <script src="../_static/js/searchbox.js"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="../_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'domain/github.com';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Training.galaxyproject.org (5)" href="training.galaxyproject.org.html" />
    <link rel="prev" title="Git.mpi-cbg.de (7)" href="git.mpi-cbg.de.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
    <meta name="docbuild:last-update" content="2025-09-16"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="../readme.html">
  
  
  
  
  
    
    
      
    
    
    <img src="../_static/logo.png" class="logo__image only-light" alt="NFDI4BioImage Training Materials - Home"/>
    <script>document.write(`<img src="../_static/logo.png" class="logo__image only-dark" alt="NFDI4BioImage Training Materials - Home"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../readme.html">
                    NFDI4BioImage Training Materials
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">What's new</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../whats_new.html">Recently added (10)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Contributing</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../contributing/index.html">How to contribute</a></li>

<li class="toctree-l1"><a class="reference internal" href="../contributing/submit_app.html">Using the Training Materials Submission App</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contributing/format.html">YML format</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By tag</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../tags/ai-ready.html">Ai-ready (84)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/artificial_intelligence.html">Artificial intelligence (46)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bioimage_analysis.html">Bioimage analysis (204)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bioinformatics.html">Bioinformatics (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/data_stewardship.html">Data stewardship (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/fair-principles.html">Fair-principles (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/fiji.html">Fiji (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/galaxy.html">Galaxy (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/imagej.html">Imagej (20)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/licensing.html">Licensing (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/metadata.html">Metadata (15)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/napari.html">Napari (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/neubias.html">Neubias (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/nfdi4bioimage.html">Nfdi4bioimage (46)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/omero.html">Omero (38)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/open_science.html">Open science (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/open_source_software.html">Open source software (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/python.html">Python (71)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/reproducibility.html">Reproducibility (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/research_data_management.html">Research data management (140)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/sharing.html">Sharing (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/workflow.html">Workflow (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/workflow_engine.html">Workflow engine (13)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By content type</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../content_types/blog%20post.html">Blog post (28)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/book.html">Book (21)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/code.html">Code (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/collection.html">Collection (86)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/data.html">Data (92)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/document.html">Document (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/documentation.html">Documentation (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/event.html">Event (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/github%20repository.html">Github repository (67)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/notebook.html">Notebook (57)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/online%20tutorial.html">Online tutorial (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/poster.html">Poster (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/preprint.html">Preprint (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/publication.html">Publication (71)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/slides.html">Slides (84)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/tutorial.html">Tutorial (48)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/video.html">Video (38)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/website.html">Website (11)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/workshop.html">Workshop (14)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By license</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../licenses/all_rights_reserved.html">All rights reserved (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/apache-2.0.html">Apache-2.0 (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-2-clause.html">Bsd-2-clause (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-3-clause.html">Bsd-3-clause (35)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-3.0.html">Cc-by-3.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-4.0.html">Cc-by-4.0 (407)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-nc-sa-4.0.html">Cc-by-nc-sa-4.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-sa-4.0.html">Cc-by-sa-4.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc0-1.0.html">Cc0-1.0 (25)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-2.0.html">Gpl-2.0 (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-3.0.html">Gpl-3.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/mit.html">Mit (31)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/unknown.html">Unknown (109)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By domain</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="bbbc.broadinstitute.org.html">Bbbc.broadinstitute.org (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="biapol.github.io.html">Biapol.github.io (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="docs.google.com.html">Docs.google.com (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="doi.org.html">Doi.org (345)</a></li>
<li class="toctree-l1"><a class="reference internal" href="f1000research.com.html">F1000research.com (11)</a></li>
<li class="toctree-l1"><a class="reference internal" href="focalplane.biologists.com.html">Focalplane.biologists.com (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="git.mpi-cbg.de.html">Git.mpi-cbg.de (7)</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Github.com (148)</a></li>
<li class="toctree-l1"><a class="reference internal" href="training.galaxyproject.org.html">Training.galaxyproject.org (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="www.biorxiv.org.html">Www.biorxiv.org (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="www.ebi.ac.uk.html">Www.ebi.ac.uk (18)</a></li>
<li class="toctree-l1"><a class="reference internal" href="www.nature.com.html">Www.nature.com (18)</a></li>
<li class="toctree-l1"><a class="reference internal" href="www.youtube.com.html">Www.youtube.com (28)</a></li>
<li class="toctree-l1"><a class="reference internal" href="zenodo.org.html">Zenodo.org (325)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Appendix</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../statistics/readme.html">Statistics</a></li>
<li class="toctree-l1"><a class="reference internal" href="../export/readme.html">Open data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../gdpr_compliance.html">GDPR Compliance Documentation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../imprint.html">Imprint</a></li>

</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training/issues/new?title=Issue%20on%20page%20%2Fdomain/github.com.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/domain/github.com.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Github.com (148)</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-survey-community-experiences-and-needs-for-the-future">2020 BioImage Analysis Survey: Community experiences and needs for the future</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#mic-workshop-on-bioimage-processing-with-python">2022 MIC Workshop on Bioimage processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#courseimageanalysis-nanocell">2023_CourseImageAnalysis_NanoCell</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#adding-a-workflow-to-biaflows">Adding a Workflow to BIAFLOWS</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#advanced-scripting-with-fiji">Advanced scripting with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#analysis-of-high-throughput-microscopy-image-data">Analysis of High-Throughput Microscopy Image Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bccd-dataset">BCCD Dataset</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bhg2023-omero-arc">BHG2023-OMERO-ARC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bids-lecture-2024">BIDS-lecture-2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#basics-of-image-processing-and-analysis">Basics of Image Processing and Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis">Bio Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-icob-2023">Bio-image Analysis ICOB 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-kioto-and-taipei-23-24">Bio-image Analysis Workshop Kioto and Taipei 23/24</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-taipei">Bio-image Analysis Workshop Taipei</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science">Bio-image Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">Bio-image analysis, biostatistics, programming and machine learning for computational biology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimaging-ai-textbook">Bioimaging AI Textbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#biomero">Biomero</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#breast-cancer-semantic-segmentation-bcss-dataset">Breast Cancer Semantic Segmentation (BCSS) dataset</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bridging-imaging-users-to-imaging-analysis-a-community-survey">Bridging Imaging Users to Imaging Analysis - A community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-bioimage-analysis-workflow-using-deep-learning">Building a Bioimage Analysis Workflow using Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022">CARE/Stardist tutorials for EMBO Practical Course — Computational optical biology 2022</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#csbdeep-and-stardist-i2k-2020">CSBDeep and StarDist @ I2K 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-introduction">CellProfiler Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-practical-neubias-lisbon-2017">CellProfiler Practical NeuBIAS Lisbon 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-tutorials">CellProfiler tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#celltrackcolab">CellTrackColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis">Collection of teaching material for deep learning for (biomedical) image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#course-on-deep-learning-for-imaging-using-pytorch">Course on Deep Learning for imaging using PyTorch</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-napari-napari-as-a-tool-for-deep-learning-project-management">DEEP NAPARI : Napari as a tool for deep learning project management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4miceverywhere">DL4MicEverywhere</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4proteins-notebooks">DL4Proteins-notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl-mbl-2021-exercises">DL@MBL 2021 Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dask-course">Dask Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-based-segmentation-for-biologists">Deep Learning Based Segmentation For Biologists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-for-image-analysis-exercises">Deep Learning for image analysis - Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-vision-and-graphics">Deep Vision and Graphics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deepprofiler-handbook">DeepProfiler Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2019-exercises-and-materials">EMBL Deep Learning course 2019 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2021-22-exercises-and-materials">EMBL Deep Learning course 2021/22 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2023-exercises-and-materials">EMBL Deep Learning course 2023 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embedseg-repository">EmbedSeg Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus">EuBIAS course 2013: Intensity Dynamics at the Periphery of Nucleus</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#example-pipeline-tutorial">Example Pipeline Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#friendsdontletfriends">FriendsDontLetFriends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-in-digital-image-processing">Fundamentals in digital image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends">Fundamentals of image analysis in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-training-material">Galaxy Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library">I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-virtual-bio-image-analysis-code-generation">I2K2024(virtual) - Bio-Image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#iafig-rms-python-for-bioimage-analysis-course">IAFIG-RMS Python for Bioimage Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#idc-tutorials">IDC-Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#itkelastix-examples">ITKElastix Examples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-with-python">Image Processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends">Image analysis and visualization in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-course-material">Image analysis course material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-for-beginners">Image processing for beginners</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#id1">Image processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-based-profiling-handbook">Image-based Profiling Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej-macro-introduction">ImageJ Macro Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-deep-learning-for-microscopy">Introduction to Deep Learning for Microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis-with-fiji">Introduction to Image Analysis with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kreshuk-lab-s-embl-eipp-predoc-course-teaching-material">Kreshuk Lab’s EMBL EIPP predoc course teaching material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#leo">LEO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-materials-of-the-deeplife-course">Lecture-materials of the DeepLife course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#midog-2021">MIDOG 2021</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-workflow-for-imaging-flow-cytometry-ifc">Machine Learning Workflow for Imaging Flow Cytometry (IFC)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#materials-for-embl-coding-club-mini-tutorials">Materials for EMBL Coding Club Mini-Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#metabolic-networks-modelling-with-cobrapy">Metabolic networks modelling with COBRApy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#models-and-applications-for-bioimage-io">Models and Applications for BioImage.IO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter">NEUBIAS Academy @HOME: Interactive Bioimage Analysis with Python and Jupyter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-analyst-school-2018">NEUBIAS Analyst School 2018</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-course-2017">NEUBIAS Bioimage Analyst Course 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2019">NEUBIAS Bioimage Analyst School 2019</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2020">NEUBIAS Bioimage Analyst School 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nlp-course">NLP Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubiaspasteur2023-advancedcellpose">NeubiasPasteur2023_AdvancedCellPose</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-demo-pipelines-for-image-processing">Nextflow Demo Pipelines for Image Processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks">OMERO - HCS analysis pipeline using Jupyter Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#onbi-image-analysis-course">ONBI Image Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-deployment-examples">Omero Deployment examples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#online-r-learning">Online_R_learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#parallel-fiji-visualizer">Parallel_Fiji_Visualizer</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-omero-batch-plugin">Plugin “omero-batch-plugin”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-omero-cli-transfer">Plugin “omero-cli-transfer”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-simple-omero-client">Plugin “simple-omero-client”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis">PoL Bio-Image Analysis Training School on GPU-Accelerated Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#predicting-axillary-lymph-node-metastasis-in-early-breast-cancer-using-deep-learning-on-primary-tumor-biopsy-slides">Predicting Axillary Lymph Node Metastasis in Early Breast Cancer Using Deep Learning on Primary Tumor Biopsy Slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-llms-course">Prompt-Engineering-LLMs-Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-bioimage-analysis-tutorial">Python BioImage Analysis Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-workshop-image-processing">Python Workshop - Image Processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-for-microscopists">Python for Microscopists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-for-python-programmers">QuPath for Python programmers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdm4mic-presentations">RDM4Mic Presentations</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdm-system-connector">RDM_system_connector</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scientific-visualization-python-matplotlib">Scientific Visualization: Python + Matplotlib</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scipy-cookbook">Scipy Cookbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scripts-filopodyanr-a-case-study-for-the-neubias-ts7-in-szeged">Scripts_FilopodyanR - a case study for the NEUBIAS TS7 in Szeged</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-a-remote-desktop-to-use-napari-in-a-browser">Setting up a remote desktop to use Napari in a browser</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#simpleitk-notebooks">SimpleITK-Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#source-control-using-git-and-github">Source Control Using Git and GitHub</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#statistical-rethinking">Statistical Rethinking</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-bioimage-analysis-with-python">Teaching Bioimage Analysis with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-imagej-fiji">Teaching ImageJ FIJI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-deep-learning-models-for-vision-compact-course">Training Deep Learning Models for Vision - Compact Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#trendsinmicroscopy2025">TrendsInMicroscopy2025</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ultrack-i2k-2024-workshop-materials">Ultrack I2K 2024 Workshop Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#v4sdb-winter-school-2025">V4SDB_Winter_School_2025</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#virtual-i2k-2024-multiview-stitcher">Virtual-I2K-2024-multiview-stitcher</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-june2024-madrid">Workshop-June2024-Madrid</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zen-python-workshop">ZEN &amp; Python workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zidas-2020-introduction-to-deep-learning">ZIDAS 2020 Introduction to Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zerocostdl4mic-exploiting-google-colab-to-develop-a-free-and-open-source-toolbox-for-deep-learning-in-microscopy">ZeroCostDL4Mic: exploiting Google Colab to develop a free and open-source toolbox for Deep-Learning in microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioformats2raw-converter">bioformats2raw Converter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimageio-chatbot">bioimageio-chatbot</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-visualization-tutorial">data_visualization_tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenbiene">datenbiene</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dmtxsamplecreator">dmtxSampleCreator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2020-s3-zarr-workshop">i2k-2020-s3-zarr-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-basics">image-processing-basics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagequantification101">imagequantification101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-generative-ai">introduction-to-generative-ai</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis">introduction-to-image-analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#napari-scipy2025-workshop">napari-scipy2025-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-workshop">nextflow-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow101">nextflow101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#numpy-pandas-course">numpy pandas course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome-ngff-validator">ome-ngff-validator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome2024-ngff-challenge">ome2024-ngff-challenge</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-arc">omero-arc</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-ontop-mappings">omero-ontop-mappings</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-vitessce">omero-vitessce</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#patho-prompt-injection">patho_prompt_injection</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quantixed-thedigitalcell-first-complete-code-set">quantixed/TheDigitalCell: First complete code set</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-workshop">qupath-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#raw2ometiff-converter">raw2ometiff Converter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rse-skills-workshop">rse-skills-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scanpy-tutorials">scanpy-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scikit-learn-mooc">scikit-learn MOOC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#shapeanalysis101">shapeanalysis101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#skimage-tutorials">skimage-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-resources">training-resources</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="github-com-148">
<h1><a class="reference external" href="http://Github.com">Github.com</a> (148)<a class="headerlink" href="#github-com-148" title="Link to this heading">#</a></h1>
<section id="bioimage-analysis-survey-community-experiences-and-needs-for-the-future">
<h2>2020 BioImage Analysis Survey: Community experiences and needs for the future<a class="headerlink" href="#bioimage-analysis-survey-community-experiences-and-needs-for-the-future" title="Link to this heading">#</a></h2>
<p>Nasim Jamali, Ellen T. A. Dobson, Kevin W. Eliceiri, Anne E. Carpenter, Beth A. Cimini</p>
<p>Published 2021</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1017/s2633903x21000039">https://doi.org/10.1017/s2633903x21000039</a></p>
<p><a class="github reference external" href="https://github.com/ciminilab/2021_Jamali_BiologicalImaging">ciminilab/2021_Jamali_BiologicalImaging</a></p>
</section>
<hr class="docutils" />
<section id="mic-workshop-on-bioimage-processing-with-python">
<h2>2022 MIC Workshop on Bioimage processing with Python<a class="headerlink" href="#mic-workshop-on-bioimage-processing-with-python" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/MICPy_Workshop_2022">guiwitz/MICPy_Workshop_2022</a></p>
</section>
<hr class="docutils" />
<section id="courseimageanalysis-nanocell">
<h2>2023_CourseImageAnalysis_NanoCell<a class="headerlink" href="#courseimageanalysis-nanocell" title="Link to this heading">#</a></h2>
<p>Fernanda Fossa</p>
<p>Published 2023-08-24T10:15:11+00:00</p>
<p>Licensed MIT</p>
<p>Course about image analysis with materials in English and videos in Portuguese.</p>
<p>Tags: Cell Profiler, Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/fefossa/2023_CourseImageAnalysis_NanoCell">fefossa/2023_CourseImageAnalysis_NanoCell</a></p>
</section>
<hr class="docutils" />
<section id="adding-a-workflow-to-biaflows">
<h2>Adding a Workflow to BIAFLOWS<a class="headerlink" href="#adding-a-workflow-to-biaflows" title="Link to this heading">#</a></h2>
<p>Sébastien Tosi, Volker Baecker, Benjamin Pavie</p>
<p>Licensed BSD-2-CLAUSE</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/RoccoDAnt/Defragmentation_TrainingSchool_EOSC-Life_2022/blob/main/Slides/Adding_a_workflow_to_BIAFLOWS.pdf">RoccoDAnt/Defragmentation_TrainingSchool_EOSC-Life_2022</a></p>
</section>
<hr class="docutils" />
<section id="advanced-scripting-with-fiji">
<h2>Advanced scripting with Fiji<a class="headerlink" href="#advanced-scripting-with-fiji" title="Link to this heading">#</a></h2>
<p>Joran Deschamps, Damian Dalle Nogare</p>
<p>Published 2023-06-14T08:25:35+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Fiji</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/nobias-fht/advanced-scripting">nobias-fht/advanced-scripting</a></p>
</section>
<hr class="docutils" />
<section id="analysis-of-high-throughput-microscopy-image-data">
<h2>Analysis of High-Throughput Microscopy Image Data<a class="headerlink" href="#analysis-of-high-throughput-microscopy-image-data" title="Link to this heading">#</a></h2>
<p>Kota Miura, Christoph Schiklenk, Clemens Lakner, Christian Tischer, Aliaksandr Halavatyi</p>
<p>Published 2014</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/miura/HTManalysisCourse/blob/master/CentreCourseProtocol.md">miura/HTManalysisCourse</a></p>
</section>
<hr class="docutils" />
<section id="bccd-dataset">
<h2>BCCD Dataset<a class="headerlink" href="#bccd-dataset" title="Link to this heading">#</a></h2>
<p>Shenggan Gan, Nicolas Chen</p>
<p>Published 2017-12-07</p>
<p>Licensed MIT</p>
<p>BCCD Dataset is a small-scale dataset for blood cells detection.</p>
<p>Tags: Ai-Ready</p>
<p>Content type: Data</p>
<p><a class="github reference external" href="https://github.com/Shenggan/BCCD_Dataset">Shenggan/BCCD_Dataset</a></p>
</section>
<hr class="docutils" />
<section id="bhg2023-omero-arc">
<h2>BHG2023-OMERO-ARC<a class="headerlink" href="#bhg2023-omero-arc" title="Link to this heading">#</a></h2>
<p>Andrea Schrader, Michele Bortolomeazzi, Niraj Kandpal, Torsten Stöter, Kevin Schneider, Peter Zentis, Josh Moore, Jeam-Marie Burel, Tom Boissonnet</p>
<p>Licensed CC-BY-4.0</p>
<p>Repository for documentation during the 2nd de.NBI BioHackathon Germany - 11.-15.12.2023 - OMERO-ARC project (in short: BHG2023-OMERO-ARC)</p>
<p>Tags: Bioinformatics, OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/BHG2023-OMERO-ARC">NFDI4BIOIMAGE/BHG2023-OMERO-ARC</a></p>
</section>
<hr class="docutils" />
<section id="bids-lecture-2024">
<h2>BIDS-lecture-2024<a class="headerlink" href="#bids-lecture-2024" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python. The material developed here between April and July 2024.</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence, Python</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ScaDS/BIDS-lecture-2024/">ScaDS/BIDS-lecture-2024</a></p>
</section>
<hr class="docutils" />
<section id="basics-of-image-processing-and-analysis">
<h2>Basics of Image Processing and Analysis<a class="headerlink" href="#basics-of-image-processing-and-analysis" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/miura/ij_textbook1/raw/76b51338e1f006c580b6f0f5cfc48fe02fba38d7/CMCIBasicCourse201102Bib.pdf">miura/ij_textbook1</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis">
<h2>Bio Image Analysis<a class="headerlink" href="#bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Christian Tischer</p>
<p>Licensed UNKNOWN</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/tischi/presentation-image-analysis">tischi/presentation-image-analysis</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-icob-2023">
<h2>Bio-image Analysis ICOB 2023<a class="headerlink" href="#bio-image-analysis-icob-2023" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/WeiChenChu/Bioimage_Analysis_ICOB_2023">WeiChenChu/Bioimage_Analysis_ICOB_2023</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-workshop-kioto-and-taipei-23-24">
<h2>Bio-image Analysis Workshop Kioto and Taipei 23/24<a class="headerlink" href="#bio-image-analysis-workshop-kioto-and-taipei-23-24" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/Koushouu/Bioimage-Analysis-Workshop-23-24">Koushouu/Bioimage-Analysis-Workshop-23-24</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-workshop-taipei">
<h2>Bio-image Analysis Workshop Taipei<a class="headerlink" href="#bio-image-analysis-workshop-taipei" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/Koushouu/Bioimage-Analysis-Workshop-Taipei/">Koushouu/Bioimage-Analysis-Workshop-Taipei</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-science">
<h2>Bio-image Data Science<a class="headerlink" href="#bio-image-data-science" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository contains training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python.</p>
<p>Tags: Research Data Management, Artificial Intelligence, Bioimage Analysis, Python</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/ScaDS/BIDS-lecture-2024">ScaDS/BIDS-lecture-2024</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">
<h2>Bio-image analysis, biostatistics, programming and machine learning for computational biology<a class="headerlink" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology" title="Link to this heading">#</a></h2>
<p>Anna Poetsch, Biotec Dresden, Marcelo Leomil Zoccoler, Johannes Richard Müller, Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Bioimage Analysis, Napari</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/BiAPoL/Bio-image_Analysis_with_Python">BiAPoL/Bio-image_Analysis_with_Python</a></p>
</section>
<hr class="docutils" />
<section id="bioimaging-ai-textbook">
<h2>Bioimaging AI Textbook<a class="headerlink" href="#bioimaging-ai-textbook" title="Link to this heading">#</a></h2>
<p>Rachel Lee, Owen Puls, Wei Ouyang, Beth Cimini</p>
<p>Published 2025-02-20</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This is a repository containing a textbook written for BioImaging scientists and other microscopists.</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Book, Github Repository</p>
<p><a class="github reference external" href="https://github.com/aicjanelia/BioImagingAI">aicjanelia/BioImagingAI</a></p>
</section>
<hr class="docutils" />
<section id="biomero">
<h2>Biomero<a class="headerlink" href="#biomero" title="Link to this heading">#</a></h2>
<p>Torec Luik, Johannes Soltwedel</p>
<p>Published 2024-07-24</p>
<p>Licensed APACHE-2.0</p>
<p>The BIOMERO framework, for BioImage analysis in OMERO, allows you to run (FAIR) bioimage analysis workflows directly from OMERO on a high-performance compute (HPC) cluster, remotely through SSH.</p>
<p>Tags: OMERO, Github</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NL-BioImaging/biomero">NL-BioImaging/biomero</a></p>
</section>
<hr class="docutils" />
<section id="breast-cancer-semantic-segmentation-bcss-dataset">
<h2>Breast Cancer Semantic Segmentation (BCSS) dataset<a class="headerlink" href="#breast-cancer-semantic-segmentation-bcss-dataset" title="Link to this heading">#</a></h2>
<p>Mohamed Amgad, Habiba Elfandy, Hagar Hussein, Lamees A Atteya, Mai A T Elsebaie, Lamia S Abo Elnasr, Rokia A Sakr, Hazem S E Salem, Ahmed F Ismail, Anas M Saad, Joumana Ahmed, Maha A T Elsebaie, Mustafijur Rahman, Inas A Ruhban, Nada M Elgazar, Yahya Alagha, Mohamed H Osman, Ahmed M Alhusseiny, Mariam M Khalaf, Abo-Alela F Younes, Ali Abdulkarim, Duaa M Younes, Ahmed M Gadallah, Ahmad M Elkashash, Salma Y Fala, Basma M Zaki, Jonathan Beezley, Deepak R Chittajallu, David Manthey, David A Gutman, Lee A D Cooper</p>
<p>Published 2019-11-09</p>
<p>Licensed CC0-1.0</p>
<p>This repo contains the necessary information and download instructions to download the dataset associated with the paper: Amgad M, Elfandy H, …, Gutman DA, Cooper LAD. Structured crowdsourcing enables convolutional segmentation of histology images. Bioinformatics. 2019. doi: 10.1093/bioinformatics/btz083. This data can be visualized in a public instance of the Digital Slide Archive at this link. If you click the “eye” image icon in the Annotations panel on the right side of the screen, you will see the results of a collaborative annotation.</p>
<p>Tags: Ai-Ready</p>
<p>Content type: Data</p>
<p><a class="github reference external" href="https://github.com/PathologyDataScience/BCSS">PathologyDataScience/BCSS</a></p>
</section>
<hr class="docutils" />
<section id="bridging-imaging-users-to-imaging-analysis-a-community-survey">
<h2>Bridging Imaging Users to Imaging Analysis - A community survey<a class="headerlink" href="#bridging-imaging-users-to-imaging-analysis-a-community-survey" title="Link to this heading">#</a></h2>
<p>Suganya Sivagurunathan, Stefania Marcotti, Carl J Nelson, Martin L Jones, David J Barry, Thomas J A Slater, Kevin W Eliceiri, Beth A Cimini</p>
<p>Published 2023</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication, Preprint</p>
<p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2023.06.05.543701v1">https://www.biorxiv.org/content/10.1101/2023.06.05.543701v1</a></p>
<p><a class="github reference external" href="https://github.com/COBA-NIH/2023_ImageAnalysisSurvey">COBA-NIH/2023_ImageAnalysisSurvey</a></p>
</section>
<hr class="docutils" />
<section id="building-a-bioimage-analysis-workflow-using-deep-learning">
<h2>Building a Bioimage Analysis Workflow using Deep Learning<a class="headerlink" href="#building-a-bioimage-analysis-workflow-using-deep-learning" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/esgomezm/NEUBIAS_chapter_DL_2020">esgomezm/NEUBIAS_chapter_DL_2020</a></p>
</section>
<hr class="docutils" />
<section id="care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022">
<h2>CARE/Stardist tutorials for EMBO Practical Course — Computational optical biology 2022<a class="headerlink" href="#care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022" title="Link to this heading">#</a></h2>
<p>Martin Weigert</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/embo_2022">maweigert/embo_2022</a></p>
</section>
<hr class="docutils" />
<section id="csbdeep-and-stardist-i2k-2020">
<h2>CSBDeep and StarDist &#64; I2K 2020<a class="headerlink" href="#csbdeep-and-stardist-i2k-2020" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Uwe Schmidt</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/stardist-i2k">maweigert/stardist-i2k</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-introduction">
<h2>CellProfiler Introduction<a class="headerlink" href="#cellprofiler-introduction" title="Link to this heading">#</a></h2>
<p>Anna Klemm</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Cellprofiler, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/ahklemm/CellProfiler_Introduction">ahklemm/CellProfiler_Introduction</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-practical-neubias-lisbon-2017">
<h2>CellProfiler Practical NeuBIAS Lisbon 2017<a class="headerlink" href="#cellprofiler-practical-neubias-lisbon-2017" title="Link to this heading">#</a></h2>
<p>Carolina Wählby, Maxime Bombrun, Christian Tischer</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Cellprofiler, Bioimage Analysis</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/tischi/cellprofiler-practical-NeuBIAS-Lisbon-2017">tischi/cellprofiler-practical-NeuBIAS-Lisbon-2017</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-tutorials">
<h2>CellProfiler tutorials<a class="headerlink" href="#cellprofiler-tutorials" title="Link to this heading">#</a></h2>
<p>Licensed BSD-3-CLAUSE</p>
<p>CellProfiler tutorials and guided exercises about translocation, segmentation, pixel-based classification and quality control</p>
<p>Tags: Cellprofiler, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/CellProfiler/tutorials">CellProfiler/tutorials</a></p>
</section>
<hr class="docutils" />
<section id="celltrackcolab">
<h2>CellTrackColab<a class="headerlink" href="#celltrackcolab" title="Link to this heading">#</a></h2>
<p>Guillaume Jacquemet</p>
<p>Licensed MIT</p>
<p>Content type: Notebook, Collection</p>
<p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2023.10.20.563252v2">https://www.biorxiv.org/content/10.1101/2023.10.20.563252v2</a></p>
<p><a class="github reference external" href="https://github.com/guijacquemet/CellTracksColab">guijacquemet/CellTracksColab</a></p>
</section>
<hr class="docutils" />
<section id="collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis">
<h2>Collection of teaching material for deep learning for (biomedical) image analysis<a class="headerlink" href="#collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis" title="Link to this heading">#</a></h2>
<p>Constantin Pape</p>
<p>Licensed MIT</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/constantinpape/dl-teaching-resources">constantinpape/dl-teaching-resources</a></p>
</section>
<hr class="docutils" />
<section id="course-on-deep-learning-for-imaging-using-pytorch">
<h2>Course on Deep Learning for imaging using PyTorch<a class="headerlink" href="#course-on-deep-learning-for-imaging-using-pytorch" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/DLImaging">guiwitz/DLImaging</a></p>
</section>
<hr class="docutils" />
<section id="deep-napari-napari-as-a-tool-for-deep-learning-project-management">
<h2>DEEP NAPARI : Napari as a tool for deep learning project management<a class="headerlink" href="#deep-napari-napari-as-a-tool-for-deep-learning-project-management" title="Link to this heading">#</a></h2>
<p>Herearii Metuarea, David Rousseau, Pejman Rasti, Valentin Gilet</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/hereariim/DEEP-NAPARI">hereariim/DEEP-NAPARI</a></p>
</section>
<hr class="docutils" />
<section id="dl4miceverywhere">
<h2>DL4MicEverywhere<a class="headerlink" href="#dl4miceverywhere" title="Link to this heading">#</a></h2>
<p>Iván Hidalgo, et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/HenriquesLab/DL4MicEverywhere">HenriquesLab/DL4MicEverywhere</a></p>
</section>
<hr class="docutils" />
<section id="dl4proteins-notebooks">
<h2>DL4Proteins-notebooks<a class="headerlink" href="#dl4proteins-notebooks" title="Link to this heading">#</a></h2>
<p>Michael Chungyoun, Courtney Thomas, Britnie Carpentier, GabeAu79, puv-sreev, Jeffrey Gray</p>
<p>Published 2024-09-04T12:24:24+00:00</p>
<p>Colab Notebooks covering deep learning tools for biomolecular structure prediction and design</p>
<p>Tags: Bioinformatics</p>
<p>Content type: Github Repository, Collection</p>
<p><a class="github reference external" href="https://github.com/Graylab/DL4Proteins-notebooks">Graylab/DL4Proteins-notebooks</a></p>
</section>
<hr class="docutils" />
<section id="dl-mbl-2021-exercises">
<h2>DL&#64;MBL 2021 Exercises<a class="headerlink" href="#dl-mbl-2021-exercises" title="Link to this heading">#</a></h2>
<p>Jan Funke, Constantin Pape, Morgan Schwartz, Xiaoyan</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/JLrumberger/DL-MBL-2021">JLrumberger/DL-MBL-2021</a></p>
</section>
<hr class="docutils" />
<section id="dask-course">
<h2>Dask Course<a class="headerlink" href="#dask-course" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Bioimage Analysis, Big Data</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/DaskCourse">guiwitz/DaskCourse</a></p>
</section>
<hr class="docutils" />
<section id="deep-learning-based-segmentation-for-biologists">
<h2>Deep Learning Based Segmentation For Biologists<a class="headerlink" href="#deep-learning-based-segmentation-for-biologists" title="Link to this heading">#</a></h2>
<p>Licensed AGPL-3.0</p>
<p>Tags: Python, R, Artificial Intelligence</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/tpecot/DeepLearningBasedSegmentationForBiologists/">tpecot/DeepLearningBasedSegmentationForBiologists</a></p>
</section>
<hr class="docutils" />
<section id="deep-learning-for-image-analysis-exercises">
<h2>Deep Learning for image analysis - Exercises<a class="headerlink" href="#deep-learning-for-image-analysis-exercises" title="Link to this heading">#</a></h2>
<p>Martin Weigert</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Fiji, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/zidas_2020_DL_intro_Part_2">maweigert/zidas_2020_DL_intro_Part_2</a></p>
</section>
<hr class="docutils" />
<section id="deep-vision-and-graphics">
<h2>Deep Vision and Graphics<a class="headerlink" href="#deep-vision-and-graphics" title="Link to this heading">#</a></h2>
<p>Victor Yurchenko, Fedor Ratnikov, Viktoriia Checkalina</p>
<p>Licensed MIT</p>
<p>Tags: Python, Artificial Intelligence</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/yandexdataschool/deep_vision_and_graphics">yandexdataschool/deep_vision_and_graphics</a></p>
</section>
<hr class="docutils" />
<section id="deepprofiler-handbook">
<h2>DeepProfiler Handbook<a class="headerlink" href="#deepprofiler-handbook" title="Link to this heading">#</a></h2>
<p>Michael Bornholdt, Juan Caicedo, Yu Han, Nikita Moshkov, Rebecca Senft</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/cytomining/DeepProfiler-handbook">cytomining/DeepProfiler-handbook</a></p>
<p><a class="reference external" href="https://cytomining.github.io/DeepProfiler-handbook/docs/00-welcome.html">https://cytomining.github.io/DeepProfiler-handbook/docs/00-welcome.html</a></p>
</section>
<hr class="docutils" />
<section id="embl-deep-learning-course-2019-exercises-and-materials">
<h2>EMBL Deep Learning course 2019 exercises and materials<a class="headerlink" href="#embl-deep-learning-course-2019-exercises-and-materials" title="Link to this heading">#</a></h2>
<p>Valentyna Zinchenko, Pejman Rasti, Martin Weigert, Szymon Stoma</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/kreshuklab/teaching-dl-course-2019">kreshuklab/teaching-dl-course-2019</a></p>
</section>
<hr class="docutils" />
<section id="embl-deep-learning-course-2021-22-exercises-and-materials">
<h2>EMBL Deep Learning course 2021/22 exercises and materials<a class="headerlink" href="#embl-deep-learning-course-2021-22-exercises-and-materials" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Constantin Pape</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/kreshuklab/teaching-dl-course-2022">kreshuklab/teaching-dl-course-2022</a></p>
</section>
<hr class="docutils" />
<section id="embl-deep-learning-course-2023-exercises-and-materials">
<h2>EMBL Deep Learning course 2023 exercises and materials<a class="headerlink" href="#embl-deep-learning-course-2023-exercises-and-materials" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Uwe Schmidt, Benjamin Gallusser, Albert Dominguez Mantes, Buglakova Alyona</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/kreshuklab/teaching-dl-course-2023">kreshuklab/teaching-dl-course-2023</a></p>
</section>
<hr class="docutils" />
<section id="embedseg-repository">
<h2>EmbedSeg Repository<a class="headerlink" href="#embedseg-repository" title="Link to this heading">#</a></h2>
<p>Manan Lalit, Joran Deschamps, Florian Jug, Ajinkya Kulkarni</p>
<p>Licensed CC-BY-NC-4.0</p>
<p>Code Implementation for EmbedSeg, an Instance Segmentation Method for Microscopy Images</p>
<p>Tags: Bioimage Analysis, Instance Segmentation</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/juglab/EmbedSeg">juglab/EmbedSeg</a></p>
</section>
<hr class="docutils" />
<section id="eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus">
<h2>EuBIAS course 2013: Intensity Dynamics at the Periphery of Nucleus<a class="headerlink" href="#eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Published 2013</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Tutorial, Book</p>
<p><a class="github reference external" href="https://github.com/miura/BIAS_Nucleus_Segmentation/blob/master/module9.pdf">miura/BIAS_Nucleus_Segmentation</a></p>
</section>
<hr class="docutils" />
<section id="example-pipeline-tutorial">
<h2>Example Pipeline Tutorial<a class="headerlink" href="#example-pipeline-tutorial" title="Link to this heading">#</a></h2>
<p>Tim Monko</p>
<p>Published 2024-10-28</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Napari-ndev is a collection of widgets intended to serve any person seeking to process microscopy images from start to finish. The goal of this example pipeline is to get the user familiar with working with napari-ndev for batch processing and reproducibility (view Image Utilities and Workflow Widget).</p>
<p>Tags: Napari, Bioimage Analysis</p>
<p>Content type: Documentation, Github Repository, Tutorial</p>
<p><a class="reference external" href="https://timmonko.github.io/napari-ndev/tutorial/01_example_pipeline/">https://timmonko.github.io/napari-ndev/tutorial/01_example_pipeline/</a></p>
<p><a class="github reference external" href="https://github.com/timmonko/napari-ndev">timmonko/napari-ndev</a></p>
</section>
<hr class="docutils" />
<section id="friendsdontletfriends">
<h2>FriendsDontLetFriends<a class="headerlink" href="#friendsdontletfriends" title="Link to this heading">#</a></h2>
<p>C. Li</p>
<p>Published 2024-03-10T15:34:07+00:00</p>
<p>Licensed MIT</p>
<p>Friends don’t let friends make certain types of data visualization - What are they and why are they bad.</p>
<p>Tags: Visualization</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/cxli233/FriendsDontLetFriends">cxli233/FriendsDontLetFriends</a></p>
</section>
<hr class="docutils" />
<section id="fundamentals-in-digital-image-processing">
<h2>Fundamentals in digital image processing<a class="headerlink" href="#fundamentals-in-digital-image-processing" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/Fundamentals_image_processing">guiwitz/Fundamentals_image_processing</a></p>
</section>
<hr class="docutils" />
<section id="fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends">
<h2>Fundamentals of image analysis in Python with scikit-image, napari, and friends<a class="headerlink" href="#fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/jni/halfway-to-i2k-skimage-napari">jni/halfway-to-i2k-skimage-napari</a></p>
</section>
<hr class="docutils" />
<section id="galaxy-training-material">
<h2>Galaxy Training Material<a class="headerlink" href="#galaxy-training-material" title="Link to this heading">#</a></h2>
<p>Licensed MIT</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="github reference external" href="https://github.com/galaxyproject/training-material">galaxyproject/training-material</a></p>
</section>
<hr class="docutils" />
<section id="i2k-2024-clesperanto-gpu-accelerated-image-processing-library">
<h2>I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library<a class="headerlink" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library" title="Link to this heading">#</a></h2>
<p>Stephane Rigaud, Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Course and material for the clEsperanto workshop presented at I2K 2024 &#64; Human Technopol (Milan, Italy). The workshop is an hands-on demo of the clesperanto project, focussing on how to use the library for users who want use GPU-acceleration for their Image Processing pipeline.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Workshop, Tutorial, Notebook</p>
<p><a class="github reference external" href="https://github.com/StRigaud/clesperanto_workshop_I2K24?tab=readme-ov-file">StRigaud/clesperanto_workshop_I2K24</a></p>
</section>
<hr class="docutils" />
<section id="i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">
<h2>I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark<a class="headerlink" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark" title="Link to this heading">#</a></h2>
<p>Stephan Saalfeld, Tobias Pietzsch</p>
<p>Published None</p>
<p>Licensed APACHE-2.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Notebook, Github Repository</p>
<p><a class="reference external" href="https://saalfeldlab.github.io/i2k2024-lazy-workshop/">https://saalfeldlab.github.io/i2k2024-lazy-workshop/</a></p>
<p><a class="github reference external" href="https://github.com/saalfeldlab/i2k2024-lazy-workshop">saalfeldlab/i2k2024-lazy-workshop</a></p>
</section>
<hr class="docutils" />
<section id="i2k2024-virtual-bio-image-analysis-code-generation">
<h2>I2K2024(virtual) - Bio-Image Analysis Code Generation<a class="headerlink" href="#i2k2024-virtual-bio-image-analysis-code-generation" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This repository contains training materials for the Tutorial “Bio-Image Analysis Code Generation” at the From Images To Knowledge (I2K) Conference (virtual) October 28th-30th 2024.</p>
<p>Tags: Bioimage Analysis, Notebooks, Biabob</p>
<p>Content type: Github Repository, Tutorial, Notebook</p>
<p><a class="github reference external" href="https://github.com/haesleinhuepf/i2k2024-ai-code-generation">haesleinhuepf/i2k2024-ai-code-generation</a></p>
</section>
<hr class="docutils" />
<section id="iafig-rms-python-for-bioimage-analysis-course">
<h2>IAFIG-RMS Python for Bioimage Analysis Course<a class="headerlink" href="#iafig-rms-python-for-bioimage-analysis-course" title="Link to this heading">#</a></h2>
<p>Aurelien Barbotin, Chas Nelson, Dominic Waithe, Ola (Alexandra) Tarkowska, Mikolaj Kundegorski, Stephen Cross, Todd Fallesen</p>
<p>Licensed GPL-3.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/RMS-DAIM/Python-for-Bioimage-Analysis">RMS-DAIM/Python-for-Bioimage-Analysis</a></p>
</section>
<hr class="docutils" />
<section id="idc-tutorials">
<h2>IDC-Tutorials<a class="headerlink" href="#idc-tutorials" title="Link to this heading">#</a></h2>
<p>Andrey Fedorov, Daniela Schacherer, Dennis Bontempi, Bill Clifford, Vamsi Thiriveedhi, Brianna Major, wlongabaugh, dependabot[bot], Deepa Krishnaswamy</p>
<p>Published 2024-11-28T17:20:53+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Self-guided notebook tutorials to help get started with IDC</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ImagingDataCommons/IDC-Tutorials">ImagingDataCommons/IDC-Tutorials</a></p>
</section>
<hr class="docutils" />
<section id="itkelastix-examples">
<h2>ITKElastix Examples<a class="headerlink" href="#itkelastix-examples" title="Link to this heading">#</a></h2>
<p>Licensed APACHE-2.0</p>
<p>Tags: Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/InsightSoftwareConsortium/ITKElastix/tree/main/examples">InsightSoftwareConsortium/ITKElastix</a></p>
</section>
<hr class="docutils" />
<section id="image-processing-with-python">
<h2>Image Processing with Python<a class="headerlink" href="#image-processing-with-python" title="Link to this heading">#</a></h2>
<p>Jacob Deppen, Kimberly Meechan, David Palmquist, Ulf Schiller, Robert Turner, Marianne Corvellec, Toby Hodges, et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://datacarpentry.org/image-processing/">https://datacarpentry.org/image-processing/</a></p>
<p><a class="github reference external" href="https://github.com/datacarpentry/image-processing">datacarpentry/image-processing</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends">
<h2>Image analysis and visualization in Python with scikit-image, napari, and friends<a class="headerlink" href="#image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/scipy-2023-image-analysis/tutorial">scipy-2023-image-analysis/tutorial</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-course-material">
<h2>Image analysis course material<a class="headerlink" href="#image-analysis-course-material" title="Link to this heading">#</a></h2>
<p>Christian Tischer</p>
<p>Licensed MIT</p>
<p>Training materials about image registration, big warp and elastix</p>
<p><a class="github reference external" href="https://github.com/tischi/image-analysis-course-material">tischi/image-analysis-course-material</a></p>
</section>
<hr class="docutils" />
<section id="image-processing-for-beginners">
<h2>Image processing for beginners<a class="headerlink" href="#image-processing-for-beginners" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/PyImageCourse_beginner">guiwitz/PyImageCourse_beginner</a></p>
</section>
<hr class="docutils" />
<section id="id1">
<h2>Image processing with Python<a class="headerlink" href="#id1" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed MIT</p>
<p>Series of Notebooks exposing how to do mostly basic and some advanced image processing using Python. It uses standard packages (Numpy, Maplotlib) and for the image processing parts is heavily based on the scikit-image package.</p>
<p>Tags: Python</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/Python_image_processing">guiwitz/Python_image_processing</a></p>
</section>
<hr class="docutils" />
<section id="image-based-profiling-handbook">
<h2>Image-based Profiling Handbook<a class="headerlink" href="#image-based-profiling-handbook" title="Link to this heading">#</a></h2>
<p>Beth Cimini, Tim Becker, Shantanu Singh, Gregory Way, Hamdah Abbasi, Callum Tromans-Coia</p>
<p>Licensed CC0-1.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/cytomining/profiling-handbook">cytomining/profiling-handbook</a></p>
<p><a class="reference external" href="https://cytomining.github.io/profiling-handbook/">https://cytomining.github.io/profiling-handbook/</a></p>
</section>
<hr class="docutils" />
<section id="imagej-macro-introduction">
<h2>ImageJ Macro Introduction<a class="headerlink" href="#imagej-macro-introduction" title="Link to this heading">#</a></h2>
<p>Anna Klemm</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Imagej Macro, Bioimage Analysis</p>
<p>Content type: Slides, Code</p>
<p><a class="github reference external" href="https://github.com/ahklemm/ImageJMacro_Introduction">ahklemm/ImageJMacro_Introduction</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-deep-learning-for-microscopy">
<h2>Introduction to Deep Learning for Microscopy<a class="headerlink" href="#introduction-to-deep-learning-for-microscopy" title="Link to this heading">#</a></h2>
<p>Costantin Pape</p>
<p>Licensed MIT</p>
<p>This course consists of lectures and exercises that teach the background of deep learning for image analysis and show applications to classification and segmentation analysis problems.</p>
<p>Tags: Artificial Intelligence, Python</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/computational-cell-analytics/dl-for-micro">computational-cell-analytics/dl-for-micro</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-image-analysis-with-fiji">
<h2>Introduction to Image Analysis with Fiji<a class="headerlink" href="#introduction-to-image-analysis-with-fiji" title="Link to this heading">#</a></h2>
<p>Robert Haase, Benoit Lombardot</p>
<p>Licensed CC-BY-NC-4.0</p>
<p>Tags: Imagej, Fiji, Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/mpicbg-scicomp/CourseIntroToIA">mpicbg-scicomp/CourseIntroToIA</a></p>
</section>
<hr class="docutils" />
<section id="kreshuk-lab-s-embl-eipp-predoc-course-teaching-material">
<h2>Kreshuk Lab’s EMBL EIPP predoc course teaching material<a class="headerlink" href="#kreshuk-lab-s-embl-eipp-predoc-course-teaching-material" title="Link to this heading">#</a></h2>
<p>Adrian Wolny, Johannes Hugger, Qin Yu, Buglakova Alyona</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/kreshuklab/predoc-course">kreshuklab/predoc-course</a></p>
</section>
<hr class="docutils" />
<section id="leo">
<h2>LEO<a class="headerlink" href="#leo" title="Link to this heading">#</a></h2>
<p>Rodrigo Escobar Diaz Guerrero</p>
<p>Published 2025-01-08T10:20:30+00:00</p>
<p>Licensed MIT</p>
<p>Linking Electronic Lab Notebooks and other sources with OMERO objects</p>
<p>Tags: OMERO, Research Data Management, Electronic Lab Notebooks</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/LEO">NFDI4BIOIMAGE/LEO</a></p>
</section>
<hr class="docutils" />
<section id="lecture-materials-of-the-deeplife-course">
<h2>Lecture-materials of the DeepLife course<a class="headerlink" href="#lecture-materials-of-the-deeplife-course" title="Link to this heading">#</a></h2>
<p>Carl Herrmann, annavonbachmann, David Hoksza, Martin Schätz, Dario Malchiodi, jnguyenvan, Britta Velten, Elodie Laine, JanaBraunger, barwil</p>
<p>Published 2023-12-06</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioinformatics</p>
<p>Content type: Github Repository, Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/deeplife4eu/Lecture-materials/">deeplife4eu/Lecture-materials</a></p>
</section>
<hr class="docutils" />
<section id="midog-2021">
<h2>MIDOG 2021<a class="headerlink" href="#midog-2021" title="Link to this heading">#</a></h2>
<p>Marc Aubreville, Frauke Wilm</p>
<p>Published 2021-03-16</p>
<p>Licensed UNLICENSED</p>
<p>Mitosis domain generation. Here you can find code of our own evaluations and a dockered reference algorithm for mitotic figures to use as a template.</p>
<p>Tags: Ai-Ready</p>
<p>Content type: Data</p>
<p><a class="github reference external" href="https://github.com/DeepMicroscopy/MIDOG">DeepMicroscopy/MIDOG</a></p>
</section>
<hr class="docutils" />
<section id="machine-learning-workflow-for-imaging-flow-cytometry-ifc">
<h2>Machine Learning Workflow for Imaging Flow Cytometry (IFC)<a class="headerlink" href="#machine-learning-workflow-for-imaging-flow-cytometry-ifc" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: Cellprofiler, Bioimage Analysis</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/holgerhennig/machine-learning-IFC">holgerhennig/machine-learning-IFC</a></p>
</section>
<hr class="docutils" />
<section id="materials-for-embl-coding-club-mini-tutorials">
<h2>Materials for EMBL Coding Club Mini-Tutorials<a class="headerlink" href="#materials-for-embl-coding-club-mini-tutorials" title="Link to this heading">#</a></h2>
<p>Jonas Hartmann, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python</p>
<p>Content type: Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/WhoIsJack/EMBL-CodingClub">WhoIsJack/EMBL-CodingClub</a></p>
<p><a class="reference external" href="https://bio-it.embl.de/Coding%20Club/Curated%20Tutorials/">https://bio-it.embl.de/Coding%20Club/Curated%20Tutorials/</a></p>
</section>
<hr class="docutils" />
<section id="metabolic-networks-modelling-with-cobrapy">
<h2>Metabolic networks modelling with COBRApy<a class="headerlink" href="#metabolic-networks-modelling-with-cobrapy" title="Link to this heading">#</a></h2>
<p>Karin Sasaki, Aleksej Zelezniak</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/karinsasaki/metabolic-networks-modelling">karinsasaki/metabolic-networks-modelling</a></p>
</section>
<hr class="docutils" />
<section id="models-and-applications-for-bioimage-io">
<h2>Models and Applications for <a class="reference external" href="http://BioImage.IO">BioImage.IO</a><a class="headerlink" href="#models-and-applications-for-bioimage-io" title="Link to this heading">#</a></h2>
<p>Wei Ouyang, et al.</p>
<p>Licensed UNKNOWN</p>
<p><a class="github reference external" href="https://github.com/imjoy-team/bioimage-io-resources">imjoy-team/bioimage-io-resources</a></p>
</section>
<hr class="docutils" />
<section id="multiplexed-tissue-imaging-tools-and-approaches">
<h2>Multiplexed tissue imaging - tools and approaches<a class="headerlink" href="#multiplexed-tissue-imaging-tools-and-approaches" title="Link to this heading">#</a></h2>
<p>Agustín Andrés Corbat, OmFrederic, Jonas Windhager, Kristína Lidayová</p>
<p>Licensed CC-BY-4.0</p>
<p>Material for the I2K 2024 “Multiplexed tissue imaging - tools and approaches” workshop</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Slides, Workshop</p>
<p><a class="github reference external" href="https://github.com/BIIFSweden/I2K2024-MTIWorkshop">BIIFSweden/I2K2024-MTIWorkshop</a></p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1">https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1</a></p>
</section>
<hr class="docutils" />
<section id="neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter">
<h2>NEUBIAS Academy &#64;HOME: Interactive Bioimage Analysis with Python and Jupyter<a class="headerlink" href="#neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter" title="Link to this heading">#</a></h2>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Neubias, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/neubias_academy_biapy">guiwitz/neubias_academy_biapy</a></p>
</section>
<hr class="docutils" />
<section id="neubias-analyst-school-2018">
<h2>NEUBIAS Analyst School 2018<a class="headerlink" href="#neubias-analyst-school-2018" title="Link to this heading">#</a></h2>
<p>Assaf Zaritsky, Csaba Molnar, Vasja Urbancic, Richard Butler, Anna Kreshuk, Vannary Meas-Yedid</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2018">miura/NEUBIAS_AnalystSchool2018</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-course-2017">
<h2>NEUBIAS Bioimage Analyst Course 2017<a class="headerlink" href="#neubias-bioimage-analyst-course-2017" title="Link to this heading">#</a></h2>
<p>Curtis Rueden, Florian Levet, J.B. Sibarta, Alexandre Dafour, Daniel Sage, Sebastien Tosi, Michal Kozubek, Jean-Yves Tinevez, Kota Miura, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_Bioimage_Analyst_Course2017">miura/NEUBIAS_Bioimage_Analyst_Course2017</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-school-2019">
<h2>NEUBIAS Bioimage Analyst School 2019<a class="headerlink" href="#neubias-bioimage-analyst-school-2019" title="Link to this heading">#</a></h2>
<p>Kota Miura, Chong Zhang, Jean-Yves Tinevez, Robert Haase, Julius Hossein, Pejamn Rasti, David Rousseau, Ignacio Arganda-Carreras, Siân Culley, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2019">miura/NEUBIAS_AnalystSchool2019</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-school-2020">
<h2>NEUBIAS Bioimage Analyst School 2020<a class="headerlink" href="#neubias-bioimage-analyst-school-2020" title="Link to this heading">#</a></h2>
<p>Marion Louveaux, Stéphane Verger, Arianne Bercowsky Rama, Ignacio Arganda-Carreras, Estibaliz Gómez-de-Mariscal, Kota Miura, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2020">miura/NEUBIAS_AnalystSchool2020</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon">
<h2>NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon)<a class="headerlink" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon" title="Link to this heading">#</a></h2>
<p>Mohamed M. Abdrabbou, Mehrnaz Babaki, Tom Boissonnet, Michele Bortolomeazzi, Eik Dahms, Vanessa A. F. Fuchs, Moritz Hoevels, Niraj Kandpal, Christoph Möhl, Joshua A. Moore, Astrid Schauss, Andrea Schrader, Torsten Stöter, Julia Thönnißen, Monica Valencia-S., H. Lukas Weil, Jens Wendt and Peter Zentis</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Arc, Dataplant, Hackathon, Nfdi4Bioimage, OMERO, Python, Research Data Management</p>
<p>Content type: Event, Publication, Documentation</p>
<p><a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/Cologne-Hackathon-2023">NFDI4BIOIMAGE/Cologne-Hackathon-2023</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10609770">https://doi.org/10.5281/zenodo.10609770</a></p>
</section>
<hr class="docutils" />
<section id="nlp-course">
<h2>NLP Course<a class="headerlink" href="#nlp-course" title="Link to this heading">#</a></h2>
<p>Yandex School of Data Analysis</p>
<p>Licensed MIT</p>
<p>YSDA course in Natural Language Processing</p>
<p>Tags: Natural Language Processing</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/yandexdataschool/nlp_course">yandexdataschool/nlp_course</a></p>
</section>
<hr class="docutils" />
<section id="neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">
<h2>Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist<a class="headerlink" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Olivier Burri, Siân Culley, Uwe Schmidt</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Neubias, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/neubias_academy_stardist">maweigert/neubias_academy_stardist</a></p>
</section>
<hr class="docutils" />
<section id="neubiaspasteur2023-advancedcellpose">
<h2>NeubiasPasteur2023_AdvancedCellPose<a class="headerlink" href="#neubiaspasteur2023-advancedcellpose" title="Link to this heading">#</a></h2>
<p>Gaelle Letort</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tutorial for running CellPose advanced functions</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/gletort/NeubiasPasteur2023_AdvancedCellPose">gletort/NeubiasPasteur2023_AdvancedCellPose</a></p>
</section>
<hr class="docutils" />
<section id="nextflow-demo-pipelines-for-image-processing">
<h2>Nextflow Demo Pipelines for Image Processing<a class="headerlink" href="#nextflow-demo-pipelines-for-image-processing" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Content type: Code</p>
<p><a class="github reference external" href="https://github.com/JaneliaSciComp/nf-demos">JaneliaSciComp/nf-demos</a></p>
</section>
<hr class="docutils" />
<section id="omero-hcs-analysis-pipeline-using-jupyter-notebooks">
<h2>OMERO - HCS analysis pipeline using Jupyter Notebooks<a class="headerlink" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks" title="Link to this heading">#</a></h2>
<p>Riccardo Massei</p>
<p>Licensed MIT</p>
<p>Material and solutions for the course ‘Bioimage data management and analysis with OMERO’ held in Heidelberg (13th May 2024) - Module 3 (1.45 pm - 3.45 pm): OMERO and Jupyter Notebooks. Main goal of the workflow is to show the potential of JN to perform reproducible image analysis in connection with an OMERO instance. In this specific example, we are performing a simple nuclei segmentation from raw images uploaded in OMERO.</p>
<p>Tags: Teaching, Bioimage Analysis, Notebooks, Python, OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/rmassei/2024-jn-omero-pipeline">rmassei/2024-jn-omero-pipeline</a></p>
</section>
<hr class="docutils" />
<section id="onbi-image-analysis-course">
<h2>ONBI Image Analysis Course<a class="headerlink" href="#onbi-image-analysis-course" title="Link to this heading">#</a></h2>
<p>Mark Jenkinson, Jens Rittscher, Dominic Waithe</p>
<p>Licensed GPL-2.0</p>
<p>This repository contains the materials for the University of Oxford DTC ONBI Image Analysis course.</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/dwaithe/ONBI_image_analysis">dwaithe/ONBI_image_analysis</a></p>
</section>
<hr class="docutils" />
<section id="object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">
<h2>Object Tracking and Track Analysis using TrackMate and CellTracksColab<a class="headerlink" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab" title="Link to this heading">#</a></h2>
<p>Joanna Pylvänäinen</p>
<p>Published None</p>
<p>Licensed GPL-3.0</p>
<p>I2K 2024 workshop materials for “Object Tracking and Track Analysis using TrackMate and CellTracksColab”</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Tutorial, Workshop, Slides</p>
<p><a class="github reference external" href="https://github.com/CellMigrationLab/I2K_2024">CellMigrationLab/I2K_2024</a></p>
</section>
<hr class="docutils" />
<section id="omero-deployment-examples">
<h2>Omero Deployment examples<a class="headerlink" href="#omero-deployment-examples" title="Link to this heading">#</a></h2>
<p>Licensed BSD-2-CLAUSE</p>
<p>Tags: OMERO</p>
<p>Content type: Collection</p>
<p><a class="github reference external" href="https://github.com/ome/omero-deployment-examples">ome/omero-deployment-examples</a></p>
</section>
<hr class="docutils" />
<section id="online-r-learning">
<h2>Online_R_learning<a class="headerlink" href="#online-r-learning" title="Link to this heading">#</a></h2>
<p>C. Li</p>
<p>Published 2023-07-09T06:27:14+00:00</p>
<p>Licensed CC0-1.0</p>
<p>Online R learning for applied statistics</p>
<p>Tags: Statistics</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/cxli233/Online_R_learning">cxli233/Online_R_learning</a></p>
</section>
<hr class="docutils" />
<section id="parallel-fiji-visualizer">
<h2>Parallel_Fiji_Visualizer<a class="headerlink" href="#parallel-fiji-visualizer" title="Link to this heading">#</a></h2>
<p>Matthew Mueller, Aaron, Advanced Bioimaging Center</p>
<p>Published 2024-05-15T06:14:24+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Fiji</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/abcucberkeley/Parallel_Fiji_Visualizer">abcucberkeley/Parallel_Fiji_Visualizer</a></p>
</section>
<hr class="docutils" />
<section id="plugin-omero-batch-plugin">
<h2>Plugin “omero-batch-plugin”<a class="headerlink" href="#plugin-omero-batch-plugin" title="Link to this heading">#</a></h2>
<p>Licensed GPL-2.0</p>
<p>An ImageJ plugin to run a script or macro on a batch of images from/to OMERO.</p>
<p>Tags: OMERO, Imagej, Imagej Macro, Github</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/GReD-Clermont/omero_batch-plugin">GReD-Clermont/omero_batch-plugin</a></p>
</section>
<hr class="docutils" />
<section id="plugin-omero-cli-transfer">
<h2>Plugin “omero-cli-transfer”<a class="headerlink" href="#plugin-omero-cli-transfer" title="Link to this heading">#</a></h2>
<p>Erick Martins Ratamero, jean-marie burel, Will Moore, Guillaume Gay, Christoph Moehl, et al.</p>
<p>Published 2024-09-14</p>
<p>Licensed GPL-2.0</p>
<p>An OMERO CLI plugin for creating and using transfer packets between OMERO servers.</p>
<p>Tags: OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ome/omero-cli-transfer">ome/omero-cli-transfer</a></p>
</section>
<hr class="docutils" />
<section id="plugin-simple-omero-client">
<h2>Plugin “simple-omero-client”<a class="headerlink" href="#plugin-simple-omero-client" title="Link to this heading">#</a></h2>
<p>Pierre Pouchin, Rdornier, kekunn, jean-marie burel</p>
<p>Licensed GPL-2.0</p>
<p>A wrapper library which can be called from scripts in Fiji, but can mostly be used in Maven projects to wrap calls to the underlying OMERO Java Gateway.</p>
<p>Tags: OMERO, Github, Fiji</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/GReD-Clermont/simple-omero-client">GReD-Clermont/simple-omero-client</a></p>
</section>
<hr class="docutils" />
<section id="pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis">
<h2>PoL Bio-Image Analysis Training School on GPU-Accelerated Image Analysis<a class="headerlink" href="#pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis" title="Link to this heading">#</a></h2>
<p>Stephane Rigaud, Brian Northan, Till Korten, Neringa Jurenaite, Apurv Deepak Kulkarni, Peter Steinbach, Sebastian Starke, Johannes Soltwedel, Marvin Albert, Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository hosts notebooks, information and data for the GPU-Accelerated Image Analysis Track of the PoL Bio-Image Analysis Symposium.</p>
<p>Tags: Gpu, Clesperanto, Dask, Python</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/BiAPoL/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/">BiAPoL/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis</a></p>
</section>
<hr class="docutils" />
<section id="predicting-axillary-lymph-node-metastasis-in-early-breast-cancer-using-deep-learning-on-primary-tumor-biopsy-slides">
<h2>Predicting Axillary Lymph Node Metastasis in Early Breast Cancer Using Deep Learning on Primary Tumor Biopsy Slides<a class="headerlink" href="#predicting-axillary-lymph-node-metastasis-in-early-breast-cancer-using-deep-learning-on-primary-tumor-biopsy-slides" title="Link to this heading">#</a></h2>
<p>Wenqi Tang, MIC Group</p>
<p>Published 2021-12-12</p>
<p>Licensed UNLICENSED</p>
<p>This repo is the official implementation of our paper “Predicting Axillary Lymph Node Metastasis in Early Breast Cancer Using Deep Learning on Primary Tumor Biopsy Slides”.</p>
<p>Tags: Ai-Ready</p>
<p>Content type: Data</p>
<p><a class="github reference external" href="https://github.com/bupt-ai-cz/BALNMP">bupt-ai-cz/BALNMP</a></p>
</section>
<hr class="docutils" />
<section id="prompt-engineering-llms-course">
<h2>Prompt-Engineering-LLMs-Course<a class="headerlink" href="#prompt-engineering-llms-course" title="Link to this heading">#</a></h2>
<p>Published 2024-09-11T07:45:30+00:00</p>
<p>Licensed MIT</p>
<p>Tags: Llms, Prompt Engineering, Code Generation</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/HelmholtzAI-Consultants-Munich/Prompt-Engineering-LLMs-Course">HelmholtzAI-Consultants-Munich/Prompt-Engineering-LLMs-Course</a></p>
</section>
<hr class="docutils" />
<section id="python-bioimage-analysis-tutorial">
<h2>Python BioImage Analysis Tutorial<a class="headerlink" href="#python-bioimage-analysis-tutorial" title="Link to this heading">#</a></h2>
<p>Jonas Hartmann</p>
<p>Licensed MIT</p>
<p>Tags: Python, Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/WhoIsJack/python-bioimage-analysis-tutorial">WhoIsJack/python-bioimage-analysis-tutorial</a></p>
</section>
<hr class="docutils" />
<section id="python-workshop-image-processing">
<h2>Python Workshop - Image Processing<a class="headerlink" href="#python-workshop-image-processing" title="Link to this heading">#</a></h2>
<p>Karin Sasaki, Jonas Hartmann</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python</p>
<p>Content type: Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/karinsasaki/python-workshop-image-processing">karinsasaki/python-workshop-image-processing</a></p>
</section>
<hr class="docutils" />
<section id="python-for-microscopists">
<h2>Python for Microscopists<a class="headerlink" href="#python-for-microscopists" title="Link to this heading">#</a></h2>
<p>Sreenivas Bhattiprolu</p>
<p>Licensed MIT</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/bnsreenu/python_for_microscopists">bnsreenu/python_for_microscopists</a></p>
</section>
<hr class="docutils" />
<section id="qupath-for-python-programmers">
<h2>QuPath for Python programmers<a class="headerlink" href="#qupath-for-python-programmers" title="Link to this heading">#</a></h2>
<p>Alan O’Callaghan, Léo Leplat</p>
<p>Published 2024-10-23</p>
<p>Licensed UNKNOWN</p>
<p>These are the notebooks and associated files for the i2k 2024 QuPath for Python programmers workshop.</p>
<p>Tags: Python, Notebooks, Open Source Software, Bioimage Analysis</p>
<p>Content type: Github Repository, Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/qupath/i2k-qupath-for-python-programmers">qupath/i2k-qupath-for-python-programmers</a></p>
</section>
<hr class="docutils" />
<section id="rdm4mic-presentations">
<h2>RDM4Mic Presentations<a class="headerlink" href="#rdm4mic-presentations" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management</p>
<p>Content type: Collection</p>
<p><a class="github reference external" href="https://github.com/German-BioImaging/RDM4mic/tree/master/presentations">German-BioImaging/RDM4mic</a></p>
</section>
<hr class="docutils" />
<section id="rdm-system-connector">
<h2>RDM_system_connector<a class="headerlink" href="#rdm-system-connector" title="Link to this heading">#</a></h2>
<p>SaibotMagd</p>
<p>Licensed UNKNOWN</p>
<p>This tool is intended to link different research data management platforms with each other.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/SaibotMagd/RDM_system_connector">SaibotMagd/RDM_system_connector</a></p>
</section>
<hr class="docutils" />
<section id="scientific-visualization-python-matplotlib">
<h2>Scientific Visualization: Python + Matplotlib<a class="headerlink" href="#scientific-visualization-python-matplotlib" title="Link to this heading">#</a></h2>
<p>Nicolas P. Rougier</p>
<p>Licensed CC-BY-ND-SA-4.0</p>
<p>Tags: Python</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/rougier/scientific-visualization-book">rougier/scientific-visualization-book</a></p>
<p><a class="reference external" href="https://inria.hal.science/hal-03427242/document">https://inria.hal.science/hal-03427242/document</a></p>
</section>
<hr class="docutils" />
<section id="scipy-cookbook">
<h2>Scipy Cookbook<a class="headerlink" href="#scipy-cookbook" title="Link to this heading">#</a></h2>
<p>Pauli Virtanen et al.</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This is a conversion and second life of SciPy Cookbook as a bunch of Ipython notebooks.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scipy/scipy-cookbook">scipy/scipy-cookbook</a></p>
</section>
<hr class="docutils" />
<section id="scripts-filopodyanr-a-case-study-for-the-neubias-ts7-in-szeged">
<h2>Scripts_FilopodyanR - a case study for the NEUBIAS TS7 in Szeged<a class="headerlink" href="#scripts-filopodyanr-a-case-study-for-the-neubias-ts7-in-szeged" title="Link to this heading">#</a></h2>
<p>Marion Louveaux</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Code</p>
<p><a class="github reference external" href="https://github.com/marionlouveaux/NEUBIAS2018_TS7/">marionlouveaux/NEUBIAS2018_TS7</a></p>
</section>
<hr class="docutils" />
<section id="setting-up-a-remote-desktop-to-use-napari-in-a-browser">
<h2>Setting up a remote desktop to use Napari in a browser<a class="headerlink" href="#setting-up-a-remote-desktop-to-use-napari-in-a-browser" title="Link to this heading">#</a></h2>
<p>Volker Hilsenstein</p>
<p>Licensed UNKNOWN</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/VolkerH/Jupyter-Napari-Desktop">VolkerH/Jupyter-Napari-Desktop</a></p>
</section>
<hr class="docutils" />
<section id="simpleitk-notebooks">
<h2>SimpleITK-Notebooks<a class="headerlink" href="#simpleitk-notebooks" title="Link to this heading">#</a></h2>
<p>Ziv Yaniv et al.</p>
<p>Licensed APACHE-2.0</p>
<p>Jupyter notebooks for learning how to use SimpleITK</p>
<p>Tags: Bioimage Analysis, Simpleitk</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/InsightSoftwareConsortium/SimpleITK-Notebooks">InsightSoftwareConsortium/SimpleITK-Notebooks</a></p>
</section>
<hr class="docutils" />
<section id="source-control-using-git-and-github">
<h2>Source Control Using Git and GitHub<a class="headerlink" href="#source-control-using-git-and-github" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/JaneliaSciComp/2020AprilGitCourse">JaneliaSciComp/2020AprilGitCourse</a></p>
</section>
<hr class="docutils" />
<section id="statistical-rethinking">
<h2>Statistical Rethinking<a class="headerlink" href="#statistical-rethinking" title="Link to this heading">#</a></h2>
<p>Richard McElreath</p>
<p>Published 2024-03-01</p>
<p>Licensed CC0-1.0</p>
<p>This course teaches data analysis, but it focuses on scientific models. The unfortunate truth about data is that nothing much can be done with it, until we say what caused it. We will prioritize conceptual, causal models and precise questions about those models. We will use Bayesian data analysis to connect scientific models to evidence. And we will learn powerful computational tools for coping with high-dimension, imperfect data of the kind that biologists and social scientists face.</p>
<p>Tags: Statistics</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/rmcelreath/stat_rethinking_2024">rmcelreath/stat_rethinking_2024</a></p>
</section>
<hr class="docutils" />
<section id="teaching-bioimage-analysis-with-python">
<h2>Teaching Bioimage Analysis with Python<a class="headerlink" href="#teaching-bioimage-analysis-with-python" title="Link to this heading">#</a></h2>
<p>Rafael Camacho</p>
<p>Licensed MIT</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/CamachoDejay/teaching-bioimage-analysis-python">CamachoDejay/teaching-bioimage-analysis-python</a></p>
</section>
<hr class="docutils" />
<section id="teaching-imagej-fiji">
<h2>Teaching ImageJ FIJI<a class="headerlink" href="#teaching-imagej-fiji" title="Link to this heading">#</a></h2>
<p>Rafael Camacho</p>
<p>Licensed MIT</p>
<p>Tags: Fiji, Bioimage Analysis</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/CamachoDejay/Teaching-ImageJ-FIJI">CamachoDejay/Teaching-ImageJ-FIJI</a></p>
</section>
<hr class="docutils" />
<section id="training-deep-learning-models-for-vision-compact-course">
<h2>Training Deep Learning Models for Vision - Compact Course<a class="headerlink" href="#training-deep-learning-models-for-vision-compact-course" title="Link to this heading">#</a></h2>
<p>Constantin Pape, Adrian Wolny</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/constantinpape/training-deep-learning-models-for-vison">constantinpape/training-deep-learning-models-for-vison</a></p>
</section>
<hr class="docutils" />
<section id="trendsinmicroscopy2025">
<h2>TrendsInMicroscopy2025<a class="headerlink" href="#trendsinmicroscopy2025" title="Link to this heading">#</a></h2>
<p>Marcelo Zoccoler, Johannes Soltwedel</p>
<p>Published 2025-03-10T13:42:57+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Course contents for biapol course at Trends in Microscopy conference 2025</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="reference external" href="https://biapol.github.io/TrendsInMicroscopy_2025/">https://biapol.github.io/TrendsInMicroscopy_2025/</a></p>
<p><a class="github reference external" href="https://github.com/BiAPoL/TrendsInMicroscopy_2025">BiAPoL/TrendsInMicroscopy_2025</a></p>
</section>
<hr class="docutils" />
<section id="ultrack-i2k-2024-workshop-materials">
<h2>Ultrack I2K 2024 Workshop Materials<a class="headerlink" href="#ultrack-i2k-2024-workshop-materials" title="Link to this heading">#</a></h2>
<p>Jordão Bragantini, Teun Huijben</p>
<p>Licensed BSD3-CLAUSE</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/royerlab/ultrack-i2k2024">royerlab/ultrack-i2k2024</a></p>
<p><a class="reference external" href="https://royerlab.github.io/ultrack-i2k2024/">https://royerlab.github.io/ultrack-i2k2024/</a></p>
</section>
<hr class="docutils" />
<section id="v4sdb-winter-school-2025">
<h2>V4SDB_Winter_School_2025<a class="headerlink" href="#v4sdb-winter-school-2025" title="Link to this heading">#</a></h2>
<p>Joanna Pylvänäinen</p>
<p>Published 2025-01-13T08:29:22+00:00</p>
<p>Training materials for V4SDB Student Winter School, 28th-31st January 2025 at ELTE Eötvös Loránd University in Budapest, Hungary</p>
<p>Tags: Cell Tracking, Bioimage Analysis</p>
<p>Content type: Github Repository, Collection</p>
<p><a class="github reference external" href="https://github.com/CellMigrationLab/V4SDB_Winter_School_2025">CellMigrationLab/V4SDB_Winter_School_2025</a></p>
</section>
<hr class="docutils" />
<section id="virtual-i2k-2024-multiview-stitcher">
<h2>Virtual-I2K-2024-multiview-stitcher<a class="headerlink" href="#virtual-i2k-2024-multiview-stitcher" title="Link to this heading">#</a></h2>
<p>Published 2024-10-30T07:38:11+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Repository accompanying the multiview-stitcher tutorial for Virtual I2K 2024</p>
<p>Tags: Big Data, Bioimageanalysis</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/m-albert/Virtual-I2K-2024-multiview-stitcher">m-albert/Virtual-I2K-2024-multiview-stitcher</a></p>
</section>
<hr class="docutils" />
<section id="workshop-june2024-madrid">
<h2>Workshop-June2024-Madrid<a class="headerlink" href="#workshop-june2024-madrid" title="Link to this heading">#</a></h2>
<p>Licensed MIT</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/bioimage-io/Workshop-June2024-Madrid">bioimage-io/Workshop-June2024-Madrid</a></p>
</section>
<hr class="docutils" />
<section id="ymia-python-based-event-series-training-material">
<h2>YMIA - Python-Based Event Series Training Material<a class="headerlink" href="#ymia-python-based-event-series-training-material" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Robert Haase, ENicolay</p>
<p>Published None</p>
<p>Licensed MIT</p>
<p>This repository offer access to teaching material and useful resources for the YMIA - Python-Based Event Series.</p>
<p>Tags: Python, Artifical Intelligence, Bioimage Analysis</p>
<p>Content type: Github Repository, Slides</p>
<p><a class="github reference external" href="https://github.com/rmassei/ymia_python_event_series_material">rmassei/ymia_python_event_series_material</a></p>
</section>
<hr class="docutils" />
<section id="zen-python-workshop">
<h2>ZEN &amp; Python workshop<a class="headerlink" href="#zen-python-workshop" title="Link to this heading">#</a></h2>
<p>Licensed GPL-3.0</p>
<p>Tags: Python, Napari, Bioimage Analysis</p>
<p>Content type: Collection, Notebook</p>
<p><a class="github reference external" href="https://github.com/zeissmicroscopy/ZEN_Python_OAD_workshop">zeissmicroscopy/ZEN_Python_OAD_workshop</a></p>
</section>
<hr class="docutils" />
<section id="zidas-2020-introduction-to-deep-learning">
<h2>ZIDAS 2020 Introduction to Deep Learning<a class="headerlink" href="#zidas-2020-introduction-to-deep-learning" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/esgomezm/zidas2020_intro_DL">esgomezm/zidas2020_intro_DL</a></p>
</section>
<hr class="docutils" />
<section id="zerocostdl4mic-exploiting-google-colab-to-develop-a-free-and-open-source-toolbox-for-deep-learning-in-microscopy">
<h2>ZeroCostDL4Mic: exploiting Google Colab to develop a free and open-source toolbox for Deep-Learning in microscopy<a class="headerlink" href="#zerocostdl4mic-exploiting-google-colab-to-develop-a-free-and-open-source-toolbox-for-deep-learning-in-microscopy" title="Link to this heading">#</a></h2>
<p>Lucas von Chamier, Romain F. Laine, Johanna Jukkala, Christoph Spahn, Daniel Krentzel, Elias Nehme, Martina Lerche, Sara Hernández-pérez, Pieta Mattila, Eleni Karinou, Séamus Holden, Ahmet Can Solak, Alexander Krull, Tim-Oliver Buchholz, Martin L Jones, Loic Alain Royer, Christophe Leterrier, Yoav Shechtman, Florian Jug, Mike Heilemann, Guillaume Jacquemet, Ricardo Henriques</p>
<p>Licensed MIT</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/HenriquesLab/ZeroCostDL4Mic">HenriquesLab/ZeroCostDL4Mic</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/s41467-021-22518-0">https://www.nature.com/articles/s41467-021-22518-0</a></p>
<p><a class="reference external" href="https://doi.org/10.1038/s41467-021-22518-0">https://doi.org/10.1038/s41467-021-22518-0</a></p>
</section>
<hr class="docutils" />
<section id="bioformats2raw-converter">
<h2>bioformats2raw Converter<a class="headerlink" href="#bioformats2raw-converter" title="Link to this heading">#</a></h2>
<p>Melissa Linkert, Chris Allan, Josh Moore, Sébastien Besson, David Gault, et al.</p>
<p>Licensed GPL-2.0</p>
<p>Java application to convert image file formats, including .mrxs, to an intermediate Zarr structure compatible with the OME-NGFF specification.</p>
<p>Tags: Open Source Software</p>
<p>Content type: Application, Github Repository</p>
<p><a class="github reference external" href="https://github.com/glencoesoftware/bioformats2raw">glencoesoftware/bioformats2raw</a></p>
</section>
<hr class="docutils" />
<section id="bioimageio-chatbot">
<h2>bioimageio-chatbot<a class="headerlink" href="#bioimageio-chatbot" title="Link to this heading">#</a></h2>
<p>Wei Ouyang, Wanlu Lei, Caterina Fuster-Barceló, Gabe Reder, arratemunoz, Weize, Curtis Rueden, Matt McCormick</p>
<p>Published 2023-10-10T16:05:49+00:00</p>
<p>Licensed MIT</p>
<p>Your Personal Assistant in Computational Bioimaging.</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/bioimage-io/bioimageio-chatbot">bioimage-io/bioimageio-chatbot</a></p>
</section>
<hr class="docutils" />
<section id="data-visualization-tutorial">
<h2>data_visualization_tutorial<a class="headerlink" href="#data-visualization-tutorial" title="Link to this heading">#</a></h2>
<p>Stefan Halfpap</p>
<p>Published 2025-04-25T21:00:39+00:00</p>
<p>Tags: Data Visualization</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/klauck/data_visualization_tutorial">klauck/data_visualization_tutorial</a></p>
</section>
<hr class="docutils" />
<section id="datenbiene">
<h2>datenbiene<a class="headerlink" href="#datenbiene" title="Link to this heading">#</a></h2>
<p>Torsten Stöter</p>
<p>Published 2025-02-02T18:50:20+00:00</p>
<p>Licensed GNU GENERAL PUBLIC LICENSE V3.0</p>
<p>Tags: Research Data Management</p>
<p>Content type: Github Repository, Software</p>
<p><a class="github reference external" href="https://github.com/tstoeter/datenbiene">tstoeter/datenbiene</a></p>
</section>
<hr class="docutils" />
<section id="dmtxsamplecreator">
<h2>dmtxSampleCreator<a class="headerlink" href="#dmtxsamplecreator" title="Link to this heading">#</a></h2>
<p>SaibotMagd</p>
<p>Published 2023-06-06T11:52:14+00:00</p>
<p>Licensed APACHE-2.0</p>
<p>firefox extension: reads datamatrix code from camera and create a sample in an inventory to link it into an ELN.</p>
<p>Tags: Nfdi4Bioimage</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/SaibotMagd/dmtxSampleCreator">SaibotMagd/dmtxSampleCreator</a></p>
</section>
<hr class="docutils" />
<section id="i2k-2020-s3-zarr-workshop">
<h2>i2k-2020-s3-zarr-workshop<a class="headerlink" href="#i2k-2020-s3-zarr-workshop" title="Link to this heading">#</a></h2>
<p>Constantin Pape, Christian Tischer</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Big Data</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/tischi/i2k-2020-s3-zarr-workshop">tischi/i2k-2020-s3-zarr-workshop</a></p>
</section>
<hr class="docutils" />
<section id="image-processing-basics">
<h2>image-processing-basics<a class="headerlink" href="#image-processing-basics" title="Link to this heading">#</a></h2>
<p>Myles Scolnick</p>
<p>Published 2025-05-27T08:31:14+00:00</p>
<p>Licensed APACHE-2.0</p>
<p>Teaching material for image processing and analysis</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/fmi-faim/image-processing-basics">fmi-faim/image-processing-basics</a></p>
</section>
<hr class="docutils" />
<section id="imagequantification101">
<h2>imagequantification101<a class="headerlink" href="#imagequantification101" title="Link to this heading">#</a></h2>
<p>Virginie, Guillaume Witz, Joel Lüthi</p>
<p>Published 2022-08-17T14:55:16+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/uhlmanngroup/imagequantification101">uhlmanngroup/imagequantification101</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-generative-ai">
<h2>introduction-to-generative-ai<a class="headerlink" href="#introduction-to-generative-ai" title="Link to this heading">#</a></h2>
<p>Bruna Piereck, Alexander Botzki</p>
<p>Published 2024-09-27T14:38:51+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Course repository for Strategic Use of Generative AI</p>
<p>Tags: Artificial Intelligence</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/vibbits/introduction-to-generative-ai">vibbits/introduction-to-generative-ai</a></p>
<p><a class="reference external" href="https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/introduction-to-generative-ai/refs/heads/main/README.md">https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/introduction-to-generative-ai/refs/heads/main/README.md</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-image-analysis">
<h2>introduction-to-image-analysis<a class="headerlink" href="#introduction-to-image-analysis" title="Link to this heading">#</a></h2>
<p>Dave Barry, Stefania Marcotti, Martin Jones</p>
<p>Published 2024-10-23T14:05:55+00:00</p>
<p>Licensed CC-BY-SA-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Notebooks</p>
<p><a class="github reference external" href="https://github.com/RMS-DAIM/introduction-to-image-analysis">RMS-DAIM/introduction-to-image-analysis</a></p>
</section>
<hr class="docutils" />
<section id="napari-scipy2025-workshop">
<h2>napari-scipy2025-workshop<a class="headerlink" href="#napari-scipy2025-workshop" title="Link to this heading">#</a></h2>
<p>Draga Doncila Pop</p>
<p>Published 2025-02-28T23:41:10+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This is a three-part workshop guiding you through using napari to view images, a brief bioimaging analysis application, and extending napari’s functionality with your own custom workflows.</p>
<p>Tags: Python, Napari</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/DragaDoncila/napari-scipy2025-workshop">DragaDoncila/napari-scipy2025-workshop</a></p>
</section>
<hr class="docutils" />
<section id="nextflow-workshop">
<h2>nextflow-workshop<a class="headerlink" href="#nextflow-workshop" title="Link to this heading">#</a></h2>
<p>Tuur Muyldermans, Kris Davie, Alexander, Nicolas Vannieuwkerke, Kobe Lavaerts, Marcel Ribeiro-Dantas, Bruna Piereck, Steff Taelman</p>
<p>Published 2023-03-29T10:40:04+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Nextflow workshop materials March 2023</p>
<p>Tags: Workflow, Nextflow</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/vibbits/nextflow-workshop">vibbits/nextflow-workshop</a></p>
<p><a class="reference external" href="https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/nextflow-workshop/main/README.md#1">https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/nextflow-workshop/main/README.md#1</a></p>
</section>
<hr class="docutils" />
<section id="nextflow101">
<h2>nextflow101<a class="headerlink" href="#nextflow101" title="Link to this heading">#</a></h2>
<p>Virginie</p>
<p>Published 2025-08-19T14:51:21+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>NextFlow 101</p>
<p>Tags: Bioimage Analysis, Workflows</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/uhlmanngroup/nextflow101">uhlmanngroup/nextflow101</a></p>
</section>
<hr class="docutils" />
<section id="numpy-pandas-course">
<h2>numpy pandas course<a class="headerlink" href="#numpy-pandas-course" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/NumpyPandas_course">guiwitz/NumpyPandas_course</a></p>
</section>
<hr class="docutils" />
<section id="ome-ngff-validator">
<h2>ome-ngff-validator<a class="headerlink" href="#ome-ngff-validator" title="Link to this heading">#</a></h2>
<p>Will Moore, Josh Moore, Yaroslav Halchenko, Sébastien Besson</p>
<p>Published 2022-09-29</p>
<p>Licensed BSD-2-CLAUSE</p>
<p>Web page for validating OME-NGFF files.</p>
<p>Content type: Github Repository, Application</p>
<p><a class="reference external" href="https://ome.github.io/ome-ngff-validator/">https://ome.github.io/ome-ngff-validator/</a></p>
<p><a class="github reference external" href="https://github.com/ome/ome-ngff-validator">ome/ome-ngff-validator</a></p>
</section>
<hr class="docutils" />
<section id="ome2024-ngff-challenge">
<h2>ome2024-ngff-challenge<a class="headerlink" href="#ome2024-ngff-challenge" title="Link to this heading">#</a></h2>
<p>Will Moore, Josh Moore, sherwoodf, jean-marie burel, Norman Rzepka, dependabot[bot], JensWendt, Joost de Folter, Torsten St\xF6ter, AybukeKY, Eric Perlman, Tom Boissonnet</p>
<p>Published 2024-08-30T12:00:53+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Project planning and material repository for the 2024 challenge to generate 1 PB of OME-Zarr data</p>
<p>Tags: Sharing, Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ome/ome2024-ngff-challenge">ome/ome2024-ngff-challenge</a></p>
</section>
<hr class="docutils" />
<section id="omero-arc">
<h2>omero-arc<a class="headerlink" href="#omero-arc" title="Link to this heading">#</a></h2>
<p>Christoph Moehl, Peter Zentis, Niraj Kandpal</p>
<p>Published 2023-12-18T16:11:04+00:00</p>
<p>Licensed GNU GENERAL PUBLIC LICENSE V3.0</p>
<p>Library to export OMERO projects to ARC repositories</p>
<p>Tags: OMERO, Research Data Management</p>
<p>Content type: Github Repository, Software</p>
<p><a class="github reference external" href="https://github.com/cmohl2013/omero-arc">cmohl2013/omero-arc</a></p>
</section>
<hr class="docutils" />
<section id="omero-ontop-mappings">
<h2>omero-ontop-mappings<a class="headerlink" href="#omero-ontop-mappings" title="Link to this heading">#</a></h2>
<p>Carsten Fortmann-Grote, andrawaag, Jerven Bolleman</p>
<p>Published 2024-09-13T08:01:09+00:00</p>
<p>ONTOP module for querying OMERO with SPARQL</p>
<p>Tags: OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/German-BioImaging/omero-ontop-mappings">German-BioImaging/omero-ontop-mappings</a></p>
</section>
<hr class="docutils" />
<section id="omero-vitessce">
<h2>omero-vitessce<a class="headerlink" href="#omero-vitessce" title="Link to this heading">#</a></h2>
<p>Michele Bortolomeazzi</p>
<p>Published 2024-11-25T10:51:01+00:00</p>
<p>Licensed AGPL-3.0</p>
<p><a class="reference external" href="http://OMERO.web">OMERO.web</a> plugin for the Vitessce multimodal data viewer.</p>
<p>Tags: OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/omero-vitessce">NFDI4BIOIMAGE/omero-vitessce</a></p>
</section>
<hr class="docutils" />
<section id="patho-prompt-injection">
<h2>patho_prompt_injection<a class="headerlink" href="#patho-prompt-injection" title="Link to this heading">#</a></h2>
<p>JanClusmann, Tim Lenz</p>
<p>Published 2024-11-08T08:32:03+00:00</p>
<p>Licensed GPL-3.0</p>
<p>Tags: Histopathology, Bioimage Analysis</p>
<p>Content type: Github Repository, Notebook</p>
<p><a class="github reference external" href="https://github.com/KatherLab/patho_prompt_injection">KatherLab/patho_prompt_injection</a></p>
</section>
<hr class="docutils" />
<section id="quantixed-thedigitalcell-first-complete-code-set">
<h2>quantixed/TheDigitalCell: First complete code set<a class="headerlink" href="#quantixed-thedigitalcell-first-complete-code-set" title="Link to this heading">#</a></h2>
<p>Stephen Royle</p>
<p>Published 2019-04-17</p>
<p>Licensed GPL-3.0</p>
<p>First complete code set for The Digital Cell book.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Code</p>
<p><a class="github reference external" href="https://github.com/quantixed/TheDigitalCell">quantixed/TheDigitalCell</a></p>
<p><a class="reference external" href="https://zenodo.org/records/2643411">https://zenodo.org/records/2643411</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.2643411">https://doi.org/10.5281/zenodo.2643411</a></p>
</section>
<hr class="docutils" />
<section id="qupath-workshop">
<h2>qupath-workshop<a class="headerlink" href="#qupath-workshop" title="Link to this heading">#</a></h2>
<p>Antoine A. Ruzette, dependabot[bot]</p>
<p>Published 2025-01-16T14:05:02+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Materials supporting the QuPath workshop at Harvard Medical School.</p>
<p>Tags: Notebook, Slides, Collection</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/HMS-IAC/qupath-workshop">HMS-IAC/qupath-workshop</a></p>
</section>
<hr class="docutils" />
<section id="raw2ometiff-converter">
<h2>raw2ometiff Converter<a class="headerlink" href="#raw2ometiff-converter" title="Link to this heading">#</a></h2>
<p>Melissa Linkert, Chris Allan, Sébastien Besson, Josh Moore</p>
<p>Licensed GPL-2.0</p>
<p>Java application to convert a directory of tiles to an OME-TIFF pyramid. This is the second half of iSyntax/.mrxs =&gt; OME-TIFF conversion.</p>
<p>Tags: Open Source Software</p>
<p>Content type: Application, Github Repository</p>
<p><a class="github reference external" href="https://github.com/glencoesoftware/raw2ometiff">glencoesoftware/raw2ometiff</a></p>
</section>
<hr class="docutils" />
<section id="rse-skills-workshop">
<h2>rse-skills-workshop<a class="headerlink" href="#rse-skills-workshop" title="Link to this heading">#</a></h2>
<p>Jack Atkinson</p>
<p>Published 2023-12-22T17:39:48+00:00</p>
<p>Licensed GPL-3.0</p>
<p>Teaching materials for improving research software writing abilities.</p>
<p>Tags: Research Software Engineering</p>
<p>Content type: Github Repository, Slides</p>
<p><a class="github reference external" href="https://github.com/jatkinson1000/rse-skills-workshop">jatkinson1000/rse-skills-workshop</a></p>
</section>
<hr class="docutils" />
<section id="scanpy-tutorials">
<h2>scanpy-tutorials<a class="headerlink" href="#scanpy-tutorials" title="Link to this heading">#</a></h2>
<p>Alex Wolf, pre-commit-ci[bot], Philipp A., Isaac Virshup, Ilan Gold, Giovanni Palla, Fidel Ramirez, G\xF6k\xE7en Eraslan, Sergei Rybakov, Abolfazl (Abe), Adam Gayoso, Dinesh Palli, Gregor Sturm, Jan Lause, Karin Hrovatin, Krzysztof Polanski, RaphaelBuzzi, Yimin Zheng, Yishen Miao, evanbiederstedt</p>
<p>Published 2018-12-16T03:42:46+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Scanpy Tutorials.</p>
<p>Tags: Single-Cell Analysis, Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scverse/scanpy-tutorials">scverse/scanpy-tutorials</a></p>
</section>
<hr class="docutils" />
<section id="scikit-learn-mooc">
<h2>scikit-learn MOOC<a class="headerlink" href="#scikit-learn-mooc" title="Link to this heading">#</a></h2>
<p>Loïc Estève et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>Machine learning in Python with scikit-learn MOOC</p>
<p>Tags: Bioimage Analysis, Machine Learning</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/INRIA/scikit-learn-mooc">INRIA/scikit-learn-mooc</a></p>
</section>
<hr class="docutils" />
<section id="shapeanalysis101">
<h2>shapeanalysis101<a class="headerlink" href="#shapeanalysis101" title="Link to this heading">#</a></h2>
<p>Virginie, Johannes Hugger</p>
<p>Published 2021-05-17T13:32:40+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>A tutorial of classical shape analysis methods</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/uhlmanngroup/shapeanalysis101">uhlmanngroup/shapeanalysis101</a></p>
</section>
<hr class="docutils" />
<section id="skimage-tutorials">
<h2>skimage-tutorials<a class="headerlink" href="#skimage-tutorials" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias et al.</p>
<p>Licensed CC0 1.0 UNIVERSAL</p>
<p>skimage-tutorials - a collection of tutorials for the scikit-image package.</p>
<p>Tags: Bioimage Analysis, Scikit-Image</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scikit-image/skimage-tutorials">scikit-image/skimage-tutorials</a></p>
</section>
<hr class="docutils" />
<section id="training-resources">
<h2>training-resources<a class="headerlink" href="#training-resources" title="Link to this heading">#</a></h2>
<p>Christian Tischer, Antonio Politi, Toby Hodges, maulakhan, grinic, bugraoezdemir, Tim-Oliver Buchholz, Elnaz Fazeli, Aliaksandr Halavatyi, Dominik Kutra, Stefania Marcotti, AnniekStok, Felix, jhennies, Severina Klaus, Martin Schorb, Nima Vakili, Sebastian Gonzalez Tirado, Stefan Helfrich, Yi Sun, Ziqiang Huang, Jan Eglinger, Constantin Pape, Joel Lüthi, Matt McCormick, Oane Gros</p>
<p>Published 2020-04-23T07:51:38+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Resources for teaching/preparing to teach bioimage analysis</p>
<p>Tags: Bioimageanalysis, Neurobias</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NEUBIAS/training-resources">NEUBIAS/training-resources</a></p>
<hr class="docutils" />
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./domain"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="git.mpi-cbg.de.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Git.mpi-cbg.de (7)</p>
      </div>
    </a>
    <a class="right-next"
       href="training.galaxyproject.org.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Training.galaxyproject.org (5)</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-survey-community-experiences-and-needs-for-the-future">2020 BioImage Analysis Survey: Community experiences and needs for the future</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#mic-workshop-on-bioimage-processing-with-python">2022 MIC Workshop on Bioimage processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#courseimageanalysis-nanocell">2023_CourseImageAnalysis_NanoCell</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#adding-a-workflow-to-biaflows">Adding a Workflow to BIAFLOWS</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#advanced-scripting-with-fiji">Advanced scripting with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#analysis-of-high-throughput-microscopy-image-data">Analysis of High-Throughput Microscopy Image Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bccd-dataset">BCCD Dataset</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bhg2023-omero-arc">BHG2023-OMERO-ARC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bids-lecture-2024">BIDS-lecture-2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#basics-of-image-processing-and-analysis">Basics of Image Processing and Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis">Bio Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-icob-2023">Bio-image Analysis ICOB 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-kioto-and-taipei-23-24">Bio-image Analysis Workshop Kioto and Taipei 23/24</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-taipei">Bio-image Analysis Workshop Taipei</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science">Bio-image Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">Bio-image analysis, biostatistics, programming and machine learning for computational biology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimaging-ai-textbook">Bioimaging AI Textbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#biomero">Biomero</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#breast-cancer-semantic-segmentation-bcss-dataset">Breast Cancer Semantic Segmentation (BCSS) dataset</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bridging-imaging-users-to-imaging-analysis-a-community-survey">Bridging Imaging Users to Imaging Analysis - A community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-bioimage-analysis-workflow-using-deep-learning">Building a Bioimage Analysis Workflow using Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022">CARE/Stardist tutorials for EMBO Practical Course — Computational optical biology 2022</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#csbdeep-and-stardist-i2k-2020">CSBDeep and StarDist @ I2K 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-introduction">CellProfiler Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-practical-neubias-lisbon-2017">CellProfiler Practical NeuBIAS Lisbon 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-tutorials">CellProfiler tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#celltrackcolab">CellTrackColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis">Collection of teaching material for deep learning for (biomedical) image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#course-on-deep-learning-for-imaging-using-pytorch">Course on Deep Learning for imaging using PyTorch</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-napari-napari-as-a-tool-for-deep-learning-project-management">DEEP NAPARI : Napari as a tool for deep learning project management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4miceverywhere">DL4MicEverywhere</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4proteins-notebooks">DL4Proteins-notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl-mbl-2021-exercises">DL@MBL 2021 Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dask-course">Dask Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-based-segmentation-for-biologists">Deep Learning Based Segmentation For Biologists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-for-image-analysis-exercises">Deep Learning for image analysis - Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-vision-and-graphics">Deep Vision and Graphics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deepprofiler-handbook">DeepProfiler Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2019-exercises-and-materials">EMBL Deep Learning course 2019 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2021-22-exercises-and-materials">EMBL Deep Learning course 2021/22 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2023-exercises-and-materials">EMBL Deep Learning course 2023 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embedseg-repository">EmbedSeg Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus">EuBIAS course 2013: Intensity Dynamics at the Periphery of Nucleus</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#example-pipeline-tutorial">Example Pipeline Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#friendsdontletfriends">FriendsDontLetFriends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-in-digital-image-processing">Fundamentals in digital image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends">Fundamentals of image analysis in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-training-material">Galaxy Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library">I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-virtual-bio-image-analysis-code-generation">I2K2024(virtual) - Bio-Image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#iafig-rms-python-for-bioimage-analysis-course">IAFIG-RMS Python for Bioimage Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#idc-tutorials">IDC-Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#itkelastix-examples">ITKElastix Examples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-with-python">Image Processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends">Image analysis and visualization in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-course-material">Image analysis course material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-for-beginners">Image processing for beginners</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#id1">Image processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-based-profiling-handbook">Image-based Profiling Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej-macro-introduction">ImageJ Macro Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-deep-learning-for-microscopy">Introduction to Deep Learning for Microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis-with-fiji">Introduction to Image Analysis with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kreshuk-lab-s-embl-eipp-predoc-course-teaching-material">Kreshuk Lab’s EMBL EIPP predoc course teaching material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#leo">LEO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-materials-of-the-deeplife-course">Lecture-materials of the DeepLife course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#midog-2021">MIDOG 2021</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-workflow-for-imaging-flow-cytometry-ifc">Machine Learning Workflow for Imaging Flow Cytometry (IFC)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#materials-for-embl-coding-club-mini-tutorials">Materials for EMBL Coding Club Mini-Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#metabolic-networks-modelling-with-cobrapy">Metabolic networks modelling with COBRApy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#models-and-applications-for-bioimage-io">Models and Applications for BioImage.IO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter">NEUBIAS Academy @HOME: Interactive Bioimage Analysis with Python and Jupyter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-analyst-school-2018">NEUBIAS Analyst School 2018</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-course-2017">NEUBIAS Bioimage Analyst Course 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2019">NEUBIAS Bioimage Analyst School 2019</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2020">NEUBIAS Bioimage Analyst School 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nlp-course">NLP Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubiaspasteur2023-advancedcellpose">NeubiasPasteur2023_AdvancedCellPose</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-demo-pipelines-for-image-processing">Nextflow Demo Pipelines for Image Processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks">OMERO - HCS analysis pipeline using Jupyter Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#onbi-image-analysis-course">ONBI Image Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-deployment-examples">Omero Deployment examples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#online-r-learning">Online_R_learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#parallel-fiji-visualizer">Parallel_Fiji_Visualizer</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-omero-batch-plugin">Plugin “omero-batch-plugin”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-omero-cli-transfer">Plugin “omero-cli-transfer”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-simple-omero-client">Plugin “simple-omero-client”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis">PoL Bio-Image Analysis Training School on GPU-Accelerated Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#predicting-axillary-lymph-node-metastasis-in-early-breast-cancer-using-deep-learning-on-primary-tumor-biopsy-slides">Predicting Axillary Lymph Node Metastasis in Early Breast Cancer Using Deep Learning on Primary Tumor Biopsy Slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-llms-course">Prompt-Engineering-LLMs-Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-bioimage-analysis-tutorial">Python BioImage Analysis Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-workshop-image-processing">Python Workshop - Image Processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-for-microscopists">Python for Microscopists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-for-python-programmers">QuPath for Python programmers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdm4mic-presentations">RDM4Mic Presentations</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdm-system-connector">RDM_system_connector</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scientific-visualization-python-matplotlib">Scientific Visualization: Python + Matplotlib</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scipy-cookbook">Scipy Cookbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scripts-filopodyanr-a-case-study-for-the-neubias-ts7-in-szeged">Scripts_FilopodyanR - a case study for the NEUBIAS TS7 in Szeged</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-a-remote-desktop-to-use-napari-in-a-browser">Setting up a remote desktop to use Napari in a browser</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#simpleitk-notebooks">SimpleITK-Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#source-control-using-git-and-github">Source Control Using Git and GitHub</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#statistical-rethinking">Statistical Rethinking</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-bioimage-analysis-with-python">Teaching Bioimage Analysis with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-imagej-fiji">Teaching ImageJ FIJI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-deep-learning-models-for-vision-compact-course">Training Deep Learning Models for Vision - Compact Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#trendsinmicroscopy2025">TrendsInMicroscopy2025</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ultrack-i2k-2024-workshop-materials">Ultrack I2K 2024 Workshop Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#v4sdb-winter-school-2025">V4SDB_Winter_School_2025</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#virtual-i2k-2024-multiview-stitcher">Virtual-I2K-2024-multiview-stitcher</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-june2024-madrid">Workshop-June2024-Madrid</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zen-python-workshop">ZEN &amp; Python workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zidas-2020-introduction-to-deep-learning">ZIDAS 2020 Introduction to Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zerocostdl4mic-exploiting-google-colab-to-develop-a-free-and-open-source-toolbox-for-deep-learning-in-microscopy">ZeroCostDL4Mic: exploiting Google Colab to develop a free and open-source toolbox for Deep-Learning in microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioformats2raw-converter">bioformats2raw Converter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimageio-chatbot">bioimageio-chatbot</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-visualization-tutorial">data_visualization_tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenbiene">datenbiene</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dmtxsamplecreator">dmtxSampleCreator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2020-s3-zarr-workshop">i2k-2020-s3-zarr-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-basics">image-processing-basics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagequantification101">imagequantification101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-generative-ai">introduction-to-generative-ai</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis">introduction-to-image-analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#napari-scipy2025-workshop">napari-scipy2025-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-workshop">nextflow-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow101">nextflow101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#numpy-pandas-course">numpy pandas course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome-ngff-validator">ome-ngff-validator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome2024-ngff-challenge">ome2024-ngff-challenge</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-arc">omero-arc</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-ontop-mappings">omero-ontop-mappings</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-vitessce">omero-vitessce</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#patho-prompt-injection">patho_prompt_injection</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quantixed-thedigitalcell-first-complete-code-set">quantixed/TheDigitalCell: First complete code set</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-workshop">qupath-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#raw2ometiff-converter">raw2ometiff Converter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rse-skills-workshop">rse-skills-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scanpy-tutorials">scanpy-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scikit-learn-mooc">scikit-learn MOOC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#shapeanalysis101">shapeanalysis101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#skimage-tutorials">skimage-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-resources">training-resources</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Robert Haase, Clément Caporal,... and the NFDI4BioImage Initiative
</p>

  </div>
  
  <div class="footer-item">
    

  </div>
  
  <div class="footer-item">
    <p class="last-updated">
  Last updated on 2025-09-16.
  <br/>
</p>
  </div>
  
  <div class="footer-item">
    
<div class="extra_footer">
  <p>
Copyright: Licensed <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY 4.0</a> unless mentioned otherwise. 
Contributions and feedback are welcome.
</p>

</div>
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>