{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "4a41efb5-2f4a-41dd-98ba-eeecf01fdb68",
   "metadata": {},
   "source": [
    "# Importing new entries from Zenodo communities\n",
    "This notebook allows to import entries from Zenodo communities. It does not re-import entries that are already in our database."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "22ccdd0e-5d18-448d-a343-1ab634771d26",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import requests\n",
    "import bia_bob\n",
    "import shutil\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "7065b8ba-f726-41f0-ab49-48c9767fcda1",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'./generate_link_lists.py'"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# workaround: Until our utilities are a python library, we need to copy it here.\n",
    "shutil.copy('../scripts/generate_link_lists.py', './generate_link_lists.py')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "a1863536-3d4e-4144-a86e-92eff0cca25a",
   "metadata": {},
   "outputs": [],
   "source": [
    "from generate_link_lists import load_dataframe, update_yaml_file, complete_zenodo_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "f6c69566-1d52-4ec0-8733-2cc11e002cac",
   "metadata": {},
   "outputs": [],
   "source": [
    "token = os.getenv('ZENODO_API_KEY')\n",
    "community = 'nfdi4bioimage'\n",
    "\n",
    "response = requests.get('https://zenodo.org/api/records',\n",
    "                        params={'communities': community,\n",
    "                                'access_token': token})"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ecd9050d-8dcd-4b69-9d1f-00fa1a37db80",
   "metadata": {},
   "source": [
    "## That's what's listed in the community"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "efa9ffa0-159c-4996-a6bd-c5e1f4197c2a",
   "metadata": {},
   "outputs": [],
   "source": [
    "online_data = response.json()\n",
    "hits = online_data[\"hits\"][\"hits\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "1c59f531-ab9d-480a-81a2-7e5dbb9f8609",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "25"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(hits)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "28cbcc3a-a43c-47ee-b249-30032957411a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['https://zenodo.org/records/14014252',\n",
       " 'https://zenodo.org/records/14013026',\n",
       " 'https://zenodo.org/records/14001388',\n",
       " 'https://zenodo.org/records/13991322',\n",
       " 'https://zenodo.org/records/13837146',\n",
       " 'https://zenodo.org/records/13831274',\n",
       " 'https://zenodo.org/records/11503289',\n",
       " 'https://zenodo.org/records/13684187',\n",
       " 'https://zenodo.org/records/13640979',\n",
       " 'https://zenodo.org/records/13380289',\n",
       " 'https://zenodo.org/records/13168693',\n",
       " 'https://zenodo.org/records/12699637',\n",
       " 'https://zenodo.org/records/11548617',\n",
       " 'https://zenodo.org/records/11501662',\n",
       " 'https://zenodo.org/records/11350689',\n",
       " 'https://zenodo.org/records/11235513',\n",
       " 'https://zenodo.org/records/11146807',\n",
       " 'https://zenodo.org/records/11109616',\n",
       " 'https://zenodo.org/records/11031747',\n",
       " 'https://zenodo.org/records/10939520',\n",
       " 'https://zenodo.org/records/10886750',\n",
       " 'https://zenodo.org/records/10808486',\n",
       " 'https://zenodo.org/records/10793700',\n",
       " 'https://zenodo.org/records/10730424',\n",
       " 'https://zenodo.org/records/10687659']"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "urls = [u[\"links\"][\"self_html\"] for u in hits]\n",
    "urls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "09767176-77c2-4b9a-b43e-3678d27525f3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://zenodo.org/api/records/14014252\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'url': ['https://zenodo.org/records/14014252',\n",
       "  'https://doi.org/10.5281/zenodo.14014252'],\n",
       " 'name': 'Excel template for adding Key-Value Pairs to images',\n",
       " 'publication_date': '2024-10-30',\n",
       " 'description': 'This Excel Workbook contains some simple Macros to help with the generation of a .csv in the necessary format for Key-Value pair annotations of images in OMERO.\\nThe format is tailored for the OMERO.web script&nbsp;\"KeyVal_from_csv.py\"&nbsp; (from the version &lt;=5.8.3 of the core omero-scripts).\\nAttached is also a video of Thomas Zobel, the head of the imaging core facility Uni M&uuml;nster, showcasing the use of the Excel workbook.The video uses a slightly older version of the workbook and OMERO, but the core functionality remains unchanged.\\nPlease keep in mind, that the OMERO.web script(s) to handle Key-Value Pairs from/to .csv files will undergo a major change very soon.This might break the compatibility with the format used now for the generated .csv from the workbook.',\n",
       " 'authors': 'Zobel, Thomas, Wendt, Jens',\n",
       " 'license': 'cc-by-4.0',\n",
       " 'num_downloads': 47}"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "url = urls[0]\n",
    "\n",
    "data = complete_zenodo_data(url)\n",
    "data\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "658e8be1-37bd-4830-8e0d-4a6d6b79452d",
   "metadata": {},
   "source": [
    "## Checking what we already have"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "1bba931f-1766-4b4c-b700-9550efdc0fec",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Adding nfdi4bioimage.yml\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "\"['https://focalplane.biologists.com/2023/07/26/sharing-your-poster-on-figshare/', 'https://biapol.github.io/blog/marcelo_zoccoler/omero_scripts/readme.html', 'https://biapol.github.io/blog/robert_haase/browsing_idr/readme.html', 'https://biapol.github.io/blog/mara_lampert/getting_started_with_mambaforge_and_python/readme.html', 'https://datamanagement.hms.harvard.edu/news/promoting-data-management-nikon-imaging-center-and-cell-biology-microscopy-facility', 'https://blog.delmic.com/data-handling-in-large-scale-electron-microscopy', 'https://focalplane.biologists.com/2023/06/01/tracking-in-napari/', 'https://focalplane.biologists.com/2023/05/03/feature-extraction-in-napari/', 'https://focalplane.biologists.com/2023/03/30/annotating-3d-images-in-napari/', 'https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/', 'https://focalplane.biologists.com/2023/04/13/quality-assurance-of-segmentation-results/', 'https://focalplane.biologists.com/2023/03/02/rescaling-images-and-pixel-anisotropy/', 'https://forum.image.sc/t/user-friendly-image-metadata-annotation-tool-workflow-for-omero/87925/1', 'https://focalplane.biologists.com/2024/04/03/how-to-write-a-bug-report/', 'https://focalplane.biologists.com/2024/07/18/prompt-engineering-in-bio-image-analysis/', 'https://tess.elixir-europe.org/events', 'https://globalbioimaging.org/international-training-courses', 'https://www.openmicroscopy.org/events/', 'https://www.embl.org/about/info/course-and-conference-office/events/bia23-01/', 'https://biapol.github.io/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/intro.html', 'https://biapol.github.io/PoL-BioImage-Analysis-TS-Early-Career-Track/intro.html', 'https://www.dtls.nl/fair-data/byod/', 'https://github.com/bioimage-io/Workshop-June2024-Madrid', 'https://github.com/Koushouu/Bioimage-Analysis-Workshop-Taipei/', 'https://github.com/Koushouu/Bioimage-Analysis-Workshop-23-24', 'https://github.com/WeiChenChu/Bioimage_Analysis_ICOB_2023', 'https://indico.dkfz.de/event/1022/', 'https://github.com/German-BioImaging/RDM4mic/tree/master/presentations', 'https://biii.eu/training-material', 'https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/intro.html', 'https://bioimagebook.github.io/index.html', 'https://f1000research.com/slides/12-971', 'https://microscopydb.io/', 'https://www.openmicroscopy.org/docs/', 'https://omero.readthedocs.io/en/stable/', 'https://omero-guides.readthedocs.io/en/latest/', 'https://omero-guides.readthedocs.io/en/latest/example_facility_manager.html', 'https://omero-guides.readthedocs.io/en/latest/example.html', 'https://bio.tools/', 'https://gerbi-gmb.de/resources/teaching-resources/', 'https://forschungsdaten.info/', 'https://www.forschungsdaten.org/', ['https://zenodo.org/record/4071471', 'https://doi.org/10.5281/zenodo.4071471'], ['https://zenodo.org/record/3490058', 'https://doi.org/10.5281/zenodo.3490058'], 'https://rdmkit.elixir-europe.org/all_training_resources', 'https://tess.elixir-europe.org/materials', 'https://gerbi-gmb.de/i3dbio/i3dbio-teaching/train-mat/bioimagelist/', 'https://github.com/CellProfiler/tutorials', 'https://github.com/dwaithe/ONBI_image_analysis', 'https://syn.mrc-lmb.cam.ac.uk/acardona/fiji-tutorial/', 'https://git.mpi-cbg.de/rhaase/lecture_applied_bioimage_analysis', 'https://git.mpi-cbg.de/rhaase/lecture_imagej2_dev', 'https://git.mpi-cbg.de/rhaase/lecture_multiview_registration', 'https://git.mpi-cbg.de/rhaase/lecture_tracking_trackmate', 'https://git.mpi-cbg.de/rhaase/lecture_working_with_pixels', 'https://git.mpi-cbg.de/rhaase/lecture_working_with_objects_in_2d_and_3d', 'https://github.com/guiwitz/Python_image_processing', 'https://github.com/tpecot/DeepLearningBasedSegmentationForBiologists/', 'https://www.fabriziomusacchio.com/teaching/teaching_bioimage_analysis/', 'https://github.com/zeissmicroscopy/ZEN_Python_OAD_workshop', ['https://zenodo.org/record/6602101', 'https://doi.org/10.5281/zenodo.6602101'], 'https://www.regeringen.se/contentassets/55e7d8fbf6df4a54ac56942b98d94e4f/lund-declaration-on-maximising-the-benefits-of-research-data-pa-engelska.pdf', ['https://doi.org/10.1371/journal.pbio.3002167', 'https://www.bioimagingguide.org/'], ['https://www.nature.com/articles/nmeth.2083', 'https://bbbc.broadinstitute.org/'], 'https://coderefinery.github.io/github-without-command-line/doi/', 'https://the-turing-way.netlify.app/communication/citable/citable-cff.html', 'https://the-turing-way.netlify.app/reproducible-research/reproducible-research', 'https://github.com/miura/ij_textbook1/raw/76b51338e1f006c580b6f0f5cfc48fe02fba38d7/CMCIBasicCourse201102Bib.pdf', ['https://www.scienceeurope.org/our-resources/practical-guide-to-the-international-alignment-of-research-data-management/', 'https://doi.org/10.5281/zenodo.4915861'], 'http://celltrackingchallenge.net/2d-datasets/', 'http://celltrackingchallenge.net/3d-datasets/', 'https://www.microlist.org/', 'https://git.mpi-cbg.de/scicomp/bioimage_team/coursematerialimageanalysis/tree/master/ImageJMacro_24h_2017-01', 'https://github.com/mpicbg-scicomp/CourseIntroToIA', 'https://neubias.github.io/training-resources/', 'https://bioimage.io/', ['https://www.ebi.ac.uk/bioimage-archive/', 'https://www.sciencedirect.com/science/article/abs/pii/S0022283622000791'], ['https://idr.openmicroscopy.org/', 'https://www.nature.com/articles/nmeth.4326'], 'https://github.com/ome/omero-deployment-examples', 'https://ai4life.eurobioimaging.eu/announcing-bioengine/', 'https://kevinyamauchi.github.io/open-image-data/how_tos/conda_forge_packaging.html', 'https://ai4life.eurobioimaging.eu/ai4life-teams-up-with-galaxy-training-network-gtn-to-enhance-training-resources/', ['https://zenodo.org/records/8323588', 'https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU', 'https://doi.org/10.5281/zenodo.8323588'], 'https://github.com/InsightSoftwareConsortium/ITKElastix/tree/main/examples', 'https://www.edx.org/learn/image-analysis/ecole-polytechnique-federale-de-lausanne-image-processing-and-analysis-for-life-scientists', 'https://www.wiley.com/en-us/Standard+and+Super+Resolution+Bioimaging+Data+Analysis%3A+A+Primer-p-9781119096900', 'https://imagej.net/plugins/morpholibj', 'https://github.com/RMS-DAIM/Python-for-Bioimage-Analysis', 'https://zmb.dozuki.com/c/Image_Analysis', 'https://github.com/guiwitz/Fundamentals_image_processing', 'https://github.com/guiwitz/NumpyPandas_course', 'https://github.com/guiwitz/neubias_academy_biapy', 'https://github.com/bnsreenu/python_for_microscopists', 'https://github.com/VolkerH/Jupyter-Napari-Desktop', ['https://github.com/rougier/scientific-visualization-book', 'https://inria.hal.science/hal-03427242/document'], 'https://github.com/CamachoDejay/teaching-bioimage-analysis-python', 'https://github.com/CamachoDejay/Teaching-ImageJ-FIJI', 'https://github.com/jni/halfway-to-i2k-skimage-napari', 'https://github.com/scipy-2023-image-analysis/tutorial', 'https://github.com/guiwitz/DaskCourse', 'https://github.com/guiwitz/DLImaging', 'https://github.com/guiwitz/MICPy_Workshop_2022', 'https://github.com/miura/NEUBIAS_Bioimage_Analyst_Course2017', ['https://github.com/quantixed/TheDigitalCell', 'https://zenodo.org/records/2643411', 'https://doi.org/10.5281/zenodo.2643411'], 'https://cshlpress.com/default.tpl?cart=1700309488232283050&fromlink=T&linkaction=full&linksortby=oop_title&--eqSKUdatarq=1282', 'https://github.com/miura/NEUBIAS_AnalystSchool2020', 'https://github.com/miura/NEUBIAS_AnalystSchool2019', 'https://github.com/miura/NEUBIAS_AnalystSchool2018', 'https://github.com/marionlouveaux/NEUBIAS2018_TS7/', 'https://github.com/miura/BIAS_Nucleus_Segmentation/blob/master/module9.pdf', 'https://github.com/miura/HTManalysisCourse/blob/master/CentreCourseProtocol.md', 'https://github.com/tischi/presentation-image-analysis', ['https://github.com/WhoIsJack/python-bioimage-analysis-tutorial'], ['https://github.com/WhoIsJack/EMBL-CodingClub', 'https://bio-it.embl.de/Coding%20Club/Curated%20Tutorials/'], 'https://github.com/karinsasaki/python-workshop-image-processing', 'https://github.com/karinsasaki/metabolic-networks-modelling', 'https://github.com/tischi/cellprofiler-practical-NeuBIAS-Lisbon-2017', 'https://github.com/tischi/i2k-2020-s3-zarr-workshop', ['https://datacarpentry.org/image-processing/', 'https://github.com/datacarpentry/image-processing'], 'https://datacarpentry.org/semester-biology/', 'https://github.com/kreshuklab/teaching-dl-course-2019', 'https://github.com/kreshuklab/teaching-dl-course-2023', 'https://github.com/kreshuklab/teaching-dl-course-2022', 'https://github.com/yandexdataschool/deep_vision_and_graphics', 'https://github.com/kreshuklab/predoc-course', 'https://github.com/imjoy-team/bioimage-io-resources', 'https://github.com/constantinpape/training-deep-learning-models-for-vison', 'https://github.com/constantinpape/dl-teaching-resources', 'https://github.com/JaneliaSciComp/nf-demos', 'https://github.com/JaneliaSciComp/2020AprilGitCourse', ['https://github.com/HenriquesLab/ZeroCostDL4Mic', 'https://www.nature.com/articles/s41467-021-22518-0', 'https://doi.org/10.1038/s41467-021-22518-0'], 'https://github.com/HenriquesLab/DL4MicEverywhere', ['https://www.biorxiv.org/content/10.1101/2023.10.20.563252v2', 'https://github.com/guijacquemet/CellTracksColab'], 'https://github.com/tischi/image-analysis-course-material', 'https://github.com/guiwitz/PyImageCourse_beginner', 'https://github.com/ahklemm/ImageJMacro_Introduction', 'https://github.com/ahklemm/CellProfiler_Introduction', 'https://github.com/maweigert/embo_2022', 'https://github.com/maweigert/neubias_academy_stardist', 'https://github.com/maweigert/stardist-i2k', 'https://github.com/maweigert/zidas_2020_DL_intro_Part_2', 'https://github.com/holgerhennig/machine-learning-IFC', ['https://github.com/cytomining/profiling-handbook', 'https://cytomining.github.io/profiling-handbook/'], ['https://github.com/cytomining/DeepProfiler-handbook', 'https://cytomining.github.io/DeepProfiler-handbook/docs/00-welcome.html'], ['https://www.ebi.ac.uk/training/events/methods-bioimage-analysis/', 'https://doi.org/10.6019/TOL.BioImageAnalysis22-w.2022.00001.1', 'https://drive.google.com/file/d/1MhuqfKhZcYu3bchWMqogIybKjamU5Msg/view'], 'https://github.com/esgomezm/NEUBIAS_chapter_DL_2020', 'https://github.com/esgomezm/zidas2020_intro_DL', 'https://raw.githubusercontent.com/esgomezm/esgomezm.github.io/master/assets/pdf/SPAOM2018/MachineLearning_SPAOMworkshop_public.pdf', 'https://github.com/JLrumberger/DL-MBL-2021', 'https://github.com/galaxyproject/training-material', 'https://zenodo.org/doi/10.5281/zenodo.4330625', 'https://zenodo.org/communities/eoss', ['https://zenodo.org/records/4334697', 'https://doi.org/10.5281/zenodo.4334697'], ['https://zenodo.org/records/4328911', 'https://doi.org/10.5281/zenodo.4328911'], ['https://zenodo.org/records/4317149', 'https://doi.org/10.5281/zenodo.4317149'], 'https://docs.google.com/presentation/d/1q8q1xE-c35tvCsRXZay98s2UYWwXpp0cfCljBmMFpco/edit#slide=id.ga456d5535c_2_53', 'https://docs.google.com/presentation/d/1henPIDTpHT3bc1Y26AltItAHJ2C5xCOl/edit#slide=id.p1', 'https://www.dropbox.com/s/5abw3cvxrhpobg4/20220923_DefragmentationTS.pdf?dl=0', 'https://git.embl.de/oezdemir/course_scripts#ome-zarr-course', 'https://docs.google.com/presentation/d/1oJoy9gHmUuSmUwCkPs_InJf_WZAzmLlUNvK1FUEB4PA/edit#slide=id.ge3a24e733b_0_54', ['https://f1000research.com/slides/11-1171', 'https://doi.org/10.7490/f1000research.1119154.1'], 'https://docs.google.com/presentation/d/1WG_4307XmKsGfWT3taxMvX2rZiG1k0SM1E7SAENJQkI/edit#slide=id.p', 'https://github.com/RoccoDAnt/Defragmentation_TrainingSchool_EOSC-Life_2022/blob/main/Slides/Adding_a_workflow_to_BIAFLOWS.pdf', 'https://analyticalscience.wiley.com/do/10.1002/was.00050003/full/bioimagedataanalysis.pdf', 'https://github.com/hereariim/DEEP-NAPARI', 'https://kevinyamauchi.github.io/open-image-data/intro.html', 'https://github.com/BiAPoL/Bio-image_Analysis_with_Python', ['https://zenodo.org/records/10083555', 'https://doi.org/10.5281/zenodo.10083555'], ['https://zenodo.org/records/10815329', 'https://doi.org/10.5281/zenodo.10815329'], 'https://github.com/BiAPoL/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/', 'https://github.com/ScaDS/BIDS-lecture-2024', 'https://github.com/computational-cell-analytics/dl-for-micro', 'https://zoccoler.github.io/QM_Course_Bio_Image_Analysis_with_napari_2024', 'https://bethac07.github.io/qi_2024_analysis_lab_manual/intro.html', ['https://zenodo.org/records/10679054', 'https://doi.org/10.5281/zenodo.10679054'], ['https://zenodo.org/records/10654775', 'https://doi.org/10.5281/zenodo.10654775'], ['https://zenodo.org/records/10008465', 'https://doi.org/10.5281/zenodo.10008465'], 'https://m-albert.github.io/elastix_tutorial/intro.html', 'https://thejacksonlaboratory.github.io/intro-napari-slides/#/section', 'https://github.com/gletort/NeubiasPasteur2023_AdvancedCellPose', 'https://www.ebi.ac.uk/training/materials/microscopy-data-analysis-machine-learning-and-the-bioimage-archive-materials/', ['https://zenodo.org/records/10972692', 'https://doi.org/10.5281/zenodo.10972692'], ['https://zenodo.org/records/10990107', 'https://doi.org/10.5281/zenodo.10990107'], ['https://zenodo.org/records/10970869', 'https://doi.org/10.5281/zenodo.10970869'], ['https://zenodo.org/records/11066250', 'https://doi.org/10.5281/zenodo.11066250'], 'https://f1000research.com/slides/13-147', 'https://www.ebi.ac.uk/bioimage-archive/galleries/AI.html', 'https://www.ebi.ac.uk/bioimage-archive/galleries/visualisation.html', 'https://www.ebi.ac.uk/bioimage-archive/galleries/vem.html', 'https://github.com/rmassei/2024-jn-omero-pipeline', 'https://drive.google.com/file/d/1pPVUUMi5w2Ojw_SaBzSQVaXUuIKtQ7Ma/view', ['https://zenodo.org/records/11473803', 'https://doi.org/10.5281/zenodo.11473803'], ['https://zenodo.org/records/11474407', 'https://doi.org/10.5281/zenodo.11474407'], ['https://zenodo.org/records/11548617', 'https://doi.org/10.5281/zenodo.11548617'], 'https://zenodo.org/doi/10.5281/zenodo.10687658', 'https://doi.org/10.5281/zenodo.8434325', 'https://zenodo.org/doi/10.5281/zenodo.8349562', 'https://zenodo.org/doi/10.5281/zenodo.8340247', 'https://zenodo.org/doi/10.5281/zenodo.8139353', 'https://zenodo.org/doi/10.5281/zenodo.10886749', 'https://zenodo.org/doi/10.5281/zenodo.10793699', ['https://zenodo.org/records/11265038', 'https://doi.org/10.5281/zenodo.11265038'], ['https://zenodo.org/records/11107798', 'https://doi.org/10.5281/zenodo.11107798'], ['https://zenodo.org/records/10942559', 'https://doi.org/10.5281/zenodo.10942559'], ['https://zenodo.org/records/10816895', 'https://doi.org/10.5281/zenodo.10816895'], ['https://zenodo.org/records/4461261', 'https://doi.org/10.5281/zenodo.4461261'], ['https://zenodo.org/records/11472148', 'https://doi.org/10.5281/zenodo.11472148'], ['https://zenodo.org/records/11396199', 'https://doi.org/10.5281/zenodo.11396199'], ['https://zenodo.org/records/4748510', 'https://doi.org/10.5281/zenodo.4748510'], ['https://zenodo.org/records/4630788', 'https://doi.org/10.5281/zenodo.4630788'], ['https://zenodo.org/records/4748534', 'https://doi.org/10.5281/zenodo.4748534'], ['https://zenodo.org/records/3778431', 'https://doi.org/10.5281/zenodo.3778431'], 'https://zenodo.org/records/12623730', 'https://github.com/ScaDS/BIDS-lecture-2024/', 'https://zenodo.org/records/11201216', 'https://zenodo.org/records/12744715', 'https://archiv.ub.uni-heidelberg.de/volltextserver/29489/', 'https://f1000research.com/articles/11-638', 'https://f1000research.com/slides/10-519', 'https://focalplane.biologists.com/2023/05/06/if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work/', 'https://focalplane.biologists.com/2023/02/15/sharing-research-data-with-zenodo/', 'https://focalplane.biologists.com/2021/09/04/collaborative-bio-image-analysis-script-editing-with-git/', 'https://analyticalscience.wiley.com/do/10.1002/was.0004000267/', 'https://doi.org/10.1515/hsz-2022-0304', 'https://doi.org/10.52825/cordi.v1i.285', ['https://zenodo.org/record/7018929', 'https://doi.org/10.5281/zenodo.7018929'], 'https://analyticalscience.wiley.com/do/10.1002/was.0004000112/', 'https://doi.org/10.1038/s41592-019-0650-1', 'https://febs.onlinelibrary.wiley.com/doi/full/10.1002/1873-3468.14451', 'https://onlinelibrary.wiley.com/doi/full/10.1111/jmi.13192', 'https://doi.org/10.1038/s41592-022-01602-3', 'https://f1000research.com/articles/10-302', 'https://www.frontiersin.org/articles/10.3389/fbinf.2022.889755/full', 'https://f1000research.com/articles/10-334/v1', 'https://www.nature.com/articles/s41592-021-01288-z', 'https://onlinelibrary.wiley.com/doi/10.1111/jmi.13041', 'https://www.nature.com/articles/s41592-021-01326-w', 'https://ceur-ws.org/Vol-3415/paper-27.pdf', 'https://www.biorxiv.org/content/10.1101/2023.05.05.539647v1.abstract', 'https://www.nature.com/articles/s41592-023-01987-9', 'https://doi.org/10.1093/bioinformatics/btab106', 'https://f1000research.com/posters/9-162', ['https://zenodo.org/records/8329306', 'https://doi.org/10.5281/zenodo.8329306'], 'https://f1000research.com/slides/12-1054', ['https://github.com/NFDI4BIOIMAGE/Cologne-Hackathon-2023', 'https://doi.org/10.5281/zenodo.10609770'], 'https://zenodo.org/doi/10.5281/zenodo.10008464', 'https://doi.org/10.5281/zenodo.8070038', 'https://zenodo.org/doi/10.5281/zenodo.10609770', 'https://zenodo.org/doi/10.5281/zenodo.11235512', 'https://zenodo.org/doi/10.5281/zenodo.8329305', 'https://zenodo.org/doi/10.5281/zenodo.7928332', 'https://zenodo.org/doi/10.5281/zenodo.8414318', 'https://zenodo.org/doi/10.5281/zenodo.10730423', 'https://zenodo.org/doi/10.5281/zenodo.10939519', 'https://zenodo.org/doi/10.5281/zenodo.11031746', ['https://zenodo.org/records/13168693', 'https://doi.org/10.5281/zenodo.13168693'], ['https://zenodo.org/records/11318151', 'https://doi.org/10.5281/zenodo.11318151'], ['https://zenodo.org/records/11521029', 'https://doi.org/10.5281/zenodo.11521029'], 'https://www.nature.com/articles/s41592-021-01290-5', 'https://doi.org/10.1038/s41592-021-01315-z', 'https://rupress.org/jcb/article/189/5/777/35828/Metadata-matters-access-to-image-data-in-the-real', 'https://www.nature.com/articles/s41592-018-0195-8', ['https://doi.org/10.1017/s2633903x21000039', 'https://github.com/ciminilab/2021_Jamali_BiologicalImaging'], ['https://www.biorxiv.org/content/10.1101/2023.06.05.543701v1', 'https://github.com/COBA-NIH/2023_ImageAnalysisSurvey'], 'https://arxiv.org/abs/2302.01790v3', 'https://link.springer.com/article/10.1007/s00418-023-02203-7', 'https://figshare.com/articles/poster/I2K_Poster_Haase_V6_ImageJ_repro_pdf/7409525', 'https://www.biorxiv.org/content/10.1101/2023.10.09.561479v1', ['https://academic.oup.com/plcell/article/34/7/e1/6628764', 'https://plantsandpython.github.io/PlantsAndPython/00_Opening_page.html'], ['https://onlinelibrary.wiley.com/doi/10.1111/jmi.13317'], 'https://doi.org/10.1016/j.vibspec.2024.103685', ['https://www.nature.com/articles/s41592-022-01744-4', 'https://jipipe.hki-jena.de/'], ['https://openresearchsoftware.metajnl.com/articles/10.5334/jors.103', 'https://mitobo.informatik.uni-halle.de/'], 'https://biapy.readthedocs.io/', 'https://usegalaxy.org/', 'https://fractal-analytics-platform.github.io/', ['https://snakemake.readthedocs.io/en/stable/', 'https://academic.oup.com/bioinformatics/article/28/19/2520/290322'], ['https://doi.org/10.1111/jmi.13227', 'https://mianalysis.github.io/'], 'https://www.nextflow.io/', 'https://nf-co.re/', 'https://bioimage-io.github.io/bioengine/#/', 'https://www.youtube.com/neubias', ['https://www.youtube.com/digitalsreeni', 'https://www.youtube.com/watch?v=A4po9z61TME'], 'https://www.youtube.com/GlobalBioImaging', 'https://www.youtube.com/watch?v=e-2DbkUwKk4&list=PL5ESQNfM5lc7SAMstEu082ivW4BDMvd0U', 'https://www.youtube.com/OpenMicroscopyEnvironment', 'https://www.youtube.com/c/eurobioimagingcommunication', 'https://www.ibiology.org/online-biology-courses/bioimage-analysis-course/', 'https://www.youtube.com/@cobacenterforopenbioimagea1864', 'https://www.youtube.com/@melbourneadvancedmicroscop2617', 'https://www.youtube.com/playlist?list=PL5Edc1v41fyCLFZbBCLo41zFO-_cXBfAb', 'https://imaris.oxinst.com/tutorials', 'https://www.youtube.com/playlist?list=PLRc9dt9lEZh_SVRC4G5moOvgHuvjPwmv0', 'https://www.youtube.com/watch?v=aRHNHk07t3Q&list=PLyCNTVs-UBvuJF7WausQ5q7v7pI1vEpI1', 'https://www.youtube.com/watch?v=OH3VKI7ErAE', ['https://www.youtube.com/@chrishavlin', 'https://www.youtube.com/playlist?list=PLqbhAmYZU5KxuAcnNBIxyBkivUEiKswq1'], ['https://www.youtube.com/@RDM4mic'], ['https://www.youtube.com/watch?v=8zd4KTy-oYI&list=PLW-oxncaXRqU4XqduJzwFHvWLF06PvdVm'], ['https://zenodo.org/records/13379394', 'https://doi.org/10.5281/zenodo.13379394'], ['https://zenodo.org/records/13371196', 'https://doi.org/10.5281/zenodo.13371196'], ['https://zenodo.org/records/13506641', 'https://doi.org/10.5281/zenodo.13506641'], 'https://hal.science/hal-02267597/document', 'https://www.biorxiv.org/content/10.1101/2024.08.20.608021v1', 'https://biapol.github.io/omero-tools/', 'https://github.com/juglab/EmbedSeg', 'https://github.com/scipy/scipy-cookbook', 'https://github.com/InsightSoftwareConsortium/SimpleITK-Notebooks', 'https://github.com/scikit-image/skimage-tutorials', 'https://github.com/INRIA/scikit-learn-mooc', 'https://github.com/yandexdataschool/nlp_course', 'https://quarep-limi.github.io/WG12_checklists_for_image_publishing/intro.html', 'https://www.ebi.ac.uk/training/on-demand?facets=type:Course%20materials&query=', ['https://zenodo.org/records/11109616', 'https://doi.org/10.5281/zenodo.11109616'], ['https://zenodo.org/records/13908480', 'https://doi.org/10.5281/zenodo.13908480'], ['https://zenodo.org/records/13908108', 'https://doi.org/10.5281/zenodo.13908108'], 'https://www.ugent.be/en/research/openscience/datamanagement', 'https://www.go-fair.org/resources/rdm-starter-kit/', ['https://f1000research.com/slides/11-746', 'https://doi.org/10.7490/f1000research.1119026.1'], 'https://www.mdpi.com/2304-6775/5/1/2', ['https://www.nature.com/articles/sdata201618', 'https://doi.org/10.1038/sdata.2016.18'], ['https://pubmed.ncbi.nlm.nih.gov/36371407/', 'https://www.nature.com/articles/s41597-022-01815-3'], 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9271325/', ['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015/', 'https://www.nature.com/articles/s41592-021-01166-8', 'https://doi.org/10.1038/s41592-021-01166-8'], 'https://doi.org/10.12688/f1000research.12344.2', 'https://doi.org/10.11588/heidok.00029489', 'https://rdmkit.elixir-europe.org/data_life_cycle', 'https://tess.elixir-europe.org/search?q=data+life+cycle#materials', ['https://bausteine-fdm.de/article/view/7945', 'https://doi.org/10.17192/bfdm.2018.1.7945'], 'https://www.o-bib.de/bib/article/view/2018H2S1-13', ['https://discovery.ucl.ac.uk/id/eprint/1546596/1/26_Learn_Guidance_137-140.pdf', 'https://doi.org/10.14324/000.learn.27'], 'https://ds-wizard.org/', 'https://rdmorganiser.github.io/', 'https://focalplane.biologists.com/2023/11/06/creating-a-research-data-management-plan-using-chatgpt/', 'https://hup.sub.uni-hamburg.de/oa-pub/catalog/book/205', 'https://omero-guides.readthedocs.io/en/latest/fiji/docs/index.html', ['https://gerbi-gmb.de/i3dbio/i3dbio-rdm/i3dbio-bioimage-metadata/'], 'https://guide-kvpairs-scripts.readthedocs.io/en/latest/', ['https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-5-r47', 'https://doi.org/10.1186/gb-2005-6-5-r47'], 'https://www.frontiersin.org/journals/neuroscience/articles/10.3389/fnins.2022.871228/full', 'https://www.ebi.ac.uk/bioimage-archive/rembi-help-overview/', 'https://www.nature.com/articles/s41597-023-02367-w', ['https://www.fdm.tu-clausthal.de/fileadmin/FDM/documents/Manual_eLab_v0.3_20200323.pdf', 'https://www.elabftw.net/'], 'http://opendatahandbook.org/guide/en/what-is-open-data/', 'https://web.library.uq.edu.au/blog/2022/03/five-great-reasons-share-your-research-data', 'https://www.publisso.de/en/advice/publishing-advice-faqs/research-data', 'https://gerbi-gmb.de/2023/06/01/finding-and-choosing-a-repository/', 'https://www.re3data.org/', 'https://www.go-fair.org/fair-principles/', 'https://www.ebi.ac.uk/bioimage-archive/submit/', ['https://www.sciencedirect.com/science/article/pii/S0022283622000791?via%3Dihub', 'https://doi.org/10.1016/j.jmb.2022.167505'], 'https://www.nfdi.de/', 'https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007854', 'https://www.nature.com/articles/s41431-018-0160-0', 'https://omero-fbi.fr/slides/elmi23_cfd/main.html#/title-slide', 'https://nfdi4plants.org/nfdi4plants.knowledgebase/index.html', 'https://www.glencoesoftware.com/products/ngff-converter/', 'https://github.com/glencoesoftware/bioformats2raw', 'https://github.com/glencoesoftware/raw2ometiff', 'https://www.nature.com/collections/djiciihhjh', 'https://forum.image.sc/t/checklists-for-publishing-images-and-image-analysis/86304', 'https://workflowhub.eu/', 'https://galaxyproject.org/learn/advanced-workflow/', 'https://galaxy-au-training.github.io/tutorials/modules/workflows/', 'https://training.galaxyproject.org/', 'https://www.knime.com/community/image-processing', 'https://base4nfdi.de/images/AbstractDMP4NFDI.pdf', 'https://www.youtube.com/@OpenMicroscopyEnvironment', 'https://www.youtube.com/watch?v=GmhyDNm1RsM', 'https://github.com/GReD-Clermont/simple-omero-client', 'https://github.com/GReD-Clermont/omero_batch-plugin', 'https://doi.org/10.1016/j.patter.2024.101024', 'https://github.com/NL-BioImaging/biomero', 'https://github.com/ome/omero-cli-transfer', ['https://ome.github.io/ome-ngff-validator/', 'https://github.com/ome/ome-ngff-validator'], 'https://bio-formats.readthedocs.io/en/v8.0.0/users/comlinetools/index.html', ['https://zenodo.org/records/13928832', 'https://doi.org/10.5281/zenodo.13928832'], ['https://zenodo.org/records/13331351', 'https://doi.org/10.5281/zenodo.13331351'], 'https://wiki-biop.epfl.ch/en/data-management/omero/qupath', ['https://www.nature.com/articles/s41592-024-02334-2', 'https://arxiv.org/abs/2407.19778'], ['https://zenodo.org/records/14001044', 'https://doi.org/10.5281/zenodo.14001044'], 'https://github.com/qupath/i2k-qupath-for-python-programmers', ['https://saalfeldlab.github.io/i2k2024-lazy-workshop/', 'https://github.com/saalfeldlab/i2k2024-lazy-workshop'], ['https://github.com/royerlab/ultrack-i2k2024', 'https://royerlab.github.io/ultrack-i2k2024/'], ['https://github.com/BIIFSweden/I2K2024-MTIWorkshop', 'https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1'], 'https://github.com/haesleinhuepf/i2k2024-ai-code-generation', 'https://github.com/CellMigrationLab/I2K_2024', 'https://github.com/StRigaud/clesperanto_workshop_I2K24?tab=readme-ov-file', ['https://timmonko.github.io/napari-ndev/tutorial/01_example_pipeline/', 'https://github.com/timmonko/napari-ndev'], ['https://zenodo.org/records/14001388', 'https://doi.org/10.5281/zenodo.14001388'], ['https://zenodo.org/records/13831274', 'https://doi.org/10.5281/zenodo.13831274'], ['https://zenodo.org/records/13991322', 'https://doi.org/10.5281/zenodo.13991322'], ['https://zenodo.org/records/14014252', 'https://doi.org/10.5281/zenodo.14014252'], 'https://www.glencoesoftware.com/media/webinars/', 'https://f1000research.com/articles/11-638/v2', ['https://github.com/m-albert/Virtual-I2K-2024-multiview-stitcher'], ['https://zenodo.org/records/14032908', 'https://doi.org/10.5281/zenodo.14032908'], ['https://zenodo.org/records/14030307', 'https://doi.org/10.5281/zenodo.14030307'], 'https://github.com/HelmholtzAI-Consultants-Munich/Prompt-Engineering-LLMs-Course', ['https://zenodo.org/records/14044640', 'https://doi.org/10.5281/zenodo.14044640'], ['https://github.com/vibbits/introduction-to-generative-ai', 'https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/introduction-to-generative-ai/refs/heads/main/README.md'], ['https://github.com/vibbits/nextflow-workshop', 'https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/nextflow-workshop/main/README.md#1'], 'https://forum.image.sc/t/upcoming-image-analysis-events/60018/67', 'https://github.com/rmassei/ymia_python_event_series_material', 'https://github.com/SaibotMagd/RDM_system_connector', 'https://www.ebi.ac.uk/training/online/courses/finding-using-public-data/', ['https://zenodo.org/records/14013026', 'https://doi.org/10.5281/zenodo.14013026']]\""
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = load_dataframe(\"../resources/\")\n",
    "\n",
    "all_urls = str(df[\"url\"].tolist())\n",
    "all_urls"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5183ae2c-5144-4bff-a08f-d7a81f89ccef",
   "metadata": {},
   "source": [
    "## Identifying entries we are missing yet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "4d91a514-be3f-4629-85fa-fda961ebe275",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://zenodo.org/records/14014252\n",
      "https://zenodo.org/api/records/14014252\n",
      "Yes\n",
      "Yes\n",
      "https://zenodo.org/records/14013026\n",
      "https://zenodo.org/api/records/14013026\n",
      "Yes\n",
      "Yes\n",
      "https://zenodo.org/records/14001388\n",
      "https://zenodo.org/api/records/14001388\n",
      "Yes\n",
      "Yes\n",
      "https://zenodo.org/records/13991322\n",
      "https://zenodo.org/api/records/13991322\n",
      "Yes\n",
      "Yes\n",
      "https://zenodo.org/records/13837146\n",
      "https://zenodo.org/api/records/13837146\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/13831274\n",
      "https://zenodo.org/api/records/13831274\n",
      "Yes\n",
      "Yes\n",
      "https://zenodo.org/records/11503289\n",
      "https://zenodo.org/api/records/11503289\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/13684187\n",
      "https://zenodo.org/api/records/13684187\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/13640979\n",
      "https://zenodo.org/api/records/13640979\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/13380289\n",
      "https://zenodo.org/api/records/13380289\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/13168693\n",
      "https://zenodo.org/api/records/13168693\n",
      "Yes\n",
      "Yes\n",
      "https://zenodo.org/records/12699637\n",
      "https://zenodo.org/api/records/12699637\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/11548617\n",
      "https://zenodo.org/api/records/11548617\n",
      "Yes\n",
      "Yes\n",
      "https://zenodo.org/records/11501662\n",
      "https://zenodo.org/api/records/11501662\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/11350689\n",
      "https://zenodo.org/api/records/11350689\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/11235513\n",
      "https://zenodo.org/api/records/11235513\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/11146807\n",
      "https://zenodo.org/api/records/11146807\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/11109616\n",
      "https://zenodo.org/api/records/11109616\n",
      "Yes\n",
      "Yes\n",
      "https://zenodo.org/records/11031747\n",
      "https://zenodo.org/api/records/11031747\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/10939520\n",
      "https://zenodo.org/api/records/10939520\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/10886750\n",
      "https://zenodo.org/api/records/10886750\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/10808486\n",
      "https://zenodo.org/api/records/10808486\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/10793700\n",
      "https://zenodo.org/api/records/10793700\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/10730424\n",
      "https://zenodo.org/api/records/10730424\n",
      "No\n",
      "No\n",
      "https://zenodo.org/records/10687659\n",
      "https://zenodo.org/api/records/10687659\n",
      "No\n",
      "No\n"
     ]
    }
   ],
   "source": [
    "new_data = []\n",
    "for url in urls:\n",
    "    print(url)\n",
    "    data = complete_zenodo_data(url)\n",
    "    \n",
    "    if isinstance(data[\"url\"], str):\n",
    "        data[\"url\"] = [data[\"url\"]]\n",
    "\n",
    "    not_in_data_yet = True\n",
    "    for u in data[\"url\"]:\n",
    "        if u in all_urls:\n",
    "            print(\"Yes\")\n",
    "            not_in_data_yet = False\n",
    "        else:\n",
    "            print(\"No\")     \n",
    "\n",
    "    if not_in_data_yet:\n",
    "        new_data.append(data)\n",
    "\n",
    "#new_data"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "11fe5ad3-b68f-46cb-b9d6-3c9e6875cb64",
   "metadata": {},
   "source": [
    "## Saving new entries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "db54051a-0433-4710-a5b4-6f6c9ebe80aa",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- authors: \"J\\xFCngst, Christian, Zentis, Peter\"\n",
      "    description: Raw microscopy image from the NFDI4Bioimage calendar October 2024\n",
      "    license: cc-by-4.0\n",
      "    name: NFDI4Bioimage Calendar 2024 October; original image\n",
      "    num_downloads: 8\n",
      "    publication_date: '2024-09-25'\n",
      "    url:\n",
      "    - https://zenodo.org/records/13837146\n",
      "    - https://doi.org/10.5281/zenodo.13837146\n",
      "  - authors: Dvoretskii, Stefan\n",
      "    license: cc-by-4.0\n",
      "    name: Insights from Acquiring Open Medical Imaging  Datasets for Foundation Model\n",
      "      Development\n",
      "    num_downloads: 41\n",
      "    publication_date: '2024-04-10'\n",
      "    url:\n",
      "    - https://zenodo.org/records/11503289\n",
      "    - https://doi.org/10.5281/zenodo.11503289\n",
      "  - authors: Wetzker, Cornelia, Schlierf, Michael\n",
      "    description: The poster is part of the work of the German consortium NFDI4BIOIMAGE\n",
      "      funded by the Deutsche Forschungsgemeinschaft (DFG grant number NFDI 46/1, project\n",
      "      number 501864659).\n",
      "    license: cc-by-4.0\n",
      "    name: RESEARCH DATA MANAGEMENT on Campus and in NFDI4BIOIMAGE\n",
      "    num_downloads: 26\n",
      "    publication_date: '2024-08-29'\n",
      "    url:\n",
      "    - https://zenodo.org/records/13684187\n",
      "    - https://doi.org/10.5281/zenodo.13684187\n",
      "  - authors: Haase, Robert\n",
      "    description: This talk will present the initiatives of the NFDI4BioImage consortium\n",
      "      aimed at the long-term preservation of life science data. We will discuss our\n",
      "      efforts to establish metadata standards, which are crucial for ensuring data reusability\n",
      "      and integrity. The development of sustainable infrastructure is another key focus,\n",
      "      enabling seamless data integration and analysis in the cloud. We will take a look\n",
      "      at how we manage training materials and communicate with our community. Through\n",
      "      these actions, NFDI4BioImage seeks to enable FAIR bioimage data management for\n",
      "      German researchers, across disciplines and embedded in the international framework.\n",
      "    license: cc-by-4.0\n",
      "    name: Towards Preservation of Life Science Data with NFDI4BIOIMAGE\n",
      "    num_downloads: 225\n",
      "    publication_date: '2024-09-03'\n",
      "    url:\n",
      "    - https://zenodo.org/records/13640979\n",
      "    - https://doi.org/10.5281/zenodo.13640979\n",
      "  - authors: Dvoretskii, Stefan\n",
      "    license: cc-by-4.0\n",
      "    name: Insights from Acquiring Open Medical Imaging Datasets for Foundation Model\n",
      "      Development\n",
      "    num_downloads: 46\n",
      "    publication_date: '2024-04-10'\n",
      "    url:\n",
      "    - https://zenodo.org/records/13380289\n",
      "    - https://doi.org/10.5281/zenodo.13380289\n",
      "  - authors: Habinger, Sophie, Heber, Maximilian, Kralj, Sonja, Mikautsch, Emilia\n",
      "    description: 'The rise of Open Science (OS) and the academic community&rsquo;s needs\n",
      "      that come with it bring about a range of challenges for academic libraries. To\n",
      "      face these challenges, the University of Konstanz has created a competence unit\n",
      "      called&nbsp;Team Open Science&nbsp;in the Communication, Information, Media Center\n",
      "      (KIM) - a joint unit of library and IT infrastructure. The Team creates synergies\n",
      "      within itself and across the library. In December 2023, it involved 12 staff members\n",
      "      specialising in open access (OA), research data management (RDM), open educational\n",
      "      resources (OER) and virtual research environments (VRE). It collaborates closely\n",
      "      with other KIM departments. This submission shall serve as a best practice example\n",
      "      for the impact of OS on research libraries and, beyond that, the impact of research\n",
      "      libraries on universities.\n",
      "  \n",
      "      To enhance and foster OS, the Team provides individual consultations, services\n",
      "      and office hours for researchers. Here, it collaborates closely with other librarians\n",
      "      like subject specialists and the Team University Publications. Along similar lines,\n",
      "      the KIM offers institutional repositories for publications (KOPS) and research\n",
      "      data (KonDATA). Beyond that, the Team provides solutions to host OA journals and\n",
      "      analyses researchers&rsquo; VRE needs to decide on implementation options. In\n",
      "      sum, the Team is the central OS contact point for the entire university, underlining\n",
      "      the major role the library holds in making institutional impact.\n",
      "  \n",
      "      Furthermore, the Team had the leading role in creating the University of Konstanz&rsquo;\n",
      "      OS Policy, one of the first ones passed by a German university. This policy stands\n",
      "      out because it encompasses various OS domains. It demands, among other things,\n",
      "      that text publications be made OA and that research data be managed according\n",
      "      to relevant subject-specific standards. If permissible and reasonable, it demands\n",
      "      that research data should be made publicly available at the earliest possible\n",
      "      time. Along these lines, the policy has a large impact on how the library handles\n",
      "      closed access books and subscription-based journals. As a consequence, OA is pursued\n",
      "      wherever possible, leading to the highest OA quota of all German universities.\n",
      "      In that sense, the Team is a crucial driving force of OS in the University of\n",
      "      Konstanz, which ties in with the library&rsquo;s major role of open research transformation.\n",
      "  \n",
      "      Beyond the University of Konstanz, the Team is involved in a range of national\n",
      "      and international projects collaborating with other libraries. On a national level,\n",
      "      they lead the project open.access-network which provides an information platform\n",
      "      for researchers and librarians and connects the German-speaking OA community through\n",
      "      events like bar camps. The project KOALA-AV supports libraries in establishing\n",
      "      consortial solutions for financing Diamond OA publications. Moreover, the Team\n",
      "      is involved in the federal state initiative for RDM in Baden-W&uuml;rttemberg\n",
      "      (bwFDM). Here, the Team is in charge of forschungsdaten.info, the German-speaking\n",
      "      countries&rsquo; leading RDM information platform, which will be offered in English\n",
      "      within the next years. Internationally, the Team cooperates with librarians and\n",
      "      other OS professionals from the European Reform University Alliance (ERUA) and\n",
      "      the European University for Well-Being (EUniWell), establishing formats for best\n",
      "      practice exchange, such as monthly OS Meet-Ups.'\n",
      "    license: cc-by-4.0\n",
      "    name: 'Institutionalization and Collaboration as a Way of Addressing the Challenges\n",
      "      Open Science Presents to Libraries: The University of Konstanz as a National Pioneer'\n",
      "    num_downloads: 119\n",
      "    publication_date: '2024-07-09'\n",
      "    url:\n",
      "    - https://zenodo.org/records/12699637\n",
      "    - https://doi.org/10.5281/zenodo.12699637\n",
      "  - authors: Massei, Riccardo, Schmidt, Christian, Bortolomeazzi, Michele, Thoennissen,\n",
      "      Julia, Bumberger, Jan, Dickscheid, Timo, Mallm, Jan-Philipp, Ferrando-May, Elisa\n",
      "    description: Germany&rsquo;s National Research Data Infrastructure (NFDI) aims to\n",
      "      establish a sustained, cross-disciplinary research data management (RDM) infrastructure\n",
      "      that enables researchers to handle FAIR (findable, accessible, interoperable,\n",
      "      reusable) data. While&nbsp;FAIR principles have been&nbsp;adopted by funders,\n",
      "      policymakers, and publishers, their practical implementation remains an ongoing\n",
      "      effort. In the field of bio-imaging, harmonization of&nbsp;data formats, metadata\n",
      "      ontologies, and open data repositories is necessary&nbsp;to achieve FAIR data.&nbsp;The\n",
      "      NFDI4BIOIMAGE was established&nbsp;to address these issues and&nbsp;develop tools\n",
      "      and best practices to facilitate FAIR microscopy and image analysis data in alignment\n",
      "      with international community activities. The&nbsp;consortium operates through\n",
      "      its Data Stewards team to provide expertise and direct support to help overcome\n",
      "      RDM challenges. The three Helmholtz Centers in NFDI4BIOIMAGE aim to collaborate\n",
      "      closely with other centers and initiatives, such as HMC, Helmholtz AI, and HIP.\n",
      "      Here we present NFDI4BIOIMAGE&rsquo;s work and its significance for research in\n",
      "      Helmholtz and beyond\n",
      "    license: cc-by-4.0\n",
      "    name: The role of Helmholtz Centers in NFDI4BIOIMAGE - A national consortium enhancing\n",
      "      FAIR data management for microscopy and bioimage analysis\n",
      "    num_downloads: 47\n",
      "    publication_date: '2024-06-06'\n",
      "    url:\n",
      "    - https://zenodo.org/records/11501662\n",
      "    - https://doi.org/10.5281/zenodo.11501662\n",
      "  - authors: Massei, Riccardo, Bortolomeazzi, Michele, Schmidt, Christian\n",
      "    description: 'Here we share the material used in a workshop held on May 13th, 2024,\n",
      "      at the German Cancer Research Center in Heidelberg (on-premise)\n",
      "  \n",
      "      Description:Microscopy experiments generate information-rich, multi-dimensional\n",
      "      data, allowing us to investigate biological processes at high spatial and temporal\n",
      "      resolution. Image processing and analysis is a standard procedure to retrieve\n",
      "      quantitative information from biological imaging. Due to the complex nature of\n",
      "      bioimaging files that often come in proprietary formats, it can be challenging\n",
      "      to organize, structure, and annotate bioimaging data throughout a project. Data\n",
      "      often needs to be moved between collaboration partners, transformed into open\n",
      "      formats, processed with a variety of software tools, and exported to smaller-sized\n",
      "      images for presentation. The path from image acquisition to final publication\n",
      "      figures with quantitative results must be documented and reproducible.\n",
      "  \n",
      "      In this workshop, participants learn how to use OMERO to organize their data and\n",
      "      enrich the bioimage data with structured metadata annotations.We also focus on\n",
      "      image analysis workflows in combination with OMERO based on the Fiji/ImageJ software\n",
      "      and using Jupyter Notebooks. In the last part, we explore how OMERO can be used\n",
      "      to create publication figures and prepare bioimage data for publication in a suitable\n",
      "      repository such as the Bioimage Archive.\n",
      "  \n",
      "      Module 1&nbsp;(9 am - 10.15 am):&nbsp;Basics of OMERO, data structuring and annotation\n",
      "  \n",
      "      Module 2&nbsp;(10.45 am - 12.45 pm):&nbsp;OMERO and Fiji\n",
      "  \n",
      "      Module 3&nbsp;(1.45 pm - 3.45 pm):&nbsp;OMERO and Jupyter Notebooks\n",
      "  \n",
      "      Module 4&nbsp;(4.15 pm - 6. pm):&nbsp;Publication-ready figures and data with\n",
      "      OMERO\n",
      "  \n",
      "      The target group for this workshopThis workshop is directed at researchers at\n",
      "      all career levels who plan to or have started to use OMERO for their microscopy\n",
      "      research data management.&nbsp;We encourage the workshop participants to bring\n",
      "      example data from their research to discuss suitable metadata annotation for their\n",
      "      everyday practice.\n",
      "  \n",
      "      Prerequisites:Users should bring their laptops and have access to the internet\n",
      "      through one of the following options:- eduroam- institutional WiFi- VPN connection\n",
      "      to their institutional networks to access OMERO\n",
      "  \n",
      "      Who are the trainers?\n",
      "  \n",
      "      Dr. Riccardo Massei (Helmholtz-Center for Environmental Research, UFZ, Leipzig)\n",
      "      - Data Steward for Bioimaging Data in NFDI4BIOIMAGE\n",
      "  \n",
      "      Dr. Michele Bortolomeazzi (DKFZ, Single cell Open Lab, bioimage data specialist,\n",
      "      bioinformatician, staff scientist in the NFDI4BIOIMAGE project)\n",
      "  \n",
      "      Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging,\n",
      "      German Cancer Research Center, Heidelberg, Project Coordinator of the NFDI4BIOIMAGE\n",
      "      project)'\n",
      "    license: cc-by-4.0\n",
      "    name: '[Workshop] Bioimage data management and analysis with OMERO'\n",
      "    num_downloads: 208\n",
      "    publication_date: '2024-05-13'\n",
      "    url:\n",
      "    - https://zenodo.org/records/11350689\n",
      "    - https://doi.org/10.5281/zenodo.11350689\n",
      "  - authors: Moore, Josh, Kunis, Susanne\n",
      "    description: Poster presented at the European Light Microscopy Initiative meeting\n",
      "      in Liverpool (https://www.elmi2024.org/)\n",
      "    license: cc-by-4.0\n",
      "    name: '[ELMI 2024]  AI''s Dirty Little Secret: Without\n",
      "  \n",
      "      FAIR Data, It''s Just Fancy Math'\n",
      "    num_downloads: 53\n",
      "    publication_date: '2024-05-21'\n",
      "    url:\n",
      "    - https://zenodo.org/records/11235513\n",
      "    - https://doi.org/10.5281/zenodo.11235513\n",
      "  - authors: Escobar Diaz Guerrero, Rodrigo\n",
      "    description: First updates of LEO (Linking ELN with OMERO)\n",
      "    license: cc-by-4.0\n",
      "    name: 'LEO: Linking ELN with OMERO'\n",
      "    num_downloads: 3\n",
      "    publication_date: '2024-05-08'\n",
      "    url:\n",
      "    - https://zenodo.org/records/11146807\n",
      "    - https://doi.org/10.5281/zenodo.11146807\n",
      "  - authors: Fortmann-Grote, Carsten\n",
      "    description: This presentation was given at the 2nd MPG-NFDI Workshop on April 18th.\n",
      "    license: cc-by-4.0\n",
      "    name: NFDI4BIOIMAGE\n",
      "    num_downloads: 81\n",
      "    publication_date: '2024-04-22'\n",
      "    url:\n",
      "    - https://zenodo.org/records/11031747\n",
      "    - https://doi.org/10.5281/zenodo.11031747\n",
      "  - authors: Schmidt, Christian\n",
      "    description: Short Talk about the NFDI4BIOIMAGE consortium presented at the RDM\n",
      "      in (Bio-)Medicine Information Event on April 10th, 2024, organized C&sup3;RDM\n",
      "      &amp; ZB MED.\n",
      "    license: cc-by-4.0\n",
      "    name: '[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure'\n",
      "    num_downloads: 47\n",
      "    publication_date: '2024-04-10'\n",
      "    url:\n",
      "    - https://zenodo.org/records/10939520\n",
      "    - https://doi.org/10.5281/zenodo.10939520\n",
      "  - authors: Margineanu, Anca, Stringari, Chiara, Zoccoler, Marcelo, Wetzker, Cornelia\n",
      "    description: The presentations introduce open-source software to read in, visualize\n",
      "      and analyse fluorescence lifetime imaging microscopy (FLIM) raw data developed\n",
      "      for life scientists. The slides were presented at German Bioimaging (GerBI) FLIM\n",
      "      Workshop held February 26 to 29 2024 at the Biomedical Center of LMU M&uuml;nchen\n",
      "      by Anca Margineanu, Chiara Stringari and Conni Wetzker.&nbsp;\n",
      "    license: cc-by-4.0\n",
      "    name: A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and\n",
      "      napari-flim-phasor-plotter\n",
      "    num_downloads: 217\n",
      "    publication_date: '2024-03-27'\n",
      "    url:\n",
      "    - https://zenodo.org/records/10886750\n",
      "    - https://doi.org/10.5281/zenodo.10886750\n",
      "  - authors: Fortmann-Grote, Carsten\n",
      "    license: cc-by-4.0\n",
      "    name: Linked (Open) Data for Microbial Population Biology\n",
      "    num_downloads: 121\n",
      "    publication_date: '2024-03-12'\n",
      "    url:\n",
      "    - https://zenodo.org/records/10808486\n",
      "    - https://doi.org/10.5281/zenodo.10808486\n",
      "  - authors: Massei, Riccardo\n",
      "    description: 'Results of the project \"Conversion of KNIME image analysis workflows\n",
      "      to Galaxy\" during the Hackathon \"Image Analysis in Galaxy\" (Freiburg 26 Feb -\n",
      "      01 Mar 2024)\n",
      "  \n",
      "      &nbsp;'\n",
      "    license: cc-by-4.0\n",
      "    name: Hackaton Results - Conversion of KNIME image analysis workflows to Galaxy\n",
      "    num_downloads: 46\n",
      "    publication_date: '2024-03-07'\n",
      "    url:\n",
      "    - https://zenodo.org/records/10793700\n",
      "    - https://doi.org/10.5281/zenodo.10793700\n",
      "  - authors: \"Fuchs, Vanessa Aphaia Fiona, Wendt, Jens, M\\xFCller, Maximilian, Ahmadi,\\\n",
      "      \\ Mohsen, Massei, Riccardo, Wetzker, Cornelia\"\n",
      "    description: The Data Steward Team of the NFDI4BIOIMAGE consortium presents themselves\n",
      "      and the services (including the Helpdesk) that we offer.\n",
      "    license: cc-by-4.0\n",
      "    name: Who you gonna call? - Data Stewards to the rescue\n",
      "    num_downloads: 89\n",
      "    publication_date: '2024-03-01'\n",
      "    url:\n",
      "    - https://zenodo.org/records/10730424\n",
      "    - https://doi.org/10.5281/zenodo.10730424\n",
      "  - authors: Moore, Josh, Waagmeester, Andra, Hettne, Kristina, Wolstencroft, Katherine,\n",
      "      Kunis, Susanne\n",
      "    description: 'In 2005, the first version of OMERO stored RDF natively. However,\n",
      "      just a year after the 1.0 release of RDF, performance considerations led to the\n",
      "      development of a more traditional SQL approach for OMERO. A binary protocol makes\n",
      "      it possible to query and retrieve metadata but the resulting information cannot\n",
      "      immediately be combined with other sources. This is the adventure of rediscovering\n",
      "      the benefit of RDF triples as a -- if not the -- common exchange mechanism.\n",
      "  \n",
      "      &nbsp;\n",
      "  \n",
      "      This poster was presented at SWAT4HCLS in Leiden, NL, 2024 as Poster 54. See https://www.swat4ls.org/workshops/leiden2024\n",
      "      for more information.'\n",
      "    license: cc-by-4.0\n",
      "    name: RDF as a bridge to domain-platforms like OMERO, or There and back again.\n",
      "    num_downloads: 120\n",
      "    publication_date: '2024-02-26'\n",
      "    url:\n",
      "    - https://zenodo.org/records/10687659\n",
      "    - https://doi.org/10.5281/zenodo.10687659\n",
      "  \n"
     ]
    }
   ],
   "source": [
    "import yaml\n",
    "zenodo_yml = yaml.dump(new_data).replace(\"\\n\", \"\\n  \")\n",
    "print(zenodo_yml)\n",
    "\n",
    "with open('../resources/nfdi4bioimage.yml', 'a') as file:\n",
    "    file.write(\"\\n\")\n",
    "    file.write(zenodo_yml)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8fa55de8-b51a-45ce-934c-75c090166dab",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
