
<!DOCTYPE html>


<html lang="en" data-content_root="../" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Christian schmidt (26) &#8212; NFDI4BioImage Training Materials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="../_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="../_static/styles/sphinx-book-theme.css?v=eba8b062" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.8ecb98da25f57f5357bf6f572d296f466b2cfe2517ffebfabe82451661e28f02.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="../_static/css/searchbox.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-design.min.css?v=95c83b7e" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="../_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="../_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="../_static/doctools.js?v=9a2dae69"></script>
    <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="../_static/copybutton.js?v=f281be69"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/lunr.js/0.6.0/lunr.min.js"></script>
    <script src="../_static/js/searchbox.js"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="../_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'authors/christian_schmidt';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
    <meta name="docbuild:last-update" content="2025-11-25"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
        
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="../readme.html">
  
  
  
  
  
    
    
      
    
    
    <img src="../_static/logo.png" class="logo__image only-light" alt="NFDI4BioImage Training Materials - Home"/>
    <script>document.write(`<img src="../_static/logo.png" class="logo__image only-dark" alt="NFDI4BioImage Training Materials - Home"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../readme.html">
                    NFDI4BioImage Training Materials
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">What's new</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../whats_new.html">Recently added (10)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Contributing</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../contributing/index.html">How to contribute</a></li>

<li class="toctree-l1"><a class="reference internal" href="../contributing/submit_app.html">Using the Training Materials Submission App</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contributing/format.html">YML format</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By tag</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../tags/ai-ready.html">Ai-ready (86)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/artificial_intelligence.html">Artificial intelligence (52)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bioimage_analysis.html">Bioimage analysis (214)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bioinformatics.html">Bioinformatics (20)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/data_stewardship.html">Data stewardship (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/exclude_from_dalia.html">Exclude from dalia (460)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/fair-principles.html">Fair-principles (29)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/fiji.html">Fiji (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/galaxy.html">Galaxy (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/imagej.html">Imagej (20)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/include_in_dalia.html">Include in dalia (338)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/licensing.html">Licensing (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/metadata.html">Metadata (15)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/napari.html">Napari (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/neubias.html">Neubias (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/nfdi4bioimage.html">Nfdi4bioimage (80)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/omero.html">Omero (42)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/open_science.html">Open science (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/open_source_software.html">Open source software (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/python.html">Python (71)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/reproducibility.html">Reproducibility (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/research_data_management.html">Research data management (148)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/sharing.html">Sharing (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/workflow.html">Workflow (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/workflow_engine.html">Workflow engine (13)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By content type</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../content_types/blog%20post.html">Blog post (28)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/book.html">Book (21)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/code.html">Code (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/collection.html">Collection (86)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/data.html">Data (94)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/document.html">Document (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/documentation.html">Documentation (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/event.html">Event (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/github%20repository.html">Github repository (70)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/notebook.html">Notebook (57)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/online%20tutorial.html">Online tutorial (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/poster.html">Poster (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/preprint.html">Preprint (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/publication.html">Publication (74)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/slides.html">Slides (85)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/tutorial.html">Tutorial (48)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/video.html">Video (40)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/website.html">Website (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/workshop.html">Workshop (14)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By license</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../licenses/all_rights_reserved.html">All rights reserved (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/apache-2.0.html">Apache-2.0 (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-2-clause.html">Bsd-2-clause (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-3-clause.html">Bsd-3-clause (37)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-3.0.html">Cc-by-3.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-4.0.html">Cc-by-4.0 (432)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-nc-sa-4.0.html">Cc-by-nc-sa-4.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-sa-4.0.html">Cc-by-sa-4.0 (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc0-1.0.html">Cc0-1.0 (25)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-2.0.html">Gpl-2.0 (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-3.0.html">Gpl-3.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/mit.html">Mit (32)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/unknown.html">Unknown (111)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By domain</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../domain/bbbc.broadinstitute.org.html">Bbbc.broadinstitute.org (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/biapol.github.io.html">Biapol.github.io (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/docs.google.com.html">Docs.google.com (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/doi.org.html">Doi.org (367)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/f1000research.com.html">F1000research.com (11)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/focalplane.biologists.com.html">Focalplane.biologists.com (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/git.mpi-cbg.de.html">Git.mpi-cbg.de (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/github.com.html">Github.com (151)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/training.galaxyproject.org.html">Training.galaxyproject.org (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.biorxiv.org.html">Www.biorxiv.org (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.ebi.ac.uk.html">Www.ebi.ac.uk (18)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.nature.com.html">Www.nature.com (21)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.youtube.com.html">Www.youtube.com (29)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/zenodo.org.html">Zenodo.org (348)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Appendix</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../statistics/readme.html">Statistics</a></li>
<li class="toctree-l1"><a class="reference internal" href="../export/readme.html">Open data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../gdpr_compliance.html">GDPR Compliance Documentation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../imprint.html">Imprint</a></li>

</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training/issues/new?title=Issue%20on%20page%20%2Fauthors/christian_schmidt.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/authors/christian_schmidt.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Christian schmidt (26)</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-journey-to-fair-microscopy-data">A journey to FAIR microscopy data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-practical-guide-to-bioimaging-research-data-management-in-core-facilities">A practical guide to bioimaging research data management in core facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#finding-and-choosing-a-data-repository">Finding and Choosing a Data Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-information-infrastructure-for-bioimage-data-bioimage-metadata">I3D bio – Information Infrastructure for BioImage Data - Bioimage Metadata</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introducing-omero-vitessce-an-omero-web-plugin-for-multi-modal-data">Introducing OMERO-vitessce: an OMERO.web plugin for multi-modal data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#key-value-pair-template-for-annotation-of-datasets-in-omero-perikles-study">Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#key-value-pair-template-for-annotation-of-datasets-in-omero-for-light-and-electron-microscopy-data-within-the-research-group-of-prof-muller-reichert">Key-Value pair template for annotation of datasets in OMERO for light- and electron microscopy data within the research group of Prof. Müller-Reichert</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#mdemic-a-metadata-annotation-tool-to-facilitate-management-of-fair-image-data-in-the-bioimaging-community">MDEmic: a metadata annotation tool to facilitate management of FAIR image data in the bioimaging community</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data">NFDI4BIOIMAGE - An Initiative for a National Research Data Infrastructure for Microscopy Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omexcavator-a-tool-for-exporting-and-connecting-bioimaging-specific-metadata-in-wider-knowledge-graphs">OMExcavator: a tool for exporting and connecting  Bioimaging-specific metadata in wider knowledge graphs</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omexcavator-a-tool-for-exporting-and-connecting-domain-specific-metadata-in-a-wider-knowledge-graph">OMExcavator: a tool for exporting and connecting domain-specific metadata in a wider knowledge graph</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey">Research data management for bioimaging - the 2021 NFDI4BIOIMAGE community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#id1">Research data management for bioimaging: the 2021 NFDI4BIOIMAGE community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-an-institutional-omero-environment-for-bioimage-data-perspectives-from-both-facility-staff-and-users">Setting up an institutional OMERO environment for bioimage data: Perspectives from both facility staff and users</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-information-infrastructure-for-bioimage-data-i3d-bio-project-to-advance-fair-microscopy-data-management-for-the-community">The Information Infrastructure for BioImage Data (I3D:bio) project to advance FAIR microscopy data management for the community</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-role-of-helmholtz-centers-in-nfdi4bioimage-a-national-consortium-enhancing-fair-data-management-for-microscopy-and-bioimage-analysis">The role of Helmholtz Centers in NFDI4BIOIMAGE - A national consortium enhancing FAIR data management for microscopy and bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#community-meeting-2024-supporting-and-financing-rdm-projects-within-gerbi">[Community Meeting 2024] Supporting and financing RDM projects within GerBI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi2025-bridging-communities-with-ome-zarr">[ELMI2025] Bridging communities with OME-Zarr</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#webinar-a-journey-to-fair-bioimage-data">[Webinar] A journey to FAIR bioimage data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg">[Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-bioimage-data-management-and-analysis-with-omero">[Workshop] Bioimage data management and analysis with OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero">[Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-managing-fair-microscopy-data-at-scale-for-universities-and-research-institutions-an-introduction-for-non-imaging-stakeholders">[Workshop] Managing FAIR microscopy data at scale for universities and research institutions: an introduction for non-imaging stakeholders</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-research-data-management-for-microscopy-and-bioimage-analysis">[Workshop] Research Data Management for Microscopy and BioImage Analysis</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="christian-schmidt-26">
<h1>Christian schmidt (26)<a class="headerlink" href="#christian-schmidt-26" title="Link to this heading">#</a></h1>
<section id="a-journey-to-fair-microscopy-data">
<h2>A journey to FAIR microscopy data<a class="headerlink" href="#a-journey-to-fair-microscopy-data" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters, Janina Hanne, Christian Schmidt</p>
<p>Published 2023-05-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Oral presentation, 32nd MoMAN “From Molecules to Man” Seminar, Ulm, online. Monday February 6th, 2023</p>
<p>Abstract:</p>
<p>Research data management is essential in nowadays research, and one of the big opportunities to accelerate collaborative and innovative scientific projects. To achieve this goal, all our data needs to be FAIR (findable, accessible, interoperable, reproducible). For data acquired on microscopes, however, a common ground for FAIR data sharing is still to be established. Plenty of work on file formats, data bases, and training needs to be performed to highlight the value of data sharing and exploit its potential for bioimaging data.</p>
<p>In this presentation, Stefanie Weidtkamp-Peters will introduce the challenges for bioimaging data management, and the necessary steps to achieve data FAIRification. German BioImaging - GMB e.V., together with other institutions, contributes to this endeavor. Janina Hanne will present how the network of imaging core facilities, research groups and industry partners is key to the German bioimaging community’s aligned collaboration toward FAIR bioimaging data. These activities have paved the way for two data management initiatives in Germany: I3D:bio (Information Infrastructure for BioImage Data) and NFDI4BIOIMAGE, a consortium of the National Research Data Infrastructure. Christian Schmidt will introduce the goals and measures of these initiatives to the benefit of imaging scientist’s work and everyday practice.  </p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/7890311">https://zenodo.org/records/7890311</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.7890311">https://doi.org/10.5281/zenodo.7890311</a></p>
</section>
<hr class="docutils" />
<section id="a-practical-guide-to-bioimaging-research-data-management-in-core-facilities">
<h2>A practical guide to bioimaging research data management in core facilities<a class="headerlink" href="#a-practical-guide-to-bioimaging-research-data-management-in-core-facilities" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Tom Boissonnet, Julia Dohle, Karen Bernhardt, Elisa Ferrando-May, Tobias Wernet, Roland Nitschke, Susanne Kunis, Stefanie Weidtkamp-Peters</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://onlinelibrary.wiley.com/doi/10.1111/jmi.13317">https://onlinelibrary.wiley.com/doi/10.1111/jmi.13317</a></p>
</section>
<hr class="docutils" />
<section id="finding-and-choosing-a-data-repository">
<h2>Finding and Choosing a Data Repository<a class="headerlink" href="#finding-and-choosing-a-data-repository" title="Link to this heading">#</a></h2>
<p>Christian Schmidt</p>
<p>Published 2023-07-01</p>
<p>Licensed UNKNOWN</p>
<p>Funding agencies may demand that original source data of a publication be published, too. So the question is - where should one publish the data? And how does it get there?</p>
<p>Tags: I3Dbio, Research Data Management, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="reference external" href="https://gerbi-gmb.de/2023/06/01/finding-and-choosing-a-repository/">https://gerbi-gmb.de/2023/06/01/finding-and-choosing-a-repository/</a></p>
</section>
<hr class="docutils" />
<section id="i3d-bio-information-infrastructure-for-bioimage-data-bioimage-metadata">
<h2>I3D bio – Information Infrastructure for BioImage Data - Bioimage Metadata<a class="headerlink" href="#i3d-bio-information-infrastructure-for-bioimage-data-bioimage-metadata" title="Link to this heading">#</a></h2>
<p>Christian Schmidt</p>
<p>Licensed UNKNOWN</p>
<p>A Microscopy Research Data Management Resource.</p>
<p>Tags: Metadata, I3Dbio, Research Data Management, Exclude From Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://gerbi-gmb.de/i3dbio/i3dbio-rdm/i3dbio-bioimage-metadata/">https://gerbi-gmb.de/i3dbio/i3dbio-rdm/i3dbio-bioimage-metadata/</a></p>
</section>
<hr class="docutils" />
<section id="i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">
<h2>I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training<a class="headerlink" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Michele Bortolomeazzi, Tom Boissonnet, Carsten Fortmann-Grote, Julia Dohle, Peter Zentis, Niraj Kandpal, Susanne Kunis, Thomas Zobel, Stefanie Weidtkamp-Peters, Elisa Ferrando-May</p>
<p>Published 2023-11-13</p>
<p>Licensed CC-BY-4.0</p>
<p>The open-source software OME Remote Objects (OMERO) is a data management software that allows storing, organizing, and annotating bioimaging/microscopy data. OMERO has become one of the best-known systems for bioimage data management in the bioimaging community. The Information Infrastructure for BioImage Data (I3D:bio) project facilitates the uptake of OMERO into research data management (RDM) practices at universities and research institutions in Germany. Since the adoption of OMERO into researchers’ daily routines requires intensive training, a broad portfolio of training resources for OMERO is an asset. On top of using the OMERO guides curated by the Open Microscopy Environment Consortium (OME) team, imaging core facility staff at institutions where OMERO is used often prepare additional material tailored to be applicable for their own OMERO instances. Based on experience gathered in the Research Data Management for Microscopy group (RDM4mic) in Germany, and in the use cases in the I3D:bio project, we created a set of reusable, adjustable, openly available slide decks to serve as the basis for tailored training lectures, video tutorials, and self-guided instruction manuals directed at beginners in using OMERO. The material is published as an open educational resource complementing the existing resources for OMERO contributed by the community.</p>
<p>Tags: OMERO, Research Data Management, Nfdi4Bioimage, I3Dbio, Include In Dalia</p>
<p>Content type: Slides, Video</p>
<p><a class="reference external" href="https://zenodo.org/records/8323588">https://zenodo.org/records/8323588</a></p>
<p><a class="reference external" href="https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU">https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8323588">https://doi.org/10.5281/zenodo.8323588</a></p>
</section>
<hr class="docutils" />
<section id="introducing-omero-vitessce-an-omero-web-plugin-for-multi-modal-data">
<h2>Introducing OMERO-vitessce: an <a class="reference external" href="http://OMERO.web">OMERO.web</a> plugin for multi-modal data<a class="headerlink" href="#introducing-omero-vitessce-an-omero-web-plugin-for-multi-modal-data" title="Link to this heading">#</a></h2>
<p>Michele Bortolomeazzi, Christian Schmidt, Jan-Philipp Mallm</p>
<p>Published 2025-02-07</p>
<p>Licensed CC-BY-4.0</p>
<p>omero-vitessce: an <a class="reference external" href="http://OMERO.web">OMERO.web</a> plugin for multi-modal data viewing.
OMERO is the most used research data management system (RDM) in the bioimaging domain, and has been adopted as a centralized RDM solution by several academic and research institutions. A main reason for this is the ability to directly view and annotate images from a web-based interface. However, this feature of OMERO is currently underpowered for the visualization of very large or multimodal datasets. These datasets, are becoming a more and more common foundation for biological and biomedical studies, due to the recent developments in imaging, and sequencing technologies which enabled their application to spatial-omics. In order to begin to provide this multimodal-data capability to OMERO, we developed omero-vitessce (<a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/omero-vitessce/tree/main">NFDI4BIOIMAGE/omero-vitessce</a>), a new <a class="reference external" href="http://OMERO.web">OMERO.web</a> plugin for viewing data stored in OMERO with the Vitessce (<a class="reference external" href="http://vitessce.io/">http://vitessce.io/</a>) multimodal data viewer. omero-vitessce can be installed as an <a class="reference external" href="http://OMERO.web">OMERO.web</a> plugin with PiPy (<a class="reference external" href="https://pypi.org/project/omero-vitessce/">https://pypi.org/project/omero-vitessce/</a>), and allows users to set up interactive visualizations of their images of cells and tissues through interactive plots which are directly linked to the image. This enables the visual exploration of bioimage-analysis results and of multimodal data such as those generated through spatial-omics experiments. The data visualization is highly customizable and can be configured not only through custom configuration files, but also with the graphical interface provided by the plugin, thus making omero-vitessce a highly user-friendly solution for multimodal data viewing. most biological datasets. We plan to extend the interoperability of omero-vitessce with the OME-NGFF and SpatialData file formats to leverage the efficiency of these cloud optimized formats.
The three files in this Zenodo Record are all the same poster saved in different format all with high resolution images.</p>
<p>Tags: Nfdi4Bioimage, Exclude From Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14832855">https://zenodo.org/records/14832855</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14832855">https://doi.org/10.5281/zenodo.14832855</a></p>
</section>
<hr class="docutils" />
<section id="key-value-pair-template-for-annotation-of-datasets-in-omero-perikles-study">
<h2>Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)<a class="headerlink" href="#key-value-pair-template-for-annotation-of-datasets-in-omero-perikles-study" title="Link to this heading">#</a></h2>
<p>Anett Jannasch, Silke Tulok, Vanessa Aphaia Fiona Fuchs, Tom Boissonnet, Christian Schmidt, Michele Bortolomeazzi, Gunar Fabig, Chukwuebuka Okafornta</p>
<p>Published 2024-06-26</p>
<p>Licensed CC-BY-4.0</p>
<p>This is a Key-Value pair template used for the annotation of datasets in OMERO. It is tailored for a research study (PERIKLES project) on the biocompatibility of newly designed biomaterials out of pericardial tissue for cardiovascular substitutes (<a class="reference external" href="https://doi.org/10.1063/5.0182672">https://doi.org/10.1063/5.0182672</a>) conducted in the research department of Cardiac Surgery at the Faculty of Medicine Carl Gustav Carus at the Technische Universität Dresden . A corresponding public example dataset is used in the publication “Setting up an institutional OMERO environment for bioimage data: perspectives from both facility staff and users” and is available here
(<a class="reference external" href="https://omero.med.tu-dresden.de/webclient/?show=dataset-1557">https://omero.med.tu-dresden.de/webclient/?show=dataset-1557</a>).
The template is based on the REMBI recommendations (<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015</a>) and it was developed during the PoL-Bio-Image Analysis Symposium in Dresden Aug 28th- Sept 1th 2023. 
With this template it is possible to create a csv-file, that can be used to annotate a dataset in OMERO using the annotation script (<a class="github reference external" href="https://github.com/ome/omero-scripts/blob/develop/omero/annotation_scripts/">ome/omero-scripts</a>).
How to use:
select and copy the data range containing Keys and Values
open a new excel sheet and paste transpose in column B1
type in A1 ‘dataset’
insert in A2 the exact name of the dataset, which should be annotated in OMERO
save the new excel sheet in csv- (comma seperated values) file format</p>
<p>Example can be seen in sheet 1 ‘csv import’. Important note; the code has to be 8-Bit UCS transformation format (UTF-8) otherwise several characters (for example µ, %,°) might not be able to decode by the annotation script. We encountered this issue with old Microsoft Office versions (e.g. MS Office 2016). 
Note: By filling the values in the excel sheet, avoid the usage of decimal delimiter.
 
See cross reference:
10.5281/zenodo.12547566 Key-Value pair template for annotation of datasets in OMERO (light- and electron microscopy data within the research group of Prof. Mueller-Reichert)
10.5281/zenodo.12578084 Key-Value pair template for annotation in OMERO for light microscopy data acquired with AxioScan7 - Core Facility Cellular Imaging (CFCI)</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Exclude From Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/12546808">https://zenodo.org/records/12546808</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.12546808">https://doi.org/10.5281/zenodo.12546808</a></p>
</section>
<hr class="docutils" />
<section id="key-value-pair-template-for-annotation-of-datasets-in-omero-for-light-and-electron-microscopy-data-within-the-research-group-of-prof-muller-reichert">
<h2>Key-Value pair template for annotation of datasets in OMERO for light- and electron microscopy data within the research group of Prof. Müller-Reichert<a class="headerlink" href="#key-value-pair-template-for-annotation-of-datasets-in-omero-for-light-and-electron-microscopy-data-within-the-research-group-of-prof-muller-reichert" title="Link to this heading">#</a></h2>
<p>Gunar Fabig, Anett Jannasch, Chukwuebuka Okafornta, Tom Boissonnet, Christian Schmidt, Michele Bortolomeazzi, Vanessa Aphaia Fiona Fuchs, Maria Koeckert, Aayush Poddar, Martin Vogel, Hanna-Margareta Schwarzbach, Andy Vogelsang, Michael Gerlach, Anja Nobst, Thomas Müller-Reichert, Silke Tulok</p>
<p>Published 2024-06-26</p>
<p>Licensed CC-BY-4.0</p>
<p>This are a two Key-Value pair templates used for the annotation of datasets in OMERO. They are tailored for light- and electron microcopy data for all research projects of the research group of Prof. T. Mueller-Reichert.  All members of the Core Facility Cellular Imaging agreed for using these templates to annotate data in OMERO. Furthermore, there are a corresponding public example datasets used in the publication “Setting up an institutional OMERO environment for bioimage data: perspectives from both facility staff and users” and are available here:
<a class="reference external" href="https://omero.med.tu-dresden.de/webclient/?show=dataset-1552">https://omero.med.tu-dresden.de/webclient/?show=dataset-1552</a> –&gt; for lattice-light sheet microscopy
<a class="reference external" href="https://omero.med.tu-dresden.de/webclient/?show=dataset-1555--&amp;amp;gt">https://omero.med.tu-dresden.de/webclient/?show=dataset-1555–&amp;gt</a>; for electron microscopy data
That templates are based on the REMBI recommendations (<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015</a>) and were developed during the PoL-Bio-Image Analysis Symposium in Dresden Aug 28th- Sept 1st in 2023 and further adapeted during the usage of OMERO. 
With every template it is possible to create a csv-file, that can be used to annotate a dataset in OMERO using the annotation script (<a class="github reference external" href="https://github.com/ome/omero-scripts/blob/develop/omero/annotation_scripts/">ome/omero-scripts</a>).
How to use:</p>
<p>fill the template with metadata
select and copy the data range containing the Keys and Values
open a new excel sheet and paste transpose in cell A1
Important: cell A1 contains always the name ‘dataset’ and cell A2 contains the exact name of the dataset, which should be annotated in OMERO
save the new excel sheet in csv-file (comma separated values) format</p>
<p>Examples can be seen in sheet 3 ‘csv_TOMO’ and sheet 5 csv_TEM’.
Important note: The code has to be 8-Bit UCS transformation format (UTF-8) otherwise several characters (for example µ, %,°) might be not able to decode by the annotation script. We encountered this issue with old Microsoft-Office versions (MS Office 2016). 
Note: By filling the values in the excel sheet, avoid the usage of comma as decimal delimiter.
See cross reference:
10.5281/zenodo.12546808 Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)
10.5281/zenodo.12578084 Key-Value pair template for annotation in OMERO for light microscopy data acquired with AxioScan7 - Core Facility Cellular Imaging (CFCI)
 </p>
<p>Tags: Exclude From Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/12547566">https://zenodo.org/records/12547566</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.12547566">https://doi.org/10.5281/zenodo.12547566</a></p>
</section>
<hr class="docutils" />
<section id="mdemic-a-metadata-annotation-tool-to-facilitate-management-of-fair-image-data-in-the-bioimaging-community">
<h2>MDEmic: a metadata annotation tool to facilitate management of FAIR image data in the bioimaging community<a class="headerlink" href="#mdemic-a-metadata-annotation-tool-to-facilitate-management-of-fair-image-data-in-the-bioimaging-community" title="Link to this heading">#</a></h2>
<p>Susanne Kunis, Sebastian Hänsch, Christian Schmidt, Frances Wong, Caterina Strambio-De-Castillia, Stefanie Weidtkamp-Peters</p>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>Tags: Research Data Management, Metadata, Exclude From Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.nature.com/articles/s41592-021-01288-z">https://www.nature.com/articles/s41592-021-01288-z</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data">
<h2>NFDI4BIOIMAGE - An Initiative for a National Research Data Infrastructure for Microscopy Data<a class="headerlink" href="#nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Elisa Ferrando-May</p>
<p>Published 2021-04-29</p>
<p>Licensed CCY-BY-SA-4.0</p>
<p>Align existing and establish novel services &amp; solutions for data management tasks throughout the bioimage data lifecycle.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Exclude From Dalia</p>
<p>Content type: Conference Abstract, Slides</p>
<p><a class="reference external" href="https://doi.org/10.11588/heidok.00029489">https://doi.org/10.11588/heidok.00029489</a></p>
</section>
<hr class="docutils" />
<section id="omexcavator-a-tool-for-exporting-and-connecting-bioimaging-specific-metadata-in-wider-knowledge-graphs">
<h2>OMExcavator: a tool for exporting and connecting  Bioimaging-specific metadata in wider knowledge graphs<a class="headerlink" href="#omexcavator-a-tool-for-exporting-and-connecting-bioimaging-specific-metadata-in-wider-knowledge-graphs" title="Link to this heading">#</a></h2>
<p>Stefan Dvoretskii, Klaus Maier-Hein, Marco Nolden, Christian Schmidt, Michele Bortolomeazzi, Josh Moore</p>
<p>Published 2025-05-15</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15423904">https://zenodo.org/records/15423904</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15423904">https://doi.org/10.5281/zenodo.15423904</a></p>
</section>
<hr class="docutils" />
<section id="omexcavator-a-tool-for-exporting-and-connecting-domain-specific-metadata-in-a-wider-knowledge-graph">
<h2>OMExcavator: a tool for exporting and connecting domain-specific metadata in a wider knowledge graph<a class="headerlink" href="#omexcavator-a-tool-for-exporting-and-connecting-domain-specific-metadata-in-a-wider-knowledge-graph" title="Link to this heading">#</a></h2>
<p>Stefan Dvoretskii, Michele Bortolomeazzi, Marco Nolden, Christian Schmidt, Klaus Maier-Hein, Josh Moore</p>
<p>Published 2025-02-21</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage, Exclude From Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15268798">https://zenodo.org/records/15268798</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15268798">https://doi.org/10.5281/zenodo.15268798</a></p>
</section>
<hr class="docutils" />
<section id="research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey">
<h2>Research data management for bioimaging - the 2021 NFDI4BIOIMAGE community survey<a class="headerlink" href="#research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Janina Hanne, Josh Moore, Christian Meesters, Elisa Ferrando-May, et al.</p>
<p>Published 2022-09-20</p>
<p>Licensed CC-BY-4.0</p>
<p>As an initiative within Germany’s National Research Data Infrastructure, the authors conducted this community survey in summer 2021 to assess the state of the art of bioimaging RDM and the community needs.</p>
<p>Tags: Research Data Management, Exclude From Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://f1000research.com/articles/11-638/v2">https://f1000research.com/articles/11-638/v2</a></p>
</section>
<hr class="docutils" />
<section id="id1">
<h2>Research data management for bioimaging: the 2021 NFDI4BIOIMAGE community survey<a class="headerlink" href="#id1" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Janina Hanne, Josh Moore, Christian Meesters, Elisa Ferrando-May, Stefanie Weidtkamp-Peters, members of the NFDI4BIOIMAGE initiative</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Exclude From Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://f1000research.com/articles/11-638">https://f1000research.com/articles/11-638</a></p>
</section>
<hr class="docutils" />
<section id="setting-up-an-institutional-omero-environment-for-bioimage-data-perspectives-from-both-facility-staff-and-users">
<h2>Setting up an institutional OMERO environment for bioimage data: Perspectives from both facility staff and users<a class="headerlink" href="#setting-up-an-institutional-omero-environment-for-bioimage-data-perspectives-from-both-facility-staff-and-users" title="Link to this heading">#</a></h2>
<p>Anett Jannasch, Silke Tulok, Chukwuebuka William Okafornta, Thomas Kugel, Michele Bortolomeazzi, Tom Boissonnet, Christian Schmidt, Andy Vogelsang</p>
<p>Published 2024-09-14</p>
<p>Licensed CC-BY-4.0</p>
<p>Modern bioimaging core facilities at research institutions are essential for managing and maintaining high-end instruments, providing training and support for researchers in experimental design, image acquisition and data analysis.</p>
<p>Tags: Nfdi4Bioimage, OMERO, Bioimage Analysis, Exclude From Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://onlinelibrary.wiley.com/doi/10.1111/jmi.13360">https://onlinelibrary.wiley.com/doi/10.1111/jmi.13360</a></p>
</section>
<hr class="docutils" />
<section id="the-information-infrastructure-for-bioimage-data-i3d-bio-project-to-advance-fair-microscopy-data-management-for-the-community">
<h2>The Information Infrastructure for BioImage Data (I3D:bio) project to advance FAIR microscopy data management for the community<a class="headerlink" href="#the-information-infrastructure-for-bioimage-data-i3d-bio-project-to-advance-fair-microscopy-data-management-for-the-community" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Michele Bortolomeazzi, Tom Boissonnet, Julia Dohle, Tobias Wernet, Janina Hanne, Roland Nitschke, Susanne Kunis, Karen Bernhardt, Stefanie Weidtkamp-Peters, Elisa Ferrando-May</p>
<p>Published 2024-03-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Research data management (RDM) in microscopy and image analysis is a challenging task. Large files in proprietary formats, complex N-dimensional array structures, and various metadata models and formats can make image data handling inconvenient and difficult. For data organization, annotation, and sharing, researchers need solutions that fit everyday practice and comply with the FAIR (Findable, Accessible, Interoperable, Reusable) principles. International community-based efforts have begun creating open data models (OME), an open file format and translation library (OME-TIFF, Bio-Formats), data management software platforms, and microscopy metadata recommendations and annotation tools. Bringing these developments into practice requires support and training. Iterative feedback and tool improvement is needed to foster practical adoption by the scientific community. The Information Infrastructure for BioImage Data (I3D:bio) project works on guidelines, training resources, and practical assistance for FAIR microscopy RDM adoption with a focus on the management platform OMERO and metadata annotations.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Exclude From Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/10805204">https://zenodo.org/records/10805204</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10805204">https://doi.org/10.5281/zenodo.10805204</a></p>
</section>
<hr class="docutils" />
<section id="the-role-of-helmholtz-centers-in-nfdi4bioimage-a-national-consortium-enhancing-fair-data-management-for-microscopy-and-bioimage-analysis">
<h2>The role of Helmholtz Centers in NFDI4BIOIMAGE - A national consortium enhancing FAIR data management for microscopy and bioimage analysis<a class="headerlink" href="#the-role-of-helmholtz-centers-in-nfdi4bioimage-a-national-consortium-enhancing-fair-data-management-for-microscopy-and-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Christian Schmidt, Michele Bortolomeazzi, Julia Thoennissen, Jan Bumberger, Timo Dickscheid, Jan-Philipp Mallm, Elisa Ferrando-May</p>
<p>Published 2024-06-06</p>
<p>Licensed CC-BY-4.0</p>
<p>Germany’s National Research Data Infrastructure (NFDI) aims to establish a sustained, cross-disciplinary research data management (RDM) infrastructure that enables researchers to handle FAIR (findable, accessible, interoperable, reusable) data. While FAIR principles have been adopted by funders, policymakers, and publishers, their practical implementation remains an ongoing effort. In the field of bio-imaging, harmonization of data formats, metadata ontologies, and open data repositories is necessary to achieve FAIR data. The NFDI4BIOIMAGE was established to address these issues and develop tools and best practices to facilitate FAIR microscopy and image analysis data in alignment with international community activities. The consortium operates through its Data Stewards team to provide expertise and direct support to help overcome RDM challenges. The three Helmholtz Centers in NFDI4BIOIMAGE aim to collaborate closely with other centers and initiatives, such as HMC, Helmholtz AI, and HIP. Here we present NFDI4BIOIMAGE’s work and its significance for research in Helmholtz and beyond</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Exclude From Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/11501662">https://zenodo.org/records/11501662</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11501662">https://doi.org/10.5281/zenodo.11501662</a></p>
</section>
<hr class="docutils" />
<section id="community-meeting-2024-supporting-and-financing-rdm-projects-within-gerbi">
<h2>[Community Meeting 2024] Supporting and financing RDM projects within GerBI<a class="headerlink" href="#community-meeting-2024-supporting-and-financing-rdm-projects-within-gerbi" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters, Josh Moore, Christian Schmidt, Roland Nitschke, Susanne Kunis, Thomas Zobel</p>
<p>Published 2024-03-28</p>
<p>Licensed CC-BY-4.0</p>
<p>Overview of GerBI RDM projects: why and how?</p>
<p>Tags: Exclude From Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/10889694">https://zenodo.org/records/10889694</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10889694">https://doi.org/10.5281/zenodo.10889694</a></p>
</section>
<hr class="docutils" />
<section id="elmi2025-bridging-communities-with-ome-zarr">
<h2>[ELMI2025] Bridging communities with OME-Zarr<a class="headerlink" href="#elmi2025-bridging-communities-with-ome-zarr" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Aastha Mathur, Josh Moore</p>
<p>Published 2025-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Presented at ELMI2025
 
<a class="reference external" href="https://www.embl.org/about/info/course-and-conference-office/events/elmi2025/">https://www.embl.org/about/info/course-and-conference-office/events/elmi2025/</a></p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15393592">https://zenodo.org/records/15393592</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15393592">https://doi.org/10.5281/zenodo.15393592</a></p>
</section>
<hr class="docutils" />
<section id="short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">
<h2>[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure<a class="headerlink" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure" title="Link to this heading">#</a></h2>
<p>Christian Schmidt</p>
<p>Published 2024-04-10</p>
<p>Licensed CC-BY-4.0</p>
<p>Short Talk about the NFDI4BIOIMAGE consortium presented at the RDM in (Bio-)Medicine Information Event on April 10th, 2024, organized C³RDM &amp; ZB MED.</p>
<p>Tags: Exclude From Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/10939520">https://zenodo.org/records/10939520</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10939520">https://doi.org/10.5281/zenodo.10939520</a></p>
</section>
<hr class="docutils" />
<section id="webinar-a-journey-to-fair-bioimage-data">
<h2>[Webinar] A journey to FAIR bioimage data<a class="headerlink" href="#webinar-a-journey-to-fair-bioimage-data" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters, Tom Boissonnet, Christian Schmidt</p>
<p>Published 2025-07-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation slides from an EMBL-EBI Webinar Talk within the webinar series:
“How to organise and share my imaging data? - Multimodal data management for marine biologists, environmental scientists and imaging specialists”
 
Abstract / Description
Bioimaging is a pervasive and indispensable methodological approach in the life and biomedical sciences. Due to the development of new technologies and the easier access to compute resources, bioimaging experiments have become a big data discipline, facing the same challenges as other omics technologies within the life sciences. However, to fully exploit the potential of bioimage data, it is necessary to make the data FAIR. In this webinar we will present viable solutions for storing, processing, analysing, and, first and foremost, sharing bioimaging data. We will introduce services provided to the scientific community, that are dealing with various aspects of the bioimage data life cycle such as:</p>
<ul class="simple">
<li><p>Where to get support for bioimage data management- Local bioimage data management: OMERO and beyond- Annotation of bioimage data: metadata, ontologies, REMBI etc- Linking your image data with experimental protocols and analysis results- Large data living in the cloud: ome.zarr- Publication of bioimage data
Who is this course for?
This webinar is suitable for marine biologists and environmental scientists collecting samples from the natural environment, generating, visualising, annotating and analysing large, multimodal datasets such as imaging data, and sharing their data by submitting them to public data repositories. The webinar will support you to set up an efficient data flow that is aligned with FAIR principles.
This event is part of a webinar series organised by the STANDFLOW project, an initiative supported by EMBL’s Planetary biology Transversal Theme. STANDFLOW is about a collaborative effort towards creating a standardised data management workflow. The project primarily utilises imaging data derived from samples collected through the TREC (Traversing European Coastlines) and the Roscoff Culture Collection. For details on all topics covered in this series and registration information, please visit the following link: How to organise and share my imaging data?: Multimodal data management for marine biologists, and environmental scientists and imaging specialists
Outcomes
By the end of the webinar you will be able to: </p></li>
</ul>
<p>Find resources and support for bioimage data management
Get started with bioimage data annotation
Identify the dos and don’ts for bioimage data publication</p>
<p> 
(taken from: <a class="reference external" href="https://www.ebi.ac.uk/training/events/journey-fair-bioimage-data/">https://www.ebi.ac.uk/training/events/journey-fair-bioimage-data/</a>)</p>
<p>Tags: Nfdi4Bioimage, Fair Principles, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15796252">https://zenodo.org/records/15796252</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15796252">https://doi.org/10.5281/zenodo.15796252</a></p>
</section>
<hr class="docutils" />
<section id="workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg">
<h2>[Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg<a class="headerlink" href="#workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg" title="Link to this heading">#</a></h2>
<p>Tom Boissonnet, Bettina Hagen, Susanne Kunis, Christian Schmidt, Stefanie Weidtkamp-Peters</p>
<p>Published 2024-11-18</p>
<p>Licensed CC-BY-4.0</p>
<p>Fit for OMERO: How imaging facilities and IT departments work together to enable RDM for bioimaging
Description:
Research data management (RDM) in bioimaging is challenging because of large file sizes, heterogeneous file formats and the variability of imaging methods. The image data management system OMERO (OME Remote Objects) allows for centralized and secure storage, organization, annotation, and interrogation of microscopy data by researchers. It is an internationally well-supported open-source software tool that has become one of the best-known image data management tools among bioimaging scientists. Nevertheless, the de novo setup of OMERO at an institute is a multi-stakeholder process that demands time, funds, organization and iterative implementation. In this workshop, participants learn how to begin setting up OMERO-based image data management at their institution. The topics include:</p>
<p>Stakeholder identification at the university / research institute
Process management, time line expectations, and resources planning
Learning about each other‘s perspectives on chances and challenges for RDM
Funding opportunities and strategies for IT and imaging core facilities
Hands-on: Setting up an OMERO server in a virtual machine environment</p>
<p>Target audience:
This workshop was directed at universities and research institutions who consider or plan to implement OMERO, or are in an early phase of implementation. This workshop was intended for teams from IT departments and imaging facilities to participate together with one person from the IT department, and one person from the imaging core facility at the same institution.
The trainers:</p>
<p>Prof. Dr. Stefanie Weidtkamp-Peters (Imaging Core Facility Head, Center for Advanced Imaging, Heinrich Heine University of Düsseldorf)
Dr. Susanne Kunis (Software architect, OMERO administrator, metadata specialist, University of Osnabrück)
Dr. Tom Boissonnet (OMERO admin and image metadata specialist, Center for Advanced Imaging, Heinrich Heine University of Düsseldorf)
Dr. Bettina Hagen (IT Administration and service specialist, Max Planck Institute for the Biology of Ageing, Cologne) 
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center (DKFZ), Heidelberg)</p>
<p>Time and place
The format was a two-day, in-person workshop (October 16-17, 2024). Location: Heidelberg, Germany
Workshop learning goals</p>
<p>Learn the steps to establish a local RDM environment fit for bioimaging data
Create a network of IT experts and bioimaging specialists for bioimage RDM across institutions
Establish a stakeholder process management for installing OMERO-based RDM
Learn from each other, leverage different expertise
Learn how to train users, establish sustainability strategies, and foster FAIR RDM for bioimaging at your institution</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Exclude From Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14178789">https://zenodo.org/records/14178789</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14178789">https://doi.org/10.5281/zenodo.14178789</a></p>
</section>
<hr class="docutils" />
<section id="workshop-bioimage-data-management-and-analysis-with-omero">
<h2>[Workshop] Bioimage data management and analysis with OMERO<a class="headerlink" href="#workshop-bioimage-data-management-and-analysis-with-omero" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Michele Bortolomeazzi, Christian Schmidt</p>
<p>Published 2024-05-13</p>
<p>Licensed CC-BY-4.0</p>
<p>Here we share the material used in a workshop held on May 13th, 2024, at the German Cancer Research Center in Heidelberg (on-premise)
Description:Microscopy experiments generate information-rich, multi-dimensional data, allowing us to investigate biological processes at high spatial and temporal resolution. Image processing and analysis is a standard procedure to retrieve quantitative information from biological imaging. Due to the complex nature of bioimaging files that often come in proprietary formats, it can be challenging to organize, structure, and annotate bioimaging data throughout a project. Data often needs to be moved between collaboration partners, transformed into open formats, processed with a variety of software tools, and exported to smaller-sized images for presentation. The path from image acquisition to final publication figures with quantitative results must be documented and reproducible.
In this workshop, participants learn how to use OMERO to organize their data and enrich the bioimage data with structured metadata annotations.We also focus on image analysis workflows in combination with OMERO based on the Fiji/ImageJ software and using Jupyter Notebooks. In the last part, we explore how OMERO can be used to create publication figures and prepare bioimage data for publication in a suitable repository such as the Bioimage Archive.
Module 1 (9 am - 10.15 am): Basics of OMERO, data structuring and annotation
Module 2 (10.45 am - 12.45 pm): OMERO and Fiji
Module 3 (1.45 pm - 3.45 pm): OMERO and Jupyter Notebooks
Module 4 (4.15 pm - 6. pm): Publication-ready figures and data with OMERO
The target group for this workshopThis workshop is directed at researchers at all career levels who plan to or have started to use OMERO for their microscopy research data management. We encourage the workshop participants to bring example data from their research to discuss suitable metadata annotation for their everyday practice.
Prerequisites:Users should bring their laptops and have access to the internet through one of the following options:- eduroam- institutional WiFi- VPN connection to their institutional networks to access OMERO
Who are the trainers?
Dr. Riccardo Massei (Helmholtz-Center for Environmental Research, UFZ, Leipzig) - Data Steward for Bioimaging Data in NFDI4BIOIMAGE
Dr. Michele Bortolomeazzi (DKFZ, Single cell Open Lab, bioimage data specialist, bioinformatician, staff scientist in the NFDI4BIOIMAGE project)
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center, Heidelberg, Project Coordinator of the NFDI4BIOIMAGE project)</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/11350689">https://zenodo.org/records/11350689</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11350689">https://doi.org/10.5281/zenodo.11350689</a></p>
</section>
<hr class="docutils" />
<section id="workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero">
<h2>[Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO<a class="headerlink" href="#workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero" title="Link to this heading">#</a></h2>
<p>Vanessa Fiona Aphaia Fuchs, Christian Schmidt, Tom Boissonnet</p>
<p>Published 2024-05-06</p>
<p>Licensed CC-BY-4.0</p>
<p>Description
Microscopy experiments generate information-rich, multi-dimensional data, allowing us to investigate biological processes at high spatial and temporal resolution. Image processing and analysis is a standard procedure to retrieve quantitative information from biological imaging. Due to the complex nature of bioimaging files that often come in proprietary formats, it can be challenging to organize, structure, and annotate bioimaging data throughout a project. Data often needs to be moved between collaboration partners, transformed into open formats, processed with a variety of software tools, and exported to smaller-sized images for presentation. The path from image acquisition to final publication figures with quantitative results must be documented and reproducible.
In this workshop, participants learn how to use structured metadata annotations in the image data management platform OMERO (OME Remote Objects) to optimize their data handling. This strategy helps both with organizing data for easier processing and analysis and for the preparation of data publication in journal manuscripts and in public repositories such as the BioImage Archive. Participants learn the principles of leveraging object-oriented data organization in OMERO to enhance findability and usability of their data, also in collaborative settings. The integration of OMERO with image analysis tools, in particular ImageJ/Fiji, will be trained. Moreover, users learn about community-accepted metadata checklists (REMBI) to enrich the value of their data toward reproducibility and reusability. In this workshop, we will provide hands-on training and recommendations on:</p>
<p>Structured metadata annotation features in OMERO and how to use them
Types of metadata in bioimaging: Technical metadata, sample metadata, analysis metadata
The use of ontologies and terminologies for metadata annotation
REMBI, the recommended metadata for biological images
Metadata-assisted image analysis streamlining
Tools for metadata annotation in OMERO</p>
<p>The target group for this workshop
This workshop is directed at researchers at all career levels who have started using OMERO for their microscopy research data management. We encourage the workshop participants to bring example data from their research to discuss suitable metadata annotation for their everyday practice.
Who are the trainers (see trainer description below for more details)</p>
<p>Dr. Vanessa Fuchs (NFDI4BIOIMAGE Data Steward, Center for Advanced Imaging, Heinrich-Heine University of Düsseldorf)
Dr. Tom Boissonnet (OMERO admin and image metadata specialist, Center for Advanced Imaging, Heinrich-Heine University of Düsseldorf)
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center, Heidelberg)</p>
<p>Material Description
Published here are the presentation slides that were used for input from the trainers during the different sessions of the programme. Additionally, a Fiji Macro is published that depends on the OMERO Extensions Plugin by Pouchin et al, 2022, F100Research, <a class="reference external" href="https://doi.org/10.12688/f1000research.110385.2">https://doi.org/10.12688/f1000research.110385.2</a> 
Programme Overview
Day 1 - April 29th, 2024 09.00 a.m. to 10.00 a.m.: Session 1 - Welcome and Introduction
10.00 a.m. to 10.30 a.m.:  Session 2 - Introduction to the FAIR principles &amp; data annotation
10:30 a.m. to 10:45 a.m.: Coffee break
10.45 a.m. to 12.00 a.m.: Session 3 - Data structure (datasets in OMERO) and organization with Tags 
12.00 a.m. to 1.00 p.m.:  Lunch Break
1.00 p.m. to 2.00 p.m.:  Session 4 - REMBI, Key-Value pair annotations in bioimaging
2:00 p.m. to 2.30 p.m.:  Session 5 - Ontologies for Key-Value Pairs in OMERO
2:30 p.m. to 2:45 p.m. Coffee break
2.45 p.m. to 3.45 p.m.:  Wrap-up, discussion, outlook on day 2
Day 2 - April 30th, 2024
09.00 a.m. to 09.30 a.m.:  Arrival and Start into day 2
09.30 a.m. to 11.30 a.m.:  Session 6 - Hands-on : REMBI-based Key-Value Pair annotation in OMERO
11.30 a.m. to 12.30 a.m.:  Lunch Break
12.30 a.m. to 1.15 p.m.: Session 7 - OMERO and OMERO.plugins
1.15 p.m. to 2.00 p.m.: Session 8 - Loading OMERO-hosted data into Fiji
2.00 p.m. to 2.15 p.m.: Coffee break 
2.15 p.m. to 3.00 p.m.: Discussion, Outlook</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/11109616">https://zenodo.org/records/11109616</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11109616">https://doi.org/10.5281/zenodo.11109616</a></p>
</section>
<hr class="docutils" />
<section id="workshop-managing-fair-microscopy-data-at-scale-for-universities-and-research-institutions-an-introduction-for-non-imaging-stakeholders">
<h2>[Workshop] Managing FAIR microscopy data at scale for universities and research institutions: an introduction for non-imaging stakeholders<a class="headerlink" href="#workshop-managing-fair-microscopy-data-at-scale-for-universities-and-research-institutions-an-introduction-for-non-imaging-stakeholders" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Michele Bortolomeazzi, Ksenia Krooß, Jan-Philipp Mallm, Elisa Ferrando-May, Stefanie Weidtkamp-Peters</p>
<p>Published 2025-03-14</p>
<p>Licensed CC-BY-4.0</p>
<p>These slides were used in a workshop at the 2025 E-Science Tage in Heidelberg.
Workshop Abstract:
Effective Research Data Management (RDM) requires collaboration between infrastructure providers, support units, and domain-specific experts across scientific disciplines. Microscopy, or bioimaging, is a widely used technology at universities and research institutions, generating large, multi-dimensional datasets. Scientists now routinely produce microscopy data using advanced imaging modalities, often through centrally-provided instruments maintained by core facilities.
Bioimaging data management presents unique challenges: files are often large (e.g., 15+ GB for whole slide images), come in various proprietary formats, and are accessed frequently for viewing as well as for complex image processing and analysis workflows. Collaboration between experimenters, clinicians, group leaders, core facility staff, and image analysts adds to the complexity, increasing the risk of data fragmentation and metadata loss.
The DFG-funded project I3D:bio and the consortium NFDI4BIOIMAGE, part of Germany’s National Research Data Infrastructure (NFDI), are addressing these challenges by developing solutions and best practices for managing large, complex microscopy datasets. This workshop introduces the challenges of bioimaging RDM to institutional support personnel, including, for example, library staff, IT departments, and data stewards. Participants will explore the bioimaging RDM system OMERO, and apply structured metadata annotation and object-oriented data organization to a simple training dataset. OMERO offers centralized, secure access to data, allowing collaboration and reducing the data fragementation risk. Moreover, participants will experience the benefits of OME-Zarr, a chunked open file format designed for FAIR data sharing and remote access. OME-Zarr enables streaming of large, N-dimensional array-typed data over the Internet without the need to download whole files. An expanding toolbox for leveraging OME-Zarr for bioimaging data renders this file type a promising candidate for a standard file format suitable for use in FAIR Digital Object (FDO) implementations for microscopy data. OME-Zarr has become a pillar for imaging data sharing in two bioimaging-specific data repositories, i.e., the Image Data Resource (IDR) and the BioImage Archive (BIA). The team of Data Stewards from both abovenmentioned projects help researchers and research support staff to manage und publish bioimaging data.
By the end of the workshop, participants will have gained hands-on experience with bioimaging data and will be aware of support resources like the NFDI4BIOIMAGE Help Desk for addressing specific local use cases. Our goal is to promote collaboration across disciplines to effectively manage complex bioimaging data in compliance with the FAIR principles.
 </p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15026373">https://zenodo.org/records/15026373</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15026373">https://doi.org/10.5281/zenodo.15026373</a></p>
</section>
<hr class="docutils" />
<section id="workshop-research-data-management-for-microscopy-and-bioimage-analysis">
<h2>[Workshop] Research Data Management for Microscopy and BioImage Analysis<a class="headerlink" href="#workshop-research-data-management-for-microscopy-and-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Tom Boissonnet, Michele Bortolomeazzi, Ksenia Krooß</p>
<p>Published 2024-09-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Research Data Management for Microscopy and BioImage Analysis</p>
<p>Introduction to BioImaging Research Data Management, NFDI4BIOIMAGE and I3D:bioChristian Schmidt /DKFZ Heidelberg
OMERO as a tool for bioimaging data managementTom Boissonnet /Heinrich-Heine Universität Düsseldorf
Reproducible image analysis workflows with OMERO software APIsMichele Bortolomeazzi /DKFZ Heidelberg
Publishing datasets in public archives for bioimage dataKsenia Krooß /Heinrich-Heine Universität Düsseldorf</p>
<p>Date &amp; Venue:Thursday, Sept. 26, 5.30 p.m.Haus 22 / Paul Ehrlich Lecture Hall (H22-1)University Hospital Frankfurt</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13861026">https://zenodo.org/records/13861026</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13861026">https://doi.org/10.5281/zenodo.13861026</a></p>
<hr class="docutils" />
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./authors"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-journey-to-fair-microscopy-data">A journey to FAIR microscopy data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-practical-guide-to-bioimaging-research-data-management-in-core-facilities">A practical guide to bioimaging research data management in core facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#finding-and-choosing-a-data-repository">Finding and Choosing a Data Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-information-infrastructure-for-bioimage-data-bioimage-metadata">I3D bio – Information Infrastructure for BioImage Data - Bioimage Metadata</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introducing-omero-vitessce-an-omero-web-plugin-for-multi-modal-data">Introducing OMERO-vitessce: an OMERO.web plugin for multi-modal data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#key-value-pair-template-for-annotation-of-datasets-in-omero-perikles-study">Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#key-value-pair-template-for-annotation-of-datasets-in-omero-for-light-and-electron-microscopy-data-within-the-research-group-of-prof-muller-reichert">Key-Value pair template for annotation of datasets in OMERO for light- and electron microscopy data within the research group of Prof. Müller-Reichert</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#mdemic-a-metadata-annotation-tool-to-facilitate-management-of-fair-image-data-in-the-bioimaging-community">MDEmic: a metadata annotation tool to facilitate management of FAIR image data in the bioimaging community</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data">NFDI4BIOIMAGE - An Initiative for a National Research Data Infrastructure for Microscopy Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omexcavator-a-tool-for-exporting-and-connecting-bioimaging-specific-metadata-in-wider-knowledge-graphs">OMExcavator: a tool for exporting and connecting  Bioimaging-specific metadata in wider knowledge graphs</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omexcavator-a-tool-for-exporting-and-connecting-domain-specific-metadata-in-a-wider-knowledge-graph">OMExcavator: a tool for exporting and connecting domain-specific metadata in a wider knowledge graph</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey">Research data management for bioimaging - the 2021 NFDI4BIOIMAGE community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#id1">Research data management for bioimaging: the 2021 NFDI4BIOIMAGE community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-an-institutional-omero-environment-for-bioimage-data-perspectives-from-both-facility-staff-and-users">Setting up an institutional OMERO environment for bioimage data: Perspectives from both facility staff and users</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-information-infrastructure-for-bioimage-data-i3d-bio-project-to-advance-fair-microscopy-data-management-for-the-community">The Information Infrastructure for BioImage Data (I3D:bio) project to advance FAIR microscopy data management for the community</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-role-of-helmholtz-centers-in-nfdi4bioimage-a-national-consortium-enhancing-fair-data-management-for-microscopy-and-bioimage-analysis">The role of Helmholtz Centers in NFDI4BIOIMAGE - A national consortium enhancing FAIR data management for microscopy and bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#community-meeting-2024-supporting-and-financing-rdm-projects-within-gerbi">[Community Meeting 2024] Supporting and financing RDM projects within GerBI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi2025-bridging-communities-with-ome-zarr">[ELMI2025] Bridging communities with OME-Zarr</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#webinar-a-journey-to-fair-bioimage-data">[Webinar] A journey to FAIR bioimage data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg">[Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-bioimage-data-management-and-analysis-with-omero">[Workshop] Bioimage data management and analysis with OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero">[Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-managing-fair-microscopy-data-at-scale-for-universities-and-research-institutions-an-introduction-for-non-imaging-stakeholders">[Workshop] Managing FAIR microscopy data at scale for universities and research institutions: an introduction for non-imaging stakeholders</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-research-data-management-for-microscopy-and-bioimage-analysis">[Workshop] Research Data Management for Microscopy and BioImage Analysis</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Robert Haase, Clément Caporal,... and the NFDI4BioImage Initiative
</p>

  </div>
  
  <div class="footer-item">
    

  </div>
  
  <div class="footer-item">
    <p class="last-updated">
  Last updated on 2025-11-25.
  <br/>
</p>
  </div>
  
  <div class="footer-item">
    
<div class="extra_footer">
  <p>
Copyright: Licensed <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY 4.0</a> unless mentioned otherwise. 
Contributions and feedback are welcome.
</p>

</div>
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>