
<!DOCTYPE html>


<html lang="en" data-content_root="../" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Github repository (54) &#8212; NFDI4BioImage Training Materials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="../_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="../_static/styles/sphinx-book-theme.css?v=eba8b062" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-design.min.css?v=95c83b7e" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="../_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="../_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="../_static/doctools.js?v=9a2dae69"></script>
    <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="../_static/copybutton.js?v=f281be69"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="../_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'content_types/github repository';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Notebook (55)" href="notebook.html" />
    <link rel="prev" title="Event (8)" href="event.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
    <meta name="docbuild:last-update" content="2025-03-03"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="../readme.html">
  
  
  
  
  
    
    
      
    
    
    <img src="../_static/logo.png" class="logo__image only-light" alt="NFDI4BioImage Training Materials - Home"/>
    <script>document.write(`<img src="../_static/logo.png" class="logo__image only-dark" alt="NFDI4BioImage Training Materials - Home"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../readme.html">
                    NFDI4BioImage Training Materials
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">What's new</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../whats_new.html">Recently added (10)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Contributing</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../contributing/index.html">How to contribute</a></li>

<li class="toctree-l1"><a class="reference internal" href="../contributing/submit_app.html">Using the Training Materials Submission App</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contributing/format.html">YML format</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By tag</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../tags/artificial_intelligence.html">Artificial intelligence (44)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bioimage_analysis.html">Bioimage analysis (186)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bioinformatics.html">Bioinformatics (18)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/data_stewardship.html">Data stewardship (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/fair-principles.html">Fair-principles (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/fiji.html">Fiji (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/imagej.html">Imagej (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/licensing.html">Licensing (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/metadata.html">Metadata (15)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/napari.html">Napari (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/neubias.html">Neubias (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/nfdi4bioimage.html">Nfdi4bioimage (45)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/omero.html">Omero (35)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/open_science.html">Open science (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/open_source_software.html">Open source software (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/python.html">Python (71)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/reproducibility.html">Reproducibility (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/research_data_management.html">Research data management (132)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/sharing.html">Sharing (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/workflow.html">Workflow (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/workflow_engine.html">Workflow engine (13)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By content type</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="blog%20post.html">Blog post (23)</a></li>
<li class="toctree-l1"><a class="reference internal" href="book.html">Book (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="code.html">Code (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="collection.html">Collection (82)</a></li>
<li class="toctree-l1"><a class="reference internal" href="data.html">Data (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="document.html">Document (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="documentation.html">Documentation (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="event.html">Event (8)</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Github repository (54)</a></li>
<li class="toctree-l1"><a class="reference internal" href="notebook.html">Notebook (55)</a></li>
<li class="toctree-l1"><a class="reference internal" href="online%20tutorial.html">Online tutorial (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="poster.html">Poster (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="preprint.html">Preprint (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="publication.html">Publication (62)</a></li>
<li class="toctree-l1"><a class="reference internal" href="slides.html">Slides (83)</a></li>
<li class="toctree-l1"><a class="reference internal" href="tutorial.html">Tutorial (45)</a></li>
<li class="toctree-l1"><a class="reference internal" href="video.html">Video (32)</a></li>
<li class="toctree-l1"><a class="reference internal" href="website.html">Website (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="workshop.html">Workshop (12)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By license</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../licenses/all_rights_reserved.html">All rights reserved (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/apache-2.0.html">Apache-2.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-2-clause.html">Bsd-2-clause (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-3-clause.html">Bsd-3-clause (30)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-4.0.html">Cc-by-4.0 (259)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-sa-4.0.html">Cc-by-sa-4.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc0-1.0.html">Cc0-1.0 (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-2.0.html">Gpl-2.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-3.0.html">Gpl-3.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/mit.html">Mit (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/unknown.html">Unknown (98)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By domain</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../domain/biapol.github.io.html">Biapol.github.io (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/docs.google.com.html">Docs.google.com (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/doi.org.html">Doi.org (197)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/f1000research.com.html">F1000research.com (11)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/focalplane.biologists.com.html">Focalplane.biologists.com (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/git.mpi-cbg.de.html">Git.mpi-cbg.de (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/github.com.html">Github.com (132)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.biorxiv.org.html">Www.biorxiv.org (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.ebi.ac.uk.html">Www.ebi.ac.uk (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.nature.com.html">Www.nature.com (17)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.youtube.com.html">Www.youtube.com (25)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/zenodo.org.html">Zenodo.org (185)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Appendix</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../statistics/readme.html">Statistics</a></li>
<li class="toctree-l1"><a class="reference internal" href="../export/readme.html">Open data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../imprint.html">Imprint</a></li>

</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training/issues/new?title=Issue%20on%20page%20%2Fcontent_types/github repository.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/content_types/github repository.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Github repository (54)</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#advanced-scripting-with-fiji">Advanced scripting with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bids-lecture-2024">BIDS-lecture-2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#biomero">Biomero</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4proteins-notebooks">DL4Proteins-notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embedseg-repository">EmbedSeg Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#example-pipeline-tutorial">Example Pipeline Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#friendsdontletfriends">FriendsDontLetFriends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library">I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-virtual-bio-image-analysis-code-generation">I2K2024(virtual) - Bio-Image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#leo">LEO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-materials-of-the-deeplife-course">Lecture-materials of the DeepLife course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nlp-course">NLP Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubiaspasteur2023-advancedcellpose">NeubiasPasteur2023_AdvancedCellPose</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks">OMERO - HCS analysis pipeline using Jupyter Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-tools">Omero-tools</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#online-r-learning">Online_R_learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#parallel-fiji-visualizer">Parallel_Fiji_Visualizer</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-omero-batch-plugin">Plugin “omero-batch-plugin”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-omero-cli-transfer">Plugin “omero-cli-transfer”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-simple-omero-client">Plugin “simple-omero-client”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-llms-course">Prompt-Engineering-LLMs-Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-for-python-programmers">QuPath for Python programmers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdm-system-connector">RDM_system_connector</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scipy-cookbook">Scipy Cookbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#simpleitk-notebooks">SimpleITK-Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#statistical-rethinking">Statistical Rethinking</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ultrack-i2k-2024-workshop-materials">Ultrack I2K 2024 Workshop Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#v4sdb-winter-school-2025">V4SDB_Winter_School_2025</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#virtual-i2k-2024-multiview-stitcher">Virtual-I2K-2024-multiview-stitcher</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioformats2raw-converter">bioformats2raw Converter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenbiene">datenbiene</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dmtxsamplecreator">dmtxSampleCreator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2020-s3-zarr-workshop">i2k-2020-s3-zarr-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-generative-ai">introduction-to-generative-ai</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis">introduction-to-image-analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#napari-scipy2025-workshop">napari-scipy2025-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-workshop">nextflow-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome-ngff-validator">ome-ngff-validator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome2024-ngff-challenge">ome2024-ngff-challenge</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-arc">omero-arc</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-ontop-mappings">omero-ontop-mappings</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-vitessce">omero-vitessce</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#patho-prompt-injection">patho_prompt_injection</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#raw2ometiff-converter">raw2ometiff Converter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rse-skills-workshop">rse-skills-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scanpy-tutorials">scanpy-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scikit-learn-mooc">scikit-learn MOOC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#skimage-tutorials">skimage-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-resources">training-resources</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="github-repository-54">
<h1>Github repository (54)<a class="headerlink" href="#github-repository-54" title="Link to this heading">#</a></h1>
<section id="advanced-scripting-with-fiji">
<h2>Advanced scripting with Fiji<a class="headerlink" href="#advanced-scripting-with-fiji" title="Link to this heading">#</a></h2>
<p>Joran Deschamps, Damian Dalle Nogare</p>
<p>Published 2023-06-14T08:25:35+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Fiji</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/nobias-fht/advanced-scripting">nobias-fht/advanced-scripting</a></p>
</section>
<hr class="docutils" />
<section id="bids-lecture-2024">
<h2>BIDS-lecture-2024<a class="headerlink" href="#bids-lecture-2024" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python. The material developed here between April and July 2024.</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence, Python</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ScaDS/BIDS-lecture-2024/">ScaDS/BIDS-lecture-2024</a></p>
</section>
<hr class="docutils" />
<section id="biomero">
<h2>Biomero<a class="headerlink" href="#biomero" title="Link to this heading">#</a></h2>
<p>Torec Luik, Johannes Soltwedel</p>
<p>Published 2024-07-24</p>
<p>Licensed APACHE-2.0</p>
<p>The BIOMERO framework, for BioImage analysis in OMERO, allows you to run (FAIR) bioimage analysis workflows directly from OMERO on a high-performance compute (HPC) cluster, remotely through SSH.</p>
<p>Tags: OMERO, Github</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NL-BioImaging/biomero">NL-BioImaging/biomero</a></p>
</section>
<hr class="docutils" />
<section id="dl4proteins-notebooks">
<h2>DL4Proteins-notebooks<a class="headerlink" href="#dl4proteins-notebooks" title="Link to this heading">#</a></h2>
<p>Michael Chungyoun, Courtney Thomas, Britnie Carpentier, GabeAu79, puv-sreev, Jeffrey Gray</p>
<p>Published 2024-09-04T12:24:24+00:00</p>
<p>Colab Notebooks covering deep learning tools for biomolecular structure prediction and design</p>
<p>Tags: Bioinformatics</p>
<p>Content type: Github Repository, Collection</p>
<p><a class="github reference external" href="https://github.com/Graylab/DL4Proteins-notebooks">Graylab/DL4Proteins-notebooks</a></p>
</section>
<hr class="docutils" />
<section id="embedseg-repository">
<h2>EmbedSeg Repository<a class="headerlink" href="#embedseg-repository" title="Link to this heading">#</a></h2>
<p>Manan Lalit, Joran Deschamps, Florian Jug, Ajinkya Kulkarni</p>
<p>Licensed CC-BY-NC-4.0</p>
<p>Code Implementation for EmbedSeg, an Instance Segmentation Method for Microscopy Images</p>
<p>Tags: Bioimage Analysis, Instance Segmentation</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/juglab/EmbedSeg">juglab/EmbedSeg</a></p>
</section>
<hr class="docutils" />
<section id="example-pipeline-tutorial">
<h2>Example Pipeline Tutorial<a class="headerlink" href="#example-pipeline-tutorial" title="Link to this heading">#</a></h2>
<p>Tim Monko</p>
<p>Published 2024-10-28</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Napari-ndev is a collection of widgets intended to serve any person seeking to process microscopy images from start to finish. The goal of this example pipeline is to get the user familiar with working with napari-ndev for batch processing and reproducibility (view Image Utilities and Workflow Widget).</p>
<p>Tags: Napari, Bioimage Analysis</p>
<p>Content type: Documentation, Github Repository, Tutorial</p>
<p><a class="reference external" href="https://timmonko.github.io/napari-ndev/tutorial/01_example_pipeline/">https://timmonko.github.io/napari-ndev/tutorial/01_example_pipeline/</a></p>
<p><a class="github reference external" href="https://github.com/timmonko/napari-ndev">timmonko/napari-ndev</a></p>
</section>
<hr class="docutils" />
<section id="friendsdontletfriends">
<h2>FriendsDontLetFriends<a class="headerlink" href="#friendsdontletfriends" title="Link to this heading">#</a></h2>
<p>C. Li</p>
<p>Published 2024-03-10T15:34:07+00:00</p>
<p>Licensed MIT</p>
<p>Friends don’t let friends make certain types of data visualization - What are they and why are they bad.</p>
<p>Tags: Visualization</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/cxli233/FriendsDontLetFriends">cxli233/FriendsDontLetFriends</a></p>
</section>
<hr class="docutils" />
<section id="i2k-2024-clesperanto-gpu-accelerated-image-processing-library">
<h2>I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library<a class="headerlink" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library" title="Link to this heading">#</a></h2>
<p>Stephane Rigaud, Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Course and material for the clEsperanto workshop presented at I2K 2024 &#64; Human Technopol (Milan, Italy). The workshop is an hands-on demo of the clesperanto project, focussing on how to use the library for users who want use GPU-acceleration for their Image Processing pipeline.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Workshop, Tutorial, Notebook</p>
<p><a class="github reference external" href="https://github.com/StRigaud/clesperanto_workshop_I2K24?tab=readme-ov-file">StRigaud/clesperanto_workshop_I2K24</a></p>
</section>
<hr class="docutils" />
<section id="i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">
<h2>I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark<a class="headerlink" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark" title="Link to this heading">#</a></h2>
<p>Stephan Saalfeld, Tobias Pietzsch</p>
<p>Published None</p>
<p>Licensed APACHE-2.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Notebook, Github Repository</p>
<p><a class="reference external" href="https://saalfeldlab.github.io/i2k2024-lazy-workshop/">https://saalfeldlab.github.io/i2k2024-lazy-workshop/</a></p>
<p><a class="github reference external" href="https://github.com/saalfeldlab/i2k2024-lazy-workshop">saalfeldlab/i2k2024-lazy-workshop</a></p>
</section>
<hr class="docutils" />
<section id="i2k2024-virtual-bio-image-analysis-code-generation">
<h2>I2K2024(virtual) - Bio-Image Analysis Code Generation<a class="headerlink" href="#i2k2024-virtual-bio-image-analysis-code-generation" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This repository contains training materials for the Tutorial “Bio-Image Analysis Code Generation” at the From Images To Knowledge (I2K) Conference (virtual) October 28th-30th 2024.</p>
<p>Tags: Bioimage Analysis, Notebooks, Biabob</p>
<p>Content type: Github Repository, Tutorial, Notebook</p>
<p><a class="github reference external" href="https://github.com/haesleinhuepf/i2k2024-ai-code-generation">haesleinhuepf/i2k2024-ai-code-generation</a></p>
</section>
<hr class="docutils" />
<section id="leo">
<h2>LEO<a class="headerlink" href="#leo" title="Link to this heading">#</a></h2>
<p>Rodrigo Escobar Diaz Guerrero</p>
<p>Published 2025-01-08T10:20:30+00:00</p>
<p>Licensed MIT</p>
<p>Linking Electronic Lab Notebooks and other sources with OMERO objects</p>
<p>Tags: OMERO, Research Data Management, Electronic Lab Notebooks</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/LEO">NFDI4BIOIMAGE/LEO</a></p>
</section>
<hr class="docutils" />
<section id="lecture-materials-of-the-deeplife-course">
<h2>Lecture-materials of the DeepLife course<a class="headerlink" href="#lecture-materials-of-the-deeplife-course" title="Link to this heading">#</a></h2>
<p>Carl Herrmann, annavonbachmann, David Hoksza, Martin Schätz, Dario Malchiodi, jnguyenvan, Britta Velten, Elodie Laine, JanaBraunger, barwil</p>
<p>Published 2023-12-06</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioinformatics</p>
<p>Content type: Github Repository, Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/deeplife4eu/Lecture-materials/">deeplife4eu/Lecture-materials</a></p>
</section>
<hr class="docutils" />
<section id="multiplexed-tissue-imaging-tools-and-approaches">
<h2>Multiplexed tissue imaging - tools and approaches<a class="headerlink" href="#multiplexed-tissue-imaging-tools-and-approaches" title="Link to this heading">#</a></h2>
<p>Agustín Andrés Corbat, OmFrederic, Jonas Windhager, Kristína Lidayová</p>
<p>Licensed CC-BY-4.0</p>
<p>Material for the I2K 2024 “Multiplexed tissue imaging - tools and approaches” workshop</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Slides, Workshop</p>
<p><a class="github reference external" href="https://github.com/BIIFSweden/I2K2024-MTIWorkshop">BIIFSweden/I2K2024-MTIWorkshop</a></p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1">https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">
<h2>NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)<a class="headerlink" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository" title="Link to this heading">#</a></h2>
<p>Mohamed Abdrabbou, Mehrnaz Babaki, Tom Boissonnet, Michele Bortolomeazzi, Eik Dahms, Vanessa Fuchs, A. F. Moritz Hoevels, Niraj Kandpal, Christoph Möhl, Joshua A. Moore, Astrid Schauss, Andrea Schrader, Torsten Stöter, Julia Thönnißen, Monica Valencia-S., H. Lukas Weil, Jens Wendt, Peter Zentis</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository documents the first NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon), where topics like ‘Interoperability’, ‘REMBI / Mapping’, and ‘Neuroglancer (OMERO / zarr)’ were explored through collaborative discussions and workflow sessions, culminating in reports that bridge NFDI4Bioimage to DataPLANT. Funded by various DFG initiatives, this event emphasized documentation and use cases, contributing preparatory work for future interoperability projects at the 2nd de.NBI BioHackathon in Bielefeld.</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Github Repository</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10609770">https://zenodo.org/doi/10.5281/zenodo.10609770</a></p>
</section>
<hr class="docutils" />
<section id="nlp-course">
<h2>NLP Course<a class="headerlink" href="#nlp-course" title="Link to this heading">#</a></h2>
<p>Yandex School of Data Analysis</p>
<p>Licensed MIT</p>
<p>YSDA course in Natural Language Processing</p>
<p>Tags: Natural Language Processing</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/yandexdataschool/nlp_course">yandexdataschool/nlp_course</a></p>
</section>
<hr class="docutils" />
<section id="neubiaspasteur2023-advancedcellpose">
<h2>NeubiasPasteur2023_AdvancedCellPose<a class="headerlink" href="#neubiaspasteur2023-advancedcellpose" title="Link to this heading">#</a></h2>
<p>Gaelle Letort</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tutorial for running CellPose advanced functions</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/gletort/NeubiasPasteur2023_AdvancedCellPose">gletort/NeubiasPasteur2023_AdvancedCellPose</a></p>
</section>
<hr class="docutils" />
<section id="omero-hcs-analysis-pipeline-using-jupyter-notebooks">
<h2>OMERO - HCS analysis pipeline using Jupyter Notebooks<a class="headerlink" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks" title="Link to this heading">#</a></h2>
<p>Riccardo Massei</p>
<p>Licensed MIT</p>
<p>Material and solutions for the course ‘Bioimage data management and analysis with OMERO’ held in Heidelberg (13th May 2024) - Module 3 (1.45 pm - 3.45 pm): OMERO and Jupyter Notebooks. Main goal of the workflow is to show the potential of JN to perform reproducible image analysis in connection with an OMERO instance. In this specific example, we are performing a simple nuclei segmentation from raw images uploaded in OMERO.</p>
<p>Tags: Teaching, Bioimage Analysis, Notebooks, Python, OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/rmassei/2024-jn-omero-pipeline">rmassei/2024-jn-omero-pipeline</a></p>
</section>
<hr class="docutils" />
<section id="object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">
<h2>Object Tracking and Track Analysis using TrackMate and CellTracksColab<a class="headerlink" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab" title="Link to this heading">#</a></h2>
<p>Joanna Pylvänäinen</p>
<p>Published None</p>
<p>Licensed GPL-3.0</p>
<p>I2K 2024 workshop materials for “Object Tracking and Track Analysis using TrackMate and CellTracksColab”</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Tutorial, Workshop, Slides</p>
<p><a class="github reference external" href="https://github.com/CellMigrationLab/I2K_2024">CellMigrationLab/I2K_2024</a></p>
</section>
<hr class="docutils" />
<section id="omero-tools">
<h2>Omero-tools<a class="headerlink" href="#omero-tools" title="Link to this heading">#</a></h2>
<p>Johannes Soltwedel</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository contains a collection of tools for working with OMERO. Such tools can be working with the OMERO command line interface to transfer datasets between repositories, etc.</p>
<p>Tags: OMERO, Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="reference external" href="https://biapol.github.io/omero-tools/">https://biapol.github.io/omero-tools/</a></p>
</section>
<hr class="docutils" />
<section id="online-r-learning">
<h2>Online_R_learning<a class="headerlink" href="#online-r-learning" title="Link to this heading">#</a></h2>
<p>C. Li</p>
<p>Published 2023-07-09T06:27:14+00:00</p>
<p>Licensed CC0-1.0</p>
<p>Online R learning for applied statistics</p>
<p>Tags: Statistics</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/cxli233/Online_R_learning">cxli233/Online_R_learning</a></p>
</section>
<hr class="docutils" />
<section id="parallel-fiji-visualizer">
<h2>Parallel_Fiji_Visualizer<a class="headerlink" href="#parallel-fiji-visualizer" title="Link to this heading">#</a></h2>
<p>Matthew Mueller, Aaron, Advanced Bioimaging Center</p>
<p>Published 2024-05-15T06:14:24+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Fiji</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/abcucberkeley/Parallel_Fiji_Visualizer">abcucberkeley/Parallel_Fiji_Visualizer</a></p>
</section>
<hr class="docutils" />
<section id="plugin-omero-batch-plugin">
<h2>Plugin “omero-batch-plugin”<a class="headerlink" href="#plugin-omero-batch-plugin" title="Link to this heading">#</a></h2>
<p>Licensed GPL-2.0</p>
<p>An ImageJ plugin to run a script or macro on a batch of images from/to OMERO.</p>
<p>Tags: OMERO, Imagej, Imagej Macro, Github</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/GReD-Clermont/omero_batch-plugin">GReD-Clermont/omero_batch-plugin</a></p>
</section>
<hr class="docutils" />
<section id="plugin-omero-cli-transfer">
<h2>Plugin “omero-cli-transfer”<a class="headerlink" href="#plugin-omero-cli-transfer" title="Link to this heading">#</a></h2>
<p>Erick Martins Ratamero, Jean-Marie Burel, Will Moore, Guillaume Gay, Christoph Moehl, et al.</p>
<p>Published 2024-09-14</p>
<p>Licensed GPL-2.0</p>
<p>An OMERO CLI plugin for creating and using transfer packets between OMERO servers.</p>
<p>Tags: OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ome/omero-cli-transfer">ome/omero-cli-transfer</a></p>
</section>
<hr class="docutils" />
<section id="plugin-simple-omero-client">
<h2>Plugin “simple-omero-client”<a class="headerlink" href="#plugin-simple-omero-client" title="Link to this heading">#</a></h2>
<p>Pierre Pouchin, Rdornier, kekunn, Jean-Marie Burel</p>
<p>Licensed GPL-2.0</p>
<p>A wrapper library which can be called from scripts in Fiji, but can mostly be used in Maven projects to wrap calls to the underlying OMERO Java Gateway.</p>
<p>Tags: OMERO, Github, Fiji</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/GReD-Clermont/simple-omero-client">GReD-Clermont/simple-omero-client</a></p>
</section>
<hr class="docutils" />
<section id="prompt-engineering-llms-course">
<h2>Prompt-Engineering-LLMs-Course<a class="headerlink" href="#prompt-engineering-llms-course" title="Link to this heading">#</a></h2>
<p>Published 2024-09-11T07:45:30+00:00</p>
<p>Licensed MIT</p>
<p>Tags: Llms, Prompt Engineering, Code Generation</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/HelmholtzAI-Consultants-Munich/Prompt-Engineering-LLMs-Course">HelmholtzAI-Consultants-Munich/Prompt-Engineering-LLMs-Course</a></p>
</section>
<hr class="docutils" />
<section id="qupath-for-python-programmers">
<h2>QuPath for Python programmers<a class="headerlink" href="#qupath-for-python-programmers" title="Link to this heading">#</a></h2>
<p>Alan O’Callaghan, Léo Leplat</p>
<p>Published 2024-10-23</p>
<p>Licensed UNKNOWN</p>
<p>These are the notebooks and associated files for the i2k 2024 QuPath for Python programmers workshop.</p>
<p>Tags: Python, Notebooks, Open Source Software, Bioimage Analysis</p>
<p>Content type: Github Repository, Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/qupath/i2k-qupath-for-python-programmers">qupath/i2k-qupath-for-python-programmers</a></p>
</section>
<hr class="docutils" />
<section id="rdm-system-connector">
<h2>RDM_system_connector<a class="headerlink" href="#rdm-system-connector" title="Link to this heading">#</a></h2>
<p>SaibotMagd</p>
<p>Licensed UNKNOWN</p>
<p>This tool is intended to link different research data management platforms with each other.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/SaibotMagd/RDM_system_connector">SaibotMagd/RDM_system_connector</a></p>
</section>
<hr class="docutils" />
<section id="scipy-cookbook">
<h2>Scipy Cookbook<a class="headerlink" href="#scipy-cookbook" title="Link to this heading">#</a></h2>
<p>Pauli Virtanen et al.</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This is a conversion and second life of SciPy Cookbook as a bunch of Ipython notebooks.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scipy/scipy-cookbook">scipy/scipy-cookbook</a></p>
</section>
<hr class="docutils" />
<section id="simpleitk-notebooks">
<h2>SimpleITK-Notebooks<a class="headerlink" href="#simpleitk-notebooks" title="Link to this heading">#</a></h2>
<p>Ziv Yaniv et al.</p>
<p>Licensed APACHE-2.0</p>
<p>Jupyter notebooks for learning how to use SimpleITK</p>
<p>Tags: Bioimage Analysis, Simpleitk</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/InsightSoftwareConsortium/SimpleITK-Notebooks">InsightSoftwareConsortium/SimpleITK-Notebooks</a></p>
</section>
<hr class="docutils" />
<section id="statistical-rethinking">
<h2>Statistical Rethinking<a class="headerlink" href="#statistical-rethinking" title="Link to this heading">#</a></h2>
<p>Richard McElreath</p>
<p>Published 2024-03-01</p>
<p>Licensed CC0-1.0</p>
<p>This course teaches data analysis, but it focuses on scientific models. The unfortunate truth about data is that nothing much can be done with it, until we say what caused it. We will prioritize conceptual, causal models and precise questions about those models. We will use Bayesian data analysis to connect scientific models to evidence. And we will learn powerful computational tools for coping with high-dimension, imperfect data of the kind that biologists and social scientists face.</p>
<p>Tags: Statistics</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/rmcelreath/stat_rethinking_2024">rmcelreath/stat_rethinking_2024</a></p>
</section>
<hr class="docutils" />
<section id="ultrack-i2k-2024-workshop-materials">
<h2>Ultrack I2K 2024 Workshop Materials<a class="headerlink" href="#ultrack-i2k-2024-workshop-materials" title="Link to this heading">#</a></h2>
<p>Jordão Bragantini, Teun Huijben</p>
<p>Licensed BSD3-CLAUSE</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/royerlab/ultrack-i2k2024">royerlab/ultrack-i2k2024</a></p>
<p><a class="reference external" href="https://royerlab.github.io/ultrack-i2k2024/">https://royerlab.github.io/ultrack-i2k2024/</a></p>
</section>
<hr class="docutils" />
<section id="v4sdb-winter-school-2025">
<h2>V4SDB_Winter_School_2025<a class="headerlink" href="#v4sdb-winter-school-2025" title="Link to this heading">#</a></h2>
<p>Joanna Pylvänäinen</p>
<p>Published 2025-01-13T08:29:22+00:00</p>
<p>Training materials for V4SDB Student Winter School, 28th-31st January 2025 at ELTE Eötvös Loránd University in Budapest, Hungary</p>
<p>Tags: Cell Tracking, Bioimage Analysis</p>
<p>Content type: Github Repository, Collection</p>
<p><a class="github reference external" href="https://github.com/CellMigrationLab/V4SDB_Winter_School_2025">CellMigrationLab/V4SDB_Winter_School_2025</a></p>
</section>
<hr class="docutils" />
<section id="virtual-i2k-2024-multiview-stitcher">
<h2>Virtual-I2K-2024-multiview-stitcher<a class="headerlink" href="#virtual-i2k-2024-multiview-stitcher" title="Link to this heading">#</a></h2>
<p>Published 2024-10-30T07:38:11+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Repository accompanying the multiview-stitcher tutorial for Virtual I2K 2024</p>
<p>Tags: Big Data, Bioimageanalysis</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/m-albert/Virtual-I2K-2024-multiview-stitcher">m-albert/Virtual-I2K-2024-multiview-stitcher</a></p>
</section>
<hr class="docutils" />
<section id="ymia-python-based-event-series-training-material">
<h2>YMIA - Python-Based Event Series Training Material<a class="headerlink" href="#ymia-python-based-event-series-training-material" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Robert Haase, ENicolay</p>
<p>Published None</p>
<p>Licensed MIT</p>
<p>This repository offer access to teaching material and useful resources for the YMIA - Python-Based Event Series.</p>
<p>Tags: Python, Artifical Intelligence, Bioimage Analysis</p>
<p>Content type: Github Repository, Slides</p>
<p><a class="github reference external" href="https://github.com/rmassei/ymia_python_event_series_material">rmassei/ymia_python_event_series_material</a></p>
</section>
<hr class="docutils" />
<section id="bioformats2raw-converter">
<h2>bioformats2raw Converter<a class="headerlink" href="#bioformats2raw-converter" title="Link to this heading">#</a></h2>
<p>Melissa Linkert, Chris Allan, Josh Moore, Sébastien Besson, David Gault, et al.</p>
<p>Licensed GPL-2.0</p>
<p>Java application to convert image file formats, including .mrxs, to an intermediate Zarr structure compatible with the OME-NGFF specification.</p>
<p>Tags: Open Source Software</p>
<p>Content type: Application, Github Repository</p>
<p><a class="github reference external" href="https://github.com/glencoesoftware/bioformats2raw">glencoesoftware/bioformats2raw</a></p>
</section>
<hr class="docutils" />
<section id="datenbiene">
<h2>datenbiene<a class="headerlink" href="#datenbiene" title="Link to this heading">#</a></h2>
<p>Torsten Stöter</p>
<p>Published 2025-02-02T18:50:20+00:00</p>
<p>Licensed GNU GENERAL PUBLIC LICENSE V3.0</p>
<p>Tags: Research Data Management</p>
<p>Content type: Github Repository, Software</p>
<p><a class="github reference external" href="https://github.com/tstoeter/datenbiene">tstoeter/datenbiene</a></p>
</section>
<hr class="docutils" />
<section id="dmtxsamplecreator">
<h2>dmtxSampleCreator<a class="headerlink" href="#dmtxsamplecreator" title="Link to this heading">#</a></h2>
<p>SaibotMagd</p>
<p>Published 2023-06-06T11:52:14+00:00</p>
<p>Licensed APACHE-2.0</p>
<p>firefox extension: reads datamatrix code from camera and create a sample in an inventory to link it into an ELN.</p>
<p>Tags: Nfdi4Bioimage</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/SaibotMagd/dmtxSampleCreator">SaibotMagd/dmtxSampleCreator</a></p>
</section>
<hr class="docutils" />
<section id="i2k-2020-s3-zarr-workshop">
<h2>i2k-2020-s3-zarr-workshop<a class="headerlink" href="#i2k-2020-s3-zarr-workshop" title="Link to this heading">#</a></h2>
<p>Constantin Pape, Christian Tischer</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Big Data</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/tischi/i2k-2020-s3-zarr-workshop">tischi/i2k-2020-s3-zarr-workshop</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-generative-ai">
<h2>introduction-to-generative-ai<a class="headerlink" href="#introduction-to-generative-ai" title="Link to this heading">#</a></h2>
<p>Bruna Piereck, Alexander Botzki</p>
<p>Published 2024-09-27T14:38:51+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Course repository for Strategic Use of Generative AI</p>
<p>Tags: Artificial Intelligence</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/vibbits/introduction-to-generative-ai">vibbits/introduction-to-generative-ai</a></p>
<p><a class="reference external" href="https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/introduction-to-generative-ai/refs/heads/main/README.md">https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/introduction-to-generative-ai/refs/heads/main/README.md</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-image-analysis">
<h2>introduction-to-image-analysis<a class="headerlink" href="#introduction-to-image-analysis" title="Link to this heading">#</a></h2>
<p>Dave Barry, Stefania Marcotti, Martin Jones</p>
<p>Published 2024-10-23T14:05:55+00:00</p>
<p>Licensed CC-BY-SA-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Notebooks</p>
<p><a class="github reference external" href="https://github.com/RMS-DAIM/introduction-to-image-analysis">RMS-DAIM/introduction-to-image-analysis</a></p>
</section>
<hr class="docutils" />
<section id="napari-scipy2025-workshop">
<h2>napari-scipy2025-workshop<a class="headerlink" href="#napari-scipy2025-workshop" title="Link to this heading">#</a></h2>
<p>Draga Doncila Pop</p>
<p>Published 2025-02-28T23:41:10+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This is a three-part workshop guiding you through using napari to view images, a brief bioimaging analysis application, and extending napari’s functionality with your own custom workflows.</p>
<p>Tags: Python, Napari</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/DragaDoncila/napari-scipy2025-workshop">DragaDoncila/napari-scipy2025-workshop</a></p>
</section>
<hr class="docutils" />
<section id="nextflow-workshop">
<h2>nextflow-workshop<a class="headerlink" href="#nextflow-workshop" title="Link to this heading">#</a></h2>
<p>Tuur Muyldermans, Kris Davie, Alexander, Nicolas Vannieuwkerke, Kobe Lavaerts, Marcel Ribeiro-Dantas, Bruna Piereck, Steff Taelman</p>
<p>Published 2023-03-29T10:40:04+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Nextflow workshop materials March 2023</p>
<p>Tags: Workflow, Nextflow</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/vibbits/nextflow-workshop">vibbits/nextflow-workshop</a></p>
<p><a class="reference external" href="https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/nextflow-workshop/main/README.md#1">https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/nextflow-workshop/main/README.md#1</a></p>
</section>
<hr class="docutils" />
<section id="ome-ngff-validator">
<h2>ome-ngff-validator<a class="headerlink" href="#ome-ngff-validator" title="Link to this heading">#</a></h2>
<p>Will Moore, Josh Moore, Yaroslav Halchenko, Sébastien Besson</p>
<p>Published 2022-09-29</p>
<p>Licensed BSD-2-CLAUSE</p>
<p>Web page for validating OME-NGFF files.</p>
<p>Content type: Github Repository, Application</p>
<p><a class="reference external" href="https://ome.github.io/ome-ngff-validator/">https://ome.github.io/ome-ngff-validator/</a></p>
<p><a class="github reference external" href="https://github.com/ome/ome-ngff-validator">ome/ome-ngff-validator</a></p>
</section>
<hr class="docutils" />
<section id="ome2024-ngff-challenge">
<h2>ome2024-ngff-challenge<a class="headerlink" href="#ome2024-ngff-challenge" title="Link to this heading">#</a></h2>
<p>Will Moore, Josh Moore, sherwoodf, Jean-Marie Burel, Norman Rzepka, dependabot[bot], JensWendt, Joost de Folter, Torsten St\xF6ter, AybukeKY, Eric Perlman, Tom Boissonnet</p>
<p>Published 2024-08-30T12:00:53+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Project planning and material repository for the 2024 challenge to generate 1 PB of OME-Zarr data</p>
<p>Tags: Sharing, Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ome/ome2024-ngff-challenge">ome/ome2024-ngff-challenge</a></p>
</section>
<hr class="docutils" />
<section id="omero-arc">
<h2>omero-arc<a class="headerlink" href="#omero-arc" title="Link to this heading">#</a></h2>
<p>Christoph Moehl, Peter Zentis, Niraj Kandpal</p>
<p>Published 2023-12-18T16:11:04+00:00</p>
<p>Licensed GNU GENERAL PUBLIC LICENSE V3.0</p>
<p>Library to export OMERO projects to ARC repositories</p>
<p>Tags: OMERO, Research Data Management</p>
<p>Content type: Github Repository, Software</p>
<p><a class="github reference external" href="https://github.com/cmohl2013/omero-arc">cmohl2013/omero-arc</a></p>
</section>
<hr class="docutils" />
<section id="omero-ontop-mappings">
<h2>omero-ontop-mappings<a class="headerlink" href="#omero-ontop-mappings" title="Link to this heading">#</a></h2>
<p>Carsten Fortmann-Grote, andrawaag, Jerven Bolleman</p>
<p>Published 2024-09-13T08:01:09+00:00</p>
<p>ONTOP module for querying OMERO with SPARQL</p>
<p>Tags: OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/German-BioImaging/omero-ontop-mappings">German-BioImaging/omero-ontop-mappings</a></p>
</section>
<hr class="docutils" />
<section id="omero-vitessce">
<h2>omero-vitessce<a class="headerlink" href="#omero-vitessce" title="Link to this heading">#</a></h2>
<p>Michele Bortolomeazzi</p>
<p>Published 2024-11-25T10:51:01+00:00</p>
<p>Licensed AGPL-3.0</p>
<p><a class="reference external" href="http://OMERO.web">OMERO.web</a> plugin for the Vitessce multimodal data viewer.</p>
<p>Tags: OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/omero-vitessce">NFDI4BIOIMAGE/omero-vitessce</a></p>
</section>
<hr class="docutils" />
<section id="patho-prompt-injection">
<h2>patho_prompt_injection<a class="headerlink" href="#patho-prompt-injection" title="Link to this heading">#</a></h2>
<p>JanClusmann, Tim Lenz</p>
<p>Published 2024-11-08T08:32:03+00:00</p>
<p>Licensed GPL-3.0</p>
<p>Tags: Histopathology, Bioimage Analysis</p>
<p>Content type: Github Repository, Notebook</p>
<p><a class="github reference external" href="https://github.com/KatherLab/patho_prompt_injection">KatherLab/patho_prompt_injection</a></p>
</section>
<hr class="docutils" />
<section id="raw2ometiff-converter">
<h2>raw2ometiff Converter<a class="headerlink" href="#raw2ometiff-converter" title="Link to this heading">#</a></h2>
<p>Melissa Linkert, Chris Allan, Sébastien Besson, Josh Moore</p>
<p>Licensed GPL-2.0</p>
<p>Java application to convert a directory of tiles to an OME-TIFF pyramid. This is the second half of iSyntax/.mrxs =&gt; OME-TIFF conversion.</p>
<p>Tags: Open Source Software</p>
<p>Content type: Application, Github Repository</p>
<p><a class="github reference external" href="https://github.com/glencoesoftware/raw2ometiff">glencoesoftware/raw2ometiff</a></p>
</section>
<hr class="docutils" />
<section id="rse-skills-workshop">
<h2>rse-skills-workshop<a class="headerlink" href="#rse-skills-workshop" title="Link to this heading">#</a></h2>
<p>Jack Atkinson</p>
<p>Published 2023-12-22T17:39:48+00:00</p>
<p>Licensed GPL-3.0</p>
<p>Teaching materials for improving research software writing abilities.</p>
<p>Tags: Research Software Engineering</p>
<p>Content type: Github Repository, Slides</p>
<p><a class="github reference external" href="https://github.com/jatkinson1000/rse-skills-workshop">jatkinson1000/rse-skills-workshop</a></p>
</section>
<hr class="docutils" />
<section id="scanpy-tutorials">
<h2>scanpy-tutorials<a class="headerlink" href="#scanpy-tutorials" title="Link to this heading">#</a></h2>
<p>Alex Wolf, pre-commit-ci[bot], Philipp A., Isaac Virshup, Ilan Gold, Giovanni Palla, Fidel Ramirez, G\xF6k\xE7en Eraslan, Sergei Rybakov, Abolfazl (Abe), Adam Gayoso, Dinesh Palli, Gregor Sturm, Jan Lause, Karin Hrovatin, Krzysztof Polanski, RaphaelBuzzi, Yimin Zheng, Yishen Miao, evanbiederstedt</p>
<p>Published 2018-12-16T03:42:46+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Scanpy Tutorials.</p>
<p>Tags: Single-Cell Analysis, Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scverse/scanpy-tutorials">scverse/scanpy-tutorials</a></p>
</section>
<hr class="docutils" />
<section id="scikit-learn-mooc">
<h2>scikit-learn MOOC<a class="headerlink" href="#scikit-learn-mooc" title="Link to this heading">#</a></h2>
<p>Loïc Estève et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>Machine learning in Python with scikit-learn MOOC</p>
<p>Tags: Bioimage Analysis, Machine Learning</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/INRIA/scikit-learn-mooc">INRIA/scikit-learn-mooc</a></p>
</section>
<hr class="docutils" />
<section id="skimage-tutorials">
<h2>skimage-tutorials<a class="headerlink" href="#skimage-tutorials" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias et al.</p>
<p>Licensed CC0 1.0 UNIVERSAL</p>
<p>skimage-tutorials - a collection of tutorials for the scikit-image package.</p>
<p>Tags: Bioimage Analysis, Scikit-Image</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scikit-image/skimage-tutorials">scikit-image/skimage-tutorials</a></p>
</section>
<hr class="docutils" />
<section id="training-resources">
<h2>training-resources<a class="headerlink" href="#training-resources" title="Link to this heading">#</a></h2>
<p>Christian Tischer, Antonio Politi, Toby Hodges, maulakhan, grinic, bugraoezdemir, Tim-Oliver Buchholz, Elnaz Fazeli, Aliaksandr Halavatyi, Dominik Kutra, Stefania Marcotti, AnniekStok, Felix, jhennies, Severina Klaus, Martin Schorb, Nima Vakili, Sebastian Gonzalez Tirado, Stefan Helfrich, Yi Sun, Ziqiang Huang, Jan Eglinger, Constantin Pape, Joel Lüthi, Matt McCormick, Oane Gros</p>
<p>Published 2020-04-23T07:51:38+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Resources for teaching/preparing to teach bioimage analysis</p>
<p>Tags: Bioimageanalysis, Neurobias</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NEUBIAS/training-resources">NEUBIAS/training-resources</a></p>
<hr class="docutils" />
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./content_types"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="event.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Event (8)</p>
      </div>
    </a>
    <a class="right-next"
       href="notebook.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Notebook (55)</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#advanced-scripting-with-fiji">Advanced scripting with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bids-lecture-2024">BIDS-lecture-2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#biomero">Biomero</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4proteins-notebooks">DL4Proteins-notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embedseg-repository">EmbedSeg Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#example-pipeline-tutorial">Example Pipeline Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#friendsdontletfriends">FriendsDontLetFriends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library">I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-virtual-bio-image-analysis-code-generation">I2K2024(virtual) - Bio-Image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#leo">LEO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-materials-of-the-deeplife-course">Lecture-materials of the DeepLife course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nlp-course">NLP Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubiaspasteur2023-advancedcellpose">NeubiasPasteur2023_AdvancedCellPose</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks">OMERO - HCS analysis pipeline using Jupyter Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-tools">Omero-tools</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#online-r-learning">Online_R_learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#parallel-fiji-visualizer">Parallel_Fiji_Visualizer</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-omero-batch-plugin">Plugin “omero-batch-plugin”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-omero-cli-transfer">Plugin “omero-cli-transfer”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plugin-simple-omero-client">Plugin “simple-omero-client”</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-llms-course">Prompt-Engineering-LLMs-Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-for-python-programmers">QuPath for Python programmers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdm-system-connector">RDM_system_connector</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scipy-cookbook">Scipy Cookbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#simpleitk-notebooks">SimpleITK-Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#statistical-rethinking">Statistical Rethinking</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ultrack-i2k-2024-workshop-materials">Ultrack I2K 2024 Workshop Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#v4sdb-winter-school-2025">V4SDB_Winter_School_2025</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#virtual-i2k-2024-multiview-stitcher">Virtual-I2K-2024-multiview-stitcher</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioformats2raw-converter">bioformats2raw Converter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenbiene">datenbiene</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dmtxsamplecreator">dmtxSampleCreator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2020-s3-zarr-workshop">i2k-2020-s3-zarr-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-generative-ai">introduction-to-generative-ai</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis">introduction-to-image-analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#napari-scipy2025-workshop">napari-scipy2025-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-workshop">nextflow-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome-ngff-validator">ome-ngff-validator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome2024-ngff-challenge">ome2024-ngff-challenge</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-arc">omero-arc</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-ontop-mappings">omero-ontop-mappings</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-vitessce">omero-vitessce</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#patho-prompt-injection">patho_prompt_injection</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#raw2ometiff-converter">raw2ometiff Converter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rse-skills-workshop">rse-skills-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scanpy-tutorials">scanpy-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scikit-learn-mooc">scikit-learn MOOC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#skimage-tutorials">skimage-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-resources">training-resources</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Robert Haase, Clément Caporal,... and the NFDI4BioImage Initiative
</p>

  </div>
  
  <div class="footer-item">
    

  </div>
  
  <div class="footer-item">
    <p class="last-updated">
  Last updated on 2025-03-03.
  <br/>
</p>
  </div>
  
  <div class="footer-item">
    
<div class="extra_footer">
  <p>
Copyright: Licensed <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY 4.0</a> unless mentioned otherwise. 
Contributions and feedback are welcome.
</p>

</div>
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>