
<!DOCTYPE html>


<html lang="en" data-content_root="../" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Slides (83) &#8212; NFDI4BioImage Training Materials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="../_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="../_static/styles/sphinx-book-theme.css?v=eba8b062" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-design.min.css?v=95c83b7e" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="../_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="../_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="../_static/doctools.js?v=9a2dae69"></script>
    <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="../_static/copybutton.js?v=f281be69"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="../_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'content_types/slides';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Tutorial (45)" href="tutorial.html" />
    <link rel="prev" title="Publication (62)" href="publication.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
    <meta name="docbuild:last-update" content="2025-03-04"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="../readme.html">
  
  
  
  
  
    
    
      
    
    
    <img src="../_static/logo.png" class="logo__image only-light" alt="NFDI4BioImage Training Materials - Home"/>
    <script>document.write(`<img src="../_static/logo.png" class="logo__image only-dark" alt="NFDI4BioImage Training Materials - Home"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../readme.html">
                    NFDI4BioImage Training Materials
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">What's new</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../whats_new.html">Recently added (10)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Contributing</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../contributing/index.html">How to contribute</a></li>

<li class="toctree-l1"><a class="reference internal" href="../contributing/submit_app.html">Using the Training Materials Submission App</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contributing/format.html">YML format</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By tag</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../tags/artificial_intelligence.html">Artificial intelligence (44)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bioimage_analysis.html">Bioimage analysis (186)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/bioinformatics.html">Bioinformatics (18)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/data_stewardship.html">Data stewardship (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/fair-principles.html">Fair-principles (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/fiji.html">Fiji (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/imagej.html">Imagej (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/licensing.html">Licensing (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/metadata.html">Metadata (15)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/napari.html">Napari (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/neubias.html">Neubias (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/nfdi4bioimage.html">Nfdi4bioimage (45)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/omero.html">Omero (35)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/open_science.html">Open science (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/open_source_software.html">Open source software (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/python.html">Python (71)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/reproducibility.html">Reproducibility (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/research_data_management.html">Research data management (132)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/sharing.html">Sharing (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/workflow.html">Workflow (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../tags/workflow_engine.html">Workflow engine (13)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By content type</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="blog%20post.html">Blog post (23)</a></li>
<li class="toctree-l1"><a class="reference internal" href="book.html">Book (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="code.html">Code (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="collection.html">Collection (82)</a></li>
<li class="toctree-l1"><a class="reference internal" href="data.html">Data (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="document.html">Document (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="documentation.html">Documentation (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="event.html">Event (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="github%20repository.html">Github repository (54)</a></li>
<li class="toctree-l1"><a class="reference internal" href="notebook.html">Notebook (55)</a></li>
<li class="toctree-l1"><a class="reference internal" href="online%20tutorial.html">Online tutorial (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="poster.html">Poster (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="preprint.html">Preprint (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="publication.html">Publication (62)</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Slides (83)</a></li>
<li class="toctree-l1"><a class="reference internal" href="tutorial.html">Tutorial (45)</a></li>
<li class="toctree-l1"><a class="reference internal" href="video.html">Video (32)</a></li>
<li class="toctree-l1"><a class="reference internal" href="website.html">Website (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="workshop.html">Workshop (12)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By license</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../licenses/all_rights_reserved.html">All rights reserved (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/apache-2.0.html">Apache-2.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-2-clause.html">Bsd-2-clause (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-3-clause.html">Bsd-3-clause (30)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-4.0.html">Cc-by-4.0 (260)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-sa-4.0.html">Cc-by-sa-4.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc0-1.0.html">Cc0-1.0 (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-2.0.html">Gpl-2.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-3.0.html">Gpl-3.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/mit.html">Mit (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/unknown.html">Unknown (98)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By domain</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../domain/biapol.github.io.html">Biapol.github.io (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/docs.google.com.html">Docs.google.com (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/doi.org.html">Doi.org (199)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/f1000research.com.html">F1000research.com (11)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/focalplane.biologists.com.html">Focalplane.biologists.com (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/git.mpi-cbg.de.html">Git.mpi-cbg.de (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/github.com.html">Github.com (132)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.biorxiv.org.html">Www.biorxiv.org (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.ebi.ac.uk.html">Www.ebi.ac.uk (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.nature.com.html">Www.nature.com (17)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.youtube.com.html">Www.youtube.com (25)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/zenodo.org.html">Zenodo.org (187)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Appendix</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../statistics/readme.html">Statistics</a></li>
<li class="toctree-l1"><a class="reference internal" href="../export/readme.html">Open data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../imprint.html">Imprint</a></li>

</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training/issues/new?title=Issue%20on%20page%20%2Fcontent_types/slides.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/content_types/slides.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Slides (83)</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium">“ZENODO und Co.” Was bringt und wer braucht ein Repositorium?</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ai-ml-dl-in-bioimage-analysis-webinar">AI ML DL in Bioimage Analysis - Webinar</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#adding-a-workflow-to-biaflows">Adding a Workflow to BIAFLOWS</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten">Alles meins – oder!? Urheberrechte klären für Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis">Bio Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-with-the-help-of-large-language-models">Bio-image Analysis with the Help of Large Language Models</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science-lectures-uni-leipzig-scads-ai">Bio-image Data Science Lectures @ Uni Leipzig / ScaDS.AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-bioimage-analysis-workflow-using-deep-learning">Building a Bioimage Analysis Workflow using Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-introduction">CellProfiler Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">Challenges and opportunities for bio-image analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#crashkurs-forschungsdatenmanagement">Crashkurs Forschungsdatenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#creating-open-computational-curricula">Creating open computational curricula</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training">Cultivating Open Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training-to-advance-bio-image-analysis">Cultivating Open Training to advance Bio-image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl-mbl-2021-exercises">DL@MBL 2021 Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-management-at-france-bioimaging">Data management at France BioImaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagement">Datenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz">Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-1">Datenmanagementpläne erstellen - Teil 1</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-2">Datenmanagementpläne erstellen - Teil 2</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy">Dr Guillaume Jacquemet on studying cancer cell metastasis in the era of deep learning for microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten">Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fairy-deep-learning-for-bioimage-analysis">FAIRy deep-learning for bioImage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers">From Paper to Pixels: Navigation through your Research Data - presentations of speakers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-training-material">Galaxy Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#generative-artificial-intelligence-for-bio-image-analysis">Generative artificial intelligence for bio-image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#hackaton-results-conversion-of-knime-image-analysis-workflows-to-galaxy">Hackaton Results - Conversion of KNIME image analysis workflows to Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#hitchhiking-through-a-diverse-bio-image-analysis-software-universe">Hitchhiking through a diverse Bio-image Analysis Software Universe</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-data-services-at-euro-bioimaging-community-efforts-towards-fair-image-data-and-analysis-services">Image Data Services at Euro-BioImaging: Community efforts towards FAIR Image Data and Analysis Services</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-in-galaxy">Image analysis in Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej-macro-introduction">ImageJ Macro Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej2-api-beating">ImageJ2 API-beating</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#intro-napari-slides">Intro napari slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden">Introduction to ImageJ macro programming, Scientific Computing Facility, MPI CBG Dresden</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-research-data-management-and-open-research">Introduction to Research Data Management and Open Research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#jupyter-for-interactive-cloud-computing">Jupyter for interactive cloud computing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kollaboratives-arbeiten-und-versionskontrolle-mit-git">Kollaboratives Arbeiten und Versionskontrolle mit Git</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-applied-bioimage-analysis-2020">Lecture Applied Bioimage Analysis 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-materials-of-the-deeplife-course">Lecture-materials of the DeepLife course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-deep-learning-applications-to-bioimage-analysis">Machine Learning - Deep Learning. Applications to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-and-deep-learning-on-the-cloud-segmentation">Machine and Deep Learning on the cloud: Segmentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#methods-in-bioimage-analysis">Methods in bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microsam-talks">MicroSam-Talks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive">Microscopy data analysis: machine learning and the BioImage Archive</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multi-view-fusion">Multi-view fusion</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud">My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-analyst-school-2018">NEUBIAS Analyst School 2018</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-course-2017">NEUBIAS Bioimage Analyst Course 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2019">NEUBIAS Bioimage Analyst School 2019</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2020">NEUBIAS Bioimage Analyst School 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage">NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data">NFDI4BIOIMAGE - An Initiative for a National Research Data Infrastructure for Microscopy Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-scalable-and-reproducible-scientific-workflows">Nextflow: Scalable and reproducible scientific workflows</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-science-sharing-licensing">Open Science, Sharing &amp; Licensing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing">Parallelization and heterogeneous computing: from pure CPU to GPU-accelerated image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#promoting-reproducibility-in-biomedical-research-through-image-analysis">Promoting Reproducibility in Biomedical Research through Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-open-source-software-for-analysing-awkward-images">QuPath: Open source software for analysing (awkward) images</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again">RDF as a bridge to domain-platforms like OMERO, or There and back again.</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-management-seminar-slides">Research Data Management Seminar - Slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sharing-and-licensing-material">Sharing and licensing material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten">So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sustainable-data-stewardship">Sustainable Data Stewardship</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#thinking-data-management-on-different-scales">Thinking data management on different scales</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-theory-trackmate-and-mastodon">Tracking Theory, TrackMate, and Mastodon</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#welcome-to-bioimage-town">Welcome to BioImage Town</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-bioimage-analysis-an-introduction">What is Bioimage Analysis? An Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-objects-in-2d-and-3d">Working with objects in 2D and 3D</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-pixels">Working with pixels</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zidas-2020-introduction-to-deep-learning">ZIDAS 2020 Introduction to Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#n4bi-ahm-welcome-to-bioimage-town">[N4BI AHM] Welcome to BioImage Town</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ilastik-interactive-machine-learning-for-bio-image-analysis">ilastik: interactive machine learning for (bio)image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rse-skills-workshop">rse-skills-workshop</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="slides-83">
<h1>Slides (83)<a class="headerlink" href="#slides-83" title="Link to this heading">#</a></h1>
<section id="zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium">
<h2>“ZENODO und Co.” Was bringt und wer braucht ein Repositorium?<a class="headerlink" href="#zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium" title="Link to this heading">#</a></h2>
<p>Elfi Hesse, Jan-Christoph Deinert, Christian Löschen</p>
<p>Published 2021-01-25</p>
<p>Licensed CC-BY-4.0</p>
<p>Die Online-Veranstaltung fand am 21.01.2021 im Rahmen der SaxFDM-Veranstaltungsreihe “Digital Kitchen - Küchengespräche mit SaxFDM” statt. SaxFDM-Sprecherin Elfi Hesse (HTW Dresden) erläuterte zunächst Grundsätzliches zum Thema Repositorien. Anschließend teilten Nutzer (Jan Deinert – HZDR) und Anbieter (Christian Löschen – TU Dresden/ZIH) lokaler Repositorien ihre Erfahrungen mit uns.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4461261">https://zenodo.org/records/4461261</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4461261">https://doi.org/10.5281/zenodo.4461261</a></p>
</section>
<hr class="docutils" />
<section id="a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">
<h2>A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter<a class="headerlink" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter" title="Link to this heading">#</a></h2>
<p>Anca Margineanu, Chiara Stringari, Marcelo Zoccoler, Cornelia Wetzker</p>
<p>Licensed CC-BY-4.0</p>
<p>The presentations introduce open-source software to read in, visualize and analyse fluorescence lifetime imaging microscopy (FLIM) raw data developed for life scientists. The slides were presented at German Bioimaging (GerBI) FLIM Workshop held February 26 to 29 2024 at the Biomedical Center of LMU München by Anca Margineanu, Chiara Stringari and Conni Wetzker.</p>
<p>Tags: Bioimage Analysis, Flim</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10886749">https://zenodo.org/doi/10.5281/zenodo.10886749</a></p>
</section>
<hr class="docutils" />
<section id="ai-ml-dl-in-bioimage-analysis-webinar">
<h2>AI ML DL in Bioimage Analysis - Webinar<a class="headerlink" href="#ai-ml-dl-in-bioimage-analysis-webinar" title="Link to this heading">#</a></h2>
<p>Yannick KREMPP</p>
<p>Published 2024-11-14</p>
<p>Licensed UNKNOWN</p>
<p>A review of the tools, methods and concepts useful for biologists and life scientists as well as bioimage analysts.</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Video, Slides, Webinar</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=TJXNMIWtdac">https://www.youtube.com/watch?v=TJXNMIWtdac</a></p>
</section>
<hr class="docutils" />
<section id="adding-a-workflow-to-biaflows">
<h2>Adding a Workflow to BIAFLOWS<a class="headerlink" href="#adding-a-workflow-to-biaflows" title="Link to this heading">#</a></h2>
<p>Sébastien Tosi, Volker Baecker, Benjamin Pavie</p>
<p>Licensed BSD-2-CLAUSE</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/RoccoDAnt/Defragmentation_TrainingSchool_EOSC-Life_2022/blob/main/Slides/Adding_a_workflow_to_BIAFLOWS.pdf">RoccoDAnt/Defragmentation_TrainingSchool_EOSC-Life_2022</a></p>
</section>
<hr class="docutils" />
<section id="alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten">
<h2>Alles meins – oder!? Urheberrechte klären für Forschungsdaten<a class="headerlink" href="#alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten" title="Link to this heading">#</a></h2>
<p>Stephan Wünsche</p>
<p>Published 2024-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Wem gehören Forschungsdaten? Diese Frage stellt sich bei Daten, an deren Entstehung mehrere Personen beteiligt waren, und besonders bei Textdaten, Bildern und Videos. Hier lernen Sie, für Ihr eigenes Forschungsvorhaben zu erkennen, wessen Urheber- und Leistungsschutzrechte zu berücksichtigen sind. Sie erfahren, wie Sie mit Hilfe von Vereinbarungen frühzeitig Rechtssicherheit herstellen, etwa um Daten weitergeben oder publizieren zu können.
 
 </p>
<p>Tags: Research Data Management, Licensing</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11472148">https://zenodo.org/records/11472148</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11472148">https://doi.org/10.5281/zenodo.11472148</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis">
<h2>Bio Image Analysis<a class="headerlink" href="#bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Christian Tischer</p>
<p>Licensed UNKNOWN</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/tischi/presentation-image-analysis">tischi/presentation-image-analysis</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">
<h2>Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities<a class="headerlink" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities" title="Link to this heading">#</a></h2>
<p>Cornelia Wetzker</p>
<p>Published 2023-11-08</p>
<p>Licensed CC-BY-4.0</p>
<p>This presentation gives a short outline of the complexity of data and metadata in the bioimaging universe. It introduces NFDI4BIOIMAGE as a newly formed consortium as part of the German ‘Nationale Forschungsdateninfrastruktur’ (NFDI) and its goals and tools for data management including its current members on TU Dresden campus.  </p>
<p>Tags: Research Data Management, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10083555">https://zenodo.org/records/10083555</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10083555">https://doi.org/10.5281/zenodo.10083555</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-with-the-help-of-large-language-models">
<h2>Bio-image Analysis with the Help of Large Language Models<a class="headerlink" href="#bio-image-analysis-with-the-help-of-large-language-models" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-03-13</p>
<p>Licensed CC-BY-4.0</p>
<p>Large Language Models (LLMs) change the way how we use computers. This also has impact on the bio-image analysis community. We can generate code that analyzes biomedical image data if we ask the right prompts. This talk outlines introduces basic principles, explains prompt engineering and how to apply it to bio-image analysis. We also compare how different LLM vendors perform on code generation tasks and which challenges are ahead for the bio-image analysis community.</p>
<p>Tags: Artificial Intelligence, Python</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10815329">https://zenodo.org/records/10815329</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10815329">https://doi.org/10.5281/zenodo.10815329</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-science-lectures-uni-leipzig-scads-ai">
<h2>Bio-image Data Science Lectures &#64; Uni Leipzig / <a class="reference external" href="http://ScaDS.AI">ScaDS.AI</a><a class="headerlink" href="#bio-image-data-science-lectures-uni-leipzig-scads-ai" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>These are the PPTx training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python. The material developed here between April and July 2024.</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence, Python</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/12623730">https://zenodo.org/records/12623730</a></p>
</section>
<hr class="docutils" />
<section id="building-a-bioimage-analysis-workflow-using-deep-learning">
<h2>Building a Bioimage Analysis Workflow using Deep Learning<a class="headerlink" href="#building-a-bioimage-analysis-workflow-using-deep-learning" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/esgomezm/NEUBIAS_chapter_DL_2020">esgomezm/NEUBIAS_chapter_DL_2020</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-introduction">
<h2>CellProfiler Introduction<a class="headerlink" href="#cellprofiler-introduction" title="Link to this heading">#</a></h2>
<p>Anna Klemm</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Cellprofiler, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/ahklemm/CellProfiler_Introduction">ahklemm/CellProfiler_Introduction</a></p>
</section>
<hr class="docutils" />
<section id="challenges-and-opportunities-for-bio-image-analysis-core-facilities">
<h2>Challenges and opportunities for bio-image analysis core-facilities<a class="headerlink" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/12-1054">https://f1000research.com/slides/12-1054</a></p>
</section>
<hr class="docutils" />
<section id="crashkurs-forschungsdatenmanagement">
<h2>Crashkurs Forschungsdatenmanagement<a class="headerlink" href="#crashkurs-forschungsdatenmanagement" title="Link to this heading">#</a></h2>
<p>Barbara Weiner, Stephan Wünsche, Stefan Kühne, Pia Voigt, Sebastian Frericks, Clemens Hoffmann, Romy Elze, Ronny Gey</p>
<p>Published 2020-04-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Diese Präsentation bietet einen Einstieg in alle relevanten Bereiche des Forschungsdatenmanagements an der Universität Leipzig. Behandelt werden Grundlagen des Forschungsdatenmanagements, technische, ethische und rechtliche Aspekte sowie die Archivierung und Publikation von Forschungsdaten. Die Präsentation enthält zahlreiche weiterführende Links (rot) und Literaturhinweise.</p>
<p>Ergänzend hierzu wird eine Präsentation mit Übungsaufgaben angeboten, die helfen soll, das Gelernte zu festigen und in der eigenen Forschungspraxis umzusetzen. Den Aufgaben folgen jeweils eine Antwortfolie sowie deren Auflösung.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/3778431">https://zenodo.org/records/3778431</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.3778431">https://doi.org/10.5281/zenodo.3778431</a></p>
</section>
<hr class="docutils" />
<section id="creating-open-computational-curricula">
<h2>Creating open computational curricula<a class="headerlink" href="#creating-open-computational-curricula" title="Link to this heading">#</a></h2>
<p>Kari Jordan, Zhian Kamvar, Toby Hodges</p>
<p>Published 2020-12-11</p>
<p>Licensed CC-BY-4.0</p>
<p>In this interactive session, Carpentries team members will guide attendees through three stages of the backward design process to create a lesson development plan for the open source tool of their choosing. Attendees will leave having identified what practical skills they aim to teach (learning objectives), an approach for designing challenge questions (formative assessment), and mechanisms to give and receive feedback.</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4317149">https://zenodo.org/records/4317149</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4317149">https://doi.org/10.5281/zenodo.4317149</a></p>
</section>
<hr class="docutils" />
<section id="cultivating-open-training">
<h2>Cultivating Open Training<a class="headerlink" href="#cultivating-open-training" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-03-14</p>
<p>Licensed CC-BY-4.0</p>
<p>In this SaxFDM Digital Kitchen, I introduced current challenges and potential solutions for openly sharing training materials, softly focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and uploading materials to publicly acessible repositories are proposed to fill this gap.</p>
<p>Tags: Open Science, Research Data Management, FAIR-Principles, Bioimage Analysis, Licensing</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10816895">https://zenodo.org/records/10816895</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10816895">https://doi.org/10.5281/zenodo.10816895</a></p>
</section>
<hr class="docutils" />
<section id="cultivating-open-training-to-advance-bio-image-analysis">
<h2>Cultivating Open Training to advance Bio-image Analysis<a class="headerlink" href="#cultivating-open-training-to-advance-bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-04-25</p>
<p>Licensed CC-BY-4.0</p>
<p>These slides introduce current challenges and potential solutions for openly sharing training materials, focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and publicly acessible repositories are proposed to fill this gap.</p>
<p>Tags: Research Data Management, Licensing, FAIR-Principles</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11066250">https://zenodo.org/records/11066250</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11066250">https://doi.org/10.5281/zenodo.11066250</a></p>
</section>
<hr class="docutils" />
<section id="dl-mbl-2021-exercises">
<h2>DL&#64;MBL 2021 Exercises<a class="headerlink" href="#dl-mbl-2021-exercises" title="Link to this heading">#</a></h2>
<p>Jan Funke, Constantin Pape, Morgan Schwartz, Xiaoyan</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/JLrumberger/DL-MBL-2021">JLrumberger/DL-MBL-2021</a></p>
</section>
<hr class="docutils" />
<section id="data-management-at-france-bioimaging">
<h2>Data management at France BioImaging<a class="headerlink" href="#data-management-at-france-bioimaging" title="Link to this heading">#</a></h2>
<p>Published 2023-07-05</p>
<p>Licensed CC-BY-SA-4.0</p>
<p>Tags: Research Data Management, Bioimage Analysis, Open Science</p>
<p>Content type: Slides, Presentation</p>
<p><a class="reference external" href="https://omero-fbi.fr/slides/elmi23_cfd/main.html#/title-slide">https://omero-fbi.fr/slides/elmi23_cfd/main.html#/title-slide</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagement">
<h2>Datenmanagement<a class="headerlink" href="#datenmanagement" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-04-14</p>
<p>Licensed CC-BY-4.0</p>
<p>In dieser Data Management Session wird der Lebenszyklus von Daten näher beleuchtet. Wie entstehen Daten, was passiert mit ihnen, wenn sie verarbeitet werden? Wem gehören die Daten und wer ist dafür verantwortlich, sie zu veröffentlichen, zu archivieren und gegebenenfalls wiederzuverwenden? Wir werden einen Datenmanagementplan in Gruppenarbeit entwerfen, ggf. mit Hilfe von ChatGPT.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10970869">https://zenodo.org/records/10970869</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10970869">https://doi.org/10.5281/zenodo.10970869</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz">
<h2>Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz<a class="headerlink" href="#datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz" title="Link to this heading">#</a></h2>
<p>Pia Voigt, Carolin Hundt</p>
<p>Published 2024-04-19</p>
<p>Licensed CC-BY-4.0</p>
<p>Workshop zum Thema „Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz“ auf der Data Week Leipzig
Der Umgang mit Daten ist im Alltag nicht immer leicht: Wie und wo speichert man Daten idealerweise? Welche Strategien helfen, den Überblick zu behalten und wie geht man mit personenbezogenen Daten um? Diese Fragen möchten wir gemeinsam mit Ihnen anhand individueller Datenprobleme besprechen und Ihnen Lösungen aufzeigen, wie Sie ihr Datenmanagement effizient gestalten können.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11107798">https://zenodo.org/records/11107798</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11107798">https://doi.org/10.5281/zenodo.11107798</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagementplane-erstellen-teil-1">
<h2>Datenmanagementpläne erstellen - Teil 1<a class="headerlink" href="#datenmanagementplane-erstellen-teil-1" title="Link to this heading">#</a></h2>
<p>Pia Voigt, Barbara Weiner</p>
<p>Published 2021-03-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Was ist ein Datenmanagementplan? Welche Vorgaben sollte ich beachten? Wie erstelle ich einen solchen für mein Forschungsprojekt und welche nützlichen Tools kann ich hierfür verwenden?</p>
<p>Die Anforderungen der Forschungsförderer zum Datenmanagement steigen stetig. Damit verbunden ist häufig auch das Erstellen eines Datenmanagementplans. Dabei erwarten DFG, BMBF oder die EU jeweils unterschiedliche Angaben zur Erhebung, Speicherung und Veröffentlichung von projektbezogenen Forschungsdaten. Zudem bietet das Erstellen eines Datenmanagementplans viele Vorteile und hilft Ihnen nicht zuletzt, die Anforderungen der guten wissenschaftlichen Praxis strukturiert umzusetzen.</p>
<p>Was im ersten Moment unübersichtlich und überfordernd wirkt, soll in diesem Kurs anhand einer grundlegenden theoretischen Einführung im ersten und praxisorientierter Beispiele im zweiten Teil der Veranstaltung handhabbar gemacht werden. Sie lernen, was hinter den Anforderungen der Forschungsförderer steckt, welche Elemente ein Datenmanagementplan enthalten sollte und wie sie einen solchen mithilfe interaktiver Tools selbst erstellen können.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4630788">https://zenodo.org/records/4630788</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4630788">https://doi.org/10.5281/zenodo.4630788</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagementplane-erstellen-teil-2">
<h2>Datenmanagementpläne erstellen - Teil 2<a class="headerlink" href="#datenmanagementplane-erstellen-teil-2" title="Link to this heading">#</a></h2>
<p>Pia Voigt, Barbara Weiner</p>
<p>Published 2021-03-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Was ist ein Datenmanagementplan? Welche Vorgaben sollte ich beachten? Wie erstelle ich einen solchen für mein Forschungsprojekt und welche nützlichen Tools kann ich hierfür verwenden?</p>
<p>Die Anforderungen der Forschungsförderer zum Datenmanagement steigen stetig. Damit verbunden ist häufig auch das Erstellen eines Datenmanagementplans. Dabei erwarten DFG, BMBF oder die EU jeweils unterschiedliche Angaben zur Erhebung, Speicherung und Veröffentlichung von projektbezogenen Forschungsdaten. Zudem bietet das Erstellen eines Datenmanagementplans viele Vorteile und hilft Ihnen nicht zuletzt, die Anforderungen der guten wissenschaftlichen Praxis strukturiert umzusetzen.</p>
<p>Was im ersten Moment unübersichtlich und überfordernd wirkt, soll in diesem Kurs anhand einer grundlegenden theoretischen Einführung im ersten und praxisorientierter Beispiele im zweiten Teil der Veranstaltung handhabbar gemacht werden. Sie lernen, was hinter den Anforderungen der Forschungsförderer steckt, welche Elemente ein Datenmanagementplan enthalten sollte und wie sie einen solchen mithilfe interaktiver Tools selbst erstellen können.</p>
<p>Version 2 enthält aktuelle Links und weiterführende Hinweise zu einzelnen Aspekten eines Datenmanagementplans.</p>
<p>Version 3 ist die überarbeitete und aktualisierte Version der ersten beiden und enthält u.a. Hinweise zur Lizenzierung und zu Nutzungsrechten an Forschungsdaten.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4748534">https://zenodo.org/records/4748534</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4748534">https://doi.org/10.5281/zenodo.4748534</a></p>
</section>
<hr class="docutils" />
<section id="dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy">
<h2>Dr Guillaume Jacquemet on studying cancer cell metastasis in the era of deep learning for microscopy<a class="headerlink" href="#dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy" title="Link to this heading">#</a></h2>
<p>Guillaume Jacquemet</p>
<p>Published 2024-10-24</p>
<p>Licensed UNKNOWN</p>
<p>Leukocyte extravasation is a critical component of the innate immune response, while circulating tumour cell extravasation is a crucial step in metastasis formation. Despite their importance, these extravasation mechanisms remain incompletely understood. In this talk, Guillaume Jacquemet presents a novel imaging framework that integrates microfluidics with high-speed, label-free imaging to study the arrest of pancreatic cancer cells (PDAC) on human endothelial layers under physiological flow conditions.</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Video, Slides</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=KTdZBgSCYJQ">https://www.youtube.com/watch?v=KTdZBgSCYJQ</a></p>
</section>
<hr class="docutils" />
<section id="einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten">
<h2>Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten<a class="headerlink" href="#einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten" title="Link to this heading">#</a></h2>
<p>Stephan Wünsche, Pia Voigt</p>
<p>Published 2021-05-11</p>
<p>Licensed CC-BY-4.0</p>
<p>Diese Präsentation wurde im Zuge der digitalen Veranstaltungsreihe “Einblicke ins Forschungsdatenmanagement” erstellt. Diese findet seit dem SS 2020 an der Universität Leipzig für alle Interessierten zu verschiedenen Themen des Forschungsdatenmanagements statt.</p>
<p>Dieser Teil der Reihe dreht sich um Rechtsfragen im Umgang mit Forschungsdaten und deren Bedeutung für die wissenschaftliche Praxis. Sie finden in der vorliegenden Präsentation einen Überblick über relevante Rechtsbereiche sowie Erläuterungen zum Datenschutz, Urheberrecht und den Grundsätzen der guten wissenschaftlichen Praxis mit Fokus auf deren Bedeutung im Forschungsdatenmanagement.</p>
<p>Tags: Research Data Management, Data Protection</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4748510">https://zenodo.org/records/4748510</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4748510">https://doi.org/10.5281/zenodo.4748510</a></p>
</section>
<hr class="docutils" />
<section id="euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">
<h2>Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks<a class="headerlink" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks" title="Link to this heading">#</a></h2>
<p>Isabel Kemmer, Euro-BioImaging ERIC</p>
<p>Published 2024-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Hands-on exercises on FAIR Bioimage Data from the interactive online workshop “Euro-BioImaging’s Guide to FAIR BioImage Data 2024” (<a class="reference external" href="https://www.eurobioimaging.eu/news/a-guide-to-fair-bioimage-data-2024/">https://www.eurobioimaging.eu/news/a-guide-to-fair-bioimage-data-2024/</a>).  Types of tasks included: FAIR characteristics of a real world dataset Data Management Plan (DMP) Journal Policies on FAIR data sharing Ontology search Metadata according to REMBI scheme (Image from: Sarkans, U., Chiu, W., Collinson, L. et al. REMBI: Recommended Metadata for Biological Images—enabling reuse of microscopy data in biology. Nat Methods 18, 1418–1422 (2021). <a class="reference external" href="https://doi.org/10.1038/s41592-021-01166-8">https://doi.org/10.1038/s41592-021-01166-8</a>) Matching datasets to bioimage repositories Browsing bioimage repositories</p>
<p>Tags: Bioimage Analysis, FAIR-Principles, Research Data Management</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="reference external" href="https://zenodo.org/records/11474407">https://zenodo.org/records/11474407</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11474407">https://doi.org/10.5281/zenodo.11474407</a></p>
</section>
<hr class="docutils" />
<section id="fairy-deep-learning-for-bioimage-analysis">
<h2>FAIRy deep-learning for bioImage analysis<a class="headerlink" href="#fairy-deep-learning-for-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez de Mariscal</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to FAIR deep learning. Furthermore, tools to deploy trained DL models (deepImageJ), easily train and evaluate them (ZeroCostDL4Mic and DeepBacs) ensure reproducibility (DL4MicEverywhere), and share this technology in an open-source and reproducible manner (BioImage Model Zoo) are introduced.</p>
<p>Tags: Artificial Intelligence, FAIR-Principles, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/13-147">https://f1000research.com/slides/13-147</a></p>
</section>
<hr class="docutils" />
<section id="from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers">
<h2>From Paper to Pixels: Navigation through your Research Data - presentations of speakers<a class="headerlink" href="#from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers" title="Link to this heading">#</a></h2>
<p>Marcelo Zoccoler, Simon Bekemeier, Tom Boissonnet, Simon Parker, Luca Bertinetti, Marc Gentzel, Riccardo Massei, Cornelia Wetzker</p>
<p>Published 2024-06-10</p>
<p>Licensed CC-BY-4.0</p>
<p>The workshop introduced key topics of research data management (RDM) and the implementation thereof on a life science campus. Internal and external experts of RDM including scientists that apply chosen software tools presented the basic concepts and their implementation to a broad audience. 
Talks covered general aspects of data handling and sorting, naming conventions, data storage repositories and archives, licensing of material, data and code management using git, data protection particularly regarding patient data and in genome sequencing and more. Two data management concepts and exemplary tools were highlighted in particular, being electronic lab notebooks with eLabFTW and the bio-image management software OMERO. Those were chosen because of three aspects: the large benefit of these management tools for a life science campus, their free availability as open source tools with the option of contribution of required functionalities and first existing use cases on campus already supported by CMCB/PoL IT.
Two talks by Robert Haase (<a class="reference external" href="http://ScaDS.AI/">ScaDS.AI/</a> Uni Leipzig) and Robert Müller (Kontaktstelle Forschungsdaten, TU Dresden with contributions from Denise Dörfel) that opened the symposium were shared independently:
<a class="reference external" href="https://zenodo.org/records/11382341">https://zenodo.org/records/11382341</a>
<a class="reference external" href="https://zenodo.org/records/11261115">https://zenodo.org/records/11261115</a>
The workshop organization was funded by the CMCB/PoL Networking Grant and supported by the consortium NFDI4BIOIMAGE (funded by DFG grant number NFDI 46/1, project number 501864659).</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11548617">https://zenodo.org/records/11548617</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11548617">https://doi.org/10.5281/zenodo.11548617</a></p>
</section>
<hr class="docutils" />
<section id="galaxy-training-material">
<h2>Galaxy Training Material<a class="headerlink" href="#galaxy-training-material" title="Link to this heading">#</a></h2>
<p>Licensed MIT</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="github reference external" href="https://github.com/galaxyproject/training-material">galaxyproject/training-material</a></p>
</section>
<hr class="docutils" />
<section id="generative-artificial-intelligence-for-bio-image-analysis">
<h2>Generative artificial intelligence for bio-image analysis<a class="headerlink" href="#generative-artificial-intelligence-for-bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/12-971">https://f1000research.com/slides/12-971</a></p>
</section>
<hr class="docutils" />
<section id="hackaton-results-conversion-of-knime-image-analysis-workflows-to-galaxy">
<h2>Hackaton Results - Conversion of KNIME image analysis workflows to Galaxy<a class="headerlink" href="#hackaton-results-conversion-of-knime-image-analysis-workflows-to-galaxy" title="Link to this heading">#</a></h2>
<p>Riccardo Massei</p>
<p>Licensed CC-BY-4.0</p>
<p>Results of the project ‘Conversion of KNIME image analysis workflows to Galaxy’ during the Hackathon ‘Image Analysis in Galaxy’ (Freiburg 26 Feb - 01 Mar 2024)</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10793699">https://zenodo.org/doi/10.5281/zenodo.10793699</a></p>
</section>
<hr class="docutils" />
<section id="hitchhiking-through-a-diverse-bio-image-analysis-software-universe">
<h2>Hitchhiking through a diverse Bio-image Analysis Software Universe<a class="headerlink" href="#hitchhiking-through-a-diverse-bio-image-analysis-software-universe" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2022-07-22</p>
<p>Licensed CC-BY-4.0</p>
<p>Overview about decision making and how to influence decisions in the bio-image analysis software context.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Slides, Presentation</p>
<p><a class="reference external" href="https://f1000research.com/slides/11-746">https://f1000research.com/slides/11-746</a></p>
<p><a class="reference external" href="https://doi.org/10.7490/f1000research.1119026.1">https://doi.org/10.7490/f1000research.1119026.1</a></p>
</section>
<hr class="docutils" />
<section id="i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">
<h2>I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training<a class="headerlink" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Michele Bortolomeazzi, Tom Boissonnet, Carsten Fortmann-Grote, Julia Dohle, Peter Zentis, Niraj Kandpal, Susanne Kunis, Thomas Zobel, Stefanie Weidtkamp-Peters, Elisa Ferrando-May</p>
<p>Published 2023-11-13</p>
<p>Licensed CC-BY-4.0</p>
<p>The open-source software OME Remote Objects (OMERO) is a data management software that allows storing, organizing, and annotating bioimaging/microscopy data. OMERO has become one of the best-known systems for bioimage data management in the bioimaging community. The Information Infrastructure for BioImage Data (I3D:bio) project facilitates the uptake of OMERO into research data management (RDM) practices at universities and research institutions in Germany. Since the adoption of OMERO into researchers’ daily routines requires intensive training, a broad portfolio of training resources for OMERO is an asset. On top of using the OMERO guides curated by the Open Microscopy Environment Consortium (OME) team, imaging core facility staff at institutions where OMERO is used often prepare additional material tailored to be applicable for their own OMERO instances. Based on experience gathered in the Research Data Management for Microscopy group (RDM4mic) in Germany, and in the use cases in the I3D:bio project, we created a set of reusable, adjustable, openly available slide decks to serve as the basis for tailored training lectures, video tutorials, and self-guided instruction manuals directed at beginners in using OMERO. The material is published as an open educational resource complementing the existing resources for OMERO contributed by the community.</p>
<p>Tags: OMERO, Research Data Management, Nfdi4Bioimage, I3Dbio</p>
<p>Content type: Slides, Video</p>
<p><a class="reference external" href="https://zenodo.org/records/8323588">https://zenodo.org/records/8323588</a></p>
<p><a class="reference external" href="https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU">https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8323588">https://doi.org/10.5281/zenodo.8323588</a></p>
</section>
<hr class="docutils" />
<section id="image-data-services-at-euro-bioimaging-community-efforts-towards-fair-image-data-and-analysis-services">
<h2>Image Data Services at Euro-BioImaging: Community efforts towards FAIR Image Data and Analysis Services<a class="headerlink" href="#image-data-services-at-euro-bioimaging-community-efforts-towards-fair-image-data-and-analysis-services" title="Link to this heading">#</a></h2>
<p>Aastha Mathur</p>
<p>Licensed UNKNOWN</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1henPIDTpHT3bc1Y26AltItAHJ2C5xCOl/edit#slide=id.p1">https://docs.google.com/presentation/d/1henPIDTpHT3bc1Y26AltItAHJ2C5xCOl/edit#slide=id.p1</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-in-galaxy">
<h2>Image analysis in Galaxy<a class="headerlink" href="#image-analysis-in-galaxy" title="Link to this heading">#</a></h2>
<p>Beatriz Serrano-Solano, Björn Grüning</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1WG_4307XmKsGfWT3taxMvX2rZiG1k0SM1E7SAENJQkI/edit#slide=id.p">https://docs.google.com/presentation/d/1WG_4307XmKsGfWT3taxMvX2rZiG1k0SM1E7SAENJQkI/edit#slide=id.p</a></p>
</section>
<hr class="docutils" />
<section id="imagej-macro-introduction">
<h2>ImageJ Macro Introduction<a class="headerlink" href="#imagej-macro-introduction" title="Link to this heading">#</a></h2>
<p>Anna Klemm</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Imagej Macro, Bioimage Analysis</p>
<p>Content type: Slides, Code</p>
<p><a class="github reference external" href="https://github.com/ahklemm/ImageJMacro_Introduction">ahklemm/ImageJMacro_Introduction</a></p>
</section>
<hr class="docutils" />
<section id="imagej2-api-beating">
<h2>ImageJ2 API-beating<a class="headerlink" href="#imagej2-api-beating" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_imagej2_dev">https://git.mpi-cbg.de/rhaase/lecture_imagej2_dev</a></p>
</section>
<hr class="docutils" />
<section id="intro-napari-slides">
<h2>Intro napari slides<a class="headerlink" href="#intro-napari-slides" title="Link to this heading">#</a></h2>
<p>Peter Sobolewski</p>
<p>Licensed MIT</p>
<p>Introduction to napari workshop run at JAX (Spring 2024).</p>
<p>Tags: Napari</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://thejacksonlaboratory.github.io/intro-napari-slides/#/section">https://thejacksonlaboratory.github.io/intro-napari-slides/#/section</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden">
<h2>Introduction to ImageJ macro programming, Scientific Computing Facility, MPI CBG Dresden<a class="headerlink" href="#introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden" title="Link to this heading">#</a></h2>
<p>Robert Haase, Benoit Lombardot</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/scicomp/bioimage_team/coursematerialimageanalysis/tree/master/ImageJMacro_24h_2017-01">https://git.mpi-cbg.de/scicomp/bioimage_team/coursematerialimageanalysis/tree/master/ImageJMacro_24h_2017-01</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-research-data-management-and-open-research">
<h2>Introduction to Research Data Management and Open Research<a class="headerlink" href="#introduction-to-research-data-management-and-open-research" title="Link to this heading">#</a></h2>
<p>Shanmugasundaram</p>
<p>Published 2024-05-17</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to RDM primarily for researchers. Can be seen as primer to all other materials in this catalogue.</p>
<p>Tags: Research Data Management, Open Science</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4778265">https://zenodo.org/records/4778265</a></p>
</section>
<hr class="docutils" />
<section id="jupyter-for-interactive-cloud-computing">
<h2>Jupyter for interactive cloud computing<a class="headerlink" href="#jupyter-for-interactive-cloud-computing" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1q8q1xE-c35tvCsRXZay98s2UYWwXpp0cfCljBmMFpco/edit#slide=id.ga456d5535c_2_53">https://docs.google.com/presentation/d/1q8q1xE-c35tvCsRXZay98s2UYWwXpp0cfCljBmMFpco/edit#slide=id.ga456d5535c_2_53</a></p>
</section>
<hr class="docutils" />
<section id="kollaboratives-arbeiten-und-versionskontrolle-mit-git">
<h2>Kollaboratives Arbeiten und Versionskontrolle mit Git<a class="headerlink" href="#kollaboratives-arbeiten-und-versionskontrolle-mit-git" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-04-15</p>
<p>Licensed CC-BY-4.0</p>
<p>Gemeinsames Arbeiten im Internet stellt uns vor neue Herausforderungen: Wer hat eine Datei wann hochgeladen? Wer hat zum Inhalt beigetragen? Wie kann man Inhalte zusammenfuehren, wenn mehrere Mitarbeiter gleichzeitig Aenderungen gemacht haben? Das Versionskontrollwerkzeug git stellt eine umfassende Loesung fuer solche Fragen bereit. Die Onlineplatform <a class="reference external" href="http://github.com">github.com</a> stellt nicht nur Softwareentwicklern weltweit eine git-getriebene Platform zur Verfuegung und erlaubt ihnen effektiv zusammen zu arbeiten. In diesem Workshop lernen wir:</p>
<p>Infuerung in FAIR-Prinzipien im Softwarecontext
Arbeiten mit git: Pull-requests
Aufloesen von Merge-Konflikten
Automatisiertes Archivieren von Inhalten nach <a class="reference external" href="http://Zenodo.org">Zenodo.org</a>
Eigene Webseiten auf <a class="reference external" href="http://github.io">github.io</a> publizieren</p>
<p>Tags: Research Data Management, FAIR-Principles, Git, Zenodo</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10972692">https://zenodo.org/records/10972692</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10972692">https://doi.org/10.5281/zenodo.10972692</a></p>
</section>
<hr class="docutils" />
<section id="lecture-applied-bioimage-analysis-2020">
<h2>Lecture Applied Bioimage Analysis 2020<a class="headerlink" href="#lecture-applied-bioimage-analysis-2020" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Slides, scripts, data and other exercise materials of the BioImage Analysis lecture at CMCB TU Dresden 2020</p>
<p>Tags: Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_applied_bioimage_analysis">https://git.mpi-cbg.de/rhaase/lecture_applied_bioimage_analysis</a></p>
</section>
<hr class="docutils" />
<section id="lecture-materials-of-the-deeplife-course">
<h2>Lecture-materials of the DeepLife course<a class="headerlink" href="#lecture-materials-of-the-deeplife-course" title="Link to this heading">#</a></h2>
<p>Carl Herrmann, annavonbachmann, David Hoksza, Martin Schätz, Dario Malchiodi, jnguyenvan, Britta Velten, Elodie Laine, JanaBraunger, barwil</p>
<p>Published 2023-12-06</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioinformatics</p>
<p>Content type: Github Repository, Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/deeplife4eu/Lecture-materials/">deeplife4eu/Lecture-materials</a></p>
</section>
<hr class="docutils" />
<section id="machine-learning-deep-learning-applications-to-bioimage-analysis">
<h2>Machine Learning - Deep Learning. Applications to Bioimage Analysis<a class="headerlink" href="#machine-learning-deep-learning-applications-to-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://raw.githubusercontent.com/esgomezm/esgomezm.github.io/master/assets/pdf/SPAOM2018/MachineLearning_SPAOMworkshop_public.pdf">https://raw.githubusercontent.com/esgomezm/esgomezm.github.io/master/assets/pdf/SPAOM2018/MachineLearning_SPAOMworkshop_public.pdf</a></p>
</section>
<hr class="docutils" />
<section id="machine-and-deep-learning-on-the-cloud-segmentation">
<h2>Machine and Deep Learning on the cloud: Segmentation<a class="headerlink" href="#machine-and-deep-learning-on-the-cloud-segmentation" title="Link to this heading">#</a></h2>
<p>Ignacio Arganda-Carreras</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1oJoy9gHmUuSmUwCkPs_InJf_WZAzmLlUNvK1FUEB4PA/edit#slide=id.ge3a24e733b_0_54">https://docs.google.com/presentation/d/1oJoy9gHmUuSmUwCkPs_InJf_WZAzmLlUNvK1FUEB4PA/edit#slide=id.ge3a24e733b_0_54</a></p>
</section>
<hr class="docutils" />
<section id="methods-in-bioimage-analysis">
<h2>Methods in bioimage analysis<a class="headerlink" href="#methods-in-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Christian Tischer</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Online Tutorial, Video, Slides</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/events/methods-bioimage-analysis/">https://www.ebi.ac.uk/training/events/methods-bioimage-analysis/</a></p>
<p><a class="reference external" href="https://doi.org/10.6019/TOL.BioImageAnalysis22-w.2022.00001.1">https://doi.org/10.6019/TOL.BioImageAnalysis22-w.2022.00001.1</a></p>
<p><a class="reference external" href="https://drive.google.com/file/d/1MhuqfKhZcYu3bchWMqogIybKjamU5Msg/view">https://drive.google.com/file/d/1MhuqfKhZcYu3bchWMqogIybKjamU5Msg/view</a></p>
</section>
<hr class="docutils" />
<section id="microsam-talks">
<h2>MicroSam-Talks<a class="headerlink" href="#microsam-talks" title="Link to this heading">#</a></h2>
<p>Constantin Pape</p>
<p>Published 2024-05-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Talks about Segment Anything for Microscopy: <a class="github reference external" href="https://github.com/computational-cell-analytics/micro-sam">computational-cell-analytics/micro-sam</a>.
Currently contains slides for two talks:</p>
<p>Overview of Segment Anythign for Microscopy given at the SWISSBIAS online meeting in April 2024
Talk about vision foundation models and Segment Anything for Microscopy given at Human Technopole as part of the EMBO Deep Learning Course in May 2024</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11265038">https://zenodo.org/records/11265038</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11265038">https://doi.org/10.5281/zenodo.11265038</a></p>
</section>
<hr class="docutils" />
<section id="microscopy-data-analysis-machine-learning-and-the-bioimage-archive">
<h2>Microscopy data analysis: machine learning and the BioImage Archive<a class="headerlink" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive" title="Link to this heading">#</a></h2>
<p>Andrii Iudin, Anna Foix-Romero, Anna Kreshuk, Awais Athar, Beth Cimini, Dominik Kutra, Estibalis Gomez de Mariscal, Frances Wong, Guillaume Jacquemet, Kedar Narayan, Martin Weigert, Nodar Gogoberidze, Osman Salih, Petr Walczysko, Ryan Conrad, Simone Weyend, Sriram Sundar Somasundharam, Suganya Sivagurunathan, Ugis Sarkans</p>
<p>Licensed CC-BY-4.0</p>
<p>The Microscopy data analysis: machine learning and the BioImage Archive course, which focused on introducing programmatic approaches used in the analysis of bioimage data via the BioImage Archive, ran in May 2023.</p>
<p>Tags: Bioimage Analysis, Python, Artificial Intelligence</p>
<p>Content type: Video, Slides</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/materials/microscopy-data-analysis-machine-learning-and-the-bioimage-archive-materials/">https://www.ebi.ac.uk/training/materials/microscopy-data-analysis-machine-learning-and-the-bioimage-archive-materials/</a></p>
</section>
<hr class="docutils" />
<section id="multi-view-fusion">
<h2>Multi-view fusion<a class="headerlink" href="#multi-view-fusion" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Lecture slides of a session on Multiview Fusion in Fiji</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_multiview_registration">https://git.mpi-cbg.de/rhaase/lecture_multiview_registration</a></p>
</section>
<hr class="docutils" />
<section id="multiplexed-tissue-imaging-tools-and-approaches">
<h2>Multiplexed tissue imaging - tools and approaches<a class="headerlink" href="#multiplexed-tissue-imaging-tools-and-approaches" title="Link to this heading">#</a></h2>
<p>Agustín Andrés Corbat, OmFrederic, Jonas Windhager, Kristína Lidayová</p>
<p>Licensed CC-BY-4.0</p>
<p>Material for the I2K 2024 “Multiplexed tissue imaging - tools and approaches” workshop</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Slides, Workshop</p>
<p><a class="github reference external" href="https://github.com/BIIFSweden/I2K2024-MTIWorkshop">BIIFSweden/I2K2024-MTIWorkshop</a></p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1">https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1</a></p>
</section>
<hr class="docutils" />
<section id="my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud">
<h2>My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud<a class="headerlink" href="#my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud" title="Link to this heading">#</a></h2>
<p>Elnaz Fazeli</p>
<p>Published 2024-02-19</p>
<p>Licensed CC-BY-4.0</p>
<p>In these slides I introducemy journey through teaching bioimage analysis courses in different formats, from in person courses to online material. I have an overview of different training formats and comparing these for different audiences. </p>
<p>Tags: Teaching</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10679054">https://zenodo.org/records/10679054</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10679054">https://doi.org/10.5281/zenodo.10679054</a></p>
</section>
<hr class="docutils" />
<section id="neubias-analyst-school-2018">
<h2>NEUBIAS Analyst School 2018<a class="headerlink" href="#neubias-analyst-school-2018" title="Link to this heading">#</a></h2>
<p>Assaf Zaritsky, Csaba Molnar, Vasja Urbancic, Richard Butler, Anna Kreshuk, Vannary Meas-Yedid</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2018">miura/NEUBIAS_AnalystSchool2018</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-course-2017">
<h2>NEUBIAS Bioimage Analyst Course 2017<a class="headerlink" href="#neubias-bioimage-analyst-course-2017" title="Link to this heading">#</a></h2>
<p>Curtis Rueden, Florian Levet, J.B. Sibarta, Alexandre Dafour, Daniel Sage, Sebastien Tosi, Michal Kozubek, Jean-Yves Tinevez, Kota Miura, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_Bioimage_Analyst_Course2017">miura/NEUBIAS_Bioimage_Analyst_Course2017</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-school-2019">
<h2>NEUBIAS Bioimage Analyst School 2019<a class="headerlink" href="#neubias-bioimage-analyst-school-2019" title="Link to this heading">#</a></h2>
<p>Kota Miura, Chong Zhang, Jean-Yves Tinevez, Robert Haase, Julius Hossein, Pejamn Rasti, David Rousseau, Ignacio Arganda-Carreras, Siân Culley, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2019">miura/NEUBIAS_AnalystSchool2019</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-school-2020">
<h2>NEUBIAS Bioimage Analyst School 2020<a class="headerlink" href="#neubias-bioimage-analyst-school-2020" title="Link to this heading">#</a></h2>
<p>Marion Louveaux, Stéphane Verger, Arianne Bercowsky Rama, Ignacio Arganda-Carreras, Estibaliz Gómez-de-Mariscal, Kota Miura, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2020">miura/NEUBIAS_AnalystSchool2020</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage">
<h2>NFDI4BIOIMAGE<a class="headerlink" href="#nfdi4bioimage" title="Link to this heading">#</a></h2>
<p>Carsten Fortmann-Grote</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation was given at the 2nd MPG-NFDI Workshop on April 18th about e NFDI4BIOIMAGE Consortium, FAIRification of Image (meta)data, Zarr, RFC, Training (TA5), contributing.</p>
<p>Tags: Research Data Management, Bioimage Analysis, FAIR-Principles, Zarr, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.11031746">https://zenodo.org/doi/10.5281/zenodo.11031746</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data">
<h2>NFDI4BIOIMAGE - An Initiative for a National Research Data Infrastructure for Microscopy Data<a class="headerlink" href="#nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Elisa Ferrando-May</p>
<p>Published 2021-04-29</p>
<p>Licensed CCY-BY-SA-4.0</p>
<p>Align existing and establish novel services &amp; solutions for data management tasks throughout the bioimage data lifecycle.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Conference Abstract, Slides</p>
<p><a class="reference external" href="https://doi.org/10.11588/heidok.00029489">https://doi.org/10.11588/heidok.00029489</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters</p>
<p>Licensed CC-BY-4.0</p>
<p>NFDI4BIOIMAGE core mission, bioimage data challenge, task areas, FAIR bioimage workflows.</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8070038">https://doi.org/10.5281/zenodo.8070038</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg" title="Link to this heading">#</a></h2>
<p>Riccardo Massei</p>
<p>Licensed CC-BY-4.0</p>
<p>NFDI4BIOIMAGE is a consortium within the framework of the National Research Data Infrastructure (NFDI) in Germany. In this talk, the consortium and the contribution to the work programme by the Helmholtz Centre for Environmental Research (UFZ) in Leipzig are outlined.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8414318">https://zenodo.org/doi/10.5281/zenodo.8414318</a></p>
</section>
<hr class="docutils" />
<section id="neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">
<h2>Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist<a class="headerlink" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Olivier Burri, Siân Culley, Uwe Schmidt</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Neubias, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/neubias_academy_stardist">maweigert/neubias_academy_stardist</a></p>
</section>
<hr class="docutils" />
<section id="nextflow-scalable-and-reproducible-scientific-workflows">
<h2>Nextflow: Scalable and reproducible scientific workflows<a class="headerlink" href="#nextflow-scalable-and-reproducible-scientific-workflows" title="Link to this heading">#</a></h2>
<p>Floden Evan, Di Tommaso Paolo</p>
<p>Published 2020-12-17</p>
<p>Licensed CC-BY-4.0</p>
<p>Nextflow is an open-source workflow management system that prioritizes portability and reproducibility. It enables users to develop and seamlessly scale genomics workflows locally, on HPC clusters, or in major cloud providers’ infrastructures. Developed since 2014 and backed by a fast-growing community, the Nextflow ecosystem is made up of users and developers across academia, government and industry. It counts over 1M downloads and over 10K users worldwide.</p>
<p>Tags: Workflow Engine</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4334697">https://zenodo.org/records/4334697</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4334697">https://doi.org/10.5281/zenodo.4334697</a></p>
</section>
<hr class="docutils" />
<section id="object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">
<h2>Object Tracking and Track Analysis using TrackMate and CellTracksColab<a class="headerlink" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab" title="Link to this heading">#</a></h2>
<p>Joanna Pylvänäinen</p>
<p>Published None</p>
<p>Licensed GPL-3.0</p>
<p>I2K 2024 workshop materials for “Object Tracking and Track Analysis using TrackMate and CellTracksColab”</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Tutorial, Workshop, Slides</p>
<p><a class="github reference external" href="https://github.com/CellMigrationLab/I2K_2024">CellMigrationLab/I2K_2024</a></p>
</section>
<hr class="docutils" />
<section id="open-science-sharing-licensing">
<h2>Open Science, Sharing &amp; Licensing<a class="headerlink" href="#open-science-sharing-licensing" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-04-18</p>
<p>Licensed CC-BY-4.0</p>
<p>Wir tauchen ein in die Welt der Open Science und definieren Begriffe wie Open Source, Open Access und die FAIR-Prinzipien (Findable, Accessible, Interoperable and Reuasable). Wir diskutieren, wie diese Methoden der [wissenschaftlichen] Kommunikation und des Datenmanagements die Welt verändern und wie wir sie praktisch in unsere Arbeit integrieren können. Dabei spielen Aspekte wie Copyright und Lizenzierung eine wichtige Rolle.</p>
<p>Tags: Research Data Management, Open Access, FAIR-Principles, Licensing</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10990107">https://zenodo.org/records/10990107</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10990107">https://doi.org/10.5281/zenodo.10990107</a></p>
</section>
<hr class="docutils" />
<section id="parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing">
<h2>Parallelization and heterogeneous computing: from pure CPU to GPU-accelerated image processing<a class="headerlink" href="#parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/11-1171">https://f1000research.com/slides/11-1171</a></p>
<p><a class="reference external" href="https://doi.org/10.7490/f1000research.1119154.1">https://doi.org/10.7490/f1000research.1119154.1</a></p>
</section>
<hr class="docutils" />
<section id="promoting-reproducibility-in-biomedical-research-through-image-analysis">
<h2>Promoting Reproducibility in Biomedical Research through Image Analysis<a class="headerlink" href="#promoting-reproducibility-in-biomedical-research-through-image-analysis" title="Link to this heading">#</a></h2>
<p>David Barry</p>
<p>Published 2025-01-29</p>
<p>Licensed CC-BY-4.0 INTERNATIONAL</p>
<p>Slides presented at Empowering Healthcare with Automated Analysis at London Metropolitan University.</p>
<p>Tags: Reproducibility, Image Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14767944">https://doi.org/10.5281/zenodo.14767944</a></p>
</section>
<hr class="docutils" />
<section id="qupath-open-source-software-for-analysing-awkward-images">
<h2>QuPath: Open source software for analysing (awkward) images<a class="headerlink" href="#qupath-open-source-software-for-analysing-awkward-images" title="Link to this heading">#</a></h2>
<p>Peter Bankhead</p>
<p>Published 2020-12-16</p>
<p>Licensed CC-BY-4.0</p>
<p>Slides from the CZI/EOSS online meeting in December 2020.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4328911">https://zenodo.org/records/4328911</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4328911">https://doi.org/10.5281/zenodo.4328911</a></p>
</section>
<hr class="docutils" />
<section id="rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again">
<h2>RDF as a bridge to domain-platforms like OMERO, or There and back again.<a class="headerlink" href="#rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again" title="Link to this heading">#</a></h2>
<p>Josh Moore, Andra Waagmeester, Kristina Hettne, Katherine Wolstencroft, Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>In 2005, the first version of OMERO stored RDF natively. However, just a year after the 1.0 release of RDF, performance considerations led to the development of a more traditional SQL approach for OMERO. A binary protocol makes it possible to query and retrieve metadata but the resulting information cannot immediately be combined with other sources. This is the adventure of rediscovering the benefit of RDF triples as a – if not the – common exchange mechanism.</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10687658">https://zenodo.org/doi/10.5281/zenodo.10687658</a></p>
</section>
<hr class="docutils" />
<section id="research-data-management-seminar-slides">
<h2>Research Data Management Seminar - Slides<a class="headerlink" href="#research-data-management-seminar-slides" title="Link to this heading">#</a></h2>
<p>Stefano Della Chiesa</p>
<p>Published 2022-05-18</p>
<p>Licensed CC-BY-4.0</p>
<p>This Research Data Management (RDM) Slides introduce to the multidisciplinary knowledge and competencies required to address policy compliance and research data management best practices throughout a project lifecycle, and beyond it.</p>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>Module 1 - Introduces the RDM giving its context in the Research Data Governance
Module 2 - Illustrates the most important RDM policies and principles
Module 3 - Provides the most relevant RDM knowledge bricks
Module 4 - Discuss the Data Management Plans (DMPs), examples, templates and guidance
</pre></div>
</div>
<p> </p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/record/6602101">https://zenodo.org/record/6602101</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.6602101">https://doi.org/10.5281/zenodo.6602101</a></p>
</section>
<hr class="docutils" />
<section id="sharing-and-licensing-material">
<h2>Sharing and licensing material<a class="headerlink" href="#sharing-and-licensing-material" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to sharing resources online and licensing</p>
<p>Tags: Sharing, Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/10-519">https://f1000research.com/slides/10-519</a></p>
</section>
<hr class="docutils" />
<section id="so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten">
<h2>So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten<a class="headerlink" href="#so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten" title="Link to this heading">#</a></h2>
<p>Pia Voigt</p>
<p>Published 2024-05-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Der Umgang mit personenbezogenen Daten stellt Forschende oft vor rechtliche Herausforderungen: Unter welchen Bedingungen dürfen personenbezogene Daten verarbeitet werden? Welche Voraussetzungen müssen erfüllt sein und welche Strategien können angewendet werden, um Daten sicher speichern, verarbeiten, teilen und aufbewahren zu können? Mit Hilfe dieses Foliensatzes erhalten Sie Einblicke in datenschutzrechtliche Aspekte beim Umgang mit Ihren Forschungsdaten. </p>
<p>Tags: Research Data Management, Data Protection, FAIR-Principles</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11396199">https://zenodo.org/records/11396199</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11396199">https://doi.org/10.5281/zenodo.11396199</a></p>
</section>
<hr class="docutils" />
<section id="sustainable-data-stewardship">
<h2>Sustainable Data Stewardship<a class="headerlink" href="#sustainable-data-stewardship" title="Link to this heading">#</a></h2>
<p>Stefano Della Chiesa</p>
<p>Published 2024-03-25</p>
<p>Licensed CC-BY-4.0</p>
<p>These slides were presented at the 2. SaxFDM-Beratungsstammtisch and delve into the strategic integration of Research Data Management (RDM) within research organizations. The Leibniz IOER presented an insightful overview of RDM activities and approaches, emphasizing the criticality of embedding RDM strategically within research institutions. The presentation showcases some best practices in RDM implementation through practical examples, offering valuable insights for optimizing data stewardship processes.</p>
<p>Tags: Research Data Management, Data Stewardship</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10942559">https://zenodo.org/records/10942559</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10942559">https://doi.org/10.5281/zenodo.10942559</a></p>
</section>
<hr class="docutils" />
<section id="thinking-data-management-on-different-scales">
<h2>Thinking data management on different scales<a class="headerlink" href="#thinking-data-management-on-different-scales" title="Link to this heading">#</a></h2>
<p>Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation given at PoL BioImage Analysis Symposium Dresden 2023</p>
<p>Tags: Research Data Management, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8329305">https://zenodo.org/doi/10.5281/zenodo.8329305</a></p>
</section>
<hr class="docutils" />
<section id="tracking-theory-trackmate-and-mastodon">
<h2>Tracking Theory, TrackMate, and Mastodon<a class="headerlink" href="#tracking-theory-trackmate-and-mastodon" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Lecture slides of a session on Cell Tracking in Fiji</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_tracking_trackmate">https://git.mpi-cbg.de/rhaase/lecture_tracking_trackmate</a></p>
</section>
<hr class="docutils" />
<section id="welcome-to-bioimage-town">
<h2>Welcome to BioImage Town<a class="headerlink" href="#welcome-to-bioimage-town" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Licensed CC-BY-4.0</p>
<p>Welcome at NFDI4BIOIMAGE All-Hands Meeting in Düsseldorf, Germany, October 16, 2023</p>
<p>Tags: OMERO, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10008464">https://zenodo.org/doi/10.5281/zenodo.10008464</a></p>
</section>
<hr class="docutils" />
<section id="what-is-bioimage-analysis-an-introduction">
<h2>What is Bioimage Analysis? An Introduction<a class="headerlink" href="#what-is-bioimage-analysis-an-introduction" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://www.dropbox.com/s/5abw3cvxrhpobg4/20220923_DefragmentationTS.pdf?dl=0">https://www.dropbox.com/s/5abw3cvxrhpobg4/20220923_DefragmentationTS.pdf?dl=0</a></p>
</section>
<hr class="docutils" />
<section id="working-with-objects-in-2d-and-3d">
<h2>Working with objects in 2D and 3D<a class="headerlink" href="#working-with-objects-in-2d-and-3d" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_working_with_objects_in_2d_and_3d">https://git.mpi-cbg.de/rhaase/lecture_working_with_objects_in_2d_and_3d</a></p>
</section>
<hr class="docutils" />
<section id="working-with-pixels">
<h2>Working with pixels<a class="headerlink" href="#working-with-pixels" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_working_with_pixels">https://git.mpi-cbg.de/rhaase/lecture_working_with_pixels</a></p>
</section>
<hr class="docutils" />
<section id="ymia-python-based-event-series-training-material">
<h2>YMIA - Python-Based Event Series Training Material<a class="headerlink" href="#ymia-python-based-event-series-training-material" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Robert Haase, ENicolay</p>
<p>Published None</p>
<p>Licensed MIT</p>
<p>This repository offer access to teaching material and useful resources for the YMIA - Python-Based Event Series.</p>
<p>Tags: Python, Artifical Intelligence, Bioimage Analysis</p>
<p>Content type: Github Repository, Slides</p>
<p><a class="github reference external" href="https://github.com/rmassei/ymia_python_event_series_material">rmassei/ymia_python_event_series_material</a></p>
</section>
<hr class="docutils" />
<section id="zidas-2020-introduction-to-deep-learning">
<h2>ZIDAS 2020 Introduction to Deep Learning<a class="headerlink" href="#zidas-2020-introduction-to-deep-learning" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/esgomezm/zidas2020_intro_DL">esgomezm/zidas2020_intro_DL</a></p>
</section>
<hr class="docutils" />
<section id="n4bi-ahm-welcome-to-bioimage-town">
<h2>[N4BI AHM] Welcome to BioImage Town<a class="headerlink" href="#n4bi-ahm-welcome-to-bioimage-town" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2023-10-16</p>
<p>Licensed CC-BY-4.0</p>
<p>Keynote at the NFDI4BIOIMAGE All-Hands Meeting in Düsseldorf, Germany, October 16, 2023.</p>
<p>Tags: Research Data Management</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10008465">https://zenodo.org/records/10008465</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10008465">https://doi.org/10.5281/zenodo.10008465</a></p>
</section>
<hr class="docutils" />
<section id="short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">
<h2>[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure<a class="headerlink" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure" title="Link to this heading">#</a></h2>
<p>Christian Schmidt</p>
<p>Licensed CC-BY-4.0</p>
<p>Short Talk about the NFDI4BIOIMAGE consortium presented at the RDM in (Bio-)Medicine Information Event on April 10th, 2024, organized C³RDM &amp; ZB MED.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10939519">https://zenodo.org/doi/10.5281/zenodo.10939519</a></p>
</section>
<hr class="docutils" />
<section id="ilastik-interactive-machine-learning-for-bio-image-analysis">
<h2>ilastik: interactive machine learning for (bio)image analysis<a class="headerlink" href="#ilastik-interactive-machine-learning-for-bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Anna Kreshuk, Dominik Kutra</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.4330625">https://zenodo.org/doi/10.5281/zenodo.4330625</a></p>
</section>
<hr class="docutils" />
<section id="rse-skills-workshop">
<h2>rse-skills-workshop<a class="headerlink" href="#rse-skills-workshop" title="Link to this heading">#</a></h2>
<p>Jack Atkinson</p>
<p>Published 2023-12-22T17:39:48+00:00</p>
<p>Licensed GPL-3.0</p>
<p>Teaching materials for improving research software writing abilities.</p>
<p>Tags: Research Software Engineering</p>
<p>Content type: Github Repository, Slides</p>
<p><a class="github reference external" href="https://github.com/jatkinson1000/rse-skills-workshop">jatkinson1000/rse-skills-workshop</a></p>
<hr class="docutils" />
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./content_types"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="publication.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Publication (62)</p>
      </div>
    </a>
    <a class="right-next"
       href="tutorial.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Tutorial (45)</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium">“ZENODO und Co.” Was bringt und wer braucht ein Repositorium?</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ai-ml-dl-in-bioimage-analysis-webinar">AI ML DL in Bioimage Analysis - Webinar</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#adding-a-workflow-to-biaflows">Adding a Workflow to BIAFLOWS</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten">Alles meins – oder!? Urheberrechte klären für Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis">Bio Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-with-the-help-of-large-language-models">Bio-image Analysis with the Help of Large Language Models</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science-lectures-uni-leipzig-scads-ai">Bio-image Data Science Lectures @ Uni Leipzig / ScaDS.AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-bioimage-analysis-workflow-using-deep-learning">Building a Bioimage Analysis Workflow using Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-introduction">CellProfiler Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">Challenges and opportunities for bio-image analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#crashkurs-forschungsdatenmanagement">Crashkurs Forschungsdatenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#creating-open-computational-curricula">Creating open computational curricula</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training">Cultivating Open Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training-to-advance-bio-image-analysis">Cultivating Open Training to advance Bio-image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl-mbl-2021-exercises">DL@MBL 2021 Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-management-at-france-bioimaging">Data management at France BioImaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagement">Datenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz">Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-1">Datenmanagementpläne erstellen - Teil 1</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-2">Datenmanagementpläne erstellen - Teil 2</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy">Dr Guillaume Jacquemet on studying cancer cell metastasis in the era of deep learning for microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten">Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fairy-deep-learning-for-bioimage-analysis">FAIRy deep-learning for bioImage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers">From Paper to Pixels: Navigation through your Research Data - presentations of speakers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-training-material">Galaxy Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#generative-artificial-intelligence-for-bio-image-analysis">Generative artificial intelligence for bio-image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#hackaton-results-conversion-of-knime-image-analysis-workflows-to-galaxy">Hackaton Results - Conversion of KNIME image analysis workflows to Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#hitchhiking-through-a-diverse-bio-image-analysis-software-universe">Hitchhiking through a diverse Bio-image Analysis Software Universe</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-data-services-at-euro-bioimaging-community-efforts-towards-fair-image-data-and-analysis-services">Image Data Services at Euro-BioImaging: Community efforts towards FAIR Image Data and Analysis Services</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-in-galaxy">Image analysis in Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej-macro-introduction">ImageJ Macro Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej2-api-beating">ImageJ2 API-beating</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#intro-napari-slides">Intro napari slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden">Introduction to ImageJ macro programming, Scientific Computing Facility, MPI CBG Dresden</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-research-data-management-and-open-research">Introduction to Research Data Management and Open Research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#jupyter-for-interactive-cloud-computing">Jupyter for interactive cloud computing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kollaboratives-arbeiten-und-versionskontrolle-mit-git">Kollaboratives Arbeiten und Versionskontrolle mit Git</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-applied-bioimage-analysis-2020">Lecture Applied Bioimage Analysis 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-materials-of-the-deeplife-course">Lecture-materials of the DeepLife course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-deep-learning-applications-to-bioimage-analysis">Machine Learning - Deep Learning. Applications to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-and-deep-learning-on-the-cloud-segmentation">Machine and Deep Learning on the cloud: Segmentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#methods-in-bioimage-analysis">Methods in bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microsam-talks">MicroSam-Talks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive">Microscopy data analysis: machine learning and the BioImage Archive</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multi-view-fusion">Multi-view fusion</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud">My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-analyst-school-2018">NEUBIAS Analyst School 2018</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-course-2017">NEUBIAS Bioimage Analyst Course 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2019">NEUBIAS Bioimage Analyst School 2019</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2020">NEUBIAS Bioimage Analyst School 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage">NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data">NFDI4BIOIMAGE - An Initiative for a National Research Data Infrastructure for Microscopy Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-scalable-and-reproducible-scientific-workflows">Nextflow: Scalable and reproducible scientific workflows</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-science-sharing-licensing">Open Science, Sharing &amp; Licensing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing">Parallelization and heterogeneous computing: from pure CPU to GPU-accelerated image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#promoting-reproducibility-in-biomedical-research-through-image-analysis">Promoting Reproducibility in Biomedical Research through Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-open-source-software-for-analysing-awkward-images">QuPath: Open source software for analysing (awkward) images</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again">RDF as a bridge to domain-platforms like OMERO, or There and back again.</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-management-seminar-slides">Research Data Management Seminar - Slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sharing-and-licensing-material">Sharing and licensing material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten">So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sustainable-data-stewardship">Sustainable Data Stewardship</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#thinking-data-management-on-different-scales">Thinking data management on different scales</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-theory-trackmate-and-mastodon">Tracking Theory, TrackMate, and Mastodon</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#welcome-to-bioimage-town">Welcome to BioImage Town</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-bioimage-analysis-an-introduction">What is Bioimage Analysis? An Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-objects-in-2d-and-3d">Working with objects in 2D and 3D</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-pixels">Working with pixels</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zidas-2020-introduction-to-deep-learning">ZIDAS 2020 Introduction to Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#n4bi-ahm-welcome-to-bioimage-town">[N4BI AHM] Welcome to BioImage Town</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ilastik-interactive-machine-learning-for-bio-image-analysis">ilastik: interactive machine learning for (bio)image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rse-skills-workshop">rse-skills-workshop</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Robert Haase, Clément Caporal,... and the NFDI4BioImage Initiative
</p>

  </div>
  
  <div class="footer-item">
    

  </div>
  
  <div class="footer-item">
    <p class="last-updated">
  Last updated on 2025-03-04.
  <br/>
</p>
  </div>
  
  <div class="footer-item">
    
<div class="extra_footer">
  <p>
Copyright: Licensed <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY 4.0</a> unless mentioned otherwise. 
Contributions and feedback are welcome.
</p>

</div>
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>