
<!DOCTYPE html>


<html lang="en" data-content_root="../" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Bioimage analysis (183) &#8212; NFDI4BioImage Training Materials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  <!--
    this give us a css class that will be invisible only if js is disabled
  -->
  <noscript>
    <style>
      .pst-js-only { display: none !important; }

    </style>
  </noscript>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=8878045cc6db502f8baf" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=8878045cc6db502f8baf" rel="stylesheet" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="../_static/styles/sphinx-book-theme.css?v=a3416100" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-design.min.css?v=95c83b7e" />
  
  <!-- So that users can add custom icons -->
  <script src="../_static/scripts/fontawesome.js?digest=8878045cc6db502f8baf"></script>
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=8878045cc6db502f8baf" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=8878045cc6db502f8baf" />

    <script src="../_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="../_static/doctools.js?v=9a2dae69"></script>
    <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="../_static/copybutton.js?v=f281be69"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="../_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'tags/bioimage_analysis';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Bioinformatics (10)" href="bioinformatics.html" />
    <link rel="prev" title="Artificial intelligence (44)" href="artificial_intelligence.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  <meta name="docsearch:version" content="" />
    <meta name="docbuild:last-update" content="2025-02-13"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <dialog id="pst-search-dialog">
    
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form>
  </dialog>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <dialog id="pst-primary-sidebar-modal"></dialog>
      <div id="pst-primary-sidebar" class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="../readme.html">
  
  
  
  
  
    
    
      
    
    
    <img src="../_static/logo.png" class="logo__image only-light" alt="NFDI4BioImage Training Materials - Home"/>
    <img src="../_static/logo.png" class="logo__image only-dark pst-js-only" alt="NFDI4BioImage Training Materials - Home"/>
  
  
</a></div>
        <div class="sidebar-primary-item">

<button class="btn search-button-field search-button__button pst-js-only" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
 <i class="fa-solid fa-magnifying-glass"></i>
 <span class="search-button__default-text">Search</span>
 <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
</button></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../readme.html">
                    NFDI4BioImage Training Materials
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">What's new</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../whats_new.html">Recently added (10)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Contributing</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../contributing/index.html">How to contribute</a></li>

<li class="toctree-l1"><a class="reference internal" href="../contributing/submit_app.html">Using the Training Materials Submission App</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contributing/format.html">YML format</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By tag</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="artificial_intelligence.html">Artificial intelligence (44)</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Bioimage analysis (183)</a></li>
<li class="toctree-l1"><a class="reference internal" href="bioinformatics.html">Bioinformatics (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="data_stewardship.html">Data stewardship (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="fair-principles.html">Fair-principles (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="imagej.html">Imagej (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="licensing.html">Licensing (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="metadata.html">Metadata (15)</a></li>
<li class="toctree-l1"><a class="reference internal" href="napari.html">Napari (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="neubias.html">Neubias (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="nfdi4bioimage.html">Nfdi4bioimage (44)</a></li>
<li class="toctree-l1"><a class="reference internal" href="omero.html">Omero (31)</a></li>
<li class="toctree-l1"><a class="reference internal" href="open_science.html">Open science (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="open_source_software.html">Open source software (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="python.html">Python (70)</a></li>
<li class="toctree-l1"><a class="reference internal" href="research_data_management.html">Research data management (128)</a></li>
<li class="toctree-l1"><a class="reference internal" href="sharing.html">Sharing (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="workflow.html">Workflow (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="workflow_engine.html">Workflow engine (13)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By content type</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../content_types/blog%20post.html">Blog post (23)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/book.html">Book (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/code.html">Code (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/collection.html">Collection (81)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/data.html">Data (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/document.html">Document (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/documentation.html">Documentation (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/event.html">Event (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/github%20repository.html">Github repository (45)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/notebook.html">Notebook (55)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/online%20tutorial.html">Online tutorial (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/poster.html">Poster (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/preprint.html">Preprint (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/publication.html">Publication (61)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/slides.html">Slides (82)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/tutorial.html">Tutorial (42)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/video.html">Video (32)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/website.html">Website (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/workshop.html">Workshop (12)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By license</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../licenses/all_rights_reserved.html">All rights reserved (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-2-clause.html">Bsd-2-clause (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-3-clause.html">Bsd-3-clause (26)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-4.0.html">Cc-by-4.0 (251)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-sa-4.0.html">Cc-by-sa-4.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc0-1.0.html">Cc0-1.0 (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-2.0.html">Gpl-2.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-3.0.html">Gpl-3.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/mit.html">Mit (24)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/unknown.html">Unknown (89)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By domain</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../domain/biapol.github.io.html">Biapol.github.io (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/docs.google.com.html">Docs.google.com (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/doi.org.html">Doi.org (186)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/f1000research.com.html">F1000research.com (11)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/focalplane.biologists.com.html">Focalplane.biologists.com (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/git.mpi-cbg.de.html">Git.mpi-cbg.de (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/github.com.html">Github.com (123)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.biorxiv.org.html">Www.biorxiv.org (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.ebi.ac.uk.html">Www.ebi.ac.uk (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.nature.com.html">Www.nature.com (16)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.youtube.com.html">Www.youtube.com (22)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/zenodo.org.html">Zenodo.org (175)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Appendix</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../statistics/readme.html">Statistics</a></li>
<li class="toctree-l1"><a class="reference internal" href="../export/readme.html">Open data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../imprint.html">Imprint</a></li>

</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
      <div class="sidebar-primary-item">
<div id="ethical-ad-placement"
      class="flat"
      data-ea-publisher="readthedocs"
      data-ea-type="readthedocs-sidebar"
      data-ea-manual="true">
</div></div>
  </div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training/issues/new?title=Issue%20on%20page%20%2Ftags/bioimage_analysis.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/tags/bioimage_analysis.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button pst-js-only" aria-label="Color mode" data-bs-title="Color mode"  data-bs-placement="bottom" data-bs-toggle="tooltip">
  <i class="theme-switch fa-solid fa-sun                fa-lg" data-mode="light" title="Light"></i>
  <i class="theme-switch fa-solid fa-moon               fa-lg" data-mode="dark"  title="Dark"></i>
  <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"  title="System Settings"></i>
</button>


<button class="btn btn-sm pst-navbar-icon search-button search-button__button pst-js-only" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
</button>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Bioimage analysis (183)</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-survey-community-experiences-and-needs-for-the-future">2020 BioImage Analysis Survey: Community experiences and needs for the future</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#mic-workshop-on-bioimage-processing-with-python">2022 MIC Workshop on Bioimage processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-fiji-scripting-tutorial">A Fiji Scripting Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe">A Hitchhiker’s guide through the bio-image analysis software universe</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ai-ml-dl-in-bioimage-analysis-webinar">AI ML DL in Bioimage Analysis - Webinar</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ai4life-teams-up-with-galaxy-training-network-gtn-to-enhance-training-resources">AI4Life teams up with Galaxy Training Network (GTN) to enhance training resources</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#adding-a-workflow-to-biaflows">Adding a Workflow to BIAFLOWS</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#analysis-of-high-throughput-microscopy-image-data">Analysis of High-Throughput Microscopy Image Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#annotating-3d-images-in-napari">Annotating 3D images in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bids-lecture-2024">BIDS-lecture-2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#biomero-a-scalable-and-extensible-image-analysis-framework">BIOMERO - A scalable and extensible image analysis framework</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#basics-of-image-processing-and-analysis">Basics of Image Processing and Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bigdataprocessor2-a-free-and-open-source-fiji-plugin-for-inspection-and-processing-of-tb-sized-image-data">BigDataProcessor2: A free and open-source Fiji plugin for inspection and processing of TB sized image data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-lecture-2020">Bio Image Analysis Lecture 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-code-generation-using-bia-bob">Bio-image Analysis Code Generation using bia-bob</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-icob-2023">Bio-image Analysis ICOB 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-kioto-and-taipei-23-24">Bio-image Analysis Workshop Kioto and Taipei 23/24</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-taipei">Bio-image Analysis Workshop Taipei</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science">Bio-image Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science-lectures-uni-leipzig-scads-ai">Bio-image Data Science Lectures @ Uni Leipzig / ScaDS.AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">Bio-image analysis, biostatistics, programming and machine learning for computational biology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-notebooks">BioImage Analysis Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-archive-ai-gallery">BioImage Archive AI Gallery</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-archive-visual-gallery">BioImage Archive Visual Gallery</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-archive-volume-em-gallery">BioImage Archive Volume EM Gallery</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-data-analysis">BioImage Data Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-informatics-index-training-materials">BioImage Informatics Index Training Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-model-zoo">Bioimage Model Zoo</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-with-napari">Bioimage analysis with Napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bridging-imaging-users-to-imaging-analysis-a-community-survey">Bridging Imaging Users to Imaging Analysis - A community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bring-your-own-data-workshops">Bring your own data workshops</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-bioimage-analysis-workflow-using-deep-learning">Building a Bioimage Analysis Workflow using Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022">CARE/Stardist tutorials for EMBO Practical Course — Computational optical biology 2022</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#clij-gpu-accelerated-image-processing-for-everyone">CLIJ: GPU-accelerated image processing for everyone</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#coba-center-for-open-bioimage-analysis-youtube-channel">COBA: Center for Open Bioimage Analysis YouTube Channel</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#csbdeep-and-stardist-i2k-2020">CSBDeep and StarDist @ I2K 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cell-tracking-challenge-2d-datasets">Cell Tracking Challenge - 2D Datasets</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cell-tracking-challenge-3d-datasets">Cell Tracking Challenge - 3D Datasets</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-introduction">CellProfiler Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-practical-neubias-lisbon-2017">CellProfiler Practical NeuBIAS Lisbon 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-tutorials">CellProfiler tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">Challenges and opportunities for bio-image analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bioimage-analysis-core-facilities">Challenges and opportunities for bioimage analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#checklists-for-publishing-images-and-image-analysis">Checklists for publishing images and image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#chris-halvin-youtube-channel">Chris Halvin YouTube channel</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis">Collection of teaching material for deep learning for (biomedical) image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#community-developed-checklists-for-publishing-images-and-image-analyses">Community-developed checklists for publishing images and image analyses</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#course-on-deep-learning-for-imaging-using-pytorch">Course on Deep Learning for imaging using PyTorch</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training">Cultivating Open Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-napari-napari-as-a-tool-for-deep-learning-project-management">DEEP NAPARI : Napari as a tool for deep learning project management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4miceverywhere">DL4MicEverywhere</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl-mbl-2021-exercises">DL@MBL 2021 Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dask-course">Dask Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-management-at-france-bioimaging">Data management at France BioImaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-for-image-analysis-exercises">Deep Learning for image analysis - Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deepprofiler-handbook">DeepProfiler Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#developing-semi-automatic-analysis-pipelines-and-technological-solutions-for-metadata-annotation-and-management-in-high-content-screening-hcs-bioimaging">Developing (semi)automatic analysis pipelines and technological solutions for metadata annotation and management in high-content screening (HCS) bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#diffusion-models-for-image-restoration-an-introduction">Diffusion Models for Image Restoration - An Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy">Dr Guillaume Jacquemet on studying cancer cell metastasis in the era of deep learning for microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats">EDAM-bioimaging - The ontology of bioimage informatics operations, topics, data, and formats</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embo-practical-course-advanced-methods-in-bioimage-analysis">EMBO Practical Course Advanced methods in bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#epflx-image-processing-and-analysis-for-life-scientists">EPFLx: Image Processing and Analysis for Life Scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embedseg-repository">EmbedSeg Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#erick-martins-ratamero-expanding-the-ome-ecosystem-for-imaging-data-management-scipy-2024">Erick Martins Ratamero - Expanding the OME ecosystem for imaging data management | SciPy 2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus">EuBIAS course 2013: Intensity Dynamics at the Periphery of Nucleus</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-s-template-for-research-data-management-plans">Euro-BioImaging’s Template for Research Data Management Plans</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#example-pipeline-tutorial">Example Pipeline Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fair-bioimage-data">FAIR BioImage Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fairy-deep-learning-for-bioimage-analysis">FAIRy deep-learning for bioImage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#feature-extraction-in-napari">Feature extraction in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fiji-is-just-imagej-tutorials">Fiji Is Just ImageJ Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging">Fit for OMERO: How imaging facilities and IT departments work together to enable RDM for bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-in-digital-image-processing">Fundamentals in digital image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends">Fundamentals of image analysis in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-training">Galaxy Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#generative-artificial-intelligence-for-bio-image-analysis">Generative artificial intelligence for bio-image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#global-bioimaging-training-database">Global BioImaging Training Database</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero">High throughput &amp; automated data analysis and data management workflow with Cellprofiler and OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists">Highlights from the 2016-2020 NEUBIAS training schools for Bioimage Analysts: a success story and key asset for analysts and life scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#hitchhiking-through-a-diverse-bio-image-analysis-software-universe">Hitchhiking through a diverse Bio-image Analysis Software Universe</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library">I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-virtual-bio-image-analysis-code-generation">I2K2024(virtual) - Bio-Image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#iafig-rms-python-for-bioimage-analysis-course">IAFIG-RMS Python for Bioimage Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#itkelastix-examples">ITKElastix Examples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ibiology-bioimage-analysis-course-the-life-cycle-of-an-image-data-set">Ibiology. Bioimage Analysis Course. The Life Cycle of an Image Data Set</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-training-resources">Image Analysis Training Resources</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-with-python">Image Processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends">Image analysis and visualization in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-in-galaxy">Image analysis in Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-for-beginners">Image processing for beginners</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-based-profiling-handbook">Image-based Profiling Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej-macro-introduction">ImageJ Macro Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej2-api-beating">ImageJ2 API-beating</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imaris-tutorials">Imaris Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-bioimage-analysis">Introduction to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis-with-fiji">Introduction to Image Analysis with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden">Introduction to ImageJ macro programming, Scientific Computing Facility, MPI CBG Dresden</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#jupyter-for-interactive-cloud-computing">Jupyter for interactive cloud computing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-applied-bioimage-analysis-2020">Lecture Applied Bioimage Analysis 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-deep-learning-applications-to-bioimage-analysis">Machine Learning - Deep Learning. Applications to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-workflow-for-imaging-flow-cytometry-ifc">Machine Learning Workflow for Imaging Flow Cytometry (IFC)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-and-deep-learning-on-the-cloud-segmentation">Machine and Deep Learning on the cloud: Segmentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#making-the-most-of-bioimaging-data-through-interdisciplinary-interactions">Making the most of bioimaging data through interdisciplinary interactions</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration">Meeting in the Middle: Towards Successful Multidisciplinary Bioimage Analysis Collaboration</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#melbourne-advanced-microscopy-facility">Melbourne Advanced Microscopy Facility</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#methods-in-bioimage-analysis">Methods in bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#metrics-reloaded-a-framework-for-trustworthy-image-analysis-validation">Metrics Reloaded - A framework for trustworthy image analysis validation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microsam-talks">MicroSam-Talks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive">Microscopy data analysis: machine learning and the BioImage Archive</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#modular-training-resources-for-bioimage-analysis">Modular training resources for bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#morpholibj-documentation">MorphoLibJ documentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multi-view-fusion">Multi-view fusion</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multimodal-large-language-models-for-bioimage-analysis">Multimodal large language models for bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter">NEUBIAS Academy @HOME: Interactive Bioimage Analysis with Python and Jupyter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-analyst-school-2018">NEUBIAS Analyst School 2018</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-course-2017">NEUBIAS Bioimage Analyst Course 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2019">NEUBIAS Bioimage Analyst School 2019</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2020">NEUBIAS Bioimage Analyst School 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage">NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubiaspasteur2023-advancedcellpose">NeubiasPasteur2023_AdvancedCellPose</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks">OMERO - HCS analysis pipeline using Jupyter Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-qupath">OMERO - QuPath</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#onbi-image-analysis-course">ONBI Image Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-tools">Omero-tools</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-image-data-handbook">Open Image Data Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-early-career-bio-image-analysis-training-school">PoL Bio-Image Analysis Early Career Bio-image Analysis Training School</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-gpu-accelerated-image-analysis-training-school">PoL Bio-Image Analysis GPU Accelerated Image Analysis Training School</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-in-bio-image-analysis">Prompt Engineering in Bio-image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-bioimage-analysis-tutorial">Python BioImage Analysis Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-for-microscopists">Python for Microscopists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-for-python-programmers">QuPath for Python programmers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-open-source-software-for-analysing-awkward-images">QuPath: Open source software for analysing (awkward) images</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quality-assurance-of-segmentation-results">Quality assurance of segmentation results</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again">RDF as a bridge to domain-platforms like OMERO, or There and back again.</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rescaling-images-and-pixel-an-isotropy">Rescaling images and pixel (an)isotropy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#running-deep-learning-scripts-in-the-bia-pol-omero-server">Running Deep-Learning Scripts in the BiA-PoL Omero Server</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scipy-cookbook">Scipy Cookbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scripts-filopodyanr-a-case-study-for-the-neubias-ts7-in-szeged">Scripts_FilopodyanR - a case study for the NEUBIAS TS7 in Szeged</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#simpleitk-notebooks">SimpleITK-Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-bioimage-analysis-with-python">Teaching Bioimage Analysis with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-imagej-fiji">Teaching ImageJ FIJI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-digital-cell-cell-biology-as-a-data-science">The Digital Cell: Cell Biology as a Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-open-microscopy-environment-ome-data-model-and-xml-file-open-tools-for-informatics-and-quantitative-analysis-in-biological-imaging">The Open Microscopy Environment (OME) Data Model and XML file - open tools for informatics and quantitative analysis in biological imaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#towards-community-driven-metadata-standards-for-light-microscopy-tiered-specifications-extending-the-ome-model">Towards community-driven metadata standards for light microscopy - tiered specifications extending the OME model</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-theory-trackmate-and-mastodon">Tracking Theory, TrackMate, and Mastodon</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-in-napari">Tracking in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-of-mitochondria-and-capturing-mitoflashes">Tracking of mitochondria and capturing mitoflashes</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-deep-learning-models-for-vision-compact-course">Training Deep Learning Models for Vision - Compact Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ultrack-i2k-2024-workshop-materials">Ultrack I2K 2024 Workshop Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#understanding-metric-related-pitfalls-in-image-analysis-validation">Understanding metric-related pitfalls in image analysis validation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#upcoming-image-analysis-events">Upcoming Image Analysis Events</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#using-glittr-org-to-find-compare-and-re-use-online-training-materials">Using Glittr.org to find, compare and re-use online training materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#v4sdb-winter-school-2025">V4SDB_Winter_School_2025</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#welcome-to-bioimage-town">Welcome to BioImage Town</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-bioimage-analysis-an-introduction">What is Bioimage Analysis? An Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#who-you-gonna-call-data-stewards-to-the-rescue">Who you gonna call? - Data Stewards to the rescue</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-objects-in-2d-and-3d">Working with objects in 2D and 3D</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-pixels">Working with pixels</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-june2024-madrid">Workshop-June2024-Madrid</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zen-python-workshop">ZEN &amp; Python workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zidas-2020-introduction-to-deep-learning">ZIDAS 2020 Introduction to Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zerocostdl4mic-exploiting-google-colab-to-develop-a-free-and-open-source-toolbox-for-deep-learning-in-microscopy">ZeroCostDL4Mic: exploiting Google Colab to develop a free and open-source toolbox for Deep-Learning in microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cordi-2023-zarr-a-cloud-optimized-storage-for-interactive-access-of-large-arrays">[CORDI 2023] Zarr: A Cloud-Optimized Storage for Interactive Access of Large Arrays</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#arivis-vision4d-tutorials">arivis Vision4D Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ilastik-interactive-machine-learning-for-bio-image-analysis">ilastik: interactive machine learning for (bio)image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis">introduction-to-image-analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#patho-prompt-injection">patho_prompt_injection</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quantixed-thedigitalcell-first-complete-code-set">quantixed/TheDigitalCell: First complete code set</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scanpy-tutorials">scanpy-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scikit-learn-mooc">scikit-learn MOOC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#skimage-tutorials">skimage-tutorials</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="bioimage-analysis-183">
<h1>Bioimage analysis (183)<a class="headerlink" href="#bioimage-analysis-183" title="Link to this heading">#</a></h1>
<section id="bioimage-analysis-survey-community-experiences-and-needs-for-the-future">
<h2>2020 BioImage Analysis Survey: Community experiences and needs for the future<a class="headerlink" href="#bioimage-analysis-survey-community-experiences-and-needs-for-the-future" title="Link to this heading">#</a></h2>
<p>Nasim Jamali, Ellen T. A. Dobson, Kevin W. Eliceiri, Anne E. Carpenter, Beth A. Cimini</p>
<p>Published 2021</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1017/s2633903x21000039">https://doi.org/10.1017/s2633903x21000039</a></p>
<p><a class="github reference external" href="https://github.com/ciminilab/2021_Jamali_BiologicalImaging">ciminilab/2021_Jamali_BiologicalImaging</a></p>
</section>
<hr class="docutils" />
<section id="mic-workshop-on-bioimage-processing-with-python">
<h2>2022 MIC Workshop on Bioimage processing with Python<a class="headerlink" href="#mic-workshop-on-bioimage-processing-with-python" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/MICPy_Workshop_2022">guiwitz/MICPy_Workshop_2022</a></p>
</section>
<hr class="docutils" />
<section id="a-fiji-scripting-tutorial">
<h2>A Fiji Scripting Tutorial<a class="headerlink" href="#a-fiji-scripting-tutorial" title="Link to this heading">#</a></h2>
<p>Albert Cardona</p>
<p>Licensed CC0-1.0</p>
<p>Tags: Imagej, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="reference external" href="https://syn.mrc-lmb.cam.ac.uk/acardona/fiji-tutorial/">https://syn.mrc-lmb.cam.ac.uk/acardona/fiji-tutorial/</a></p>
</section>
<hr class="docutils" />
<section id="a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">
<h2>A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter<a class="headerlink" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter" title="Link to this heading">#</a></h2>
<p>Anca Margineanu, Chiara Stringari, Marcelo Zoccoler, Cornelia Wetzker</p>
<p>Licensed CC-BY-4.0</p>
<p>The presentations introduce open-source software to read in, visualize and analyse fluorescence lifetime imaging microscopy (FLIM) raw data developed for life scientists. The slides were presented at German Bioimaging (GerBI) FLIM Workshop held February 26 to 29 2024 at the Biomedical Center of LMU München by Anca Margineanu, Chiara Stringari and Conni Wetzker.</p>
<p>Tags: Bioimage Analysis, Flim</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10886749">https://zenodo.org/doi/10.5281/zenodo.10886749</a></p>
</section>
<hr class="docutils" />
<section id="a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe">
<h2>A Hitchhiker’s guide through the bio-image analysis software universe<a class="headerlink" href="#a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe" title="Link to this heading">#</a></h2>
<p>Robert Haase, Elnaz Fazeli, David Legland, Michael Doube, Siân Culley, Ilya Belevich, Eija Jokitalo, Martin Schorb, Anna Klemm, Christian Tischer</p>
<p>Licensed CC-BY-4.0</p>
<p>This article gives an overview about commonly used bioimage analysis software and which aspects to consider when choosing a software for a specific project.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://febs.onlinelibrary.wiley.com/doi/full/10.1002/1873-3468.14451">https://febs.onlinelibrary.wiley.com/doi/full/10.1002/1873-3468.14451</a></p>
</section>
<hr class="docutils" />
<section id="ai-ml-dl-in-bioimage-analysis-webinar">
<h2>AI ML DL in Bioimage Analysis - Webinar<a class="headerlink" href="#ai-ml-dl-in-bioimage-analysis-webinar" title="Link to this heading">#</a></h2>
<p>Yannick KREMPP</p>
<p>Published 2024-11-14</p>
<p>Licensed UNKNOWN</p>
<p>A review of the tools, methods and concepts useful for biologists and life scientists as well as bioimage analysts.</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Video, Slides, Webinar</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=TJXNMIWtdac">https://www.youtube.com/watch?v=TJXNMIWtdac</a></p>
</section>
<hr class="docutils" />
<section id="ai4life-teams-up-with-galaxy-training-network-gtn-to-enhance-training-resources">
<h2>AI4Life teams up with Galaxy Training Network (GTN) to enhance training resources<a class="headerlink" href="#ai4life-teams-up-with-galaxy-training-network-gtn-to-enhance-training-resources" title="Link to this heading">#</a></h2>
<p>Caterina Fuster-Barceló</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Workflow Engine, Bioimage Analysis</p>
<p>Content type: Documentation</p>
<p><a class="reference external" href="https://ai4life.eurobioimaging.eu/ai4life-teams-up-with-galaxy-training-network-gtn-to-enhance-training-resources/">https://ai4life.eurobioimaging.eu/ai4life-teams-up-with-galaxy-training-network-gtn-to-enhance-training-resources/</a></p>
</section>
<hr class="docutils" />
<section id="adding-a-workflow-to-biaflows">
<h2>Adding a Workflow to BIAFLOWS<a class="headerlink" href="#adding-a-workflow-to-biaflows" title="Link to this heading">#</a></h2>
<p>Sébastien Tosi, Volker Baecker, Benjamin Pavie</p>
<p>Licensed BSD-2-CLAUSE</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/RoccoDAnt/Defragmentation_TrainingSchool_EOSC-Life_2022/blob/main/Slides/Adding_a_workflow_to_BIAFLOWS.pdf">RoccoDAnt/Defragmentation_TrainingSchool_EOSC-Life_2022</a></p>
</section>
<hr class="docutils" />
<section id="analysis-of-high-throughput-microscopy-image-data">
<h2>Analysis of High-Throughput Microscopy Image Data<a class="headerlink" href="#analysis-of-high-throughput-microscopy-image-data" title="Link to this heading">#</a></h2>
<p>Kota Miura, Christoph Schiklenk, Clemens Lakner, Christian Tischer, Aliaksandr Halavatyi</p>
<p>Published 2014</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/miura/HTManalysisCourse/blob/master/CentreCourseProtocol.md">miura/HTManalysisCourse</a></p>
</section>
<hr class="docutils" />
<section id="annotating-3d-images-in-napari">
<h2>Annotating 3D images in napari<a class="headerlink" href="#annotating-3d-images-in-napari" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/03/30/annotating-3d-images-in-napari/">https://focalplane.biologists.com/2023/03/30/annotating-3d-images-in-napari/</a></p>
</section>
<hr class="docutils" />
<section id="bids-lecture-2024">
<h2>BIDS-lecture-2024<a class="headerlink" href="#bids-lecture-2024" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python. The material developed here between April and July 2024.</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence, Python</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ScaDS/BIDS-lecture-2024/">ScaDS/BIDS-lecture-2024</a></p>
</section>
<hr class="docutils" />
<section id="biomero-a-scalable-and-extensible-image-analysis-framework">
<h2>BIOMERO - A scalable and extensible image analysis framework<a class="headerlink" href="#biomero-a-scalable-and-extensible-image-analysis-framework" title="Link to this heading">#</a></h2>
<p>Torec T. Luik, Rodrigo Rosas-Bertolini, Eric A.J. Reits, Ron A. Hoebe, Przemek M. Krawczyk</p>
<p>Published None</p>
<p>Licensed CC-BY-4.0</p>
<p>The authors introduce BIOMERO (bioimage analysis in OMERO), a bridge connecting OMERO, a renowned bioimaging data management platform, FAIR workflows, and high-performance computing (HPC) environments.</p>
<p>Tags: OMERO, Workflow, Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1016/j.patter.2024.101024">https://doi.org/10.1016/j.patter.2024.101024</a></p>
</section>
<hr class="docutils" />
<section id="basics-of-image-processing-and-analysis">
<h2>Basics of Image Processing and Analysis<a class="headerlink" href="#basics-of-image-processing-and-analysis" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/miura/ij_textbook1/raw/76b51338e1f006c580b6f0f5cfc48fe02fba38d7/CMCIBasicCourse201102Bib.pdf">miura/ij_textbook1</a></p>
</section>
<hr class="docutils" />
<section id="bigdataprocessor2-a-free-and-open-source-fiji-plugin-for-inspection-and-processing-of-tb-sized-image-data">
<h2>BigDataProcessor2: A free and open-source Fiji plugin for inspection and processing of TB sized image data<a class="headerlink" href="#bigdataprocessor2-a-free-and-open-source-fiji-plugin-for-inspection-and-processing-of-tb-sized-image-data" title="Link to this heading">#</a></h2>
<p>Christian Tischer, Ashis Ravindran, Sabine Reither, Nicolas Chiaruttini, Rainer Pepperkok, Nils Norlin</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1093/bioinformatics/btab106">https://doi.org/10.1093/bioinformatics/btab106</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-lecture-2020">
<h2>Bio Image Analysis Lecture 2020<a class="headerlink" href="#bio-image-analysis-lecture-2020" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=e-2DbkUwKk4&amp;amp;list=PL5ESQNfM5lc7SAMstEu082ivW4BDMvd0U">https://www.youtube.com/watch?v=e-2DbkUwKk4&amp;list=PL5ESQNfM5lc7SAMstEu082ivW4BDMvd0U</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-code-generation-using-bia-bob">
<h2>Bio-image Analysis Code Generation using bia-bob<a class="headerlink" href="#bio-image-analysis-code-generation-using-bia-bob" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-10-09</p>
<p>Licensed CC-BY-4.0</p>
<p>In this presentation I introduce bia-bob, an AI-based code generator that integrates into Jupyter Lab and allows for easy generation of Bio-Image Analysis Python code. It highlights how to get started with using large language models and prompt engineering to get high-quality bio-image analysis code.</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p><a class="reference external" href="https://zenodo.org/records/13908108">https://zenodo.org/records/13908108</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13908108">https://doi.org/10.5281/zenodo.13908108</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-icob-2023">
<h2>Bio-image Analysis ICOB 2023<a class="headerlink" href="#bio-image-analysis-icob-2023" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/WeiChenChu/Bioimage_Analysis_ICOB_2023">WeiChenChu/Bioimage_Analysis_ICOB_2023</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-workshop-kioto-and-taipei-23-24">
<h2>Bio-image Analysis Workshop Kioto and Taipei 23/24<a class="headerlink" href="#bio-image-analysis-workshop-kioto-and-taipei-23-24" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/Koushouu/Bioimage-Analysis-Workshop-23-24">Koushouu/Bioimage-Analysis-Workshop-23-24</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-workshop-taipei">
<h2>Bio-image Analysis Workshop Taipei<a class="headerlink" href="#bio-image-analysis-workshop-taipei" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/Koushouu/Bioimage-Analysis-Workshop-Taipei/">Koushouu/Bioimage-Analysis-Workshop-Taipei</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-science">
<h2>Bio-image Data Science<a class="headerlink" href="#bio-image-data-science" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository contains training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python.</p>
<p>Tags: Research Data Management, Artificial Intelligence, Bioimage Analysis, Python</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/ScaDS/BIDS-lecture-2024">ScaDS/BIDS-lecture-2024</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-science-lectures-uni-leipzig-scads-ai">
<h2>Bio-image Data Science Lectures &#64; Uni Leipzig / <a class="reference external" href="http://ScaDS.AI">ScaDS.AI</a><a class="headerlink" href="#bio-image-data-science-lectures-uni-leipzig-scads-ai" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>These are the PPTx training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python. The material developed here between April and July 2024.</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence, Python</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/12623730">https://zenodo.org/records/12623730</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">
<h2>Bio-image analysis, biostatistics, programming and machine learning for computational biology<a class="headerlink" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology" title="Link to this heading">#</a></h2>
<p>Anna Poetsch, Biotec Dresden, Marcelo Leomil Zoccoler, Johannes Richard Müller, Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Bioimage Analysis, Napari</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/BiAPoL/Bio-image_Analysis_with_Python">BiAPoL/Bio-image_Analysis_with_Python</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-analysis-notebooks">
<h2>BioImage Analysis Notebooks<a class="headerlink" href="#bioimage-analysis-notebooks" title="Link to this heading">#</a></h2>
<p>Robert Haase et al.</p>
<p>Licensed [‘CC-BY-4.0’, ‘BSD-3-CLAUSE’]</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Book, Notebook</p>
<p><a class="reference external" href="https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/intro.html">https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-archive-ai-gallery">
<h2>BioImage Archive AI Gallery<a class="headerlink" href="#bioimage-archive-ai-gallery" title="Link to this heading">#</a></h2>
<p>Licensed CC0-1.0</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Collection, Data</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/bioimage-archive/galleries/AI.html">https://www.ebi.ac.uk/bioimage-archive/galleries/AI.html</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-archive-visual-gallery">
<h2>BioImage Archive Visual Gallery<a class="headerlink" href="#bioimage-archive-visual-gallery" title="Link to this heading">#</a></h2>
<p>Licensed CC0-1.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Data</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/bioimage-archive/galleries/visualisation.html">https://www.ebi.ac.uk/bioimage-archive/galleries/visualisation.html</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-archive-volume-em-gallery">
<h2>BioImage Archive Volume EM Gallery<a class="headerlink" href="#bioimage-archive-volume-em-gallery" title="Link to this heading">#</a></h2>
<p>Licensed CC0-1.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Data</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/bioimage-archive/galleries/vem.html">https://www.ebi.ac.uk/bioimage-archive/galleries/vem.html</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-data-analysis">
<h2>BioImage Data Analysis<a class="headerlink" href="#bioimage-data-analysis" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://analyticalscience.wiley.com/do/10.1002/was.00050003/full/bioimagedataanalysis.pdf">https://analyticalscience.wiley.com/do/10.1002/was.00050003/full/bioimagedataanalysis.pdf</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-informatics-index-training-materials">
<h2>BioImage Informatics Index Training Materials<a class="headerlink" href="#bioimage-informatics-index-training-materials" title="Link to this heading">#</a></h2>
<p>Licensed ODC-BY-1.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://biii.eu/training-material">https://biii.eu/training-material</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-model-zoo">
<h2>Bioimage Model Zoo<a class="headerlink" href="#bioimage-model-zoo" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://bioimage.io/">https://bioimage.io/</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-analysis-with-napari">
<h2>Bioimage analysis with Napari<a class="headerlink" href="#bioimage-analysis-with-napari" title="Link to this heading">#</a></h2>
<p>Fabrizio Musacchio</p>
<p>Licensed CC-BY-ND-SA-4.0</p>
<p>Tags: Python, Napari, Bioimage Analysis</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://www.fabriziomusacchio.com/teaching/teaching_bioimage_analysis/">https://www.fabriziomusacchio.com/teaching/teaching_bioimage_analysis/</a></p>
</section>
<hr class="docutils" />
<section id="bridging-imaging-users-to-imaging-analysis-a-community-survey">
<h2>Bridging Imaging Users to Imaging Analysis - A community survey<a class="headerlink" href="#bridging-imaging-users-to-imaging-analysis-a-community-survey" title="Link to this heading">#</a></h2>
<p>Suganya Sivagurunathan, Stefania Marcotti, Carl J Nelson, Martin L Jones, David J Barry, Thomas J A Slater, Kevin W Eliceiri, Beth A Cimini</p>
<p>Published 2023</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication, Preprint</p>
<p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2023.06.05.543701v1">https://www.biorxiv.org/content/10.1101/2023.06.05.543701v1</a></p>
<p><a class="github reference external" href="https://github.com/COBA-NIH/2023_ImageAnalysisSurvey">COBA-NIH/2023_ImageAnalysisSurvey</a></p>
</section>
<hr class="docutils" />
<section id="bring-your-own-data-workshops">
<h2>Bring your own data workshops<a class="headerlink" href="#bring-your-own-data-workshops" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis, Research Data Management</p>
<p>Content type: Workshop</p>
<p><a class="reference external" href="https://www.dtls.nl/fair-data/byod/">https://www.dtls.nl/fair-data/byod/</a></p>
</section>
<hr class="docutils" />
<section id="building-a-bioimage-analysis-workflow-using-deep-learning">
<h2>Building a Bioimage Analysis Workflow using Deep Learning<a class="headerlink" href="#building-a-bioimage-analysis-workflow-using-deep-learning" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/esgomezm/NEUBIAS_chapter_DL_2020">esgomezm/NEUBIAS_chapter_DL_2020</a></p>
</section>
<hr class="docutils" />
<section id="care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022">
<h2>CARE/Stardist tutorials for EMBO Practical Course — Computational optical biology 2022<a class="headerlink" href="#care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022" title="Link to this heading">#</a></h2>
<p>Martin Weigert</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/embo_2022">maweigert/embo_2022</a></p>
</section>
<hr class="docutils" />
<section id="clij-gpu-accelerated-image-processing-for-everyone">
<h2>CLIJ: GPU-accelerated image processing for everyone<a class="headerlink" href="#clij-gpu-accelerated-image-processing-for-everyone" title="Link to this heading">#</a></h2>
<p>Robert Haase, Loic Royer, et al.</p>
<p>Published 2020</p>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>CLIJ is a collection of image processing functions that use graphics processing units for accelerated processing.</p>
<p>Tags: Imagej, Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1038/s41592-019-0650-1">https://doi.org/10.1038/s41592-019-0650-1</a></p>
</section>
<hr class="docutils" />
<section id="coba-center-for-open-bioimage-analysis-youtube-channel">
<h2>COBA: Center for Open Bioimage Analysis YouTube Channel<a class="headerlink" href="#coba-center-for-open-bioimage-analysis-youtube-channel" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://www.youtube.com/&#64;cobacenterforopenbioimagea1864">https://www.youtube.com/&#64;cobacenterforopenbioimagea1864</a></p>
</section>
<hr class="docutils" />
<section id="csbdeep-and-stardist-i2k-2020">
<h2>CSBDeep and StarDist &#64; I2K 2020<a class="headerlink" href="#csbdeep-and-stardist-i2k-2020" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Uwe Schmidt</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/stardist-i2k">maweigert/stardist-i2k</a></p>
</section>
<hr class="docutils" />
<section id="cell-tracking-challenge-2d-datasets">
<h2>Cell Tracking Challenge - 2D Datasets<a class="headerlink" href="#cell-tracking-challenge-2d-datasets" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Data</p>
<p><a class="reference external" href="http://celltrackingchallenge.net/2d-datasets/">http://celltrackingchallenge.net/2d-datasets/</a></p>
</section>
<hr class="docutils" />
<section id="cell-tracking-challenge-3d-datasets">
<h2>Cell Tracking Challenge - 3D Datasets<a class="headerlink" href="#cell-tracking-challenge-3d-datasets" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Data</p>
<p><a class="reference external" href="http://celltrackingchallenge.net/3d-datasets/">http://celltrackingchallenge.net/3d-datasets/</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-introduction">
<h2>CellProfiler Introduction<a class="headerlink" href="#cellprofiler-introduction" title="Link to this heading">#</a></h2>
<p>Anna Klemm</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Cellprofiler, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/ahklemm/CellProfiler_Introduction">ahklemm/CellProfiler_Introduction</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-practical-neubias-lisbon-2017">
<h2>CellProfiler Practical NeuBIAS Lisbon 2017<a class="headerlink" href="#cellprofiler-practical-neubias-lisbon-2017" title="Link to this heading">#</a></h2>
<p>Carolina Wählby, Maxime Bombrun, Christian Tischer</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Cellprofiler, Bioimage Analysis</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/tischi/cellprofiler-practical-NeuBIAS-Lisbon-2017">tischi/cellprofiler-practical-NeuBIAS-Lisbon-2017</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-tutorials">
<h2>CellProfiler tutorials<a class="headerlink" href="#cellprofiler-tutorials" title="Link to this heading">#</a></h2>
<p>Licensed BSD-3-CLAUSE</p>
<p>CellProfiler tutorials and guided exercises about translocation, segmentation, pixel-based classification and quality control</p>
<p>Tags: Cellprofiler, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/CellProfiler/tutorials">CellProfiler/tutorials</a></p>
</section>
<hr class="docutils" />
<section id="challenges-and-opportunities-for-bio-image-analysis-core-facilities">
<h2>Challenges and opportunities for bio-image analysis core-facilities<a class="headerlink" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/12-1054">https://f1000research.com/slides/12-1054</a></p>
</section>
<hr class="docutils" />
<section id="challenges-and-opportunities-for-bioimage-analysis-core-facilities">
<h2>Challenges and opportunities for bioimage analysis core-facilities<a class="headerlink" href="#challenges-and-opportunities-for-bioimage-analysis-core-facilities" title="Link to this heading">#</a></h2>
<p>Johannes Richard Soltwedel, Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>This article outlines common reasons for founding bioimage analysis core-facilities, services they can provide to fulfill certain need and conflicts of interest that arise from these services.</p>
<p>Tags: Bioimage Analysis, Research Data Management</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://onlinelibrary.wiley.com/doi/full/10.1111/jmi.13192">https://onlinelibrary.wiley.com/doi/full/10.1111/jmi.13192</a></p>
</section>
<hr class="docutils" />
<section id="checklists-for-publishing-images-and-image-analysis">
<h2>Checklists for publishing images and image analysis<a class="headerlink" href="#checklists-for-publishing-images-and-image-analysis" title="Link to this heading">#</a></h2>
<p>Christopher Schmied</p>
<p>Published 2023-09-14</p>
<p>Licensed CC0-1.0</p>
<p>In this paper we introduce two sets of checklists. One is concerned with the publication of images. The other one gives instructions for the publication of image analysis.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Forum Post</p>
<p><a class="reference external" href="https://forum.image.sc/t/checklists-for-publishing-images-and-image-analysis/86304">https://forum.image.sc/t/checklists-for-publishing-images-and-image-analysis/86304</a></p>
</section>
<hr class="docutils" />
<section id="chris-halvin-youtube-channel">
<h2>Chris Halvin YouTube channel<a class="headerlink" href="#chris-halvin-youtube-channel" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: Napari, Python, Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://www.youtube.com/&#64;chrishavlin">https://www.youtube.com/&#64;chrishavlin</a></p>
<p><a class="reference external" href="https://www.youtube.com/playlist?list=PLqbhAmYZU5KxuAcnNBIxyBkivUEiKswq1">https://www.youtube.com/playlist?list=PLqbhAmYZU5KxuAcnNBIxyBkivUEiKswq1</a></p>
</section>
<hr class="docutils" />
<section id="collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis">
<h2>Collection of teaching material for deep learning for (biomedical) image analysis<a class="headerlink" href="#collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis" title="Link to this heading">#</a></h2>
<p>Constantin Pape</p>
<p>Licensed MIT</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/constantinpape/dl-teaching-resources">constantinpape/dl-teaching-resources</a></p>
</section>
<hr class="docutils" />
<section id="community-developed-checklists-for-publishing-images-and-image-analyses">
<h2>Community-developed checklists for publishing images and image analyses<a class="headerlink" href="#community-developed-checklists-for-publishing-images-and-image-analyses" title="Link to this heading">#</a></h2>
<p>Beth Cimini et al.</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This book is a companion to the Nature Methods publication Community-developed checklists for publishing images and image analyses. In this paper, members of QUAREP-LiMi have proposed 3 sets of standards for publishing image figures and image analysis - minimal requirements, recommended additions, and ideal comprehensive goals. By following this guidance, we hope to remove some of the stress non-experts may face in determining what they need to do, and we also believe that researchers will find their science more interpretable and more reproducible.</p>
<p>Tags: Bioimage Analysis, Research Data Management</p>
<p>Content type: Notebook, Collection</p>
<p><a class="reference external" href="https://quarep-limi.github.io/WG12_checklists_for_image_publishing/intro.html">https://quarep-limi.github.io/WG12_checklists_for_image_publishing/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="course-on-deep-learning-for-imaging-using-pytorch">
<h2>Course on Deep Learning for imaging using PyTorch<a class="headerlink" href="#course-on-deep-learning-for-imaging-using-pytorch" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/DLImaging">guiwitz/DLImaging</a></p>
</section>
<hr class="docutils" />
<section id="cultivating-open-training">
<h2>Cultivating Open Training<a class="headerlink" href="#cultivating-open-training" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-03-14</p>
<p>Licensed CC-BY-4.0</p>
<p>In this SaxFDM Digital Kitchen, I introduced current challenges and potential solutions for openly sharing training materials, softly focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and uploading materials to publicly acessible repositories are proposed to fill this gap.</p>
<p>Tags: Open Science, Research Data Management, FAIR-Principles, Bioimage Analysis, Licensing</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10816895">https://zenodo.org/records/10816895</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10816895">https://doi.org/10.5281/zenodo.10816895</a></p>
</section>
<hr class="docutils" />
<section id="deep-napari-napari-as-a-tool-for-deep-learning-project-management">
<h2>DEEP NAPARI : Napari as a tool for deep learning project management<a class="headerlink" href="#deep-napari-napari-as-a-tool-for-deep-learning-project-management" title="Link to this heading">#</a></h2>
<p>Herearii Metuarea, David Rousseau, Pejman Rasti, Valentin Gilet</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/hereariim/DEEP-NAPARI">hereariim/DEEP-NAPARI</a></p>
</section>
<hr class="docutils" />
<section id="dl4miceverywhere">
<h2>DL4MicEverywhere<a class="headerlink" href="#dl4miceverywhere" title="Link to this heading">#</a></h2>
<p>Iván Hidalgo, et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/HenriquesLab/DL4MicEverywhere">HenriquesLab/DL4MicEverywhere</a></p>
</section>
<hr class="docutils" />
<section id="dl-mbl-2021-exercises">
<h2>DL&#64;MBL 2021 Exercises<a class="headerlink" href="#dl-mbl-2021-exercises" title="Link to this heading">#</a></h2>
<p>Jan Funke, Constantin Pape, Morgan Schwartz, Xiaoyan</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/JLrumberger/DL-MBL-2021">JLrumberger/DL-MBL-2021</a></p>
</section>
<hr class="docutils" />
<section id="dask-course">
<h2>Dask Course<a class="headerlink" href="#dask-course" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Bioimage Analysis, Big Data</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/DaskCourse">guiwitz/DaskCourse</a></p>
</section>
<hr class="docutils" />
<section id="data-management-at-france-bioimaging">
<h2>Data management at France BioImaging<a class="headerlink" href="#data-management-at-france-bioimaging" title="Link to this heading">#</a></h2>
<p>Published 2023-07-05</p>
<p>Licensed CC-BY-SA-4.0</p>
<p>Tags: Research Data Management, Bioimage Analysis, Open Science</p>
<p>Content type: Slides, Presentation</p>
<p><a class="reference external" href="https://omero-fbi.fr/slides/elmi23_cfd/main.html#/title-slide">https://omero-fbi.fr/slides/elmi23_cfd/main.html#/title-slide</a></p>
</section>
<hr class="docutils" />
<section id="deep-learning-for-image-analysis-exercises">
<h2>Deep Learning for image analysis - Exercises<a class="headerlink" href="#deep-learning-for-image-analysis-exercises" title="Link to this heading">#</a></h2>
<p>Martin Weigert</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Fiji, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/zidas_2020_DL_intro_Part_2">maweigert/zidas_2020_DL_intro_Part_2</a></p>
</section>
<hr class="docutils" />
<section id="deepprofiler-handbook">
<h2>DeepProfiler Handbook<a class="headerlink" href="#deepprofiler-handbook" title="Link to this heading">#</a></h2>
<p>Michael Bornholdt, Juan Caicedo, Yu Han, Nikita Moshkov, Rebecca Senft</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/cytomining/DeepProfiler-handbook">cytomining/DeepProfiler-handbook</a></p>
<p><a class="reference external" href="https://cytomining.github.io/DeepProfiler-handbook/docs/00-welcome.html">https://cytomining.github.io/DeepProfiler-handbook/docs/00-welcome.html</a></p>
</section>
<hr class="docutils" />
<section id="developing-semi-automatic-analysis-pipelines-and-technological-solutions-for-metadata-annotation-and-management-in-high-content-screening-hcs-bioimaging">
<h2>Developing (semi)automatic analysis pipelines and technological solutions for metadata annotation and management in high-content screening (HCS) bioimaging<a class="headerlink" href="#developing-semi-automatic-analysis-pipelines-and-technological-solutions-for-metadata-annotation-and-management-in-high-content-screening-hcs-bioimaging" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Stefan Scholz, Wibke Busch, Thomas Schnike, Hannes Bohring, Jan Bumberger</p>
<p>Licensed CC-BY-4.0</p>
<p>High-content screening (HCS) bioimaging automates the imaging and analysis of numerous biological samples, generating extensive metadata that is crucial for effective image management and interpretation. Efficiently handling this complex data is essential to implementing FAIR principles and realizing HCS’s full potential for scientific discoveries.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8434325">https://doi.org/10.5281/zenodo.8434325</a></p>
</section>
<hr class="docutils" />
<section id="diffusion-models-for-image-restoration-an-introduction">
<h2>Diffusion Models for Image Restoration - An Introduction<a class="headerlink" href="#diffusion-models-for-image-restoration-an-introduction" title="Link to this heading">#</a></h2>
<p>Anirban Ray</p>
<p>Licensed UNKNOWN</p>
<p>Presentation given at the EMBO-DL4MIA 2024, Advanced Topic Seminar, May-11-2024</p>
<p>Tags: Bioimage Analysis, Diffusion Models, Tu Dresden</p>
<p>Content type: Presentation</p>
<p><a class="reference external" href="https://drive.google.com/file/d/1pPVUUMi5w2Ojw_SaBzSQVaXUuIKtQ7Ma/view">https://drive.google.com/file/d/1pPVUUMi5w2Ojw_SaBzSQVaXUuIKtQ7Ma/view</a></p>
</section>
<hr class="docutils" />
<section id="dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy">
<h2>Dr Guillaume Jacquemet on studying cancer cell metastasis in the era of deep learning for microscopy<a class="headerlink" href="#dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy" title="Link to this heading">#</a></h2>
<p>Guillaume Jacquemet</p>
<p>Published 2024-10-24</p>
<p>Licensed UNKNOWN</p>
<p>Leukocyte extravasation is a critical component of the innate immune response, while circulating tumour cell extravasation is a crucial step in metastasis formation. Despite their importance, these extravasation mechanisms remain incompletely understood. In this talk, Guillaume Jacquemet presents a novel imaging framework that integrates microfluidics with high-speed, label-free imaging to study the arrest of pancreatic cancer cells (PDAC) on human endothelial layers under physiological flow conditions.</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Video, Slides</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=KTdZBgSCYJQ">https://www.youtube.com/watch?v=KTdZBgSCYJQ</a></p>
</section>
<hr class="docutils" />
<section id="edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats">
<h2>EDAM-bioimaging - The ontology of bioimage informatics operations, topics, data, and formats<a class="headerlink" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats" title="Link to this heading">#</a></h2>
<p>Matúš Kalaš et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>EDAM-bioimaging is an extension of the EDAM ontology, dedicated to bioimage analysis, bioimage informatics, and bioimaging.</p>
<p>Tags: Ontology, Bioimage Analysis</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://hal.science/hal-02267597/document">https://hal.science/hal-02267597/document</a></p>
</section>
<hr class="docutils" />
<section id="embo-practical-course-advanced-methods-in-bioimage-analysis">
<h2>EMBO Practical Course Advanced methods in bioimage analysis<a class="headerlink" href="#embo-practical-course-advanced-methods-in-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Event</p>
<p><a class="reference external" href="https://www.embl.org/about/info/course-and-conference-office/events/bia23-01/">https://www.embl.org/about/info/course-and-conference-office/events/bia23-01/</a></p>
</section>
<hr class="docutils" />
<section id="epflx-image-processing-and-analysis-for-life-scientists">
<h2>EPFLx: Image Processing and Analysis for Life Scientists<a class="headerlink" href="#epflx-image-processing-and-analysis-for-life-scientists" title="Link to this heading">#</a></h2>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Online Tutorial</p>
<p><a class="reference external" href="https://www.edx.org/learn/image-analysis/ecole-polytechnique-federale-de-lausanne-image-processing-and-analysis-for-life-scientists">https://www.edx.org/learn/image-analysis/ecole-polytechnique-federale-de-lausanne-image-processing-and-analysis-for-life-scientists</a></p>
</section>
<hr class="docutils" />
<section id="embedseg-repository">
<h2>EmbedSeg Repository<a class="headerlink" href="#embedseg-repository" title="Link to this heading">#</a></h2>
<p>Manan Lalit, Joran Deschamps, Florian Jug, Ajinkya Kulkarni</p>
<p>Licensed CC-BY-NC-4.0</p>
<p>Code Implementation for EmbedSeg, an Instance Segmentation Method for Microscopy Images</p>
<p>Tags: Bioimage Analysis, Instance Segmentation</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/juglab/EmbedSeg">juglab/EmbedSeg</a></p>
</section>
<hr class="docutils" />
<section id="erick-martins-ratamero-expanding-the-ome-ecosystem-for-imaging-data-management-scipy-2024">
<h2>Erick Martins Ratamero - Expanding the OME ecosystem for imaging data management | SciPy 2024<a class="headerlink" href="#erick-martins-ratamero-expanding-the-ome-ecosystem-for-imaging-data-management-scipy-2024" title="Link to this heading">#</a></h2>
<p>SciPy, Erick Martins Ratamero</p>
<p>Published 2024-08-19</p>
<p>Licensed YOUTUBE STANDARD LICENSE</p>
<p>Tags: OMERO, Bioimage Analysis</p>
<p>Content type: Video, Presentation</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=GmhyDNm1RsM">https://www.youtube.com/watch?v=GmhyDNm1RsM</a></p>
</section>
<hr class="docutils" />
<section id="eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus">
<h2>EuBIAS course 2013: Intensity Dynamics at the Periphery of Nucleus<a class="headerlink" href="#eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Published 2013</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Tutorial, Book</p>
<p><a class="github reference external" href="https://github.com/miura/BIAS_Nucleus_Segmentation/blob/master/module9.pdf">miura/BIAS_Nucleus_Segmentation</a></p>
</section>
<hr class="docutils" />
<section id="euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">
<h2>Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks<a class="headerlink" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks" title="Link to this heading">#</a></h2>
<p>Isabel Kemmer, Euro-BioImaging ERIC</p>
<p>Published 2024-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Hands-on exercises on FAIR Bioimage Data from the interactive online workshop “Euro-BioImaging’s Guide to FAIR BioImage Data 2024” (<a class="reference external" href="https://www.eurobioimaging.eu/news/a-guide-to-fair-bioimage-data-2024/">https://www.eurobioimaging.eu/news/a-guide-to-fair-bioimage-data-2024/</a>).  Types of tasks included: FAIR characteristics of a real world dataset Data Management Plan (DMP) Journal Policies on FAIR data sharing Ontology search Metadata according to REMBI scheme (Image from: Sarkans, U., Chiu, W., Collinson, L. et al. REMBI: Recommended Metadata for Biological Images—enabling reuse of microscopy data in biology. Nat Methods 18, 1418–1422 (2021). <a class="reference external" href="https://doi.org/10.1038/s41592-021-01166-8">https://doi.org/10.1038/s41592-021-01166-8</a>) Matching datasets to bioimage repositories Browsing bioimage repositories</p>
<p>Tags: Bioimage Analysis, FAIR-Principles, Research Data Management</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="reference external" href="https://zenodo.org/records/11474407">https://zenodo.org/records/11474407</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11474407">https://doi.org/10.5281/zenodo.11474407</a></p>
</section>
<hr class="docutils" />
<section id="euro-bioimaging-s-template-for-research-data-management-plans">
<h2>Euro-BioImaging’s Template for Research Data Management Plans<a class="headerlink" href="#euro-bioimaging-s-template-for-research-data-management-plans" title="Link to this heading">#</a></h2>
<p>Isabel Kemmer, Euro-BioImaging ERIC</p>
<p>Published 2024-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Euro-BioImaging has developed a Data Management Plan (DMP) template with questions tailored to bioimaging research projects. Outlining data management practices in this way ensures traceability of project data, allowing for a continuous and unambiguous flow of information throughout the research project. This template can be used to satisfy the requirement to submit a DMP to certain funders. Regardless of the funder, Euro-BioImaging users are encouraged to provide a DMP and can use this template accordingly. 
This DMP template is available as a fillable PDF with further instructions and sample responses available by hovering over the fillable fields. </p>
<p>Tags: Bioimage Analysis, FAIR-Principles, Research Data Management</p>
<p>Content type: Collection, Tutorial</p>
<p><a class="reference external" href="https://zenodo.org/records/11473803">https://zenodo.org/records/11473803</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11473803">https://doi.org/10.5281/zenodo.11473803</a></p>
</section>
<hr class="docutils" />
<section id="example-pipeline-tutorial">
<h2>Example Pipeline Tutorial<a class="headerlink" href="#example-pipeline-tutorial" title="Link to this heading">#</a></h2>
<p>Tim Monko</p>
<p>Published 2024-10-28</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Napari-ndev is a collection of widgets intended to serve any person seeking to process microscopy images from start to finish. The goal of this example pipeline is to get the user familiar with working with napari-ndev for batch processing and reproducibility (view Image Utilities and Workflow Widget).</p>
<p>Tags: Napari, Bioimage Analysis</p>
<p>Content type: Documentation, Github Repository, Tutorial</p>
<p><a class="reference external" href="https://timmonko.github.io/napari-ndev/tutorial/01_example_pipeline/">https://timmonko.github.io/napari-ndev/tutorial/01_example_pipeline/</a></p>
<p><a class="github reference external" href="https://github.com/timmonko/napari-ndev">timmonko/napari-ndev</a></p>
</section>
<hr class="docutils" />
<section id="fair-bioimage-data">
<h2>FAIR BioImage Data<a class="headerlink" href="#fair-bioimage-data" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Fair, Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=8zd4KTy-oYI&amp;amp;list=PLW-oxncaXRqU4XqduJzwFHvWLF06PvdVm">https://www.youtube.com/watch?v=8zd4KTy-oYI&amp;list=PLW-oxncaXRqU4XqduJzwFHvWLF06PvdVm</a></p>
</section>
<hr class="docutils" />
<section id="fairy-deep-learning-for-bioimage-analysis">
<h2>FAIRy deep-learning for bioImage analysis<a class="headerlink" href="#fairy-deep-learning-for-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez de Mariscal</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to FAIR deep learning. Furthermore, tools to deploy trained DL models (deepImageJ), easily train and evaluate them (ZeroCostDL4Mic and DeepBacs) ensure reproducibility (DL4MicEverywhere), and share this technology in an open-source and reproducible manner (BioImage Model Zoo) are introduced.</p>
<p>Tags: Artificial Intelligence, FAIR-Principles, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/13-147">https://f1000research.com/slides/13-147</a></p>
</section>
<hr class="docutils" />
<section id="feature-extraction-in-napari">
<h2>Feature extraction in napari<a class="headerlink" href="#feature-extraction-in-napari" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/05/03/feature-extraction-in-napari/">https://focalplane.biologists.com/2023/05/03/feature-extraction-in-napari/</a></p>
</section>
<hr class="docutils" />
<section id="fiji-is-just-imagej-tutorials">
<h2>Fiji Is Just ImageJ Tutorials<a class="headerlink" href="#fiji-is-just-imagej-tutorials" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://www.youtube.com/playlist?list=PL5Edc1v41fyCLFZbBCLo41zFO-_cXBfAb">https://www.youtube.com/playlist?list=PL5Edc1v41fyCLFZbBCLo41zFO-_cXBfAb</a></p>
</section>
<hr class="docutils" />
<section id="fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging">
<h2>Fit for OMERO: How imaging facilities and IT departments work together to enable RDM for bioimaging<a class="headerlink" href="#fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging" title="Link to this heading">#</a></h2>
<p>Starts Oct 16, 2024, 9:00 AM, Ends Oct 17, 2024, 5:00 PM</p>
<p>Tags: Bioimage Analysis, OMERO, Research Data Management</p>
<p>Content type: Workshop</p>
<p><a class="reference external" href="https://indico.dkfz.de/event/1022/">https://indico.dkfz.de/event/1022/</a></p>
</section>
<hr class="docutils" />
<section id="fundamentals-in-digital-image-processing">
<h2>Fundamentals in digital image processing<a class="headerlink" href="#fundamentals-in-digital-image-processing" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/Fundamentals_image_processing">guiwitz/Fundamentals_image_processing</a></p>
</section>
<hr class="docutils" />
<section id="fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends">
<h2>Fundamentals of image analysis in Python with scikit-image, napari, and friends<a class="headerlink" href="#fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/jni/halfway-to-i2k-skimage-napari">jni/halfway-to-i2k-skimage-napari</a></p>
</section>
<hr class="docutils" />
<section id="galaxy-training">
<h2>Galaxy Training<a class="headerlink" href="#galaxy-training" title="Link to this heading">#</a></h2>
<p>Published None</p>
<p>Licensed CC-BY-4.0</p>
<p>Collection of tutorials developed and maintained by the worldwide Galaxy community.</p>
<p>Tags: Bioimage Analysis, Data Analysis</p>
<p>Content type: Collection, Tutorial</p>
<p><a class="reference external" href="https://training.galaxyproject.org/">https://training.galaxyproject.org/</a></p>
</section>
<hr class="docutils" />
<section id="generative-artificial-intelligence-for-bio-image-analysis">
<h2>Generative artificial intelligence for bio-image analysis<a class="headerlink" href="#generative-artificial-intelligence-for-bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/12-971">https://f1000research.com/slides/12-971</a></p>
</section>
<hr class="docutils" />
<section id="global-bioimaging-training-database">
<h2>Global BioImaging Training Database<a class="headerlink" href="#global-bioimaging-training-database" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Event</p>
<p><a class="reference external" href="https://globalbioimaging.org/international-training-courses">https://globalbioimaging.org/international-training-courses</a></p>
</section>
<hr class="docutils" />
<section id="high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero">
<h2>High throughput &amp; automated data analysis and data management workflow with Cellprofiler and OMERO<a class="headerlink" href="#high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero" title="Link to this heading">#</a></h2>
<p>Sarah Weischer, Jens Wendt, Thomas Zobel</p>
<p>Licensed CC-BY-4.0</p>
<p>In this workshop a fully integrated data analysis solutions employing OMERO and commonly applied image analysis tools (e.g., CellProfiler, Fiji) using existing python interfaces (OMERO Python language bindings, ezOmero, Cellprofiler Python API) is presented.</p>
<p>Tags: OMERO, Data Analysis, Bioimage Analysis</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8139353">https://zenodo.org/doi/10.5281/zenodo.8139353</a></p>
</section>
<hr class="docutils" />
<section id="highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists">
<h2>Highlights from the 2016-2020 NEUBIAS training schools for Bioimage Analysts: a success story and key asset for analysts and life scientists<a class="headerlink" href="#highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists" title="Link to this heading">#</a></h2>
<p>Gabriel G. Martins, Fabrice P. Cordelières, Julien Colombelli, Rocco D’Antuono, Ofra Golani, Romain Guiet, Robert Haase, Anna H. Klemm, Marion Louveaux, Perrine Paul-Gilloteaux, Jean-Yves Tinevez, Kota Miura</p>
<p>Published 2021</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis, Neubias</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://f1000research.com/articles/10-334/v1">https://f1000research.com/articles/10-334/v1</a></p>
</section>
<hr class="docutils" />
<section id="hitchhiking-through-a-diverse-bio-image-analysis-software-universe">
<h2>Hitchhiking through a diverse Bio-image Analysis Software Universe<a class="headerlink" href="#hitchhiking-through-a-diverse-bio-image-analysis-software-universe" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2022-07-22</p>
<p>Licensed CC-BY-4.0</p>
<p>Overview about decision making and how to influence decisions in the bio-image analysis software context.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Slides, Presentation</p>
<p><a class="reference external" href="https://f1000research.com/slides/11-746">https://f1000research.com/slides/11-746</a></p>
<p><a class="reference external" href="https://doi.org/10.7490/f1000research.1119026.1">https://doi.org/10.7490/f1000research.1119026.1</a></p>
</section>
<hr class="docutils" />
<section id="i2k-2024-clesperanto-gpu-accelerated-image-processing-library">
<h2>I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library<a class="headerlink" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library" title="Link to this heading">#</a></h2>
<p>Stephane Rigaud, Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Course and material for the clEsperanto workshop presented at I2K 2024 &#64; Human Technopol (Milan, Italy). The workshop is an hands-on demo of the clesperanto project, focussing on how to use the library for users who want use GPU-acceleration for their Image Processing pipeline.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Workshop, Tutorial, Notebook</p>
<p><a class="github reference external" href="https://github.com/StRigaud/clesperanto_workshop_I2K24?tab=readme-ov-file">StRigaud/clesperanto_workshop_I2K24</a></p>
</section>
<hr class="docutils" />
<section id="i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">
<h2>I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark<a class="headerlink" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark" title="Link to this heading">#</a></h2>
<p>Stephan Saalfeld, Tobias Pietzsch</p>
<p>Published None</p>
<p>Licensed APACHE-2.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Notebook, Github Repository</p>
<p><a class="reference external" href="https://saalfeldlab.github.io/i2k2024-lazy-workshop/">https://saalfeldlab.github.io/i2k2024-lazy-workshop/</a></p>
<p><a class="github reference external" href="https://github.com/saalfeldlab/i2k2024-lazy-workshop">saalfeldlab/i2k2024-lazy-workshop</a></p>
</section>
<hr class="docutils" />
<section id="i2k2024-virtual-bio-image-analysis-code-generation">
<h2>I2K2024(virtual) - Bio-Image Analysis Code Generation<a class="headerlink" href="#i2k2024-virtual-bio-image-analysis-code-generation" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This repository contains training materials for the Tutorial “Bio-Image Analysis Code Generation” at the From Images To Knowledge (I2K) Conference (virtual) October 28th-30th 2024.</p>
<p>Tags: Bioimage Analysis, Notebooks, Biabob</p>
<p>Content type: Github Repository, Tutorial, Notebook</p>
<p><a class="github reference external" href="https://github.com/haesleinhuepf/i2k2024-ai-code-generation">haesleinhuepf/i2k2024-ai-code-generation</a></p>
</section>
<hr class="docutils" />
<section id="iafig-rms-python-for-bioimage-analysis-course">
<h2>IAFIG-RMS Python for Bioimage Analysis Course<a class="headerlink" href="#iafig-rms-python-for-bioimage-analysis-course" title="Link to this heading">#</a></h2>
<p>Aurelien Barbotin, Chas Nelson, Dominic Waithe, Ola (Alexandra) Tarkowska, Mikolaj Kundegorski, Stephen Cross, Todd Fallesen</p>
<p>Licensed GPL-3.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/RMS-DAIM/Python-for-Bioimage-Analysis">RMS-DAIM/Python-for-Bioimage-Analysis</a></p>
</section>
<hr class="docutils" />
<section id="itkelastix-examples">
<h2>ITKElastix Examples<a class="headerlink" href="#itkelastix-examples" title="Link to this heading">#</a></h2>
<p>Licensed APACHE-2.0</p>
<p>Tags: Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/InsightSoftwareConsortium/ITKElastix/tree/main/examples">InsightSoftwareConsortium/ITKElastix</a></p>
</section>
<hr class="docutils" />
<section id="ibiology-bioimage-analysis-course-the-life-cycle-of-an-image-data-set">
<h2>Ibiology. Bioimage Analysis Course. The Life Cycle of an Image Data Set<a class="headerlink" href="#ibiology-bioimage-analysis-course-the-life-cycle-of-an-image-data-set" title="Link to this heading">#</a></h2>
<p>Licensed CC BY-NC-ND 3.0 DEED</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://www.ibiology.org/online-biology-courses/bioimage-analysis-course/">https://www.ibiology.org/online-biology-courses/bioimage-analysis-course/</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-training-resources">
<h2>Image Analysis Training Resources<a class="headerlink" href="#image-analysis-training-resources" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://neubias.github.io/training-resources/">https://neubias.github.io/training-resources/</a></p>
</section>
<hr class="docutils" />
<section id="image-processing-with-python">
<h2>Image Processing with Python<a class="headerlink" href="#image-processing-with-python" title="Link to this heading">#</a></h2>
<p>Mark Meysenburg, Toby Hodges, Dominik Kutra, Erin Becker, David Palmquist, et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>This lesson shows how to use Python and scikit-image to do basic image processing.</p>
<p>Tags: Bioimage Analysis, Python</p>
<p>Content type: Tutorial, Workflow</p>
<p><a class="reference external" href="https://datacarpentry.org/image-processing/key-points.html">https://datacarpentry.org/image-processing/key-points.html</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends">
<h2>Image analysis and visualization in Python with scikit-image, napari, and friends<a class="headerlink" href="#image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/scipy-2023-image-analysis/tutorial">scipy-2023-image-analysis/tutorial</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-in-galaxy">
<h2>Image analysis in Galaxy<a class="headerlink" href="#image-analysis-in-galaxy" title="Link to this heading">#</a></h2>
<p>Beatriz Serrano-Solano, Björn Grüning</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1WG_4307XmKsGfWT3taxMvX2rZiG1k0SM1E7SAENJQkI/edit#slide=id.p">https://docs.google.com/presentation/d/1WG_4307XmKsGfWT3taxMvX2rZiG1k0SM1E7SAENJQkI/edit#slide=id.p</a></p>
</section>
<hr class="docutils" />
<section id="image-processing-for-beginners">
<h2>Image processing for beginners<a class="headerlink" href="#image-processing-for-beginners" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/PyImageCourse_beginner">guiwitz/PyImageCourse_beginner</a></p>
</section>
<hr class="docutils" />
<section id="image-based-profiling-handbook">
<h2>Image-based Profiling Handbook<a class="headerlink" href="#image-based-profiling-handbook" title="Link to this heading">#</a></h2>
<p>Beth Cimini, Tim Becker, Shantanu Singh, Gregory Way, Hamdah Abbasi, Callum Tromans-Coia</p>
<p>Licensed CC0-1.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/cytomining/profiling-handbook">cytomining/profiling-handbook</a></p>
<p><a class="reference external" href="https://cytomining.github.io/profiling-handbook/">https://cytomining.github.io/profiling-handbook/</a></p>
</section>
<hr class="docutils" />
<section id="imagej-macro-introduction">
<h2>ImageJ Macro Introduction<a class="headerlink" href="#imagej-macro-introduction" title="Link to this heading">#</a></h2>
<p>Anna Klemm</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Imagej Macro, Bioimage Analysis</p>
<p>Content type: Slides, Code</p>
<p><a class="github reference external" href="https://github.com/ahklemm/ImageJMacro_Introduction">ahklemm/ImageJMacro_Introduction</a></p>
</section>
<hr class="docutils" />
<section id="imagej2-api-beating">
<h2>ImageJ2 API-beating<a class="headerlink" href="#imagej2-api-beating" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_imagej2_dev">https://git.mpi-cbg.de/rhaase/lecture_imagej2_dev</a></p>
</section>
<hr class="docutils" />
<section id="imaris-tutorials">
<h2>Imaris Tutorials<a class="headerlink" href="#imaris-tutorials" title="Link to this heading">#</a></h2>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://imaris.oxinst.com/tutorials">https://imaris.oxinst.com/tutorials</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-bioimage-analysis">
<h2>Introduction to Bioimage Analysis<a class="headerlink" href="#introduction-to-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Pete Bankhead</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Imagej, Bioimage Analysis</p>
<p>Content type: Book, Notebook</p>
<p><a class="reference external" href="https://bioimagebook.github.io/index.html">https://bioimagebook.github.io/index.html</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-image-analysis-with-fiji">
<h2>Introduction to Image Analysis with Fiji<a class="headerlink" href="#introduction-to-image-analysis-with-fiji" title="Link to this heading">#</a></h2>
<p>Robert Haase, Benoit Lombardot</p>
<p>Licensed CC-BY-NC-4.0</p>
<p>Tags: Imagej, Fiji, Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/mpicbg-scicomp/CourseIntroToIA">mpicbg-scicomp/CourseIntroToIA</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden">
<h2>Introduction to ImageJ macro programming, Scientific Computing Facility, MPI CBG Dresden<a class="headerlink" href="#introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden" title="Link to this heading">#</a></h2>
<p>Robert Haase, Benoit Lombardot</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/scicomp/bioimage_team/coursematerialimageanalysis/tree/master/ImageJMacro_24h_2017-01">https://git.mpi-cbg.de/scicomp/bioimage_team/coursematerialimageanalysis/tree/master/ImageJMacro_24h_2017-01</a></p>
</section>
<hr class="docutils" />
<section id="jupyter-for-interactive-cloud-computing">
<h2>Jupyter for interactive cloud computing<a class="headerlink" href="#jupyter-for-interactive-cloud-computing" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1q8q1xE-c35tvCsRXZay98s2UYWwXpp0cfCljBmMFpco/edit#slide=id.ga456d5535c_2_53">https://docs.google.com/presentation/d/1q8q1xE-c35tvCsRXZay98s2UYWwXpp0cfCljBmMFpco/edit#slide=id.ga456d5535c_2_53</a></p>
</section>
<hr class="docutils" />
<section id="lecture-applied-bioimage-analysis-2020">
<h2>Lecture Applied Bioimage Analysis 2020<a class="headerlink" href="#lecture-applied-bioimage-analysis-2020" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Slides, scripts, data and other exercise materials of the BioImage Analysis lecture at CMCB TU Dresden 2020</p>
<p>Tags: Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_applied_bioimage_analysis">https://git.mpi-cbg.de/rhaase/lecture_applied_bioimage_analysis</a></p>
</section>
<hr class="docutils" />
<section id="machine-learning-deep-learning-applications-to-bioimage-analysis">
<h2>Machine Learning - Deep Learning. Applications to Bioimage Analysis<a class="headerlink" href="#machine-learning-deep-learning-applications-to-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://raw.githubusercontent.com/esgomezm/esgomezm.github.io/master/assets/pdf/SPAOM2018/MachineLearning_SPAOMworkshop_public.pdf">https://raw.githubusercontent.com/esgomezm/esgomezm.github.io/master/assets/pdf/SPAOM2018/MachineLearning_SPAOMworkshop_public.pdf</a></p>
</section>
<hr class="docutils" />
<section id="machine-learning-workflow-for-imaging-flow-cytometry-ifc">
<h2>Machine Learning Workflow for Imaging Flow Cytometry (IFC)<a class="headerlink" href="#machine-learning-workflow-for-imaging-flow-cytometry-ifc" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: Cellprofiler, Bioimage Analysis</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/holgerhennig/machine-learning-IFC">holgerhennig/machine-learning-IFC</a></p>
</section>
<hr class="docutils" />
<section id="machine-and-deep-learning-on-the-cloud-segmentation">
<h2>Machine and Deep Learning on the cloud: Segmentation<a class="headerlink" href="#machine-and-deep-learning-on-the-cloud-segmentation" title="Link to this heading">#</a></h2>
<p>Ignacio Arganda-Carreras</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1oJoy9gHmUuSmUwCkPs_InJf_WZAzmLlUNvK1FUEB4PA/edit#slide=id.ge3a24e733b_0_54">https://docs.google.com/presentation/d/1oJoy9gHmUuSmUwCkPs_InJf_WZAzmLlUNvK1FUEB4PA/edit#slide=id.ge3a24e733b_0_54</a></p>
</section>
<hr class="docutils" />
<section id="making-the-most-of-bioimaging-data-through-interdisciplinary-interactions">
<h2>Making the most of bioimaging data through interdisciplinary interactions<a class="headerlink" href="#making-the-most-of-bioimaging-data-through-interdisciplinary-interactions" title="Link to this heading">#</a></h2>
<p>Virginie Uhlmann, Matthew Hartley, Josh Moore, Erin Weisbart, Assaf Zaritsky</p>
<p>Published 2024-10-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis, Open Science, Microscopy</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://journals.biologists.com/jcs/article/137/20/jcs262139/362478/Making-the-most-of-bioimaging-data-through">https://journals.biologists.com/jcs/article/137/20/jcs262139/362478/Making-the-most-of-bioimaging-data-through</a></p>
</section>
<hr class="docutils" />
<section id="meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration">
<h2>Meeting in the Middle: Towards Successful Multidisciplinary Bioimage Analysis Collaboration<a class="headerlink" href="#meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration" title="Link to this heading">#</a></h2>
<p>Anjalie Schlaeppi, Wilson Adams, Robert Haase, Jan Huisken, Ryan B. MacDonald, Kevin W. Eliceiri, Elisabeth C. Kugler</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.frontiersin.org/articles/10.3389/fbinf.2022.889755/full">https://www.frontiersin.org/articles/10.3389/fbinf.2022.889755/full</a></p>
</section>
<hr class="docutils" />
<section id="melbourne-advanced-microscopy-facility">
<h2>Melbourne Advanced Microscopy Facility<a class="headerlink" href="#melbourne-advanced-microscopy-facility" title="Link to this heading">#</a></h2>
<p>Collection of tutorial videos for Fiji users</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://www.youtube.com/&#64;melbourneadvancedmicroscop2617">https://www.youtube.com/&#64;melbourneadvancedmicroscop2617</a></p>
</section>
<hr class="docutils" />
<section id="methods-in-bioimage-analysis">
<h2>Methods in bioimage analysis<a class="headerlink" href="#methods-in-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Christian Tischer</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Online Tutorial, Video, Slides</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/events/methods-bioimage-analysis/">https://www.ebi.ac.uk/training/events/methods-bioimage-analysis/</a></p>
<p><a class="reference external" href="https://doi.org/10.6019/TOL.BioImageAnalysis22-w.2022.00001.1">https://doi.org/10.6019/TOL.BioImageAnalysis22-w.2022.00001.1</a></p>
<p><a class="reference external" href="https://drive.google.com/file/d/1MhuqfKhZcYu3bchWMqogIybKjamU5Msg/view">https://drive.google.com/file/d/1MhuqfKhZcYu3bchWMqogIybKjamU5Msg/view</a></p>
</section>
<hr class="docutils" />
<section id="metrics-reloaded-a-framework-for-trustworthy-image-analysis-validation">
<h2>Metrics Reloaded - A framework for trustworthy image analysis validation<a class="headerlink" href="#metrics-reloaded-a-framework-for-trustworthy-image-analysis-validation" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>The mission of Metrics Reloaded is to guide researchers in the selection of appropriate performance metrics for biomedical image analysis problems, as well as provide a comprehensive online resource for metric-related information and pitfalls</p>
<p>Tags: Bioimage Analysis, Quality Control</p>
<p>Content type: Website, Collection</p>
<p><a class="reference external" href="https://metrics-reloaded.dkfz.de/">https://metrics-reloaded.dkfz.de/</a></p>
</section>
<hr class="docutils" />
<section id="microsam-talks">
<h2>MicroSam-Talks<a class="headerlink" href="#microsam-talks" title="Link to this heading">#</a></h2>
<p>Constantin Pape</p>
<p>Published 2024-05-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Talks about Segment Anything for Microscopy: <a class="github reference external" href="https://github.com/computational-cell-analytics/micro-sam">computational-cell-analytics/micro-sam</a>.
Currently contains slides for two talks:</p>
<p>Overview of Segment Anythign for Microscopy given at the SWISSBIAS online meeting in April 2024
Talk about vision foundation models and Segment Anything for Microscopy given at Human Technopole as part of the EMBO Deep Learning Course in May 2024</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11265038">https://zenodo.org/records/11265038</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11265038">https://doi.org/10.5281/zenodo.11265038</a></p>
</section>
<hr class="docutils" />
<section id="microscopy-data-analysis-machine-learning-and-the-bioimage-archive">
<h2>Microscopy data analysis: machine learning and the BioImage Archive<a class="headerlink" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive" title="Link to this heading">#</a></h2>
<p>Andrii Iudin, Anna Foix-Romero, Anna Kreshuk, Awais Athar, Beth Cimini, Dominik Kutra, Estibalis Gomez de Mariscal, Frances Wong, Guillaume Jacquemet, Kedar Narayan, Martin Weigert, Nodar Gogoberidze, Osman Salih, Petr Walczysko, Ryan Conrad, Simone Weyend, Sriram Sundar Somasundharam, Suganya Sivagurunathan, Ugis Sarkans</p>
<p>Licensed CC-BY-4.0</p>
<p>The Microscopy data analysis: machine learning and the BioImage Archive course, which focused on introducing programmatic approaches used in the analysis of bioimage data via the BioImage Archive, ran in May 2023.</p>
<p>Tags: Bioimage Analysis, Python, Artificial Intelligence</p>
<p>Content type: Video, Slides</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/materials/microscopy-data-analysis-machine-learning-and-the-bioimage-archive-materials/">https://www.ebi.ac.uk/training/materials/microscopy-data-analysis-machine-learning-and-the-bioimage-archive-materials/</a></p>
</section>
<hr class="docutils" />
<section id="modular-training-resources-for-bioimage-analysis">
<h2>Modular training resources for bioimage analysis<a class="headerlink" href="#modular-training-resources-for-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Christian Tischer, Antonio Politi, Tim-Oliver Buchholz, Elnaz Fazeli, Nicola Gritti, Aliaksandr Halavatyi, Sebastian Gonzalez Tirado, Julian Hennies, Toby Hodges, Arif Khan, Dominik Kutra, Stefania Marcotti, Bugra Oezdemir, Felix Schneider, Martin Schorb, Anniek Stokkermans, Yi Sun, Nima Vakili</p>
<p>Published 2024-12-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Resources for teaching/preparing to teach bioimage analysis</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p><a class="reference external" href="https://zenodo.org/records/14264885">https://zenodo.org/records/14264885</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14264885">https://doi.org/10.5281/zenodo.14264885</a></p>
</section>
<hr class="docutils" />
<section id="morpholibj-documentation">
<h2>MorphoLibJ documentation<a class="headerlink" href="#morpholibj-documentation" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Document</p>
<p><a class="reference external" href="https://imagej.net/plugins/morpholibj">https://imagej.net/plugins/morpholibj</a></p>
</section>
<hr class="docutils" />
<section id="multi-view-fusion">
<h2>Multi-view fusion<a class="headerlink" href="#multi-view-fusion" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Lecture slides of a session on Multiview Fusion in Fiji</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_multiview_registration">https://git.mpi-cbg.de/rhaase/lecture_multiview_registration</a></p>
</section>
<hr class="docutils" />
<section id="multimodal-large-language-models-for-bioimage-analysis">
<h2>Multimodal large language models for bioimage analysis<a class="headerlink" href="#multimodal-large-language-models-for-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Shanghang Zhang, Gaole Dai, Tiejun Huang, Jianxu Chen</p>
<p>Licensed CC-BY-NC-SA</p>
<p>Multimodal large language models have been recognized as a historical milestone in the field of artificial intelligence and have demonstrated revolutionary potentials not only in commercial applications, but also for many scientific fields. Here we give a brief overview of multimodal large language models through the lens of bioimage analysis and discuss how we could build these models as a community to facilitate biology research</p>
<p>Tags: Bioimage Analysis, FAIR-Principles, Workflow</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.nature.com/articles/s41592-024-02334-2">https://www.nature.com/articles/s41592-024-02334-2</a></p>
<p><a class="reference external" href="https://arxiv.org/abs/2407.19778">https://arxiv.org/abs/2407.19778</a></p>
</section>
<hr class="docutils" />
<section id="multiplexed-tissue-imaging-tools-and-approaches">
<h2>Multiplexed tissue imaging - tools and approaches<a class="headerlink" href="#multiplexed-tissue-imaging-tools-and-approaches" title="Link to this heading">#</a></h2>
<p>Agustín Andrés Corbat, OmFrederic, Jonas Windhager, Kristína Lidayová</p>
<p>Licensed CC-BY-4.0</p>
<p>Material for the I2K 2024 “Multiplexed tissue imaging - tools and approaches” workshop</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Slides, Workshop</p>
<p><a class="github reference external" href="https://github.com/BIIFSweden/I2K2024-MTIWorkshop">BIIFSweden/I2K2024-MTIWorkshop</a></p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1">https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1</a></p>
</section>
<hr class="docutils" />
<section id="neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter">
<h2>NEUBIAS Academy &#64;HOME: Interactive Bioimage Analysis with Python and Jupyter<a class="headerlink" href="#neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter" title="Link to this heading">#</a></h2>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Neubias, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/neubias_academy_biapy">guiwitz/neubias_academy_biapy</a></p>
</section>
<hr class="docutils" />
<section id="neubias-analyst-school-2018">
<h2>NEUBIAS Analyst School 2018<a class="headerlink" href="#neubias-analyst-school-2018" title="Link to this heading">#</a></h2>
<p>Assaf Zaritsky, Csaba Molnar, Vasja Urbancic, Richard Butler, Anna Kreshuk, Vannary Meas-Yedid</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2018">miura/NEUBIAS_AnalystSchool2018</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-course-2017">
<h2>NEUBIAS Bioimage Analyst Course 2017<a class="headerlink" href="#neubias-bioimage-analyst-course-2017" title="Link to this heading">#</a></h2>
<p>Curtis Rueden, Florian Levet, J.B. Sibarta, Alexandre Dafour, Daniel Sage, Sebastien Tosi, Michal Kozubek, Jean-Yves Tinevez, Kota Miura, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_Bioimage_Analyst_Course2017">miura/NEUBIAS_Bioimage_Analyst_Course2017</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-school-2019">
<h2>NEUBIAS Bioimage Analyst School 2019<a class="headerlink" href="#neubias-bioimage-analyst-school-2019" title="Link to this heading">#</a></h2>
<p>Kota Miura, Chong Zhang, Jean-Yves Tinevez, Robert Haase, Julius Hossein, Pejamn Rasti, David Rousseau, Ignacio Arganda-Carreras, Siân Culley, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2019">miura/NEUBIAS_AnalystSchool2019</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-school-2020">
<h2>NEUBIAS Bioimage Analyst School 2020<a class="headerlink" href="#neubias-bioimage-analyst-school-2020" title="Link to this heading">#</a></h2>
<p>Marion Louveaux, Stéphane Verger, Arianne Bercowsky Rama, Ignacio Arganda-Carreras, Estibaliz Gómez-de-Mariscal, Kota Miura, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2020">miura/NEUBIAS_AnalystSchool2020</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage">
<h2>NFDI4BIOIMAGE<a class="headerlink" href="#nfdi4bioimage" title="Link to this heading">#</a></h2>
<p>Carsten Fortmann-Grote</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation was given at the 2nd MPG-NFDI Workshop on April 18th about e NFDI4BIOIMAGE Consortium, FAIRification of Image (meta)data, Zarr, RFC, Training (TA5), contributing.</p>
<p>Tags: Research Data Management, Bioimage Analysis, FAIR-Principles, Zarr, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.11031746">https://zenodo.org/doi/10.5281/zenodo.11031746</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters</p>
<p>Licensed CC-BY-4.0</p>
<p>NFDI4BIOIMAGE core mission, bioimage data challenge, task areas, FAIR bioimage workflows.</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8070038">https://doi.org/10.5281/zenodo.8070038</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg" title="Link to this heading">#</a></h2>
<p>Riccardo Massei</p>
<p>Licensed CC-BY-4.0</p>
<p>NFDI4BIOIMAGE is a consortium within the framework of the National Research Data Infrastructure (NFDI) in Germany. In this talk, the consortium and the contribution to the work programme by the Helmholtz Centre for Environmental Research (UFZ) in Leipzig are outlined.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8414318">https://zenodo.org/doi/10.5281/zenodo.8414318</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">
<h2>NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)<a class="headerlink" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository" title="Link to this heading">#</a></h2>
<p>Mohamed Abdrabbou, Mehrnaz Babaki, Tom Boissonnet, Michele Bortolomeazzi, Eik Dahms, Vanessa Fuchs, A. F. Moritz Hoevels, Niraj Kandpal, Christoph Möhl, Joshua A. Moore, Astrid Schauss, Andrea Schrader, Torsten Stöter, Julia Thönnißen, Monica Valencia-S., H. Lukas Weil, Jens Wendt, Peter Zentis</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository documents the first NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon), where topics like ‘Interoperability’, ‘REMBI / Mapping’, and ‘Neuroglancer (OMERO / zarr)’ were explored through collaborative discussions and workflow sessions, culminating in reports that bridge NFDI4Bioimage to DataPLANT. Funded by various DFG initiatives, this event emphasized documentation and use cases, contributing preparatory work for future interoperability projects at the 2nd de.NBI BioHackathon in Bielefeld.</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Github Repository</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10609770">https://zenodo.org/doi/10.5281/zenodo.10609770</a></p>
</section>
<hr class="docutils" />
<section id="neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">
<h2>Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist<a class="headerlink" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Olivier Burri, Siân Culley, Uwe Schmidt</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Neubias, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/neubias_academy_stardist">maweigert/neubias_academy_stardist</a></p>
</section>
<hr class="docutils" />
<section id="neubiaspasteur2023-advancedcellpose">
<h2>NeubiasPasteur2023_AdvancedCellPose<a class="headerlink" href="#neubiaspasteur2023-advancedcellpose" title="Link to this heading">#</a></h2>
<p>Gaelle Letort</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tutorial for running CellPose advanced functions</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/gletort/NeubiasPasteur2023_AdvancedCellPose">gletort/NeubiasPasteur2023_AdvancedCellPose</a></p>
</section>
<hr class="docutils" />
<section id="omero-hcs-analysis-pipeline-using-jupyter-notebooks">
<h2>OMERO - HCS analysis pipeline using Jupyter Notebooks<a class="headerlink" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks" title="Link to this heading">#</a></h2>
<p>Riccardo Massei</p>
<p>Licensed MIT</p>
<p>Material and solutions for the course ‘Bioimage data management and analysis with OMERO’ held in Heidelberg (13th May 2024) - Module 3 (1.45 pm - 3.45 pm): OMERO and Jupyter Notebooks. Main goal of the workflow is to show the potential of JN to perform reproducible image analysis in connection with an OMERO instance. In this specific example, we are performing a simple nuclei segmentation from raw images uploaded in OMERO.</p>
<p>Tags: Teaching, Bioimage Analysis, Notebooks, Python, OMERO</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/rmassei/2024-jn-omero-pipeline">rmassei/2024-jn-omero-pipeline</a></p>
</section>
<hr class="docutils" />
<section id="omero-qupath">
<h2>OMERO - QuPath<a class="headerlink" href="#omero-qupath" title="Link to this heading">#</a></h2>
<p>Rémy Jean Daniel Dornier</p>
<p>Licensed CC-BY-NC-SA-4.0</p>
<p>OMERO-RAW extension for QuPath allows to directly access to the raw pixels of images. All types of images (RGB, fluorescence, …) are supported with this extension.</p>
<p>Tags: Bioimage Analysis, OMERO</p>
<p>Content type: Online Tutorial</p>
<p><a class="reference external" href="https://wiki-biop.epfl.ch/en/data-management/omero/qupath">https://wiki-biop.epfl.ch/en/data-management/omero/qupath</a></p>
</section>
<hr class="docutils" />
<section id="onbi-image-analysis-course">
<h2>ONBI Image Analysis Course<a class="headerlink" href="#onbi-image-analysis-course" title="Link to this heading">#</a></h2>
<p>Mark Jenkinson, Jens Rittscher, Dominic Waithe</p>
<p>Licensed GPL-2.0</p>
<p>This repository contains the materials for the University of Oxford DTC ONBI Image Analysis course.</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/dwaithe/ONBI_image_analysis">dwaithe/ONBI_image_analysis</a></p>
</section>
<hr class="docutils" />
<section id="object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">
<h2>Object Tracking and Track Analysis using TrackMate and CellTracksColab<a class="headerlink" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab" title="Link to this heading">#</a></h2>
<p>Joanna Pylvänäinen</p>
<p>Published None</p>
<p>Licensed GPL-3.0</p>
<p>I2K 2024 workshop materials for “Object Tracking and Track Analysis using TrackMate and CellTracksColab”</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Tutorial, Workshop, Slides</p>
<p><a class="github reference external" href="https://github.com/CellMigrationLab/I2K_2024">CellMigrationLab/I2K_2024</a></p>
</section>
<hr class="docutils" />
<section id="omero-tools">
<h2>Omero-tools<a class="headerlink" href="#omero-tools" title="Link to this heading">#</a></h2>
<p>Johannes Soltwedel</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository contains a collection of tools for working with OMERO. Such tools can be working with the OMERO command line interface to transfer datasets between repositories, etc.</p>
<p>Tags: OMERO, Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="reference external" href="https://biapol.github.io/omero-tools/">https://biapol.github.io/omero-tools/</a></p>
</section>
<hr class="docutils" />
<section id="open-image-data-handbook">
<h2>Open Image Data Handbook<a class="headerlink" href="#open-image-data-handbook" title="Link to this heading">#</a></h2>
<p>Kevin Yamauchi</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Neubias, Research Data Management, Napari, Python, Bioimage Analysis</p>
<p>Content type: Book, Notebook</p>
<p><a class="reference external" href="https://kevinyamauchi.github.io/open-image-data/intro.html">https://kevinyamauchi.github.io/open-image-data/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="pol-bio-image-analysis-early-career-bio-image-analysis-training-school">
<h2>PoL Bio-Image Analysis Early Career Bio-image Analysis Training School<a class="headerlink" href="#pol-bio-image-analysis-early-career-bio-image-analysis-training-school" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis, Python</p>
<p>Content type: Event, Notebook</p>
<p><a class="reference external" href="https://biapol.github.io/PoL-BioImage-Analysis-TS-Early-Career-Track/intro.html">https://biapol.github.io/PoL-BioImage-Analysis-TS-Early-Career-Track/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="pol-bio-image-analysis-gpu-accelerated-image-analysis-training-school">
<h2>PoL Bio-Image Analysis GPU Accelerated Image Analysis Training School<a class="headerlink" href="#pol-bio-image-analysis-gpu-accelerated-image-analysis-training-school" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis, Python</p>
<p>Content type: Event, Notebook</p>
<p><a class="reference external" href="https://biapol.github.io/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/intro.html">https://biapol.github.io/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="prompt-engineering-in-bio-image-analysis">
<h2>Prompt Engineering in Bio-image Analysis<a class="headerlink" href="#prompt-engineering-in-bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Jupyter, Bioimage Analysis, Prompt Engineering, Biabob</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2024/07/18/prompt-engineering-in-bio-image-analysis/">https://focalplane.biologists.com/2024/07/18/prompt-engineering-in-bio-image-analysis/</a></p>
</section>
<hr class="docutils" />
<section id="python-bioimage-analysis-tutorial">
<h2>Python BioImage Analysis Tutorial<a class="headerlink" href="#python-bioimage-analysis-tutorial" title="Link to this heading">#</a></h2>
<p>Jonas Hartmann</p>
<p>Licensed MIT</p>
<p>Tags: Python, Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/WhoIsJack/python-bioimage-analysis-tutorial">WhoIsJack/python-bioimage-analysis-tutorial</a></p>
</section>
<hr class="docutils" />
<section id="python-for-microscopists">
<h2>Python for Microscopists<a class="headerlink" href="#python-for-microscopists" title="Link to this heading">#</a></h2>
<p>Sreenivas Bhattiprolu</p>
<p>Licensed MIT</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/bnsreenu/python_for_microscopists">bnsreenu/python_for_microscopists</a></p>
</section>
<hr class="docutils" />
<section id="qupath-for-python-programmers">
<h2>QuPath for Python programmers<a class="headerlink" href="#qupath-for-python-programmers" title="Link to this heading">#</a></h2>
<p>Alan O’Callaghan, Léo Leplat</p>
<p>Published 2024-10-23</p>
<p>Licensed UNKNOWN</p>
<p>These are the notebooks and associated files for the i2k 2024 QuPath for Python programmers workshop.</p>
<p>Tags: Python, Notebooks, Open Source Software, Bioimage Analysis</p>
<p>Content type: Github Repository, Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/qupath/i2k-qupath-for-python-programmers">qupath/i2k-qupath-for-python-programmers</a></p>
</section>
<hr class="docutils" />
<section id="qupath-open-source-software-for-analysing-awkward-images">
<h2>QuPath: Open source software for analysing (awkward) images<a class="headerlink" href="#qupath-open-source-software-for-analysing-awkward-images" title="Link to this heading">#</a></h2>
<p>Peter Bankhead</p>
<p>Published 2020-12-16</p>
<p>Licensed CC-BY-4.0</p>
<p>Slides from the CZI/EOSS online meeting in December 2020.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4328911">https://zenodo.org/records/4328911</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4328911">https://doi.org/10.5281/zenodo.4328911</a></p>
</section>
<hr class="docutils" />
<section id="quality-assurance-of-segmentation-results">
<h2>Quality assurance of segmentation results<a class="headerlink" href="#quality-assurance-of-segmentation-results" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/04/13/quality-assurance-of-segmentation-results/">https://focalplane.biologists.com/2023/04/13/quality-assurance-of-segmentation-results/</a></p>
</section>
<hr class="docutils" />
<section id="rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again">
<h2>RDF as a bridge to domain-platforms like OMERO, or There and back again.<a class="headerlink" href="#rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again" title="Link to this heading">#</a></h2>
<p>Josh Moore, Andra Waagmeester, Kristina Hettne, Katherine Wolstencroft, Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>In 2005, the first version of OMERO stored RDF natively. However, just a year after the 1.0 release of RDF, performance considerations led to the development of a more traditional SQL approach for OMERO. A binary protocol makes it possible to query and retrieve metadata but the resulting information cannot immediately be combined with other sources. This is the adventure of rediscovering the benefit of RDF triples as a – if not the – common exchange mechanism.</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10687658">https://zenodo.org/doi/10.5281/zenodo.10687658</a></p>
</section>
<hr class="docutils" />
<section id="rescaling-images-and-pixel-an-isotropy">
<h2>Rescaling images and pixel (an)isotropy<a class="headerlink" href="#rescaling-images-and-pixel-an-isotropy" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/03/02/rescaling-images-and-pixel-anisotropy/">https://focalplane.biologists.com/2023/03/02/rescaling-images-and-pixel-anisotropy/</a></p>
</section>
<hr class="docutils" />
<section id="running-deep-learning-scripts-in-the-bia-pol-omero-server">
<h2>Running Deep-Learning Scripts in the BiA-PoL Omero Server<a class="headerlink" href="#running-deep-learning-scripts-in-the-bia-pol-omero-server" title="Link to this heading">#</a></h2>
<p>Marcelo Zoccoler</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://biapol.github.io/blog/marcelo_zoccoler/omero_scripts/readme.html">https://biapol.github.io/blog/marcelo_zoccoler/omero_scripts/readme.html</a></p>
</section>
<hr class="docutils" />
<section id="scipy-cookbook">
<h2>Scipy Cookbook<a class="headerlink" href="#scipy-cookbook" title="Link to this heading">#</a></h2>
<p>Pauli Virtanen et al.</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This is a conversion and second life of SciPy Cookbook as a bunch of Ipython notebooks.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scipy/scipy-cookbook">scipy/scipy-cookbook</a></p>
</section>
<hr class="docutils" />
<section id="scripts-filopodyanr-a-case-study-for-the-neubias-ts7-in-szeged">
<h2>Scripts_FilopodyanR - a case study for the NEUBIAS TS7 in Szeged<a class="headerlink" href="#scripts-filopodyanr-a-case-study-for-the-neubias-ts7-in-szeged" title="Link to this heading">#</a></h2>
<p>Marion Louveaux</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Code</p>
<p><a class="github reference external" href="https://github.com/marionlouveaux/NEUBIAS2018_TS7/">marionlouveaux/NEUBIAS2018_TS7</a></p>
</section>
<hr class="docutils" />
<section id="simpleitk-notebooks">
<h2>SimpleITK-Notebooks<a class="headerlink" href="#simpleitk-notebooks" title="Link to this heading">#</a></h2>
<p>Ziv Yaniv et al.</p>
<p>Licensed APACHE-2.0</p>
<p>Jupyter notebooks for learning how to use SimpleITK</p>
<p>Tags: Bioimage Analysis, Simpleitk</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/InsightSoftwareConsortium/SimpleITK-Notebooks">InsightSoftwareConsortium/SimpleITK-Notebooks</a></p>
</section>
<hr class="docutils" />
<section id="teaching-bioimage-analysis-with-python">
<h2>Teaching Bioimage Analysis with Python<a class="headerlink" href="#teaching-bioimage-analysis-with-python" title="Link to this heading">#</a></h2>
<p>Rafael Camacho</p>
<p>Licensed MIT</p>
<p>Tags: Python, Bioimage Analysis</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/CamachoDejay/teaching-bioimage-analysis-python">CamachoDejay/teaching-bioimage-analysis-python</a></p>
</section>
<hr class="docutils" />
<section id="teaching-imagej-fiji">
<h2>Teaching ImageJ FIJI<a class="headerlink" href="#teaching-imagej-fiji" title="Link to this heading">#</a></h2>
<p>Rafael Camacho</p>
<p>Licensed MIT</p>
<p>Tags: Fiji, Bioimage Analysis</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/CamachoDejay/Teaching-ImageJ-FIJI">CamachoDejay/Teaching-ImageJ-FIJI</a></p>
</section>
<hr class="docutils" />
<section id="the-digital-cell-cell-biology-as-a-data-science">
<h2>The Digital Cell: Cell Biology as a Data Science<a class="headerlink" href="#the-digital-cell-cell-biology-as-a-data-science" title="Link to this heading">#</a></h2>
<p>Stephen Royle</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://cshlpress.com/default.tpl?cart=1700309488232283050&amp;amp;fromlink=T&amp;amp;linkaction=full&amp;amp;linksortby=oop_title&amp;amp;--eqSKUdatarq=1282">https://cshlpress.com/default.tpl?cart=1700309488232283050&amp;fromlink=T&amp;linkaction=full&amp;linksortby=oop_title&amp;–eqSKUdatarq=1282</a></p>
</section>
<hr class="docutils" />
<section id="the-open-microscopy-environment-ome-data-model-and-xml-file-open-tools-for-informatics-and-quantitative-analysis-in-biological-imaging">
<h2>The Open Microscopy Environment (OME) Data Model and XML file - open tools for informatics and quantitative analysis in biological imaging<a class="headerlink" href="#the-open-microscopy-environment-ome-data-model-and-xml-file-open-tools-for-informatics-and-quantitative-analysis-in-biological-imaging" title="Link to this heading">#</a></h2>
<p>Ilya G. Goldberg, Chris Allan, jean-marie burel, Doug Creager, Andrea Falconi, et. al</p>
<p>Published 2005-05-03</p>
<p>Licensed CC-BY-4.0</p>
<p>The Open Microscopy Environment (OME) defines a data model and a software implementation to serve as an informatics framework for imaging in biological microscopy experiments, including representation of acquisition parameters, annotations and image analysis results.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-5-r47">https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-5-r47</a></p>
<p><a class="reference external" href="https://doi.org/10.1186/gb-2005-6-5-r47">https://doi.org/10.1186/gb-2005-6-5-r47</a></p>
</section>
<hr class="docutils" />
<section id="towards-community-driven-metadata-standards-for-light-microscopy-tiered-specifications-extending-the-ome-model">
<h2>Towards community-driven metadata standards for light microscopy - tiered specifications extending the OME model<a class="headerlink" href="#towards-community-driven-metadata-standards-for-light-microscopy-tiered-specifications-extending-the-ome-model" title="Link to this heading">#</a></h2>
<p>Mathias Hammer, Maximiliaan Huisman, Alessandro Rigano, Ulrike Boehm, James J. Chambers, et al.</p>
<p>Published 2022-07-10</p>
<p>Licensed UNKNOWN</p>
<p>Rigorous record-keeping and quality control are required to ensure the quality, reproducibility and value of imaging data. The 4DN Initiative and BINA here propose light Microscopy Metadata specifications that extend the OME data model, scale with experimental intent and complexity, and make it possible for scientists to create comprehensive records of imaging experiments.</p>
<p>Tags: Reproducibility, Bioimage Analysis, Metadata</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9271325/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9271325/</a></p>
</section>
<hr class="docutils" />
<section id="tracking-theory-trackmate-and-mastodon">
<h2>Tracking Theory, TrackMate, and Mastodon<a class="headerlink" href="#tracking-theory-trackmate-and-mastodon" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Lecture slides of a session on Cell Tracking in Fiji</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_tracking_trackmate">https://git.mpi-cbg.de/rhaase/lecture_tracking_trackmate</a></p>
</section>
<hr class="docutils" />
<section id="tracking-in-napari">
<h2>Tracking in napari<a class="headerlink" href="#tracking-in-napari" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/06/01/tracking-in-napari/">https://focalplane.biologists.com/2023/06/01/tracking-in-napari/</a></p>
</section>
<hr class="docutils" />
<section id="tracking-of-mitochondria-and-capturing-mitoflashes">
<h2>Tracking of mitochondria and capturing mitoflashes<a class="headerlink" href="#tracking-of-mitochondria-and-capturing-mitoflashes" title="Link to this heading">#</a></h2>
<p>Leonid Kostrykin, Diana Chiang Jurado</p>
<p>Published 2024-11-20</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioinformatics, Bioimage Analysis</p>
<p>Content type: Workflow, Tutorial</p>
<p><a class="reference external" href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/detection-of-mitoflashes/tutorial.html#tracking-of-mitochondria-and-capturing-mitoflashes">https://training.galaxyproject.org/training-material/topics/imaging/tutorials/detection-of-mitoflashes/tutorial.html#tracking-of-mitochondria-and-capturing-mitoflashes</a></p>
</section>
<hr class="docutils" />
<section id="training-deep-learning-models-for-vision-compact-course">
<h2>Training Deep Learning Models for Vision - Compact Course<a class="headerlink" href="#training-deep-learning-models-for-vision-compact-course" title="Link to this heading">#</a></h2>
<p>Constantin Pape, Adrian Wolny</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p><a class="github reference external" href="https://github.com/constantinpape/training-deep-learning-models-for-vison">constantinpape/training-deep-learning-models-for-vison</a></p>
</section>
<hr class="docutils" />
<section id="ultrack-i2k-2024-workshop-materials">
<h2>Ultrack I2K 2024 Workshop Materials<a class="headerlink" href="#ultrack-i2k-2024-workshop-materials" title="Link to this heading">#</a></h2>
<p>Jordão Bragantini, Teun Huijben</p>
<p>Licensed BSD3-CLAUSE</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/royerlab/ultrack-i2k2024">royerlab/ultrack-i2k2024</a></p>
<p><a class="reference external" href="https://royerlab.github.io/ultrack-i2k2024/">https://royerlab.github.io/ultrack-i2k2024/</a></p>
</section>
<hr class="docutils" />
<section id="understanding-metric-related-pitfalls-in-image-analysis-validation">
<h2>Understanding metric-related pitfalls in image analysis validation<a class="headerlink" href="#understanding-metric-related-pitfalls-in-image-analysis-validation" title="Link to this heading">#</a></h2>
<p>Annika Reinke, et al.</p>
<p>Published 2023</p>
<p>This article gives a detailed overview about pitfalls when using metric for image analysis algorithm validation.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Publication, Preprint</p>
<p><a class="reference external" href="https://arxiv.org/abs/2302.01790v3">https://arxiv.org/abs/2302.01790v3</a></p>
</section>
<hr class="docutils" />
<section id="upcoming-image-analysis-events">
<h2>Upcoming Image Analysis Events<a class="headerlink" href="#upcoming-image-analysis-events" title="Link to this heading">#</a></h2>
<p>Curtis Rueden, Albane de la Villegeorges, Simon F. Nørrelykke, Romain Guiet, Olivier Burri, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Event, Forum Post, Workshop</p>
<p><a class="reference external" href="https://forum.image.sc/t/upcoming-image-analysis-events/60018/67">https://forum.image.sc/t/upcoming-image-analysis-events/60018/67</a></p>
</section>
<hr class="docutils" />
<section id="using-glittr-org-to-find-compare-and-re-use-online-training-materials">
<h2>Using <a class="reference external" href="http://Glittr.org">Glittr.org</a> to find, compare and re-use online training materials<a class="headerlink" href="#using-glittr-org-to-find-compare-and-re-use-online-training-materials" title="Link to this heading">#</a></h2>
<p>Geert van Geest, Yann Haefliger, Monique Zahn-Zabal, Patricia M. Palagi</p>
<p>Licensed CC-BY-4.0</p>
<p><a class="reference external" href="http://Glittr.org">Glittr.org</a> is a platform that aggregates and indexes training materials on computational life sciences from public git repositories, making it easier for users to find, compare, and analyze these resources based on various metrics. By providing insights into the availability of materials, collaboration patterns, and licensing practices, <a class="reference external" href="http://Glittr.org">Glittr.org</a> supports adherence to the FAIR principles, benefiting the broader life sciences community.</p>
<p>Tags: Bioimage Analysis, Research Data Management</p>
<p>Content type: Publication, Preprint</p>
<p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2024.08.20.608021v1">https://www.biorxiv.org/content/10.1101/2024.08.20.608021v1</a></p>
</section>
<hr class="docutils" />
<section id="v4sdb-winter-school-2025">
<h2>V4SDB_Winter_School_2025<a class="headerlink" href="#v4sdb-winter-school-2025" title="Link to this heading">#</a></h2>
<p>Joanna Pylvänäinen</p>
<p>Published 2025-01-13T08:29:22+00:00</p>
<p>Training materials for V4SDB Student Winter School, 28th-31st January 2025 at ELTE Eötvös Loránd University in Budapest, Hungary</p>
<p>Tags: Cell Tracking, Bioimage Analysis</p>
<p>Content type: Github Repository, Collection</p>
<p><a class="github reference external" href="https://github.com/CellMigrationLab/V4SDB_Winter_School_2025">CellMigrationLab/V4SDB_Winter_School_2025</a></p>
</section>
<hr class="docutils" />
<section id="welcome-to-bioimage-town">
<h2>Welcome to BioImage Town<a class="headerlink" href="#welcome-to-bioimage-town" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Licensed CC-BY-4.0</p>
<p>Welcome at NFDI4BIOIMAGE All-Hands Meeting in Düsseldorf, Germany, October 16, 2023</p>
<p>Tags: OMERO, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10008464">https://zenodo.org/doi/10.5281/zenodo.10008464</a></p>
</section>
<hr class="docutils" />
<section id="what-is-bioimage-analysis-an-introduction">
<h2>What is Bioimage Analysis? An Introduction<a class="headerlink" href="#what-is-bioimage-analysis-an-introduction" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://www.dropbox.com/s/5abw3cvxrhpobg4/20220923_DefragmentationTS.pdf?dl=0">https://www.dropbox.com/s/5abw3cvxrhpobg4/20220923_DefragmentationTS.pdf?dl=0</a></p>
</section>
<hr class="docutils" />
<section id="who-you-gonna-call-data-stewards-to-the-rescue">
<h2>Who you gonna call? - Data Stewards to the rescue<a class="headerlink" href="#who-you-gonna-call-data-stewards-to-the-rescue" title="Link to this heading">#</a></h2>
<p>Vanessa Aphaia Fiona Fuchs, Jens Wendt, Maximilian Müller, Mohsen Ahmadi, Riccardo Massei, Cornelia Wetzker</p>
<p>Licensed CC-BY-4.0</p>
<p>The Data Steward Team of the NFDI4BIOIMAGE consortium presents themselves and the services (including the Helpdesk) that we offer.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Data Stewardship, Nfdi4Bioimage</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10730423">https://zenodo.org/doi/10.5281/zenodo.10730423</a></p>
</section>
<hr class="docutils" />
<section id="working-with-objects-in-2d-and-3d">
<h2>Working with objects in 2D and 3D<a class="headerlink" href="#working-with-objects-in-2d-and-3d" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_working_with_objects_in_2d_and_3d">https://git.mpi-cbg.de/rhaase/lecture_working_with_objects_in_2d_and_3d</a></p>
</section>
<hr class="docutils" />
<section id="working-with-pixels">
<h2>Working with pixels<a class="headerlink" href="#working-with-pixels" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_working_with_pixels">https://git.mpi-cbg.de/rhaase/lecture_working_with_pixels</a></p>
</section>
<hr class="docutils" />
<section id="workshop-june2024-madrid">
<h2>Workshop-June2024-Madrid<a class="headerlink" href="#workshop-june2024-madrid" title="Link to this heading">#</a></h2>
<p>Licensed MIT</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/bioimage-io/Workshop-June2024-Madrid">bioimage-io/Workshop-June2024-Madrid</a></p>
</section>
<hr class="docutils" />
<section id="ymia-python-based-event-series-training-material">
<h2>YMIA - Python-Based Event Series Training Material<a class="headerlink" href="#ymia-python-based-event-series-training-material" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Robert Haase, ENicolay</p>
<p>Published None</p>
<p>Licensed MIT</p>
<p>This repository offer access to teaching material and useful resources for the YMIA - Python-Based Event Series.</p>
<p>Tags: Python, Artifical Intelligence, Bioimage Analysis</p>
<p>Content type: Github Repository, Slides</p>
<p><a class="github reference external" href="https://github.com/rmassei/ymia_python_event_series_material">rmassei/ymia_python_event_series_material</a></p>
</section>
<hr class="docutils" />
<section id="zen-python-workshop">
<h2>ZEN &amp; Python workshop<a class="headerlink" href="#zen-python-workshop" title="Link to this heading">#</a></h2>
<p>Licensed GPL-3.0</p>
<p>Tags: Python, Napari, Bioimage Analysis</p>
<p>Content type: Collection, Notebook</p>
<p><a class="github reference external" href="https://github.com/zeissmicroscopy/ZEN_Python_OAD_workshop">zeissmicroscopy/ZEN_Python_OAD_workshop</a></p>
</section>
<hr class="docutils" />
<section id="zidas-2020-introduction-to-deep-learning">
<h2>ZIDAS 2020 Introduction to Deep Learning<a class="headerlink" href="#zidas-2020-introduction-to-deep-learning" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/esgomezm/zidas2020_intro_DL">esgomezm/zidas2020_intro_DL</a></p>
</section>
<hr class="docutils" />
<section id="zerocostdl4mic-exploiting-google-colab-to-develop-a-free-and-open-source-toolbox-for-deep-learning-in-microscopy">
<h2>ZeroCostDL4Mic: exploiting Google Colab to develop a free and open-source toolbox for Deep-Learning in microscopy<a class="headerlink" href="#zerocostdl4mic-exploiting-google-colab-to-develop-a-free-and-open-source-toolbox-for-deep-learning-in-microscopy" title="Link to this heading">#</a></h2>
<p>Lucas von Chamier, Romain F. Laine, Johanna Jukkala, Christoph Spahn, Daniel Krentzel, Elias Nehme, Martina Lerche, Sara Hernández-pérez, Pieta Mattila, Eleni Karinou, Séamus Holden, Ahmet Can Solak, Alexander Krull, Tim-Oliver Buchholz, Martin L Jones, Loic Alain Royer, Christophe Leterrier, Yoav Shechtman, Florian Jug, Mike Heilemann, Guillaume Jacquemet, Ricardo Henriques</p>
<p>Licensed MIT</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/HenriquesLab/ZeroCostDL4Mic">HenriquesLab/ZeroCostDL4Mic</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/s41467-021-22518-0">https://www.nature.com/articles/s41467-021-22518-0</a></p>
<p><a class="reference external" href="https://doi.org/10.1038/s41467-021-22518-0">https://doi.org/10.1038/s41467-021-22518-0</a></p>
</section>
<hr class="docutils" />
<section id="cordi-2023-zarr-a-cloud-optimized-storage-for-interactive-access-of-large-arrays">
<h2>[CORDI 2023] Zarr: A Cloud-Optimized Storage for Interactive Access of Large Arrays<a class="headerlink" href="#cordi-2023-zarr-a-cloud-optimized-storage-for-interactive-access-of-large-arrays" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Licensed CC-BY-4.0</p>
<p>For decades, the sharing of large N-dimensional datasets has posed issues across multiple domains. Interactively accessing terabyte-scale data has previously required significant server resources to properly prepare cropped or down-sampled representations on the fly. Now, a cloud-native chunked format easing this burden has been adopted in the bioimaging domain for standardization. The format — Zarr — is potentially of interest for other consortia and sections of NFDI.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Data Science</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8340247">https://zenodo.org/doi/10.5281/zenodo.8340247</a></p>
</section>
<hr class="docutils" />
<section id="elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">
<h2>[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math<a class="headerlink" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math" title="Link to this heading">#</a></h2>
<p>Josh Moore, Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presented at the European Light Microscopy Initiative meeting in Liverpool (<a class="reference external" href="https://www.elmi2024.org/">https://www.elmi2024.org/</a>)</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.11235512">https://zenodo.org/doi/10.5281/zenodo.11235512</a></p>
</section>
<hr class="docutils" />
<section id="short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">
<h2>[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure<a class="headerlink" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure" title="Link to this heading">#</a></h2>
<p>Christian Schmidt</p>
<p>Licensed CC-BY-4.0</p>
<p>Short Talk about the NFDI4BIOIMAGE consortium presented at the RDM in (Bio-)Medicine Information Event on April 10th, 2024, organized C³RDM &amp; ZB MED.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10939519">https://zenodo.org/doi/10.5281/zenodo.10939519</a></p>
</section>
<hr class="docutils" />
<section id="arivis-vision4d-tutorials">
<h2>arivis Vision4D Tutorials<a class="headerlink" href="#arivis-vision4d-tutorials" title="Link to this heading">#</a></h2>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Collection, Video</p>
<p><a class="reference external" href="https://www.youtube.com/playlist?list=PLRc9dt9lEZh_SVRC4G5moOvgHuvjPwmv0">https://www.youtube.com/playlist?list=PLRc9dt9lEZh_SVRC4G5moOvgHuvjPwmv0</a></p>
</section>
<hr class="docutils" />
<section id="ilastik-interactive-machine-learning-for-bio-image-analysis">
<h2>ilastik: interactive machine learning for (bio)image analysis<a class="headerlink" href="#ilastik-interactive-machine-learning-for-bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Anna Kreshuk, Dominik Kutra</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.4330625">https://zenodo.org/doi/10.5281/zenodo.4330625</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-image-analysis">
<h2>introduction-to-image-analysis<a class="headerlink" href="#introduction-to-image-analysis" title="Link to this heading">#</a></h2>
<p>Dave Barry, Stefania Marcotti, Martin Jones</p>
<p>Published 2024-10-23T14:05:55+00:00</p>
<p>Licensed CC-BY-SA-4.0</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Github Repository, Notebooks</p>
<p><a class="github reference external" href="https://github.com/RMS-DAIM/introduction-to-image-analysis">RMS-DAIM/introduction-to-image-analysis</a></p>
</section>
<hr class="docutils" />
<section id="patho-prompt-injection">
<h2>patho_prompt_injection<a class="headerlink" href="#patho-prompt-injection" title="Link to this heading">#</a></h2>
<p>JanClusmann, Tim Lenz</p>
<p>Published 2024-11-08T08:32:03+00:00</p>
<p>Licensed GPL-3.0</p>
<p>Tags: Histopathology, Bioimage Analysis</p>
<p>Content type: Github Repository, Notebook</p>
<p><a class="github reference external" href="https://github.com/KatherLab/patho_prompt_injection">KatherLab/patho_prompt_injection</a></p>
</section>
<hr class="docutils" />
<section id="quantixed-thedigitalcell-first-complete-code-set">
<h2>quantixed/TheDigitalCell: First complete code set<a class="headerlink" href="#quantixed-thedigitalcell-first-complete-code-set" title="Link to this heading">#</a></h2>
<p>Stephen Royle</p>
<p>Published 2019-04-17</p>
<p>Licensed GPL-3.0</p>
<p>First complete code set for The Digital Cell book.</p>
<p>Tags: Bioimage Analysis</p>
<p>Content type: Code</p>
<p><a class="github reference external" href="https://github.com/quantixed/TheDigitalCell">quantixed/TheDigitalCell</a></p>
<p><a class="reference external" href="https://zenodo.org/records/2643411">https://zenodo.org/records/2643411</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.2643411">https://doi.org/10.5281/zenodo.2643411</a></p>
</section>
<hr class="docutils" />
<section id="scanpy-tutorials">
<h2>scanpy-tutorials<a class="headerlink" href="#scanpy-tutorials" title="Link to this heading">#</a></h2>
<p>Alex Wolf, pre-commit-ci[bot], Philipp A., Isaac Virshup, Ilan Gold, Giovanni Palla, Fidel Ramirez, G\xF6k\xE7en Eraslan, Sergei Rybakov, Abolfazl (Abe), Adam Gayoso, Dinesh Palli, Gregor Sturm, Jan Lause, Karin Hrovatin, Krzysztof Polanski, RaphaelBuzzi, Yimin Zheng, Yishen Miao, evanbiederstedt</p>
<p>Published 2018-12-16T03:42:46+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Scanpy Tutorials.</p>
<p>Tags: Single-Cell Analysis, Bioimage Analysis</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scverse/scanpy-tutorials">scverse/scanpy-tutorials</a></p>
</section>
<hr class="docutils" />
<section id="scikit-learn-mooc">
<h2>scikit-learn MOOC<a class="headerlink" href="#scikit-learn-mooc" title="Link to this heading">#</a></h2>
<p>Loïc Estève et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>Machine learning in Python with scikit-learn MOOC</p>
<p>Tags: Bioimage Analysis, Machine Learning</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/INRIA/scikit-learn-mooc">INRIA/scikit-learn-mooc</a></p>
</section>
<hr class="docutils" />
<section id="skimage-tutorials">
<h2>skimage-tutorials<a class="headerlink" href="#skimage-tutorials" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias et al.</p>
<p>Licensed CC0 1.0 UNIVERSAL</p>
<p>skimage-tutorials - a collection of tutorials for the scikit-image package.</p>
<p>Tags: Bioimage Analysis, Scikit-Image</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scikit-image/skimage-tutorials">scikit-image/skimage-tutorials</a></p>
<hr class="docutils" />
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./tags"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="artificial_intelligence.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Artificial intelligence (44)</p>
      </div>
    </a>
    <a class="right-next"
       href="bioinformatics.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Bioinformatics (10)</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <dialog id="pst-secondary-sidebar-modal"></dialog>
                <div id="pst-secondary-sidebar" class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-survey-community-experiences-and-needs-for-the-future">2020 BioImage Analysis Survey: Community experiences and needs for the future</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#mic-workshop-on-bioimage-processing-with-python">2022 MIC Workshop on Bioimage processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-fiji-scripting-tutorial">A Fiji Scripting Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe">A Hitchhiker’s guide through the bio-image analysis software universe</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ai-ml-dl-in-bioimage-analysis-webinar">AI ML DL in Bioimage Analysis - Webinar</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ai4life-teams-up-with-galaxy-training-network-gtn-to-enhance-training-resources">AI4Life teams up with Galaxy Training Network (GTN) to enhance training resources</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#adding-a-workflow-to-biaflows">Adding a Workflow to BIAFLOWS</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#analysis-of-high-throughput-microscopy-image-data">Analysis of High-Throughput Microscopy Image Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#annotating-3d-images-in-napari">Annotating 3D images in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bids-lecture-2024">BIDS-lecture-2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#biomero-a-scalable-and-extensible-image-analysis-framework">BIOMERO - A scalable and extensible image analysis framework</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#basics-of-image-processing-and-analysis">Basics of Image Processing and Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bigdataprocessor2-a-free-and-open-source-fiji-plugin-for-inspection-and-processing-of-tb-sized-image-data">BigDataProcessor2: A free and open-source Fiji plugin for inspection and processing of TB sized image data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-lecture-2020">Bio Image Analysis Lecture 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-code-generation-using-bia-bob">Bio-image Analysis Code Generation using bia-bob</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-icob-2023">Bio-image Analysis ICOB 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-kioto-and-taipei-23-24">Bio-image Analysis Workshop Kioto and Taipei 23/24</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-taipei">Bio-image Analysis Workshop Taipei</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science">Bio-image Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science-lectures-uni-leipzig-scads-ai">Bio-image Data Science Lectures @ Uni Leipzig / ScaDS.AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">Bio-image analysis, biostatistics, programming and machine learning for computational biology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-notebooks">BioImage Analysis Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-archive-ai-gallery">BioImage Archive AI Gallery</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-archive-visual-gallery">BioImage Archive Visual Gallery</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-archive-volume-em-gallery">BioImage Archive Volume EM Gallery</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-data-analysis">BioImage Data Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-informatics-index-training-materials">BioImage Informatics Index Training Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-model-zoo">Bioimage Model Zoo</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-with-napari">Bioimage analysis with Napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bridging-imaging-users-to-imaging-analysis-a-community-survey">Bridging Imaging Users to Imaging Analysis - A community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bring-your-own-data-workshops">Bring your own data workshops</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-bioimage-analysis-workflow-using-deep-learning">Building a Bioimage Analysis Workflow using Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022">CARE/Stardist tutorials for EMBO Practical Course — Computational optical biology 2022</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#clij-gpu-accelerated-image-processing-for-everyone">CLIJ: GPU-accelerated image processing for everyone</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#coba-center-for-open-bioimage-analysis-youtube-channel">COBA: Center for Open Bioimage Analysis YouTube Channel</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#csbdeep-and-stardist-i2k-2020">CSBDeep and StarDist @ I2K 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cell-tracking-challenge-2d-datasets">Cell Tracking Challenge - 2D Datasets</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cell-tracking-challenge-3d-datasets">Cell Tracking Challenge - 3D Datasets</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-introduction">CellProfiler Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-practical-neubias-lisbon-2017">CellProfiler Practical NeuBIAS Lisbon 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-tutorials">CellProfiler tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">Challenges and opportunities for bio-image analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bioimage-analysis-core-facilities">Challenges and opportunities for bioimage analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#checklists-for-publishing-images-and-image-analysis">Checklists for publishing images and image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#chris-halvin-youtube-channel">Chris Halvin YouTube channel</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis">Collection of teaching material for deep learning for (biomedical) image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#community-developed-checklists-for-publishing-images-and-image-analyses">Community-developed checklists for publishing images and image analyses</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#course-on-deep-learning-for-imaging-using-pytorch">Course on Deep Learning for imaging using PyTorch</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training">Cultivating Open Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-napari-napari-as-a-tool-for-deep-learning-project-management">DEEP NAPARI : Napari as a tool for deep learning project management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4miceverywhere">DL4MicEverywhere</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl-mbl-2021-exercises">DL@MBL 2021 Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dask-course">Dask Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-management-at-france-bioimaging">Data management at France BioImaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-for-image-analysis-exercises">Deep Learning for image analysis - Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deepprofiler-handbook">DeepProfiler Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#developing-semi-automatic-analysis-pipelines-and-technological-solutions-for-metadata-annotation-and-management-in-high-content-screening-hcs-bioimaging">Developing (semi)automatic analysis pipelines and technological solutions for metadata annotation and management in high-content screening (HCS) bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#diffusion-models-for-image-restoration-an-introduction">Diffusion Models for Image Restoration - An Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy">Dr Guillaume Jacquemet on studying cancer cell metastasis in the era of deep learning for microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats">EDAM-bioimaging - The ontology of bioimage informatics operations, topics, data, and formats</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embo-practical-course-advanced-methods-in-bioimage-analysis">EMBO Practical Course Advanced methods in bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#epflx-image-processing-and-analysis-for-life-scientists">EPFLx: Image Processing and Analysis for Life Scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embedseg-repository">EmbedSeg Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#erick-martins-ratamero-expanding-the-ome-ecosystem-for-imaging-data-management-scipy-2024">Erick Martins Ratamero - Expanding the OME ecosystem for imaging data management | SciPy 2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus">EuBIAS course 2013: Intensity Dynamics at the Periphery of Nucleus</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-s-template-for-research-data-management-plans">Euro-BioImaging’s Template for Research Data Management Plans</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#example-pipeline-tutorial">Example Pipeline Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fair-bioimage-data">FAIR BioImage Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fairy-deep-learning-for-bioimage-analysis">FAIRy deep-learning for bioImage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#feature-extraction-in-napari">Feature extraction in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fiji-is-just-imagej-tutorials">Fiji Is Just ImageJ Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging">Fit for OMERO: How imaging facilities and IT departments work together to enable RDM for bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-in-digital-image-processing">Fundamentals in digital image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends">Fundamentals of image analysis in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-training">Galaxy Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#generative-artificial-intelligence-for-bio-image-analysis">Generative artificial intelligence for bio-image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#global-bioimaging-training-database">Global BioImaging Training Database</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero">High throughput &amp; automated data analysis and data management workflow with Cellprofiler and OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists">Highlights from the 2016-2020 NEUBIAS training schools for Bioimage Analysts: a success story and key asset for analysts and life scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#hitchhiking-through-a-diverse-bio-image-analysis-software-universe">Hitchhiking through a diverse Bio-image Analysis Software Universe</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library">I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-virtual-bio-image-analysis-code-generation">I2K2024(virtual) - Bio-Image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#iafig-rms-python-for-bioimage-analysis-course">IAFIG-RMS Python for Bioimage Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#itkelastix-examples">ITKElastix Examples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ibiology-bioimage-analysis-course-the-life-cycle-of-an-image-data-set">Ibiology. Bioimage Analysis Course. The Life Cycle of an Image Data Set</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-training-resources">Image Analysis Training Resources</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-with-python">Image Processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends">Image analysis and visualization in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-in-galaxy">Image analysis in Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-for-beginners">Image processing for beginners</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-based-profiling-handbook">Image-based Profiling Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej-macro-introduction">ImageJ Macro Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej2-api-beating">ImageJ2 API-beating</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imaris-tutorials">Imaris Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-bioimage-analysis">Introduction to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis-with-fiji">Introduction to Image Analysis with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden">Introduction to ImageJ macro programming, Scientific Computing Facility, MPI CBG Dresden</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#jupyter-for-interactive-cloud-computing">Jupyter for interactive cloud computing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-applied-bioimage-analysis-2020">Lecture Applied Bioimage Analysis 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-deep-learning-applications-to-bioimage-analysis">Machine Learning - Deep Learning. Applications to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-workflow-for-imaging-flow-cytometry-ifc">Machine Learning Workflow for Imaging Flow Cytometry (IFC)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-and-deep-learning-on-the-cloud-segmentation">Machine and Deep Learning on the cloud: Segmentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#making-the-most-of-bioimaging-data-through-interdisciplinary-interactions">Making the most of bioimaging data through interdisciplinary interactions</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration">Meeting in the Middle: Towards Successful Multidisciplinary Bioimage Analysis Collaboration</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#melbourne-advanced-microscopy-facility">Melbourne Advanced Microscopy Facility</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#methods-in-bioimage-analysis">Methods in bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#metrics-reloaded-a-framework-for-trustworthy-image-analysis-validation">Metrics Reloaded - A framework for trustworthy image analysis validation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microsam-talks">MicroSam-Talks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive">Microscopy data analysis: machine learning and the BioImage Archive</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#modular-training-resources-for-bioimage-analysis">Modular training resources for bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#morpholibj-documentation">MorphoLibJ documentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multi-view-fusion">Multi-view fusion</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multimodal-large-language-models-for-bioimage-analysis">Multimodal large language models for bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter">NEUBIAS Academy @HOME: Interactive Bioimage Analysis with Python and Jupyter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-analyst-school-2018">NEUBIAS Analyst School 2018</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-course-2017">NEUBIAS Bioimage Analyst Course 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2019">NEUBIAS Bioimage Analyst School 2019</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2020">NEUBIAS Bioimage Analyst School 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage">NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubiaspasteur2023-advancedcellpose">NeubiasPasteur2023_AdvancedCellPose</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks">OMERO - HCS analysis pipeline using Jupyter Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-qupath">OMERO - QuPath</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#onbi-image-analysis-course">ONBI Image Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-tools">Omero-tools</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-image-data-handbook">Open Image Data Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-early-career-bio-image-analysis-training-school">PoL Bio-Image Analysis Early Career Bio-image Analysis Training School</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-gpu-accelerated-image-analysis-training-school">PoL Bio-Image Analysis GPU Accelerated Image Analysis Training School</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-in-bio-image-analysis">Prompt Engineering in Bio-image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-bioimage-analysis-tutorial">Python BioImage Analysis Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-for-microscopists">Python for Microscopists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-for-python-programmers">QuPath for Python programmers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-open-source-software-for-analysing-awkward-images">QuPath: Open source software for analysing (awkward) images</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quality-assurance-of-segmentation-results">Quality assurance of segmentation results</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdf-as-a-bridge-to-domain-platforms-like-omero-or-there-and-back-again">RDF as a bridge to domain-platforms like OMERO, or There and back again.</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rescaling-images-and-pixel-an-isotropy">Rescaling images and pixel (an)isotropy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#running-deep-learning-scripts-in-the-bia-pol-omero-server">Running Deep-Learning Scripts in the BiA-PoL Omero Server</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scipy-cookbook">Scipy Cookbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scripts-filopodyanr-a-case-study-for-the-neubias-ts7-in-szeged">Scripts_FilopodyanR - a case study for the NEUBIAS TS7 in Szeged</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#simpleitk-notebooks">SimpleITK-Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-bioimage-analysis-with-python">Teaching Bioimage Analysis with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-imagej-fiji">Teaching ImageJ FIJI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-digital-cell-cell-biology-as-a-data-science">The Digital Cell: Cell Biology as a Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-open-microscopy-environment-ome-data-model-and-xml-file-open-tools-for-informatics-and-quantitative-analysis-in-biological-imaging">The Open Microscopy Environment (OME) Data Model and XML file - open tools for informatics and quantitative analysis in biological imaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#towards-community-driven-metadata-standards-for-light-microscopy-tiered-specifications-extending-the-ome-model">Towards community-driven metadata standards for light microscopy - tiered specifications extending the OME model</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-theory-trackmate-and-mastodon">Tracking Theory, TrackMate, and Mastodon</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-in-napari">Tracking in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-of-mitochondria-and-capturing-mitoflashes">Tracking of mitochondria and capturing mitoflashes</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-deep-learning-models-for-vision-compact-course">Training Deep Learning Models for Vision - Compact Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ultrack-i2k-2024-workshop-materials">Ultrack I2K 2024 Workshop Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#understanding-metric-related-pitfalls-in-image-analysis-validation">Understanding metric-related pitfalls in image analysis validation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#upcoming-image-analysis-events">Upcoming Image Analysis Events</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#using-glittr-org-to-find-compare-and-re-use-online-training-materials">Using Glittr.org to find, compare and re-use online training materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#v4sdb-winter-school-2025">V4SDB_Winter_School_2025</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#welcome-to-bioimage-town">Welcome to BioImage Town</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-bioimage-analysis-an-introduction">What is Bioimage Analysis? An Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#who-you-gonna-call-data-stewards-to-the-rescue">Who you gonna call? - Data Stewards to the rescue</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-objects-in-2d-and-3d">Working with objects in 2D and 3D</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-pixels">Working with pixels</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-june2024-madrid">Workshop-June2024-Madrid</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zen-python-workshop">ZEN &amp; Python workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zidas-2020-introduction-to-deep-learning">ZIDAS 2020 Introduction to Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zerocostdl4mic-exploiting-google-colab-to-develop-a-free-and-open-source-toolbox-for-deep-learning-in-microscopy">ZeroCostDL4Mic: exploiting Google Colab to develop a free and open-source toolbox for Deep-Learning in microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cordi-2023-zarr-a-cloud-optimized-storage-for-interactive-access-of-large-arrays">[CORDI 2023] Zarr: A Cloud-Optimized Storage for Interactive Access of Large Arrays</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#arivis-vision4d-tutorials">arivis Vision4D Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ilastik-interactive-machine-learning-for-bio-image-analysis">ilastik: interactive machine learning for (bio)image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis">introduction-to-image-analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#patho-prompt-injection">patho_prompt_injection</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quantixed-thedigitalcell-first-complete-code-set">quantixed/TheDigitalCell: First complete code set</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scanpy-tutorials">scanpy-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scikit-learn-mooc">scikit-learn MOOC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#skimage-tutorials">skimage-tutorials</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Robert Haase, Clément Caporal,... and the NFDI4BioImage Initiative
</p>

  </div>
  
  <div class="footer-item">
    

  </div>
  
  <div class="footer-item">
    <p class="last-updated">
  Last updated on 2025-02-13.
  <br/>
</p>
  </div>
  
  <div class="footer-item">
    
<div class="extra_footer">
  <p>
Copyright: Licensed <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY 4.0</a> unless mentioned otherwise. 
Contributions and feedback are welcome.
</p>

</div>
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script defer src="../_static/scripts/bootstrap.js?digest=8878045cc6db502f8baf"></script>
<script defer src="../_static/scripts/pydata-sphinx-theme.js?digest=8878045cc6db502f8baf"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>