
<!DOCTYPE html>


<html lang="en" data-content_root="../" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Include in dalia (338) &#8212; NFDI4BioImage Training Materials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="../_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="../_static/styles/sphinx-book-theme.css?v=eba8b062" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.8ecb98da25f57f5357bf6f572d296f466b2cfe2517ffebfabe82451661e28f02.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="../_static/css/searchbox.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-design.min.css?v=95c83b7e" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="../_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="../_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="../_static/doctools.js?v=9a2dae69"></script>
    <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="../_static/copybutton.js?v=f281be69"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/lunr.js/0.6.0/lunr.min.js"></script>
    <script src="../_static/js/searchbox.js"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="../_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'tags/include_in_dalia';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Licensing (7)" href="licensing.html" />
    <link rel="prev" title="Imagej (20)" href="imagej.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
    <meta name="docbuild:last-update" content="2025-10-21"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="../readme.html">
  
  
  
  
  
    
    
      
    
    
    <img src="../_static/logo.png" class="logo__image only-light" alt="NFDI4BioImage Training Materials - Home"/>
    <script>document.write(`<img src="../_static/logo.png" class="logo__image only-dark" alt="NFDI4BioImage Training Materials - Home"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../readme.html">
                    NFDI4BioImage Training Materials
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">What's new</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../whats_new.html">Recently added (9)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Contributing</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../contributing/index.html">How to contribute</a></li>

<li class="toctree-l1"><a class="reference internal" href="../contributing/submit_app.html">Using the Training Materials Submission App</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contributing/format.html">YML format</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By tag</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="ai-ready.html">Ai-ready (86)</a></li>
<li class="toctree-l1"><a class="reference internal" href="artificial_intelligence.html">Artificial intelligence (52)</a></li>
<li class="toctree-l1"><a class="reference internal" href="bioimage_analysis.html">Bioimage analysis (214)</a></li>
<li class="toctree-l1"><a class="reference internal" href="bioinformatics.html">Bioinformatics (20)</a></li>
<li class="toctree-l1"><a class="reference internal" href="data_stewardship.html">Data stewardship (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="exclude_from_dalia.html">Exclude from dalia (460)</a></li>
<li class="toctree-l1"><a class="reference internal" href="fair-principles.html">Fair-principles (29)</a></li>
<li class="toctree-l1"><a class="reference internal" href="fiji.html">Fiji (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="galaxy.html">Galaxy (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="imagej.html">Imagej (20)</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Include in dalia (338)</a></li>
<li class="toctree-l1"><a class="reference internal" href="licensing.html">Licensing (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="metadata.html">Metadata (15)</a></li>
<li class="toctree-l1"><a class="reference internal" href="napari.html">Napari (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="neubias.html">Neubias (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="nfdi4bioimage.html">Nfdi4bioimage (80)</a></li>
<li class="toctree-l1"><a class="reference internal" href="omero.html">Omero (42)</a></li>
<li class="toctree-l1"><a class="reference internal" href="open_science.html">Open science (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="open_source_software.html">Open source software (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="python.html">Python (71)</a></li>
<li class="toctree-l1"><a class="reference internal" href="reproducibility.html">Reproducibility (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="research_data_management.html">Research data management (148)</a></li>
<li class="toctree-l1"><a class="reference internal" href="sharing.html">Sharing (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="workflow.html">Workflow (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="workflow_engine.html">Workflow engine (13)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By content type</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../content_types/blog%20post.html">Blog post (28)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/book.html">Book (21)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/code.html">Code (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/collection.html">Collection (86)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/data.html">Data (94)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/document.html">Document (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/documentation.html">Documentation (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/event.html">Event (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/github%20repository.html">Github repository (70)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/notebook.html">Notebook (57)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/online%20tutorial.html">Online tutorial (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/poster.html">Poster (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/preprint.html">Preprint (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/publication.html">Publication (74)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/slides.html">Slides (85)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/tutorial.html">Tutorial (48)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/video.html">Video (40)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/website.html">Website (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/workshop.html">Workshop (14)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By license</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../licenses/all_rights_reserved.html">All rights reserved (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/apache-2.0.html">Apache-2.0 (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-2-clause.html">Bsd-2-clause (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-3-clause.html">Bsd-3-clause (37)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-3.0.html">Cc-by-3.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-4.0.html">Cc-by-4.0 (418)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-nc-sa-4.0.html">Cc-by-nc-sa-4.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-sa-4.0.html">Cc-by-sa-4.0 (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc0-1.0.html">Cc0-1.0 (25)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-2.0.html">Gpl-2.0 (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-3.0.html">Gpl-3.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/mit.html">Mit (32)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/unknown.html">Unknown (111)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By domain</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../domain/bbbc.broadinstitute.org.html">Bbbc.broadinstitute.org (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/biapol.github.io.html">Biapol.github.io (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/docs.google.com.html">Docs.google.com (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/doi.org.html">Doi.org (353)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/f1000research.com.html">F1000research.com (11)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/focalplane.biologists.com.html">Focalplane.biologists.com (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/git.mpi-cbg.de.html">Git.mpi-cbg.de (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/github.com.html">Github.com (151)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/training.galaxyproject.org.html">Training.galaxyproject.org (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.biorxiv.org.html">Www.biorxiv.org (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.ebi.ac.uk.html">Www.ebi.ac.uk (18)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.nature.com.html">Www.nature.com (21)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.youtube.com.html">Www.youtube.com (29)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/zenodo.org.html">Zenodo.org (334)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Appendix</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../statistics/readme.html">Statistics</a></li>
<li class="toctree-l1"><a class="reference internal" href="../export/readme.html">Open data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../gdpr_compliance.html">GDPR Compliance Documentation</a></li>
<li class="toctree-l1"><a class="reference internal" href="../imprint.html">Imprint</a></li>

</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training/issues/new?title=Issue%20on%20page%20%2Ftags/include_in_dalia.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/tags/include_in_dalia.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Include in dalia (338)</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#management-of-microscopy-image-data-an-overview-of-omero-bioimage-archive-and-image-data-resource-2025-uni-leipzig">“Management of Microscopy Image Data: An overview of OMERO, BioImage Archive and Image Data Resource”  2025 @ Uni Leipzig</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium">“ZENODO und Co.” Was bringt und wer braucht ein Repositorium?</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#mic-workshop-on-bioimage-processing-with-python">2022 MIC Workshop on Bioimage processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#courseimageanalysis-nanocell">2023_CourseImageAnalysis_NanoCell</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#steps-towards-reproducible-research">6 Steps Towards Reproducible Research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-fiji-scripting-tutorial">A Fiji Scripting Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe">A Hitchhiker’s guide through the bio-image analysis software universe</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-perspective-on-fair-and-scalable-access-to-large-image-data">A Perspective on FAIR and Scalable Access to Large Image Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-biologists-guide-to-planning-and-performing-quantitative-bioimaging-experiments">A biologist’s guide to planning and performing quantitative bioimaging experiments</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-global-view-of-standards-for-open-image-data-formats-and-repositories">A global view of standards for open image data formats and repositories</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-journey-to-fair-microscopy-data">A journey to FAIR microscopy data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-practical-guide-to-bioimaging-research-data-management-in-core-facilities">A practical guide to bioimaging research data management in core facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ai-ml-dl-in-bioimage-analysis-webinar">AI ML DL in Bioimage Analysis - Webinar</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#accessible-interactive-spatial-omics-data-visualizations-with-vitessce-and-omero">Accessible Interactive Spatial-Omics Data Visualizations with Vitessce and OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#advanced-scripting-with-fiji">Advanced scripting with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#advancing-fair-image-analysis-in-galaxy-tools-workflows-and-training">Advancing FAIR Image Analysis in Galaxy: Tools, Workflows, and Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten">Alles meins – oder!? Urheberrechte klären für Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#analysis-of-high-throughput-microscopy-image-data">Analysis of High-Throughput Microscopy Image Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#annotating-3d-images-in-napari">Annotating 3D images in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#artificial-intelligence-for-digital-pathology">Artificial Intelligence for Digital Pathology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#basics-of-image-processing-and-analysis">Basics of Image Processing and Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#best-practice-data-life-cycle-approaches-for-the-life-sciences">Best practice data life cycle approaches for the life sciences</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-code-generation">Bio-image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-code-generation-using-bia-bob">Bio-image Analysis Code Generation using bia-bob</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-kioto-and-taipei-23-24">Bio-image Analysis Workshop Kioto and Taipei 23/24</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-taipei">Bio-image Analysis Workshop Taipei</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-with-the-help-of-large-language-models">Bio-image Analysis with the Help of Large Language Models</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science">Bio-image Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science-lectures-2025-uni-leipzig-scads-ai">Bio-image Data Science Lectures 2025 @ Uni Leipzig / ScaDS.AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">Bio-image analysis, biostatistics, programming and machine learning for computational biology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-notebooks">BioImage Analysis Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-and-superresolution-microscopy-workshop-2023-at-dartmouth-college">BioImage Analysis and Superresolution Microscopy Workshop 2023 (at Dartmouth College)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-data-analysis">BioImage Data Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-io-chatbot-globias-seminar">BioImage.IO Chatbot, GloBIAS Seminar</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-with-napari">Bioimage analysis with Napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimaging-ai-textbook">Bioimaging AI Textbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#browsing-the-open-microscopy-image-data-resource-with-python">Browsing the Open Microscopy Image Data Resource with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-fair-image-analysis-pipelines-for-high-content-screening-hcs-data-using-galaxy">Building FAIR image analysis pipelines for high-content-screening (HCS) data using Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-bioimage-analysis-workflow-using-deep-learning">Building a Bioimage Analysis Workflow using Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-fair-image-data-ecosystem-for-microscopy-communities">Building a FAIR image data ecosystem for microscopy communities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-national-research-data-infrastructure">Building a National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022">CARE/Stardist tutorials for EMBO Practical Course — Computational optical biology 2022</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cs231n-convolutional-neural-networks-for-visual-recognition">CS231n Convolutional Neural Networks for Visual Recognition</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#csbdeep-and-stardist-i2k-2020">CSBDeep and StarDist @ I2K 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-introduction">CellProfiler Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-practical-neubias-lisbon-2017">CellProfiler Practical NeuBIAS Lisbon 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-tutorials">CellProfiler tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#center-for-microscopy-and-image-analysis-how-to-guides">Center for Microscopy and Image Analysis How-to Guides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">Challenges and opportunities for bio-image analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bioimage-analysis-core-facilities">Challenges and opportunities for bioimage analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#chatgpt-for-image-analysis">ChatGPT for Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cloud-native-formats-enable-federated-repositories-at-peta-scale">Cloud-Native Formats Enable Federated Repositories at Peta-Scale</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collaborative-working-and-version-control-with-git-hub">Collaborative Working and Version Control with git[hub]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collaborative-bio-image-analysis-script-editing-with-git">Collaborative bio-image analysis script editing with git</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis">Collection of teaching material for deep learning for (biomedical) image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#combining-the-bids-and-arc-directory-structures-for-multimodal-research-data-organization">Combining the BIDS and ARC Directory Structures for Multimodal Research Data Organization</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#community-developed-checklists-for-publishing-images-and-image-analyses">Community-developed checklists for publishing images and image analyses</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#conda-container-and-bots-how-to-build-and-maintain-tool-dependencies-in-workflows-and-training-materials">Conda, Container and Bots - How to Build and Maintain Tool Dependencies in Workflows and Training Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#course-on-deep-learning-for-imaging-using-pytorch">Course on Deep Learning for imaging using PyTorch</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#crashkurs-forschungsdatenmanagement">Crashkurs Forschungsdatenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#creating-workflows-and-advanced-workflow-options">Creating Workflows and Advanced Workflow Options</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#creating-a-research-data-management-plan-using-chatgpt">Creating a Research Data Management Plan using chatGPT</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#creating-open-computational-curricula">Creating open computational curricula</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training">Cultivating Open Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training-to-advance-bio-image-analysis">Cultivating Open Training to advance Bio-image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cytovi-deep-generative-modeling-of-antibody-based-single-cell-technologies">CytoVI Deep generative modeling of antibody-based single cell technologies</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dalia-interchange-format">DALIA Interchange Format</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4miceverywhere-overcoming-reproducibility-challenges-in-deep-learning-microscopy-imaging">DL4MicEverywhere – Overcoming reproducibility challenges in deep learning microscopy imaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-carpentry-for-biologists">Data Carpentry for Biologists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-visualization-with-flying-colors">Data Visualization with Flying Colors</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-handling-in-large-scale-electron-microscopy">Data handling in large-scale electron microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-management-at-france-bioimaging">Data management at France BioImaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-stewardship-and-research-data-management-tools-for-multimodal-linking-of-imaging-data-in-plasma-medicine">Data stewardship and research data management tools for multimodal linking of imaging data in plasma medicine</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dataplant-knowledge-base">DataPLANT knowledge base</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dataviz-protocols-an-introduction-to-data-visualization-protocols-for-wet-lab-scientists">DataViz protocols - An introduction to data visualization protocols for wet lab scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagement">Datenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz">Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-1">Datenmanagementpläne erstellen - Teil 1</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-2">Datenmanagementpläne erstellen - Teil 2</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-based-segmentation-for-biologists">Deep Learning Based Segmentation For Biologists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-for-image-analysis-exercises">Deep Learning for image analysis - Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-vision-and-graphics">Deep Vision and Graphics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deepprofiler-handbook">DeepProfiler Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#developing-a-training-strategy">Developing a Training Strategy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#developing-open-source-software-for-bioimage-analysis-opportunities-and-challenges">Developing open-source software for bioimage analysis: opportunities and challenges</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#development-of-a-platform-for-advanced-optics-education-training-and-prototyping">Development of a platform for advanced optics education, training and prototyping</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#diataxis-a-systematic-approach-to-technical-documentation-authoring">Diátaxis - A systematic approach to technical documentation authoring.</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#docker-mastery-with-kubernetes-swarm-from-a-docker-captain">Docker Mastery - with Kubernetes + Swarm from a Docker Captain</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy">Dr Guillaume Jacquemet on studying cancer cell metastasis in the era of deep learning for microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats-update-2020">EDAM-bioimaging: The ontology of bioimage informatics operations, topics, data, and formats (update 2020)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embi-recommended-metadata-for-biological-images-metadata-guidelines-for-bioimaging-data">EMBI - Recommended Metadata for Biological Images – metadata guidelines for bioimaging data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2019-exercises-and-materials">EMBL Deep Learning course 2019 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2021-22-exercises-and-materials">EMBL Deep Learning course 2021/22 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2023-exercises-and-materials">EMBL Deep Learning course 2023 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#efficiently-starting-institutional-research-data-management">Efficiently starting institutional research data management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten">Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elastix-tutorial">Elastix tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#end-to-end-tissue-microarray-image-analysis-with-galaxy-me">End-to-End Tissue Microarray Image Analysis with Galaxy-ME</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#engineering-a-software-environment-for-research-data-management-of-microscopy-image-data-in-a-core-facility">Engineering a Software Environment for Research Data Management of Microscopy Image Data in a Core Facility</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#erstellung-und-realisierung-einer-institutionellen-forschungsdaten-policy">Erstellung und Realisierung einer institutionellen Forschungsdaten-Policy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus">EuBIAS course 2013: Intensity Dynamics at the Periphery of Nucleus</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-evolve-deliverable-6-2-landscape-analysis-of-existing-training-resources-for-the-nodes">Euro-BioImaging - EVOLVE Deliverable 6.2 - Landscape analysis of existing training resources for the Nodes</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#expansion-and-fluctuations-enhanced-microscopy-for-nanoscale-molecular-profiling-of-cells-and-tissues-data-processing-manual">Expansion and fluctuations-enhanced microscopy for nanoscale molecular profiling of cells and tissues - Data processing manual</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#explainable-ai-for-computer-vision">Explainable AI for Computer Vision</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fair-priciples">FAIR Priciples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fairy-deep-learning-for-bioimage-analysis">FAIRy deep-learning for bioImage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#faq-kunstliche-intelligenz-und-gute-wissenschaftliche-praxis">FAQ Künstliche Intelligenz und gute wissenschaftliche Praxis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#feature-extraction-in-napari">Feature extraction in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#file-naming-convention-worksheet">File Naming Convention Worksheet</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#finding-and-choosing-a-data-repository">Finding and Choosing a Data Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#finding-and-using-publicly-available-data">Finding and using publicly available data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#five-great-reasons-to-share-your-research-data">Five great reasons to share your research data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#five-ways-to-effectively-use-animation-in-data-visualization">Five ways to effectively use animation in data visualization</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#forschungsdaten-info">Forschungsdaten.info</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#forschungsdaten-org">Forschungsdaten.org</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#friendsdontletfriends">FriendsDontLetFriends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#from-cells-to-pixels-bridging-biologists-and-image-analysts-through-a-common-language">From Cells to Pixels: Bridging Biologists and  Image Analysts Through a Common Language</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers">From Paper to Pixels: Navigation through your Research Data - presentations of speakers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#from-bioimaging-projects-to-communities-globias-bia-seminar-series">From bioimaging projects to communities - GloBIAS BIA Seminar Series</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-in-digital-image-processing">Fundamentals in digital image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends">Fundamentals of image analysis in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-meets-omero-overview-on-the-galaxy-omero-suite-and-vizarr-viewer">Galaxy meets OMERO! Overview on the Galaxy OMERO-suite and Vizarr Viewer</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-workflows">Galaxy workflows</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#generative-artificial-intelligence-for-bio-image-analysis">Generative artificial intelligence for bio-image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#getting-started-with-mambaforge-and-python">Getting started with Mambaforge and Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#getting-started-with-python-intro-and-set-up-a-conda-environment">Getting started with Python: intro and set-up a conda environment</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#guidance-for-developing-a-research-data-management-rdm-policy">Guidance for Developing a Research Data Management (RDM) Policy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero">High throughput &amp; automated data analysis and data management workflow with Cellprofiler and OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#high-content-screening-hcs-workflows-for-fair-image-data-management-with-omero">High-content screening (HCS) workflows for FAIR image data management with OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists">Highlights from the 2016-2020 NEUBIAS training schools for Bioimage Analysts: a success story and key asset for analysts and life scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#how-to-write-a-bug-report">How to write a bug report</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library">I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-virtual-bio-image-analysis-code-generation">I2K2024(virtual) - Bio-Image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#iafig-rms-python-for-bioimage-analysis-course">IAFIG-RMS Python for Bioimage Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#idc-tutorials">IDC-Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work">If you license it, it’ll be harder to steal it. Why we should license our work</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-training-resources">Image Analysis Training Resources</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-in-healthcare-the-current-landscape-trends-and-collaborative-opportunities">Image Analysis in Healthcare - The Current Landscape, Trends, and Collaborative Opportunities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-using-galaxy">Image Analysis using Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-with-python">Image Processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-repository-decision-tree-where-do-i-deposit-my-imaging-data">Image Repository Decision Tree - Where do I deposit my imaging data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends">Image analysis and visualization in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-course-material">Image analysis course material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-handling-using-fiji-training-materials">Image handling using Fiji - training materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-for-beginners">Image processing for beginners</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#id1">Image processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-based-profiling-handbook">Image-based Profiling Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej-macro-introduction">ImageJ Macro Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej2-api-beating">ImageJ2 API-beating</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#increasing-the-fairness-of-electron-microscopy-data-in-life-and-material-science-research">Increasing the FAIRness of electron microscopy data in life and material science research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#interactive-bioimage-analysis-workflow-with-clij-eabias-2025-training-event">Interactive Bioimage Analysis Workflow with CLIJ (@EABIAS 2025 training event)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#interactive-image-data-flow-graphs">Interactive Image Data Flow Graphs</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#internal-alm-bioimage-analysis-workshop-2023">Internal ALM BioImage Analysis workshop 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#intro-napari-slides">Intro napari slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-bioimage-analysis">Introduction to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-deep-learning-for-microscopy">Introduction to Deep Learning for Microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-high-performance-computing-for-life-scientists">Introduction to High Performance Computing for Life Scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis">Introduction to Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis-with-fiji">Introduction to Image Analysis with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden">Introduction to ImageJ macro programming, Scientific Computing Facility, MPI CBG Dresden</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-omero-frankfurt-online">Introduction to OMERO - Frankfurt - online</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-research-data-management-and-open-research">Introduction to Research Data Management and Open Research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#jipipe-spring-course-jsc-2025-workshop-recordings-slides-homework-and-materials">JIPipe Spring Course (JSC) 2025: Workshop Recordings, Slides, Homework, and Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kollaboratives-arbeiten-und-versionskontrolle-mit-git">Kollaboratives Arbeiten und Versionskontrolle mit Git</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kreshuk-lab-s-embl-eipp-predoc-course-teaching-material">Kreshuk Lab’s EMBL EIPP predoc course teaching material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kriterienkatalog-fur-materialien-aus-dem-themenbereich-forschungsdatenmanagement">Kriterienkatalog für Materialien aus dem Themenbereich Forschungsdatenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#large-language-models-an-introduction-for-life-scientists">Large Language Models: An Introduction for Life Scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#latent-space-visualisation-pca-t-sne-umap-deep-learning-animated">Latent Space Visualisation: PCA, t-SNE, UMAP | Deep Learning Animated</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#learning-and-training-bio-image-analysis-in-the-age-of-ai">Learning and Training Bio-image Analysis in the Age of AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-applied-bioimage-analysis-2020">Lecture Applied Bioimage Analysis 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#leitfaden-zur-digitalen-datensparsamkeit-mit-praxisbeispielen">Leitfaden zur digitalen Datensparsamkeit (mit Praxisbeispielen)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#liver-micrometastases-area-quantification-using-qupath-and-pixel-classifier">Liver Micrometastases area quantification using QuPath and pixel classifier</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-deep-learning-applications-to-bioimage-analysis">Machine Learning - Deep Learning. Applications to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-workflow-for-imaging-flow-cytometry-ifc">Machine Learning Workflow for Imaging Flow Cytometry (IFC)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#making-the-most-of-bioimaging-data-through-interdisciplinary-interactions">Making the most of bioimaging data through interdisciplinary interactions</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#making-your-package-available-on-conda-forge">Making your package available on conda-forge</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#making-your-project-citable">Making your project citable</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#managing-scientific-python-environments-using-conda-mamba-and-friends">Managing Scientific Python environments using Conda, Mamba and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration">Meeting in the Middle: Towards Successful Multidisciplinary Bioimage Analysis Collaboration</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#metabolic-networks-modelling-with-cobrapy">Metabolic networks modelling with COBRApy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#metadata-matters-access-to-image-data-in-the-real-world">Metadata matters: access to image data in the real world</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#methods-in-bioimage-analysis">Methods in bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive">Microscopy data analysis: machine learning and the BioImage Archive</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#modular-training-resources-for-bioimage-analysis">Modular training resources for bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#morphological-analysis-of-neural-cells-with-weka-and-snt-fiji-plugins">Morphological analysis of neural cells with WEKA and SNT Fiji plugins</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multi-template-matching-for-object-detection-slides">Multi-Template-Matching for object-detection (slides)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multi-view-fusion">Multi-view fusion</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud">My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter">NEUBIAS Academy @HOME: Interactive Bioimage Analysis with Python and Jupyter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-analyst-school-2018">NEUBIAS Analyst School 2018</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-course-2017">NEUBIAS Bioimage Analyst Course 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2019">NEUBIAS Bioimage Analyst School 2019</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2020">NEUBIAS Bioimage Analyst School 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage">NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-data-management-illustrations-by-henning-falk">NFDI4BIOIMAGE data management illustrations by Henning Falk</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nlp-course">NLP Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#name-styles">Name Styles</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#navigating-the-bioimage-analysis-landscape-understanding-the-community-and-its-collaborative-dynamics">Navigating the Bioimage Analysis Landscape: Understanding the Community  and its Collaborative Dynamics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubiaspasteur2023-advancedcellpose">NeubiasPasteur2023_AdvancedCellPose</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-demo-pipelines-for-image-processing">Nextflow Demo Pipelines for Image Processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome-documentation">OME Documentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks">OMERO - HCS analysis pipeline using Jupyter Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-guide">OMERO Guide</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-and-arc-workflow">OMERO and ARC Workflow</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-documentation">OMERO documentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-walkthrough-example">OMERO walkthrough example</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-walkthrough-for-facility-managers">OMERO walkthrough for facility managers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omexcavator-a-tool-for-exporting-and-connecting-bioimaging-specific-metadata-in-wider-knowledge-graphs">OMExcavator: a tool for exporting and connecting  Bioimaging-specific metadata in wider knowledge graphs</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#onbi-image-analysis-course">ONBI Image Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-deployment-examples">Omero Deployment examples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#online-r-learning">Online_R_learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-image-data-handbook">Open Image Data Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-science-sharing-licensing">Open Science, Sharing &amp; Licensing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-source-ai-tools-for-bioimage-analysis-workshop-2024-icob-academia-sinica-taiwan">Open source AI Tools for bioimage analysis workshop (2024) @ICOB, Academia Sinica, Taiwan</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#overview-of-the-galaxy-omero-suite-upload-images-and-metadata-in-omero-using-galaxy">Overview of the Galaxy OMERO-suite - Upload images and metadata in OMERO using Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing">Parallelization and heterogeneous computing: from pure CPU to GPU-accelerated image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#photonic-data-analysis-in-2050">Photonic data analysis in 2050</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plants-python-a-series-of-lessons-in-coding-plant-biology-computation-and-bioinformatics">Plants &amp; Python - A series of lessons in coding, plant biology, computation, and bioinformatics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-early-career-bio-image-analysis-training-school">PoL Bio-Image Analysis Early Career Bio-image Analysis Training School</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-gpu-accelerated-image-analysis-training-school">PoL Bio-Image Analysis GPU Accelerated Image Analysis Training School</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis">PoL Bio-Image Analysis Training School on GPU-Accelerated Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#practical-guide-to-the-international-alignment-of-research-data-management-extended-edition">Practical Guide to the International Alignment of Research Data Management - Extended Edition</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#practical-considerations-for-data-exploration-in-quantitative-cell-biology">Practical considerations for data exploration in quantitative cell biology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#promoting-data-management-at-the-nikon-imaging-center-and-cell-biology-microscopy-facility">Promoting Data Management at the Nikon Imaging Center and Cell Biology Microscopy Facility</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#promoting-reproducibility-in-biomedical-research-through-image-analysis">Promoting Reproducibility in Biomedical Research through Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-in-bio-image-analysis">Prompt Engineering in Bio-image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-agentic-workflows-and-multi-modal-large-language-models">Prompt Engineering, Agentic Workflows and Multi-modal Large Language Models</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-llms-course">Prompt-Engineering-LLMs-Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pygameprompts">PygamePrompts</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-bioimage-analysis-tutorial">Python BioImage Analysis Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-workshop-image-processing">Python Workshop - Image Processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-for-bioimage-analysis-basic-tools-and-setup-on-windows">Python for Bioimage Analysis Basic Tools and Setup on Windows</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-for-microscopists">Python for Microscopists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qi-2024-analysis-lab-manual">QI 2024 Analysis Lab Manual</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qm-course-lectures-on-bio-image-analysis-with-napari-2024">QM Course Lectures on Bio-Image Analysis with napari 2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quarep-limi-a-community-driven-initiative-to-establish-guidelines-for-quality-assessment-and-reproducibility-for-instruments-and-images-in-light-microscopy">QUAREP-LiMi: A community-driven initiative to establish guidelines for quality assessment and reproducibility for instruments and images in light microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-for-python-programmers">QuPath for Python programmers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-open-source-software-for-analysing-awkward-images">QuPath: Open source software for analysing (awkward) images</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quality-assurance-of-segmentation-results">Quality assurance of segmentation results</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdm-starter-kit">RDM Starter Kit</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdmkit-training-resources">RDMKit Training Resources</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rembi-overview">REMBI Overview</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rechtsfragen-bei-open-science-ein-leitfaden">Rechtsfragen bei Open Science - Ein Leitfaden</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#report-on-a-pilot-study-implementation-of-omero-for-microscopy-data-management">Report on a pilot study:  Implementation of OMERO for  microscopy data management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#reproducible-bio-image-analysis-using-python-napari-jupyter-and-ai">Reproducible Bio-Image Analysis using Python, Napari, Jupyter and AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rescaling-images-and-pixel-an-isotropy">Rescaling images and pixel (an)isotropy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-management-seminar-slides">Research Data Management Seminar - Slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-managemet-and-how-not-to-get-overwhelmed-with-data">Research Data Managemet and how not to get overwhelmed with data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#running-deep-learning-scripts-in-the-bia-pol-omero-server">Running Deep-Learning Scripts in the BiA-PoL Omero Server</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#swc-gcnu-software-skills">SWC/GCNU Software Skills</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scientific-visualization-python-matplotlib">Scientific Visualization: Python + Matplotlib</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scipy-cookbook">Scipy Cookbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-a-data-management-infrastructure-for-bioimaging">Setting up a data management infrastructure for bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-a-remote-desktop-to-use-napari-in-a-browser">Setting up a remote desktop to use Napari in a browser</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sharing-your-poster-on-figshare-a-community-guide-to-how-to-and-why">Sharing Your Poster on Figshare: A Community Guide to How-To and Why</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sharing-and-licensing-material">Sharing and licensing material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sharing-research-data-with-zenodo">Sharing research data with Zenodo</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#simpleitk-notebooks">SimpleITK-Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#slides-about-flute-a-python-gui-for-interactive-phasor-analysis-of-flim-data">Slides about FLUTE: a Python GUI for interactive phasor analysis of FLIM data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten">So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#software-citation-with-citation-cff">Software Citation with CITATION.cff</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#source-control-using-git-and-github">Source Control Using Git and GitHub</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data">Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sustainable-data-stewardship">Sustainable Data Stewardship</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-bioimage-analysis-with-python">Teaching Bioimage Analysis with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-imagej-fiji">Teaching ImageJ FIJI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ten-simple-rules-for-making-training-materials-fair">Ten simple rules for making training materials FAIR</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-digital-cell-cell-biology-as-a-data-science">The Digital Cell: Cell Biology as a Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-turing-way-guide-for-reproducible-research">The Turing Way: Guide for reproducible research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-crucial-role-of-bioimage-analysts-in-scientific-research-and-publication">The crucial role of bioimage analysts in scientific research and publication</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#things-that-shouldn-t-be-in-file-names-for-1000-alex">Things that shouldn’t be in file names for $1000 Alex</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#thinking-data-management-on-different-scales">Thinking data management on different scales</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#towards-open-and-standardised-imaging-data-an-introduction-to-bio-formats-ome-tiff-and-ome-zarr">Towards open and standardised imaging data: an introduction to Bio-Formats, OME-TIFF, and OME-Zarr</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-theory-trackmate-and-mastodon">Tracking Theory, TrackMate, and Mastodon</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-in-napari">Tracking in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-of-mitochondria-and-capturing-mitoflashes">Tracking of mitochondria and capturing mitoflashes</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#train-the-trainer-concept-on-research-data-management">Train-the-Trainer Concept on Research Data Management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-computational-skills-in-the-age-of-ai">Training Computational Skills in the Age of AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-deep-learning-models-for-vision-compact-course">Training Deep Learning Models for Vision - Compact Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-concepts-in-research-data-management-and-data-science-with-the-focus-on-health-research">Training concepts in research data management and data science with the focus on health research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ultrack-i2k-2024-workshop-materials">Ultrack I2K 2024 Workshop Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#user-friendly-image-metadata-annotation-tool-workflow-for-omero">User friendly Image metadata annotation tool/workflow for OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#virtual-i2k-2024-multiview-stitcher">Virtual-I2K-2024-multiview-stitcher</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#vision-language-models-for-bio-image-data-science">Vision Language Models for Bio-image Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-not-to-do-when-creating-a-data-management-plan-dmp">WHAT NOT TO DO WHEN CREATING A DATA MANAGEMENT PLAN (DMP)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-open-data">What is Open Data?</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#when-data-doesn-t-fit">When Data Doesn’t Fit</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#who-you-gonna-call-data-stewards-to-the-rescue">Who you gonna call? - Data Stewards to the rescue</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workflow-for-user-introduction-into-microscopy-omero-and-data-management-at-center-for-advanced-imaging">Workflow for user introduction into microscopy, OMERO and data management at Center for Advanced imaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-objects-in-2d-and-3d">Working with objects in 2D and 3D</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-pixels">Working with pixels</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zen-python-workshop">ZEN &amp; Python workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zidas-2020-introduction-to-deep-learning">ZIDAS 2020 Introduction to Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cidas-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[CIDAS] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cmcb-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[CMCB] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without">[ELMI 2024]  AI’s Dirty Little Secret: Without</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi2025-bridging-communities-with-ome-zarr">[ELMI2025] Bridging communities with OME-Zarr</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi2025-the-road-to-ome-zarr-1-0">[ELMI2025] The Road to OME-Zarr 1.0</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi2025-workshop-fair101-navigating-fair-data-from-principles-to-practice">[ELMI2025] Workshop: FAIR101 - Navigating FAIR data from principles to practice</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#gbi-eoe-vii-five-or-ten-must-have-items-for-making-it-infrastructure-for-managing-bioimage-data">[GBI EOE VII] Five (or ten) must-have items for making IT infrastructure for managing bioimage data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[I2K] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi-tech-talk-cloud-based-image-science">[NFDI Tech Talk] Cloud Based Image Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard">[SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#webinar-a-journey-to-fair-bioimage-data">[Webinar] A journey to FAIR bioimage data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg">[Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-bioimage-data-management-and-analysis-with-omero">[Workshop] Bioimage data management and analysis with OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero">[Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-managing-fair-microscopy-data-at-scale-for-universities-and-research-institutions-an-introduction-for-non-imaging-stakeholders">[Workshop] Managing FAIR microscopy data at scale for universities and research institutions: an introduction for non-imaging stakeholders</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-research-data-management-for-microscopy-and-bioimage-analysis">[Workshop] Research Data Management for Microscopy and BioImage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-visualization-tutorial">data_visualization_tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-basics">image-processing-basics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagequantification101">imagequantification101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-generative-ai">introduction-to-generative-ai</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#id2">introduction-to-image-analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#napari-scipy2025-workshop">napari-scipy2025-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-workshop">nextflow-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow101">nextflow101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#numpy-pandas-course">numpy pandas course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-workshop">qupath-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rse-skills-workshop">rse-skills-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scanpy-tutorials">scanpy-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scikit-learn-mooc">scikit-learn MOOC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#shapeanalysis101">shapeanalysis101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#skimage-tutorials">skimage-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-resources">training-resources</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="include-in-dalia-338">
<h1>Include in dalia (338)<a class="headerlink" href="#include-in-dalia-338" title="Link to this heading">#</a></h1>
<section id="management-of-microscopy-image-data-an-overview-of-omero-bioimage-archive-and-image-data-resource-2025-uni-leipzig">
<h2>“Management of Microscopy Image Data: An overview of OMERO, BioImage Archive and Image Data Resource”  2025 &#64; Uni Leipzig<a class="headerlink" href="#management-of-microscopy-image-data-an-overview-of-omero-bioimage-archive-and-image-data-resource-2025-uni-leipzig" title="Link to this heading">#</a></h2>
<p>Vellutini, Bruno C., Cuenca, Marina B., Abhijeet Krishna, Alicja Szałapak, Modes, Carl D., Pavel Tomančák</p>
<p>Published 2025-08-15</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation matherial from the course “Management of Microscopy Image Data: An Overview of OMERO, BioImage Archive and Image Data Resource” helded in Uni Leipzig on the 04/07/2025.
This course is part of the RDM lecture series organized by Dr. Dr. habil. Dagmar Quandt.
Link to the event:
<a class="reference external" href="https://fortbildung.uni-leipzig.de/fortbildung.html?id=2436">https://fortbildung.uni-leipzig.de/fortbildung.html?id=2436</a>
M. Massei is funded by the Deutsche Forschungsgemeinschaft (DFG) – project number [NFDI46/1] - 501864659</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/16880913">https://zenodo.org/records/16880913</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.16880913">https://doi.org/10.5281/zenodo.16880913</a></p>
</section>
<hr class="docutils" />
<section id="zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium">
<h2>“ZENODO und Co.” Was bringt und wer braucht ein Repositorium?<a class="headerlink" href="#zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium" title="Link to this heading">#</a></h2>
<p>Elfi Hesse, Jan-Christoph Deinert, Christian Löschen</p>
<p>Published 2021-01-25</p>
<p>Licensed CC-BY-4.0</p>
<p>Die Online-Veranstaltung fand am 21.01.2021 im Rahmen der SaxFDM-Veranstaltungsreihe “Digital Kitchen - Küchengespräche mit SaxFDM” statt. SaxFDM-Sprecherin Elfi Hesse (HTW Dresden) erläuterte zunächst Grundsätzliches zum Thema Repositorien. Anschließend teilten Nutzer (Jan Deinert – HZDR) und Anbieter (Christian Löschen – TU Dresden/ZIH) lokaler Repositorien ihre Erfahrungen mit uns.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4461261">https://zenodo.org/records/4461261</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4461261">https://doi.org/10.5281/zenodo.4461261</a></p>
</section>
<hr class="docutils" />
<section id="mic-workshop-on-bioimage-processing-with-python">
<h2>2022 MIC Workshop on Bioimage processing with Python<a class="headerlink" href="#mic-workshop-on-bioimage-processing-with-python" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/MICPy_Workshop_2022">guiwitz/MICPy_Workshop_2022</a></p>
</section>
<hr class="docutils" />
<section id="courseimageanalysis-nanocell">
<h2>2023_CourseImageAnalysis_NanoCell<a class="headerlink" href="#courseimageanalysis-nanocell" title="Link to this heading">#</a></h2>
<p>Fernanda Fossa</p>
<p>Published 2023-08-24T10:15:11+00:00</p>
<p>Licensed MIT</p>
<p>Course about image analysis with materials in English and videos in Portuguese.</p>
<p>Tags: Cell Profiler, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/fefossa/2023_CourseImageAnalysis_NanoCell">fefossa/2023_CourseImageAnalysis_NanoCell</a></p>
</section>
<hr class="docutils" />
<section id="steps-towards-reproducible-research">
<h2>6 Steps Towards Reproducible Research<a class="headerlink" href="#steps-towards-reproducible-research" title="Link to this heading">#</a></h2>
<p>Heidi Seibold</p>
<p>Licensed CC-BY-4.0</p>
<p>A short book with 6 steps that get you closer to making your work reproducible.</p>
<p>Tags: Reproducibility, Research Data Management, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://zenodo.org/records/12744715">https://zenodo.org/records/12744715</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.12744715">https://doi.org/10.5281/zenodo.12744715</a></p>
</section>
<hr class="docutils" />
<section id="a-fiji-scripting-tutorial">
<h2>A Fiji Scripting Tutorial<a class="headerlink" href="#a-fiji-scripting-tutorial" title="Link to this heading">#</a></h2>
<p>Albert Cardona</p>
<p>Licensed CC0-1.0</p>
<p>Tags: Imagej, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="reference external" href="https://syn.mrc-lmb.cam.ac.uk/acardona/fiji-tutorial/">https://syn.mrc-lmb.cam.ac.uk/acardona/fiji-tutorial/</a></p>
</section>
<hr class="docutils" />
<section id="a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">
<h2>A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter<a class="headerlink" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter" title="Link to this heading">#</a></h2>
<p>Anca Margineanu, Chiara Stringari, Marcelo Zoccoler, Cornelia Wetzker</p>
<p>Licensed CC-BY-4.0</p>
<p>The presentations introduce open-source software to read in, visualize and analyse fluorescence lifetime imaging microscopy (FLIM) raw data developed for life scientists. The slides were presented at German Bioimaging (GerBI) FLIM Workshop held February 26 to 29 2024 at the Biomedical Center of LMU München by Anca Margineanu, Chiara Stringari and Conni Wetzker.</p>
<p>Tags: Bioimage Analysis, Flim, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10886749">https://zenodo.org/doi/10.5281/zenodo.10886749</a></p>
</section>
<hr class="docutils" />
<section id="a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe">
<h2>A Hitchhiker’s guide through the bio-image analysis software universe<a class="headerlink" href="#a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe" title="Link to this heading">#</a></h2>
<p>Robert Haase, Elnaz Fazeli, David Legland, Michael Doube, Siân Culley, Ilya Belevich, Eija Jokitalo, Martin Schorb, Anna Klemm, Christian Tischer</p>
<p>Licensed CC-BY-4.0</p>
<p>This article gives an overview about commonly used bioimage analysis software and which aspects to consider when choosing a software for a specific project.</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://febs.onlinelibrary.wiley.com/doi/full/10.1002/1873-3468.14451">https://febs.onlinelibrary.wiley.com/doi/full/10.1002/1873-3468.14451</a></p>
</section>
<hr class="docutils" />
<section id="a-perspective-on-fair-and-scalable-access-to-large-image-data">
<h2>A Perspective on FAIR and Scalable Access to Large Image Data<a class="headerlink" href="#a-perspective-on-fair-and-scalable-access-to-large-image-data" title="Link to this heading">#</a></h2>
<p>Julia Thönnißen, Sarah Oliveira, Alexander Oberstrass, Jan-Oliver Kropp, Xiao Gui, Christian Schiffer, Timo Dickscheid</p>
<p>Published 2025-08-04</p>
<p>Licensed CC-BY-4.0</p>
<p>The rapid development of new imaging technologies across scientific domains–especially high-throughput technologies–results in a growing volume of image datasets in the Tera- to Petabyte scale. Efficient visualization and analysis of such massive image resources is critical but remains challenging due to the sheer size of the data, its continuous growth, and the limitations of conventional software tools to address these problems. Tools for visualization, annotation and analysis of large image data are confronted with the fundamental dilemma of balancing computational efficiency and memory requirements. Many tools are unable to process large datasets due to memory constraints, requiring workarounds like downsampling. On the other hand, solutions that can handle large data efficiently often rely on specialized or even proprietary file formats, limiting interoperability with other software. This reflects diverging requirements: storage favours compression for efficiency, analysis demands fast data access, and visualization requires tiled, multi-resolution representations. Lacking a unified approach for these conflicting needs, the operation of large and dynamically evolving image repositories in practice often requires undesirable data conversions and costly data duplication. In addressing these challenges, the bioimaging community increasingly adheres to the FAIR principles [1] through national and international initiatives [2], [3], [4]. For example, the Open Microscopy Environment (OME) fosters standards such as OME-TIFF [5] and its cloud-native successor OME-NGFF [6]; BioFormats [7] and OMERO [8] facilitate metadata-rich data handling across diverse platforms; and BrAinPI [9] provides web-based visualization of images via Neuroglancer [10]. These tools represent important developments towards more efficient and standardized use of bioimaging data. However, for very large and dynamically growing repositories, it is still not feasible to settle on a single standard for a subset of these tools, in particular in the light of very diverging needs for massively parallel processing on HPC systems. Therefore, converting data to a single target format is often not a practical solution. We propose a concept for a modular image delivery service which acts as a middleware between large image data resources and applications, serving image data from a cloud resource in multiple requested representations on demand. The service allows reading data stored in different input file formats, applying coordinate transformations and filtering operations on-the-fly, and serving the results in a range of different output formats and layouts. Building upon a common framework for reading and transforming data, an extensible set of access points connects the service to client applications: Lightweight REST APIs allow web-based mutli-resolution access (e.g., in common formats such as used in Neuroglancer and OpenSeadragon base viewers); mountable filesystem interfaces enable linking the repository to file-oriented solutions (e.g., OMERO, ImageJ); and programmatic access from customizable software tools (e.g., Napari). To provide compatibility with upcoming image data standards like BIDS [11] and minimize conversion efforts, the service is able to dynamically expose standard-conform views into arbitrarily organized datasets. The proposed approach for reading and transforming data on-the-fly eliminates the need for redundant storage and application-specific conversions of datasets, improving workflow efficiency and sustainability. In summary, we advocate for the development of a flexible and extensible image data service that supports large-scale analysis, dynamic transformations, multi-tool interoperability, and compatibility with community standards for large image datasets. This way it supports the FAIR principles, reduces integration barriers, meets the performance demands of modern imaging research, and still fosters the use of existing community developments.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/16736220">https://zenodo.org/records/16736220</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.16736220">https://doi.org/10.5281/zenodo.16736220</a></p>
</section>
<hr class="docutils" />
<section id="a-biologists-guide-to-planning-and-performing-quantitative-bioimaging-experiments">
<h2>A biologist’s guide to planning and performing quantitative bioimaging experiments<a class="headerlink" href="#a-biologists-guide-to-planning-and-performing-quantitative-bioimaging-experiments" title="Link to this heading">#</a></h2>
<p>Rebecca A. Senft, Barbara Diaz-Rohrer, Pina Colarusso, Lucy Swift, Nasim Jamali, Helena Jambor, Thomas Pengo, Craig Brideau, Paula Montero Llopis, Virginie Uhlmann, Jason Kirk, Kevin Andrew Gonzales, Peter Bankhead, Edward L. Evans III, Kevin W. Eliceiri, Beth A. Cimini</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Collection, Publication</p>
<p><a class="reference external" href="https://doi.org/10.1371/journal.pbio.3002167">https://doi.org/10.1371/journal.pbio.3002167</a></p>
<p><a class="reference external" href="https://www.bioimagingguide.org/">https://www.bioimagingguide.org/</a></p>
</section>
<hr class="docutils" />
<section id="a-global-view-of-standards-for-open-image-data-formats-and-repositories">
<h2>A global view of standards for open image data formats and repositories<a class="headerlink" href="#a-global-view-of-standards-for-open-image-data-formats-and-repositories" title="Link to this heading">#</a></h2>
<p>J.R. Swedlow, P. Kankaanpää, U. Sarkans, et al.</p>
<p>A comprehensive overview of existing standards for image data formats in biomedicine, including DICOM, OME-TIFF and NIfTI. It discusses standardisation challenges and provides recommendations for improving the interoperability and FAIRness of image data.</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1038/s41592-021-01113-7">https://doi.org/10.1038/s41592-021-01113-7</a></p>
</section>
<hr class="docutils" />
<section id="a-journey-to-fair-microscopy-data">
<h2>A journey to FAIR microscopy data<a class="headerlink" href="#a-journey-to-fair-microscopy-data" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters, Janina Hanne, Christian Schmidt</p>
<p>Published 2023-05-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Oral presentation, 32nd MoMAN “From Molecules to Man” Seminar, Ulm, online. Monday February 6th, 2023</p>
<p>Abstract:</p>
<p>Research data management is essential in nowadays research, and one of the big opportunities to accelerate collaborative and innovative scientific projects. To achieve this goal, all our data needs to be FAIR (findable, accessible, interoperable, reproducible). For data acquired on microscopes, however, a common ground for FAIR data sharing is still to be established. Plenty of work on file formats, data bases, and training needs to be performed to highlight the value of data sharing and exploit its potential for bioimaging data.</p>
<p>In this presentation, Stefanie Weidtkamp-Peters will introduce the challenges for bioimaging data management, and the necessary steps to achieve data FAIRification. German BioImaging - GMB e.V., together with other institutions, contributes to this endeavor. Janina Hanne will present how the network of imaging core facilities, research groups and industry partners is key to the German bioimaging community’s aligned collaboration toward FAIR bioimaging data. These activities have paved the way for two data management initiatives in Germany: I3D:bio (Information Infrastructure for BioImage Data) and NFDI4BIOIMAGE, a consortium of the National Research Data Infrastructure. Christian Schmidt will introduce the goals and measures of these initiatives to the benefit of imaging scientist’s work and everyday practice.  </p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/7890311">https://zenodo.org/records/7890311</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.7890311">https://doi.org/10.5281/zenodo.7890311</a></p>
</section>
<hr class="docutils" />
<section id="a-practical-guide-to-bioimaging-research-data-management-in-core-facilities">
<h2>A practical guide to bioimaging research data management in core facilities<a class="headerlink" href="#a-practical-guide-to-bioimaging-research-data-management-in-core-facilities" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Tom Boissonnet, Julia Dohle, Karen Bernhardt, Elisa Ferrando-May, Tobias Wernet, Roland Nitschke, Susanne Kunis, Stefanie Weidtkamp-Peters</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://onlinelibrary.wiley.com/doi/10.1111/jmi.13317">https://onlinelibrary.wiley.com/doi/10.1111/jmi.13317</a></p>
</section>
<hr class="docutils" />
<section id="ai-ml-dl-in-bioimage-analysis-webinar">
<h2>AI ML DL in Bioimage Analysis - Webinar<a class="headerlink" href="#ai-ml-dl-in-bioimage-analysis-webinar" title="Link to this heading">#</a></h2>
<p>Yannick KREMPP</p>
<p>Published 2024-11-14</p>
<p>Licensed UNKNOWN</p>
<p>A review of the tools, methods and concepts useful for biologists and life scientists as well as bioimage analysts.</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Video, Slides, Webinar</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=TJXNMIWtdac">https://www.youtube.com/watch?v=TJXNMIWtdac</a></p>
</section>
<hr class="docutils" />
<section id="accessible-interactive-spatial-omics-data-visualizations-with-vitessce-and-omero">
<h2>Accessible Interactive Spatial-Omics Data Visualizations with Vitessce and OMERO<a class="headerlink" href="#accessible-interactive-spatial-omics-data-visualizations-with-vitessce-and-omero" title="Link to this heading">#</a></h2>
<p>Bortolomeazzi Michele</p>
<p>Published 2025-06-30</p>
<p>Licensed CC-BY-4.0</p>
<p>OMERO is the most used research data management system (RDM) in the bioimaging domain, and has been adopted as a centralized RDM solution by several academic and research institutions. A main reason for this is the ability to directly view and annotate images from a web-based interface. However, this feature of OMERO is currently underpowered for the visualization of very large or multimodal datasets. These datasets, are becoming a more and more common foundation for biological and biomedical studies, due to the recent developments in imaging, and sequencing technologies which enabled their application to spatial-omics. In order to begin to provide this multimodal-data capability to OMERO, we developed omero-vitessce (<a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/omero-vitessce/tree/main">NFDI4BIOIMAGE/omero-vitessce</a>), a new <a class="reference external" href="http://OMERO.web">OMERO.web</a> plugin for viewing data stored in OMERO with the Vitessce (<a class="reference external" href="http://vitessce.io/">http://vitessce.io/</a>) multimodal data viewer. omero-vitessce can be installed as an <a class="reference external" href="http://OMERO.web">OMERO.web</a> plugin with PiPy (<a class="reference external" href="https://pypi.org/project/omero-vitessce/">https://pypi.org/project/omero-vitessce/</a>), and allows users to set up interactive visualizations of their images of cells and tissues through interactive plots which are directly linked to the image. This enables the visual exploration of bioimage-analysis results and of multimodal data such as those generated through spatial-omics experiments. The data visualization is highly customizable and can be configured not only through custom configuration files, but also with the graphical interface provided by the plugin, thus making omero-vitessce a highly user-friendly solution for multimodal data viewing. most biological datasets. We plan to extend the interoperability of omero-vitessce with the OME-NGFF and SpatialData file formats to leverage the efficiency of these cloud optimized formats.</p>
<p>Tags: Nfdi4Bioimage, OMERO, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15771899">https://zenodo.org/records/15771899</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15771899">https://doi.org/10.5281/zenodo.15771899</a></p>
</section>
<hr class="docutils" />
<section id="advanced-scripting-with-fiji">
<h2>Advanced scripting with Fiji<a class="headerlink" href="#advanced-scripting-with-fiji" title="Link to this heading">#</a></h2>
<p>Joran Deschamps, Damian Dalle Nogare</p>
<p>Published 2023-06-14T08:25:35+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Fiji, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/nobias-fht/advanced-scripting">nobias-fht/advanced-scripting</a></p>
</section>
<hr class="docutils" />
<section id="advancing-fair-image-analysis-in-galaxy-tools-workflows-and-training">
<h2>Advancing FAIR Image Analysis in Galaxy: Tools, Workflows, and Training<a class="headerlink" href="#advancing-fair-image-analysis-in-galaxy-tools-workflows-and-training" title="Link to this heading">#</a></h2>
<p>Chiang Jurado, Diana, Riccardo Massei, Pavankumar Videm, Anup Kumar, Anne Fouilloux, Leonid Kostrykin, Beatriz Serrano-Solano, Björn Grüning</p>
<p>Published 2025-03-06</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14979253">https://zenodo.org/records/14979253</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14979253">https://doi.org/10.5281/zenodo.14979253</a></p>
</section>
<hr class="docutils" />
<section id="alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten">
<h2>Alles meins – oder!? Urheberrechte klären für Forschungsdaten<a class="headerlink" href="#alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten" title="Link to this heading">#</a></h2>
<p>Stephan Wünsche</p>
<p>Published 2024-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Wem gehören Forschungsdaten? Diese Frage stellt sich bei Daten, an deren Entstehung mehrere Personen beteiligt waren, und besonders bei Textdaten, Bildern und Videos. Hier lernen Sie, für Ihr eigenes Forschungsvorhaben zu erkennen, wessen Urheber- und Leistungsschutzrechte zu berücksichtigen sind. Sie erfahren, wie Sie mit Hilfe von Vereinbarungen frühzeitig Rechtssicherheit herstellen, etwa um Daten weitergeben oder publizieren zu können.
 
 </p>
<p>Tags: Research Data Management, Licensing, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11472148">https://zenodo.org/records/11472148</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11472148">https://doi.org/10.5281/zenodo.11472148</a></p>
</section>
<hr class="docutils" />
<section id="analysis-of-high-throughput-microscopy-image-data">
<h2>Analysis of High-Throughput Microscopy Image Data<a class="headerlink" href="#analysis-of-high-throughput-microscopy-image-data" title="Link to this heading">#</a></h2>
<p>Kota Miura, Christoph Schiklenk, Clemens Lakner, Christian Tischer, Aliaksandr Halavatyi</p>
<p>Published 2014</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p><a class="github reference external" href="https://github.com/miura/HTManalysisCourse/blob/master/CentreCourseProtocol.md">miura/HTManalysisCourse</a></p>
</section>
<hr class="docutils" />
<section id="annotating-3d-images-in-napari">
<h2>Annotating 3D images in napari<a class="headerlink" href="#annotating-3d-images-in-napari" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/03/30/annotating-3d-images-in-napari/">https://focalplane.biologists.com/2023/03/30/annotating-3d-images-in-napari/</a></p>
</section>
<hr class="docutils" />
<section id="artificial-intelligence-for-digital-pathology">
<h2>Artificial Intelligence for Digital Pathology<a class="headerlink" href="#artificial-intelligence-for-digital-pathology" title="Link to this heading">#</a></h2>
<p>Jakob Nikolas Kather, Faisal Mahmood, Florian Jug</p>
<p>Published 2024-11-08</p>
<p>Licensed UNKNOWN</p>
<p>How can artificial intelligence be used for digital pathology?</p>
<p>Tags: Artificial Intelligence, Include In Dalia</p>
<p>Content type: Video</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=Om9tl4Dh2yw">https://www.youtube.com/watch?v=Om9tl4Dh2yw</a></p>
</section>
<hr class="docutils" />
<section id="basics-of-image-processing-and-analysis">
<h2>Basics of Image Processing and Analysis<a class="headerlink" href="#basics-of-image-processing-and-analysis" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/miura/ij_textbook1/raw/76b51338e1f006c580b6f0f5cfc48fe02fba38d7/CMCIBasicCourse201102Bib.pdf">miura/ij_textbook1</a></p>
</section>
<hr class="docutils" />
<section id="best-practice-data-life-cycle-approaches-for-the-life-sciences">
<h2>Best practice data life cycle approaches for the life sciences<a class="headerlink" href="#best-practice-data-life-cycle-approaches-for-the-life-sciences" title="Link to this heading">#</a></h2>
<p>Philippa C. Griffin, Jyoti Khadake, Kate S. LeMay, Suzanna E. Lewis, Sandra Orchard, et al.</p>
<p>Published 2018-06-04</p>
<p>Licensed UNKNOWN</p>
<p>The authors provide an overview of best practice data life cycle approaches for researchers in the life sciences/bioinformatics space with a particular focus on ‘omics’ datasets and computer-based data processing and analysis.</p>
<p>Tags: Bioinformatics, Reproducibility, Research Data Management, Sharing, Open Science, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.12688/f1000research.12344.2">https://doi.org/10.12688/f1000research.12344.2</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">
<h2>Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities<a class="headerlink" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities" title="Link to this heading">#</a></h2>
<p>Cornelia Wetzker</p>
<p>Published 2023-11-08</p>
<p>Licensed CC-BY-4.0</p>
<p>This presentation gives a short outline of the complexity of data and metadata in the bioimaging universe. It introduces NFDI4BIOIMAGE as a newly formed consortium as part of the German ‘Nationale Forschungsdateninfrastruktur’ (NFDI) and its goals and tools for data management including its current members on TU Dresden campus.  </p>
<p>Tags: Research Data Management, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10083555">https://zenodo.org/records/10083555</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10083555">https://doi.org/10.5281/zenodo.10083555</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-code-generation">
<h2>Bio-image Analysis Code Generation<a class="headerlink" href="#bio-image-analysis-code-generation" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-10-28</p>
<p>Licensed CC-BY-4.0</p>
<p>Large Language Models are changing the way we interact with computers, especially how we write code. In this tutorial, we will generate bio-image analysis code using two LLM-based code generators, bia-bob and git-bob.
<a class="github reference external" href="https://github.com/haesleinhuepf/bia-bob">haesleinhuepf/bia-bob</a>
<a class="github reference external" href="https://github.com/haesleinhuepf/git-bob">haesleinhuepf/git-bob</a>
 </p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14001044">https://zenodo.org/records/14001044</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14001044">https://doi.org/10.5281/zenodo.14001044</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-code-generation-using-bia-bob">
<h2>Bio-image Analysis Code Generation using bia-bob<a class="headerlink" href="#bio-image-analysis-code-generation-using-bia-bob" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-10-09</p>
<p>Licensed CC-BY-4.0</p>
<p>In this presentation I introduce bia-bob, an AI-based code generator that integrates into Jupyter Lab and allows for easy generation of Bio-Image Analysis Python code. It highlights how to get started with using large language models and prompt engineering to get high-quality bio-image analysis code.</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13908108">https://zenodo.org/records/13908108</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13908108">https://doi.org/10.5281/zenodo.13908108</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-workshop-kioto-and-taipei-23-24">
<h2>Bio-image Analysis Workshop Kioto and Taipei 23/24<a class="headerlink" href="#bio-image-analysis-workshop-kioto-and-taipei-23-24" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/Koushouu/Bioimage-Analysis-Workshop-23-24">Koushouu/Bioimage-Analysis-Workshop-23-24</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-workshop-taipei">
<h2>Bio-image Analysis Workshop Taipei<a class="headerlink" href="#bio-image-analysis-workshop-taipei" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Workshop, Collection</p>
<p><a class="github reference external" href="https://github.com/Koushouu/Bioimage-Analysis-Workshop-Taipei/">Koushouu/Bioimage-Analysis-Workshop-Taipei</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-with-the-help-of-large-language-models">
<h2>Bio-image Analysis with the Help of Large Language Models<a class="headerlink" href="#bio-image-analysis-with-the-help-of-large-language-models" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-03-13</p>
<p>Licensed CC-BY-4.0</p>
<p>Large Language Models (LLMs) change the way how we use computers. This also has impact on the bio-image analysis community. We can generate code that analyzes biomedical image data if we ask the right prompts. This talk outlines introduces basic principles, explains prompt engineering and how to apply it to bio-image analysis. We also compare how different LLM vendors perform on code generation tasks and which challenges are ahead for the bio-image analysis community.</p>
<p>Tags: Artificial Intelligence, Python, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10815329">https://zenodo.org/records/10815329</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10815329">https://doi.org/10.5281/zenodo.10815329</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-science">
<h2>Bio-image Data Science<a class="headerlink" href="#bio-image-data-science" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository contains training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python.</p>
<p>Tags: Research Data Management, Artificial Intelligence, Bioimage Analysis, Python, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/ScaDS/BIDS-lecture-2024">ScaDS/BIDS-lecture-2024</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-science-lectures-2025-uni-leipzig-scads-ai">
<h2>Bio-image Data Science Lectures 2025 &#64; Uni Leipzig / <a class="reference external" href="http://ScaDS.AI">ScaDS.AI</a><a class="headerlink" href="#bio-image-data-science-lectures-2025-uni-leipzig-scads-ai" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2025-05-29</p>
<p>Licensed CC-BY-4.0</p>
<p>These are the PPTx training resources for Students at Uni Leipzig who want to dive into bio-image data science with Python. The material will develop here and in the corresponding github repository between April and July 2025.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15546497">https://zenodo.org/records/15546497</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15546497">https://doi.org/10.5281/zenodo.15546497</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">
<h2>Bio-image analysis, biostatistics, programming and machine learning for computational biology<a class="headerlink" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology" title="Link to this heading">#</a></h2>
<p>Anna Poetsch, Biotec Dresden, Marcelo Leomil Zoccoler, Johannes Richard Müller, Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Bioimage Analysis, Napari, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/BiAPoL/Bio-image_Analysis_with_Python">BiAPoL/Bio-image_Analysis_with_Python</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-analysis-notebooks">
<h2>BioImage Analysis Notebooks<a class="headerlink" href="#bioimage-analysis-notebooks" title="Link to this heading">#</a></h2>
<p>Robert Haase et al.</p>
<p>Licensed [‘CC-BY-4.0’, ‘BSD-3-CLAUSE’]</p>
<p>Tags: Python, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Book, Notebook</p>
<p><a class="reference external" href="https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/intro.html">https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-analysis-and-superresolution-microscopy-workshop-2023-at-dartmouth-college">
<h2>BioImage Analysis and Superresolution Microscopy Workshop 2023 (at Dartmouth College)<a class="headerlink" href="#bioimage-analysis-and-superresolution-microscopy-workshop-2023-at-dartmouth-college" title="Link to this heading">#</a></h2>
<p>Zuzana Burdíková, Zdeněk Švindrych, Martin Schätz, Jakub Soukup, Pat Robison</p>
<p>Published 2023-06-07</p>
<p>Licensed CC-BY-4.0</p>
<p>The full program is available in the repository (Schedule DMW 2023.rtf) with most of the presentations and all exercises. All exercise files are in ZIP files Data.zip and ThunderSTORM sample data 2023.zip.</p>
<p>Date: June 6-7, 2023
Time: 9am - 5pm
Location: Dartmouth College, 74 College St, Hanover, Kellogg 200
Sponsor: bioMT (Lunch and coffee breaks)</p>
<p>Summary:
The BioImage Analysis and Superresolution Microscopy Workshop 2023 took place at Dartmouth College, offering participants a comprehensive learning experience in the field of bioimage analysis and superresolution microscopy. Over the course of two days, researchers, scientists, and students immersed themselves in cutting-edge microscopy techniques and expanded their knowledge and practical skills.</p>
<p>Day 1 (Tue, June 6):
The workshop began with Zdenek Svindrych providing an overview of microscopy principles, methods, and theoretical foundations. Participants gained insights into image formation in fluorescence microscopes, resolution, and noise. This was followed by an introduction to superresolution microscopy techniques, including Single-Molecule localization Microscopy (STORM, PALM, DNA-PAINT) and Structured Illumination Microscopy (SIM, ISM, Airyscan, SoRa). Zuzana Burdikova then guided attendees through the theoretical aspects of bioimage processing in Fiji, covering image formats, multi-dimensional image analysis, data visualization, and quantitative analysis. Practical sessions allowed participants to apply their knowledge, exploring two-channel colocalization, image filtering, and quantitative measurements in Fiji.</p>
<p>Day 2 (Wed, June 7):
The second day commenced with a remote presentation by <a class="reference external" href="http://SVI.nl">SVI.nl</a>, introducing participants to the Huygens deconvolution software used in widefield, confocal, and superresolution microscopy. Zdenek Svindrych demonstrated the practical applications of ThunderSTORM, an ImageJ plug-in for single molecule localization microscopy (SMLM) data analysis and superresolution imaging. The session included an overview of the ThunderSTORM workflow, covering localization, filtering, rendering, and 3D STORM using the astigmatism method. Jakub Soukup explored advanced noise reduction algorithms such as Noise2Void and StarDist. Martin Schätz discussed data management strategies.</p>
<p>In the afternoon, participants engaged in practical sessions. Martin Schätz presented Ilastik, a versatile software for image classification and segmentation. Pat Robison delivered a scientific lecture on the role of detyrosinated microtubules in contracting cardiomyocytes. Zdenek Svindrych, along with other experts, led a hands-on session on customizing Fiji using the ImageJ Macro language. The day concluded with a practical session on the interactive design of GPU-accelerated image data flow graphs in Fiji, guided by Martin Schätz and the workshop team.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/8025067">https://zenodo.org/records/8025067</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8025067">https://doi.org/10.5281/zenodo.8025067</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-data-analysis">
<h2>BioImage Data Analysis<a class="headerlink" href="#bioimage-data-analysis" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>Tags: Neubias, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://analyticalscience.wiley.com/do/10.1002/was.00050003/full/bioimagedataanalysis.pdf">https://analyticalscience.wiley.com/do/10.1002/was.00050003/full/bioimagedataanalysis.pdf</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-io-chatbot-globias-seminar">
<h2><a class="reference external" href="http://BioImage.IO">BioImage.IO</a> Chatbot, GloBIAS Seminar<a class="headerlink" href="#bioimage-io-chatbot-globias-seminar" title="Link to this heading">#</a></h2>
<p>Caterina Fuster-Barcelo</p>
<p>Published 2024-10-02</p>
<p>Licensed CC-BY-4.0</p>
<p>The dynamic field of bioimage analysis continually seeks innovative tools to democratize access to analysis tools and its documentation. The <a class="reference external" href="http://BioImage.IO">BioImage.IO</a> Chatbot, leveraging state-of-the-art AI technologies including Large Language Models (LLMs) and Retrieval Augmented Generation (RAG), provides an interactive platform that significantly integrates the exploration and application of bioimage analysis tools and models. This seminar will introduce the <a class="reference external" href="http://BioImage.IO">BioImage.IO</a> Chatbot’s capabilities, focusing on how it facilitates access to advanced analysis tools and documentation, allows for the execution of complex models, and enables users to create customized extensions adjusted to specific research needs. In a live demo, attendees will see how to interact with the chatbot and all its assistants and capabilities. Join us to explore how the <a class="reference external" href="http://BioImage.IO">BioImage.IO</a> Chatbot ca transform your research by making sophisticated analysis more intuitive and accessible.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13880367">https://zenodo.org/records/13880367</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13880367">https://doi.org/10.5281/zenodo.13880367</a></p>
</section>
<hr class="docutils" />
<section id="bioimage-analysis-with-napari">
<h2>Bioimage analysis with Napari<a class="headerlink" href="#bioimage-analysis-with-napari" title="Link to this heading">#</a></h2>
<p>Fabrizio Musacchio</p>
<p>Licensed CC-BY-ND-SA-4.0</p>
<p>Tags: Python, Napari, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://www.fabriziomusacchio.com/teaching/teaching_bioimage_analysis/">https://www.fabriziomusacchio.com/teaching/teaching_bioimage_analysis/</a></p>
</section>
<hr class="docutils" />
<section id="bioimaging-ai-textbook">
<h2>Bioimaging AI Textbook<a class="headerlink" href="#bioimaging-ai-textbook" title="Link to this heading">#</a></h2>
<p>Rachel Lee, Owen Puls, Wei Ouyang, Beth Cimini</p>
<p>Published 2025-02-20</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This is a repository containing a textbook written for BioImaging scientists and other microscopists.</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Book, Github Repository</p>
<p><a class="github reference external" href="https://github.com/aicjanelia/BioImagingAI">aicjanelia/BioImagingAI</a></p>
</section>
<hr class="docutils" />
<section id="browsing-the-open-microscopy-image-data-resource-with-python">
<h2>Browsing the Open Microscopy Image Data Resource with Python<a class="headerlink" href="#browsing-the-open-microscopy-image-data-resource-with-python" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: OMERO, Python, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://biapol.github.io/blog/robert_haase/browsing_idr/readme.html">https://biapol.github.io/blog/robert_haase/browsing_idr/readme.html</a></p>
</section>
<hr class="docutils" />
<section id="building-fair-image-analysis-pipelines-for-high-content-screening-hcs-data-using-galaxy">
<h2>Building FAIR image analysis pipelines for high-content-screening (HCS) data using Galaxy<a class="headerlink" href="#building-fair-image-analysis-pipelines-for-high-content-screening-hcs-data-using-galaxy" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Matthias Bernt, Leonid Kostrykin, Jan Bumberger</p>
<p>Published 2024-05-14</p>
<p>Licensed MIT</p>
<p>Imaging plays a crucial role across various scientific disciplines, particularly in life sciences. However, image data often proves complex, and the volume of images requiring analysis is steadily increasing, especially in high-content screening (HCS) experiments involving cell lines or other organisms. Specifically, analysis pipelines must align to the FAIR principles, ensuring they are reusable and interchangeable across different domains</p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15047849">https://zenodo.org/records/15047849</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15047849">https://doi.org/10.5281/zenodo.15047849</a></p>
</section>
<hr class="docutils" />
<section id="building-a-bioimage-analysis-workflow-using-deep-learning">
<h2>Building a Bioimage Analysis Workflow using Deep Learning<a class="headerlink" href="#building-a-bioimage-analysis-workflow-using-deep-learning" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/esgomezm/NEUBIAS_chapter_DL_2020">esgomezm/NEUBIAS_chapter_DL_2020</a></p>
</section>
<hr class="docutils" />
<section id="building-a-fair-image-data-ecosystem-for-microscopy-communities">
<h2>Building a FAIR image data ecosystem for microscopy communities<a class="headerlink" href="#building-a-fair-image-data-ecosystem-for-microscopy-communities" title="Link to this heading">#</a></h2>
<p>Isabel Kemmer, Antje Keppler, Beatriz Serrano-Solano, Arina Rybina, Bugra Özdemir, Johanna Bischof, Ayoub El Ghadraoui, John E. Eriksson, Aastha Mathur</p>
<p>Published 2023-03-31</p>
<p>Licensed CC-BY-4.0</p>
<p>Bioimaging has now entered the era of big data with faster than ever development of complex microscopy technologies leading to increasingly complex datasets. This enormous increase in data size and informational complexity within those datasets has brought with it several difficulties in terms of common and harmonized data handling, analysis and management practices, which are currently hampering the full potential of image data being realized. Here we outline a wide range of efforts and solutions currently being developed by the microscopy community to address these challenges on the path towards FAIR bioimage data. We also highlight how different actors in the microscopy ecosystem are working together, creating synergies that develop new approaches, and how research infrastructures, such as Euro-BioImaging, are fostering these interactions to shape the field. </p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/7788899">https://zenodo.org/records/7788899</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.7788899">https://doi.org/10.5281/zenodo.7788899</a></p>
</section>
<hr class="docutils" />
<section id="building-a-national-research-data-infrastructure">
<h2>Building a National Research Data Infrastructure<a class="headerlink" href="#building-a-national-research-data-infrastructure" title="Link to this heading">#</a></h2>
<p>for Microscopy and BioImage Analysis</p>
<p>Josh Moore</p>
<p>Published 2025-06-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation for the BioImagingUK Meeting taking place from 1200 - 1500 BST on Monday 30 June 2025 at mmc2025 <a class="reference external" href="https://www.mmc-series.org.uk/meetings-features/bioimaginguk-meeting.html">https://www.mmc-series.org.uk/meetings-features/bioimaginguk-meeting.html</a>
This pre-congress meeting provides an opportunity for the UK Bioimaging community to discuss priorities and strategies in national infrastructure, technology development, training, careers and ways to share knowledge across different disciplines. The session will consist of short talks from members of the BioImagingUK community and industrial/institute collaboration partners to update on progress, new opportunities and initiatives. There will be interactive Q+A sessions to encourage discussion and enable emerging priorities and ideas to be highlighted.</p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15756866">https://zenodo.org/records/15756866</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15756866">https://doi.org/10.5281/zenodo.15756866</a></p>
</section>
<hr class="docutils" />
<section id="care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022">
<h2>CARE/Stardist tutorials for EMBO Practical Course — Computational optical biology 2022<a class="headerlink" href="#care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022" title="Link to this heading">#</a></h2>
<p>Martin Weigert</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/embo_2022">maweigert/embo_2022</a></p>
</section>
<hr class="docutils" />
<section id="cs231n-convolutional-neural-networks-for-visual-recognition">
<h2>CS231n Convolutional Neural Networks for Visual Recognition<a class="headerlink" href="#cs231n-convolutional-neural-networks-for-visual-recognition" title="Link to this heading">#</a></h2>
<p>Geeth Sethi, Sumith Kulal, Kevin Zakka, William Shen, Rachel Gardner</p>
<p>Licensed UNKNOWN</p>
<p>ConvNet architectures make the explicit assumption that the inputs are images, which allows us to encode certain properties into the architecture. These then make the forward function more efficient to implement and vastly reduce the amount of parameters in the network.</p>
<p>Tags: Deep Learning, Computer Vision, Neural Networks, Machine Learning, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://cs231n.github.io/convolutional-networks/">https://cs231n.github.io/convolutional-networks/</a></p>
</section>
<hr class="docutils" />
<section id="csbdeep-and-stardist-i2k-2020">
<h2>CSBDeep and StarDist &#64; I2K 2020<a class="headerlink" href="#csbdeep-and-stardist-i2k-2020" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Uwe Schmidt</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/stardist-i2k">maweigert/stardist-i2k</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-introduction">
<h2>CellProfiler Introduction<a class="headerlink" href="#cellprofiler-introduction" title="Link to this heading">#</a></h2>
<p>Anna Klemm</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Cellprofiler, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/ahklemm/CellProfiler_Introduction">ahklemm/CellProfiler_Introduction</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-practical-neubias-lisbon-2017">
<h2>CellProfiler Practical NeuBIAS Lisbon 2017<a class="headerlink" href="#cellprofiler-practical-neubias-lisbon-2017" title="Link to this heading">#</a></h2>
<p>Carolina Wählby, Maxime Bombrun, Christian Tischer</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Cellprofiler, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/tischi/cellprofiler-practical-NeuBIAS-Lisbon-2017">tischi/cellprofiler-practical-NeuBIAS-Lisbon-2017</a></p>
</section>
<hr class="docutils" />
<section id="cellprofiler-tutorials">
<h2>CellProfiler tutorials<a class="headerlink" href="#cellprofiler-tutorials" title="Link to this heading">#</a></h2>
<p>Licensed BSD-3-CLAUSE</p>
<p>CellProfiler tutorials and guided exercises about translocation, segmentation, pixel-based classification and quality control</p>
<p>Tags: Cellprofiler, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/CellProfiler/tutorials">CellProfiler/tutorials</a></p>
</section>
<hr class="docutils" />
<section id="center-for-microscopy-and-image-analysis-how-to-guides">
<h2>Center for Microscopy and Image Analysis How-to Guides<a class="headerlink" href="#center-for-microscopy-and-image-analysis-how-to-guides" title="Link to this heading">#</a></h2>
<p>Tags: Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://zmb.dozuki.com/c/Image_Analysis">https://zmb.dozuki.com/c/Image_Analysis</a></p>
</section>
<hr class="docutils" />
<section id="challenges-and-opportunities-for-bio-image-analysis-core-facilities">
<h2>Challenges and opportunities for bio-image analysis core-facilities<a class="headerlink" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/12-1054">https://f1000research.com/slides/12-1054</a></p>
</section>
<hr class="docutils" />
<section id="challenges-and-opportunities-for-bioimage-analysis-core-facilities">
<h2>Challenges and opportunities for bioimage analysis core-facilities<a class="headerlink" href="#challenges-and-opportunities-for-bioimage-analysis-core-facilities" title="Link to this heading">#</a></h2>
<p>Johannes Richard Soltwedel, Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>This article outlines common reasons for founding bioimage analysis core-facilities, services they can provide to fulfill certain need and conflicts of interest that arise from these services.</p>
<p>Tags: Bioimage Analysis, Research Data Management, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://onlinelibrary.wiley.com/doi/full/10.1111/jmi.13192">https://onlinelibrary.wiley.com/doi/full/10.1111/jmi.13192</a></p>
</section>
<hr class="docutils" />
<section id="chatgpt-for-image-analysis">
<h2>ChatGPT for Image Analysis<a class="headerlink" href="#chatgpt-for-image-analysis" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-08-25</p>
<p>Licensed CC-BY-4.0</p>
<p>Large Language Models (LLMs) such as ChatGPT are changing the way we interact with computers, including how we analye microscopy imaging data. In this talk I introduce basic concepts of asking LLMs to write code and how to modify the questions to get the best out of it. For trying out these prompt engineering basics there are additional online resources available: <a class="reference external" href="https://scads.github.io/prompt-engineering-basics-2024/intro.html">https://scads.github.io/prompt-engineering-basics-2024/intro.html</a></p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13371196">https://zenodo.org/records/13371196</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13371196">https://doi.org/10.5281/zenodo.13371196</a></p>
</section>
<hr class="docutils" />
<section id="cloud-native-formats-enable-federated-repositories-at-peta-scale">
<h2>Cloud-Native Formats Enable Federated Repositories at Peta-Scale<a class="headerlink" href="#cloud-native-formats-enable-federated-repositories-at-peta-scale" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2025-09-27</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presentation for the abstract “Enabling Peta-Scale Federated Repositories through Cloud-Native Formats: Lessons from a fast-paced challenge in the bioimaging community” submitted to 2nd Conference on Research Data Infrastructure (CoRDI) 2025</p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/16911980">https://zenodo.org/records/16911980</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.16911980">https://doi.org/10.5281/zenodo.16911980</a></p>
</section>
<hr class="docutils" />
<section id="collaborative-working-and-version-control-with-git-hub">
<h2>Collaborative Working and Version Control with git[hub]<a class="headerlink" href="#collaborative-working-and-version-control-with-git-hub" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-01-10</p>
<p>Licensed CC-BY-4.0</p>
<p>This slide deck introduces the version control tool git, related terminology and the Github Desktop app for managing files in Git[hub] repositories. We furthermore dive into:* Working with repositories* Collaborative with others* Github-Zenodo integration* Github pages* Artificial Intelligence answering Github Issues</p>
<p>Tags: Nfdi4Bioimage, Globias, Research Data Management, Research Software Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14626054">https://zenodo.org/records/14626054</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14626054">https://doi.org/10.5281/zenodo.14626054</a></p>
</section>
<hr class="docutils" />
<section id="collaborative-bio-image-analysis-script-editing-with-git">
<h2>Collaborative bio-image analysis script editing with git<a class="headerlink" href="#collaborative-bio-image-analysis-script-editing-with-git" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to version control using git for collaborative, reproducible script editing.</p>
<p>Tags: Sharing, Research Data Management, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2021/09/04/collaborative-bio-image-analysis-script-editing-with-git/">https://focalplane.biologists.com/2021/09/04/collaborative-bio-image-analysis-script-editing-with-git/</a></p>
</section>
<hr class="docutils" />
<section id="collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis">
<h2>Collection of teaching material for deep learning for (biomedical) image analysis<a class="headerlink" href="#collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis" title="Link to this heading">#</a></h2>
<p>Constantin Pape</p>
<p>Licensed MIT</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p><a class="github reference external" href="https://github.com/constantinpape/dl-teaching-resources">constantinpape/dl-teaching-resources</a></p>
</section>
<hr class="docutils" />
<section id="combining-the-bids-and-arc-directory-structures-for-multimodal-research-data-organization">
<h2>Combining the BIDS and ARC Directory Structures for Multimodal Research Data Organization<a class="headerlink" href="#combining-the-bids-and-arc-directory-structures-for-multimodal-research-data-organization" title="Link to this heading">#</a></h2>
<p>Torsten Stöter, Tobias Gottschall, Andrea Schrager, Peter Zentis, Monica Valencia-Schneider, Niraj Kandpal, Werner Zuschratter, Astrid Schauss, Timo Dickscheid, Timo Mühlhaus, Dirk von Suchodoletz</p>
<p>Published 2023-09-12</p>
<p>Licensed CC-BY-4.0</p>
<p>Interdisciplinary collaboration and integration of large and diverse datasets are becoming increasingly important. Answering complex research questions requires combining and analysing multimodal datasets. Research data management follows the FAIR principles making data findable, accessible, interoperable, and reusable. However, there are challenges in capturing the entire research cycle and contextualizing data according, not only for the DataPLANT and NFDI4BIOIMAGE communities. To address these challenges, DataPLANT developed a data structure called Annotated Research Context (ARC). The Brain Imaging Data Structure (BIDS) originated from the neuroimaging community extended for microscopic image data. Both concepts provide standardised and file system based data storage structures for organising and sharing research data accompanied with metadata. We exemplarily compare the ARC and BIDS designs and propose structural and metadata mapping.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/records/8349563">https://zenodo.org/records/8349563</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8349563">https://doi.org/10.5281/zenodo.8349563</a></p>
</section>
<hr class="docutils" />
<section id="community-developed-checklists-for-publishing-images-and-image-analyses">
<h2>Community-developed checklists for publishing images and image analyses<a class="headerlink" href="#community-developed-checklists-for-publishing-images-and-image-analyses" title="Link to this heading">#</a></h2>
<p>Beth Cimini et al.</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This book is a companion to the Nature Methods publication Community-developed checklists for publishing images and image analyses. In this paper, members of QUAREP-LiMi have proposed 3 sets of standards for publishing image figures and image analysis - minimal requirements, recommended additions, and ideal comprehensive goals. By following this guidance, we hope to remove some of the stress non-experts may face in determining what they need to do, and we also believe that researchers will find their science more interpretable and more reproducible.</p>
<p>Tags: Bioimage Analysis, Research Data Management, Include In Dalia</p>
<p>Content type: Notebook, Collection</p>
<p><a class="reference external" href="https://quarep-limi.github.io/WG12_checklists_for_image_publishing/intro.html">https://quarep-limi.github.io/WG12_checklists_for_image_publishing/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="conda-container-and-bots-how-to-build-and-maintain-tool-dependencies-in-workflows-and-training-materials">
<h2>Conda, Container and Bots - How to Build and Maintain Tool Dependencies in Workflows and Training Materials<a class="headerlink" href="#conda-container-and-bots-how-to-build-and-maintain-tool-dependencies-in-workflows-and-training-materials" title="Link to this heading">#</a></h2>
<p>Paul Zierep, Sanjay Kumar Srikakulam, Sebastian Schaaf, Bjoern Gruening</p>
<p>Published 2023-09-07</p>
<p>Licensed CC-BY-4.0</p>
<p>The lifecycle of scientific tools comprises the creation of code releases, packages and containers which can be deployed into cloud platforms, such as the Galaxy Project, where they are run and integrated into workflows. The tools and workflows are further used to create training material that benefits a broad community. The need to organize and streamline this tool development lifecycle has led to a sophisticated development and deployment architecture.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.tib-op.org/ojs/index.php/CoRDI/article/view/417">https://www.tib-op.org/ojs/index.php/CoRDI/article/view/417</a></p>
</section>
<hr class="docutils" />
<section id="course-on-deep-learning-for-imaging-using-pytorch">
<h2>Course on Deep Learning for imaging using PyTorch<a class="headerlink" href="#course-on-deep-learning-for-imaging-using-pytorch" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Bioimage Analysis, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/DLImaging">guiwitz/DLImaging</a></p>
</section>
<hr class="docutils" />
<section id="crashkurs-forschungsdatenmanagement">
<h2>Crashkurs Forschungsdatenmanagement<a class="headerlink" href="#crashkurs-forschungsdatenmanagement" title="Link to this heading">#</a></h2>
<p>Barbara Weiner, Stephan Wünsche, Stefan Kühne, Pia Voigt, Sebastian Frericks, Clemens Hoffmann, Romy Elze, Ronny Gey</p>
<p>Published 2020-04-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Diese Präsentation bietet einen Einstieg in alle relevanten Bereiche des Forschungsdatenmanagements an der Universität Leipzig. Behandelt werden Grundlagen des Forschungsdatenmanagements, technische, ethische und rechtliche Aspekte sowie die Archivierung und Publikation von Forschungsdaten. Die Präsentation enthält zahlreiche weiterführende Links (rot) und Literaturhinweise.</p>
<p>Ergänzend hierzu wird eine Präsentation mit Übungsaufgaben angeboten, die helfen soll, das Gelernte zu festigen und in der eigenen Forschungspraxis umzusetzen. Den Aufgaben folgen jeweils eine Antwortfolie sowie deren Auflösung.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/3778431">https://zenodo.org/records/3778431</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.3778431">https://doi.org/10.5281/zenodo.3778431</a></p>
</section>
<hr class="docutils" />
<section id="creating-workflows-and-advanced-workflow-options">
<h2>Creating Workflows and Advanced Workflow Options<a class="headerlink" href="#creating-workflows-and-advanced-workflow-options" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Workflow, Include In Dalia</p>
<p>Content type: Tutorial, Online Tutorial</p>
<p><a class="reference external" href="https://galaxyproject.org/learn/advanced-workflow/">https://galaxyproject.org/learn/advanced-workflow/</a></p>
</section>
<hr class="docutils" />
<section id="creating-a-research-data-management-plan-using-chatgpt">
<h2>Creating a Research Data Management Plan using chatGPT<a class="headerlink" href="#creating-a-research-data-management-plan-using-chatgpt" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2023-11-06</p>
<p>Licensed CC-BY-4.0</p>
<p>In this blog post the author demonstrates how chatGPT can be used to combine a fictive project description with a DMP specification to produce a project-specific DMP.</p>
<p>Tags: Research Data Management, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/11/06/creating-a-research-data-management-plan-using-chatgpt/">https://focalplane.biologists.com/2023/11/06/creating-a-research-data-management-plan-using-chatgpt/</a></p>
</section>
<hr class="docutils" />
<section id="creating-open-computational-curricula">
<h2>Creating open computational curricula<a class="headerlink" href="#creating-open-computational-curricula" title="Link to this heading">#</a></h2>
<p>Kari Jordan, Zhian Kamvar, Toby Hodges</p>
<p>Published 2020-12-11</p>
<p>Licensed CC-BY-4.0</p>
<p>In this interactive session, Carpentries team members will guide attendees through three stages of the backward design process to create a lesson development plan for the open source tool of their choosing. Attendees will leave having identified what practical skills they aim to teach (learning objectives), an approach for designing challenge questions (formative assessment), and mechanisms to give and receive feedback.</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4317149">https://zenodo.org/records/4317149</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4317149">https://doi.org/10.5281/zenodo.4317149</a></p>
</section>
<hr class="docutils" />
<section id="cultivating-open-training">
<h2>Cultivating Open Training<a class="headerlink" href="#cultivating-open-training" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-03-14</p>
<p>Licensed CC-BY-4.0</p>
<p>In this SaxFDM Digital Kitchen, I introduced current challenges and potential solutions for openly sharing training materials, softly focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and uploading materials to publicly acessible repositories are proposed to fill this gap.</p>
<p>Tags: Open Science, Research Data Management, FAIR-Principles, Bioimage Analysis, Licensing, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10816895">https://zenodo.org/records/10816895</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10816895">https://doi.org/10.5281/zenodo.10816895</a></p>
</section>
<hr class="docutils" />
<section id="cultivating-open-training-to-advance-bio-image-analysis">
<h2>Cultivating Open Training to advance Bio-image Analysis<a class="headerlink" href="#cultivating-open-training-to-advance-bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-04-25</p>
<p>Licensed CC-BY-4.0</p>
<p>These slides introduce current challenges and potential solutions for openly sharing training materials, focusing on bio-image analysis. In this field a lot of training materials circulate in private channels, but openly shared, reusable materials, according to the FAIR-principles, are still rare. Using the CC-BY license and publicly acessible repositories are proposed to fill this gap.</p>
<p>Tags: Research Data Management, Licensing, FAIR-Principles, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11066250">https://zenodo.org/records/11066250</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11066250">https://doi.org/10.5281/zenodo.11066250</a></p>
</section>
<hr class="docutils" />
<section id="cytovi-deep-generative-modeling-of-antibody-based-single-cell-technologies">
<h2>CytoVI Deep generative modeling of antibody-based single cell technologies<a class="headerlink" href="#cytovi-deep-generative-modeling-of-antibody-based-single-cell-technologies" title="Link to this heading">#</a></h2>
<p>Florian Ingelfinger, Nathan Levy, Can Ergen, Artemy Bakulin, Alexander Becker, Pierre Boyeau, Martin Kim, Diana Ditz, Jan Dirks, Jonas Maaskola, Tobias Wertheimer, Robert Zeiser, Corinne C. Widmer, Ido Amit, Nir Yosef</p>
<p>Published 2025-09-12</p>
<p>Licensed CC-BY-4.0</p>
<p>CytoVI is a probabilistic generative model that enables statistically rigorous and integrative analysis of antibody-based single cell technologies, outperforming existing methods and enabling key functionalities like cell embeddings and differential protein expression testing, with applications in B cell maturation, non-Hodgkin lymphoma, and diagnostic flow cytometry.</p>
<p>Tags: Artificial Intelligence, Bioinformatics, Include In Dalia</p>
<p>Content type: Preprint</p>
<p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2025.09.07.674699v1">https://www.biorxiv.org/content/10.1101/2025.09.07.674699v1</a></p>
</section>
<hr class="docutils" />
<section id="dalia-interchange-format">
<h2>DALIA Interchange Format<a class="headerlink" href="#dalia-interchange-format" title="Link to this heading">#</a></h2>
<p>Jonathan Geiger, Petra Steiner, Abdelmoneim Amer Desouki, Frank Lange</p>
<p>Published 2024-06-07</p>
<p>Licensed CC-BY-SA-4.0</p>
<p>The DALIA (Data Literacy Alliance) project aims to develop a knowledge graph for FAIR teaching and learning materials on data literacy, data competencies and research data management (RDM) skills within the National Research Data Infrastructure (NFDI) and the RDM landscape. Such a platform thrives on the participation of users who want to search, create, manage or use teaching and learning materials.
A schematization of the metadata is necessary for the interoperability of teaching and learning materials. This is done by the DALIA Interchange Format (DIF), which provides a framework for making the metadata of teaching and learning materials transparent, comparable and smooth to integrate into the DALIA platform. It includes the description and explanation of the data fields for the online publication of educational resources.
The DIF was developed in close consultation with the scientific community. This development process included several feedback rounds in which valuable feedback was provided and subsequently incorporated into the DIF. This not only contributed to the clear, transparent and user-oriented definitions of the data fields, and to a clear structure, but also to the integration of many existing data standards and to the (special) requirements of the scientific community. The selection of elements is based on the Dublin Core Application Profile.
The DIF is provided as a PDF document and in table form (ODS) to convey the attributes of the teaching and learning materials and their definitions in an easily understandable form and to facilitate communication. It also includes a legend and an example in tabular form. In addition, a template (CSV) with the attributes as column headers is provided, which can be used for recording the metadata of the teaching and learning materials. The tables can also be transferred to technical application profiles.
We would like to thank all the commentators of the previous versions, especially Susanne Arndt, Sophie Boße, Sonja Felder, Marc Fuhrmans, Jan-Michael Haugwitz, Marina Lemaire, Karoline Lemke, Birte Lindstädt, Juliane Röder, and Jakob Voß. Without their feedback and advice, the DIF would be less transparent.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/11521029">https://zenodo.org/records/11521029</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11521029">https://doi.org/10.5281/zenodo.11521029</a></p>
</section>
<hr class="docutils" />
<section id="dl4miceverywhere-overcoming-reproducibility-challenges-in-deep-learning-microscopy-imaging">
<h2>DL4MicEverywhere – Overcoming reproducibility challenges in deep learning microscopy imaging<a class="headerlink" href="#dl4miceverywhere-overcoming-reproducibility-challenges-in-deep-learning-microscopy-imaging" title="Link to this heading">#</a></h2>
<p>Iván Hidalgo-Cenalmor</p>
<p>Published 2024-07-29</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bio Image Analysis, Artifical Intelligence, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2024/07/29/dl4miceverywhere-overcoming-reproducibility-challenges-in-deep-learning-microscopy-imaging/">https://focalplane.biologists.com/2024/07/29/dl4miceverywhere-overcoming-reproducibility-challenges-in-deep-learning-microscopy-imaging/</a></p>
</section>
<hr class="docutils" />
<section id="data-carpentry-for-biologists">
<h2>Data Carpentry for Biologists<a class="headerlink" href="#data-carpentry-for-biologists" title="Link to this heading">#</a></h2>
<p>Licensed [‘CC-BY-4.0’, ‘MIT’]</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Tutorial, Code</p>
<p><a class="reference external" href="https://datacarpentry.org/semester-biology/">https://datacarpentry.org/semester-biology/</a></p>
</section>
<hr class="docutils" />
<section id="data-visualization-with-flying-colors">
<h2>Data Visualization with Flying Colors<a class="headerlink" href="#data-visualization-with-flying-colors" title="Link to this heading">#</a></h2>
<p>Joachim Goedhart</p>
<p>Published 2019-08-29</p>
<p>Licensed UNKNOWN</p>
<p>The author discusses a number of color palettes that are suitable for coloring graphical elements in plots.</p>
<p>Tags: Data Visualization, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://thenode.biologists.com/data-visualization-with-flying-colors/research/">https://thenode.biologists.com/data-visualization-with-flying-colors/research/</a></p>
</section>
<hr class="docutils" />
<section id="data-handling-in-large-scale-electron-microscopy">
<h2>Data handling in large-scale electron microscopy<a class="headerlink" href="#data-handling-in-large-scale-electron-microscopy" title="Link to this heading">#</a></h2>
<p>Job Fermie</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://blog.delmic.com/data-handling-in-large-scale-electron-microscopy">https://blog.delmic.com/data-handling-in-large-scale-electron-microscopy</a></p>
</section>
<hr class="docutils" />
<section id="data-management-at-france-bioimaging">
<h2>Data management at France BioImaging<a class="headerlink" href="#data-management-at-france-bioimaging" title="Link to this heading">#</a></h2>
<p>Published 2024-29-05</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Research Data Management, Bioimage Analysis, Open Science, Include In Dalia</p>
<p>Content type: Slides, Presentation</p>
<p><a class="reference external" href="https://downloads.openmicroscopy.org/presentations/2024/Dundee/Day1/Research%20Data%20Management%20at%20France%20BioImaging.pdf">https://downloads.openmicroscopy.org/presentations/2024/Dundee/Day1/Research%20Data%20Management%20at%20France%20BioImaging.pdf</a></p>
</section>
<hr class="docutils" />
<section id="data-stewardship-and-research-data-management-tools-for-multimodal-linking-of-imaging-data-in-plasma-medicine">
<h2>Data stewardship and research data management tools for multimodal linking of imaging data in plasma medicine<a class="headerlink" href="#data-stewardship-and-research-data-management-tools-for-multimodal-linking-of-imaging-data-in-plasma-medicine" title="Link to this heading">#</a></h2>
<p>Mohsen Ahmadi, Robert Wagner, Philipp Mattern, Nick Plathe, Sander Bekeschus, Markus M. Becker, Torsten Stöter, Stefanie Weidtkamp-Peters</p>
<p>Published 2023-11-03</p>
<p>Licensed CC-BY-4.0</p>
<p>A more detailed understanding of the effect of plasmas on biological systems can be fostered by combining data from different imaging modalities, such as optical imaging, fluorescence imaging, and mass spectrometry imaging. This, however, requires the implementation and use of sophisticated research data management (RDM) solutions to incorporate the influence of plasma parameters and treatment procedures as well as the effects of plasma on the treated targets. In order to address this, RDM activities on different levels and from different perspectives are started and brought together within the framework of the NFDI consortium NFDI4BIOIMAGE.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/10069368">https://zenodo.org/records/10069368</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10069368">https://doi.org/10.5281/zenodo.10069368</a></p>
</section>
<hr class="docutils" />
<section id="dataplant-knowledge-base">
<h2>DataPLANT knowledge base<a class="headerlink" href="#dataplant-knowledge-base" title="Link to this heading">#</a></h2>
<p>Published 2022-12-14</p>
<p>Licensed CC-BY-4.0</p>
<p>Explore fundamental topics on research data management (RDM), how DataPLANT implements these aspects to support plant researchers with RDM tools and services, read guides and manuals or search for some teaching materials.</p>
<p>Tags: Research Data Management, Dataplant, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://nfdi4plants.org/nfdi4plants.knowledgebase/index.html">https://nfdi4plants.org/nfdi4plants.knowledgebase/index.html</a></p>
</section>
<hr class="docutils" />
<section id="dataviz-protocols-an-introduction-to-data-visualization-protocols-for-wet-lab-scientists">
<h2>DataViz protocols - An introduction to data visualization protocols for wet lab scientists<a class="headerlink" href="#dataviz-protocols-an-introduction-to-data-visualization-protocols-for-wet-lab-scientists" title="Link to this heading">#</a></h2>
<p>Joachim Goedhart</p>
<p>Published 2024-12-10</p>
<p>Licensed CC-BY-NC-SA-4.0</p>
<p>Tags: Data Visualization, R, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://zenodo.org/records/7257808">https://zenodo.org/records/7257808</a></p>
<p><a class="reference external" href="https://joachimgoedhart.github.io/DataViz-protocols/">https://joachimgoedhart.github.io/DataViz-protocols/</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.7257808">https://doi.org/10.5281/zenodo.7257808</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagement">
<h2>Datenmanagement<a class="headerlink" href="#datenmanagement" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-04-14</p>
<p>Licensed CC-BY-4.0</p>
<p>In dieser Data Management Session wird der Lebenszyklus von Daten näher beleuchtet. Wie entstehen Daten, was passiert mit ihnen, wenn sie verarbeitet werden? Wem gehören die Daten und wer ist dafür verantwortlich, sie zu veröffentlichen, zu archivieren und gegebenenfalls wiederzuverwenden? Wir werden einen Datenmanagementplan in Gruppenarbeit entwerfen, ggf. mit Hilfe von ChatGPT.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10970869">https://zenodo.org/records/10970869</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10970869">https://doi.org/10.5281/zenodo.10970869</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz">
<h2>Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz<a class="headerlink" href="#datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz" title="Link to this heading">#</a></h2>
<p>Pia Voigt, Carolin Hundt</p>
<p>Published 2024-04-19</p>
<p>Licensed CC-BY-4.0</p>
<p>Workshop zum Thema „Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz“ auf der Data Week Leipzig
Der Umgang mit Daten ist im Alltag nicht immer leicht: Wie und wo speichert man Daten idealerweise? Welche Strategien helfen, den Überblick zu behalten und wie geht man mit personenbezogenen Daten um? Diese Fragen möchten wir gemeinsam mit Ihnen anhand individueller Datenprobleme besprechen und Ihnen Lösungen aufzeigen, wie Sie ihr Datenmanagement effizient gestalten können.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11107798">https://zenodo.org/records/11107798</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11107798">https://doi.org/10.5281/zenodo.11107798</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagementplane-erstellen-teil-1">
<h2>Datenmanagementpläne erstellen - Teil 1<a class="headerlink" href="#datenmanagementplane-erstellen-teil-1" title="Link to this heading">#</a></h2>
<p>Pia Voigt, Barbara Weiner</p>
<p>Published 2021-03-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Was ist ein Datenmanagementplan? Welche Vorgaben sollte ich beachten? Wie erstelle ich einen solchen für mein Forschungsprojekt und welche nützlichen Tools kann ich hierfür verwenden?</p>
<p>Die Anforderungen der Forschungsförderer zum Datenmanagement steigen stetig. Damit verbunden ist häufig auch das Erstellen eines Datenmanagementplans. Dabei erwarten DFG, BMBF oder die EU jeweils unterschiedliche Angaben zur Erhebung, Speicherung und Veröffentlichung von projektbezogenen Forschungsdaten. Zudem bietet das Erstellen eines Datenmanagementplans viele Vorteile und hilft Ihnen nicht zuletzt, die Anforderungen der guten wissenschaftlichen Praxis strukturiert umzusetzen.</p>
<p>Was im ersten Moment unübersichtlich und überfordernd wirkt, soll in diesem Kurs anhand einer grundlegenden theoretischen Einführung im ersten und praxisorientierter Beispiele im zweiten Teil der Veranstaltung handhabbar gemacht werden. Sie lernen, was hinter den Anforderungen der Forschungsförderer steckt, welche Elemente ein Datenmanagementplan enthalten sollte und wie sie einen solchen mithilfe interaktiver Tools selbst erstellen können.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4630788">https://zenodo.org/records/4630788</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4630788">https://doi.org/10.5281/zenodo.4630788</a></p>
</section>
<hr class="docutils" />
<section id="datenmanagementplane-erstellen-teil-2">
<h2>Datenmanagementpläne erstellen - Teil 2<a class="headerlink" href="#datenmanagementplane-erstellen-teil-2" title="Link to this heading">#</a></h2>
<p>Pia Voigt, Barbara Weiner</p>
<p>Published 2021-03-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Was ist ein Datenmanagementplan? Welche Vorgaben sollte ich beachten? Wie erstelle ich einen solchen für mein Forschungsprojekt und welche nützlichen Tools kann ich hierfür verwenden?</p>
<p>Die Anforderungen der Forschungsförderer zum Datenmanagement steigen stetig. Damit verbunden ist häufig auch das Erstellen eines Datenmanagementplans. Dabei erwarten DFG, BMBF oder die EU jeweils unterschiedliche Angaben zur Erhebung, Speicherung und Veröffentlichung von projektbezogenen Forschungsdaten. Zudem bietet das Erstellen eines Datenmanagementplans viele Vorteile und hilft Ihnen nicht zuletzt, die Anforderungen der guten wissenschaftlichen Praxis strukturiert umzusetzen.</p>
<p>Was im ersten Moment unübersichtlich und überfordernd wirkt, soll in diesem Kurs anhand einer grundlegenden theoretischen Einführung im ersten und praxisorientierter Beispiele im zweiten Teil der Veranstaltung handhabbar gemacht werden. Sie lernen, was hinter den Anforderungen der Forschungsförderer steckt, welche Elemente ein Datenmanagementplan enthalten sollte und wie sie einen solchen mithilfe interaktiver Tools selbst erstellen können.</p>
<p>Version 2 enthält aktuelle Links und weiterführende Hinweise zu einzelnen Aspekten eines Datenmanagementplans.</p>
<p>Version 3 ist die überarbeitete und aktualisierte Version der ersten beiden und enthält u.a. Hinweise zur Lizenzierung und zu Nutzungsrechten an Forschungsdaten.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4748534">https://zenodo.org/records/4748534</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4748534">https://doi.org/10.5281/zenodo.4748534</a></p>
</section>
<hr class="docutils" />
<section id="deep-learning-based-segmentation-for-biologists">
<h2>Deep Learning Based Segmentation For Biologists<a class="headerlink" href="#deep-learning-based-segmentation-for-biologists" title="Link to this heading">#</a></h2>
<p>Licensed AGPL-3.0</p>
<p>Tags: Python, R, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/tpecot/DeepLearningBasedSegmentationForBiologists/">tpecot/DeepLearningBasedSegmentationForBiologists</a></p>
</section>
<hr class="docutils" />
<section id="deep-learning-for-image-analysis-exercises">
<h2>Deep Learning for image analysis - Exercises<a class="headerlink" href="#deep-learning-for-image-analysis-exercises" title="Link to this heading">#</a></h2>
<p>Martin Weigert</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Fiji, Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/zidas_2020_DL_intro_Part_2">maweigert/zidas_2020_DL_intro_Part_2</a></p>
</section>
<hr class="docutils" />
<section id="deep-vision-and-graphics">
<h2>Deep Vision and Graphics<a class="headerlink" href="#deep-vision-and-graphics" title="Link to this heading">#</a></h2>
<p>Victor Yurchenko, Fedor Ratnikov, Viktoriia Checkalina</p>
<p>Licensed MIT</p>
<p>Tags: Python, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/yandexdataschool/deep_vision_and_graphics">yandexdataschool/deep_vision_and_graphics</a></p>
</section>
<hr class="docutils" />
<section id="deepprofiler-handbook">
<h2>DeepProfiler Handbook<a class="headerlink" href="#deepprofiler-handbook" title="Link to this heading">#</a></h2>
<p>Michael Bornholdt, Juan Caicedo, Yu Han, Nikita Moshkov, Rebecca Senft</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/cytomining/DeepProfiler-handbook">cytomining/DeepProfiler-handbook</a></p>
<p><a class="reference external" href="https://cytomining.github.io/DeepProfiler-handbook/docs/00-welcome.html">https://cytomining.github.io/DeepProfiler-handbook/docs/00-welcome.html</a></p>
</section>
<hr class="docutils" />
<section id="developing-a-training-strategy">
<h2>Developing a Training Strategy<a class="headerlink" href="#developing-a-training-strategy" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-11-08</p>
<p>Licensed CC-BY-4.0</p>
<p>When training people in topics such as programming, bio-image analysis or data science, it makes sense to define a training strategy with a wider perspective than just trainees needs. This slide deck gives insights into aspects to consider when defining a training strategy. It considers funder’s interests, financial aspects, metrics / goals, steps towards sustainability and opportunities for outreach and for founding future collaborations.</p>
<p>Tags: Nfdi4Bioimage, Artificial Intelligence, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14053758">https://zenodo.org/records/14053758</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14053758">https://doi.org/10.5281/zenodo.14053758</a></p>
</section>
<hr class="docutils" />
<section id="developing-open-source-software-for-bioimage-analysis-opportunities-and-challenges">
<h2>Developing open-source software for bioimage analysis: opportunities and challenges<a class="headerlink" href="#developing-open-source-software-for-bioimage-analysis-opportunities-and-challenges" title="Link to this heading">#</a></h2>
<p>Florian Levet, Anne E. Carpenter, Kevin W. Eliceiri, Anna Kreshuk, Peter Bankhead, Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>This article outlines common challenges and practices when developing open-source software for bio-image analysis.</p>
<p>Tags: Neubias, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://f1000research.com/articles/10-302">https://f1000research.com/articles/10-302</a></p>
</section>
<hr class="docutils" />
<section id="development-of-a-platform-for-advanced-optics-education-training-and-prototyping">
<h2>Development of a platform for advanced optics education, training and prototyping<a class="headerlink" href="#development-of-a-platform-for-advanced-optics-education-training-and-prototyping" title="Link to this heading">#</a></h2>
<p>Nadine Utz, Sabine Reither, Ruth Hans, Christian Feldhaus</p>
<p>Published 2023-10-05</p>
<p>Licensed CC-BY-4.0</p>
<p>In bio-medical research we often need to combine a broad range of expertise to run complex experiments and analyse and interpret their results. Also, it is desirable that all stakeholders of a project understand all parts of the experiment and analysis to draw and support the right conclusions. For imaging experiments this usually requires a basic understanding of the underlying physics. This has not necessarily been part of the professional training of all stakeholders, e.g. biologists or data scientists. Therefore an affordable platform for easily demonstrating and explaining imaging principles would be desirable.
Building up on a commercially available STEM Optics kit we developed extensions with widely available and affordable components to demonstrate advanced imaging techniques like e.g. confocal, lightsheet, OPT, spectral imaging. All models are quick and easy to build, yet demonstrate the important physical principles each imaging technique is based on.
Further use cases for this kit are training courses, demonstrations for imaging newbies when designing an experiment and outreach activities but also basic level prototyping.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/10925217">https://zenodo.org/records/10925217</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10925217">https://doi.org/10.5281/zenodo.10925217</a></p>
</section>
<hr class="docutils" />
<section id="diataxis-a-systematic-approach-to-technical-documentation-authoring">
<h2>Diátaxis - A systematic approach to technical documentation authoring.<a class="headerlink" href="#diataxis-a-systematic-approach-to-technical-documentation-authoring" title="Link to this heading">#</a></h2>
<p>Daniele Procida</p>
<p>Licensed CC-BY-SA-4.0</p>
<p>Diátaxis is a systematic framework for technical documentation that organizes content into four types—tutorials, how-to guides, technical reference, and explanations—to address distinct user needs, enhancing both user understanding and the documentation process.</p>
<p>Tags: Documentation, Include In Dalia</p>
<p>Content type: Website, Tutorial, Workflow</p>
<p><a class="reference external" href="https://www.diataxis.fr/">https://www.diataxis.fr/</a></p>
</section>
<hr class="docutils" />
<section id="docker-mastery-with-kubernetes-swarm-from-a-docker-captain">
<h2>Docker Mastery - with Kubernetes + Swarm from a Docker Captain<a class="headerlink" href="#docker-mastery-with-kubernetes-swarm-from-a-docker-captain" title="Link to this heading">#</a></h2>
<p>Bret Fisher</p>
<p>Licensed UNKNOWN</p>
<p>In this course you will learn how to use Docker, Compose and Kubernetes on your machine for better software building and testing.</p>
<p>Tags: Docker, Include In Dalia</p>
<p>Content type: Video, Tutorial, Online Course</p>
<p><a class="reference external" href="https://www.udemy.com/course/docker-mastery/?srsltid=AfmBOornR5gRqOg-4v8Nsap1z24CaPPUPxg8JzyqEGZ6MvW_dh-sf4Af&amp;amp;couponCode=ST2MT110724BNEW">https://www.udemy.com/course/docker-mastery/?srsltid=AfmBOornR5gRqOg-4v8Nsap1z24CaPPUPxg8JzyqEGZ6MvW_dh-sf4Af&amp;couponCode=ST2MT110724BNEW</a></p>
</section>
<hr class="docutils" />
<section id="dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy">
<h2>Dr Guillaume Jacquemet on studying cancer cell metastasis in the era of deep learning for microscopy<a class="headerlink" href="#dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy" title="Link to this heading">#</a></h2>
<p>Guillaume Jacquemet</p>
<p>Published 2024-10-24</p>
<p>Licensed UNKNOWN</p>
<p>Leukocyte extravasation is a critical component of the innate immune response, while circulating tumour cell extravasation is a crucial step in metastasis formation. Despite their importance, these extravasation mechanisms remain incompletely understood. In this talk, Guillaume Jacquemet presents a novel imaging framework that integrates microfluidics with high-speed, label-free imaging to study the arrest of pancreatic cancer cells (PDAC) on human endothelial layers under physiological flow conditions.</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Video, Slides</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=KTdZBgSCYJQ">https://www.youtube.com/watch?v=KTdZBgSCYJQ</a></p>
</section>
<hr class="docutils" />
<section id="edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats-update-2020">
<h2>EDAM-bioimaging: The ontology of bioimage informatics operations, topics, data, and formats (update 2020)<a class="headerlink" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats-update-2020" title="Link to this heading">#</a></h2>
<p>Matúš Kalaš, Laure Plantard, Joakim Lindblad, Martin Jones, Nataša Sladoje, Moritz A Kirschmann, Anatole Chessel, Leandro Scholz, Fabianne Rössler, Laura Nicolás Sáenz, Estibaliz Gómez de Mariscal, John Bogovic, Alexandre Dufour, Xavier Heiligenstein, Dominic Waithe, Marie-Charlotte Domart, Matthia Karreman, Raf Van de Plas, Robert Haase, David Hörl, Lassi Paavolainen, Ivana Vrhovac Madunić, Dean Karaica, Arrate Muñoz-Barrutia, Paula Sampaio, Daniel Sage, Sebastian Munck, Ofra Golani, Josh Moore, Florian Levet, Jon Ison, Alban Gaignard, Hervé Ménager, Chong Zhang, Kota Miura, Julien Colombelli, Perrine Paul-Gilloteaux</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Metadata, Include In Dalia</p>
<p>Content type: Publication, Poster</p>
<p><a class="reference external" href="https://f1000research.com/posters/9-162">https://f1000research.com/posters/9-162</a></p>
</section>
<hr class="docutils" />
<section id="embi-recommended-metadata-for-biological-images-metadata-guidelines-for-bioimaging-data">
<h2>EMBI - Recommended Metadata for Biological Images – metadata guidelines for bioimaging data<a class="headerlink" href="#embi-recommended-metadata-for-biological-images-metadata-guidelines-for-bioimaging-data" title="Link to this heading">#</a></h2>
<p>Jullia Jakiela, Laura Cooper, Katarzyna Kamieniecka, Krzysztof Poterlowicz</p>
<p>Published 2025-02-05</p>
<p>Licensed CC-BY-4.0 INTERNATIONAL</p>
<p>The objective is to organise bioimage metadata, find out what REMBI is and why it is useful, categorise what metadata belongs to each of the submodules of REMBI and gather the metadata for an example bioimage dataset.</p>
<p>Tags: Bioinformatics, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="reference external" href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/bioimage-REMBI/tutorial.html">https://training.galaxyproject.org/training-material/topics/imaging/tutorials/bioimage-REMBI/tutorial.html</a></p>
</section>
<hr class="docutils" />
<section id="embl-deep-learning-course-2019-exercises-and-materials">
<h2>EMBL Deep Learning course 2019 exercises and materials<a class="headerlink" href="#embl-deep-learning-course-2019-exercises-and-materials" title="Link to this heading">#</a></h2>
<p>Valentyna Zinchenko, Pejman Rasti, Martin Weigert, Szymon Stoma</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/kreshuklab/teaching-dl-course-2019">kreshuklab/teaching-dl-course-2019</a></p>
</section>
<hr class="docutils" />
<section id="embl-deep-learning-course-2021-22-exercises-and-materials">
<h2>EMBL Deep Learning course 2021/22 exercises and materials<a class="headerlink" href="#embl-deep-learning-course-2021-22-exercises-and-materials" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Constantin Pape</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/kreshuklab/teaching-dl-course-2022">kreshuklab/teaching-dl-course-2022</a></p>
</section>
<hr class="docutils" />
<section id="embl-deep-learning-course-2023-exercises-and-materials">
<h2>EMBL Deep Learning course 2023 exercises and materials<a class="headerlink" href="#embl-deep-learning-course-2023-exercises-and-materials" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Uwe Schmidt, Benjamin Gallusser, Albert Dominguez Mantes, Buglakova Alyona</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/kreshuklab/teaching-dl-course-2023">kreshuklab/teaching-dl-course-2023</a></p>
</section>
<hr class="docutils" />
<section id="efficiently-starting-institutional-research-data-management">
<h2>Efficiently starting institutional research data management<a class="headerlink" href="#efficiently-starting-institutional-research-data-management" title="Link to this heading">#</a></h2>
<p>Katarzyna Biernacka, Katrin Cortez, Kerstin Helbig</p>
<p>Published 2019-10-15</p>
<p>Licensed CC-BY-4.0</p>
<p>Researchers are increasingly often confronted with research data management (RDM) topics during their work. Higher education institutions therefore begin to offer services for RDM at some point to give support and advice. However, many groundbreaking decisions have to be made at the very beginning of RDM services. Priorities must be set and policies formulated. Likewise, the staff must first be qualified in order to provide advice and adequately deal with the manifold problems awaiting.
The FDMentor project has therefore bundled the expertise of five German universities with different experiences and levels of RDM knowledge to jointly develop strategies, roadmaps, guidelines, and open access training material. Humboldt-Universität zu Berlin, Freie Universität Berlin, Technische Universität Berlin, University of Potsdam, and European University Viadrina Frankfurt (Oder) have worked together on common solutions that are easy to adapt. With funding of the German Federal Ministry of Education and Research, the collaborative project addressed four problem areas: strategy development, legal issues, policy development, and competence enhancement. The aim of the project outcomes is to provide other higher education institutions with the best possible support for the efficient introduction of research data management. Therefore, all project results are freely accessible under the CC-BY 4.0 international license. The early involvement of the community in the form of workshops and the collection of feedback has proven its worth: the FDMentor strategies, roadmaps, guidelines, and training materials are applied and adapted beyond the partner universities.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Document</p>
<p><a class="reference external" href="https://zenodo.org/record/3490058">https://zenodo.org/record/3490058</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.3490058">https://doi.org/10.5281/zenodo.3490058</a></p>
</section>
<hr class="docutils" />
<section id="einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten">
<h2>Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten<a class="headerlink" href="#einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten" title="Link to this heading">#</a></h2>
<p>Stephan Wünsche, Pia Voigt</p>
<p>Published 2021-05-11</p>
<p>Licensed CC-BY-4.0</p>
<p>Diese Präsentation wurde im Zuge der digitalen Veranstaltungsreihe “Einblicke ins Forschungsdatenmanagement” erstellt. Diese findet seit dem SS 2020 an der Universität Leipzig für alle Interessierten zu verschiedenen Themen des Forschungsdatenmanagements statt.</p>
<p>Dieser Teil der Reihe dreht sich um Rechtsfragen im Umgang mit Forschungsdaten und deren Bedeutung für die wissenschaftliche Praxis. Sie finden in der vorliegenden Präsentation einen Überblick über relevante Rechtsbereiche sowie Erläuterungen zum Datenschutz, Urheberrecht und den Grundsätzen der guten wissenschaftlichen Praxis mit Fokus auf deren Bedeutung im Forschungsdatenmanagement.</p>
<p>Tags: Research Data Management, Data Protection, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4748510">https://zenodo.org/records/4748510</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4748510">https://doi.org/10.5281/zenodo.4748510</a></p>
</section>
<hr class="docutils" />
<section id="elastix-tutorial">
<h2>Elastix tutorial<a class="headerlink" href="#elastix-tutorial" title="Link to this heading">#</a></h2>
<p>Marvin Albert</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tutorial material for teaching the basics of (itk-)elastix for image registration in microscopy images.</p>
<p>Tags: Image Registration, Itk, Elastix, Include In Dalia</p>
<p>Content type: Notebook, Collection</p>
<p><a class="reference external" href="https://m-albert.github.io/elastix_tutorial/intro.html">https://m-albert.github.io/elastix_tutorial/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="end-to-end-tissue-microarray-image-analysis-with-galaxy-me">
<h2>End-to-End Tissue Microarray Image Analysis with Galaxy-ME<a class="headerlink" href="#end-to-end-tissue-microarray-image-analysis-with-galaxy-me" title="Link to this heading">#</a></h2>
<p>Cameron Watson, Allison Creason</p>
<p>Published 2023-02-14</p>
<p>Licensed CC-BY-4.0</p>
<p>This tutorial will demonstrate how to use the Galaxy multiplex imaging tools to process and analyze publicly available TMA test data provided by MCMICRO (Figure 1); however, the majority of the steps in this tutorial are the same for both TMAs and WSIs and notes are made throughout the tutorial where processing of these two imaging types diverge.</p>
<p>Tags: Galaxy, Multiplex Imaging, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="reference external" href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.html#end-to-end-tissue-microarray-image-analysis-with-galaxy-me">https://training.galaxyproject.org/training-material/topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.html#end-to-end-tissue-microarray-image-analysis-with-galaxy-me</a></p>
</section>
<hr class="docutils" />
<section id="engineering-a-software-environment-for-research-data-management-of-microscopy-image-data-in-a-core-facility">
<h2>Engineering a Software Environment for Research Data Management of Microscopy Image Data in a Core Facility<a class="headerlink" href="#engineering-a-software-environment-for-research-data-management-of-microscopy-image-data-in-a-core-facility" title="Link to this heading">#</a></h2>
<p>Kunis</p>
<p>Published 2022-05-30</p>
<p>This thesis deals with concepts and solutions in the field of data management in everyday scientific life for image data from microscopy. The focus of the formulated requirements has so far been on published data, which represent only a small subset of the data generated in the scientific process. More and more, everyday research data are moving into the focus of the principles for the management of research data that were formulated early on (FAIR-principles). The adequate management of this mostly multimodal data is a real challenge in terms of its heterogeneity and scope. There is a lack of standardised and established workflows and also the software solutions available so far do not adequately reflect the special requirements of this area. However, the success of any data management process depends heavily on the degree of integration into the daily work routine. Data management must, as far as possible, fit seamlessly into this process. Microscopy data in the scientific process is embedded in pre-processing, which consists of preparatory laboratory work and the analytical evaluation of the microscopy data. In terms of volume, the image data often form the largest part of data generated within this entire research process. In this paper, we focus on concepts and techniques related to the handling and description of this image data and address the necessary basics. The aim is to improve the embedding of the existing data management solution for image data (OMERO) into the everyday scientific work. For this purpose, two independent software extensions for OMERO were implemented within the framework of this thesis: OpenLink and MDEmic. OpenLink simplifies the access to the data stored in the integrated repository in order to feed them into established workflows for further evaluations and enables not only the internal but also the external exchange of data without weakening the advantages of the data repository. The focus of the second implemented software solution, MDEmic, is on the capturing of relevant metadata for microscopy. Through the extended metadata collection, a corresponding linking of the multimodal data by means of a unique description and the corresponding semantic background is aimed at. The configurability of MDEmic is designed to address the currently very dynamic development of underlying concepts and formats. The main goal of MDEmic is to minimise the workload and to automate processes. This provides the scientist with a tool to handle this complex and extensive task of metadata acquisition for microscopic data in a simple way. With the help of the software, semantic and syntactic standardisation can take place without the scientist having to deal with the technical concepts. The generated metadata descriptions are automatically integrated into the image repository and, at the same time, can be transferred by the scientists into formats that are needed when publishing the data.</p>
<p>Tags: Nfdi4Bioimage, Research Data Managementv, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/6905931">https://zenodo.org/records/6905931</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.6905931">https://doi.org/10.5281/zenodo.6905931</a></p>
</section>
<hr class="docutils" />
<section id="erstellung-und-realisierung-einer-institutionellen-forschungsdaten-policy">
<h2>Erstellung und Realisierung einer institutionellen Forschungsdaten-Policy<a class="headerlink" href="#erstellung-und-realisierung-einer-institutionellen-forschungsdaten-policy" title="Link to this heading">#</a></h2>
<p>Uli Hahn, Kerstin Helbig, Gerald Jagusch, Jessica Rex</p>
<p>Published 2018-10-22</p>
<p>Licensed CC-BY-4.0</p>
<p>Die vorliegende Empfehlung sowie die zugehörigen Erfahrungsberichte geben einen Überblick über die verschiedenen Möglichkeiten der Gestaltung einer Forschungsdatenmanagement Policy sowie Wege zu deren Erstellung.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://bausteine-fdm.de/article/view/7945">https://bausteine-fdm.de/article/view/7945</a></p>
<p><a class="reference external" href="https://doi.org/10.17192/bfdm.2018.1.7945">https://doi.org/10.17192/bfdm.2018.1.7945</a></p>
</section>
<hr class="docutils" />
<section id="eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus">
<h2>EuBIAS course 2013: Intensity Dynamics at the Periphery of Nucleus<a class="headerlink" href="#eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus" title="Link to this heading">#</a></h2>
<p>Kota Miura</p>
<p>Published 2013</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Tutorial, Book</p>
<p><a class="github reference external" href="https://github.com/miura/BIAS_Nucleus_Segmentation/blob/master/module9.pdf">miura/BIAS_Nucleus_Segmentation</a></p>
</section>
<hr class="docutils" />
<section id="euro-bioimaging-evolve-deliverable-6-2-landscape-analysis-of-existing-training-resources-for-the-nodes">
<h2>Euro-BioImaging - EVOLVE Deliverable 6.2 - Landscape analysis of existing training resources for the Nodes<a class="headerlink" href="#euro-bioimaging-evolve-deliverable-6-2-landscape-analysis-of-existing-training-resources-for-the-nodes" title="Link to this heading">#</a></h2>
<p>Euro-BioImaging ERIC</p>
<p>Published 2025-09-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Horizon Europe funded EVOLVE Deliverable 6.2 - Landscape analysis of existing training resources for the Nodes
This version has not yet been reviewed or approved by the European Commission and is made publicly available for transparency and early community feedback.
A final, EC-approved version will be published when available.
This document presents a strategic analysis of training resources for Euro-BioImaging Nodes, assessing bothNode-organized and global opportunities. By analyzing Node-organized and externally available trainingcourses, alongside insights from recent surveys and training bursary applications, this report provides afoundation for strengthening the training framework of Euro-BioImagingDelivering high-quality imaging services relies on continuous skill development, particularly as scientificadvancements and technological innovations reshape the imaging landscape. </p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/17048377">https://zenodo.org/records/17048377</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.17048377">https://doi.org/10.5281/zenodo.17048377</a></p>
</section>
<hr class="docutils" />
<section id="euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">
<h2>Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks<a class="headerlink" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks" title="Link to this heading">#</a></h2>
<p>Isabel Kemmer, Euro-BioImaging ERIC</p>
<p>Published 2024-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Hands-on exercises on FAIR Bioimage Data from the interactive online workshop “Euro-BioImaging’s Guide to FAIR BioImage Data 2024” (<a class="reference external" href="https://www.eurobioimaging.eu/news/a-guide-to-fair-bioimage-data-2024/">https://www.eurobioimaging.eu/news/a-guide-to-fair-bioimage-data-2024/</a>).  Types of tasks included: FAIR characteristics of a real world dataset Data Management Plan (DMP) Journal Policies on FAIR data sharing Ontology search Metadata according to REMBI scheme (Image from: Sarkans, U., Chiu, W., Collinson, L. et al. REMBI: Recommended Metadata for Biological Images—enabling reuse of microscopy data in biology. Nat Methods 18, 1418–1422 (2021). <a class="reference external" href="https://doi.org/10.1038/s41592-021-01166-8">https://doi.org/10.1038/s41592-021-01166-8</a>) Matching datasets to bioimage repositories Browsing bioimage repositories</p>
<p>Tags: Bioimage Analysis, FAIR-Principles, Research Data Management, Include In Dalia</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="reference external" href="https://zenodo.org/records/11474407">https://zenodo.org/records/11474407</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11474407">https://doi.org/10.5281/zenodo.11474407</a></p>
</section>
<hr class="docutils" />
<section id="expansion-and-fluctuations-enhanced-microscopy-for-nanoscale-molecular-profiling-of-cells-and-tissues-data-processing-manual">
<h2>Expansion and fluctuations-enhanced microscopy for nanoscale molecular profiling of cells and tissues - Data processing manual<a class="headerlink" href="#expansion-and-fluctuations-enhanced-microscopy-for-nanoscale-molecular-profiling-of-cells-and-tissues-data-processing-manual" title="Link to this heading">#</a></h2>
<p>Dominik Kylies, Heil, Hannah S., Vesga, Arturo G., Del Rosario, Mario, Maria Schwerk, Malte Kuehl, Wong, Milagros N., Victor Puelles, Ricardo Henriques</p>
<p>Licensed CC-BY-4.0</p>
<p>Here we provide test datasets and a training manual for the parameter optimization with eSRRF. 
The training manual will guide users through an eSRRF paramenter optimization routine and quantiative image quality assesment with both, the ImagJ-Plugin NanoJ-eSRRF (Chapter 1) and the python implementation NanoPyx-eSRRF (Chapter 2). By showcasing the optimization routine on three differnt test dataset (Chapter 3), providing intermediate results and expected outcome, the users can eaisily learn how to find the optimal processing parameters for eSRRF processing.
Three samples are provided to showcase the eSRRF reconstruction process:</p>
<ol class="arabic simple">
<li><p>Microtubules sample: Set01_DNA-PAINT_Microtubules.tif
DNA-PAINT microscopy measurement of immunolabeled microtubules in fixed COS-7 cells, showing 0.121 localizations per frame and µm^2 (data published in Laine and Heil et al.)
108x90 pixels, 500 frames, pixel size: 160 nm </p></li>
<li><p>Kidney sample: Set02_KidneySDNephrinExM.tif
ExM of human kidney biopsies stained with nephrin (data published in Kylies et al.)
150x150 pixels, 200 frames, pixel size: 102 nm </p></li>
<li><p>Single emitters simulation: Set03_simulation_groundTruth_2p5Sigma - Fluorescence stack_Avg5.tif
Simulated individual molecules emitting placed on concentric rings with radii increasing by 220 nm steps. On each ring the molecules are separated by 57.5, 115, 173, 230, 288 and 345 nm, respectively (data published in Laine and Heil et al.)
33x33 pixels, 100 frames, pixel size: 100 nm </p></li>
<li><p>Test dataset for drift/vibration correction: Set04_ExSRRF_eSRRF_vibration_correction_practice_dataset.tif
EsM of human kidney biopsies stained with nephrin (data published in Kylies et al.)
100x100 pixels, 200 frames, pixel size: 102 nm</p></li>
<li><p>Test dataset for photobleaching: Set05_Photobleaching.tif
ExM of 120 nm Nanorulers (data published in Kylies et al.)
150x150 pixels, 75 frames, pixel size: 64 nm
 
Jupyter-Notebook: ridge_detection.ipynb
With this notebook qantitative image analyis of sturctures resolved with ExSRRF can be performed.
Such as:</p></li>
</ol>
<p>calculation of the target structure density. 
identifying areas with high inter-ridge spacing by maping the distance to the nearest ridge based on Euclidean distance transform. 
measuring the spatial uniformity of the structure of interest by examining the distribution of the local densities and the distances to the nearest ridge. </p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13897937">https://zenodo.org/records/13897937</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13897937">https://doi.org/10.5281/zenodo.13897937</a></p>
</section>
<hr class="docutils" />
<section id="explainable-ai-for-computer-vision">
<h2>Explainable AI for Computer Vision<a class="headerlink" href="#explainable-ai-for-computer-vision" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2025-03-09</p>
<p>Licensed CC-BY-4.0</p>
<p>In this slide deck we learn about the basics of Explainable Artificial Intelligence with a soft focus on Computer Vision. We take a deeper dive in one method: Gradient Class Activation Maps.
Releated exercise materials are available online: <a class="reference external" href="https://haesleinhuepf.github.io/xai/">https://haesleinhuepf.github.io/xai/</a></p>
<p>Tags: Nfdi4Bioimage, Bioimage Analysis, Artificial Intelligence, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14996127">https://zenodo.org/records/14996127</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14996127">https://doi.org/10.5281/zenodo.14996127</a></p>
</section>
<hr class="docutils" />
<section id="fair-priciples">
<h2>FAIR Priciples<a class="headerlink" href="#fair-priciples" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>In 2016, the ‘FAIR Guiding Principles for scientific data management and stewardship’ were published in Scientific Data. The authors intended to provide guidelines to improve the Findability, Accessibility, Interoperability, and Reuse of digital assets.</p>
<p>Tags: FAIR-Principles, Data Stewardship, Research Data Management, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://www.go-fair.org/fair-principles/">https://www.go-fair.org/fair-principles/</a></p>
</section>
<hr class="docutils" />
<section id="fairy-deep-learning-for-bioimage-analysis">
<h2>FAIRy deep-learning for bioImage analysis<a class="headerlink" href="#fairy-deep-learning-for-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez de Mariscal</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to FAIR deep learning. Furthermore, tools to deploy trained DL models (deepImageJ), easily train and evaluate them (ZeroCostDL4Mic and DeepBacs) ensure reproducibility (DL4MicEverywhere), and share this technology in an open-source and reproducible manner (BioImage Model Zoo) are introduced.</p>
<p>Tags: Artificial Intelligence, FAIR-Principles, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/13-147">https://f1000research.com/slides/13-147</a></p>
</section>
<hr class="docutils" />
<section id="faq-kunstliche-intelligenz-und-gute-wissenschaftliche-praxis">
<h2>FAQ Künstliche Intelligenz und gute wissenschaftliche Praxis<a class="headerlink" href="#faq-kunstliche-intelligenz-und-gute-wissenschaftliche-praxis" title="Link to this heading">#</a></h2>
<p>Katrin Frisch</p>
<p>Published 2024-11-06</p>
<p>Licensed CC-BY-4.0</p>
<p>Diese FAQ versammeln Fragen, die uns häufig im Zusammenhang mit künstlicher Intelligenz (KI) und guter wissenschaftlicher Praxis (GWP) erreichen. Die Antworten sollen bei der Orientierung in einem schnelllebigen Thema helfen, ohne dabei präskriptiv zu sein. Sie stellen keine offizielle Positionierung des Ombudsman für die Wissenschaft (OfdW) dar, sondern beschreiben den Status Quo und ordnen bereits bestehende Empfehlungen aus Sicht der GWP ein, identifizieren Lücken und verweisen auf weiterführende Literatur. Diese FAQ-Sammlung richtet sich primär an Forschende. Für die Nutzung von KI in der Lehre und in studentischen (Qualifikations-)Arbeiten sind i.d.R. universitäre KI-Richtlinien, angepasste Prüfungsordnungen und Selbstständigkeitserklärungen sowie Entscheidungen individueller Lehrpersonen maßgeblich. Daher werden eventuelle Besonderheiten von KI in der Lehre und in Prüfungsangelegenheiten in diesen FAQ nicht besprochen.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14045172">https://zenodo.org/records/14045172</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14045172">https://doi.org/10.5281/zenodo.14045172</a></p>
</section>
<hr class="docutils" />
<section id="feature-extraction-in-napari">
<h2>Feature extraction in napari<a class="headerlink" href="#feature-extraction-in-napari" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/05/03/feature-extraction-in-napari/">https://focalplane.biologists.com/2023/05/03/feature-extraction-in-napari/</a></p>
</section>
<hr class="docutils" />
<section id="file-naming-convention-worksheet">
<h2>File Naming Convention Worksheet<a class="headerlink" href="#file-naming-convention-worksheet" title="Link to this heading">#</a></h2>
<p>Kristin Briney</p>
<p>Published 2020-06-02</p>
<p>Licensed CC-BY-4.0</p>
<p>This worksheet walks researchers through the process of creating a file naming convention for a group of files. This process includes: choosing metadata, encoding and ordering the metadata, adding version information, and properly formatting the file names. Two versions of the worksheet are available: a Caltech Library branded version and a generic editable version.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Worksheet</p>
<p><a class="reference external" href="https://authors.library.caltech.edu/records/mmnpf-cez11">https://authors.library.caltech.edu/records/mmnpf-cez11</a></p>
</section>
<hr class="docutils" />
<section id="finding-and-choosing-a-data-repository">
<h2>Finding and Choosing a Data Repository<a class="headerlink" href="#finding-and-choosing-a-data-repository" title="Link to this heading">#</a></h2>
<p>Christian Schmidt</p>
<p>Published 2023-07-01</p>
<p>Licensed UNKNOWN</p>
<p>Funding agencies may demand that original source data of a publication be published, too. So the question is - where should one publish the data? And how does it get there?</p>
<p>Tags: I3Dbio, Research Data Management, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="reference external" href="https://gerbi-gmb.de/2023/06/01/finding-and-choosing-a-repository/">https://gerbi-gmb.de/2023/06/01/finding-and-choosing-a-repository/</a></p>
</section>
<hr class="docutils" />
<section id="finding-and-using-publicly-available-data">
<h2>Finding and using publicly available data<a class="headerlink" href="#finding-and-using-publicly-available-data" title="Link to this heading">#</a></h2>
<p>Anna Swan</p>
<p>Published 2024-01-01</p>
<p>Licensed CC-BY-4.0</p>
<p>Sharing knowledge and data in the life sciences allows us to learn from each other and built on what others have discovered. This collection of online courses brings together a variety of training, covering topics such as biocuration, open data, restricted access data and finding publicly available data, to help you discover and make the most of publicly available data in the life sciences.</p>
<p>Tags: Open Science, Teaching, Sharing, Include In Dalia</p>
<p>Content type: Collection, Tutorial, Video</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/online/courses/finding-using-public-data/">https://www.ebi.ac.uk/training/online/courses/finding-using-public-data/</a></p>
</section>
<hr class="docutils" />
<section id="five-great-reasons-to-share-your-research-data">
<h2>Five great reasons to share your research data<a class="headerlink" href="#five-great-reasons-to-share-your-research-data" title="Link to this heading">#</a></h2>
<p>Published 2022-03-18</p>
<p>Licensed UNKNOWN</p>
<p>Sharing your data can benefit your career in some interesting ways. In this post, read why you should be making more of your research data openly available.</p>
<p>Tags: Research Data Management, Sharing, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://web.library.uq.edu.au/blog/2022/03/five-great-reasons-share-your-research-data">https://web.library.uq.edu.au/blog/2022/03/five-great-reasons-share-your-research-data</a></p>
</section>
<hr class="docutils" />
<section id="five-ways-to-effectively-use-animation-in-data-visualization">
<h2>Five ways to effectively use animation in data visualization<a class="headerlink" href="#five-ways-to-effectively-use-animation-in-data-visualization" title="Link to this heading">#</a></h2>
<p>Robert Kosara, Allison Horst</p>
<p>Published 2024-12-05</p>
<p>Licensed UNKNOWN</p>
<p>This post explores how animation can enhance data visualizations by improving viewer understanding and engagement, while also acknowledging the risks of misuse. It presents five practical techniques, supported by examples and implementation tips using D3 or Observable Plot.</p>
<p>Tags: Data Visualization, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://observablehq.com/blog/effective-animation">https://observablehq.com/blog/effective-animation</a></p>
</section>
<hr class="docutils" />
<section id="forschungsdaten-info">
<h2><a class="reference external" href="http://Forschungsdaten.info">Forschungsdaten.info</a><a class="headerlink" href="#forschungsdaten-info" title="Link to this heading">#</a></h2>
<p>Licensed PUBLIC DOMAIN</p>
<p>Research Data Management Information Portal in German</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://forschungsdaten.info/">https://forschungsdaten.info/</a></p>
</section>
<hr class="docutils" />
<section id="forschungsdaten-org">
<h2><a class="reference external" href="http://Forschungsdaten.org">Forschungsdaten.org</a><a class="headerlink" href="#forschungsdaten-org" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Research Data Management Wiki in German</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://www.forschungsdaten.org/">https://www.forschungsdaten.org/</a></p>
</section>
<hr class="docutils" />
<section id="friendsdontletfriends">
<h2>FriendsDontLetFriends<a class="headerlink" href="#friendsdontletfriends" title="Link to this heading">#</a></h2>
<p>C. Li</p>
<p>Published 2024-03-10T15:34:07+00:00</p>
<p>Licensed MIT</p>
<p>Friends don’t let friends make certain types of data visualization - What are they and why are they bad.</p>
<p>Tags: Visualization, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/cxli233/FriendsDontLetFriends">cxli233/FriendsDontLetFriends</a></p>
</section>
<hr class="docutils" />
<section id="from-cells-to-pixels-bridging-biologists-and-image-analysts-through-a-common-language">
<h2>From Cells to Pixels: Bridging Biologists and  Image Analysts Through a Common Language<a class="headerlink" href="#from-cells-to-pixels-bridging-biologists-and-image-analysts-through-a-common-language" title="Link to this heading">#</a></h2>
<p>Elnaz Fazeli, Haase Robert, Doube Michael, Miura Kota, Legland David</p>
<p>Published 2024-08-16</p>
<p>Licensed CC-BY-4.0</p>
<p>Bioimaging has transformed our understanding of biological processes, yet extracting meaningful information from complex datasets remains a challenge, particularly for early career scientists. This paper proposes a simplified, systematic approach to bioimage analysis, focusing on categorizing commonly observed structures and shapes, and providing relevant analysis methods. Our approach includes illustrative examples and a visual flowchart, enabling researchers to define analysis objectives clearly. By understanding the diversity of bioimage structures and aligning them with appropriate analysis approaches, the framework empowers researchers to navigate bioimage datasets more efficiently. It also aims to foster a common language between researchers and analysts, thereby enhancing mutual understanding and facilitating effective communication.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13331351">https://zenodo.org/records/13331351</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13331351">https://doi.org/10.5281/zenodo.13331351</a></p>
</section>
<hr class="docutils" />
<section id="from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers">
<h2>From Paper to Pixels: Navigation through your Research Data - presentations of speakers<a class="headerlink" href="#from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers" title="Link to this heading">#</a></h2>
<p>Marcelo Zoccoler, Simon Bekemeier, Tom Boissonnet, Simon Parker, Luca Bertinetti, Marc Gentzel, Riccardo Massei, Cornelia Wetzker</p>
<p>Published 2024-06-10</p>
<p>Licensed CC-BY-4.0</p>
<p>The workshop introduced key topics of research data management (RDM) and the implementation thereof on a life science campus. Internal and external experts of RDM including scientists that apply chosen software tools presented the basic concepts and their implementation to a broad audience. 
Talks covered general aspects of data handling and sorting, naming conventions, data storage repositories and archives, licensing of material, data and code management using git, data protection particularly regarding patient data and in genome sequencing and more. Two data management concepts and exemplary tools were highlighted in particular, being electronic lab notebooks with eLabFTW and the bio-image management software OMERO. Those were chosen because of three aspects: the large benefit of these management tools for a life science campus, their free availability as open source tools with the option of contribution of required functionalities and first existing use cases on campus already supported by CMCB/PoL IT.
Two talks by Robert Haase (<a class="reference external" href="http://ScaDS.AI/">ScaDS.AI/</a> Uni Leipzig) and Robert Müller (Kontaktstelle Forschungsdaten, TU Dresden with contributions from Denise Dörfel) that opened the symposium were shared independently:
<a class="reference external" href="https://zenodo.org/records/11382341">https://zenodo.org/records/11382341</a>
<a class="reference external" href="https://zenodo.org/records/11261115">https://zenodo.org/records/11261115</a>
The workshop organization was funded by the CMCB/PoL Networking Grant and supported by the consortium NFDI4BIOIMAGE (funded by DFG grant number NFDI 46/1, project number 501864659).</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11548617">https://zenodo.org/records/11548617</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11548617">https://doi.org/10.5281/zenodo.11548617</a></p>
</section>
<hr class="docutils" />
<section id="from-bioimaging-projects-to-communities-globias-bia-seminar-series">
<h2>From bioimaging projects to communities - GloBIAS BIA Seminar Series<a class="headerlink" href="#from-bioimaging-projects-to-communities-globias-bia-seminar-series" title="Link to this heading">#</a></h2>
<p>Aastha Mathur, Euro-BioImaging ERIC</p>
<p>Published 2025-07-29</p>
<p>Licensed CC-BY-4.0</p>
<p>This presentaiton sumarises Euro-BioImaging ERIC services, focussing on their Image Data Services. It briefly presents processes and challenges in image anlaysis service provison and introduces some supporting tools. It also emphasises the roll of community initiatives and networks in providing solutions and support towards Image data management and analysis. This presentaiton was part of the GloBIAS BioImage Analysis Seminar Series.
Date of presentation: 2025-07-24</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/16573999">https://zenodo.org/records/16573999</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.16573999">https://doi.org/10.5281/zenodo.16573999</a></p>
</section>
<hr class="docutils" />
<section id="fundamentals-in-digital-image-processing">
<h2>Fundamentals in digital image processing<a class="headerlink" href="#fundamentals-in-digital-image-processing" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/Fundamentals_image_processing">guiwitz/Fundamentals_image_processing</a></p>
</section>
<hr class="docutils" />
<section id="fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends">
<h2>Fundamentals of image analysis in Python with scikit-image, napari, and friends<a class="headerlink" href="#fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/jni/halfway-to-i2k-skimage-napari">jni/halfway-to-i2k-skimage-napari</a></p>
</section>
<hr class="docutils" />
<section id="galaxy-meets-omero-overview-on-the-galaxy-omero-suite-and-vizarr-viewer">
<h2>Galaxy meets OMERO! Overview on the Galaxy OMERO-suite and Vizarr Viewer<a class="headerlink" href="#galaxy-meets-omero-overview-on-the-galaxy-omero-suite-and-vizarr-viewer" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Matthias Bernt, Beatriz Serrano-Solano, Lucille Lopez-Delisle, Jan Bumberger, Björn Grüning, Leonid Kostrykin</p>
<p>Published 2025-03-05</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14975462">https://zenodo.org/records/14975462</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14975462">https://doi.org/10.5281/zenodo.14975462</a></p>
</section>
<hr class="docutils" />
<section id="galaxy-workflows">
<h2>Galaxy workflows<a class="headerlink" href="#galaxy-workflows" title="Link to this heading">#</a></h2>
<p>Licensed CC0-1.0</p>
<p>A workflow is a chain of analysis steps. In Galaxy, we can create a workflow from an existing analysis history, or we can create one visually by adding tools to a canvas. This tutorial covers building a workflow to analyse a bacterial genome, from input FASTQ sequencing reads to assembly, annotation, and visualization.</p>
<p>Tags: Workflow, Include In Dalia</p>
<p>Content type: Online Tutorial, Tutorial</p>
<p><a class="reference external" href="https://galaxy-au-training.github.io/tutorials/modules/workflows/">https://galaxy-au-training.github.io/tutorials/modules/workflows/</a></p>
</section>
<hr class="docutils" />
<section id="generative-artificial-intelligence-for-bio-image-analysis">
<h2>Generative artificial intelligence for bio-image analysis<a class="headerlink" href="#generative-artificial-intelligence-for-bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Bioimage Analysis, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/12-971">https://f1000research.com/slides/12-971</a></p>
</section>
<hr class="docutils" />
<section id="getting-started-with-mambaforge-and-python">
<h2>Getting started with Mambaforge and Python<a class="headerlink" href="#getting-started-with-mambaforge-and-python" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Conda, Mamba, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://biapol.github.io/blog/mara_lampert/getting_started_with_mambaforge_and_python/readme.html">https://biapol.github.io/blog/mara_lampert/getting_started_with_mambaforge_and_python/readme.html</a></p>
</section>
<hr class="docutils" />
<section id="getting-started-with-python-intro-and-set-up-a-conda-environment">
<h2>Getting started with Python: intro and set-up a conda environment<a class="headerlink" href="#getting-started-with-python-intro-and-set-up-a-conda-environment" title="Link to this heading">#</a></h2>
<p>Riccardo Massei</p>
<p>Published 2024-10-09</p>
<p>Licensed CC-BY-4.0</p>
<p>YMIA python event 2024
Presentation :  “Getting started with Python: intro and set-up a conda environment with Dr. Riccardo Massei”</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13908480">https://zenodo.org/records/13908480</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13908480">https://doi.org/10.5281/zenodo.13908480</a></p>
</section>
<hr class="docutils" />
<section id="guidance-for-developing-a-research-data-management-rdm-policy">
<h2>Guidance for Developing a Research Data Management (RDM) Policy<a class="headerlink" href="#guidance-for-developing-a-research-data-management-rdm-policy" title="Link to this heading">#</a></h2>
<p>Published 2017</p>
<p>Licensed CC-BY-4.0</p>
<p>This document provides the essential elements of a Research Data Management (RDM) Policy and is part of the LEARN Toolkit containing the Model Policy for Research Data Management (RDM) at Research Institutions/Institutes.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://discovery.ucl.ac.uk/id/eprint/1546596/1/26_Learn_Guidance_137-140.pdf">https://discovery.ucl.ac.uk/id/eprint/1546596/1/26_Learn_Guidance_137-140.pdf</a></p>
<p><a class="reference external" href="https://doi.org/10.14324/000.learn.27">https://doi.org/10.14324/000.learn.27</a></p>
</section>
<hr class="docutils" />
<section id="high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero">
<h2>High throughput &amp; automated data analysis and data management workflow with Cellprofiler and OMERO<a class="headerlink" href="#high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero" title="Link to this heading">#</a></h2>
<p>Sarah Weischer, Jens Wendt, Thomas Zobel</p>
<p>Licensed CC-BY-4.0</p>
<p>In this workshop a fully integrated data analysis solutions employing OMERO and commonly applied image analysis tools (e.g., CellProfiler, Fiji) using existing python interfaces (OMERO Python language bindings, ezOmero, Cellprofiler Python API) is presented.</p>
<p>Tags: OMERO, Data Analysis, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8139353">https://zenodo.org/doi/10.5281/zenodo.8139353</a></p>
</section>
<hr class="docutils" />
<section id="high-content-screening-hcs-workflows-for-fair-image-data-management-with-omero">
<h2>High-content screening (HCS) workflows for FAIR image data management with OMERO<a class="headerlink" href="#high-content-screening-hcs-workflows-for-fair-image-data-management-with-omero" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Wibke Busch, Beatriz Serrano-Solano, Matthias Bernt, Stefan Scholz, Elena K. Nicolay, Hannes Bohring, Jan Bumberger</p>
<p>Published 2025-05-09</p>
<p>Licensed CC-BY-4.0</p>
<p>This study demonstrates the use of Workflow Management Systems (WMS) and the OMERO platform to create reusable semi-automatic workflows for managing high-content screening (HCS) bioimaging data, improving data management efficiency, reducing errors, and providing a blueprint for future HCS data management systems.</p>
<p>Tags: Bioimage Analysis, OMERO, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.nature.com/articles/s41598-025-00720-0">https://www.nature.com/articles/s41598-025-00720-0</a></p>
</section>
<hr class="docutils" />
<section id="highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists">
<h2>Highlights from the 2016-2020 NEUBIAS training schools for Bioimage Analysts: a success story and key asset for analysts and life scientists<a class="headerlink" href="#highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists" title="Link to this heading">#</a></h2>
<p>Gabriel G. Martins, Fabrice P. Cordelières, Julien Colombelli, Rocco D’Antuono, Ofra Golani, Romain Guiet, Robert Haase, Anna H. Klemm, Marion Louveaux, Perrine Paul-Gilloteaux, Jean-Yves Tinevez, Kota Miura</p>
<p>Published 2021</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis, Neubias, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://f1000research.com/articles/10-334/v1">https://f1000research.com/articles/10-334/v1</a></p>
</section>
<hr class="docutils" />
<section id="how-to-write-a-bug-report">
<h2>How to write a bug report<a class="headerlink" href="#how-to-write-a-bug-report" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Github, Python, Science Communication, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2024/04/03/how-to-write-a-bug-report/">https://focalplane.biologists.com/2024/04/03/how-to-write-a-bug-report/</a></p>
</section>
<hr class="docutils" />
<section id="i2k-2024-clesperanto-gpu-accelerated-image-processing-library">
<h2>I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library<a class="headerlink" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library" title="Link to this heading">#</a></h2>
<p>Stephane Rigaud, Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Course and material for the clEsperanto workshop presented at I2K 2024 &#64; Human Technopol (Milan, Italy). The workshop is an hands-on demo of the clesperanto project, focussing on how to use the library for users who want use GPU-acceleration for their Image Processing pipeline.</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository, Workshop, Tutorial, Notebook</p>
<p><a class="github reference external" href="https://github.com/StRigaud/clesperanto_workshop_I2K24?tab=readme-ov-file">StRigaud/clesperanto_workshop_I2K24</a></p>
</section>
<hr class="docutils" />
<section id="i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">
<h2>I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark<a class="headerlink" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark" title="Link to this heading">#</a></h2>
<p>Stephan Saalfeld, Tobias Pietzsch</p>
<p>Published None</p>
<p>Licensed APACHE-2.0</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Workshop, Notebook, Github Repository</p>
<p><a class="reference external" href="https://saalfeldlab.github.io/i2k2024-lazy-workshop/">https://saalfeldlab.github.io/i2k2024-lazy-workshop/</a></p>
<p><a class="github reference external" href="https://github.com/saalfeldlab/i2k2024-lazy-workshop">saalfeldlab/i2k2024-lazy-workshop</a></p>
</section>
<hr class="docutils" />
<section id="i2k2024-virtual-bio-image-analysis-code-generation">
<h2>I2K2024(virtual) - Bio-Image Analysis Code Generation<a class="headerlink" href="#i2k2024-virtual-bio-image-analysis-code-generation" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This repository contains training materials for the Tutorial “Bio-Image Analysis Code Generation” at the From Images To Knowledge (I2K) Conference (virtual) October 28th-30th 2024.</p>
<p>Tags: Bioimage Analysis, Notebooks, Biabob, Include In Dalia</p>
<p>Content type: Github Repository, Tutorial, Notebook</p>
<p><a class="github reference external" href="https://github.com/haesleinhuepf/i2k2024-ai-code-generation">haesleinhuepf/i2k2024-ai-code-generation</a></p>
</section>
<hr class="docutils" />
<section id="i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">
<h2>I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training<a class="headerlink" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Michele Bortolomeazzi, Tom Boissonnet, Carsten Fortmann-Grote, Julia Dohle, Peter Zentis, Niraj Kandpal, Susanne Kunis, Thomas Zobel, Stefanie Weidtkamp-Peters, Elisa Ferrando-May</p>
<p>Published 2023-11-13</p>
<p>Licensed CC-BY-4.0</p>
<p>The open-source software OME Remote Objects (OMERO) is a data management software that allows storing, organizing, and annotating bioimaging/microscopy data. OMERO has become one of the best-known systems for bioimage data management in the bioimaging community. The Information Infrastructure for BioImage Data (I3D:bio) project facilitates the uptake of OMERO into research data management (RDM) practices at universities and research institutions in Germany. Since the adoption of OMERO into researchers’ daily routines requires intensive training, a broad portfolio of training resources for OMERO is an asset. On top of using the OMERO guides curated by the Open Microscopy Environment Consortium (OME) team, imaging core facility staff at institutions where OMERO is used often prepare additional material tailored to be applicable for their own OMERO instances. Based on experience gathered in the Research Data Management for Microscopy group (RDM4mic) in Germany, and in the use cases in the I3D:bio project, we created a set of reusable, adjustable, openly available slide decks to serve as the basis for tailored training lectures, video tutorials, and self-guided instruction manuals directed at beginners in using OMERO. The material is published as an open educational resource complementing the existing resources for OMERO contributed by the community.</p>
<p>Tags: OMERO, Research Data Management, Nfdi4Bioimage, I3Dbio, Include In Dalia</p>
<p>Content type: Slides, Video</p>
<p><a class="reference external" href="https://zenodo.org/records/8323588">https://zenodo.org/records/8323588</a></p>
<p><a class="reference external" href="https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU">https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8323588">https://doi.org/10.5281/zenodo.8323588</a></p>
</section>
<hr class="docutils" />
<section id="iafig-rms-python-for-bioimage-analysis-course">
<h2>IAFIG-RMS Python for Bioimage Analysis Course<a class="headerlink" href="#iafig-rms-python-for-bioimage-analysis-course" title="Link to this heading">#</a></h2>
<p>Aurelien Barbotin, Chas Nelson, Dominic Waithe, Ola (Alexandra) Tarkowska, Mikolaj Kundegorski, Stephen Cross, Todd Fallesen</p>
<p>Licensed GPL-3.0</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/RMS-DAIM/Python-for-Bioimage-Analysis">RMS-DAIM/Python-for-Bioimage-Analysis</a></p>
</section>
<hr class="docutils" />
<section id="idc-tutorials">
<h2>IDC-Tutorials<a class="headerlink" href="#idc-tutorials" title="Link to this heading">#</a></h2>
<p>Andrey Fedorov, Daniela Schacherer, Dennis Bontempi, Bill Clifford, Vamsi Thiriveedhi, Brianna Major, wlongabaugh, dependabot[bot], Deepa Krishnaswamy</p>
<p>Published 2024-11-28T17:20:53+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Self-guided notebook tutorials to help get started with IDC</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ImagingDataCommons/IDC-Tutorials">ImagingDataCommons/IDC-Tutorials</a></p>
</section>
<hr class="docutils" />
<section id="if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work">
<h2>If you license it, it’ll be harder to steal it. Why we should license our work<a class="headerlink" href="#if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Blog post about why we should license our work and what is important when choosing a license.</p>
<p>Tags: Licensing, Research Data Management, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/05/06/if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work/">https://focalplane.biologists.com/2023/05/06/if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work/</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-training-resources">
<h2>Image Analysis Training Resources<a class="headerlink" href="#image-analysis-training-resources" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Neubias, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://neubias.github.io/training-resources/">https://neubias.github.io/training-resources/</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-in-healthcare-the-current-landscape-trends-and-collaborative-opportunities">
<h2>Image Analysis in Healthcare - The Current Landscape, Trends, and Collaborative Opportunities<a class="headerlink" href="#image-analysis-in-healthcare-the-current-landscape-trends-and-collaborative-opportunities" title="Link to this heading">#</a></h2>
<p>Martin Schätz, Pérez Koldenkova, Vadim</p>
<p>Published 2025-09-09</p>
<p>Licensed CC-BY-4.0</p>
<p>Workshop presentation from XXXIV Foro de Investigación en Salud 2025.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/17080219">https://zenodo.org/records/17080219</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.17080219">https://doi.org/10.5281/zenodo.17080219</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-using-galaxy">
<h2>Image Analysis using Galaxy<a class="headerlink" href="#image-analysis-using-galaxy" title="Link to this heading">#</a></h2>
<p>Beatriz Serrano-Solano, Anne Fouilloux, Leonid Kostrykin, Riccardo Massei</p>
<p>Published 2025-03-07</p>
<p>Licensed UNKNOWN</p>
<p>This video is a tutorial on how to use the Galaxy platform for bioimage analysis.</p>
<p>Tags: Galaxy, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Tutorial, Video</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=wAHOPT6lRV0">https://www.youtube.com/watch?v=wAHOPT6lRV0</a></p>
</section>
<hr class="docutils" />
<section id="image-processing-with-python">
<h2>Image Processing with Python<a class="headerlink" href="#image-processing-with-python" title="Link to this heading">#</a></h2>
<p>Mark Meysenburg, Toby Hodges, Dominik Kutra, Erin Becker, David Palmquist, et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>This lesson shows how to use Python and scikit-image to do basic image processing.</p>
<p>Tags: Bioimage Analysis, Python, Include In Dalia</p>
<p>Content type: Tutorial, Workflow</p>
<p><a class="reference external" href="https://datacarpentry.org/image-processing/key-points.html">https://datacarpentry.org/image-processing/key-points.html</a></p>
</section>
<hr class="docutils" />
<section id="image-repository-decision-tree-where-do-i-deposit-my-imaging-data">
<h2>Image Repository Decision Tree - Where do I deposit my imaging data<a class="headerlink" href="#image-repository-decision-tree-where-do-i-deposit-my-imaging-data" title="Link to this heading">#</a></h2>
<p>Isabel Kemmer, Feriel Romdhane, Euro-BioImaging ERIC</p>
<p>Published 2025-05-15</p>
<p>Licensed CC-BY-4.0</p>
<p>Depositing data in quality data repositories is one crucial step towards FAIR (Findable, Accessible, Interoperable, and Reusable) data. Accordingly, Euro-BioImaging strongly encourages sharing scientific imaging data in established, thematic repositories. 
To guide you in the selection of appropriate repositories, we have created an overview of available repositories for different types of image data, including their scope and requirements. This decision tree guides you through questions about your data and directs you to the correct repository, and/or provides instructions for further processing to meet the critera of the repositories. 
Three seperate trees are provided for different classes of imaging data: open bioimage data, preclinical data, and human imaging data. These versions with three trees can be used for web-view. Update: also the editable versions in powerpoint format (.pptx) are now provided. Please be aware that opening the versions with another program might lead to shifted formatting.
Update: we now also provide ready-to-print versions designed to be printed on A3 format. One page shows the open bioimaging data tree and one page combines the preclinical and human imaging data trees. Also the editable versions of these are provided.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15425770">https://zenodo.org/records/15425770</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15425770">https://doi.org/10.5281/zenodo.15425770</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends">
<h2>Image analysis and visualization in Python with scikit-image, napari, and friends<a class="headerlink" href="#image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/scipy-2023-image-analysis/tutorial">scipy-2023-image-analysis/tutorial</a></p>
</section>
<hr class="docutils" />
<section id="image-analysis-course-material">
<h2>Image analysis course material<a class="headerlink" href="#image-analysis-course-material" title="Link to this heading">#</a></h2>
<p>Christian Tischer</p>
<p>Licensed MIT</p>
<p>Training materials about image registration, big warp and elastix</p>
<p>Tags: Include In Dalia</p>
<p><a class="github reference external" href="https://github.com/tischi/image-analysis-course-material">tischi/image-analysis-course-material</a></p>
</section>
<hr class="docutils" />
<section id="image-handling-using-fiji-training-materials">
<h2>Image handling using Fiji - training materials<a class="headerlink" href="#image-handling-using-fiji-training-materials" title="Link to this heading">#</a></h2>
<p>Joanna Pylvänäinen</p>
<p>Published 2025-01-30</p>
<p>Licensed MIT</p>
<p>Description:This training package provides a guide to image processing and analysis using ImageJ/Fiji, an open-source software widely used in biological and medical imaging. The manual includes step-by-step exercises demonstrating practical workflows for measuring size distribution and signal intensity using both conventional thresholding and advanced tools like StarDist. This resource is ideal for researchers, students, and professionals looking to enhance their image analysis skills using Fiji.
Key topics include:</p>
<p>Image calibration and intensity adjustments
Channel splitting and merging
Projection techniques and scale bar addition
Segmentation and thresholding methods
Quantitative analysis of nuclei and fluorescence signal intensity</p>
<p>Publication Date: January 2025
Keywords: Fiji, ImageJ, Image Analysis, Microscopy, Segmentation, Particle Analysis, 3D Visualization, StarDist
License: MIT</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14771563">https://zenodo.org/records/14771563</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14771563">https://doi.org/10.5281/zenodo.14771563</a></p>
</section>
<hr class="docutils" />
<section id="image-processing-for-beginners">
<h2>Image processing for beginners<a class="headerlink" href="#image-processing-for-beginners" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/PyImageCourse_beginner">guiwitz/PyImageCourse_beginner</a></p>
</section>
<hr class="docutils" />
<section id="id1">
<h2>Image processing with Python<a class="headerlink" href="#id1" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed MIT</p>
<p>Series of Notebooks exposing how to do mostly basic and some advanced image processing using Python. It uses standard packages (Numpy, Maplotlib) and for the image processing parts is heavily based on the scikit-image package.</p>
<p>Tags: Python, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/Python_image_processing">guiwitz/Python_image_processing</a></p>
</section>
<hr class="docutils" />
<section id="image-based-profiling-handbook">
<h2>Image-based Profiling Handbook<a class="headerlink" href="#image-based-profiling-handbook" title="Link to this heading">#</a></h2>
<p>Beth Cimini, Tim Becker, Shantanu Singh, Gregory Way, Hamdah Abbasi, Callum Tromans-Coia</p>
<p>Licensed CC0-1.0</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/cytomining/profiling-handbook">cytomining/profiling-handbook</a></p>
<p><a class="reference external" href="https://cytomining.github.io/profiling-handbook/">https://cytomining.github.io/profiling-handbook/</a></p>
</section>
<hr class="docutils" />
<section id="imagej-macro-introduction">
<h2>ImageJ Macro Introduction<a class="headerlink" href="#imagej-macro-introduction" title="Link to this heading">#</a></h2>
<p>Anna Klemm</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Imagej Macro, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides, Code</p>
<p><a class="github reference external" href="https://github.com/ahklemm/ImageJMacro_Introduction">ahklemm/ImageJMacro_Introduction</a></p>
</section>
<hr class="docutils" />
<section id="imagej2-api-beating">
<h2>ImageJ2 API-beating<a class="headerlink" href="#imagej2-api-beating" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_imagej2_dev">https://git.mpi-cbg.de/rhaase/lecture_imagej2_dev</a></p>
</section>
<hr class="docutils" />
<section id="increasing-the-fairness-of-electron-microscopy-data-in-life-and-material-science-research">
<h2>Increasing the FAIRness of electron microscopy data in life and material science research<a class="headerlink" href="#increasing-the-fairness-of-electron-microscopy-data-in-life-and-material-science-research" title="Link to this heading">#</a></h2>
<p>Cornelia Wetzker</p>
<p>Published 2025-08-31</p>
<p>Licensed CC-BY-4.0</p>
<p>The poster introduces the consortium NFDI4BIOIMAGE and presents tools of research data management in microscopy to increase the FAIRness of data at the Microscopy Conference in Karlsruhe 2025. On site, it is presented in booth 57 for joint introduction of the national research data infrastructure (NFDI) consortia matWERK, FAIRmat and NFDI4BIOIMAGE.
C.W. is funded by the German consortium NFDI4BIOIMAGE (Deutsche Forschungsgemeinschaft, grant number NFDI 46/1, project number 501864659).</p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/17014253">https://zenodo.org/records/17014253</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.17014253">https://doi.org/10.5281/zenodo.17014253</a></p>
</section>
<hr class="docutils" />
<section id="interactive-bioimage-analysis-workflow-with-clij-eabias-2025-training-event">
<h2>Interactive Bioimage Analysis Workflow with CLIJ (&#64;EABIAS 2025 training event)<a class="headerlink" href="#interactive-bioimage-analysis-workflow-with-clij-eabias-2025-training-event" title="Link to this heading">#</a></h2>
<p>Wei-Chen Chu</p>
<p>Published 2025-03-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation file used in the EABIAS training event: EABIAS/2025-ImageJ-Micro-Image-Analysis-and-Programming_Taipei (Lesson_04)Video Recording (in Mandarin): <a class="reference external" href="https://www.youtube.com/watch?v=uheSMSENnzE">https://www.youtube.com/watch?v=uheSMSENnzE</a></p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15070246">https://zenodo.org/records/15070246</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15070246">https://doi.org/10.5281/zenodo.15070246</a></p>
</section>
<hr class="docutils" />
<section id="interactive-image-data-flow-graphs">
<h2>Interactive Image Data Flow Graphs<a class="headerlink" href="#interactive-image-data-flow-graphs" title="Link to this heading">#</a></h2>
<p>Martin Schätz</p>
<p>Published 2022-10-17</p>
<p>Licensed CC-BY-4.0</p>
<p>The slides were presented during the Macro programming with ImageJ workshop (<a class="reference external" href="https://www.16mcm.cz/programme/#workshops">https://www.16mcm.cz/programme/#workshops</a>) which was part of the 16th Multinational Congress on Microscopy. It is a collection and “reshuffle” of slides originally made by Robert Haase on topics from Image Analysis in general up to User-friendly GPU-accelerated bio-image analysis and CLIJ2.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/7215114">https://zenodo.org/records/7215114</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.7215114">https://doi.org/10.5281/zenodo.7215114</a></p>
</section>
<hr class="docutils" />
<section id="internal-alm-bioimage-analysis-workshop-2023">
<h2>Internal ALM BioImage Analysis workshop 2023<a class="headerlink" href="#internal-alm-bioimage-analysis-workshop-2023" title="Link to this heading">#</a></h2>
<p>Martin Schätz, Maria Azevedo, Paula Sampaio</p>
<p>Published 2023-11-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Internal ALM BioImage Analysis Workshop 2023OverviewThe Internal ALM BioImage Analysis Workshop 2023, organized by the Advanced Light Microscopy i3S scientific platform, was a comprehensive 2.5-day internal workshop dedicated to open-source BioImage Analysis. The event combined informative presentations with hands-on sessions, utilizing the EMBL Bioimage Analysis Desktop (BAND) Platform. The used sources for datasets and presentations are in Notes.Workshop ProgramDay 1: Foundations of BioImage AnalysisIntroduction to Research Data Management [00i3S_Data_Management_2023_MP_Slido]: An exploration of data management, naming conventions, and ethical considerations in BioImage Analysis and image manipulation. Referencing content from the NEUBIASAcademy&#64;Home Webinar: “In Defense of Image Data &amp; Analysis Integrity.”Interactive Image Data Flow Graphs with CLIJ2 in FIJI [01i3S_Interactive Image Data Flow Graphs]: A hands-on session introducing CLIJ2 in FIJI, with a focus on practical applications. Relevant datasets were explored during the presentation, and are described in notes.Day 2: Advanced Techniques in BioImage AnalysisNoise2Void Denoising with CSBdeep in FIJI [02i3S_IMCF_noise2void_EN] : Exploring the Noise2Void denoising approach using the CSBdeep FIJI plugin, with hands-on examples using data from the juglab/n2v GitHub repository.StarDist for Fluorescence Nuclei Segmentation [03i3S_schatzm_stardist_21] : An introduction to StarDist through the FIJI StarDist plugin, with hands-on experience using the BBBC004 dataset.Introduction to Napari 2023 [04i3S_Introduction to Napari 2023] : A hands-on session introducing Napari and its connection with CLIJ2. The datasets used were detailed in the presentation.Ilastik Pixel Classification and BioImage Model Zoo [05i3S_schatzm_Ilastik_woHandOn_PixObj_2023]: Hands-on exploration of Ilastik with a focus on pixel and object classification. Neural networks from the BioImage Model Zoo were also introduced.BioImage Model Zoo Possibilities [06i3S_BioImageModelZoo-Kreshuk]: An overview of the concept of BioImage Model Zoo and its potential applications.DeepImageJ Plugin with Hands-on on Stardist and Unet NNs [07i3S_deepImageJ-Gomez_de_Mariscal]: Introduction to the deepImageJ plugin, capable of utilizing models from the BioImage Model Zoo. The hands-on session focused on Stardist and Unet Neural Networks.Day 3: Practical Applications and Advanced TrainingHands-on with Ilastik Neural Networks: Utilizing BioImage Model Zoos, participants engaged in hands-on exercises using the Ilastik Multicut method from the AdvanceImageAnalysisEMBO2023 GitHub repository.Hands-on with Cellpose 2.0: Practical sessions included applying pre-trained models with the Simple Visual Cellpose Cheat Sheet. Participants also had the opportunity to train new models using original Cellpose datasets and larger data from BBBC019 and BBBC003 datasets. This workshop would not be possible without the previous excellent work of many people involved in the Network of European BioImage Analysts - NEUBIAS and without my attendance at the EMBO Practical Course Advanced methods in bioimage analysis .</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/10205578">https://zenodo.org/records/10205578</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10205578">https://doi.org/10.5281/zenodo.10205578</a></p>
</section>
<hr class="docutils" />
<section id="intro-napari-slides">
<h2>Intro napari slides<a class="headerlink" href="#intro-napari-slides" title="Link to this heading">#</a></h2>
<p>Peter Sobolewski</p>
<p>Licensed MIT</p>
<p>Introduction to napari workshop run at JAX (Spring 2024).</p>
<p>Tags: Napari, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://thejacksonlaboratory.github.io/intro-napari-slides/#/section">https://thejacksonlaboratory.github.io/intro-napari-slides/#/section</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-bioimage-analysis">
<h2>Introduction to Bioimage Analysis<a class="headerlink" href="#introduction-to-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Pete Bankhead</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Imagej, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Book, Notebook</p>
<p><a class="reference external" href="https://bioimagebook.github.io/index.html">https://bioimagebook.github.io/index.html</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-deep-learning-for-microscopy">
<h2>Introduction to Deep Learning for Microscopy<a class="headerlink" href="#introduction-to-deep-learning-for-microscopy" title="Link to this heading">#</a></h2>
<p>Costantin Pape</p>
<p>Licensed MIT</p>
<p>This course consists of lectures and exercises that teach the background of deep learning for image analysis and show applications to classification and segmentation analysis problems.</p>
<p>Tags: Artificial Intelligence, Python, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/computational-cell-analytics/dl-for-micro">computational-cell-analytics/dl-for-micro</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-high-performance-computing-for-life-scientists">
<h2>Introduction to High Performance Computing for Life Scientists<a class="headerlink" href="#introduction-to-high-performance-computing-for-life-scientists" title="Link to this heading">#</a></h2>
<p>Julien Sindt</p>
<p>Published 2021-03-22</p>
<p>Licensed CC-BY-4.0</p>
<p>This course introduces life science researchers to high-performance computing (HPC), covering essential concepts and providing hands-on experience using the UK’s ARCHER2 supercomputing service. It aims to help participants understand how HPC can benefit their research and prepare them to use it effectively for tasks like biomolecular simulation.</p>
<p>Tags: High Performance Computing, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="reference external" href="https://epcced.github.io/20210322-intro-hpc-life-scientists/">https://epcced.github.io/20210322-intro-hpc-life-scientists/</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-image-analysis">
<h2>Introduction to Image Analysis<a class="headerlink" href="#introduction-to-image-analysis" title="Link to this heading">#</a></h2>
<p>Barry, David J, Stefania Marcotti, Salgueiro Torres, Sara, Martin Jones, Alicja Skórkowska</p>
<p>Published 2025-08-19</p>
<p>Licensed CC-BY-SA-4.0</p>
<p>Presentation slides associated with the Introduction to Image Analysis workshop run at the Francis Crick Institute on 18-19th August 2025:
<a class="reference external" href="https://doi.org/10.5281/zenodo.16949737">https://doi.org/10.5281/zenodo.16949737</a></p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/16949737">https://zenodo.org/records/16949737</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.16949737">https://doi.org/10.5281/zenodo.16949737</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-image-analysis-with-fiji">
<h2>Introduction to Image Analysis with Fiji<a class="headerlink" href="#introduction-to-image-analysis-with-fiji" title="Link to this heading">#</a></h2>
<p>Robert Haase, Benoit Lombardot</p>
<p>Licensed CC-BY-NC-4.0</p>
<p>Tags: Imagej, Fiji, Bioimage Analysis, Include In Dalia</p>
<p><a class="github reference external" href="https://github.com/mpicbg-scicomp/CourseIntroToIA">mpicbg-scicomp/CourseIntroToIA</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden">
<h2>Introduction to ImageJ macro programming, Scientific Computing Facility, MPI CBG Dresden<a class="headerlink" href="#introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden" title="Link to this heading">#</a></h2>
<p>Robert Haase, Benoit Lombardot</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Imagej, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/scicomp/bioimage_team/coursematerialimageanalysis/tree/master/ImageJMacro_24h_2017-01">https://git.mpi-cbg.de/scicomp/bioimage_team/coursematerialimageanalysis/tree/master/ImageJMacro_24h_2017-01</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-omero-frankfurt-online">
<h2>Introduction to OMERO - Frankfurt - online<a class="headerlink" href="#introduction-to-omero-frankfurt-online" title="Link to this heading">#</a></h2>
<p>Michele Bortolomeazzi, Tom Boissonnet</p>
<p>Published 2025-04-05</p>
<p>Licensed CC-BY-4.0</p>
<p>These slides were presented during an online introductory session to OMERO for the UB Frankfurt.
The two-hour session consisted of a first part highlighting the benefits that image data management brings to the lab. The second part showcased image analysis workflows with a Fiji macro and a Python notebook.
 </p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15152576">https://zenodo.org/records/15152576</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15152576">https://doi.org/10.5281/zenodo.15152576</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-research-data-management-and-open-research">
<h2>Introduction to Research Data Management and Open Research<a class="headerlink" href="#introduction-to-research-data-management-and-open-research" title="Link to this heading">#</a></h2>
<p>Shanmugasundaram</p>
<p>Published 2024-05-17</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to RDM primarily for researchers. Can be seen as primer to all other materials in this catalogue.</p>
<p>Tags: Research Data Management, Open Science, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4778265">https://zenodo.org/records/4778265</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4778265">https://doi.org/10.5281/zenodo.4778265</a></p>
</section>
<hr class="docutils" />
<section id="jipipe-spring-course-jsc-2025-workshop-recordings-slides-homework-and-materials">
<h2>JIPipe Spring Course (JSC) 2025: Workshop Recordings, Slides, Homework, and Materials<a class="headerlink" href="#jipipe-spring-course-jsc-2025-workshop-recordings-slides-homework-and-materials" title="Link to this heading">#</a></h2>
<p>Ruman Gerst, Zoltán Cseresnyés, Marc Thilo Figge</p>
<p>Published 2025-05-12</p>
<p>Licensed CC-BY-4.0</p>
<p>The course gives a basic introduction into microscopy, optics, and image analysis. This is followed by interactive tutorials that explain the basics of creating fully automated image analysis workflows in JIPipe using a simple blobs analysis and intermediate-level quantification of LSFM kidney images. JIPipe-specific features including annotation-guided batch processing, organization with graph compartments, expressions and path processing, and project-wide metadata and parameters are also established. Finally, an advanced real-world pipeline is showcased with detailed guidance through the individual components that include integrations of Cellpose and TrackMate.</p>
<p>Tags: Nfdi4Bioimage, Jipipe, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Workshop, Video, Tutorial, Slides</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15373555">https://doi.org/10.5281/zenodo.15373555</a></p>
</section>
<hr class="docutils" />
<section id="kollaboratives-arbeiten-und-versionskontrolle-mit-git">
<h2>Kollaboratives Arbeiten und Versionskontrolle mit Git<a class="headerlink" href="#kollaboratives-arbeiten-und-versionskontrolle-mit-git" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-04-15</p>
<p>Licensed CC-BY-4.0</p>
<p>Gemeinsames Arbeiten im Internet stellt uns vor neue Herausforderungen: Wer hat eine Datei wann hochgeladen? Wer hat zum Inhalt beigetragen? Wie kann man Inhalte zusammenfuehren, wenn mehrere Mitarbeiter gleichzeitig Aenderungen gemacht haben? Das Versionskontrollwerkzeug git stellt eine umfassende Loesung fuer solche Fragen bereit. Die Onlineplatform <a class="reference external" href="http://github.com">github.com</a> stellt nicht nur Softwareentwicklern weltweit eine git-getriebene Platform zur Verfuegung und erlaubt ihnen effektiv zusammen zu arbeiten. In diesem Workshop lernen wir:</p>
<p>Infuerung in FAIR-Prinzipien im Softwarecontext
Arbeiten mit git: Pull-requests
Aufloesen von Merge-Konflikten
Automatisiertes Archivieren von Inhalten nach <a class="reference external" href="http://Zenodo.org">Zenodo.org</a>
Eigene Webseiten auf <a class="reference external" href="http://github.io">github.io</a> publizieren</p>
<p>Tags: Research Data Management, FAIR-Principles, Git, Zenodo, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10972692">https://zenodo.org/records/10972692</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10972692">https://doi.org/10.5281/zenodo.10972692</a></p>
</section>
<hr class="docutils" />
<section id="kreshuk-lab-s-embl-eipp-predoc-course-teaching-material">
<h2>Kreshuk Lab’s EMBL EIPP predoc course teaching material<a class="headerlink" href="#kreshuk-lab-s-embl-eipp-predoc-course-teaching-material" title="Link to this heading">#</a></h2>
<p>Adrian Wolny, Johannes Hugger, Qin Yu, Buglakova Alyona</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/kreshuklab/predoc-course">kreshuklab/predoc-course</a></p>
</section>
<hr class="docutils" />
<section id="kriterienkatalog-fur-materialien-aus-dem-themenbereich-forschungsdatenmanagement">
<h2>Kriterienkatalog für Materialien aus dem Themenbereich Forschungsdatenmanagement<a class="headerlink" href="#kriterienkatalog-fur-materialien-aus-dem-themenbereich-forschungsdatenmanagement" title="Link to this heading">#</a></h2>
<p>Linda Zollitsch, Swantje Piotrowski</p>
<p>Published 2025-01-24</p>
<p>Licensed CC-BY-4.0</p>
<p>Im Rahmen von FDM-SH Kontor – einem Projekt, das im Kontext der AG Kompetenzentwicklung von der Landesinitiative FDM-SH durchgeführt wurde - haben wir zum Ziel, eine kuratierte Materialbasis für Fortbildungen und Schulungen zu schaffen. Dies stellte uns vor die Herausforderung, festzulegen, wie die Materialien ausgewählt werden sollen.
Dieser Kriterienkatalog ist ein Versuch, erste Qualitätskriterien (insbesondere hinsichtlich der Nachnutzbarkeit und den FAIR-Prinzipien) für Materialien auf Basis von Metadaten zu erstellen. Dabei wurde das Vorgehen des Open Science Learning Gates (<a class="reference external" href="https://zenodo.org/records/12772135">https://zenodo.org/records/12772135</a>), als Vorbild genommen. Neben dem Metadatenschema der RDA (<a class="reference external" href="https://zenodo.org/records/6769695#.YrrP9-xBybQ">https://zenodo.org/records/6769695#.YrrP9-xBybQ</a>) haben wir auf das Metadatenschema der DINI/nestor UAG Schulungen/Fortbildungen (<a class="reference external" href="https://zenodo.org/records/3760398">https://zenodo.org/records/3760398</a>) sowie das DALIA Interchange Format (<a class="reference external" href="https://zenodo.org/records/11521029">https://zenodo.org/records/11521029</a>) zurückgegriffen.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14729452">https://zenodo.org/records/14729452</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14729452">https://doi.org/10.5281/zenodo.14729452</a></p>
</section>
<hr class="docutils" />
<section id="large-language-models-an-introduction-for-life-scientists">
<h2>Large Language Models: An Introduction for Life Scientists<a class="headerlink" href="#large-language-models-an-introduction-for-life-scientists" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-12-12</p>
<p>Licensed CC-BY-4.0</p>
<p>This slide deck introduces Large Language Models to an audience of life-scientists. We first dive into terminology: Different kinds of Language Models and what they can be used for. The remaining slides are optional slides to allow us to dive deeper into topics such as tools for using LLMs in Science, Quality Assurance, Techniques such as Retrieval Augmented Generation and Prompt Engineering.</p>
<p>Tags: Globias, Artificial Intelligence, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14418209">https://zenodo.org/records/14418209</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14418209">https://doi.org/10.5281/zenodo.14418209</a></p>
</section>
<hr class="docutils" />
<section id="latent-space-visualisation-pca-t-sne-umap-deep-learning-animated">
<h2>Latent Space Visualisation: PCA, t-SNE, UMAP | Deep Learning Animated<a class="headerlink" href="#latent-space-visualisation-pca-t-sne-umap-deep-learning-animated" title="Link to this heading">#</a></h2>
<p>Deepia</p>
<p>Published 2024-08-05</p>
<p>Licensed UNKNOWN</p>
<p>This video provides information about three very common methods for data dimensionality reduction: PCA, t-SNE and UMAP. These are especially useful when you want to visualise the latent space of an autoencoder.</p>
<p>Tags: Dimensionality Reduction, Include In Dalia</p>
<p>Content type: Video</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=o_cAOa5fMhE">https://www.youtube.com/watch?v=o_cAOa5fMhE</a></p>
</section>
<hr class="docutils" />
<section id="learning-and-training-bio-image-analysis-in-the-age-of-ai">
<h2>Learning and Training Bio-image Analysis in the Age of AI<a class="headerlink" href="#learning-and-training-bio-image-analysis-in-the-age-of-ai" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2025-04-07</p>
<p>Licensed CC-BY-4.0</p>
<p>The advent of large language models (LLMs) such as ChatGPT changes the way we analyse images. We ask LLMs to generate code, apply it to images and spend less time on learning implementation details. This also has impact on how we learn image analysis. While coding skills are still required, we can use LLMs to explain code, make proposals how to analyse the images and yet still decide how the analysis is done.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15165424">https://zenodo.org/records/15165424</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15165424">https://doi.org/10.5281/zenodo.15165424</a></p>
</section>
<hr class="docutils" />
<section id="lecture-applied-bioimage-analysis-2020">
<h2>Lecture Applied Bioimage Analysis 2020<a class="headerlink" href="#lecture-applied-bioimage-analysis-2020" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Slides, scripts, data and other exercise materials of the BioImage Analysis lecture at CMCB TU Dresden 2020</p>
<p>Tags: Imagej, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_applied_bioimage_analysis">https://git.mpi-cbg.de/rhaase/lecture_applied_bioimage_analysis</a></p>
</section>
<hr class="docutils" />
<section id="leitfaden-zur-digitalen-datensparsamkeit-mit-praxisbeispielen">
<h2>Leitfaden zur digitalen Datensparsamkeit (mit Praxisbeispielen)<a class="headerlink" href="#leitfaden-zur-digitalen-datensparsamkeit-mit-praxisbeispielen" title="Link to this heading">#</a></h2>
<p>Maximilian Heber, Moritz Jakob, Matthias Landwehr, Jan Leendertse, Maximilian Müller, Gabriel Schneider, Dirk von Suchodoletz, Robert Ulrich</p>
<p>Published 2024-06-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Im Zuge der stetig wachsenden Brisanz des Forschungsdatenmanagements fallen immer größere Mengen an Forschungsdaten an. Diese an sich begrüßenswerte Entwicklung führt zu technischen und organisatorischen Herausforderungen nicht nur im Bereich der Speicherung von Forschungsdaten, sondern in allen Phasen des Forschungsdatenlebenszyklus. Der vorliegende Beitrag erläutert vor diesem Hintergrund mögliche Motivationen hinter digitaler Datensparsamkeit mit Blick auf organisatorische, technische und ethische Kriterien, Datenschutz und Nachhaltigkeit. Anschließend werden vor dem Hintergrund zentraler Herausforderungen Umsetzungsvorschläge für das Vorfeld sowie den Verlauf eines Forschungsvorhabens gemacht. Zudem werden grundlegende Empfehlungen zur digitalen Datensparsamkeit ausgesprochen.
Eine kürzere Ausgabe des Leitfadens ist im Mai 2024 in der Zeitschrift o | bib erschienen: <a class="reference external" href="https://doi.org/10.5282/o-bib/6036">https://doi.org/10.5282/o-bib/6036</a>
Diese Ausgabe enthält ein zusätzliches Kapitel (4.2) mit konkreten Praxisbeispielen.
Dieser Artikel wurde ins Englische übersetzt:
Heber, M., Jakob, M., Landwehr, M., Leendertse, J., Müller, M., Schneider, G., von Suchodoletz, D., &amp; Ulrich, R. (2024). A Users’ Guide to Economical Digital Data Usage. Zenodo. <a class="reference external" href="https://doi.org/10.5281/zenodo.13752220">https://doi.org/10.5281/zenodo.13752220</a></p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/11445843">https://zenodo.org/records/11445843</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11445843">https://doi.org/10.5281/zenodo.11445843</a></p>
</section>
<hr class="docutils" />
<section id="liver-micrometastases-area-quantification-using-qupath-and-pixel-classifier">
<h2>Liver Micrometastases area quantification using QuPath and pixel classifier<a class="headerlink" href="#liver-micrometastases-area-quantification-using-qupath-and-pixel-classifier" title="Link to this heading">#</a></h2>
<p>Laia Simó-Riudalbas, Romain Guiet, Olivier Burri, Julien Duc, Didier Trono</p>
<p>Published 2022-05-06</p>
<p>Licensed CC-BY-4.0</p>
<p>Sample: Mouse (NSG) liver slices with human colorectal cancer cells metastases, stained with Hematoxylin &amp; Eosin. </p>
<p>Image Acquisition: Images were acquired on an Olympus VS120 Whole Slide Scanner, using a 20x objective (UPLSAPO, N.A. 0.75) and a color camera (Pike F505 Color) with an image pixel size of 0.345 microns.</p>
<p>Image Processing and Analysis: Obtained images were analyzed using the software QuPath [1] (version 0.3.2) using groovy scripts, making use of a pixel classifier to segment and measure cancer cell clusters.</p>
<p>Files :</p>
<p>Detailed_worflow.pdf : contains a detailed description of how pixel classifier was created</p>
<p>images_for_classifier_training.zip : contains all the vsi file obtained from the microscope and used for the training</p>
<p>project_for_classifier_training.zip : contains the QuPath project, with Training Image, annotations, classifiers and scripts for analysis</p>
<p>PythonCode.txt : code ran to transform output results from QuPath to final results</p>
<p> </p>
<p>[1] Bankhead, P. et al. QuPath: Open source software for digital pathology image analysis. Scientific Reports (2017). <a class="reference external" href="https://doi.org/10.1038/s41598-017-17204-5">https://doi.org/10.1038/s41598-017-17204-5</a></p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/6523649">https://zenodo.org/records/6523649</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.6523649">https://doi.org/10.5281/zenodo.6523649</a></p>
</section>
<hr class="docutils" />
<section id="machine-learning-deep-learning-applications-to-bioimage-analysis">
<h2>Machine Learning - Deep Learning. Applications to Bioimage Analysis<a class="headerlink" href="#machine-learning-deep-learning-applications-to-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://raw.githubusercontent.com/esgomezm/esgomezm.github.io/master/assets/pdf/SPAOM2018/MachineLearning_SPAOMworkshop_public.pdf">https://raw.githubusercontent.com/esgomezm/esgomezm.github.io/master/assets/pdf/SPAOM2018/MachineLearning_SPAOMworkshop_public.pdf</a></p>
</section>
<hr class="docutils" />
<section id="machine-learning-workflow-for-imaging-flow-cytometry-ifc">
<h2>Machine Learning Workflow for Imaging Flow Cytometry (IFC)<a class="headerlink" href="#machine-learning-workflow-for-imaging-flow-cytometry-ifc" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: Cellprofiler, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/holgerhennig/machine-learning-IFC">holgerhennig/machine-learning-IFC</a></p>
</section>
<hr class="docutils" />
<section id="making-the-most-of-bioimaging-data-through-interdisciplinary-interactions">
<h2>Making the most of bioimaging data through interdisciplinary interactions<a class="headerlink" href="#making-the-most-of-bioimaging-data-through-interdisciplinary-interactions" title="Link to this heading">#</a></h2>
<p>Virginie Uhlmann, Matthew Hartley, Josh Moore, Erin Weisbart, Assaf Zaritsky</p>
<p>Published 2024-10-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis, Open Science, Microscopy, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://journals.biologists.com/jcs/article/137/20/jcs262139/362478/Making-the-most-of-bioimaging-data-through">https://journals.biologists.com/jcs/article/137/20/jcs262139/362478/Making-the-most-of-bioimaging-data-through</a></p>
</section>
<hr class="docutils" />
<section id="making-your-package-available-on-conda-forge">
<h2>Making your package available on conda-forge<a class="headerlink" href="#making-your-package-available-on-conda-forge" title="Link to this heading">#</a></h2>
<p>Kevin Yamauchi</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Deployment, Python, Include In Dalia</p>
<p>Content type: Documentation</p>
<p><a class="reference external" href="https://kevinyamauchi.github.io/open-image-data/how_tos/conda_forge_packaging.html">https://kevinyamauchi.github.io/open-image-data/how_tos/conda_forge_packaging.html</a></p>
</section>
<hr class="docutils" />
<section id="making-your-project-citable">
<h2>Making your project citable<a class="headerlink" href="#making-your-project-citable" title="Link to this heading">#</a></h2>
<p>A tutorial explaining how to make Github repositories citable by automatically creating DOIs using the Github-Zenodo integration.</p>
<p>Tags: Sharing, Citing, Research Data Management, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="reference external" href="https://coderefinery.github.io/github-without-command-line/doi/">https://coderefinery.github.io/github-without-command-line/doi/</a></p>
</section>
<hr class="docutils" />
<section id="managing-scientific-python-environments-using-conda-mamba-and-friends">
<h2>Managing Scientific Python environments using Conda, Mamba and friends<a class="headerlink" href="#managing-scientific-python-environments-using-conda-mamba-and-friends" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Conda, Mamba, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/">https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/</a></p>
</section>
<hr class="docutils" />
<section id="meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration">
<h2>Meeting in the Middle: Towards Successful Multidisciplinary Bioimage Analysis Collaboration<a class="headerlink" href="#meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration" title="Link to this heading">#</a></h2>
<p>Anjalie Schlaeppi, Wilson Adams, Robert Haase, Jan Huisken, Ryan B. MacDonald, Kevin W. Eliceiri, Elisabeth C. Kugler</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.frontiersin.org/articles/10.3389/fbinf.2022.889755/full">https://www.frontiersin.org/articles/10.3389/fbinf.2022.889755/full</a></p>
</section>
<hr class="docutils" />
<section id="metabolic-networks-modelling-with-cobrapy">
<h2>Metabolic networks modelling with COBRApy<a class="headerlink" href="#metabolic-networks-modelling-with-cobrapy" title="Link to this heading">#</a></h2>
<p>Karin Sasaki, Aleksej Zelezniak</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/karinsasaki/metabolic-networks-modelling">karinsasaki/metabolic-networks-modelling</a></p>
</section>
<hr class="docutils" />
<section id="metadata-matters-access-to-image-data-in-the-real-world">
<h2>Metadata matters: access to image data in the real world<a class="headerlink" href="#metadata-matters-access-to-image-data-in-the-real-world" title="Link to this heading">#</a></h2>
<p>Melissa Linkert, et al.</p>
<p>Published 2010</p>
<p>Tags: Metadata, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://rupress.org/jcb/article/189/5/777/35828/Metadata-matters-access-to-image-data-in-the-real">https://rupress.org/jcb/article/189/5/777/35828/Metadata-matters-access-to-image-data-in-the-real</a></p>
</section>
<hr class="docutils" />
<section id="methods-in-bioimage-analysis">
<h2>Methods in bioimage analysis<a class="headerlink" href="#methods-in-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Christian Tischer</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Online Tutorial, Video, Slides</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/events/methods-bioimage-analysis/">https://www.ebi.ac.uk/training/events/methods-bioimage-analysis/</a></p>
<p><a class="reference external" href="https://doi.org/10.6019/TOL.BioImageAnalysis22-w.2022.00001.1">https://doi.org/10.6019/TOL.BioImageAnalysis22-w.2022.00001.1</a></p>
<p><a class="reference external" href="https://drive.google.com/file/d/1MhuqfKhZcYu3bchWMqogIybKjamU5Msg/view">https://drive.google.com/file/d/1MhuqfKhZcYu3bchWMqogIybKjamU5Msg/view</a></p>
</section>
<hr class="docutils" />
<section id="microscopy-data-analysis-machine-learning-and-the-bioimage-archive">
<h2>Microscopy data analysis: machine learning and the BioImage Archive<a class="headerlink" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive" title="Link to this heading">#</a></h2>
<p>Andrii Iudin, Anna Foix-Romero, Anna Kreshuk, Awais Athar, Beth Cimini, Dominik Kutra, Estibalis Gomez de Mariscal, Frances Wong, Guillaume Jacquemet, Kedar Narayan, Martin Weigert, Nodar Gogoberidze, Osman Salih, Petr Walczysko, Ryan Conrad, Simone Weyend, Sriram Sundar Somasundharam, Suganya Sivagurunathan, Ugis Sarkans</p>
<p>Licensed CC-BY-4.0</p>
<p>The Microscopy data analysis: machine learning and the BioImage Archive course, which focused on introducing programmatic approaches used in the analysis of bioimage data via the BioImage Archive, ran in May 2023.</p>
<p>Tags: Bioimage Analysis, Python, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Video, Slides</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/materials/microscopy-data-analysis-machine-learning-and-the-bioimage-archive-materials/">https://www.ebi.ac.uk/training/materials/microscopy-data-analysis-machine-learning-and-the-bioimage-archive-materials/</a></p>
</section>
<hr class="docutils" />
<section id="modular-training-resources-for-bioimage-analysis">
<h2>Modular training resources for bioimage analysis<a class="headerlink" href="#modular-training-resources-for-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Christian Tischer, Antonio Politi, Tim-Oliver Buchholz, Elnaz Fazeli, Nicola Gritti, Aliaksandr Halavatyi, Sebastian Gonzalez Tirado, Julian Hennies, Toby Hodges, Arif Khan, Dominik Kutra, Stefania Marcotti, Bugra Oezdemir, Felix Schneider, Martin Schorb, Anniek Stokkermans, Yi Sun, Nima Vakili</p>
<p>Published 2024-12-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Resources for teaching/preparing to teach bioimage analysis</p>
<p>Tags: Neubias, Bioimage Analysis, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14264885">https://zenodo.org/records/14264885</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14264885">https://doi.org/10.5281/zenodo.14264885</a></p>
</section>
<hr class="docutils" />
<section id="morphological-analysis-of-neural-cells-with-weka-and-snt-fiji-plugins">
<h2>Morphological analysis of neural cells with WEKA and SNT Fiji plugins<a class="headerlink" href="#morphological-analysis-of-neural-cells-with-weka-and-snt-fiji-plugins" title="Link to this heading">#</a></h2>
<p>Daniel Waiger</p>
<p>Published 2022-07-14</p>
<p>Licensed CC-BY-4.0</p>
<p>A simple workflow to detect Soma and neurite paths, from light microscopy datasets.</p>
<p>Using open-source tools for beginners.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/6834214">https://zenodo.org/records/6834214</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.6834214">https://doi.org/10.5281/zenodo.6834214</a></p>
</section>
<hr class="docutils" />
<section id="multi-template-matching-for-object-detection-slides">
<h2>Multi-Template-Matching for object-detection (slides)<a class="headerlink" href="#multi-template-matching-for-object-detection-slides" title="Link to this heading">#</a></h2>
<p>Laurent Thomas</p>
<p>Published 2022-05-16</p>
<p>Licensed CC-BY-4.0</p>
<p>This presentations describes Multi-Template-Matching, a novel method extending on template-matching for object-detection in images.</p>
<p>The project was part of the PhD project of Laurent Thomas between 2017 and 2020, under supervision of Jochen Gehrig. The project was hosted at ACQUIFER Imaging with collaboration of the medical university of Heidelberg, and part of the ImageInLife Horizon2020 ITN (PhD program). </p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/6554166">https://zenodo.org/records/6554166</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.6554166">https://doi.org/10.5281/zenodo.6554166</a></p>
</section>
<hr class="docutils" />
<section id="multi-view-fusion">
<h2>Multi-view fusion<a class="headerlink" href="#multi-view-fusion" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Lecture slides of a session on Multiview Fusion in Fiji</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_multiview_registration">https://git.mpi-cbg.de/rhaase/lecture_multiview_registration</a></p>
</section>
<hr class="docutils" />
<section id="multiplexed-tissue-imaging-tools-and-approaches">
<h2>Multiplexed tissue imaging - tools and approaches<a class="headerlink" href="#multiplexed-tissue-imaging-tools-and-approaches" title="Link to this heading">#</a></h2>
<p>Agustín Andrés Corbat, OmFrederic, Jonas Windhager, Kristína Lidayová</p>
<p>Licensed CC-BY-4.0</p>
<p>Material for the I2K 2024 “Multiplexed tissue imaging - tools and approaches” workshop</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository, Slides, Workshop</p>
<p><a class="github reference external" href="https://github.com/BIIFSweden/I2K2024-MTIWorkshop">BIIFSweden/I2K2024-MTIWorkshop</a></p>
<p><a class="reference external" href="https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1">https://docs.google.com/presentation/d/1R9-4lXAmTYuyFZpTMDR85SjnLsPZhVZ8/edit#slide=id.p1</a></p>
</section>
<hr class="docutils" />
<section id="my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud">
<h2>My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud<a class="headerlink" href="#my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud" title="Link to this heading">#</a></h2>
<p>Elnaz Fazeli</p>
<p>Published 2024-02-19</p>
<p>Licensed CC-BY-4.0</p>
<p>In these slides I introducemy journey through teaching bioimage analysis courses in different formats, from in person courses to online material. I have an overview of different training formats and comparing these for different audiences. </p>
<p>Tags: Teaching, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10679054">https://zenodo.org/records/10679054</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10679054">https://doi.org/10.5281/zenodo.10679054</a></p>
</section>
<hr class="docutils" />
<section id="neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter">
<h2>NEUBIAS Academy &#64;HOME: Interactive Bioimage Analysis with Python and Jupyter<a class="headerlink" href="#neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter" title="Link to this heading">#</a></h2>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Neubias, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/neubias_academy_biapy">guiwitz/neubias_academy_biapy</a></p>
</section>
<hr class="docutils" />
<section id="neubias-analyst-school-2018">
<h2>NEUBIAS Analyst School 2018<a class="headerlink" href="#neubias-analyst-school-2018" title="Link to this heading">#</a></h2>
<p>Assaf Zaritsky, Csaba Molnar, Vasja Urbancic, Richard Butler, Anna Kreshuk, Vannary Meas-Yedid</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2018">miura/NEUBIAS_AnalystSchool2018</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-course-2017">
<h2>NEUBIAS Bioimage Analyst Course 2017<a class="headerlink" href="#neubias-bioimage-analyst-course-2017" title="Link to this heading">#</a></h2>
<p>Curtis Rueden, Florian Levet, J.B. Sibarta, Alexandre Dafour, Daniel Sage, Sebastien Tosi, Michal Kozubek, Jean-Yves Tinevez, Kota Miura, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides, Tutorial</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_Bioimage_Analyst_Course2017">miura/NEUBIAS_Bioimage_Analyst_Course2017</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-school-2019">
<h2>NEUBIAS Bioimage Analyst School 2019<a class="headerlink" href="#neubias-bioimage-analyst-school-2019" title="Link to this heading">#</a></h2>
<p>Kota Miura, Chong Zhang, Jean-Yves Tinevez, Robert Haase, Julius Hossein, Pejamn Rasti, David Rousseau, Ignacio Arganda-Carreras, Siân Culley, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2019">miura/NEUBIAS_AnalystSchool2019</a></p>
</section>
<hr class="docutils" />
<section id="neubias-bioimage-analyst-school-2020">
<h2>NEUBIAS Bioimage Analyst School 2020<a class="headerlink" href="#neubias-bioimage-analyst-school-2020" title="Link to this heading">#</a></h2>
<p>Marion Louveaux, Stéphane Verger, Arianne Bercowsky Rama, Ignacio Arganda-Carreras, Estibaliz Gómez-de-Mariscal, Kota Miura, et al.</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Neubias, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides, Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/miura/NEUBIAS_AnalystSchool2020">miura/NEUBIAS_AnalystSchool2020</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage">
<h2>NFDI4BIOIMAGE<a class="headerlink" href="#nfdi4bioimage" title="Link to this heading">#</a></h2>
<p>Carsten Fortmann-Grote</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation was given at the 2nd MPG-NFDI Workshop on April 18th about e NFDI4BIOIMAGE Consortium, FAIRification of Image (meta)data, Zarr, RFC, Training (TA5), contributing.</p>
<p>Tags: Research Data Management, Bioimage Analysis, FAIR-Principles, Zarr, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.11031746">https://zenodo.org/doi/10.5281/zenodo.11031746</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters</p>
<p>Licensed CC-BY-4.0</p>
<p>NFDI4BIOIMAGE core mission, bioimage data challenge, task areas, FAIR bioimage workflows.</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8070038">https://doi.org/10.5281/zenodo.8070038</a></p>
<p><a class="reference external" href="https://zenodo.org/records/8070038">https://zenodo.org/records/8070038</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and Bioimage Analysis<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>NFDI4BIOIMAGE Consortium</p>
<p>Published 2024-08-07</p>
<p>Licensed CC-BY-4.0</p>
<p>Bioimaging refers to a collection of methods to visualize the internal structures and mechanisms of living organisms. The fundamental tool, the microscope, has enabled seminal discoveries like that of the cell as the smallest unit of life, and continues to expand our understanding of biological processes. Today, we can follow the interaction of single molecules within nanoseconds in a living cell, and the development of complete small organisms like fish and flies over several days starting from the fertilized egg. Each image pixel encodes multiple spatiotemporal and spectral dimensions, compounding the massive volume and complexity of bioimage data. Proper handling of this data is indispensable for analysis and its lack has become a growing hindrance for the many disciplines of the life and biomedical sciences relying on bioimaging. No single domain has the expertise to tackle this bottleneck alone.
As a method-specific consortium, NFDI4BIOMAGE seeks to address these issues, enabling bioimaging data to be shared and re-used like they are acquired, i.e., independently of disciplinary boundaries. We will provide solutions for exploiting the full information content of bioimage data and enable new discoveries through sharing and re-analysis. Our RDM strategy is based on a robust needs analysis that derives not only from a community survey but also from over a decade of experience in German BioImaging, the German Society for Microscopy and Image Analysis. It considers the entire lifecycle of bioimaging data, from acquisition to archiving, including analysis and enabling re-use. A foundational element of this strategy is the definition of a common, cloud-compatible, and interoperable digital object that bundles binary images with their descriptive and provenance metadata. With members from plant biology to neuroscience, NFDI4BIOIMAGE will champion the standardization of bioimage data to create a framework that answers discipline-specific needs while ensuring communication and interoperability with data types and RDM systems across domains. Integration of bioimage data with, e.g., omics data as the basis for spatial omics, holds great promise for fields such as cancer medicine. Unlocking the full potential of bioimage data will rely on the development and broad availability of exceptional analysis tools and training sets. NFDI4BIOIMAGE will make these accessible and usable including cutting-edge AI-based methods in scalable cloud environments. NFDI4BIOIMAGE intersects with multiple NFDI consortia, most prominently with GHGA for linking image and genomics data and with DataPLANT on the definition of FAIR data objects. Last but not least, NFDI4BIOIMAGE is internationally well connected and represents the opportunity for German scientists to keep path with and have a voice in several international initiatives focusing on the FAIRification of bioimage data as one of the main challenges for the advancement of knowledge in the life and biomedical sciences.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13168693">https://zenodo.org/records/13168693</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13168693">https://doi.org/10.5281/zenodo.13168693</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-data-management-illustrations-by-henning-falk">
<h2>NFDI4BIOIMAGE data management illustrations by Henning Falk<a class="headerlink" href="#nfdi4bioimage-data-management-illustrations-by-henning-falk" title="Link to this heading">#</a></h2>
<p>NFDI4BIOIMAGE Consortium</p>
<p>Published 2024-11-29</p>
<p>Licensed CC-BY-4.0</p>
<p>These illustrations were contracted by the Heinrich Heine University Düsseldorf in the frame of the consortium NFDI4BIOIMAGE from Henning Falk for the purpose of education and public outreach. The illustrations are free to use under a CC-BY 4.0 license.AttributionPlease include an attribution similar to: “Data annoation matters”, NFDI4BIOIMAGE Consortium (2024): NFDI4BIOIMAGE data management illustrations by Henning Falk, Zenodo, <a class="reference external" href="https://doi.org/10.5281/zenodo.14186100">https://doi.org/10.5281/zenodo.14186100</a>, is used under a CC-BY 4.0 license. Modifications to this illustration include cropping.
 </p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14186101">https://zenodo.org/records/14186101</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14186101">https://doi.org/10.5281/zenodo.14186101</a></p>
</section>
<hr class="docutils" />
<section id="nlp-course">
<h2>NLP Course<a class="headerlink" href="#nlp-course" title="Link to this heading">#</a></h2>
<p>Yandex School of Data Analysis</p>
<p>Licensed MIT</p>
<p>YSDA course in Natural Language Processing</p>
<p>Tags: Natural Language Processing, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/yandexdataschool/nlp_course">yandexdataschool/nlp_course</a></p>
</section>
<hr class="docutils" />
<section id="name-styles">
<h2>Name Styles<a class="headerlink" href="#name-styles" title="Link to this heading">#</a></h2>
<p>Nick Radcliffe</p>
<p>Published None</p>
<p>Licensed UNKNOWN</p>
<p>This playful exploration delves into the quirky world of naming conventions in computing and data, humorously comparing them to Boston Box matrices and D&amp;D alignments while poking fun at the chaos of categorizing what makes sense in metadata standards.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://www.tdda.info/name-styles">https://www.tdda.info/name-styles</a></p>
</section>
<hr class="docutils" />
<section id="navigating-the-bioimage-analysis-landscape-understanding-the-community-and-its-collaborative-dynamics">
<h2>Navigating the Bioimage Analysis Landscape: Understanding the Community  and its Collaborative Dynamics<a class="headerlink" href="#navigating-the-bioimage-analysis-landscape-understanding-the-community-and-its-collaborative-dynamics" title="Link to this heading">#</a></h2>
<p>Martin Schätz</p>
<p>Published 2024-06-28</p>
<p>Licensed CC-BY-4.0</p>
<p>Presented as a part of:
Mexican Bioimaging Workshops 9: Fundamentos de Microscopía “Microscopía en la Salud”
Workshop on light microscopyJune 26th to 28th, 2024
Outreach 9June 29th, 2024
Expanding Global Access to BioimagingConnecting the Mexican Bioimaging Community</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/12584729">https://zenodo.org/records/12584729</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.12584729">https://doi.org/10.5281/zenodo.12584729</a></p>
</section>
<hr class="docutils" />
<section id="neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">
<h2>Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist<a class="headerlink" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist" title="Link to this heading">#</a></h2>
<p>Martin Weigert, Olivier Burri, Siân Culley, Uwe Schmidt</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Neubias, Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides, Notebook</p>
<p><a class="github reference external" href="https://github.com/maweigert/neubias_academy_stardist">maweigert/neubias_academy_stardist</a></p>
</section>
<hr class="docutils" />
<section id="neubiaspasteur2023-advancedcellpose">
<h2>NeubiasPasteur2023_AdvancedCellPose<a class="headerlink" href="#neubiaspasteur2023-advancedcellpose" title="Link to this heading">#</a></h2>
<p>Gaelle Letort</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tutorial for running CellPose advanced functions</p>
<p>Tags: Bioimage Analysis, Artificial Intelligence, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/gletort/NeubiasPasteur2023_AdvancedCellPose">gletort/NeubiasPasteur2023_AdvancedCellPose</a></p>
</section>
<hr class="docutils" />
<section id="nextflow-demo-pipelines-for-image-processing">
<h2>Nextflow Demo Pipelines for Image Processing<a class="headerlink" href="#nextflow-demo-pipelines-for-image-processing" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Code</p>
<p><a class="github reference external" href="https://github.com/JaneliaSciComp/nf-demos">JaneliaSciComp/nf-demos</a></p>
</section>
<hr class="docutils" />
<section id="ome-documentation">
<h2>OME Documentation<a class="headerlink" href="#ome-documentation" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: OMERO, Include In Dalia</p>
<p>Content type: Documentation</p>
<p><a class="reference external" href="https://www.openmicroscopy.org/docs/">https://www.openmicroscopy.org/docs/</a></p>
</section>
<hr class="docutils" />
<section id="omero-hcs-analysis-pipeline-using-jupyter-notebooks">
<h2>OMERO - HCS analysis pipeline using Jupyter Notebooks<a class="headerlink" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks" title="Link to this heading">#</a></h2>
<p>Riccardo Massei</p>
<p>Licensed MIT</p>
<p>Material and solutions for the course ‘Bioimage data management and analysis with OMERO’ held in Heidelberg (13th May 2024) - Module 3 (1.45 pm - 3.45 pm): OMERO and Jupyter Notebooks. Main goal of the workflow is to show the potential of JN to perform reproducible image analysis in connection with an OMERO instance. In this specific example, we are performing a simple nuclei segmentation from raw images uploaded in OMERO.</p>
<p>Tags: Teaching, Bioimage Analysis, Notebooks, Python, OMERO, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/rmassei/2024-jn-omero-pipeline">rmassei/2024-jn-omero-pipeline</a></p>
</section>
<hr class="docutils" />
<section id="omero-guide">
<h2>OMERO Guide<a class="headerlink" href="#omero-guide" title="Link to this heading">#</a></h2>
<p>Licensed BSD-2-CLAUSE</p>
<p>Tags: OMERO, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://omero-guides.readthedocs.io/en/latest/">https://omero-guides.readthedocs.io/en/latest/</a></p>
</section>
<hr class="docutils" />
<section id="omero-and-arc-workflow">
<h2>OMERO and ARC Workflow<a class="headerlink" href="#omero-and-arc-workflow" title="Link to this heading">#</a></h2>
<p>Niraj Kandpal, Peter Zentis, Monica Valencia-Schneider, Astrid Schauss</p>
<p>Published 2025-04-15</p>
<p>Licensed CC-BY-4.0</p>
<p>A tutorial for OMERO and ARC Interoprability and (meta)data exchange. This describes the relevant metadata to transfer from OMERO to ARC and the other way round.</p>
<p>Tags: Bioimage Analysis, OMERO, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/15225616">https://zenodo.org/records/15225616</a></p>
</section>
<hr class="docutils" />
<section id="omero-documentation">
<h2>OMERO documentation<a class="headerlink" href="#omero-documentation" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: OMERO, Include In Dalia</p>
<p>Content type: Documentation</p>
<p><a class="reference external" href="https://omero.readthedocs.io/en/stable/">https://omero.readthedocs.io/en/stable/</a></p>
</section>
<hr class="docutils" />
<section id="omero-walkthrough-example">
<h2>OMERO walkthrough example<a class="headerlink" href="#omero-walkthrough-example" title="Link to this heading">#</a></h2>
<p>Licensed BSD-2-CLAUSE</p>
<p>Tags: OMERO, Include In Dalia</p>
<p>Content type: Document</p>
<p><a class="reference external" href="https://omero-guides.readthedocs.io/en/latest/example.html">https://omero-guides.readthedocs.io/en/latest/example.html</a></p>
</section>
<hr class="docutils" />
<section id="omero-walkthrough-for-facility-managers">
<h2>OMERO walkthrough for facility managers<a class="headerlink" href="#omero-walkthrough-for-facility-managers" title="Link to this heading">#</a></h2>
<p>Licensed BSD-2-CLAUSE</p>
<p>Tags: OMERO, Include In Dalia</p>
<p>Content type: Document</p>
<p><a class="reference external" href="https://omero-guides.readthedocs.io/en/latest/example_facility_manager.html">https://omero-guides.readthedocs.io/en/latest/example_facility_manager.html</a></p>
</section>
<hr class="docutils" />
<section id="omexcavator-a-tool-for-exporting-and-connecting-bioimaging-specific-metadata-in-wider-knowledge-graphs">
<h2>OMExcavator: a tool for exporting and connecting  Bioimaging-specific metadata in wider knowledge graphs<a class="headerlink" href="#omexcavator-a-tool-for-exporting-and-connecting-bioimaging-specific-metadata-in-wider-knowledge-graphs" title="Link to this heading">#</a></h2>
<p>Stefan Dvoretskii, Klaus Maier-Hein, Marco Nolden, Christian Schmidt, Michele Bortolomeazzi, Josh Moore</p>
<p>Published 2025-05-15</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15423904">https://zenodo.org/records/15423904</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15423904">https://doi.org/10.5281/zenodo.15423904</a></p>
</section>
<hr class="docutils" />
<section id="onbi-image-analysis-course">
<h2>ONBI Image Analysis Course<a class="headerlink" href="#onbi-image-analysis-course" title="Link to this heading">#</a></h2>
<p>Mark Jenkinson, Jens Rittscher, Dominic Waithe</p>
<p>Licensed GPL-2.0</p>
<p>This repository contains the materials for the University of Oxford DTC ONBI Image Analysis course.</p>
<p>Tags: Python, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/dwaithe/ONBI_image_analysis">dwaithe/ONBI_image_analysis</a></p>
</section>
<hr class="docutils" />
<section id="object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">
<h2>Object Tracking and Track Analysis using TrackMate and CellTracksColab<a class="headerlink" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab" title="Link to this heading">#</a></h2>
<p>Joanna Pylvänäinen</p>
<p>Published None</p>
<p>Licensed GPL-3.0</p>
<p>I2K 2024 workshop materials for “Object Tracking and Track Analysis using TrackMate and CellTracksColab”</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository, Tutorial, Workshop, Slides</p>
<p><a class="github reference external" href="https://github.com/CellMigrationLab/I2K_2024">CellMigrationLab/I2K_2024</a></p>
</section>
<hr class="docutils" />
<section id="omero-deployment-examples">
<h2>Omero Deployment examples<a class="headerlink" href="#omero-deployment-examples" title="Link to this heading">#</a></h2>
<p>Licensed BSD-2-CLAUSE</p>
<p>Tags: OMERO, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="github reference external" href="https://github.com/ome/omero-deployment-examples">ome/omero-deployment-examples</a></p>
</section>
<hr class="docutils" />
<section id="online-r-learning">
<h2>Online_R_learning<a class="headerlink" href="#online-r-learning" title="Link to this heading">#</a></h2>
<p>C. Li</p>
<p>Published 2023-07-09T06:27:14+00:00</p>
<p>Licensed CC0-1.0</p>
<p>Online R learning for applied statistics</p>
<p>Tags: Statistics, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/cxli233/Online_R_learning">cxli233/Online_R_learning</a></p>
</section>
<hr class="docutils" />
<section id="open-image-data-handbook">
<h2>Open Image Data Handbook<a class="headerlink" href="#open-image-data-handbook" title="Link to this heading">#</a></h2>
<p>Kevin Yamauchi</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Neubias, Research Data Management, Napari, Python, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Book, Notebook</p>
<p><a class="reference external" href="https://kevinyamauchi.github.io/open-image-data/intro.html">https://kevinyamauchi.github.io/open-image-data/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="open-science-sharing-licensing">
<h2>Open Science, Sharing &amp; Licensing<a class="headerlink" href="#open-science-sharing-licensing" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-04-18</p>
<p>Licensed CC-BY-4.0</p>
<p>Wir tauchen ein in die Welt der Open Science und definieren Begriffe wie Open Source, Open Access und die FAIR-Prinzipien (Findable, Accessible, Interoperable and Reuasable). Wir diskutieren, wie diese Methoden der [wissenschaftlichen] Kommunikation und des Datenmanagements die Welt verändern und wie wir sie praktisch in unsere Arbeit integrieren können. Dabei spielen Aspekte wie Copyright und Lizenzierung eine wichtige Rolle.</p>
<p>Tags: Research Data Management, Open Access, FAIR-Principles, Licensing, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10990107">https://zenodo.org/records/10990107</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10990107">https://doi.org/10.5281/zenodo.10990107</a></p>
</section>
<hr class="docutils" />
<section id="open-source-ai-tools-for-bioimage-analysis-workshop-2024-icob-academia-sinica-taiwan">
<h2>Open source AI Tools for bioimage analysis workshop (2024) &#64;ICOB, Academia Sinica, Taiwan<a class="headerlink" href="#open-source-ai-tools-for-bioimage-analysis-workshop-2024-icob-academia-sinica-taiwan" title="Link to this heading">#</a></h2>
<p>Wei-Chen Chu</p>
<p>Published 2024-08-09</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation file used in the  Open source AI Tools for bioimage analysis workshop &#64;ICOB, Academia Sinica, Taiwan (2024)Introduce ilastik, StarDist, Cellpose, Segment Anything Model (SAM), and how to use it briefly.
Full video recording (in Chinese) is available on YouTube: <a class="reference external" href="https://youtu.be/KqwssouW0G0">https://youtu.be/KqwssouW0G0</a>
This document is part III of the previous document:Chu, W.-C. (2024). Bioimage Analysis with FIJI /ImageJ &amp; Friends workshop (2024) &#64;ICOB, Academia Sinica, Taiwan. Zenodo. <a class="reference external" href="https://doi.org/10.5281/zenodo.12803966">https://doi.org/10.5281/zenodo.12803966</a></p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13284351">https://zenodo.org/records/13284351</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13284351">https://doi.org/10.5281/zenodo.13284351</a></p>
</section>
<hr class="docutils" />
<section id="overview-of-the-galaxy-omero-suite-upload-images-and-metadata-in-omero-using-galaxy">
<h2>Overview of the Galaxy OMERO-suite - Upload images and metadata in OMERO using Galaxy<a class="headerlink" href="#overview-of-the-galaxy-omero-suite-upload-images-and-metadata-in-omero-using-galaxy" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Björn Grüning</p>
<p>Published 2024-12-02</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: OMERO, Galaxy, Metadata, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Tutorial, Framework, Workflow</p>
<p><a class="reference external" href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/omero-suite/tutorial.html">https://training.galaxyproject.org/training-material/topics/imaging/tutorials/omero-suite/tutorial.html</a></p>
</section>
<hr class="docutils" />
<section id="parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing">
<h2>Parallelization and heterogeneous computing: from pure CPU to GPU-accelerated image processing<a class="headerlink" href="#parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/11-1171">https://f1000research.com/slides/11-1171</a></p>
<p><a class="reference external" href="https://doi.org/10.7490/f1000research.1119154.1">https://doi.org/10.7490/f1000research.1119154.1</a></p>
</section>
<hr class="docutils" />
<section id="photonic-data-analysis-in-2050">
<h2>Photonic data analysis in 2050<a class="headerlink" href="#photonic-data-analysis-in-2050" title="Link to this heading">#</a></h2>
<p>Oleg Ryabchykov, Shuxia Guo, Thomas Bocklitz</p>
<p>Licensed CC-BY-4.0</p>
<p>Photonic data analysis, combining imaging, spectroscopy, machine learning, and computer science, requires flexible methods and interdisciplinary collaborations to advance. Essential developments include standardizing data infrastructure for comparability, optimizing data-driven models for complex investigations, and creating techniques to handle limited or unbalanced data and device variations.</p>
<p>Tags: FAIR-Principles, Machine Learning, Research Data Management, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1016/j.vibspec.2024.103685">https://doi.org/10.1016/j.vibspec.2024.103685</a></p>
</section>
<hr class="docutils" />
<section id="plants-python-a-series-of-lessons-in-coding-plant-biology-computation-and-bioinformatics">
<h2>Plants &amp; Python - A series of lessons in coding, plant biology, computation, and bioinformatics<a class="headerlink" href="#plants-python-a-series-of-lessons-in-coding-plant-biology-computation-and-bioinformatics" title="Link to this heading">#</a></h2>
<p>Dan Chitwood, Sourabh Palande</p>
<p>Tags: Notebook, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://academic.oup.com/plcell/article/34/7/e1/6628764">https://academic.oup.com/plcell/article/34/7/e1/6628764</a></p>
<p><a class="reference external" href="https://plantsandpython.github.io/PlantsAndPython/00_Opening_page.html">https://plantsandpython.github.io/PlantsAndPython/00_Opening_page.html</a></p>
</section>
<hr class="docutils" />
<section id="pol-bio-image-analysis-early-career-bio-image-analysis-training-school">
<h2>PoL Bio-Image Analysis Early Career Bio-image Analysis Training School<a class="headerlink" href="#pol-bio-image-analysis-early-career-bio-image-analysis-training-school" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis, Python, Include In Dalia</p>
<p>Content type: Event, Notebook</p>
<p><a class="reference external" href="https://biapol.github.io/PoL-BioImage-Analysis-TS-Early-Career-Track/intro.html">https://biapol.github.io/PoL-BioImage-Analysis-TS-Early-Career-Track/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="pol-bio-image-analysis-gpu-accelerated-image-analysis-training-school">
<h2>PoL Bio-Image Analysis GPU Accelerated Image Analysis Training School<a class="headerlink" href="#pol-bio-image-analysis-gpu-accelerated-image-analysis-training-school" title="Link to this heading">#</a></h2>
<p>Tags: Bioimage Analysis, Python, Include In Dalia</p>
<p>Content type: Event, Notebook</p>
<p><a class="reference external" href="https://biapol.github.io/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/intro.html">https://biapol.github.io/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis">
<h2>PoL Bio-Image Analysis Training School on GPU-Accelerated Image Analysis<a class="headerlink" href="#pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis" title="Link to this heading">#</a></h2>
<p>Stephane Rigaud, Brian Northan, Till Korten, Neringa Jurenaite, Apurv Deepak Kulkarni, Peter Steinbach, Sebastian Starke, Johannes Soltwedel, Marvin Albert, Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository hosts notebooks, information and data for the GPU-Accelerated Image Analysis Track of the PoL Bio-Image Analysis Symposium.</p>
<p>Tags: Gpu, Clesperanto, Dask, Python, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/BiAPoL/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis/">BiAPoL/PoL-BioImage-Analysis-TS-GPU-Accelerated-Image-Analysis</a></p>
</section>
<hr class="docutils" />
<section id="practical-guide-to-the-international-alignment-of-research-data-management-extended-edition">
<h2>Practical Guide to the International Alignment of Research Data Management - Extended Edition<a class="headerlink" href="#practical-guide-to-the-international-alignment-of-research-data-management-extended-edition" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://www.scienceeurope.org/our-resources/practical-guide-to-the-international-alignment-of-research-data-management/">https://www.scienceeurope.org/our-resources/practical-guide-to-the-international-alignment-of-research-data-management/</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4915861">https://doi.org/10.5281/zenodo.4915861</a></p>
</section>
<hr class="docutils" />
<section id="practical-considerations-for-data-exploration-in-quantitative-cell-biology">
<h2>Practical considerations for data exploration in quantitative cell biology<a class="headerlink" href="#practical-considerations-for-data-exploration-in-quantitative-cell-biology" title="Link to this heading">#</a></h2>
<p>Joanna W. Pylvänäinen, Hanna Grobe, Guillaume Jacquemet</p>
<p>Published 2025-04-07</p>
<p>Licensed CC-BY-4.0</p>
<p>This article emphasizes the importance of structured, hands-on data exploration in quantitative cell biology, offering practical advice for analyzing bioimage datasets. It also highlights how generative AI and large language models can enhance and streamline data workflows for more reliable and transparent research.</p>
<p>Tags: Bioimage Analysis, Data Exploration, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://journals.biologists.com/jcs/article/138/7/jcs263801/367617/Practical-considerations-for-data-exploration-in">https://journals.biologists.com/jcs/article/138/7/jcs263801/367617/Practical-considerations-for-data-exploration-in</a></p>
</section>
<hr class="docutils" />
<section id="promoting-data-management-at-the-nikon-imaging-center-and-cell-biology-microscopy-facility">
<h2>Promoting Data Management at the Nikon Imaging Center and Cell Biology Microscopy Facility<a class="headerlink" href="#promoting-data-management-at-the-nikon-imaging-center-and-cell-biology-microscopy-facility" title="Link to this heading">#</a></h2>
<p>Jennifer Waters</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://datamanagement.hms.harvard.edu/news/promoting-data-management-nikon-imaging-center-and-cell-biology-microscopy-facility">https://datamanagement.hms.harvard.edu/news/promoting-data-management-nikon-imaging-center-and-cell-biology-microscopy-facility</a></p>
</section>
<hr class="docutils" />
<section id="promoting-reproducibility-in-biomedical-research-through-image-analysis">
<h2>Promoting Reproducibility in Biomedical Research through Image Analysis<a class="headerlink" href="#promoting-reproducibility-in-biomedical-research-through-image-analysis" title="Link to this heading">#</a></h2>
<p>David Barry</p>
<p>Published 2025-01-29</p>
<p>Licensed CC-BY-4.0 INTERNATIONAL</p>
<p>Slides presented at Empowering Healthcare with Automated Analysis at London Metropolitan University.</p>
<p>Tags: Reproducibility, Image Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14767944">https://doi.org/10.5281/zenodo.14767944</a></p>
</section>
<hr class="docutils" />
<section id="prompt-engineering-in-bio-image-analysis">
<h2>Prompt Engineering in Bio-image Analysis<a class="headerlink" href="#prompt-engineering-in-bio-image-analysis" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Jupyter, Bioimage Analysis, Prompt Engineering, Biabob, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2024/07/18/prompt-engineering-in-bio-image-analysis/">https://focalplane.biologists.com/2024/07/18/prompt-engineering-in-bio-image-analysis/</a></p>
</section>
<hr class="docutils" />
<section id="prompt-engineering-agentic-workflows-and-multi-modal-large-language-models">
<h2>Prompt Engineering, Agentic Workflows and Multi-modal Large Language Models<a class="headerlink" href="#prompt-engineering-agentic-workflows-and-multi-modal-large-language-models" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2025-01-19</p>
<p>Licensed CC-BY-4.0</p>
<p>In these two slide-decks we explore applications of large language models. In the first slide deck we dive into prompt engineering, function calling and how to build agentic workflows. In the second slide-deck we explore multi-modal large language models focusing on vision language models and image generation models. </p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14692037">https://zenodo.org/records/14692037</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14692037">https://doi.org/10.5281/zenodo.14692037</a></p>
</section>
<hr class="docutils" />
<section id="prompt-engineering-llms-course">
<h2>Prompt-Engineering-LLMs-Course<a class="headerlink" href="#prompt-engineering-llms-course" title="Link to this heading">#</a></h2>
<p>Isra Mekki</p>
<p>Published 2024-09-11T07:45:30+00:00</p>
<p>Licensed MIT</p>
<p>Tags: Llms, Prompt Engineering, Code Generation, Include In Dalia</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/HelmholtzAI-Consultants-Munich/Prompt-Engineering-LLMs-Course">HelmholtzAI-Consultants-Munich/Prompt-Engineering-LLMs-Course</a></p>
</section>
<hr class="docutils" />
<section id="pygameprompts">
<h2>PygamePrompts<a class="headerlink" href="#pygameprompts" title="Link to this heading">#</a></h2>
<p>Lea Kabjesz, Lea Gihlein, Mara Lampert, FPGro</p>
<p>Published 2025-08-12T11:36:36+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>A collection of workshop materials for exploring prompting techniques by building a Snake game in Cursor.</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/kaabl/PygamePrompts">kaabl/PygamePrompts</a></p>
</section>
<hr class="docutils" />
<section id="python-bioimage-analysis-tutorial">
<h2>Python BioImage Analysis Tutorial<a class="headerlink" href="#python-bioimage-analysis-tutorial" title="Link to this heading">#</a></h2>
<p>Jonas Hartmann</p>
<p>Licensed MIT</p>
<p>Tags: Python, Bioimage Analysis, Include In Dalia</p>
<p><a class="github reference external" href="https://github.com/WhoIsJack/python-bioimage-analysis-tutorial">WhoIsJack/python-bioimage-analysis-tutorial</a></p>
</section>
<hr class="docutils" />
<section id="python-workshop-image-processing">
<h2>Python Workshop - Image Processing<a class="headerlink" href="#python-workshop-image-processing" title="Link to this heading">#</a></h2>
<p>Karin Sasaki, Jonas Hartmann</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Python, Include In Dalia</p>
<p>Content type: Code, Notebook</p>
<p><a class="github reference external" href="https://github.com/karinsasaki/python-workshop-image-processing">karinsasaki/python-workshop-image-processing</a></p>
</section>
<hr class="docutils" />
<section id="python-for-bioimage-analysis-basic-tools-and-setup-on-windows">
<h2>Python for Bioimage Analysis Basic Tools and Setup on Windows<a class="headerlink" href="#python-for-bioimage-analysis-basic-tools-and-setup-on-windows" title="Link to this heading">#</a></h2>
<p>Alberto Díez</p>
<p>Published 2025-09-04</p>
<p>Licensed UNKNOWN</p>
<p>Introduction to basic concepts and tools needed to set up Python for bioimage analysis on Windows.</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Video</p>
<p><a class="reference external" href="https://youtu.be/tzdFuxF2E3U">https://youtu.be/tzdFuxF2E3U</a></p>
</section>
<hr class="docutils" />
<section id="python-for-microscopists">
<h2>Python for Microscopists<a class="headerlink" href="#python-for-microscopists" title="Link to this heading">#</a></h2>
<p>Sreenivas Bhattiprolu</p>
<p>Licensed MIT</p>
<p>Tags: Python, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/bnsreenu/python_for_microscopists">bnsreenu/python_for_microscopists</a></p>
</section>
<hr class="docutils" />
<section id="qi-2024-analysis-lab-manual">
<h2>QI 2024 Analysis Lab Manual<a class="headerlink" href="#qi-2024-analysis-lab-manual" title="Link to this heading">#</a></h2>
<p>Beth Cimini, Florian Jug, QI 2024</p>
<p>Licensed CC-BY-4.0</p>
<p>This book contains the quantitative analysis labs for the QI CSHL course, 2024</p>
<p>Tags: Python, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="reference external" href="https://bethac07.github.io/qi_2024_analysis_lab_manual/intro.html">https://bethac07.github.io/qi_2024_analysis_lab_manual/intro.html</a></p>
</section>
<hr class="docutils" />
<section id="qm-course-lectures-on-bio-image-analysis-with-napari-2024">
<h2>QM Course Lectures on Bio-Image Analysis with napari 2024<a class="headerlink" href="#qm-course-lectures-on-bio-image-analysis-with-napari-2024" title="Link to this heading">#</a></h2>
<p>Marcelo Leomil Zoccoler</p>
<p>Licensed CC-BY-4.0</p>
<p>In these lectures, we will explore ways to analyze microscopy images with Python and visualize them with napari, an nD viewer open-source software. The analysis will be done in Python mostly using the scikit-image, pyclesperanto and apoc libraries, via Jupyter notebooks. We will also explore some napari plugins as an interactive and convenient alternative way of performing these analysis, especially the napari-assistant, napari-apoc and napari-flim-phasor-plotter plugins.</p>
<p>Tags: Napari, Python, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="reference external" href="https://zoccoler.github.io/QM_Course_Bio_Image_Analysis_with_napari_2024">https://zoccoler.github.io/QM_Course_Bio_Image_Analysis_with_napari_2024</a></p>
</section>
<hr class="docutils" />
<section id="quarep-limi-a-community-driven-initiative-to-establish-guidelines-for-quality-assessment-and-reproducibility-for-instruments-and-images-in-light-microscopy">
<h2>QUAREP-LiMi: A community-driven initiative to establish guidelines for quality assessment and reproducibility for instruments and images in light microscopy<a class="headerlink" href="#quarep-limi-a-community-driven-initiative-to-establish-guidelines-for-quality-assessment-and-reproducibility-for-instruments-and-images-in-light-microscopy" title="Link to this heading">#</a></h2>
<p>Glyn Nelson, Ulrike Boehme, et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Quareo-Limi, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://onlinelibrary.wiley.com/doi/10.1111/jmi.13041">https://onlinelibrary.wiley.com/doi/10.1111/jmi.13041</a></p>
</section>
<hr class="docutils" />
<section id="qupath-for-python-programmers">
<h2>QuPath for Python programmers<a class="headerlink" href="#qupath-for-python-programmers" title="Link to this heading">#</a></h2>
<p>Alan O’Callaghan, Léo Leplat</p>
<p>Published 2024-10-23</p>
<p>Licensed UNKNOWN</p>
<p>These are the notebooks and associated files for the i2k 2024 QuPath for Python programmers workshop.</p>
<p>Tags: Python, Notebooks, Open Source Software, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository, Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/qupath/i2k-qupath-for-python-programmers">qupath/i2k-qupath-for-python-programmers</a></p>
</section>
<hr class="docutils" />
<section id="qupath-open-source-software-for-analysing-awkward-images">
<h2>QuPath: Open source software for analysing (awkward) images<a class="headerlink" href="#qupath-open-source-software-for-analysing-awkward-images" title="Link to this heading">#</a></h2>
<p>Peter Bankhead</p>
<p>Published 2020-12-16</p>
<p>Licensed CC-BY-4.0</p>
<p>Slides from the CZI/EOSS online meeting in December 2020.</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/4328911">https://zenodo.org/records/4328911</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4328911">https://doi.org/10.5281/zenodo.4328911</a></p>
</section>
<hr class="docutils" />
<section id="quality-assurance-of-segmentation-results">
<h2>Quality assurance of segmentation results<a class="headerlink" href="#quality-assurance-of-segmentation-results" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/04/13/quality-assurance-of-segmentation-results/">https://focalplane.biologists.com/2023/04/13/quality-assurance-of-segmentation-results/</a></p>
</section>
<hr class="docutils" />
<section id="rdm-starter-kit">
<h2>RDM Starter Kit<a class="headerlink" href="#rdm-starter-kit" title="Link to this heading">#</a></h2>
<p>GO FAIR</p>
<p>Licensed CC-BY-4.0</p>
<p>This page is supposed to serve as a Starter Kit for research data management (RDM). It lists resources designed to help researchers get started to organize their data.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Website</p>
<p><a class="reference external" href="https://www.go-fair.org/resources/rdm-starter-kit/">https://www.go-fair.org/resources/rdm-starter-kit/</a></p>
</section>
<hr class="docutils" />
<section id="rdmkit-training-resources">
<h2>RDMKit Training Resources<a class="headerlink" href="#rdmkit-training-resources" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://rdmkit.elixir-europe.org/all_training_resources">https://rdmkit.elixir-europe.org/all_training_resources</a></p>
</section>
<hr class="docutils" />
<section id="rembi-overview">
<h2>REMBI Overview<a class="headerlink" href="#rembi-overview" title="Link to this heading">#</a></h2>
<p>Licensed CC0-1.0</p>
<p>Recommended Metadata for Biological Images (REMBI) provides guidelines for metadata for biological images to enable the FAIR sharing of scientific data.</p>
<p>Tags: FAIR-Principles, Metadata, Research Data Management, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/bioimage-archive/rembi-help-overview/">https://www.ebi.ac.uk/bioimage-archive/rembi-help-overview/</a></p>
</section>
<hr class="docutils" />
<section id="rechtsfragen-bei-open-science-ein-leitfaden">
<h2>Rechtsfragen bei Open Science - Ein Leitfaden<a class="headerlink" href="#rechtsfragen-bei-open-science-ein-leitfaden" title="Link to this heading">#</a></h2>
<p>Till Kreutzer, Henning Lahmann</p>
<p>Published 2021-05-25</p>
<p>Licensed CC-BY-4.0</p>
<p>Die Digitalisierung ermöglicht eine offene Wissenschaft (Open Science). Diese hat viele Aspekte, insbesondere den freien Zugang zu wissenschaftlichen Veröffentlichungen und Materialien (Open Access), transparente Begutachtungsverfahren (Open Peer Review) oder quelloffene Technologien (Open Source). Das Programm Hamburg Open Science (Laufzeit 2018–2020) unterstützt unter anderem den Kulturwandel in der Wissenschaft. In diesem Kontext entstand der nun vorliegende Leitfaden, der das rechtliche Umfeld greifbar machen soll. Der Leitfaden erarbeitet die betroffenen Rechtsgebiete zunächst systematisch. Im zweiten Teil werden rechtliche Fragen zu Open Science beantwortet, die direkt aus den Universitäten und Bibliotheken kommen.</p>
<p>Tags: Open Science, Open Access, Copyright, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://hup.sub.uni-hamburg.de/oa-pub/catalog/book/205">https://hup.sub.uni-hamburg.de/oa-pub/catalog/book/205</a></p>
</section>
<hr class="docutils" />
<section id="report-on-a-pilot-study-implementation-of-omero-for-microscopy-data-management">
<h2>Report on a pilot study:  Implementation of OMERO for  microscopy data management<a class="headerlink" href="#report-on-a-pilot-study-implementation-of-omero-for-microscopy-data-management" title="Link to this heading">#</a></h2>
<p>Silke Tulok, Gunar Fabig, Andy Vogelsang, Thomas Kugel, Thomas Müller-Reichert</p>
<p>Published 2023-11-10</p>
<p>Licensed CC-BY-4.0</p>
<p>The Core Facility Cellular Imaging (CFCI) at the Faculty of Medicine Carl Gustav Carus (TU Dresden) is currently running a pilot project for testing the use and handling of the OMERO software. This is done together with interested users of the imaging facility and a research group. Currently, we are pushing forward this pilot study on a small scale without any data steward. Our experiences argue so far for giving data management issues into the hands of dedicated personnel not fully involved in research projects. As funding agencies will ask for higher and higher standards for implementing FAIRdata principles in the future, this will be a releva</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/10103316">https://zenodo.org/records/10103316</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10103316">https://doi.org/10.5281/zenodo.10103316</a></p>
</section>
<hr class="docutils" />
<section id="reproducible-bio-image-analysis-using-python-napari-jupyter-and-ai">
<h2>Reproducible Bio-Image Analysis using Python, Napari, Jupyter and AI<a class="headerlink" href="#reproducible-bio-image-analysis-using-python-napari-jupyter-and-ai" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2025-09-09</p>
<p>Licensed CC-BY-4.0</p>
<p>In this slide deck we learn how to write reproducible bio-image analysis code in Jupyter notebooks. Goal is not just to have code running elsewhere reproducibly, but also enabling others to understand workflows to enable them reproducing the analysis also in their mind and potentially other tools. Additionally we cover how to generate Jupyter notebooks from Napari and using artificial intelligence, namely bia-bob.</p>
<p>Tags: Nfdi4Bioimage, Bioimage Analysis, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/17085991">https://zenodo.org/records/17085991</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.17085991">https://doi.org/10.5281/zenodo.17085991</a></p>
</section>
<hr class="docutils" />
<section id="rescaling-images-and-pixel-an-isotropy">
<h2>Rescaling images and pixel (an)isotropy<a class="headerlink" href="#rescaling-images-and-pixel-an-isotropy" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/03/02/rescaling-images-and-pixel-anisotropy/">https://focalplane.biologists.com/2023/03/02/rescaling-images-and-pixel-anisotropy/</a></p>
</section>
<hr class="docutils" />
<section id="research-data-management-seminar-slides">
<h2>Research Data Management Seminar - Slides<a class="headerlink" href="#research-data-management-seminar-slides" title="Link to this heading">#</a></h2>
<p>Stefano Della Chiesa</p>
<p>Published 2022-05-18</p>
<p>Licensed CC-BY-4.0</p>
<p>This Research Data Management (RDM) Slides introduce to the multidisciplinary knowledge and competencies required to address policy compliance and research data management best practices throughout a project lifecycle, and beyond it.</p>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>Module 1 - Introduces the RDM giving its context in the Research Data Governance
Module 2 - Illustrates the most important RDM policies and principles
Module 3 - Provides the most relevant RDM knowledge bricks
Module 4 - Discuss the Data Management Plans (DMPs), examples, templates and guidance
</pre></div>
</div>
<p> </p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/record/6602101">https://zenodo.org/record/6602101</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.6602101">https://doi.org/10.5281/zenodo.6602101</a></p>
</section>
<hr class="docutils" />
<section id="research-data-managemet-and-how-not-to-get-overwhelmed-with-data">
<h2>Research Data Managemet and how not to get overwhelmed with data<a class="headerlink" href="#research-data-managemet-and-how-not-to-get-overwhelmed-with-data" title="Link to this heading">#</a></h2>
<p>Martin Schätz</p>
<p>Published 2023-09-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Research data management and how not to get overwhelmed with data presentation is an overview of bioimage analysis with a focus on the basics for data management planning, FAIR principles, and how to practically organize folders and prepares naming convention. The presentation includes an overview of metadata, Creative Common licenses, and a sum up of electronic laboratory notebooks. The last two slides go through how all of that works in practice in open access core microscopy facility.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/8372703">https://zenodo.org/records/8372703</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8372703">https://doi.org/10.5281/zenodo.8372703</a></p>
</section>
<hr class="docutils" />
<section id="running-deep-learning-scripts-in-the-bia-pol-omero-server">
<h2>Running Deep-Learning Scripts in the BiA-PoL Omero Server<a class="headerlink" href="#running-deep-learning-scripts-in-the-bia-pol-omero-server" title="Link to this heading">#</a></h2>
<p>Marcelo Zoccoler</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Python, Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://biapol.github.io/blog/marcelo_zoccoler/omero_scripts/readme.html">https://biapol.github.io/blog/marcelo_zoccoler/omero_scripts/readme.html</a></p>
</section>
<hr class="docutils" />
<section id="swc-gcnu-software-skills">
<h2>SWC/GCNU Software Skills<a class="headerlink" href="#swc-gcnu-software-skills" title="Link to this heading">#</a></h2>
<p>Licensed CC-BY-4.0</p>
<p>Computational skills training at the UCL Sainsbury Wellcome Centre and Gatsby Computational Neuroscience Unit, delivered by members of the Neuroinformatics Unit.</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Collection, Online Course, Video, Tutorial</p>
<p><a class="reference external" href="https://software-skills.neuroinformatics.dev/index.html">https://software-skills.neuroinformatics.dev/index.html</a></p>
</section>
<hr class="docutils" />
<section id="scientific-visualization-python-matplotlib">
<h2>Scientific Visualization: Python + Matplotlib<a class="headerlink" href="#scientific-visualization-python-matplotlib" title="Link to this heading">#</a></h2>
<p>Nicolas P. Rougier</p>
<p>Licensed CC-BY-ND-SA-4.0</p>
<p>Tags: Python, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="github reference external" href="https://github.com/rougier/scientific-visualization-book">rougier/scientific-visualization-book</a></p>
<p><a class="reference external" href="https://inria.hal.science/hal-03427242/document">https://inria.hal.science/hal-03427242/document</a></p>
</section>
<hr class="docutils" />
<section id="scipy-cookbook">
<h2>Scipy Cookbook<a class="headerlink" href="#scipy-cookbook" title="Link to this heading">#</a></h2>
<p>Pauli Virtanen et al.</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This is a conversion and second life of SciPy Cookbook as a bunch of Ipython notebooks.</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scipy/scipy-cookbook">scipy/scipy-cookbook</a></p>
</section>
<hr class="docutils" />
<section id="setting-up-a-data-management-infrastructure-for-bioimaging">
<h2>Setting up a data management infrastructure for bioimaging<a class="headerlink" href="#setting-up-a-data-management-infrastructure-for-bioimaging" title="Link to this heading">#</a></h2>
<p>Susanne Kunis, Karen Bernhardt, Michael Hensel</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1515/hsz-2022-0304">https://doi.org/10.1515/hsz-2022-0304</a></p>
</section>
<hr class="docutils" />
<section id="setting-up-a-remote-desktop-to-use-napari-in-a-browser">
<h2>Setting up a remote desktop to use Napari in a browser<a class="headerlink" href="#setting-up-a-remote-desktop-to-use-napari-in-a-browser" title="Link to this heading">#</a></h2>
<p>Volker Hilsenstein</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/VolkerH/Jupyter-Napari-Desktop">VolkerH/Jupyter-Napari-Desktop</a></p>
</section>
<hr class="docutils" />
<section id="sharing-your-poster-on-figshare-a-community-guide-to-how-to-and-why">
<h2>Sharing Your Poster on Figshare: A Community Guide to How-To and Why<a class="headerlink" href="#sharing-your-poster-on-figshare-a-community-guide-to-how-to-and-why" title="Link to this heading">#</a></h2>
<p>Elisabeth Kugler</p>
<p>Tags: Sharing, Research Data Management, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/07/26/sharing-your-poster-on-figshare/">https://focalplane.biologists.com/2023/07/26/sharing-your-poster-on-figshare/</a></p>
</section>
<hr class="docutils" />
<section id="sharing-and-licensing-material">
<h2>Sharing and licensing material<a class="headerlink" href="#sharing-and-licensing-material" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Introduction to sharing resources online and licensing</p>
<p>Tags: Sharing, Research Data Management, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/10-519">https://f1000research.com/slides/10-519</a></p>
</section>
<hr class="docutils" />
<section id="sharing-research-data-with-zenodo">
<h2>Sharing research data with Zenodo<a class="headerlink" href="#sharing-research-data-with-zenodo" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Blog post about how to share data using <a class="reference external" href="http://zenodo.org">zenodo.org</a></p>
<p>Tags: Sharing, Research Data Management, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/02/15/sharing-research-data-with-zenodo/">https://focalplane.biologists.com/2023/02/15/sharing-research-data-with-zenodo/</a></p>
</section>
<hr class="docutils" />
<section id="simpleitk-notebooks">
<h2>SimpleITK-Notebooks<a class="headerlink" href="#simpleitk-notebooks" title="Link to this heading">#</a></h2>
<p>Ziv Yaniv et al.</p>
<p>Licensed APACHE-2.0</p>
<p>Jupyter notebooks for learning how to use SimpleITK</p>
<p>Tags: Bioimage Analysis, Simpleitk, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/InsightSoftwareConsortium/SimpleITK-Notebooks">InsightSoftwareConsortium/SimpleITK-Notebooks</a></p>
</section>
<hr class="docutils" />
<section id="slides-about-flute-a-python-gui-for-interactive-phasor-analysis-of-flim-data">
<h2>Slides about FLUTE: a Python GUI for interactive phasor analysis of FLIM data<a class="headerlink" href="#slides-about-flute-a-python-gui-for-interactive-phasor-analysis-of-flim-data" title="Link to this heading">#</a></h2>
<p>Chiara Stringari</p>
<p>Published 2024-03-19</p>
<p>Licensed CC-BY-4.0</p>
<p>This presentation introduces the open source software to analyze FLIM data:
FLUTE – (F)luorescence (L)ifetime (U)ltima(T)e (E)xplorer:
a Python GUI for interactive phasor analysis of FLIM data
 
The software is available on GitHub: <a class="github reference external" href="https://github.com/LaboratoryOpticsBiosciences/FLUTE">LaboratoryOpticsBiosciences/FLUTE</a>
and it is published on Biological imaging Journal: Gottlieb, D., Asadipour, B., Kostina, P., Ung, T., &amp; Stringari, C. (2023). FLUTE: A Python GUI for interactive phasor analysis of FLIM data. Biological Imaging, 1-22. doi:10.1017/S2633903X23000211
The lecture was part of the short talks on community developed FLIM-software at the German BioImaging workshop on FLIM in Munich.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/10839310">https://zenodo.org/records/10839310</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10839310">https://doi.org/10.5281/zenodo.10839310</a></p>
</section>
<hr class="docutils" />
<section id="so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten">
<h2>So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten<a class="headerlink" href="#so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten" title="Link to this heading">#</a></h2>
<p>Pia Voigt</p>
<p>Published 2024-05-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Der Umgang mit personenbezogenen Daten stellt Forschende oft vor rechtliche Herausforderungen: Unter welchen Bedingungen dürfen personenbezogene Daten verarbeitet werden? Welche Voraussetzungen müssen erfüllt sein und welche Strategien können angewendet werden, um Daten sicher speichern, verarbeiten, teilen und aufbewahren zu können? Mit Hilfe dieses Foliensatzes erhalten Sie Einblicke in datenschutzrechtliche Aspekte beim Umgang mit Ihren Forschungsdaten. </p>
<p>Tags: Research Data Management, Data Protection, FAIR-Principles, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/11396199">https://zenodo.org/records/11396199</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11396199">https://doi.org/10.5281/zenodo.11396199</a></p>
</section>
<hr class="docutils" />
<section id="software-citation-with-citation-cff">
<h2>Software Citation with CITATION.cff<a class="headerlink" href="#software-citation-with-citation-cff" title="Link to this heading">#</a></h2>
<p>How to make your Github repository citable by adding a citation.cff file.</p>
<p>Tags: Sharing, Citing, Research Data Management, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="reference external" href="https://the-turing-way.netlify.app/communication/citable/citable-cff.html">https://the-turing-way.netlify.app/communication/citable/citable-cff.html</a></p>
</section>
<hr class="docutils" />
<section id="source-control-using-git-and-github">
<h2>Source Control Using Git and GitHub<a class="headerlink" href="#source-control-using-git-and-github" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/JaneliaSciComp/2020AprilGitCourse">JaneliaSciComp/2020AprilGitCourse</a></p>
</section>
<hr class="docutils" />
<section id="structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data">
<h2>Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data<a class="headerlink" href="#structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data" title="Link to this heading">#</a></h2>
<p>Sarah Weischer, Jens Wendt, Thomas Zobel</p>
<p>Published 2022-07-12</p>
<p>Licensed CC-BY-4.0</p>
<p>Provides an overview of contexts, frameworks, and models from the world of bioimage data as well as metadata. Visualizes the techniques for structuring this data as Linked Data. (Walkthrough Video: <a class="reference external" href="https://doi.org/10.5281/zenodo.7018928">https://doi.org/10.5281/zenodo.7018928</a> )</p>
<p>Content:</p>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>Types of metadata
Data formats
Data Models Microscopy Data
Tools to edit/gather metadata
ISA Framework
FDO Framework
Ontology
RDF
JSON-LD
SPARQL
Knowledge Graph
Linked Data
Smart Data
...
</pre></div>
</div>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/7018750">https://zenodo.org/records/7018750</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.7018750">https://doi.org/10.5281/zenodo.7018750</a></p>
</section>
<hr class="docutils" />
<section id="sustainable-data-stewardship">
<h2>Sustainable Data Stewardship<a class="headerlink" href="#sustainable-data-stewardship" title="Link to this heading">#</a></h2>
<p>Stefano Della Chiesa</p>
<p>Published 2024-03-25</p>
<p>Licensed CC-BY-4.0</p>
<p>These slides were presented at the 2. SaxFDM-Beratungsstammtisch and delve into the strategic integration of Research Data Management (RDM) within research organizations. The Leibniz IOER presented an insightful overview of RDM activities and approaches, emphasizing the criticality of embedding RDM strategically within research institutions. The presentation showcases some best practices in RDM implementation through practical examples, offering valuable insights for optimizing data stewardship processes.</p>
<p>Tags: Research Data Management, Data Stewardship, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10942559">https://zenodo.org/records/10942559</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10942559">https://doi.org/10.5281/zenodo.10942559</a></p>
</section>
<hr class="docutils" />
<section id="teaching-bioimage-analysis-with-python">
<h2>Teaching Bioimage Analysis with Python<a class="headerlink" href="#teaching-bioimage-analysis-with-python" title="Link to this heading">#</a></h2>
<p>Rafael Camacho</p>
<p>Licensed MIT</p>
<p>Tags: Python, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/CamachoDejay/teaching-bioimage-analysis-python">CamachoDejay/teaching-bioimage-analysis-python</a></p>
</section>
<hr class="docutils" />
<section id="teaching-imagej-fiji">
<h2>Teaching ImageJ FIJI<a class="headerlink" href="#teaching-imagej-fiji" title="Link to this heading">#</a></h2>
<p>Rafael Camacho</p>
<p>Licensed MIT</p>
<p>Tags: Fiji, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Tutorial</p>
<p><a class="github reference external" href="https://github.com/CamachoDejay/Teaching-ImageJ-FIJI">CamachoDejay/Teaching-ImageJ-FIJI</a></p>
</section>
<hr class="docutils" />
<section id="ten-simple-rules-for-making-training-materials-fair">
<h2>Ten simple rules for making training materials FAIR<a class="headerlink" href="#ten-simple-rules-for-making-training-materials-fair" title="Link to this heading">#</a></h2>
<p>Leyla Garcia, Bérénice Batut, Melissa L. Burke, Mateusz Kuzak, Fotis Psomopoulos, et al.</p>
<p>Published 2020-05-21</p>
<p>Licensed CC-BY-4.0</p>
<p>The authors offer trainers some simple rules, to help make their training materials FAIR, enabling others to find, (re)use, and adapt them.</p>
<p>Tags: Metadata, Bioinformatics, FAIR-Principles, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007854">https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007854</a></p>
</section>
<hr class="docutils" />
<section id="the-digital-cell-cell-biology-as-a-data-science">
<h2>The Digital Cell: Cell Biology as a Data Science<a class="headerlink" href="#the-digital-cell-cell-biology-as-a-data-science" title="Link to this heading">#</a></h2>
<p>Stephen Royle</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://cshlpress.com/default.tpl?cart=1700309488232283050&amp;amp;fromlink=T&amp;amp;linkaction=full&amp;amp;linksortby=oop_title&amp;amp;--eqSKUdatarq=1282">https://cshlpress.com/default.tpl?cart=1700309488232283050&amp;fromlink=T&amp;linkaction=full&amp;linksortby=oop_title&amp;–eqSKUdatarq=1282</a></p>
</section>
<hr class="docutils" />
<section id="the-turing-way-guide-for-reproducible-research">
<h2>The Turing Way: Guide for reproducible research<a class="headerlink" href="#the-turing-way-guide-for-reproducible-research" title="Link to this heading">#</a></h2>
<p>Licensed [‘CC-BY-4.0’, ‘MIT’]</p>
<p>A guide which covers topics related to skills, tools and best practices for research reproducibility.</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://the-turing-way.netlify.app/reproducible-research/reproducible-research">https://the-turing-way.netlify.app/reproducible-research/reproducible-research</a></p>
</section>
<hr class="docutils" />
<section id="the-crucial-role-of-bioimage-analysts-in-scientific-research-and-publication">
<h2>The crucial role of bioimage analysts in scientific research and publication<a class="headerlink" href="#the-crucial-role-of-bioimage-analysts-in-scientific-research-and-publication" title="Link to this heading">#</a></h2>
<p>Beth A. Cimini, Peter Bankhead, Rocco D’ Antuono, Elnaz Fazeli, Julia Fernandez-Rondriguez, Caterina Fuster-Barcelo, Robert Haase, Helena Klara Jambor, Martin L. Jones, Florian Jug, Anna H. Klemm, Anna Kreshuk, Stefania Marcotti, Gabriel G. Martins, Sara Mc Ardle, Kota Miura, Arrate Muñoz-Barrutia, Laura C. Murphy, Michael S. Nelson, Simon F. Nørrelykke, Perrine Paul-Gilloteaux, Thomas Pengo, Joanna W. Pylvänäinen, Lior Pytowski, Arianna Ravera, Annika Reinke, Yousr Rekik, Caterina Strambio-De-Castillia, Daniel Thédié, Virginie Uhlmann, Oliver Umney, Laura Wiggins, Kevin W. Eliceiri</p>
<p>Published 2024-10-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Bioimage analysis (BIA), a crucial discipline in biological research, overcomes the limitations of subjective analysis in microscopy through the creation and application of quantitative and reproducible methods. The establishment of dedicated BIA support within academic institutions is vital to improving research quality and efficiency and can significantly advance scientific discovery. However, a lack of training resources, limited career paths and insufficient recognition of the contributions made by bioimage analysts prevent the full realization of this potential. This Perspective – the result of the recent The Company of Biologists Workshop ‘Effectively Communicating Bioimage Analysis’, which aimed to summarize the global BIA landscape, categorize obstacles and offer possible solutions – proposes strategies to bring about a cultural shift towards recognizing the value of BIA by standardizing tools, improving training and encouraging formal credit for contributions</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://journals.biologists.com/jcs/article/137/20/jcs262322/362545/The-crucial-role-of-bioimage-analysts-in">https://journals.biologists.com/jcs/article/137/20/jcs262322/362545/The-crucial-role-of-bioimage-analysts-in</a></p>
</section>
<hr class="docutils" />
<section id="things-that-shouldn-t-be-in-file-names-for-1000-alex">
<h2>Things that shouldn’t be in file names for $1000 Alex<a class="headerlink" href="#things-that-shouldn-t-be-in-file-names-for-1000-alex" title="Link to this heading">#</a></h2>
<p>Joseph Scott</p>
<p>Published 2007-02-12</p>
<p>Licensed UNKNOWN</p>
<p>In this blog post, the author emphasizes problematic characters (e.g., spaces, slashes, colons) that should be avoided in filenames to ensure cross-platform compatibility across operating systems like Windows, macOS, and Linux, emphasizing the chaos caused by differing filesystem rules and naming conventions.</p>
<p>Tags: Reseach Data Management, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://blog.josephscott.org/2007/02/12/things-that-shouldnt-be-in-file-names-for-1000-alex/">https://blog.josephscott.org/2007/02/12/things-that-shouldnt-be-in-file-names-for-1000-alex/</a></p>
</section>
<hr class="docutils" />
<section id="thinking-data-management-on-different-scales">
<h2>Thinking data management on different scales<a class="headerlink" href="#thinking-data-management-on-different-scales" title="Link to this heading">#</a></h2>
<p>Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation given at PoL BioImage Analysis Symposium Dresden 2023</p>
<p>Tags: Research Data Management, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8329305">https://zenodo.org/doi/10.5281/zenodo.8329305</a></p>
</section>
<hr class="docutils" />
<section id="towards-open-and-standardised-imaging-data-an-introduction-to-bio-formats-ome-tiff-and-ome-zarr">
<h2>Towards open and standardised imaging data: an introduction to Bio-Formats, OME-TIFF, and OME-Zarr<a class="headerlink" href="#towards-open-and-standardised-imaging-data-an-introduction-to-bio-formats-ome-tiff-and-ome-zarr" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2025-05-28</p>
<p>Licensed CC-BY-4.0</p>
<p><a class="reference external" href="https://www.ebi.ac.uk/training/events/towards-open-and-standardised-imaging-data-introduction-bio-formats-ome-tiff-and-ome-zarr/">https://www.ebi.ac.uk/training/events/towards-open-and-standardised-imaging-data-introduction-bio-formats-ome-tiff-and-ome-zarr/</a>
Microscopy and bioimaging technologies are fundamental tools for exploring biological systems, generating large, multidimensional datasets rich in experimental detail. However, the bioimaging community has historically faced major challenges around data handling: vendor-specific proprietary formats, fragmented metadata storage, and increasingly large dataset sizes that outstrip traditional storage and computing solutions.
In this webinar, key open technologies developed by the Open Microscopy Environment (OME) to address these challenges were presented. Specifically, the Bio-Formats library for accessing diverse proprietary file formats, the OME-TIFF standard for archival data storage, and the OME-Zarr format for cloud-native, scalable bioimaging workflows were presented.</p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15479606">https://zenodo.org/records/15479606</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15479606">https://doi.org/10.5281/zenodo.15479606</a></p>
</section>
<hr class="docutils" />
<section id="tracking-theory-trackmate-and-mastodon">
<h2>Tracking Theory, TrackMate, and Mastodon<a class="headerlink" href="#tracking-theory-trackmate-and-mastodon" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Lecture slides of a session on Cell Tracking in Fiji</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_tracking_trackmate">https://git.mpi-cbg.de/rhaase/lecture_tracking_trackmate</a></p>
</section>
<hr class="docutils" />
<section id="tracking-in-napari">
<h2>Tracking in napari<a class="headerlink" href="#tracking-in-napari" title="Link to this heading">#</a></h2>
<p>Mara Lampert</p>
<p>Tags: Python, Napari, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Blog Post</p>
<p><a class="reference external" href="https://focalplane.biologists.com/2023/06/01/tracking-in-napari/">https://focalplane.biologists.com/2023/06/01/tracking-in-napari/</a></p>
</section>
<hr class="docutils" />
<section id="tracking-of-mitochondria-and-capturing-mitoflashes">
<h2>Tracking of mitochondria and capturing mitoflashes<a class="headerlink" href="#tracking-of-mitochondria-and-capturing-mitoflashes" title="Link to this heading">#</a></h2>
<p>Leonid Kostrykin, Diana Chiang Jurado</p>
<p>Published 2024-11-20</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Bioinformatics, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Workflow, Tutorial</p>
<p><a class="reference external" href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/detection-of-mitoflashes/tutorial.html#tracking-of-mitochondria-and-capturing-mitoflashes">https://training.galaxyproject.org/training-material/topics/imaging/tutorials/detection-of-mitoflashes/tutorial.html#tracking-of-mitochondria-and-capturing-mitoflashes</a></p>
</section>
<hr class="docutils" />
<section id="train-the-trainer-concept-on-research-data-management">
<h2>Train-the-Trainer Concept on Research Data Management<a class="headerlink" href="#train-the-trainer-concept-on-research-data-management" title="Link to this heading">#</a></h2>
<p>Katarzyna Biernacka, Maik Bierwirth, Petra Buchholz, Dominika Dolzycka, Kerstin Helbig, Janna Neumann, Carolin Odebrecht, Cord Wiljes, Ulrike Wuttke</p>
<p>Published 2020-11-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Within the project FDMentor, a German Train-the-Trainer Programme on Research Data Management (RDM) was developed and piloted in a series of workshops. The topics cover many aspects of research data management, such as data management plans and the publication of research data, as well as didactic units on learning concepts, workshop design and a range of didactic methods.</p>
<p>After the end of the project, the concept was supplemented and updated by members of the Sub-Working Group Training/Further Education (UAG Schulungen/Fortbildungen) of the DINI/nestor Working Group Research Data (DINI/nestor-AG Forschungsdaten). The newly published English version of the Train-the-Trainer Concept contains the translated concept, the materials and all methods of the Train-the-Trainer Programme. Furthermore, additional English references and materials complement this version.</p>
<p>Tags: Research Data Management, Include In Dalia</p>
<p>Content type: Book</p>
<p><a class="reference external" href="https://zenodo.org/record/4071471">https://zenodo.org/record/4071471</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.4071471">https://doi.org/10.5281/zenodo.4071471</a></p>
</section>
<hr class="docutils" />
<section id="training-computational-skills-in-the-age-of-ai">
<h2>Training Computational Skills in the Age of AI<a class="headerlink" href="#training-computational-skills-in-the-age-of-ai" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-11-06</p>
<p>Licensed CC-BY-4.0</p>
<p>Artificial intelligence (AI) and large language models (LLMs) are changing the way we use computers in science. This slide deck introduces ways for using AI and LLMs for making training materials and for exchanging knowledge about how to use AI in joint discussions between humans and LLM-based AI-systems.</p>
<p>Tags: Nfdi4Bioimage, Artificial Intelligence, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14043615">https://zenodo.org/records/14043615</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14043615">https://doi.org/10.5281/zenodo.14043615</a></p>
</section>
<hr class="docutils" />
<section id="training-deep-learning-models-for-vision-compact-course">
<h2>Training Deep Learning Models for Vision - Compact Course<a class="headerlink" href="#training-deep-learning-models-for-vision-compact-course" title="Link to this heading">#</a></h2>
<p>Constantin Pape, Adrian Wolny</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p><a class="github reference external" href="https://github.com/constantinpape/training-deep-learning-models-for-vison">constantinpape/training-deep-learning-models-for-vison</a></p>
</section>
<hr class="docutils" />
<section id="training-concepts-in-research-data-management-and-data-science-with-the-focus-on-health-research">
<h2>Training concepts in research data management and data science with the focus on health research<a class="headerlink" href="#training-concepts-in-research-data-management-and-data-science-with-the-focus-on-health-research" title="Link to this heading">#</a></h2>
<p>Jens Dierkes, Julia Fürst, Tanja Hörner, Sebastian Klammt, Birte Lindstädt, Iris Pigeot, Katja Restel, Carsten Oliver Schmidt, Dagmar Waltemath, Atinkut Zeleke</p>
<p>Published 2023-08-22</p>
<p>Licensed CC-BY-4.0</p>
<p>This handbook provides an overview of Research Data Management (RDM) and FAIR principles, with a focus on practical implementation and training for master students, doctoral researchers, and professionals in biomedical sciences. The handbook is a living document that will be updated iteratively to address the specific needs and requirements of the target group, with a focus on the NFDI4Health domain.</p>
<p>Tags: FAIR-Principles, Research Data Management, Include In Dalia</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://repository.publisso.de/resource/frl:6453768">https://repository.publisso.de/resource/frl:6453768</a></p>
</section>
<hr class="docutils" />
<section id="ultrack-i2k-2024-workshop-materials">
<h2>Ultrack I2K 2024 Workshop Materials<a class="headerlink" href="#ultrack-i2k-2024-workshop-materials" title="Link to this heading">#</a></h2>
<p>Jordão Bragantini, Teun Huijben</p>
<p>Licensed BSD3-CLAUSE</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Workshop, Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/royerlab/ultrack-i2k2024">royerlab/ultrack-i2k2024</a></p>
<p><a class="reference external" href="https://royerlab.github.io/ultrack-i2k2024/">https://royerlab.github.io/ultrack-i2k2024/</a></p>
</section>
<hr class="docutils" />
<section id="user-friendly-image-metadata-annotation-tool-workflow-for-omero">
<h2>User friendly Image metadata annotation tool/workflow for OMERO<a class="headerlink" href="#user-friendly-image-metadata-annotation-tool-workflow-for-omero" title="Link to this heading">#</a></h2>
<p>Jens Wendt</p>
<p>Tags: Metadata, Workflow, OMERO, Include In Dalia</p>
<p>Content type: Forum Post</p>
<p><a class="reference external" href="https://forum.image.sc/t/user-friendly-image-metadata-annotation-tool-workflow-for-omero/87925/1">https://forum.image.sc/t/user-friendly-image-metadata-annotation-tool-workflow-for-omero/87925/1</a></p>
</section>
<hr class="docutils" />
<section id="virtual-i2k-2024-multiview-stitcher">
<h2>Virtual-I2K-2024-multiview-stitcher<a class="headerlink" href="#virtual-i2k-2024-multiview-stitcher" title="Link to this heading">#</a></h2>
<p>Marvin Albert</p>
<p>Published 2024-10-30T07:38:11+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Repository accompanying the multiview-stitcher tutorial for Virtual I2K 2024</p>
<p>Tags: Big Data, Bioimageanalysis, Include In Dalia</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/m-albert/Virtual-I2K-2024-multiview-stitcher">m-albert/Virtual-I2K-2024-multiview-stitcher</a></p>
</section>
<hr class="docutils" />
<section id="vision-language-models-for-bio-image-data-science">
<h2>Vision Language Models for Bio-image Data Science<a class="headerlink" href="#vision-language-models-for-bio-image-data-science" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2025-06-25</p>
<p>Licensed CC-BY-4.0</p>
<p>In this talk, I demonstrate potential use-cases for vision-language models (VLM) in bio-image data science, focusing on how to analyse microscopy image data. It covers these use-cases:</p>
<p>cell counting
bounding-box segmentation
image descriptions
VLMs guessing which algorithm to use for processing
Data analysis code generation
Answering github issues </p>
<p>The talk also points at a number of VLM-based open-source tools which start reshaping the scientific bio-image data science domain:</p>
<p>bia-bob
unprompted
git-bob
napari-chatgpt
<a class="reference external" href="http://bioimage.io">bioimage.io</a> chatbot</p>
<p>Tags: Nfdi4Bioimage, Bioimage Analysis, Artificial Intelligence, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15735577">https://zenodo.org/records/15735577</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15735577">https://doi.org/10.5281/zenodo.15735577</a></p>
</section>
<hr class="docutils" />
<section id="what-not-to-do-when-creating-a-data-management-plan-dmp">
<h2>WHAT NOT TO DO WHEN CREATING A DATA MANAGEMENT PLAN (DMP)<a class="headerlink" href="#what-not-to-do-when-creating-a-data-management-plan-dmp" title="Link to this heading">#</a></h2>
<p>Georgia Koutentaki, Martin Schätz, Jan Vališ</p>
<p>Published 2025-05-14</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15402904">https://zenodo.org/records/15402904</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15402904">https://doi.org/10.5281/zenodo.15402904</a></p>
</section>
<hr class="docutils" />
<section id="what-is-open-data">
<h2>What is Open Data?<a class="headerlink" href="#what-is-open-data" title="Link to this heading">#</a></h2>
<p>Daniel Dietrich, Jonathan Gray, Tim McNamara, Antti Poikola, Rufus Pollock, et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>This handbook is about open data but what exactly is it? In particular what makes open data open, and what sorts of data are we talking about?</p>
<p>Tags: Open Science, Include In Dalia</p>
<p>Content type: Collection</p>
<p><a class="reference external" href="http://opendatahandbook.org/guide/en/what-is-open-data/">http://opendatahandbook.org/guide/en/what-is-open-data/</a></p>
</section>
<hr class="docutils" />
<section id="when-data-doesn-t-fit">
<h2>When Data Doesn’t Fit<a class="headerlink" href="#when-data-doesn-t-fit" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2025-09-11</p>
<p>Licensed CC-BY-4.0</p>
<p>Presented at “International Symposium on Integrative Bioinformatics”, Gatersleben Research Conference Series from September 10–12, 2025,  <a class="reference external" href="https://meetings.ipk-gatersleben.de/grc-ib2025/">https://meetings.ipk-gatersleben.de/grc-ib2025/</a></p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/17087096">https://zenodo.org/records/17087096</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.17087096">https://doi.org/10.5281/zenodo.17087096</a></p>
</section>
<hr class="docutils" />
<section id="who-you-gonna-call-data-stewards-to-the-rescue">
<h2>Who you gonna call? - Data Stewards to the rescue<a class="headerlink" href="#who-you-gonna-call-data-stewards-to-the-rescue" title="Link to this heading">#</a></h2>
<p>Vanessa Aphaia Fiona Fuchs, Jens Wendt, Maximilian Müller, Mohsen Ahmadi, Riccardo Massei, Cornelia Wetzker</p>
<p>Licensed CC-BY-4.0</p>
<p>The Data Steward Team of the NFDI4BIOIMAGE consortium presents themselves and the services (including the Helpdesk) that we offer.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Data Stewardship, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10730423">https://zenodo.org/doi/10.5281/zenodo.10730423</a></p>
</section>
<hr class="docutils" />
<section id="workflow-for-user-introduction-into-microscopy-omero-and-data-management-at-center-for-advanced-imaging">
<h2>Workflow for user introduction into microscopy, OMERO and data management at Center for Advanced imaging<a class="headerlink" href="#workflow-for-user-introduction-into-microscopy-omero-and-data-management-at-center-for-advanced-imaging" title="Link to this heading">#</a></h2>
<p>Ksenia Krooß, Fuchs, Vanessa Aphaia Fiona, Tom Boissonnet, Stefanie Weidtkamp-Peters</p>
<p>Published 2025-03-07</p>
<p>Licensed CC-BY-4.0</p>
<p>At the Center for Advanced Imaging (CAi) at the Heinrich Heine University Düsseldorf, Germany, we have established a workflow to guide users through all aspects of bioimaging. The process begins with an initial consultation with our imaging specialists regarding microscopy techniques for their specific project. Users then receive training in microscope operation, ensuring they can handle the equipment effectively. If needed, our specialists also provide support in image analysis. Next, we introduce users to OMERO, highlighting its features and the advantages of using a bioimage data management system. They are then trained to structure and annotate their data within OMERO according to the Recommended Metadata for Biological Images (REMBI), taking their specific research topics into account. As users prepare for data publication, we assist with data organization and repository uploads. Our goal is to educate researchers in managing bioimage data throughout its entire lifecycle, with a strong emphasis on the FAIR (findable, accessible, interoperable, reusable) principles.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14988921">https://zenodo.org/records/14988921</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14988921">https://doi.org/10.5281/zenodo.14988921</a></p>
</section>
<hr class="docutils" />
<section id="working-with-objects-in-2d-and-3d">
<h2>Working with objects in 2D and 3D<a class="headerlink" href="#working-with-objects-in-2d-and-3d" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_working_with_objects_in_2d_and_3d">https://git.mpi-cbg.de/rhaase/lecture_working_with_objects_in_2d_and_3d</a></p>
</section>
<hr class="docutils" />
<section id="working-with-pixels">
<h2>Working with pixels<a class="headerlink" href="#working-with-pixels" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Neubias, Imagej, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://git.mpi-cbg.de/rhaase/lecture_working_with_pixels">https://git.mpi-cbg.de/rhaase/lecture_working_with_pixels</a></p>
</section>
<hr class="docutils" />
<section id="ymia-python-based-event-series-training-material">
<h2>YMIA - Python-Based Event Series Training Material<a class="headerlink" href="#ymia-python-based-event-series-training-material" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Robert Haase, ENicolay</p>
<p>Published None</p>
<p>Licensed MIT</p>
<p>This repository offer access to teaching material and useful resources for the YMIA - Python-Based Event Series.</p>
<p>Tags: Python, Artifical Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository, Slides</p>
<p><a class="github reference external" href="https://github.com/rmassei/ymia_python_event_series_material">rmassei/ymia_python_event_series_material</a></p>
</section>
<hr class="docutils" />
<section id="zen-python-workshop">
<h2>ZEN &amp; Python workshop<a class="headerlink" href="#zen-python-workshop" title="Link to this heading">#</a></h2>
<p>Licensed GPL-3.0</p>
<p>Tags: Python, Napari, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Collection, Notebook</p>
<p><a class="github reference external" href="https://github.com/zeissmicroscopy/ZEN_Python_OAD_workshop">zeissmicroscopy/ZEN_Python_OAD_workshop</a></p>
</section>
<hr class="docutils" />
<section id="zidas-2020-introduction-to-deep-learning">
<h2>ZIDAS 2020 Introduction to Deep Learning<a class="headerlink" href="#zidas-2020-introduction-to-deep-learning" title="Link to this heading">#</a></h2>
<p>Estibaliz Gómez-de-Mariscal</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Artificial Intelligence, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="github reference external" href="https://github.com/esgomezm/zidas2020_intro_DL">esgomezm/zidas2020_intro_DL</a></p>
</section>
<hr class="docutils" />
<section id="cidas-scalable-strategies-for-a-next-generation-of-fair-bioimaging">
<h2>[CIDAS] Scalable strategies for a next-generation of FAIR bioimaging<a class="headerlink" href="#cidas-scalable-strategies-for-a-next-generation-of-fair-bioimaging" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2025-01-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Talk given at Georg-August-Universität Göttingen Campus Institute Data Science23rd January 2025
<a class="reference external" href="https://www.uni-goettingen.de/en/653203.html">https://www.uni-goettingen.de/en/653203.html</a></p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14845059">https://zenodo.org/records/14845059</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14845059">https://doi.org/10.5281/zenodo.14845059</a></p>
</section>
<hr class="docutils" />
<section id="cmcb-scalable-strategies-for-a-next-generation-of-fair-bioimaging">
<h2>[CMCB] Scalable strategies for a next-generation of FAIR bioimaging<a class="headerlink" href="#cmcb-scalable-strategies-for-a-next-generation-of-fair-bioimaging" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2025-01-16</p>
<p>Licensed CC-BY-4.0</p>
<p>CMCB LIFE SCIENCES SEMINARSTechnische Universität Dresden16th January 2025
<a class="reference external" href="https://tu-dresden.de/cmcb/crtd/news-termine/termine/cmcb-life-sciences-seminar-josh-moore-german-bioimaging-e-v-society-for-microscopy-and-image-analysis-constance">https://tu-dresden.de/cmcb/crtd/news-termine/termine/cmcb-life-sciences-seminar-josh-moore-german-bioimaging-e-v-society-for-microscopy-and-image-analysis-constance</a>
 </p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14650434">https://zenodo.org/records/14650434</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14650434">https://doi.org/10.5281/zenodo.14650434</a></p>
</section>
<hr class="docutils" />
<section id="elmi-2024-ai-s-dirty-little-secret-without">
<h2>[ELMI 2024]  AI’s Dirty Little Secret: Without<a class="headerlink" href="#elmi-2024-ai-s-dirty-little-secret-without" title="Link to this heading">#</a></h2>
<p>FAIR Data, It’s Just Fancy Math</p>
<p>Josh Moore, Susanne Kunis</p>
<p>Published 2024-05-21</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presented at the European Light Microscopy Initiative meeting in Liverpool (<a class="reference external" href="https://www.elmi2024.org/">https://www.elmi2024.org/</a>)</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/11235513">https://zenodo.org/records/11235513</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11235513">https://doi.org/10.5281/zenodo.11235513</a></p>
</section>
<hr class="docutils" />
<section id="elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">
<h2>[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math<a class="headerlink" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math" title="Link to this heading">#</a></h2>
<p>Josh Moore, Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presented at the European Light Microscopy Initiative meeting in Liverpool (<a class="reference external" href="https://www.elmi2024.org/">https://www.elmi2024.org/</a>)</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.11235512">https://zenodo.org/doi/10.5281/zenodo.11235512</a></p>
</section>
<hr class="docutils" />
<section id="elmi2025-bridging-communities-with-ome-zarr">
<h2>[ELMI2025] Bridging communities with OME-Zarr<a class="headerlink" href="#elmi2025-bridging-communities-with-ome-zarr" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Aastha Mathur, Josh Moore</p>
<p>Published 2025-06-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Presented at ELMI2025
 
<a class="reference external" href="https://www.embl.org/about/info/course-and-conference-office/events/elmi2025/">https://www.embl.org/about/info/course-and-conference-office/events/elmi2025/</a></p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15393592">https://zenodo.org/records/15393592</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15393592">https://doi.org/10.5281/zenodo.15393592</a></p>
</section>
<hr class="docutils" />
<section id="elmi2025-the-road-to-ome-zarr-1-0">
<h2>[ELMI2025] The Road to OME-Zarr 1.0<a class="headerlink" href="#elmi2025-the-road-to-ome-zarr-1-0" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2025-06-05</p>
<p>Licensed CC-BY-4.0</p>
<p>Presented at <a class="reference external" href="https://www.embl.org/about/info/course-and-conference-office/events/elmi2025/">https://www.embl.org/about/info/course-and-conference-office/events/elmi2025/</a>
 
Abstract
For over 20 years, the Open Microscopy Environment (OME) has developed tools and specifications to support bioimaging data sharing. Technologies such as Bio-Formats, OMERO, and OME-TIFF have helped researchers manage the growing size, complexity, and acquisition rates of imaging datasets. However, with increasing mandates for research data management, such as the Nelson memo in the United States, and the shift toward cloud-native workflows, the bioimaging community faces new challenges in ensuring scalable and FAIR data infrastructure.
In 2024, following expanding community engagement, the focus of the Next-Generation File Format (NGFF) community was on building consensus around a Request for Comments (RFC) process. This collaborative effort has laid the foundation for future refinements and wider adoption. In parallel, we hosted the “OME2024 NGFF Challenge,” bringing together over the course of just four months hundreds of terabytes of data in a first prototype of federated image hosting, showcasing the power of OME-Zarr for handling large-scale, distributed datasets.
In 2025, we are set to take a major step toward a stable FAIR solution with OME-Zarr 1.0. This milestone marks a crucial phase towards an international standard, providing an open, cloud-optimized, and scalable solution for handling terabyte- and petabyte-scale imaging data. The 1.0 release will introduce long-awaited transforms, enabling robust support for multimodal datasets, followed by collections and an extensibility mechanism to accommodate evolving scientific needs. These additions emphasize a solid foundation on which future capabilities can be built while providing the stability needed for broader adoption of the format. This presentation will outline the path to 1.0, including community-driven refinements, vendor engagement to ensure complete metadata representation, and alignment with global bioimaging initiatives. As imaging data continues to grow in scale and complexity, consensus-driven evolution of infrastructure will be key to ensuring a truly FAIR future for bioimaging.
 </p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15597856">https://zenodo.org/records/15597856</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15597856">https://doi.org/10.5281/zenodo.15597856</a></p>
</section>
<hr class="docutils" />
<section id="elmi2025-workshop-fair101-navigating-fair-data-from-principles-to-practice">
<h2>[ELMI2025] Workshop: FAIR101 - Navigating FAIR data from principles to practice<a class="headerlink" href="#elmi2025-workshop-fair101-navigating-fair-data-from-principles-to-practice" title="Link to this heading">#</a></h2>
<p>Isabel Kemmer, Euro-BioImaging ERIC</p>
<p>Published 2025-06-12</p>
<p>Licensed CC-BY-4.0</p>
<p> This workshop was held at the ELMI Meeting 2025 in Heidelberg (<a class="reference external" href="https://www.embl.org/about/info/course-and-conference-office/events/elmi2025/">https://www.embl.org/about/info/course-and-conference-office/events/elmi2025/</a>).
Abstract
FAIR 101 - Navigating FAIR data from principles to practice
Isabel Kemmer, Euro-BioImaging ERIC
This workshop will introduce the FAIR principles in the context of bioimaging data. Designed for researchers working across scales and technologies of biological and biomedical imaging, the session will address the unique challenges posed by complex, multidimensional bioimaging datasets. With the aim of providing simple yet impactful steps for a smooth start to the FAIR journey we will explore the features and benefits of FAIR data through interactive exercises and discussions - from metadata annotation and data management planning to repository selection. By the end of the workshop, you will feel more confident in applying the FAIR concepts and be prepared to improve your imaging workflows to make your precious data even more valuable.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15647102">https://zenodo.org/records/15647102</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15647102">https://doi.org/10.5281/zenodo.15647102</a></p>
</section>
<hr class="docutils" />
<section id="gbi-eoe-vii-five-or-ten-must-have-items-for-making-it-infrastructure-for-managing-bioimage-data">
<h2>[GBI EOE VII] Five (or ten) must-have items for making IT infrastructure for managing bioimage data<a class="headerlink" href="#gbi-eoe-vii-five-or-ten-must-have-items-for-making-it-infrastructure-for-managing-bioimage-data" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2024-05-26</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation made to the GBI Image Data Management Working Group during the 7th Exchange of Experience in Uruguay.</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/11318151">https://zenodo.org/records/11318151</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11318151">https://doi.org/10.5281/zenodo.11318151</a></p>
</section>
<hr class="docutils" />
<section id="i2k-scalable-strategies-for-a-next-generation-of-fair-bioimaging">
<h2>[I2K] Scalable strategies for a next-generation of FAIR bioimaging<a class="headerlink" href="#i2k-scalable-strategies-for-a-next-generation-of-fair-bioimaging" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2024-10-25</p>
<p>Licensed CC-BY-4.0</p>
<p>or, “OME-Zarr: ‘even a talk on formats [can be] interesting’”
Presented at <a class="reference external" href="https://events.humantechnopole.it/event/1/">https://events.humantechnopole.it/event/1/</a></p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13991322">https://zenodo.org/records/13991322</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13991322">https://doi.org/10.5281/zenodo.13991322</a></p>
</section>
<hr class="docutils" />
<section id="nfdi-tech-talk-cloud-based-image-science">
<h2>[NFDI Tech Talk] Cloud Based Image Science<a class="headerlink" href="#nfdi-tech-talk-cloud-based-image-science" title="Link to this heading">#</a></h2>
<p>Josh Moore, Yi Sun</p>
<p>Published 2025-06-02</p>
<p>Licensed CC-BY-4.0</p>
<p>Slides for the NFDI Tech Talk live streamed to <a class="reference external" href="https://www.youtube.com/live/bzfmE29S270">https://www.youtube.com/live/bzfmE29S270</a>
See <a class="reference external" href="http://nfdi.de/talks">http://nfdi.de/talks</a> for more information.</p>
<p>Tags: Nfdi4Bioimage, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15575379">https://zenodo.org/records/15575379</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15575379">https://doi.org/10.5281/zenodo.15575379</a></p>
</section>
<hr class="docutils" />
<section id="swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard">
<h2>[SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard<a class="headerlink" href="#swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard" title="Link to this heading">#</a></h2>
<p>Josh Moore, Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presented at Semantic Web Applications and Tools for Health Care and Life Sciences (SWAT4HCLS 2023), Feb 13–16, 2023, Basel, Switzerland. NFDI4BIOIMAGE is a newly established German consortium dedicated to the FAIR representation of biological imaging data. A key deliverable is the definition of a semantically-compatible FAIR image object integrating RDF metadata with web-compatible storage of large n-dimensional binary data in OME-Zarr. We invite feedback from and collaboration with other endeavors during the soon-to-begin 5 year funding period.</p>
<p>Tags: Research Data Management, FAIR-Principles, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.7928332">https://zenodo.org/doi/10.5281/zenodo.7928332</a></p>
</section>
<hr class="docutils" />
<section id="short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">
<h2>[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure<a class="headerlink" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure" title="Link to this heading">#</a></h2>
<p>Christian Schmidt</p>
<p>Licensed CC-BY-4.0</p>
<p>Short Talk about the NFDI4BIOIMAGE consortium presented at the RDM in (Bio-)Medicine Information Event on April 10th, 2024, organized C³RDM &amp; ZB MED.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage, Include In Dalia</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10939519">https://zenodo.org/doi/10.5281/zenodo.10939519</a></p>
</section>
<hr class="docutils" />
<section id="webinar-a-journey-to-fair-bioimage-data">
<h2>[Webinar] A journey to FAIR bioimage data<a class="headerlink" href="#webinar-a-journey-to-fair-bioimage-data" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters, Tom Boissonnet, Christian Schmidt</p>
<p>Published 2025-07-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation slides from an EMBL-EBI Webinar Talk within the webinar series:
“How to organise and share my imaging data? - Multimodal data management for marine biologists, environmental scientists and imaging specialists”
 
Abstract / Description
Bioimaging is a pervasive and indispensable methodological approach in the life and biomedical sciences. Due to the development of new technologies and the easier access to compute resources, bioimaging experiments have become a big data discipline, facing the same challenges as other omics technologies within the life sciences. However, to fully exploit the potential of bioimage data, it is necessary to make the data FAIR. In this webinar we will present viable solutions for storing, processing, analysing, and, first and foremost, sharing bioimaging data. We will introduce services provided to the scientific community, that are dealing with various aspects of the bioimage data life cycle such as:</p>
<ul class="simple">
<li><p>Where to get support for bioimage data management- Local bioimage data management: OMERO and beyond- Annotation of bioimage data: metadata, ontologies, REMBI etc- Linking your image data with experimental protocols and analysis results- Large data living in the cloud: ome.zarr- Publication of bioimage data
Who is this course for?
This webinar is suitable for marine biologists and environmental scientists collecting samples from the natural environment, generating, visualising, annotating and analysing large, multimodal datasets such as imaging data, and sharing their data by submitting them to public data repositories. The webinar will support you to set up an efficient data flow that is aligned with FAIR principles.
This event is part of a webinar series organised by the STANDFLOW project, an initiative supported by EMBL’s Planetary biology Transversal Theme. STANDFLOW is about a collaborative effort towards creating a standardised data management workflow. The project primarily utilises imaging data derived from samples collected through the TREC (Traversing European Coastlines) and the Roscoff Culture Collection. For details on all topics covered in this series and registration information, please visit the following link: How to organise and share my imaging data?: Multimodal data management for marine biologists, and environmental scientists and imaging specialists
Outcomes
By the end of the webinar you will be able to: </p></li>
</ul>
<p>Find resources and support for bioimage data management
Get started with bioimage data annotation
Identify the dos and don’ts for bioimage data publication</p>
<p> 
(taken from: <a class="reference external" href="https://www.ebi.ac.uk/training/events/journey-fair-bioimage-data/">https://www.ebi.ac.uk/training/events/journey-fair-bioimage-data/</a>)</p>
<p>Tags: Nfdi4Bioimage, Fair Principles, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15796252">https://zenodo.org/records/15796252</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15796252">https://doi.org/10.5281/zenodo.15796252</a></p>
</section>
<hr class="docutils" />
<section id="workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg">
<h2>[Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg<a class="headerlink" href="#workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg" title="Link to this heading">#</a></h2>
<p>Tom Boissonnet, Bettina Hagen, Susanne Kunis, Christian Schmidt, Stefanie Weidtkamp-Peters</p>
<p>Published 2024-10-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Fit for OMERO: How imaging facilities and IT departments work together to enable RDM for bioimaging
Description:
Research data management (RDM) in bioimaging is challenging because of large file sizes, heterogeneous file formats and the variability of imaging methods. The image data management system OMERO (OME Remote Objects) allows for centralized and secure storage, organization, annotation, and interrogation of microscopy data by researchers. It is an internationally well-supported open-source software tool that has become one of the best-known image data management tools among bioimaging scientists. Nevertheless, the de novo setup of OMERO at an institute is a multi-stakeholder process that demands time, funds, organization and iterative implementation. In this workshop, participants learn how to begin setting up OMERO-based image data management at their institution. The topics include:</p>
<p>Stakeholder identification at the university / research institute
Process management, time line expectations, and resources planning
Learning about each other‘s perspectives on chances and challenges for RDM
Funding opportunities and strategies for IT and imaging core facilities
Hands-on: Setting up an OMERO server in a virtual machine environment</p>
<p>Target audience:
This workshop was directed at universities and research institutions who consider or plan to implement OMERO, or are in an early phase of implementation. This workshop was intended for teams from IT departments and imaging facilities to participate together with one person from the IT department, and one person from the imaging core facility at the same institution.
The trainers:</p>
<p>Prof. Dr. Stefanie Weidtkamp-Peters (Imaging Core Facility Head, Center for Advanced Imaging, Heinrich Heine University of Düsseldorf)
Dr. Susanne Kunis (Software architect, OMERO administrator, metadata specialist, University of Osnabrück)
Dr. Tom Boissonnet (OMERO admin and image metadata specialist, Center for Advanced Imaging, Heinrich Heine University of Düsseldorf)
Dr. Bettina Hagen (IT Administration and service specialist, Max Planck Institute for the Biology of Ageing, Cologne) 
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center (DKFZ), Heidelberg)</p>
<p>Time and place
The format was a two-day, in-person workshop (October 16-17, 2024). Location: Heidelberg, Germany
Workshop learning goals</p>
<p>Learn the steps to establish a local RDM environment fit for bioimaging data
Create a network of IT experts and bioimaging specialists for bioimage RDM across institutions
Establish a stakeholder process management for installing OMERO-based RDM
Learn from each other, leverage different expertise
Learn how to train users, establish sustainability strategies, and foster FAIR RDM for bioimaging at your institution</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/14013026">https://zenodo.org/records/14013026</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14013026">https://doi.org/10.5281/zenodo.14013026</a></p>
</section>
<hr class="docutils" />
<section id="workshop-bioimage-data-management-and-analysis-with-omero">
<h2>[Workshop] Bioimage data management and analysis with OMERO<a class="headerlink" href="#workshop-bioimage-data-management-and-analysis-with-omero" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Michele Bortolomeazzi, Christian Schmidt</p>
<p>Published 2024-05-13</p>
<p>Licensed CC-BY-4.0</p>
<p>Here we share the material used in a workshop held on May 13th, 2024, at the German Cancer Research Center in Heidelberg (on-premise)
Description:Microscopy experiments generate information-rich, multi-dimensional data, allowing us to investigate biological processes at high spatial and temporal resolution. Image processing and analysis is a standard procedure to retrieve quantitative information from biological imaging. Due to the complex nature of bioimaging files that often come in proprietary formats, it can be challenging to organize, structure, and annotate bioimaging data throughout a project. Data often needs to be moved between collaboration partners, transformed into open formats, processed with a variety of software tools, and exported to smaller-sized images for presentation. The path from image acquisition to final publication figures with quantitative results must be documented and reproducible.
In this workshop, participants learn how to use OMERO to organize their data and enrich the bioimage data with structured metadata annotations.We also focus on image analysis workflows in combination with OMERO based on the Fiji/ImageJ software and using Jupyter Notebooks. In the last part, we explore how OMERO can be used to create publication figures and prepare bioimage data for publication in a suitable repository such as the Bioimage Archive.
Module 1 (9 am - 10.15 am): Basics of OMERO, data structuring and annotation
Module 2 (10.45 am - 12.45 pm): OMERO and Fiji
Module 3 (1.45 pm - 3.45 pm): OMERO and Jupyter Notebooks
Module 4 (4.15 pm - 6. pm): Publication-ready figures and data with OMERO
The target group for this workshopThis workshop is directed at researchers at all career levels who plan to or have started to use OMERO for their microscopy research data management. We encourage the workshop participants to bring example data from their research to discuss suitable metadata annotation for their everyday practice.
Prerequisites:Users should bring their laptops and have access to the internet through one of the following options:- eduroam- institutional WiFi- VPN connection to their institutional networks to access OMERO
Who are the trainers?
Dr. Riccardo Massei (Helmholtz-Center for Environmental Research, UFZ, Leipzig) - Data Steward for Bioimaging Data in NFDI4BIOIMAGE
Dr. Michele Bortolomeazzi (DKFZ, Single cell Open Lab, bioimage data specialist, bioinformatician, staff scientist in the NFDI4BIOIMAGE project)
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center, Heidelberg, Project Coordinator of the NFDI4BIOIMAGE project)</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/11350689">https://zenodo.org/records/11350689</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11350689">https://doi.org/10.5281/zenodo.11350689</a></p>
</section>
<hr class="docutils" />
<section id="workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero">
<h2>[Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO<a class="headerlink" href="#workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero" title="Link to this heading">#</a></h2>
<p>Vanessa Fiona Aphaia Fuchs, Christian Schmidt, Tom Boissonnet</p>
<p>Published 2024-05-06</p>
<p>Licensed CC-BY-4.0</p>
<p>Description
Microscopy experiments generate information-rich, multi-dimensional data, allowing us to investigate biological processes at high spatial and temporal resolution. Image processing and analysis is a standard procedure to retrieve quantitative information from biological imaging. Due to the complex nature of bioimaging files that often come in proprietary formats, it can be challenging to organize, structure, and annotate bioimaging data throughout a project. Data often needs to be moved between collaboration partners, transformed into open formats, processed with a variety of software tools, and exported to smaller-sized images for presentation. The path from image acquisition to final publication figures with quantitative results must be documented and reproducible.
In this workshop, participants learn how to use structured metadata annotations in the image data management platform OMERO (OME Remote Objects) to optimize their data handling. This strategy helps both with organizing data for easier processing and analysis and for the preparation of data publication in journal manuscripts and in public repositories such as the BioImage Archive. Participants learn the principles of leveraging object-oriented data organization in OMERO to enhance findability and usability of their data, also in collaborative settings. The integration of OMERO with image analysis tools, in particular ImageJ/Fiji, will be trained. Moreover, users learn about community-accepted metadata checklists (REMBI) to enrich the value of their data toward reproducibility and reusability. In this workshop, we will provide hands-on training and recommendations on:</p>
<p>Structured metadata annotation features in OMERO and how to use them
Types of metadata in bioimaging: Technical metadata, sample metadata, analysis metadata
The use of ontologies and terminologies for metadata annotation
REMBI, the recommended metadata for biological images
Metadata-assisted image analysis streamlining
Tools for metadata annotation in OMERO</p>
<p>The target group for this workshop
This workshop is directed at researchers at all career levels who have started using OMERO for their microscopy research data management. We encourage the workshop participants to bring example data from their research to discuss suitable metadata annotation for their everyday practice.
Who are the trainers (see trainer description below for more details)</p>
<p>Dr. Vanessa Fuchs (NFDI4BIOIMAGE Data Steward, Center for Advanced Imaging, Heinrich-Heine University of Düsseldorf)
Dr. Tom Boissonnet (OMERO admin and image metadata specialist, Center for Advanced Imaging, Heinrich-Heine University of Düsseldorf)
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center, Heidelberg)</p>
<p>Material Description
Published here are the presentation slides that were used for input from the trainers during the different sessions of the programme. Additionally, a Fiji Macro is published that depends on the OMERO Extensions Plugin by Pouchin et al, 2022, F100Research, <a class="reference external" href="https://doi.org/10.12688/f1000research.110385.2">https://doi.org/10.12688/f1000research.110385.2</a> 
Programme Overview
Day 1 - April 29th, 2024 09.00 a.m. to 10.00 a.m.: Session 1 - Welcome and Introduction
10.00 a.m. to 10.30 a.m.:  Session 2 - Introduction to the FAIR principles &amp; data annotation
10:30 a.m. to 10:45 a.m.: Coffee break
10.45 a.m. to 12.00 a.m.: Session 3 - Data structure (datasets in OMERO) and organization with Tags 
12.00 a.m. to 1.00 p.m.:  Lunch Break
1.00 p.m. to 2.00 p.m.:  Session 4 - REMBI, Key-Value pair annotations in bioimaging
2:00 p.m. to 2.30 p.m.:  Session 5 - Ontologies for Key-Value Pairs in OMERO
2:30 p.m. to 2:45 p.m. Coffee break
2.45 p.m. to 3.45 p.m.:  Wrap-up, discussion, outlook on day 2
Day 2 - April 30th, 2024
09.00 a.m. to 09.30 a.m.:  Arrival and Start into day 2
09.30 a.m. to 11.30 a.m.:  Session 6 - Hands-on : REMBI-based Key-Value Pair annotation in OMERO
11.30 a.m. to 12.30 a.m.:  Lunch Break
12.30 a.m. to 1.15 p.m.: Session 7 - OMERO and OMERO.plugins
1.15 p.m. to 2.00 p.m.: Session 8 - Loading OMERO-hosted data into Fiji
2.00 p.m. to 2.15 p.m.: Coffee break 
2.15 p.m. to 3.00 p.m.: Discussion, Outlook</p>
<p>Tags: Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/11109616">https://zenodo.org/records/11109616</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11109616">https://doi.org/10.5281/zenodo.11109616</a></p>
</section>
<hr class="docutils" />
<section id="workshop-managing-fair-microscopy-data-at-scale-for-universities-and-research-institutions-an-introduction-for-non-imaging-stakeholders">
<h2>[Workshop] Managing FAIR microscopy data at scale for universities and research institutions: an introduction for non-imaging stakeholders<a class="headerlink" href="#workshop-managing-fair-microscopy-data-at-scale-for-universities-and-research-institutions-an-introduction-for-non-imaging-stakeholders" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Michele Bortolomeazzi, Ksenia Krooß, Jan-Philipp Mallm, Elisa Ferrando-May, Stefanie Weidtkamp-Peters</p>
<p>Published 2025-03-14</p>
<p>Licensed CC-BY-4.0</p>
<p>These slides were used in a workshop at the 2025 E-Science Tage in Heidelberg.
Workshop Abstract:
Effective Research Data Management (RDM) requires collaboration between infrastructure providers, support units, and domain-specific experts across scientific disciplines. Microscopy, or bioimaging, is a widely used technology at universities and research institutions, generating large, multi-dimensional datasets. Scientists now routinely produce microscopy data using advanced imaging modalities, often through centrally-provided instruments maintained by core facilities.
Bioimaging data management presents unique challenges: files are often large (e.g., 15+ GB for whole slide images), come in various proprietary formats, and are accessed frequently for viewing as well as for complex image processing and analysis workflows. Collaboration between experimenters, clinicians, group leaders, core facility staff, and image analysts adds to the complexity, increasing the risk of data fragmentation and metadata loss.
The DFG-funded project I3D:bio and the consortium NFDI4BIOIMAGE, part of Germany’s National Research Data Infrastructure (NFDI), are addressing these challenges by developing solutions and best practices for managing large, complex microscopy datasets. This workshop introduces the challenges of bioimaging RDM to institutional support personnel, including, for example, library staff, IT departments, and data stewards. Participants will explore the bioimaging RDM system OMERO, and apply structured metadata annotation and object-oriented data organization to a simple training dataset. OMERO offers centralized, secure access to data, allowing collaboration and reducing the data fragementation risk. Moreover, participants will experience the benefits of OME-Zarr, a chunked open file format designed for FAIR data sharing and remote access. OME-Zarr enables streaming of large, N-dimensional array-typed data over the Internet without the need to download whole files. An expanding toolbox for leveraging OME-Zarr for bioimaging data renders this file type a promising candidate for a standard file format suitable for use in FAIR Digital Object (FDO) implementations for microscopy data. OME-Zarr has become a pillar for imaging data sharing in two bioimaging-specific data repositories, i.e., the Image Data Resource (IDR) and the BioImage Archive (BIA). The team of Data Stewards from both abovenmentioned projects help researchers and research support staff to manage und publish bioimaging data.
By the end of the workshop, participants will have gained hands-on experience with bioimaging data and will be aware of support resources like the NFDI4BIOIMAGE Help Desk for addressing specific local use cases. Our goal is to promote collaboration across disciplines to effectively manage complex bioimaging data in compliance with the FAIR principles.
 </p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/15026373">https://zenodo.org/records/15026373</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.15026373">https://doi.org/10.5281/zenodo.15026373</a></p>
</section>
<hr class="docutils" />
<section id="workshop-research-data-management-for-microscopy-and-bioimage-analysis">
<h2>[Workshop] Research Data Management for Microscopy and BioImage Analysis<a class="headerlink" href="#workshop-research-data-management-for-microscopy-and-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Tom Boissonnet, Michele Bortolomeazzi, Ksenia Krooß</p>
<p>Published 2024-09-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Research Data Management for Microscopy and BioImage Analysis</p>
<p>Introduction to BioImaging Research Data Management, NFDI4BIOIMAGE and I3D:bioChristian Schmidt /DKFZ Heidelberg
OMERO as a tool for bioimaging data managementTom Boissonnet /Heinrich-Heine Universität Düsseldorf
Reproducible image analysis workflows with OMERO software APIsMichele Bortolomeazzi /DKFZ Heidelberg
Publishing datasets in public archives for bioimage dataKsenia Krooß /Heinrich-Heine Universität Düsseldorf</p>
<p>Date &amp; Venue:Thursday, Sept. 26, 5.30 p.m.Haus 22 / Paul Ehrlich Lecture Hall (H22-1)University Hospital Frankfurt</p>
<p>Tags: Nfdi4Bioimage, Research Data Management, Include In Dalia</p>
<p><a class="reference external" href="https://zenodo.org/records/13861026">https://zenodo.org/records/13861026</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13861026">https://doi.org/10.5281/zenodo.13861026</a></p>
</section>
<hr class="docutils" />
<section id="data-visualization-tutorial">
<h2>data_visualization_tutorial<a class="headerlink" href="#data-visualization-tutorial" title="Link to this heading">#</a></h2>
<p>Stefan Halfpap</p>
<p>Published 2025-04-25T21:00:39+00:00</p>
<p>Tags: Data Visualization, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/klauck/data_visualization_tutorial">klauck/data_visualization_tutorial</a></p>
</section>
<hr class="docutils" />
<section id="image-processing-basics">
<h2>image-processing-basics<a class="headerlink" href="#image-processing-basics" title="Link to this heading">#</a></h2>
<p>Myles Scolnick</p>
<p>Published 2025-05-27T08:31:14+00:00</p>
<p>Licensed APACHE-2.0</p>
<p>Teaching material for image processing and analysis</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/fmi-faim/image-processing-basics">fmi-faim/image-processing-basics</a></p>
</section>
<hr class="docutils" />
<section id="imagequantification101">
<h2>imagequantification101<a class="headerlink" href="#imagequantification101" title="Link to this heading">#</a></h2>
<p>Virginie, Guillaume Witz, Joel Lüthi</p>
<p>Published 2022-08-17T14:55:16+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository, Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/uhlmanngroup/imagequantification101">uhlmanngroup/imagequantification101</a></p>
</section>
<hr class="docutils" />
<section id="introduction-to-generative-ai">
<h2>introduction-to-generative-ai<a class="headerlink" href="#introduction-to-generative-ai" title="Link to this heading">#</a></h2>
<p>Bruna Piereck, Alexander Botzki</p>
<p>Published 2024-09-27T14:38:51+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Course repository for Strategic Use of Generative AI</p>
<p>Tags: Artificial Intelligence, Include In Dalia</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/vibbits/introduction-to-generative-ai">vibbits/introduction-to-generative-ai</a></p>
<p><a class="reference external" href="https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/introduction-to-generative-ai/refs/heads/main/README.md">https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/introduction-to-generative-ai/refs/heads/main/README.md</a></p>
</section>
<hr class="docutils" />
<section id="id2">
<h2>introduction-to-image-analysis<a class="headerlink" href="#id2" title="Link to this heading">#</a></h2>
<p>Dave Barry, Stefania Marcotti, Sara Salgueiro Torres, Martin Jones, AlicjaSkorkowska</p>
<p>Published 2025-08-26T14:08:22+00:00</p>
<p>Licensed CC-BY-SA-4.0</p>
<p>Tags: Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/FrancisCrickInstitute/introduction-to-image-analysis">FrancisCrickInstitute/introduction-to-image-analysis</a></p>
</section>
<hr class="docutils" />
<section id="napari-scipy2025-workshop">
<h2>napari-scipy2025-workshop<a class="headerlink" href="#napari-scipy2025-workshop" title="Link to this heading">#</a></h2>
<p>Draga Doncila Pop</p>
<p>Published 2025-02-28T23:41:10+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>This is a three-part workshop guiding you through using napari to view images, a brief bioimaging analysis application, and extending napari’s functionality with your own custom workflows.</p>
<p>Tags: Python, Napari, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/DragaDoncila/napari-scipy2025-workshop">DragaDoncila/napari-scipy2025-workshop</a></p>
</section>
<hr class="docutils" />
<section id="nextflow-workshop">
<h2>nextflow-workshop<a class="headerlink" href="#nextflow-workshop" title="Link to this heading">#</a></h2>
<p>Tuur Muyldermans, Kris Davie, Alexander, Nicolas Vannieuwkerke, Kobe Lavaerts, Marcel Ribeiro-Dantas, Bruna Piereck, Steff Taelman</p>
<p>Published 2023-03-29T10:40:04+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Nextflow workshop materials March 2023</p>
<p>Tags: Workflow, Nextflow, Include In Dalia</p>
<p>Content type: Github Repository, Tutorial</p>
<p><a class="github reference external" href="https://github.com/vibbits/nextflow-workshop">vibbits/nextflow-workshop</a></p>
<p><a class="reference external" href="https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/nextflow-workshop/main/README.md#1">https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/nextflow-workshop/main/README.md#1</a></p>
</section>
<hr class="docutils" />
<section id="nextflow101">
<h2>nextflow101<a class="headerlink" href="#nextflow101" title="Link to this heading">#</a></h2>
<p>Virginie</p>
<p>Published 2025-08-19T14:51:21+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>NextFlow 101</p>
<p>Tags: Bioimage Analysis, Workflows, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/uhlmanngroup/nextflow101">uhlmanngroup/nextflow101</a></p>
</section>
<hr class="docutils" />
<section id="numpy-pandas-course">
<h2>numpy pandas course<a class="headerlink" href="#numpy-pandas-course" title="Link to this heading">#</a></h2>
<p>Guillaume Witz</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Tags: Python, Include In Dalia</p>
<p>Content type: Notebook</p>
<p><a class="github reference external" href="https://github.com/guiwitz/NumpyPandas_course">guiwitz/NumpyPandas_course</a></p>
</section>
<hr class="docutils" />
<section id="qupath-workshop">
<h2>qupath-workshop<a class="headerlink" href="#qupath-workshop" title="Link to this heading">#</a></h2>
<p>Antoine A. Ruzette, dependabot[bot]</p>
<p>Published 2025-01-16T14:05:02+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Materials supporting the QuPath workshop at Harvard Medical School.</p>
<p>Tags: Notebook, Slides, Collection, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/HMS-IAC/qupath-workshop">HMS-IAC/qupath-workshop</a></p>
</section>
<hr class="docutils" />
<section id="rse-skills-workshop">
<h2>rse-skills-workshop<a class="headerlink" href="#rse-skills-workshop" title="Link to this heading">#</a></h2>
<p>Jack Atkinson</p>
<p>Published 2023-12-22T17:39:48+00:00</p>
<p>Licensed GPL-3.0</p>
<p>Teaching materials for improving research software writing abilities.</p>
<p>Tags: Research Software Engineering, Include In Dalia</p>
<p>Content type: Github Repository, Slides</p>
<p><a class="github reference external" href="https://github.com/jatkinson1000/rse-skills-workshop">jatkinson1000/rse-skills-workshop</a></p>
</section>
<hr class="docutils" />
<section id="scanpy-tutorials">
<h2>scanpy-tutorials<a class="headerlink" href="#scanpy-tutorials" title="Link to this heading">#</a></h2>
<p>Alex Wolf, pre-commit-ci[bot], Philipp A., Isaac Virshup, Ilan Gold, Giovanni Palla, Fidel Ramirez, G\xF6k\xE7en Eraslan, Sergei Rybakov, Abolfazl (Abe), Adam Gayoso, Dinesh Palli, Gregor Sturm, Jan Lause, Karin Hrovatin, Krzysztof Polanski, RaphaelBuzzi, Yimin Zheng, Yishen Miao, evanbiederstedt</p>
<p>Published 2018-12-16T03:42:46+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Scanpy Tutorials.</p>
<p>Tags: Single-Cell Analysis, Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scverse/scanpy-tutorials">scverse/scanpy-tutorials</a></p>
</section>
<hr class="docutils" />
<section id="scikit-learn-mooc">
<h2>scikit-learn MOOC<a class="headerlink" href="#scikit-learn-mooc" title="Link to this heading">#</a></h2>
<p>Loïc Estève et al.</p>
<p>Licensed CC-BY-4.0</p>
<p>Machine learning in Python with scikit-learn MOOC</p>
<p>Tags: Bioimage Analysis, Machine Learning, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/INRIA/scikit-learn-mooc">INRIA/scikit-learn-mooc</a></p>
</section>
<hr class="docutils" />
<section id="shapeanalysis101">
<h2>shapeanalysis101<a class="headerlink" href="#shapeanalysis101" title="Link to this heading">#</a></h2>
<p>Virginie, Johannes Hugger</p>
<p>Published 2021-05-17T13:32:40+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>A tutorial of classical shape analysis methods</p>
<p>Tags: Bioimage Analysis, Include In Dalia</p>
<p>Content type: Github Repository, Notebook, Collection</p>
<p><a class="github reference external" href="https://github.com/uhlmanngroup/shapeanalysis101">uhlmanngroup/shapeanalysis101</a></p>
</section>
<hr class="docutils" />
<section id="skimage-tutorials">
<h2>skimage-tutorials<a class="headerlink" href="#skimage-tutorials" title="Link to this heading">#</a></h2>
<p>Juan Nunez-Iglesias et al.</p>
<p>Licensed CC0 1.0 UNIVERSAL</p>
<p>skimage-tutorials - a collection of tutorials for the scikit-image package.</p>
<p>Tags: Bioimage Analysis, Scikit-Image, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/scikit-image/skimage-tutorials">scikit-image/skimage-tutorials</a></p>
</section>
<hr class="docutils" />
<section id="training-resources">
<h2>training-resources<a class="headerlink" href="#training-resources" title="Link to this heading">#</a></h2>
<p>Christian Tischer, Antonio Politi, Toby Hodges, maulakhan, grinic, bugraoezdemir, Tim-Oliver Buchholz, Elnaz Fazeli, Aliaksandr Halavatyi, Dominik Kutra, Stefania Marcotti, AnniekStok, Felix, jhennies, Severina Klaus, Martin Schorb, Nima Vakili, Sebastian Gonzalez Tirado, Stefan Helfrich, Yi Sun, Ziqiang Huang, Jan Eglinger, Constantin Pape, Joel Lüthi, Matt McCormick, Oane Gros</p>
<p>Published 2020-04-23T07:51:38+00:00</p>
<p>Licensed CC-BY-4.0</p>
<p>Resources for teaching/preparing to teach bioimage analysis</p>
<p>Tags: Bioimageanalysis, Neurobias, Include In Dalia</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/NEUBIAS/training-resources">NEUBIAS/training-resources</a></p>
<hr class="docutils" />
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./tags"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="imagej.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Imagej (20)</p>
      </div>
    </a>
    <a class="right-next"
       href="licensing.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Licensing (7)</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#management-of-microscopy-image-data-an-overview-of-omero-bioimage-archive-and-image-data-resource-2025-uni-leipzig">“Management of Microscopy Image Data: An overview of OMERO, BioImage Archive and Image Data Resource”  2025 @ Uni Leipzig</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zenodo-und-co-was-bringt-und-wer-braucht-ein-repositorium">“ZENODO und Co.” Was bringt und wer braucht ein Repositorium?</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#mic-workshop-on-bioimage-processing-with-python">2022 MIC Workshop on Bioimage processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#courseimageanalysis-nanocell">2023_CourseImageAnalysis_NanoCell</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#steps-towards-reproducible-research">6 Steps Towards Reproducible Research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-fiji-scripting-tutorial">A Fiji Scripting Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-glimpse-of-the-open-source-flim-analysis-software-tools-flimfit-flute-and-napari-flim-phasor-plotter">A Glimpse of the Open-Source FLIM Analysis Software Tools FLIMfit, FLUTE and napari-flim-phasor-plotter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-hitchhiker-s-guide-through-the-bio-image-analysis-software-universe">A Hitchhiker’s guide through the bio-image analysis software universe</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-perspective-on-fair-and-scalable-access-to-large-image-data">A Perspective on FAIR and Scalable Access to Large Image Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-biologists-guide-to-planning-and-performing-quantitative-bioimaging-experiments">A biologist’s guide to planning and performing quantitative bioimaging experiments</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-global-view-of-standards-for-open-image-data-formats-and-repositories">A global view of standards for open image data formats and repositories</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-journey-to-fair-microscopy-data">A journey to FAIR microscopy data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-practical-guide-to-bioimaging-research-data-management-in-core-facilities">A practical guide to bioimaging research data management in core facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ai-ml-dl-in-bioimage-analysis-webinar">AI ML DL in Bioimage Analysis - Webinar</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#accessible-interactive-spatial-omics-data-visualizations-with-vitessce-and-omero">Accessible Interactive Spatial-Omics Data Visualizations with Vitessce and OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#advanced-scripting-with-fiji">Advanced scripting with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#advancing-fair-image-analysis-in-galaxy-tools-workflows-and-training">Advancing FAIR Image Analysis in Galaxy: Tools, Workflows, and Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#alles-meins-oder-urheberrechte-klaren-fur-forschungsdaten">Alles meins – oder!? Urheberrechte klären für Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#analysis-of-high-throughput-microscopy-image-data">Analysis of High-Throughput Microscopy Image Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#annotating-3d-images-in-napari">Annotating 3D images in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#artificial-intelligence-for-digital-pathology">Artificial Intelligence for Digital Pathology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#basics-of-image-processing-and-analysis">Basics of Image Processing and Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#best-practice-data-life-cycle-approaches-for-the-life-sciences">Best practice data life cycle approaches for the life sciences</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-code-generation">Bio-image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-code-generation-using-bia-bob">Bio-image Analysis Code Generation using bia-bob</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-kioto-and-taipei-23-24">Bio-image Analysis Workshop Kioto and Taipei 23/24</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-workshop-taipei">Bio-image Analysis Workshop Taipei</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-with-the-help-of-large-language-models">Bio-image Analysis with the Help of Large Language Models</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science">Bio-image Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-science-lectures-2025-uni-leipzig-scads-ai">Bio-image Data Science Lectures 2025 @ Uni Leipzig / ScaDS.AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-analysis-biostatistics-programming-and-machine-learning-for-computational-biology">Bio-image analysis, biostatistics, programming and machine learning for computational biology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-notebooks">BioImage Analysis Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-and-superresolution-microscopy-workshop-2023-at-dartmouth-college">BioImage Analysis and Superresolution Microscopy Workshop 2023 (at Dartmouth College)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-data-analysis">BioImage Data Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-io-chatbot-globias-seminar">BioImage.IO Chatbot, GloBIAS Seminar</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimage-analysis-with-napari">Bioimage analysis with Napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bioimaging-ai-textbook">Bioimaging AI Textbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#browsing-the-open-microscopy-image-data-resource-with-python">Browsing the Open Microscopy Image Data Resource with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-fair-image-analysis-pipelines-for-high-content-screening-hcs-data-using-galaxy">Building FAIR image analysis pipelines for high-content-screening (HCS) data using Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-bioimage-analysis-workflow-using-deep-learning">Building a Bioimage Analysis Workflow using Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-fair-image-data-ecosystem-for-microscopy-communities">Building a FAIR image data ecosystem for microscopy communities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#building-a-national-research-data-infrastructure">Building a National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#care-stardist-tutorials-for-embo-practical-course-computational-optical-biology-2022">CARE/Stardist tutorials for EMBO Practical Course — Computational optical biology 2022</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cs231n-convolutional-neural-networks-for-visual-recognition">CS231n Convolutional Neural Networks for Visual Recognition</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#csbdeep-and-stardist-i2k-2020">CSBDeep and StarDist @ I2K 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-introduction">CellProfiler Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-practical-neubias-lisbon-2017">CellProfiler Practical NeuBIAS Lisbon 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cellprofiler-tutorials">CellProfiler tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#center-for-microscopy-and-image-analysis-how-to-guides">Center for Microscopy and Image Analysis How-to Guides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">Challenges and opportunities for bio-image analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bioimage-analysis-core-facilities">Challenges and opportunities for bioimage analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#chatgpt-for-image-analysis">ChatGPT for Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cloud-native-formats-enable-federated-repositories-at-peta-scale">Cloud-Native Formats Enable Federated Repositories at Peta-Scale</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collaborative-working-and-version-control-with-git-hub">Collaborative Working and Version Control with git[hub]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collaborative-bio-image-analysis-script-editing-with-git">Collaborative bio-image analysis script editing with git</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collection-of-teaching-material-for-deep-learning-for-biomedical-image-analysis">Collection of teaching material for deep learning for (biomedical) image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#combining-the-bids-and-arc-directory-structures-for-multimodal-research-data-organization">Combining the BIDS and ARC Directory Structures for Multimodal Research Data Organization</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#community-developed-checklists-for-publishing-images-and-image-analyses">Community-developed checklists for publishing images and image analyses</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#conda-container-and-bots-how-to-build-and-maintain-tool-dependencies-in-workflows-and-training-materials">Conda, Container and Bots - How to Build and Maintain Tool Dependencies in Workflows and Training Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#course-on-deep-learning-for-imaging-using-pytorch">Course on Deep Learning for imaging using PyTorch</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#crashkurs-forschungsdatenmanagement">Crashkurs Forschungsdatenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#creating-workflows-and-advanced-workflow-options">Creating Workflows and Advanced Workflow Options</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#creating-a-research-data-management-plan-using-chatgpt">Creating a Research Data Management Plan using chatGPT</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#creating-open-computational-curricula">Creating open computational curricula</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training">Cultivating Open Training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cultivating-open-training-to-advance-bio-image-analysis">Cultivating Open Training to advance Bio-image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cytovi-deep-generative-modeling-of-antibody-based-single-cell-technologies">CytoVI Deep generative modeling of antibody-based single cell technologies</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dalia-interchange-format">DALIA Interchange Format</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dl4miceverywhere-overcoming-reproducibility-challenges-in-deep-learning-microscopy-imaging">DL4MicEverywhere – Overcoming reproducibility challenges in deep learning microscopy imaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-carpentry-for-biologists">Data Carpentry for Biologists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-visualization-with-flying-colors">Data Visualization with Flying Colors</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-handling-in-large-scale-electron-microscopy">Data handling in large-scale electron microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-management-at-france-bioimaging">Data management at France BioImaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-stewardship-and-research-data-management-tools-for-multimodal-linking-of-imaging-data-in-plasma-medicine">Data stewardship and research data management tools for multimodal linking of imaging data in plasma medicine</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dataplant-knowledge-base">DataPLANT knowledge base</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dataviz-protocols-an-introduction-to-data-visualization-protocols-for-wet-lab-scientists">DataViz protocols - An introduction to data visualization protocols for wet lab scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagement">Datenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagement-im-fokus-organisation-speicherstrategien-und-datenschutz">Datenmanagement im Fokus: Organisation, Speicherstrategien und Datenschutz</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-1">Datenmanagementpläne erstellen - Teil 1</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#datenmanagementplane-erstellen-teil-2">Datenmanagementpläne erstellen - Teil 2</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-based-segmentation-for-biologists">Deep Learning Based Segmentation For Biologists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-learning-for-image-analysis-exercises">Deep Learning for image analysis - Exercises</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deep-vision-and-graphics">Deep Vision and Graphics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#deepprofiler-handbook">DeepProfiler Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#developing-a-training-strategy">Developing a Training Strategy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#developing-open-source-software-for-bioimage-analysis-opportunities-and-challenges">Developing open-source software for bioimage analysis: opportunities and challenges</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#development-of-a-platform-for-advanced-optics-education-training-and-prototyping">Development of a platform for advanced optics education, training and prototyping</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#diataxis-a-systematic-approach-to-technical-documentation-authoring">Diátaxis - A systematic approach to technical documentation authoring.</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#docker-mastery-with-kubernetes-swarm-from-a-docker-captain">Docker Mastery - with Kubernetes + Swarm from a Docker Captain</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dr-guillaume-jacquemet-on-studying-cancer-cell-metastasis-in-the-era-of-deep-learning-for-microscopy">Dr Guillaume Jacquemet on studying cancer cell metastasis in the era of deep learning for microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#edam-bioimaging-the-ontology-of-bioimage-informatics-operations-topics-data-and-formats-update-2020">EDAM-bioimaging: The ontology of bioimage informatics operations, topics, data, and formats (update 2020)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embi-recommended-metadata-for-biological-images-metadata-guidelines-for-bioimaging-data">EMBI - Recommended Metadata for Biological Images – metadata guidelines for bioimaging data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2019-exercises-and-materials">EMBL Deep Learning course 2019 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2021-22-exercises-and-materials">EMBL Deep Learning course 2021/22 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#embl-deep-learning-course-2023-exercises-and-materials">EMBL Deep Learning course 2023 exercises and materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#efficiently-starting-institutional-research-data-management">Efficiently starting institutional research data management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#einblicke-ins-forschungsdatenmanagement-darf-ich-das-veroffentlichen-rechtsfragen-im-umgang-mit-forschungsdaten">Einblicke ins Forschungsdatenmanagement - Darf ich das veröffentlichen? Rechtsfragen im Umgang mit Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elastix-tutorial">Elastix tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#end-to-end-tissue-microarray-image-analysis-with-galaxy-me">End-to-End Tissue Microarray Image Analysis with Galaxy-ME</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#engineering-a-software-environment-for-research-data-management-of-microscopy-image-data-in-a-core-facility">Engineering a Software Environment for Research Data Management of Microscopy Image Data in a Core Facility</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#erstellung-und-realisierung-einer-institutionellen-forschungsdaten-policy">Erstellung und Realisierung einer institutionellen Forschungsdaten-Policy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#eubias-course-2013-intensity-dynamics-at-the-periphery-of-nucleus">EuBIAS course 2013: Intensity Dynamics at the Periphery of Nucleus</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-evolve-deliverable-6-2-landscape-analysis-of-existing-training-resources-for-the-nodes">Euro-BioImaging - EVOLVE Deliverable 6.2 - Landscape analysis of existing training resources for the Nodes</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#euro-bioimaging-s-guide-to-fair-bioimage-data-practical-tasks">Euro-BioImaging’s Guide to FAIR BioImage Data - Practical Tasks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#expansion-and-fluctuations-enhanced-microscopy-for-nanoscale-molecular-profiling-of-cells-and-tissues-data-processing-manual">Expansion and fluctuations-enhanced microscopy for nanoscale molecular profiling of cells and tissues - Data processing manual</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#explainable-ai-for-computer-vision">Explainable AI for Computer Vision</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fair-priciples">FAIR Priciples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fairy-deep-learning-for-bioimage-analysis">FAIRy deep-learning for bioImage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#faq-kunstliche-intelligenz-und-gute-wissenschaftliche-praxis">FAQ Künstliche Intelligenz und gute wissenschaftliche Praxis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#feature-extraction-in-napari">Feature extraction in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#file-naming-convention-worksheet">File Naming Convention Worksheet</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#finding-and-choosing-a-data-repository">Finding and Choosing a Data Repository</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#finding-and-using-publicly-available-data">Finding and using publicly available data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#five-great-reasons-to-share-your-research-data">Five great reasons to share your research data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#five-ways-to-effectively-use-animation-in-data-visualization">Five ways to effectively use animation in data visualization</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#forschungsdaten-info">Forschungsdaten.info</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#forschungsdaten-org">Forschungsdaten.org</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#friendsdontletfriends">FriendsDontLetFriends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#from-cells-to-pixels-bridging-biologists-and-image-analysts-through-a-common-language">From Cells to Pixels: Bridging Biologists and  Image Analysts Through a Common Language</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#from-paper-to-pixels-navigation-through-your-research-data-presentations-of-speakers">From Paper to Pixels: Navigation through your Research Data - presentations of speakers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#from-bioimaging-projects-to-communities-globias-bia-seminar-series">From bioimaging projects to communities - GloBIAS BIA Seminar Series</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-in-digital-image-processing">Fundamentals in digital image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#fundamentals-of-image-analysis-in-python-with-scikit-image-napari-and-friends">Fundamentals of image analysis in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-meets-omero-overview-on-the-galaxy-omero-suite-and-vizarr-viewer">Galaxy meets OMERO! Overview on the Galaxy OMERO-suite and Vizarr Viewer</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#galaxy-workflows">Galaxy workflows</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#generative-artificial-intelligence-for-bio-image-analysis">Generative artificial intelligence for bio-image analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#getting-started-with-mambaforge-and-python">Getting started with Mambaforge and Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#getting-started-with-python-intro-and-set-up-a-conda-environment">Getting started with Python: intro and set-up a conda environment</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#guidance-for-developing-a-research-data-management-rdm-policy">Guidance for Developing a Research Data Management (RDM) Policy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#high-throughput-automated-data-analysis-and-data-management-workflow-with-cellprofiler-and-omero">High throughput &amp; automated data analysis and data management workflow with Cellprofiler and OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#high-content-screening-hcs-workflows-for-fair-image-data-management-with-omero">High-content screening (HCS) workflows for FAIR image data management with OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#highlights-from-the-2016-2020-neubias-training-schools-for-bioimage-analysts-a-success-story-and-key-asset-for-analysts-and-life-scientists">Highlights from the 2016-2020 NEUBIAS training schools for Bioimage Analysts: a success story and key asset for analysts and life scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#how-to-write-a-bug-report">How to write a bug report</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-2024-clesperanto-gpu-accelerated-image-processing-library">I2K 2024: clEsperanto - GPU-Accelerated Image Processing Library</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-workshop-material-lazy-parallel-processing-and-visualization-of-large-data-with-imglib2-bigdataviewer-the-n5-api-and-spark">I2K2024 workshop material - Lazy Parallel Processing and Visualization of Large Data with ImgLib2, BigDataViewer, the N5-API, and Spark</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k2024-virtual-bio-image-analysis-code-generation">I2K2024(virtual) - Bio-Image Analysis Code Generation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#iafig-rms-python-for-bioimage-analysis-course">IAFIG-RMS Python for Bioimage Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#idc-tutorials">IDC-Tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#if-you-license-it-itll-be-harder-to-steal-it-why-we-should-license-our-work">If you license it, it’ll be harder to steal it. Why we should license our work</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-training-resources">Image Analysis Training Resources</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-in-healthcare-the-current-landscape-trends-and-collaborative-opportunities">Image Analysis in Healthcare - The Current Landscape, Trends, and Collaborative Opportunities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-using-galaxy">Image Analysis using Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-with-python">Image Processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-repository-decision-tree-where-do-i-deposit-my-imaging-data">Image Repository Decision Tree - Where do I deposit my imaging data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-and-visualization-in-python-with-scikit-image-napari-and-friends">Image analysis and visualization in Python with scikit-image, napari, and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-analysis-course-material">Image analysis course material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-handling-using-fiji-training-materials">Image handling using Fiji - training materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-for-beginners">Image processing for beginners</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#id1">Image processing with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-based-profiling-handbook">Image-based Profiling Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej-macro-introduction">ImageJ Macro Introduction</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagej2-api-beating">ImageJ2 API-beating</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#increasing-the-fairness-of-electron-microscopy-data-in-life-and-material-science-research">Increasing the FAIRness of electron microscopy data in life and material science research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#interactive-bioimage-analysis-workflow-with-clij-eabias-2025-training-event">Interactive Bioimage Analysis Workflow with CLIJ (@EABIAS 2025 training event)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#interactive-image-data-flow-graphs">Interactive Image Data Flow Graphs</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#internal-alm-bioimage-analysis-workshop-2023">Internal ALM BioImage Analysis workshop 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#intro-napari-slides">Intro napari slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-bioimage-analysis">Introduction to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-deep-learning-for-microscopy">Introduction to Deep Learning for Microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-high-performance-computing-for-life-scientists">Introduction to High Performance Computing for Life Scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis">Introduction to Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-image-analysis-with-fiji">Introduction to Image Analysis with Fiji</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-imagej-macro-programming-scientific-computing-facility-mpi-cbg-dresden">Introduction to ImageJ macro programming, Scientific Computing Facility, MPI CBG Dresden</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-omero-frankfurt-online">Introduction to OMERO - Frankfurt - online</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-research-data-management-and-open-research">Introduction to Research Data Management and Open Research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#jipipe-spring-course-jsc-2025-workshop-recordings-slides-homework-and-materials">JIPipe Spring Course (JSC) 2025: Workshop Recordings, Slides, Homework, and Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kollaboratives-arbeiten-und-versionskontrolle-mit-git">Kollaboratives Arbeiten und Versionskontrolle mit Git</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kreshuk-lab-s-embl-eipp-predoc-course-teaching-material">Kreshuk Lab’s EMBL EIPP predoc course teaching material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#kriterienkatalog-fur-materialien-aus-dem-themenbereich-forschungsdatenmanagement">Kriterienkatalog für Materialien aus dem Themenbereich Forschungsdatenmanagement</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#large-language-models-an-introduction-for-life-scientists">Large Language Models: An Introduction for Life Scientists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#latent-space-visualisation-pca-t-sne-umap-deep-learning-animated">Latent Space Visualisation: PCA, t-SNE, UMAP | Deep Learning Animated</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#learning-and-training-bio-image-analysis-in-the-age-of-ai">Learning and Training Bio-image Analysis in the Age of AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#lecture-applied-bioimage-analysis-2020">Lecture Applied Bioimage Analysis 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#leitfaden-zur-digitalen-datensparsamkeit-mit-praxisbeispielen">Leitfaden zur digitalen Datensparsamkeit (mit Praxisbeispielen)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#liver-micrometastases-area-quantification-using-qupath-and-pixel-classifier">Liver Micrometastases area quantification using QuPath and pixel classifier</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-deep-learning-applications-to-bioimage-analysis">Machine Learning - Deep Learning. Applications to Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#machine-learning-workflow-for-imaging-flow-cytometry-ifc">Machine Learning Workflow for Imaging Flow Cytometry (IFC)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#making-the-most-of-bioimaging-data-through-interdisciplinary-interactions">Making the most of bioimaging data through interdisciplinary interactions</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#making-your-package-available-on-conda-forge">Making your package available on conda-forge</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#making-your-project-citable">Making your project citable</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#managing-scientific-python-environments-using-conda-mamba-and-friends">Managing Scientific Python environments using Conda, Mamba and friends</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#meeting-in-the-middle-towards-successful-multidisciplinary-bioimage-analysis-collaboration">Meeting in the Middle: Towards Successful Multidisciplinary Bioimage Analysis Collaboration</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#metabolic-networks-modelling-with-cobrapy">Metabolic networks modelling with COBRApy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#metadata-matters-access-to-image-data-in-the-real-world">Metadata matters: access to image data in the real world</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#methods-in-bioimage-analysis">Methods in bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microscopy-data-analysis-machine-learning-and-the-bioimage-archive">Microscopy data analysis: machine learning and the BioImage Archive</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#modular-training-resources-for-bioimage-analysis">Modular training resources for bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#morphological-analysis-of-neural-cells-with-weka-and-snt-fiji-plugins">Morphological analysis of neural cells with WEKA and SNT Fiji plugins</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multi-template-matching-for-object-detection-slides">Multi-Template-Matching for object-detection (slides)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multi-view-fusion">Multi-view fusion</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#multiplexed-tissue-imaging-tools-and-approaches">Multiplexed tissue imaging - tools and approaches</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#my-journey-through-bioimage-analysis-teaching-methods-from-classroom-to-cloud">My Journey Through Bioimage Analysis Teaching Methods From Classroom to Cloud</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-home-interactive-bioimage-analysis-with-python-and-jupyter">NEUBIAS Academy @HOME: Interactive Bioimage Analysis with Python and Jupyter</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-analyst-school-2018">NEUBIAS Analyst School 2018</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-course-2017">NEUBIAS Bioimage Analyst Course 2017</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2019">NEUBIAS Bioimage Analyst School 2019</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-bioimage-analyst-school-2020">NEUBIAS Bioimage Analyst School 2020</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage">NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and Bioimage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-data-management-illustrations-by-henning-falk">NFDI4BIOIMAGE data management illustrations by Henning Falk</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nlp-course">NLP Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#name-styles">Name Styles</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#navigating-the-bioimage-analysis-landscape-understanding-the-community-and-its-collaborative-dynamics">Navigating the Bioimage Analysis Landscape: Understanding the Community  and its Collaborative Dynamics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubias-academy-2020-introduction-to-nuclei-segmentation-with-stardist">Neubias Academy 2020: Introduction to Nuclei Segmentation with StarDist</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#neubiaspasteur2023-advancedcellpose">NeubiasPasteur2023_AdvancedCellPose</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-demo-pipelines-for-image-processing">Nextflow Demo Pipelines for Image Processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome-documentation">OME Documentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-hcs-analysis-pipeline-using-jupyter-notebooks">OMERO - HCS analysis pipeline using Jupyter Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-guide">OMERO Guide</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-and-arc-workflow">OMERO and ARC Workflow</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-documentation">OMERO documentation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-walkthrough-example">OMERO walkthrough example</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-walkthrough-for-facility-managers">OMERO walkthrough for facility managers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omexcavator-a-tool-for-exporting-and-connecting-bioimaging-specific-metadata-in-wider-knowledge-graphs">OMExcavator: a tool for exporting and connecting  Bioimaging-specific metadata in wider knowledge graphs</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#onbi-image-analysis-course">ONBI Image Analysis Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#object-tracking-and-track-analysis-using-trackmate-and-celltrackscolab">Object Tracking and Track Analysis using TrackMate and CellTracksColab</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-deployment-examples">Omero Deployment examples</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#online-r-learning">Online_R_learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-image-data-handbook">Open Image Data Handbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-science-sharing-licensing">Open Science, Sharing &amp; Licensing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#open-source-ai-tools-for-bioimage-analysis-workshop-2024-icob-academia-sinica-taiwan">Open source AI Tools for bioimage analysis workshop (2024) @ICOB, Academia Sinica, Taiwan</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#overview-of-the-galaxy-omero-suite-upload-images-and-metadata-in-omero-using-galaxy">Overview of the Galaxy OMERO-suite - Upload images and metadata in OMERO using Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#parallelization-and-heterogeneous-computing-from-pure-cpu-to-gpu-accelerated-image-processing">Parallelization and heterogeneous computing: from pure CPU to GPU-accelerated image processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#photonic-data-analysis-in-2050">Photonic data analysis in 2050</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#plants-python-a-series-of-lessons-in-coding-plant-biology-computation-and-bioinformatics">Plants &amp; Python - A series of lessons in coding, plant biology, computation, and bioinformatics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-early-career-bio-image-analysis-training-school">PoL Bio-Image Analysis Early Career Bio-image Analysis Training School</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-gpu-accelerated-image-analysis-training-school">PoL Bio-Image Analysis GPU Accelerated Image Analysis Training School</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pol-bio-image-analysis-training-school-on-gpu-accelerated-image-analysis">PoL Bio-Image Analysis Training School on GPU-Accelerated Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#practical-guide-to-the-international-alignment-of-research-data-management-extended-edition">Practical Guide to the International Alignment of Research Data Management - Extended Edition</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#practical-considerations-for-data-exploration-in-quantitative-cell-biology">Practical considerations for data exploration in quantitative cell biology</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#promoting-data-management-at-the-nikon-imaging-center-and-cell-biology-microscopy-facility">Promoting Data Management at the Nikon Imaging Center and Cell Biology Microscopy Facility</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#promoting-reproducibility-in-biomedical-research-through-image-analysis">Promoting Reproducibility in Biomedical Research through Image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-in-bio-image-analysis">Prompt Engineering in Bio-image Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-agentic-workflows-and-multi-modal-large-language-models">Prompt Engineering, Agentic Workflows and Multi-modal Large Language Models</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prompt-engineering-llms-course">Prompt-Engineering-LLMs-Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#pygameprompts">PygamePrompts</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-bioimage-analysis-tutorial">Python BioImage Analysis Tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-workshop-image-processing">Python Workshop - Image Processing</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-for-bioimage-analysis-basic-tools-and-setup-on-windows">Python for Bioimage Analysis Basic Tools and Setup on Windows</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#python-for-microscopists">Python for Microscopists</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qi-2024-analysis-lab-manual">QI 2024 Analysis Lab Manual</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qm-course-lectures-on-bio-image-analysis-with-napari-2024">QM Course Lectures on Bio-Image Analysis with napari 2024</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quarep-limi-a-community-driven-initiative-to-establish-guidelines-for-quality-assessment-and-reproducibility-for-instruments-and-images-in-light-microscopy">QUAREP-LiMi: A community-driven initiative to establish guidelines for quality assessment and reproducibility for instruments and images in light microscopy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-for-python-programmers">QuPath for Python programmers</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-open-source-software-for-analysing-awkward-images">QuPath: Open source software for analysing (awkward) images</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quality-assurance-of-segmentation-results">Quality assurance of segmentation results</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdm-starter-kit">RDM Starter Kit</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rdmkit-training-resources">RDMKit Training Resources</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rembi-overview">REMBI Overview</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rechtsfragen-bei-open-science-ein-leitfaden">Rechtsfragen bei Open Science - Ein Leitfaden</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#report-on-a-pilot-study-implementation-of-omero-for-microscopy-data-management">Report on a pilot study:  Implementation of OMERO for  microscopy data management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#reproducible-bio-image-analysis-using-python-napari-jupyter-and-ai">Reproducible Bio-Image Analysis using Python, Napari, Jupyter and AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rescaling-images-and-pixel-an-isotropy">Rescaling images and pixel (an)isotropy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-management-seminar-slides">Research Data Management Seminar - Slides</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-managemet-and-how-not-to-get-overwhelmed-with-data">Research Data Managemet and how not to get overwhelmed with data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#running-deep-learning-scripts-in-the-bia-pol-omero-server">Running Deep-Learning Scripts in the BiA-PoL Omero Server</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#swc-gcnu-software-skills">SWC/GCNU Software Skills</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scientific-visualization-python-matplotlib">Scientific Visualization: Python + Matplotlib</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scipy-cookbook">Scipy Cookbook</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-a-data-management-infrastructure-for-bioimaging">Setting up a data management infrastructure for bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-a-remote-desktop-to-use-napari-in-a-browser">Setting up a remote desktop to use Napari in a browser</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sharing-your-poster-on-figshare-a-community-guide-to-how-to-and-why">Sharing Your Poster on Figshare: A Community Guide to How-To and Why</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sharing-and-licensing-material">Sharing and licensing material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sharing-research-data-with-zenodo">Sharing research data with Zenodo</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#simpleitk-notebooks">SimpleITK-Notebooks</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#slides-about-flute-a-python-gui-for-interactive-phasor-analysis-of-flim-data">Slides about FLUTE: a Python GUI for interactive phasor analysis of FLIM data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#so-geschlossen-wie-notig-so-offen-wie-moglich-datenschutz-beim-umgang-mit-forschungsdaten">So geschlossen wie nötig, so offen wie möglich - Datenschutz beim Umgang mit Forschungsdaten</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#software-citation-with-citation-cff">Software Citation with CITATION.cff</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#source-control-using-git-and-github">Source Control Using Git and GitHub</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data">Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#sustainable-data-stewardship">Sustainable Data Stewardship</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-bioimage-analysis-with-python">Teaching Bioimage Analysis with Python</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#teaching-imagej-fiji">Teaching ImageJ FIJI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ten-simple-rules-for-making-training-materials-fair">Ten simple rules for making training materials FAIR</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-digital-cell-cell-biology-as-a-data-science">The Digital Cell: Cell Biology as a Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-turing-way-guide-for-reproducible-research">The Turing Way: Guide for reproducible research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-crucial-role-of-bioimage-analysts-in-scientific-research-and-publication">The crucial role of bioimage analysts in scientific research and publication</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#things-that-shouldn-t-be-in-file-names-for-1000-alex">Things that shouldn’t be in file names for $1000 Alex</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#thinking-data-management-on-different-scales">Thinking data management on different scales</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#towards-open-and-standardised-imaging-data-an-introduction-to-bio-formats-ome-tiff-and-ome-zarr">Towards open and standardised imaging data: an introduction to Bio-Formats, OME-TIFF, and OME-Zarr</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-theory-trackmate-and-mastodon">Tracking Theory, TrackMate, and Mastodon</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-in-napari">Tracking in napari</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#tracking-of-mitochondria-and-capturing-mitoflashes">Tracking of mitochondria and capturing mitoflashes</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#train-the-trainer-concept-on-research-data-management">Train-the-Trainer Concept on Research Data Management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-computational-skills-in-the-age-of-ai">Training Computational Skills in the Age of AI</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-deep-learning-models-for-vision-compact-course">Training Deep Learning Models for Vision - Compact Course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-concepts-in-research-data-management-and-data-science-with-the-focus-on-health-research">Training concepts in research data management and data science with the focus on health research</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ultrack-i2k-2024-workshop-materials">Ultrack I2K 2024 Workshop Materials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#user-friendly-image-metadata-annotation-tool-workflow-for-omero">User friendly Image metadata annotation tool/workflow for OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#virtual-i2k-2024-multiview-stitcher">Virtual-I2K-2024-multiview-stitcher</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#vision-language-models-for-bio-image-data-science">Vision Language Models for Bio-image Data Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-not-to-do-when-creating-a-data-management-plan-dmp">WHAT NOT TO DO WHEN CREATING A DATA MANAGEMENT PLAN (DMP)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-open-data">What is Open Data?</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#when-data-doesn-t-fit">When Data Doesn’t Fit</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#who-you-gonna-call-data-stewards-to-the-rescue">Who you gonna call? - Data Stewards to the rescue</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workflow-for-user-introduction-into-microscopy-omero-and-data-management-at-center-for-advanced-imaging">Workflow for user introduction into microscopy, OMERO and data management at Center for Advanced imaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-objects-in-2d-and-3d">Working with objects in 2D and 3D</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#working-with-pixels">Working with pixels</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ymia-python-based-event-series-training-material">YMIA - Python-Based Event Series Training Material</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zen-python-workshop">ZEN &amp; Python workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#zidas-2020-introduction-to-deep-learning">ZIDAS 2020 Introduction to Deep Learning</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cidas-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[CIDAS] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cmcb-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[CMCB] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without">[ELMI 2024]  AI’s Dirty Little Secret: Without</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi2025-bridging-communities-with-ome-zarr">[ELMI2025] Bridging communities with OME-Zarr</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi2025-the-road-to-ome-zarr-1-0">[ELMI2025] The Road to OME-Zarr 1.0</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi2025-workshop-fair101-navigating-fair-data-from-principles-to-practice">[ELMI2025] Workshop: FAIR101 - Navigating FAIR data from principles to practice</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#gbi-eoe-vii-five-or-ten-must-have-items-for-making-it-infrastructure-for-managing-bioimage-data">[GBI EOE VII] Five (or ten) must-have items for making IT infrastructure for managing bioimage data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i2k-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[I2K] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi-tech-talk-cloud-based-image-science">[NFDI Tech Talk] Cloud Based Image Science</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard">[SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#webinar-a-journey-to-fair-bioimage-data">[Webinar] A journey to FAIR bioimage data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg">[Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-bioimage-data-management-and-analysis-with-omero">[Workshop] Bioimage data management and analysis with OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-fair-data-handling-for-microscopy-structured-metadata-annotation-in-omero">[Workshop] FAIR data handling for microscopy: Structured metadata annotation in OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-managing-fair-microscopy-data-at-scale-for-universities-and-research-institutions-an-introduction-for-non-imaging-stakeholders">[Workshop] Managing FAIR microscopy data at scale for universities and research institutions: an introduction for non-imaging stakeholders</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-research-data-management-for-microscopy-and-bioimage-analysis">[Workshop] Research Data Management for Microscopy and BioImage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#data-visualization-tutorial">data_visualization_tutorial</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#image-processing-basics">image-processing-basics</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#imagequantification101">imagequantification101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-generative-ai">introduction-to-generative-ai</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#id2">introduction-to-image-analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#napari-scipy2025-workshop">napari-scipy2025-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow-workshop">nextflow-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nextflow101">nextflow101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#numpy-pandas-course">numpy pandas course</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#qupath-workshop">qupath-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#rse-skills-workshop">rse-skills-workshop</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scanpy-tutorials">scanpy-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#scikit-learn-mooc">scikit-learn MOOC</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#shapeanalysis101">shapeanalysis101</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#skimage-tutorials">skimage-tutorials</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#training-resources">training-resources</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Robert Haase, Clément Caporal,... and the NFDI4BioImage Initiative
</p>

  </div>
  
  <div class="footer-item">
    

  </div>
  
  <div class="footer-item">
    <p class="last-updated">
  Last updated on 2025-10-21.
  <br/>
</p>
  </div>
  
  <div class="footer-item">
    
<div class="extra_footer">
  <p>
Copyright: Licensed <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY 4.0</a> unless mentioned otherwise. 
Contributions and feedback are welcome.
</p>

</div>
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>