
<!DOCTYPE html>


<html lang="en" data-content_root="../" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Nfdi4bioimage (45) &#8212; NFDI4BioImage Training Materials</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="../_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="../_static/styles/sphinx-book-theme.css?v=eba8b062" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-design.min.css?v=95c83b7e" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="../_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="../_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="../_static/doctools.js?v=9a2dae69"></script>
    <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="../_static/copybutton.js?v=f281be69"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="../_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'tags/nfdi4bioimage';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Omero (35)" href="omero.html" />
    <link rel="prev" title="Neubias (27)" href="neubias.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
    <meta name="docbuild:last-update" content="2025-03-18"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="../readme.html">
  
  
  
  
  
    
    
      
    
    
    <img src="../_static/logo.png" class="logo__image only-light" alt="NFDI4BioImage Training Materials - Home"/>
    <script>document.write(`<img src="../_static/logo.png" class="logo__image only-dark" alt="NFDI4BioImage Training Materials - Home"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../readme.html">
                    NFDI4BioImage Training Materials
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">What's new</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../whats_new.html">Recently added (10)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Contributing</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../contributing/index.html">How to contribute</a></li>

<li class="toctree-l1"><a class="reference internal" href="../contributing/submit_app.html">Using the Training Materials Submission App</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contributing/format.html">YML format</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By tag</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="artificial_intelligence.html">Artificial intelligence (44)</a></li>
<li class="toctree-l1"><a class="reference internal" href="bioimage_analysis.html">Bioimage analysis (186)</a></li>
<li class="toctree-l1"><a class="reference internal" href="bioinformatics.html">Bioinformatics (18)</a></li>
<li class="toctree-l1"><a class="reference internal" href="data_stewardship.html">Data stewardship (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="fair-principles.html">Fair-principles (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="fiji.html">Fiji (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="imagej.html">Imagej (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="licensing.html">Licensing (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="metadata.html">Metadata (15)</a></li>
<li class="toctree-l1"><a class="reference internal" href="napari.html">Napari (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="neubias.html">Neubias (27)</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Nfdi4bioimage (45)</a></li>
<li class="toctree-l1"><a class="reference internal" href="omero.html">Omero (35)</a></li>
<li class="toctree-l1"><a class="reference internal" href="open_science.html">Open science (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="open_source_software.html">Open source software (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="python.html">Python (71)</a></li>
<li class="toctree-l1"><a class="reference internal" href="reproducibility.html">Reproducibility (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="research_data_management.html">Research data management (132)</a></li>
<li class="toctree-l1"><a class="reference internal" href="sharing.html">Sharing (12)</a></li>
<li class="toctree-l1"><a class="reference internal" href="workflow.html">Workflow (9)</a></li>
<li class="toctree-l1"><a class="reference internal" href="workflow_engine.html">Workflow engine (13)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By content type</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../content_types/blog%20post.html">Blog post (23)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/book.html">Book (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/code.html">Code (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/collection.html">Collection (82)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/data.html">Data (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/document.html">Document (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/documentation.html">Documentation (19)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/event.html">Event (8)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/github%20repository.html">Github repository (54)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/notebook.html">Notebook (55)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/online%20tutorial.html">Online tutorial (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/poster.html">Poster (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/preprint.html">Preprint (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/publication.html">Publication (62)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/slides.html">Slides (83)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/tutorial.html">Tutorial (45)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/video.html">Video (32)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/website.html">Website (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../content_types/workshop.html">Workshop (12)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By license</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../licenses/all_rights_reserved.html">All rights reserved (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/apache-2.0.html">Apache-2.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-2-clause.html">Bsd-2-clause (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/bsd-3-clause.html">Bsd-3-clause (30)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-4.0.html">Cc-by-4.0 (260)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc-by-sa-4.0.html">Cc-by-sa-4.0 (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/cc0-1.0.html">Cc0-1.0 (13)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-2.0.html">Gpl-2.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/gpl-3.0.html">Gpl-3.0 (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/mit.html">Mit (27)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../licenses/unknown.html">Unknown (98)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">By domain</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../domain/biapol.github.io.html">Biapol.github.io (6)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/docs.google.com.html">Docs.google.com (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/doi.org.html">Doi.org (199)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/f1000research.com.html">F1000research.com (11)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/focalplane.biologists.com.html">Focalplane.biologists.com (14)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/git.mpi-cbg.de.html">Git.mpi-cbg.de (7)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/github.com.html">Github.com (132)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.biorxiv.org.html">Www.biorxiv.org (5)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.ebi.ac.uk.html">Www.ebi.ac.uk (10)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.nature.com.html">Www.nature.com (17)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/www.youtube.com.html">Www.youtube.com (25)</a></li>
<li class="toctree-l1"><a class="reference internal" href="../domain/zenodo.org.html">Zenodo.org (187)</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Appendix</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="../statistics/readme.html">Statistics</a></li>
<li class="toctree-l1"><a class="reference internal" href="../export/readme.html">Open data</a></li>
<li class="toctree-l1"><a class="reference internal" href="../imprint.html">Imprint</a></li>

</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/NFDI4BIOIMAGE/training/issues/new?title=Issue%20on%20page%20%2Ftags/nfdi4bioimage.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/tags/nfdi4bioimage.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Nfdi4bioimage (45)</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-cloud-optimized-storage-for-interactive-access-of-large-arrays">A Cloud-Optimized Storage for Interactive Access of Large Arrays</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-journey-to-fair-microscopy-data">A journey to FAIR microscopy data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#angebote-der-nfdi-fur-die-forschung-im-bereich-zoologie">Angebote der NFDI für die Forschung im Bereich Zoologie</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">Challenges and opportunities for bio-image analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collaborative-working-and-version-control-with-git-hub">Collaborative Working and Version Control with git[hub]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#engineering-a-software-environment-for-research-data-management-of-microscopy-image-data-in-a-core-facility">Engineering a Software Environment for Research Data Management of Microscopy Image Data in a Core Facility</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#key-value-pair-template-for-annotation-in-omero-for-light-microscopy-data-acquired-with-axioscan7-core-facility-cellular-imaging-cfci">Key-Value pair template for annotation in OMERO for light microscopy data acquired with AxioScan7 - Core Facility Cellular Imaging (CFCI)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#key-value-pair-template-for-annotation-of-datasets-in-omero-perikles-study">Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi-daten-als-gemeinsames-gut-fur-exzellente-forschung-organisiert-durch-die-wissenschaft-in-deutschland">NFDI - Daten als gemeinsames Gut für exzellente Forschung, organisiert durch die Wissenschaft in Deutschland.</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage">NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data">NFDI4BIOIMAGE - An Initiative for a National Research Data Infrastructure for Microscopy Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-data-management-illustrations-by-henning-falk">NFDI4BIOIMAGE data management illustrations by Henning Falk</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-perspective-for-a-national-bioimaging-standard">NFDI4BIOIMAGE: Perspective for a national bioimaging standard</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-calendar-2024-october-original-image">NFDI4Bioimage Calendar 2024 October; original image</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome-ngff-a-next-generation-file-format-for-expanding-bioimaging-data-access-strategies">OME-NGFF: a next-generation file format for expanding bioimaging data-access strategies</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome2024-ngff-challenge-results">OME2024 NGFF Challenge Results</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-for-microscopy-research-data-management">OMERO for microscopy research data management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#overview-of-the-galaxy-omero-suite-upload-images-and-metadata-in-omero-using-galaxy">Overview of the Galaxy OMERO-suite - Upload images and metadata in OMERO using Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey">Research data management for bioimaging: the 2021 NFDI4BIOIMAGE community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-a-data-management-infrastructure-for-bioimaging">Setting up a data management infrastructure for bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data">Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-information-infrastructure-for-bioimage-data-i3d-bio-project-to-advance-fair-microscopy-data-management-for-the-community">The Information Infrastructure for BioImage Data (I3D:bio) project to advance FAIR microscopy data management for the community</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-role-of-helmholtz-centers-in-nfdi4bioimage-a-national-consortium-enhancing-fair-data-management-for-microscopy-and-bioimage-analysis">The role of Helmholtz Centers in NFDI4BIOIMAGE - A national consortium enhancing FAIR data management for microscopy and bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#thinking-data-management-on-different-scales">Thinking data management on different scales</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#towards-preservation-of-life-science-data-with-nfdi4bioimage">Towards Preservation of Life Science Data with NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#welcome-to-bioimage-town">Welcome to BioImage Town</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#who-you-gonna-call-data-stewards-to-the-rescue">Who you gonna call? - Data Stewards to the rescue</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cidas-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[CIDAS] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cmcb-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[CMCB] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#community-meeting-2024-overview-team-image-data-analysis-and-management">[Community Meeting 2024] Overview Team Image Data Analysis and Management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without">[ELMI 2024]  AI’s Dirty Little Secret: Without</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard">[SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg">[Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-bioimage-data-management-and-analysis-with-omero">[Workshop] Bioimage data management and analysis with OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-research-data-management-for-microscopy-and-bioimage-analysis">[Workshop] Research Data Management for Microscopy and BioImage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dmtxsamplecreator">dmtxSampleCreator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome2024-ngff-challenge">ome2024-ngff-challenge</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="nfdi4bioimage-45">
<h1>Nfdi4bioimage (45)<a class="headerlink" href="#nfdi4bioimage-45" title="Link to this heading">#</a></h1>
<section id="a-cloud-optimized-storage-for-interactive-access-of-large-arrays">
<h2>A Cloud-Optimized Storage for Interactive Access of Large Arrays<a class="headerlink" href="#a-cloud-optimized-storage-for-interactive-access-of-large-arrays" title="Link to this heading">#</a></h2>
<p>Josh Moore, Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Publication, Conference Abstract</p>
<p><a class="reference external" href="https://doi.org/10.52825/cordi.v1i.285">https://doi.org/10.52825/cordi.v1i.285</a></p>
</section>
<hr class="docutils" />
<section id="a-journey-to-fair-microscopy-data">
<h2>A journey to FAIR microscopy data<a class="headerlink" href="#a-journey-to-fair-microscopy-data" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters, Janina Hanne, Christian Schmidt</p>
<p>Published 2023-05-03</p>
<p>Licensed CC-BY-4.0</p>
<p>Oral presentation, 32nd MoMAN “From Molecules to Man” Seminar, Ulm, online. Monday February 6th, 2023</p>
<p>Abstract:</p>
<p>Research data management is essential in nowadays research, and one of the big opportunities to accelerate collaborative and innovative scientific projects. To achieve this goal, all our data needs to be FAIR (findable, accessible, interoperable, reproducible). For data acquired on microscopes, however, a common ground for FAIR data sharing is still to be established. Plenty of work on file formats, data bases, and training needs to be performed to highlight the value of data sharing and exploit its potential for bioimaging data.</p>
<p>In this presentation, Stefanie Weidtkamp-Peters will introduce the challenges for bioimaging data management, and the necessary steps to achieve data FAIRification. German BioImaging - GMB e.V., together with other institutions, contributes to this endeavor. Janina Hanne will present how the network of imaging core facilities, research groups and industry partners is key to the German bioimaging community’s aligned collaboration toward FAIR bioimaging data. These activities have paved the way for two data management initiatives in Germany: I3D:bio (Information Infrastructure for BioImage Data) and NFDI4BIOIMAGE, a consortium of the National Research Data Infrastructure. Christian Schmidt will introduce the goals and measures of these initiatives to the benefit of imaging scientist’s work and everyday practice.  </p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/7890311">https://zenodo.org/records/7890311</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.7890311">https://doi.org/10.5281/zenodo.7890311</a></p>
</section>
<hr class="docutils" />
<section id="angebote-der-nfdi-fur-die-forschung-im-bereich-zoologie">
<h2>Angebote der NFDI für die Forschung im Bereich Zoologie<a class="headerlink" href="#angebote-der-nfdi-fur-die-forschung-im-bereich-zoologie" title="Link to this heading">#</a></h2>
<p>Birgitta König-Ries, Robert Haase, Daniel Nüst, Konrad Förstner, Judith Sophie Engel</p>
<p>Published 2024-12-04</p>
<p>Licensed CC-BY-4.0</p>
<p>In diesem Slidedeck geben wir einen Einblick in Angebote und Dienste der Nationalen Forschungsdaten Infrastruktur (NFDI), die Relevant für die Zoologie und angrenzende Disziplinen relevant sein könnten.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/14278058">https://zenodo.org/records/14278058</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14278058">https://doi.org/10.5281/zenodo.14278058</a></p>
</section>
<hr class="docutils" />
<section id="bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">
<h2>Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities<a class="headerlink" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities" title="Link to this heading">#</a></h2>
<p>Cornelia Wetzker</p>
<p>Published 2023-11-08</p>
<p>Licensed CC-BY-4.0</p>
<p>This presentation gives a short outline of the complexity of data and metadata in the bioimaging universe. It introduces NFDI4BIOIMAGE as a newly formed consortium as part of the German ‘Nationale Forschungsdateninfrastruktur’ (NFDI) and its goals and tools for data management including its current members on TU Dresden campus.  </p>
<p>Tags: Research Data Management, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/records/10083555">https://zenodo.org/records/10083555</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10083555">https://doi.org/10.5281/zenodo.10083555</a></p>
</section>
<hr class="docutils" />
<section id="challenges-and-opportunities-for-bio-image-analysis-core-facilities">
<h2>Challenges and opportunities for bio-image analysis core-facilities<a class="headerlink" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://f1000research.com/slides/12-1054">https://f1000research.com/slides/12-1054</a></p>
</section>
<hr class="docutils" />
<section id="collaborative-working-and-version-control-with-git-hub">
<h2>Collaborative Working and Version Control with git[hub]<a class="headerlink" href="#collaborative-working-and-version-control-with-git-hub" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-01-10</p>
<p>Licensed CC-BY-4.0</p>
<p>This slide deck introduces the version control tool git, related terminology and the Github Desktop app for managing files in Git[hub] repositories. We furthermore dive into:* Working with repositories* Collaborative with others* Github-Zenodo integration* Github pages* Artificial Intelligence answering Github Issues</p>
<p>Tags: Nfdi4Bioimage, Globias, Research Data Management, Research Software Management</p>
<p><a class="reference external" href="https://zenodo.org/records/14626054">https://zenodo.org/records/14626054</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14626054">https://doi.org/10.5281/zenodo.14626054</a></p>
</section>
<hr class="docutils" />
<section id="engineering-a-software-environment-for-research-data-management-of-microscopy-image-data-in-a-core-facility">
<h2>Engineering a Software Environment for Research Data Management of Microscopy Image Data in a Core Facility<a class="headerlink" href="#engineering-a-software-environment-for-research-data-management-of-microscopy-image-data-in-a-core-facility" title="Link to this heading">#</a></h2>
<p>Kunis</p>
<p>Published 2022-05-30</p>
<p>This thesis deals with concepts and solutions in the field of data management in everyday scientific life for image data from microscopy. The focus of the formulated requirements has so far been on published data, which represent only a small subset of the data generated in the scientific process. More and more, everyday research data are moving into the focus of the principles for the management of research data that were formulated early on (FAIR-principles). The adequate management of this mostly multimodal data is a real challenge in terms of its heterogeneity and scope. There is a lack of standardised and established workflows and also the software solutions available so far do not adequately reflect the special requirements of this area. However, the success of any data management process depends heavily on the degree of integration into the daily work routine. Data management must, as far as possible, fit seamlessly into this process. Microscopy data in the scientific process is embedded in pre-processing, which consists of preparatory laboratory work and the analytical evaluation of the microscopy data. In terms of volume, the image data often form the largest part of data generated within this entire research process. In this paper, we focus on concepts and techniques related to the handling and description of this image data and address the necessary basics. The aim is to improve the embedding of the existing data management solution for image data (OMERO) into the everyday scientific work. For this purpose, two independent software extensions for OMERO were implemented within the framework of this thesis: OpenLink and MDEmic. OpenLink simplifies the access to the data stored in the integrated repository in order to feed them into established workflows for further evaluations and enables not only the internal but also the external exchange of data without weakening the advantages of the data repository. The focus of the second implemented software solution, MDEmic, is on the capturing of relevant metadata for microscopy. Through the extended metadata collection, a corresponding linking of the multimodal data by means of a unique description and the corresponding semantic background is aimed at. The configurability of MDEmic is designed to address the currently very dynamic development of underlying concepts and formats. The main goal of MDEmic is to minimise the workload and to automate processes. This provides the scientist with a tool to handle this complex and extensive task of metadata acquisition for microscopic data in a simple way. With the help of the software, semantic and syntactic standardisation can take place without the scientist having to deal with the technical concepts. The generated metadata descriptions are automatically integrated into the image repository and, at the same time, can be transferred by the scientists into formats that are needed when publishing the data.</p>
<p>Tags: Nfdi4Bioimage, Research Data Managementv</p>
<p><a class="reference external" href="https://zenodo.org/records/6905931">https://zenodo.org/records/6905931</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.6905931">https://doi.org/10.5281/zenodo.6905931</a></p>
</section>
<hr class="docutils" />
<section id="i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">
<h2>I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training<a class="headerlink" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Michele Bortolomeazzi, Tom Boissonnet, Carsten Fortmann-Grote, Julia Dohle, Peter Zentis, Niraj Kandpal, Susanne Kunis, Thomas Zobel, Stefanie Weidtkamp-Peters, Elisa Ferrando-May</p>
<p>Published 2023-11-13</p>
<p>Licensed CC-BY-4.0</p>
<p>The open-source software OME Remote Objects (OMERO) is a data management software that allows storing, organizing, and annotating bioimaging/microscopy data. OMERO has become one of the best-known systems for bioimage data management in the bioimaging community. The Information Infrastructure for BioImage Data (I3D:bio) project facilitates the uptake of OMERO into research data management (RDM) practices at universities and research institutions in Germany. Since the adoption of OMERO into researchers’ daily routines requires intensive training, a broad portfolio of training resources for OMERO is an asset. On top of using the OMERO guides curated by the Open Microscopy Environment Consortium (OME) team, imaging core facility staff at institutions where OMERO is used often prepare additional material tailored to be applicable for their own OMERO instances. Based on experience gathered in the Research Data Management for Microscopy group (RDM4mic) in Germany, and in the use cases in the I3D:bio project, we created a set of reusable, adjustable, openly available slide decks to serve as the basis for tailored training lectures, video tutorials, and self-guided instruction manuals directed at beginners in using OMERO. The material is published as an open educational resource complementing the existing resources for OMERO contributed by the community.</p>
<p>Tags: OMERO, Research Data Management, Nfdi4Bioimage, I3Dbio</p>
<p>Content type: Slides, Video</p>
<p><a class="reference external" href="https://zenodo.org/records/8323588">https://zenodo.org/records/8323588</a></p>
<p><a class="reference external" href="https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU">https://www.youtube.com/playlist?list=PL2k-L-zWPoR7SHjG1HhDIwLZj0MB_stlU</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8323588">https://doi.org/10.5281/zenodo.8323588</a></p>
</section>
<hr class="docutils" />
<section id="key-value-pair-template-for-annotation-in-omero-for-light-microscopy-data-acquired-with-axioscan7-core-facility-cellular-imaging-cfci">
<h2>Key-Value pair template for annotation in OMERO for light microscopy data acquired with AxioScan7 - Core Facility Cellular Imaging (CFCI)<a class="headerlink" href="#key-value-pair-template-for-annotation-in-omero-for-light-microscopy-data-acquired-with-axioscan7-core-facility-cellular-imaging-cfci" title="Link to this heading">#</a></h2>
<p>Silke Tulok, Anja Nobst, Anett Jannasch, Tom Boissonnet, Gunar Fabig</p>
<p>Published 2024-06-28</p>
<p>Licensed CC-BY-4.0</p>
<p>This Key-Value pair template is used for the data documentation during imaging experiments and the later data annotation in OMERO. It is tailored for the usage and image acquisition at the slide scanning system Zeiss AxioScan 7 in the Core Facility Cellular Imaging (CFCI). It contains important metadata of the imaging experiment, which are not saved in the corresponding imaging files. All users of the Core Facility Cellular Imaging are trained to use that file to document their imaging parameters directly during the data acquisition with the possibility for a later upload to OMERO. Furthermore, there is a corresponding public example image used in the publication “Setting up an institutional OMERO environment for bioimage data: perspectives from both facility staff and users” and is available here:
<a class="reference external" href="https://omero.med.tu-dresden.de/webclient/?show=image-33248">https://omero.med.tu-dresden.de/webclient/?show=image-33248</a>
This template was developed by the CFCI staff during the setup and usage of the AxioScan 7 and is based on the REMBI recommendations (<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015</a>).
With this template it is possible to create a csv-file, that can be used to annotate an image or dataset in OMERO using the annotation script (<a class="github reference external" href="https://github.com/ome/omero-scripts/blob/develop/omero/annotation_scripts/">ome/omero-scripts</a>).
How to use:</p>
<p>fill the template sheet  with your metadata
select and copy the data range containing the Keys and Values
open a new excel sheet and paste transpose in cell A1 
Important: cell A1 contains always the name ‘dataset’ and cell A2 contains the exact name of the image/dataset, which should be annotated in OMERO
save the new excel sheet in csv-file (comma separated values) format</p>
<p>An example can be seen in sheet 3 ‘csv_AxioScan’.
Important note: The code has to be 8-Bit UCS transformation format (UTF-8) otherwise several characters (for example µ, %,°) might be not able to decode by the annotation script. We encountered this issue with old Microsoft-Office versions (MS Office 2016). 
Note: By filling the values in the excel sheet, avoid the usage of comma as decimal delimiter.
See cross reference:
10.5281/zenodo.12547566 Key-Value pair template for annotation of datasets in OMERO for light- and electron microscopy data within the research group of Prof. Mueller-Reichert
10.5281/zenodo.12546808 Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/12578084">https://zenodo.org/records/12578084</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.12578084">https://doi.org/10.5281/zenodo.12578084</a></p>
</section>
<hr class="docutils" />
<section id="key-value-pair-template-for-annotation-of-datasets-in-omero-perikles-study">
<h2>Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)<a class="headerlink" href="#key-value-pair-template-for-annotation-of-datasets-in-omero-perikles-study" title="Link to this heading">#</a></h2>
<p>Anett Jannasch, Silke Tulok, Vanessa Aphaia Fiona Fuchs, Tom Boissonnet, Christian Schmidt, Michele Bortolomeazzi, Gunar Fabig, Chukwuebuka Okafornta</p>
<p>Published 2024-06-26</p>
<p>Licensed CC-BY-4.0</p>
<p>This is a Key-Value pair template used for the annotation of datasets in OMERO. It is tailored for a research study (PERIKLES project) on the biocompatibility of newly designed biomaterials out of pericardial tissue for cardiovascular substitutes (<a class="reference external" href="https://doi.org/10.1063/5.0182672">https://doi.org/10.1063/5.0182672</a>) conducted in the research department of Cardiac Surgery at the Faculty of Medicine Carl Gustav Carus at the Technische Universität Dresden . A corresponding public example dataset is used in the publication “Setting up an institutional OMERO environment for bioimage data: perspectives from both facility staff and users” and is available here
(<a class="reference external" href="https://omero.med.tu-dresden.de/webclient/?show=dataset-1557">https://omero.med.tu-dresden.de/webclient/?show=dataset-1557</a>).
The template is based on the REMBI recommendations (<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8606015</a>) and it was developed during the PoL-Bio-Image Analysis Symposium in Dresden Aug 28th- Sept 1th 2023. 
With this template it is possible to create a csv-file, that can be used to annotate a dataset in OMERO using the annotation script (<a class="github reference external" href="https://github.com/ome/omero-scripts/blob/develop/omero/annotation_scripts/">ome/omero-scripts</a>).
How to use:
select and copy the data range containing Keys and Values
open a new excel sheet and paste transpose in column B1
type in A1 ‘dataset’
insert in A2 the exact name of the dataset, which should be annotated in OMERO
save the new excel sheet in csv- (comma seperated values) file format</p>
<p>Example can be seen in sheet 1 ‘csv import’. Important note; the code has to be 8-Bit UCS transformation format (UTF-8) otherwise several characters (for example µ, %,°) might not be able to decode by the annotation script. We encountered this issue with old Microsoft Office versions (e.g. MS Office 2016). 
Note: By filling the values in the excel sheet, avoid the usage of decimal delimiter.
 
See cross reference:
10.5281/zenodo.12547566 Key-Value pair template for annotation of datasets in OMERO (light- and electron microscopy data within the research group of Prof. Mueller-Reichert)
10.5281/zenodo.12578084 Key-Value pair template for annotation in OMERO for light microscopy data acquired with AxioScan7 - Core Facility Cellular Imaging (CFCI)</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/12546808">https://zenodo.org/records/12546808</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.12546808">https://doi.org/10.5281/zenodo.12546808</a></p>
</section>
<hr class="docutils" />
<section id="nfdi-daten-als-gemeinsames-gut-fur-exzellente-forschung-organisiert-durch-die-wissenschaft-in-deutschland">
<h2>NFDI - Daten als gemeinsames Gut für exzellente Forschung, organisiert durch die Wissenschaft in Deutschland.<a class="headerlink" href="#nfdi-daten-als-gemeinsames-gut-fur-exzellente-forschung-organisiert-durch-die-wissenschaft-in-deutschland" title="Link to this heading">#</a></h2>
<p>Licensed UNKNOWN</p>
<p>Schritt für Schritt verbessern wir die Nutzungsmöglichkeiten von Daten für Wissenschaft und Gesellschaft. Durch unser Zusammenwirken im NFDI-Verein entsteht eine Dachorganisation für das Forschungsdatenmanagement in allen Wissenschaftszweigen.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Website</p>
<p><a class="reference external" href="https://www.nfdi.de/">https://www.nfdi.de/</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage">
<h2>NFDI4BIOIMAGE<a class="headerlink" href="#nfdi4bioimage" title="Link to this heading">#</a></h2>
<p>Carsten Fortmann-Grote</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation was given at the 2nd MPG-NFDI Workshop on April 18th about e NFDI4BIOIMAGE Consortium, FAIRification of Image (meta)data, Zarr, RFC, Training (TA5), contributing.</p>
<p>Tags: Research Data Management, Bioimage Analysis, FAIR-Principles, Zarr, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.11031746">https://zenodo.org/doi/10.5281/zenodo.11031746</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data">
<h2>NFDI4BIOIMAGE - An Initiative for a National Research Data Infrastructure for Microscopy Data<a class="headerlink" href="#nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Elisa Ferrando-May</p>
<p>Published 2021-04-29</p>
<p>Licensed CCY-BY-SA-4.0</p>
<p>Align existing and establish novel services &amp; solutions for data management tasks throughout the bioimage data lifecycle.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Conference Abstract, Slides</p>
<p><a class="reference external" href="https://doi.org/10.11588/heidok.00029489">https://doi.org/10.11588/heidok.00029489</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023" title="Link to this heading">#</a></h2>
<p>Stefanie Weidtkamp-Peters</p>
<p>Licensed CC-BY-4.0</p>
<p>NFDI4BIOIMAGE core mission, bioimage data challenge, task areas, FAIR bioimage workflows.</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.8070038">https://doi.org/10.5281/zenodo.8070038</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">
<h2>NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]<a class="headerlink" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg" title="Link to this heading">#</a></h2>
<p>Riccardo Massei</p>
<p>Licensed CC-BY-4.0</p>
<p>NFDI4BIOIMAGE is a consortium within the framework of the National Research Data Infrastructure (NFDI) in Germany. In this talk, the consortium and the contribution to the work programme by the Helmholtz Centre for Environmental Research (UFZ) in Leipzig are outlined.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8414318">https://zenodo.org/doi/10.5281/zenodo.8414318</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-data-management-illustrations-by-henning-falk">
<h2>NFDI4BIOIMAGE data management illustrations by Henning Falk<a class="headerlink" href="#nfdi4bioimage-data-management-illustrations-by-henning-falk" title="Link to this heading">#</a></h2>
<p>NFDI4BIOIMAGE Consortium</p>
<p>Published 2024-11-29</p>
<p>Licensed CC-BY-4.0</p>
<p>These illustrations were contracted by the Heinrich Heine University Düsseldorf in the frame of the consortium NFDI4BIOIMAGE from Henning Falk for the purpose of education and public outreach. The illustrations are free to use under a CC-BY 4.0 license.AttributionPlease include an attribution similar to: “Data annoation matters”, NFDI4BIOIMAGE Consortium (2024): NFDI4BIOIMAGE data management illustrations by Henning Falk, Zenodo, <a class="reference external" href="https://doi.org/10.5281/zenodo.14186100">https://doi.org/10.5281/zenodo.14186100</a>, is used under a CC-BY 4.0 license. Modifications to this illustration include cropping.
 </p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/14186101">https://zenodo.org/records/14186101</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14186101">https://doi.org/10.5281/zenodo.14186101</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-perspective-for-a-national-bioimaging-standard">
<h2>NFDI4BIOIMAGE: Perspective for a national bioimaging standard<a class="headerlink" href="#nfdi4bioimage-perspective-for-a-national-bioimaging-standard" title="Link to this heading">#</a></h2>
<p>Josh Moore, Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://ceur-ws.org/Vol-3415/paper-27.pdf">https://ceur-ws.org/Vol-3415/paper-27.pdf</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon">
<h2>NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon)<a class="headerlink" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon" title="Link to this heading">#</a></h2>
<p>Mohamed M. Abdrabbou, Mehrnaz Babaki, Tom Boissonnet, Michele Bortolomeazzi, Eik Dahms, Vanessa A. F. Fuchs, Moritz Hoevels, Niraj Kandpal, Christoph Möhl, Joshua A. Moore, Astrid Schauss, Andrea Schrader, Torsten Stöter, Julia Thönnißen, Monica Valencia-S., H. Lukas Weil, Jens Wendt and Peter Zentis</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Arc, Dataplant, Hackathon, Nfdi4Bioimage, OMERO, Python, Research Data Management</p>
<p>Content type: Event, Publication, Documentation</p>
<p><a class="github reference external" href="https://github.com/NFDI4BIOIMAGE/Cologne-Hackathon-2023">NFDI4BIOIMAGE/Cologne-Hackathon-2023</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10609770">https://doi.org/10.5281/zenodo.10609770</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">
<h2>NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)<a class="headerlink" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository" title="Link to this heading">#</a></h2>
<p>Mohamed Abdrabbou, Mehrnaz Babaki, Tom Boissonnet, Michele Bortolomeazzi, Eik Dahms, Vanessa Fuchs, A. F. Moritz Hoevels, Niraj Kandpal, Christoph Möhl, Joshua A. Moore, Astrid Schauss, Andrea Schrader, Torsten Stöter, Julia Thönnißen, Monica Valencia-S., H. Lukas Weil, Jens Wendt, Peter Zentis</p>
<p>Licensed CC-BY-4.0</p>
<p>This repository documents the first NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon), where topics like ‘Interoperability’, ‘REMBI / Mapping’, and ‘Neuroglancer (OMERO / zarr)’ were explored through collaborative discussions and workflow sessions, culminating in reports that bridge NFDI4Bioimage to DataPLANT. Funded by various DFG initiatives, this event emphasized documentation and use cases, contributing preparatory work for future interoperability projects at the 2nd de.NBI BioHackathon in Bielefeld.</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Github Repository</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10609770">https://zenodo.org/doi/10.5281/zenodo.10609770</a></p>
</section>
<hr class="docutils" />
<section id="nfdi4bioimage-calendar-2024-october-original-image">
<h2>NFDI4Bioimage Calendar 2024 October; original image<a class="headerlink" href="#nfdi4bioimage-calendar-2024-october-original-image" title="Link to this heading">#</a></h2>
<p>Christian Jüngst, Peter Zentis</p>
<p>Published 2024-09-25</p>
<p>Licensed CC-BY-4.0</p>
<p>Raw microscopy image from the NFDI4Bioimage calendar October 2024</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/13837146">https://zenodo.org/records/13837146</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13837146">https://doi.org/10.5281/zenodo.13837146</a></p>
</section>
<hr class="docutils" />
<section id="ome-ngff-a-next-generation-file-format-for-expanding-bioimaging-data-access-strategies">
<h2>OME-NGFF: a next-generation file format for expanding bioimaging data-access strategies<a class="headerlink" href="#ome-ngff-a-next-generation-file-format-for-expanding-bioimaging-data-access-strategies" title="Link to this heading">#</a></h2>
<p>Josh Moore, Chris Allan, Sébastien Besson, jean-marie burel, Erin Diel, David Gault, Kevin Kozlowski, Dominik Lindner, Melissa Linkert, Trevor Manz, Will Moore, Constantin Pape, Christian Tischer, Jason R. Swedlow</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://www.nature.com/articles/s41592-021-01326-w">https://www.nature.com/articles/s41592-021-01326-w</a></p>
</section>
<hr class="docutils" />
<section id="ome2024-ngff-challenge-results">
<h2>OME2024 NGFF Challenge Results<a class="headerlink" href="#ome2024-ngff-challenge-results" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2024-11-01</p>
<p>Licensed CC-BY-4.0</p>
<p>Presented at the 2024 FoundingGIDE event in Okazaki, Japan: <a class="reference external" href="https://founding-gide.eurobioimaging.eu/event/foundinggide-community-event-2024/">https://founding-gide.eurobioimaging.eu/event/foundinggide-community-event-2024/</a>
Note: much of the presentation was a demonstration of the OME2024-NGFF-Challenge – <a class="reference external" href="https://ome.github.io/ome2024-ngff-challenge/">https://ome.github.io/ome2024-ngff-challenge/</a> especially of querying an extraction of the metadata (<a class="github reference external" href="https://github.com/ome/ome2024-ngff-challenge-metadata">ome/ome2024-ngff-challenge-metadata</a>)
 </p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/14234608">https://zenodo.org/records/14234608</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14234608">https://doi.org/10.5281/zenodo.14234608</a></p>
</section>
<hr class="docutils" />
<section id="omero-for-microscopy-research-data-management">
<h2>OMERO for microscopy research data management<a class="headerlink" href="#omero-for-microscopy-research-data-management" title="Link to this heading">#</a></h2>
<p>Thomas Zobel, Sarah Weischner, Jens Wendt</p>
<p>Licensed ALL RIGHTS RESERVED</p>
<p>A use case example from the Münster Imaging Network</p>
<p>Tags: Nfdi4Bioimage, OMERO, Research Data Management</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://analyticalscience.wiley.com/do/10.1002/was.0004000267/">https://analyticalscience.wiley.com/do/10.1002/was.0004000267/</a></p>
</section>
<hr class="docutils" />
<section id="overview-of-the-galaxy-omero-suite-upload-images-and-metadata-in-omero-using-galaxy">
<h2>Overview of the Galaxy OMERO-suite - Upload images and metadata in OMERO using Galaxy<a class="headerlink" href="#overview-of-the-galaxy-omero-suite-upload-images-and-metadata-in-omero-using-galaxy" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Björn Grüning</p>
<p>Published 2024-12-02</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: OMERO, Galaxy, Metadata, Nfdi4Bioimage</p>
<p>Content type: Tutorial, Framework, Workflow</p>
<p><a class="reference external" href="https://training.galaxyproject.org/training-material/topics/imaging/tutorials/omero-suite/tutorial.html">https://training.galaxyproject.org/training-material/topics/imaging/tutorials/omero-suite/tutorial.html</a></p>
</section>
<hr class="docutils" />
<section id="research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey">
<h2>Research data management for bioimaging: the 2021 NFDI4BIOIMAGE community survey<a class="headerlink" href="#research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Janina Hanne, Josh Moore, Christian Meesters, Elisa Ferrando-May, Stefanie Weidtkamp-Peters, members of the NFDI4BIOIMAGE initiative</p>
<p>Licensed CC-BY-4.0</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://f1000research.com/articles/11-638">https://f1000research.com/articles/11-638</a></p>
</section>
<hr class="docutils" />
<section id="setting-up-a-data-management-infrastructure-for-bioimaging">
<h2>Setting up a data management infrastructure for bioimaging<a class="headerlink" href="#setting-up-a-data-management-infrastructure-for-bioimaging" title="Link to this heading">#</a></h2>
<p>Susanne Kunis, Karen Bernhardt, Michael Hensel</p>
<p>Licensed UNKNOWN</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Publication</p>
<p><a class="reference external" href="https://doi.org/10.1515/hsz-2022-0304">https://doi.org/10.1515/hsz-2022-0304</a></p>
</section>
<hr class="docutils" />
<section id="structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data">
<h2>Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data<a class="headerlink" href="#structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data" title="Link to this heading">#</a></h2>
<p>Sarah Weischer, Jens Wendt, Thomas Zobel</p>
<p>Published 2022-07-12</p>
<p>Licensed CC-BY-4.0</p>
<p>Provides an overview of contexts, frameworks, and models from the world of bioimage data as well as metadata. Visualizes the techniques for structuring this data as Linked Data. (Walkthrough Video: <a class="reference external" href="https://doi.org/10.5281/zenodo.7018928">https://doi.org/10.5281/zenodo.7018928</a> )</p>
<p>Content:</p>
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>Types of metadata
Data formats
Data Models Microscopy Data
Tools to edit/gather metadata
ISA Framework
FDO Framework
Ontology
RDF
JSON-LD
SPARQL
Knowledge Graph
Linked Data
Smart Data
...
</pre></div>
</div>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/7018750">https://zenodo.org/records/7018750</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.7018750">https://doi.org/10.5281/zenodo.7018750</a></p>
</section>
<hr class="docutils" />
<section id="the-information-infrastructure-for-bioimage-data-i3d-bio-project-to-advance-fair-microscopy-data-management-for-the-community">
<h2>The Information Infrastructure for BioImage Data (I3D:bio) project to advance FAIR microscopy data management for the community<a class="headerlink" href="#the-information-infrastructure-for-bioimage-data-i3d-bio-project-to-advance-fair-microscopy-data-management-for-the-community" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Michele Bortolomeazzi, Tom Boissonnet, Julia Dohle, Tobias Wernet, Janina Hanne, Roland Nitschke, Susanne Kunis, Karen Bernhardt, Stefanie Weidtkamp-Peters, Elisa Ferrando-May</p>
<p>Published 2024-03-04</p>
<p>Licensed CC-BY-4.0</p>
<p>Research data management (RDM) in microscopy and image analysis is a challenging task. Large files in proprietary formats, complex N-dimensional array structures, and various metadata models and formats can make image data handling inconvenient and difficult. For data organization, annotation, and sharing, researchers need solutions that fit everyday practice and comply with the FAIR (Findable, Accessible, Interoperable, Reusable) principles. International community-based efforts have begun creating open data models (OME), an open file format and translation library (OME-TIFF, Bio-Formats), data management software platforms, and microscopy metadata recommendations and annotation tools. Bringing these developments into practice requires support and training. Iterative feedback and tool improvement is needed to foster practical adoption by the scientific community. The Information Infrastructure for BioImage Data (I3D:bio) project works on guidelines, training resources, and practical assistance for FAIR microscopy RDM adoption with a focus on the management platform OMERO and metadata annotations.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/10805204">https://zenodo.org/records/10805204</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10805204">https://doi.org/10.5281/zenodo.10805204</a></p>
</section>
<hr class="docutils" />
<section id="the-role-of-helmholtz-centers-in-nfdi4bioimage-a-national-consortium-enhancing-fair-data-management-for-microscopy-and-bioimage-analysis">
<h2>The role of Helmholtz Centers in NFDI4BIOIMAGE - A national consortium enhancing FAIR data management for microscopy and bioimage analysis<a class="headerlink" href="#the-role-of-helmholtz-centers-in-nfdi4bioimage-a-national-consortium-enhancing-fair-data-management-for-microscopy-and-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Christian Schmidt, Michele Bortolomeazzi, Julia Thoennissen, Jan Bumberger, Timo Dickscheid, Jan-Philipp Mallm, Elisa Ferrando-May</p>
<p>Published 2024-06-06</p>
<p>Licensed CC-BY-4.0</p>
<p>Germany’s National Research Data Infrastructure (NFDI) aims to establish a sustained, cross-disciplinary research data management (RDM) infrastructure that enables researchers to handle FAIR (findable, accessible, interoperable, reusable) data. While FAIR principles have been adopted by funders, policymakers, and publishers, their practical implementation remains an ongoing effort. In the field of bio-imaging, harmonization of data formats, metadata ontologies, and open data repositories is necessary to achieve FAIR data. The NFDI4BIOIMAGE was established to address these issues and develop tools and best practices to facilitate FAIR microscopy and image analysis data in alignment with international community activities. The consortium operates through its Data Stewards team to provide expertise and direct support to help overcome RDM challenges. The three Helmholtz Centers in NFDI4BIOIMAGE aim to collaborate closely with other centers and initiatives, such as HMC, Helmholtz AI, and HIP. Here we present NFDI4BIOIMAGE’s work and its significance for research in Helmholtz and beyond</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/11501662">https://zenodo.org/records/11501662</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11501662">https://doi.org/10.5281/zenodo.11501662</a></p>
</section>
<hr class="docutils" />
<section id="thinking-data-management-on-different-scales">
<h2>Thinking data management on different scales<a class="headerlink" href="#thinking-data-management-on-different-scales" title="Link to this heading">#</a></h2>
<p>Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Presentation given at PoL BioImage Analysis Symposium Dresden 2023</p>
<p>Tags: Research Data Management, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.8329305">https://zenodo.org/doi/10.5281/zenodo.8329305</a></p>
</section>
<hr class="docutils" />
<section id="towards-preservation-of-life-science-data-with-nfdi4bioimage">
<h2>Towards Preservation of Life Science Data with NFDI4BIOIMAGE<a class="headerlink" href="#towards-preservation-of-life-science-data-with-nfdi4bioimage" title="Link to this heading">#</a></h2>
<p>Robert Haase</p>
<p>Published 2024-09-03</p>
<p>Licensed CC-BY-4.0</p>
<p>This talk will present the initiatives of the NFDI4BioImage consortium aimed at the long-term preservation of life science data. We will discuss our efforts to establish metadata standards, which are crucial for ensuring data reusability and integrity. The development of sustainable infrastructure is another key focus, enabling seamless data integration and analysis in the cloud. We will take a look at how we manage training materials and communicate with our community. Through these actions, NFDI4BioImage seeks to enable FAIR bioimage data management for German researchers, across disciplines and embedded in the international framework.</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/13640979">https://zenodo.org/records/13640979</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13640979">https://doi.org/10.5281/zenodo.13640979</a></p>
</section>
<hr class="docutils" />
<section id="welcome-to-bioimage-town">
<h2>Welcome to BioImage Town<a class="headerlink" href="#welcome-to-bioimage-town" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Licensed CC-BY-4.0</p>
<p>Welcome at NFDI4BIOIMAGE All-Hands Meeting in Düsseldorf, Germany, October 16, 2023</p>
<p>Tags: OMERO, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10008464">https://zenodo.org/doi/10.5281/zenodo.10008464</a></p>
</section>
<hr class="docutils" />
<section id="who-you-gonna-call-data-stewards-to-the-rescue">
<h2>Who you gonna call? - Data Stewards to the rescue<a class="headerlink" href="#who-you-gonna-call-data-stewards-to-the-rescue" title="Link to this heading">#</a></h2>
<p>Vanessa Aphaia Fiona Fuchs, Jens Wendt, Maximilian Müller, Mohsen Ahmadi, Riccardo Massei, Cornelia Wetzker</p>
<p>Licensed CC-BY-4.0</p>
<p>The Data Steward Team of the NFDI4BIOIMAGE consortium presents themselves and the services (including the Helpdesk) that we offer.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Data Stewardship, Nfdi4Bioimage</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10730423">https://zenodo.org/doi/10.5281/zenodo.10730423</a></p>
</section>
<hr class="docutils" />
<section id="cidas-scalable-strategies-for-a-next-generation-of-fair-bioimaging">
<h2>[CIDAS] Scalable strategies for a next-generation of FAIR bioimaging<a class="headerlink" href="#cidas-scalable-strategies-for-a-next-generation-of-fair-bioimaging" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2025-01-23</p>
<p>Licensed CC-BY-4.0</p>
<p>Talk given at Georg-August-Universität Göttingen Campus Institute Data Science23rd January 2025
<a class="reference external" href="https://www.uni-goettingen.de/en/653203.html">https://www.uni-goettingen.de/en/653203.html</a></p>
<p>Tags: Nfdi4Bioimage</p>
<p><a class="reference external" href="https://zenodo.org/records/14716546">https://zenodo.org/records/14716546</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14716546">https://doi.org/10.5281/zenodo.14716546</a></p>
</section>
<hr class="docutils" />
<section id="cmcb-scalable-strategies-for-a-next-generation-of-fair-bioimaging">
<h2>[CMCB] Scalable strategies for a next-generation of FAIR bioimaging<a class="headerlink" href="#cmcb-scalable-strategies-for-a-next-generation-of-fair-bioimaging" title="Link to this heading">#</a></h2>
<p>Josh Moore</p>
<p>Published 2025-01-16</p>
<p>Licensed CC-BY-4.0</p>
<p>CMCB LIFE SCIENCES SEMINARSTechnische Universität Dresden16th January 2025
<a class="reference external" href="https://tu-dresden.de/cmcb/crtd/news-termine/termine/cmcb-life-sciences-seminar-josh-moore-german-bioimaging-e-v-society-for-microscopy-and-image-analysis-constance">https://tu-dresden.de/cmcb/crtd/news-termine/termine/cmcb-life-sciences-seminar-josh-moore-german-bioimaging-e-v-society-for-microscopy-and-image-analysis-constance</a>
 </p>
<p>Tags: Nfdi4Bioimage</p>
<p><a class="reference external" href="https://zenodo.org/records/14650434">https://zenodo.org/records/14650434</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14650434">https://doi.org/10.5281/zenodo.14650434</a></p>
</section>
<hr class="docutils" />
<section id="community-meeting-2024-overview-team-image-data-analysis-and-management">
<h2>[Community Meeting 2024] Overview Team Image Data Analysis and Management<a class="headerlink" href="#community-meeting-2024-overview-team-image-data-analysis-and-management" title="Link to this heading">#</a></h2>
<p>Susanne Kunis, Thomas Zobel</p>
<p>Published 2024-03-08</p>
<p>Licensed CC-BY-4.0</p>
<p>Overview of Activities of the Team Image Data Analysis and Management of German BioImaging e.V.
 </p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/10796364">https://zenodo.org/records/10796364</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.10796364">https://doi.org/10.5281/zenodo.10796364</a></p>
</section>
<hr class="docutils" />
<section id="elmi-2024-ai-s-dirty-little-secret-without">
<h2>[ELMI 2024]  AI’s Dirty Little Secret: Without<a class="headerlink" href="#elmi-2024-ai-s-dirty-little-secret-without" title="Link to this heading">#</a></h2>
<p>FAIR Data, It’s Just Fancy Math</p>
<p>Josh Moore, Susanne Kunis</p>
<p>Published 2024-05-21</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presented at the European Light Microscopy Initiative meeting in Liverpool (<a class="reference external" href="https://www.elmi2024.org/">https://www.elmi2024.org/</a>)</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/11235513">https://zenodo.org/records/11235513</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11235513">https://doi.org/10.5281/zenodo.11235513</a></p>
</section>
<hr class="docutils" />
<section id="elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">
<h2>[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math<a class="headerlink" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math" title="Link to this heading">#</a></h2>
<p>Josh Moore, Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presented at the European Light Microscopy Initiative meeting in Liverpool (<a class="reference external" href="https://www.elmi2024.org/">https://www.elmi2024.org/</a>)</p>
<p>Tags: Research Data Management, FAIR-Principles, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.11235512">https://zenodo.org/doi/10.5281/zenodo.11235512</a></p>
</section>
<hr class="docutils" />
<section id="swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard">
<h2>[SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard<a class="headerlink" href="#swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard" title="Link to this heading">#</a></h2>
<p>Josh Moore, Susanne Kunis</p>
<p>Licensed CC-BY-4.0</p>
<p>Poster presented at Semantic Web Applications and Tools for Health Care and Life Sciences (SWAT4HCLS 2023), Feb 13–16, 2023, Basel, Switzerland. NFDI4BIOIMAGE is a newly established German consortium dedicated to the FAIR representation of biological imaging data. A key deliverable is the definition of a semantically-compatible FAIR image object integrating RDF metadata with web-compatible storage of large n-dimensional binary data in OME-Zarr. We invite feedback from and collaboration with other endeavors during the soon-to-begin 5 year funding period.</p>
<p>Tags: Research Data Management, FAIR-Principles, Nfdi4Bioimage</p>
<p>Content type: Poster</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.7928332">https://zenodo.org/doi/10.5281/zenodo.7928332</a></p>
</section>
<hr class="docutils" />
<section id="short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">
<h2>[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure<a class="headerlink" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure" title="Link to this heading">#</a></h2>
<p>Christian Schmidt</p>
<p>Licensed CC-BY-4.0</p>
<p>Short Talk about the NFDI4BIOIMAGE consortium presented at the RDM in (Bio-)Medicine Information Event on April 10th, 2024, organized C³RDM &amp; ZB MED.</p>
<p>Tags: Research Data Management, Bioimage Analysis, Nfdi4Bioimage</p>
<p>Content type: Slides</p>
<p><a class="reference external" href="https://zenodo.org/doi/10.5281/zenodo.10939519">https://zenodo.org/doi/10.5281/zenodo.10939519</a></p>
</section>
<hr class="docutils" />
<section id="workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg">
<h2>[Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg<a class="headerlink" href="#workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg" title="Link to this heading">#</a></h2>
<p>Tom Boissonnet, Bettina Hagen, Susanne Kunis, Christian Schmidt, Stefanie Weidtkamp-Peters</p>
<p>Published 2024-11-18</p>
<p>Licensed CC-BY-4.0</p>
<p>Fit for OMERO: How imaging facilities and IT departments work together to enable RDM for bioimaging
Description:
Research data management (RDM) in bioimaging is challenging because of large file sizes, heterogeneous file formats and the variability of imaging methods. The image data management system OMERO (OME Remote Objects) allows for centralized and secure storage, organization, annotation, and interrogation of microscopy data by researchers. It is an internationally well-supported open-source software tool that has become one of the best-known image data management tools among bioimaging scientists. Nevertheless, the de novo setup of OMERO at an institute is a multi-stakeholder process that demands time, funds, organization and iterative implementation. In this workshop, participants learn how to begin setting up OMERO-based image data management at their institution. The topics include:</p>
<p>Stakeholder identification at the university / research institute
Process management, time line expectations, and resources planning
Learning about each other‘s perspectives on chances and challenges for RDM
Funding opportunities and strategies for IT and imaging core facilities
Hands-on: Setting up an OMERO server in a virtual machine environment</p>
<p>Target audience:
This workshop was directed at universities and research institutions who consider or plan to implement OMERO, or are in an early phase of implementation. This workshop was intended for teams from IT departments and imaging facilities to participate together with one person from the IT department, and one person from the imaging core facility at the same institution.
The trainers:</p>
<p>Prof. Dr. Stefanie Weidtkamp-Peters (Imaging Core Facility Head, Center for Advanced Imaging, Heinrich Heine University of Düsseldorf)
Dr. Susanne Kunis (Software architect, OMERO administrator, metadata specialist, University of Osnabrück)
Dr. Tom Boissonnet (OMERO admin and image metadata specialist, Center for Advanced Imaging, Heinrich Heine University of Düsseldorf)
Dr. Bettina Hagen (IT Administration and service specialist, Max Planck Institute for the Biology of Ageing, Cologne) 
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center (DKFZ), Heidelberg)</p>
<p>Time and place
The format was a two-day, in-person workshop (October 16-17, 2024). Location: Heidelberg, Germany
Workshop learning goals</p>
<p>Learn the steps to establish a local RDM environment fit for bioimaging data
Create a network of IT experts and bioimaging specialists for bioimage RDM across institutions
Establish a stakeholder process management for installing OMERO-based RDM
Learn from each other, leverage different expertise
Learn how to train users, establish sustainability strategies, and foster FAIR RDM for bioimaging at your institution</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/14178789">https://zenodo.org/records/14178789</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.14178789">https://doi.org/10.5281/zenodo.14178789</a></p>
</section>
<hr class="docutils" />
<section id="workshop-bioimage-data-management-and-analysis-with-omero">
<h2>[Workshop] Bioimage data management and analysis with OMERO<a class="headerlink" href="#workshop-bioimage-data-management-and-analysis-with-omero" title="Link to this heading">#</a></h2>
<p>Riccardo Massei, Michele Bortolomeazzi, Christian Schmidt</p>
<p>Published 2024-05-13</p>
<p>Licensed CC-BY-4.0</p>
<p>Here we share the material used in a workshop held on May 13th, 2024, at the German Cancer Research Center in Heidelberg (on-premise)
Description:Microscopy experiments generate information-rich, multi-dimensional data, allowing us to investigate biological processes at high spatial and temporal resolution. Image processing and analysis is a standard procedure to retrieve quantitative information from biological imaging. Due to the complex nature of bioimaging files that often come in proprietary formats, it can be challenging to organize, structure, and annotate bioimaging data throughout a project. Data often needs to be moved between collaboration partners, transformed into open formats, processed with a variety of software tools, and exported to smaller-sized images for presentation. The path from image acquisition to final publication figures with quantitative results must be documented and reproducible.
In this workshop, participants learn how to use OMERO to organize their data and enrich the bioimage data with structured metadata annotations.We also focus on image analysis workflows in combination with OMERO based on the Fiji/ImageJ software and using Jupyter Notebooks. In the last part, we explore how OMERO can be used to create publication figures and prepare bioimage data for publication in a suitable repository such as the Bioimage Archive.
Module 1 (9 am - 10.15 am): Basics of OMERO, data structuring and annotation
Module 2 (10.45 am - 12.45 pm): OMERO and Fiji
Module 3 (1.45 pm - 3.45 pm): OMERO and Jupyter Notebooks
Module 4 (4.15 pm - 6. pm): Publication-ready figures and data with OMERO
The target group for this workshopThis workshop is directed at researchers at all career levels who plan to or have started to use OMERO for their microscopy research data management. We encourage the workshop participants to bring example data from their research to discuss suitable metadata annotation for their everyday practice.
Prerequisites:Users should bring their laptops and have access to the internet through one of the following options:- eduroam- institutional WiFi- VPN connection to their institutional networks to access OMERO
Who are the trainers?
Dr. Riccardo Massei (Helmholtz-Center for Environmental Research, UFZ, Leipzig) - Data Steward for Bioimaging Data in NFDI4BIOIMAGE
Dr. Michele Bortolomeazzi (DKFZ, Single cell Open Lab, bioimage data specialist, bioinformatician, staff scientist in the NFDI4BIOIMAGE project)
Dr. Christian Schmidt (Science Manager for Research Data Management in Bioimaging, German Cancer Research Center, Heidelberg, Project Coordinator of the NFDI4BIOIMAGE project)</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/11350689">https://zenodo.org/records/11350689</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.11350689">https://doi.org/10.5281/zenodo.11350689</a></p>
</section>
<hr class="docutils" />
<section id="workshop-research-data-management-for-microscopy-and-bioimage-analysis">
<h2>[Workshop] Research Data Management for Microscopy and BioImage Analysis<a class="headerlink" href="#workshop-research-data-management-for-microscopy-and-bioimage-analysis" title="Link to this heading">#</a></h2>
<p>Christian Schmidt, Tom Boissonnet, Michele Bortolomeazzi, Ksenia Krooß</p>
<p>Published 2024-09-30</p>
<p>Licensed CC-BY-4.0</p>
<p>Research Data Management for Microscopy and BioImage Analysis</p>
<p>Introduction to BioImaging Research Data Management, NFDI4BIOIMAGE and I3D:bioChristian Schmidt /DKFZ Heidelberg
OMERO as a tool for bioimaging data managementTom Boissonnet /Heinrich-Heine Universität Düsseldorf
Reproducible image analysis workflows with OMERO software APIsMichele Bortolomeazzi /DKFZ Heidelberg
Publishing datasets in public archives for bioimage dataKsenia Krooß /Heinrich-Heine Universität Düsseldorf</p>
<p>Date &amp; Venue:Thursday, Sept. 26, 5.30 p.m.Haus 22 / Paul Ehrlich Lecture Hall (H22-1)University Hospital Frankfurt</p>
<p>Tags: Nfdi4Bioimage, Research Data Management</p>
<p><a class="reference external" href="https://zenodo.org/records/13861026">https://zenodo.org/records/13861026</a></p>
<p><a class="reference external" href="https://doi.org/10.5281/zenodo.13861026">https://doi.org/10.5281/zenodo.13861026</a></p>
</section>
<hr class="docutils" />
<section id="dmtxsamplecreator">
<h2>dmtxSampleCreator<a class="headerlink" href="#dmtxsamplecreator" title="Link to this heading">#</a></h2>
<p>SaibotMagd</p>
<p>Published 2023-06-06T11:52:14+00:00</p>
<p>Licensed APACHE-2.0</p>
<p>firefox extension: reads datamatrix code from camera and create a sample in an inventory to link it into an ELN.</p>
<p>Tags: Nfdi4Bioimage</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/SaibotMagd/dmtxSampleCreator">SaibotMagd/dmtxSampleCreator</a></p>
</section>
<hr class="docutils" />
<section id="ome2024-ngff-challenge">
<h2>ome2024-ngff-challenge<a class="headerlink" href="#ome2024-ngff-challenge" title="Link to this heading">#</a></h2>
<p>Will Moore, Josh Moore, sherwoodf, jean-marie burel, Norman Rzepka, dependabot[bot], JensWendt, Joost de Folter, Torsten St\xF6ter, AybukeKY, Eric Perlman, Tom Boissonnet</p>
<p>Published 2024-08-30T12:00:53+00:00</p>
<p>Licensed BSD-3-CLAUSE</p>
<p>Project planning and material repository for the 2024 challenge to generate 1 PB of OME-Zarr data</p>
<p>Tags: Sharing, Nfdi4Bioimage, Research Data Management</p>
<p>Content type: Github Repository</p>
<p><a class="github reference external" href="https://github.com/ome/ome2024-ngff-challenge">ome/ome2024-ngff-challenge</a></p>
<hr class="docutils" />
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./tags"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="neubias.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Neubias (27)</p>
      </div>
    </a>
    <a class="right-next"
       href="omero.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Omero (35)</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-cloud-optimized-storage-for-interactive-access-of-large-arrays">A Cloud-Optimized Storage for Interactive Access of Large Arrays</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#a-journey-to-fair-microscopy-data">A journey to FAIR microscopy data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#angebote-der-nfdi-fur-die-forschung-im-bereich-zoologie">Angebote der NFDI für die Forschung im Bereich Zoologie</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bio-image-data-strudel-for-workshop-on-research-data-management-in-tu-dresden-core-facilities">Bio-Image Data Strudel for Workshop on Research Data Management in TU Dresden Core Facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#challenges-and-opportunities-for-bio-image-analysis-core-facilities">Challenges and opportunities for bio-image analysis core-facilities</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#collaborative-working-and-version-control-with-git-hub">Collaborative Working and Version Control with git[hub]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#engineering-a-software-environment-for-research-data-management-of-microscopy-image-data-in-a-core-facility">Engineering a Software Environment for Research Data Management of Microscopy Image Data in a Core Facility</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#i3d-bio-s-omero-training-material-re-usable-adjustable-multi-purpose-slides-for-local-user-training">I3D:bio’s OMERO training material: Re-usable, adjustable, multi-purpose slides for local user training</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#key-value-pair-template-for-annotation-in-omero-for-light-microscopy-data-acquired-with-axioscan7-core-facility-cellular-imaging-cfci">Key-Value pair template for annotation in OMERO for light microscopy data acquired with AxioScan7 - Core Facility Cellular Imaging (CFCI)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#key-value-pair-template-for-annotation-of-datasets-in-omero-perikles-study">Key-Value pair template for annotation of datasets in OMERO (PERIKLES study)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi-daten-als-gemeinsames-gut-fur-exzellente-forschung-organisiert-durch-die-wissenschaft-in-deutschland">NFDI - Daten als gemeinsames Gut für exzellente Forschung, organisiert durch die Wissenschaft in Deutschland.</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage">NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-an-initiative-for-a-national-research-data-infrastructure-for-microscopy-data">NFDI4BIOIMAGE - An Initiative for a National Research Data Infrastructure for Microscopy Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-online-kick-off-2023">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis - Online Kick-Off 2023</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-national-research-data-infrastructure-for-microscopy-and-bioimage-analysis-conference-talk-the-pelagic-imaging-consortium-meets-helmholtz-imaging-5-10-2023-hamburg">NFDI4BIOIMAGE - National Research Data Infrastructure for Microscopy and BioImage Analysis [conference talk: The Pelagic Imaging Consortium meets Helmholtz Imaging, 5.10.2023, Hamburg]</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-data-management-illustrations-by-henning-falk">NFDI4BIOIMAGE data management illustrations by Henning Falk</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-perspective-for-a-national-bioimaging-standard">NFDI4BIOIMAGE: Perspective for a national bioimaging standard</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne Hackathon)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-ta3-hackathon-uoc-2023-cologne-hackathon-2023-github-repository">NFDI4Bioimage - TA3-Hackathon - UoC-2023 (Cologne-Hackathon-2023, GitHub repository)</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#nfdi4bioimage-calendar-2024-october-original-image">NFDI4Bioimage Calendar 2024 October; original image</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome-ngff-a-next-generation-file-format-for-expanding-bioimaging-data-access-strategies">OME-NGFF: a next-generation file format for expanding bioimaging data-access strategies</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome2024-ngff-challenge-results">OME2024 NGFF Challenge Results</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#omero-for-microscopy-research-data-management">OMERO for microscopy research data management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#overview-of-the-galaxy-omero-suite-upload-images-and-metadata-in-omero-using-galaxy">Overview of the Galaxy OMERO-suite - Upload images and metadata in OMERO using Galaxy</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-data-management-for-bioimaging-the-2021-nfdi4bioimage-community-survey">Research data management for bioimaging: the 2021 NFDI4BIOIMAGE community survey</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#setting-up-a-data-management-infrastructure-for-bioimaging">Setting up a data management infrastructure for bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#structuring-of-data-and-metadata-in-bioimaging-concepts-and-technical-solutions-in-the-context-of-linked-data">Structuring of Data and Metadata in Bioimaging: Concepts and technical Solutions in the Context of Linked Data</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-information-infrastructure-for-bioimage-data-i3d-bio-project-to-advance-fair-microscopy-data-management-for-the-community">The Information Infrastructure for BioImage Data (I3D:bio) project to advance FAIR microscopy data management for the community</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#the-role-of-helmholtz-centers-in-nfdi4bioimage-a-national-consortium-enhancing-fair-data-management-for-microscopy-and-bioimage-analysis">The role of Helmholtz Centers in NFDI4BIOIMAGE - A national consortium enhancing FAIR data management for microscopy and bioimage analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#thinking-data-management-on-different-scales">Thinking data management on different scales</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#towards-preservation-of-life-science-data-with-nfdi4bioimage">Towards Preservation of Life Science Data with NFDI4BIOIMAGE</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#welcome-to-bioimage-town">Welcome to BioImage Town</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#who-you-gonna-call-data-stewards-to-the-rescue">Who you gonna call? - Data Stewards to the rescue</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cidas-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[CIDAS] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cmcb-scalable-strategies-for-a-next-generation-of-fair-bioimaging">[CMCB] Scalable strategies for a next-generation of FAIR bioimaging</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#community-meeting-2024-overview-team-image-data-analysis-and-management">[Community Meeting 2024] Overview Team Image Data Analysis and Management</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without">[ELMI 2024]  AI’s Dirty Little Secret: Without</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#elmi-2024-ai-s-dirty-little-secret-without-fair-data-it-s-just-fancy-math">[ELMI 2024] AI’s Dirty Little Secret: Without FAIR Data, It’s Just Fancy Math</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#swat4hcls-2023-nfdi4bioimage-perspective-for-a-national-bioimage-standard">[SWAT4HCLS 2023] NFDI4BIOIMAGE: Perspective for a national bioimage standard</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#short-talk-nfdi4bioimage-a-consortium-in-the-national-research-data-infrastructure">[Short Talk] NFDI4BIOIMAGE - A consortium in the National Research Data Infrastructure</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-material-fit-for-omero-how-imaging-facilities-and-it-departments-work-together-to-enable-rdm-for-bioimaging-october-16-17-2024-heidelberg">[Workshop Material] Fit for OMERO - How imaging facilities and IT departments work together to enable RDM for bioimaging, October 16-17, 2024, Heidelberg</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-bioimage-data-management-and-analysis-with-omero">[Workshop] Bioimage data management and analysis with OMERO</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#workshop-research-data-management-for-microscopy-and-bioimage-analysis">[Workshop] Research Data Management for Microscopy and BioImage Analysis</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#dmtxsamplecreator">dmtxSampleCreator</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#ome2024-ngff-challenge">ome2024-ngff-challenge</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Robert Haase, Clément Caporal,... and the NFDI4BioImage Initiative
</p>

  </div>
  
  <div class="footer-item">
    

  </div>
  
  <div class="footer-item">
    <p class="last-updated">
  Last updated on 2025-03-18.
  <br/>
</p>
  </div>
  
  <div class="footer-item">
    
<div class="extra_footer">
  <p>
Copyright: Licensed <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY 4.0</a> unless mentioned otherwise. 
Contributions and feedback are welcome.
</p>

</div>
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>